
# Alphabetic List
## \#
* [2DMAP](\#-1.html#2dmap)
* [2LD](\#-1.html#2ld)
* [2SNP](\#-1.html#2snp)

## A
* [ACT](a-1.html#act), Analysis of Complex Traits
* [ADEGENET](a-1.html#adegenet)
* [ADMIXMAP](a-1.html#admixmap), Admixture mapping
* [AGEINF](a-1.html#ageinf)
* [ALBERT](a-1.html#albert), A Likelihood Based Estimation of Risk in Trios
* [ALLASS](a-1.html#allass)
* [ALLEGRO](a-1.html#allegro)
* [ALLELIX](a-1.html#allelix), Allelix - Paternity Linkage Analysis Online
* [ALOHOMORA](a-1.html#alohomora)
* [ALP](a-1.html#alp), Automated Linkage Preprocessor
* [Altrans](a-1.html#altrans)
* [ALTREE](a-1.html#altree)
* [AMELIA](a-1.html#amelia), Allele Matching Empirical Locus-specific Integrated Association test
* [ANALYZE](a-1.html#analyze)
* [ANCESTRY](a-1.html#ancestry)
* [ANCESTRYMAP](a-1.html#ancestrymap)
* [ANNOVAR](a-1.html#annovar), functional ANNOtation of genetic VARiants
* [ANTMAP](a-1.html#antmap)
* [APE](a-1.html#ape), Allelic Path Explorer
* [APL-OSA](a-1.html#apl-osa), Association in the Presence of Linkage with Ordered Subset Analysis
* [APM](a-1.html#apm), Affected Pedigree-Member Method
* [ARIEL](a-1.html#ariel), Accumulation of Rare variants Integrated and Extended Locus-specific test
* [ARLEQUIN](a-1.html#arlequin), * French translation of "Arlecchino"
* [ARP.GEE](a-1.html#arp.gee)
* [ASP/ASPSHARE](a-1.html#aspaspshare)
* [ASPEX](a-1.html#aspex), Affected Sib Pairs EXclusion map
* [ASSOCIATIONVIEWER](a-1.html#associationviewer)
* [AUTOSCAN](a-1.html#autoscan)

## B
* [BAMA](b-1.html#bama), Bayesian Analysis of Multilocus Association
* [BARS](b-1.html#bars), Bayesian Adaptive Regression Splines
* [BAYESFST](b-1.html#bayesfst)
* [BDGEN](b-1.html#bdgen)
* [BEAGLE](b-1.html#beagle)
* [BEAGLECALL](b-1.html#beaglecall)
* [BEAM](b-1.html#beam), Bayesian Epistasis Association Mapping
* [BETA](b-1.html#beta)
* [BIMBAM](b-1.html#bimbam), Bayesian IMputation-Based Association Mapping
* [BIOIDE](b-1.html#bioide), BIOmedical Integrated Discovery Environment
* [BIOLAD-DB](b-1.html#biolad-db)
* [BIRDSUITE](b-1.html#birdsuite)
* [BLADE](b-1.html#blade), Bayesian LinkAge DisEquilibrium mapping
* [BLOCK](b-1.html#block), Blocking Gibbs sampler for pedigree analysis
* [BMAPBUILDER](b-1.html#bmapbuilder)
* [BOOLD](b-1.html#boold)
* [BOOST](b-1.html#boost), BOolean Operation based Screening and Testing
* [BOOSTRAPPER](b-1.html#boostrapper)
* [BOREL](b-1.html#borel)
* [BOTTLENECK](b-1.html#bottleneck)
* [BPPH](b-1.html#bpph), Berkeley method for Perfect Phylogeney Haplotyping
* [BQTL](b-1.html#bqtl), Bayesian Quantitative Trait Locus mapping
* [BREAKDANCER](b-1.html#breakdancer)

## C
* [CAROL](c-1.html#carol), Combined Annotation scoRing toOL
* [CARTHAGENE](c-1.html#carthagene)
* [CASAVA](c-1.html#casava), Consensus Assessment of Sequence And VAriation
* [CASPAR](c-1.html#caspar), Computerized Affected Sibling Pair Analyzer and Reporter
* [CATS](c-1.html#cats), power Calculator for Association with Two Stage design
* [CC-QLS](c-1.html#cc-qls), Case-Control Quasi-Likelihood Score test
* [CCRAVAT](c-1.html#ccravat), Case-Control RAre Variant Analysis Tool
* [CCREL](c-1.html#ccrel), Case-Control association analysis with RELlated individuals
* [CEPH2CRI](c-1.html#ceph2cri)
* [CEPH2MAP](c-1.html#ceph2map)
* [CFC](c-1.html#cfc), Contribution, Inbreeding (F) and Coancestry
* [CHAPLIN](c-1.html#chaplin), Case-control HAPLotype INference package
* [CHECKHET](c-1.html#checkhet)
* [CHECKMATRIX](c-1.html#checkmatrix)
* [CHIAMO](c-1.html#chiamo)
* [CHIP2SPELL](c-1.html#chip2spell)
* [CHROMOSCAN](c-1.html#chromoscan)
* [CHROMOSEG](c-1.html#chromoseg)
* [CHROMSCAN](c-1.html#chromscan)
* [CLUMP](c-1.html#clump)
* [CLUSTAG](c-1.html#clustag)
* [CMAP](c-1.html#cmap), genetic and comparative maps
* [COMBIN](c-1.html#combin)
* [COMDS](c-1.html#comds)
* [COMPOSITELD](c-1.html#compositeld)
* [COPE](c-1.html#cope), COllaborative Pedigree drawing Environment
* [COVIBD](c-1.html#covibd)
* [CRIMAP](c-1.html#crimap)
* [CRIMAP-PVM](c-1.html#crimap-pvm), CRIMAP with Parallel Virtual Machine
* [CROSSFIND](c-1.html#crossfind)
* [CYRILLIC](c-1.html#cyrillic)

## D
* [DCHIP LINKAGE](d-1.html#dchip-linkage)
* [DGENE](d-1.html#dgene)
* [DHSMAP](d-1.html#dhsmap), Decay of Haplotype Sharing MAPping software
* [DINDEL](d-1.html#dindel)
* [DISENTANGLER](d-1.html#disentangler)
* [DMAP](d-1.html#dmap)
* [DMLE](d-1.html#dmle), Disease Mapping using Linkage disEquilibrium
* [DNABASER](d-1.html#dnabaser)
* [DOLINK](d-1.html#dolink)
* [DPPH](d-1.html#dpph), Direct method for Perfect Phylogeney Haplotyping

## E
* [EAGLET](e-1.html#eaglet), Efficient Analysis of Genetic Linkage: Testing and Estimation
* [EASYLINKAGE/EASYLINKAGE-PLUS](e-1.html#easylinkageeasylinkage-plus)
* [EASYPOP](e-1.html#easypop)
* [ECLIPSE](e-1.html#eclipse), Error Correcting Likelihoods In Pedigree Structure Estimation.
* [EDAC](e-1.html#edac), power calculator for Extremely Discordant And Concordant sibpairs
* [EH](e-1.html#eh), Estimating Haplotype-frequencies
* [EHAP](e-1.html#ehap)
* [EHP](e-1.html#ehp), Estimation of Haplotype frequencies from Pooled DNA samples
* [EHPLUS](e-1.html#ehplus)
* [EIGENSOFT/EIGENSTRAT](e-1.html#eigensofteigenstrat)
* [EM-DECODER](e-1.html#em-decoder)
* [EMLD](e-1.html#emld), EM estimation of haplotype frequencies and LD calculation
* [ENDOG](e-1.html#endog)
* [ENTROPY BLOCKER](e-1.html#entropy-blocker)
* [EPDT](e-1.html#epdt), Extended Pedigree Disequilibrium Test
* [EPISTACY](e-1.html#epistacy)
* [EQTL EXPLORER](e-1.html#eqtl-explorer)
* [ERPA](e-1.html#erpa), Extended Relative Pair Analysis
* [ET-TDT](e-1.html#et-tdt), Evolutionary Tree - Transmission Disequilibrium Test
* [ETDT](e-1.html#etdt), extended transmission/disequilibrium test
* [EVOKER](e-1.html#evoker)
* [EXOMEPICKS](e-1.html#exomepicks)

## F
* [FAMHAP](f-1.html#famhap)
* [FAMOZ](f-1.html#famoz), FAther/MOther
* [FASTEHPLUS](f-1.html#fastehplus), faster EH-PLUS
* [FASTER](f-1.html#faster), FAmily SmarT EliminatoR
* [FASTLINK](f-1.html#fastlink), faster version of LINKAGE
* [FASTMAP 1](f-1.html#fastmap-1), Fluorescent Allele-calling Software Toolkit: Microsatellite Automation Package
* [FASTMAP 2](f-1.html#fastmap-2)
* [FASTSLINK](f-1.html#fastslink), faster SLINK
* [FBAT](f-1.html#fbat), (haplotype) Family Based Association Test
* [FEST](f-1.html#fest)
* [FESTA](f-1.html#festa), Fragmented Exhaustive Seearch for TAgsnps
* [FINESSE](f-1.html#finesse)
* [FINETTI](f-1.html#finetti), de FINETTI generator
* [FIRSTORD](f-1.html#firstord)
* [FISHER](f-1.html#fisher)
* [FLOSS](f-1.html#floss), FLexible Ordered SubSet analysis

## G
* [G-MENDEL](g-1.html#g-mendel)
* [GAIA](g-1.html#gaia), Genetic Association Interaction Analysis
* [GAP](g-1.html#gap), genetic analysis package
* [GAS](g-1.html#gas), Genetic Analysis System
* [GAS2](g-1.html#gas2), statistical significance in Genomewide Association Studies in 2-stage
* [GASP](g-1.html#gasp), Genometric Analysis Simulation Program
* [GASSOC](g-1.html#gassoc), Genetic ASSOCiation analysis software for cases and parent
* [GATK](g-1.html#gatk), Genome Analysis ToolKit
* [GC/GCF](g-1.html#gcgcf), Genomic Control
* [GCHAP](g-1.html#gchap), Gene Counting method for HAPlotype analysis.
* [GDA](g-1.html#gda), Genetic Data Analysis
* [GEMS](g-1.html#gems), Genetic Epidemiology ModelS
* [GENABEL](g-1.html#genabel)
* [GENECLASS](g-1.html#geneclass)
* [GENECOUNTING](g-1.html#genecounting)
* [GENEFINDER](g-1.html#genefinder)
* [GENEHUNTER](g-1.html#genehunter)
* [GENEHUNTER++SAD](g-1.html#genehuntersad)
* [GENEHUNTER-IMPRINTING](g-1.html#genehunter-imprinting)
* [GENEHUNTER-MODSCORE](g-1.html#genehunter-modscore)
* [GENEHUNTER-PLUS](g-1.html#genehunter-plus)
* [GENEHUNTER-TWOLOCUS](g-1.html#genehunter-twolocus)
* [GENEPI.JAR](g-1.html#genepi.jar)
* [GENEPOOL](g-1.html#genepool)
* [GENEPOP](g-1.html#genepop)
* [GENERECON](g-1.html#generecon)
* [GENESPRING GT](g-1.html#genespring-gt)
* [GENETIC POWER CALCULATOR](g-1.html#genetic-power-calculator)
* [GENETSIM](g-1.html#genetsim)
* [GENEVAR](g-1.html#genevar), GENe Expression VARiation
* [GENEWEAVER](g-1.html#geneweaver)
* [GENIE](g-1.html#genie)
* [GENOCHECK](g-1.html#genocheck), GenoCheck
* [GENOGRAM-MAKER](g-1.html#genogram-maker)
* [GENOME](g-1.html#genome)
* [GENOMESIMLA](g-1.html#genomesimla)
* [GENOMIZER](g-1.html#genomizer)
* [GENOOM](g-1.html#genoom), GENetic Object Oriented Modeling in population genetics
* [GENOPROOF](g-1.html#genoproof)
* [GENTOOLS](g-1.html#gentools)
* [GERMLINE](g-1.html#germline)
* [GEST](g-1.html#gest), Gene-Environment interaction and STability
* [GEVALT](g-1.html#gevalt), GEnotype Visualization and ALgorithmic Tool
* [GGSD](g-1.html#ggsd), Generic Genetic Studies Database
* [GGT](g-1.html#ggt), Graphical GenoTyping package
* [GHOST](g-1.html#ghost)
* [GIST](g-1.html#gist), Genotype Ibd Sharing Test
* [GLFSINGLE/GLFTRIO/GLFMULTIPLES](g-1.html#glfsingleglftrioglfmultiples)
* [GLIDERS](g-1.html#gliders), Genome-wide LInkage DisEquilibrium Repository and Search engine
* [GLUE](g-1.html#glue), Genetic Linkage User Environment
* [GMA](g-1.html#gma), Genetic Mixture Analysis
* [GMCHECK](g-1.html#gmcheck)
* [GOLD](g-1.html#gold), Graphical Overview of Linkage Disequilibrium
* [GREGOR](g-1.html#gregor)
* [GRIDQTL](g-1.html#gridqtl)
* [GRONLOD](g-1.html#gronlod), GRONingen university LOR score calculation
* [GRR](g-1.html#grr), Graphical Representation of Relationships
* [GS-EM](g-1.html#gs-em), GenoSpectrum Expectation-Maximuzation
* [GSMA](g-1.html#gsma), Genome Search Meta Analysis
* [GTOOL](g-1.html#gtool)
* [GWAPOWER](g-1.html#gwapower), Genome-Wide Association POWER
* [GWASELECT](g-1.html#gwaselect)

## H
* [HAP 1](h-1.html#hap-1), haplotype resolution using imperfect phylogeny
* [HAP 2](h-1.html#hap-2), HAPlotype analysis of polymorphic markers
* [HAP-SAMPLE](h-1.html#hap-sample)
* [HAPAR](h-1.html#hapar), HAplotype inference by PARsimony
* [HAPASSOC](h-1.html#hapassoc)
* [HAPBLOCK](h-1.html#hapblock), a dynamical programming algorithm for haplotype block partitioning
* [HAPBLOCK 2](h-1.html#hapblock-2)
* [HAPGEN](h-1.html#hapgen)
* [HAPINFERX](h-1.html#hapinferx)
* [HAPLO 1](h-1.html#haplo-1), * Haplotyping with computation of conditional probabilities
* [HAPLO 2](h-1.html#haplo-2)
* [HAPLO.STAT](h-1.html#haplo.stat)
* [HAPLOBLOCK](h-1.html#haploblock)
* [HAPLOBLOCKFINDER](h-1.html#haploblockfinder)
* [HAPLOBUILD](h-1.html#haplobuild)
* [HAPLOCLUSTERS](h-1.html#haploclusters)
* [HAPLOPAINTER](h-1.html#haplopainter)
* [HAPLOPOOL](h-1.html#haplopool)
* [HAPLORE](h-1.html#haplore), HAPLOtype REconstruction in pedigrees
* [HAPLOREC](h-1.html#haplorec)
* [HAPLOT](h-1.html#haplot)
* [HAPLOTTER](h-1.html#haplotter)
* [HAPLOTYPE ESTIMATION](h-1.html#haplotype-estimation)
* [HAPLOTYPER](h-1.html#haplotyper)
* [HAPLOVIEW](h-1.html#haploview)
* [HAPLOVISUAL](h-1.html#haplovisual)
* [HAPLOWSER](h-1.html#haplowser), HAPLotype brOWSER
* [HAPMINER](h-1.html#hapminer)
* [HAPMIX](h-1.html#hapmix)
* [HAPMIXMAP](h-1.html#hapmixmap)
* [HAPPY](h-1.html#happy), reconstructing HAPlotYpes
* [HAPSCOPE](h-1.html#hapscope)
* [HAPSIMU](h-1.html#hapsimu)
* [HAPSTAT](h-1.html#hapstat)
* [HARDY](h-1.html#hardy)
* [HCLUST](h-1.html#hclust)
* [HEGESMA](h-1.html#hegesma), HEterogeneity and GEnome Search Meta Analysis
* [HELIXTREE](h-1.html#helixtree), HelixTree Genetic Association Software
* [HIT](h-1.html#hit), Haplotype Inference Technique
* [HOMOG/HOMOGM](h-1.html#homoghomogm)
* [HOMOZYGOSITYMAPPER](h-1.html#homozygositymapper)
* [HOTSPOTTER](h-1.html#hotspotter)
* [HPLUS](h-1.html#hplus)
* [HS-TDT](h-1.html#hs-tdt), Haplotype Sharing Transmission Disequilibrium Test
* [HTR](h-1.html#htr), Haplotype Trend Regression
* [HTSNPER](h-1.html#htsnper), Haplotype Tagging SNPer
* [HWESTRATA](h-1.html#hwestrata)
* [HWMET](h-1.html#hwmet)

## I
* [IBDREG](i-1.html#ibdreg)
* [IGG](i-1.html#igg), Integrate Genotypes for genome-wide Genetic studies
* [ILLUMINUS](i-1.html#illuminus)
* [ILR](i-1.html#ilr), Imprinting Likelihood Ratio
* [IMPUTE](i-1.html#impute)
* [INSEGT](i-1.html#insegt), INference of SEGregation Types
* [INTEGRAYEDMAP](i-1.html#integrayedmap)
* [INTERSNP](i-1.html#intersnp)

## J
* [J/QTL](j-1.html#jqtl)
* [JENTI](j-1.html#jenti)
* [JLIN](j-1.html#jlin), a Java based LINkage disequilibrium plotter
* [JOINMAP](j-1.html#joinmap), Software for the calculation of genetic linkage maps
* [JPSGCS](j-1.html#jpsgcs), Java Programs for Statistical Genetics and Computational Statistics

## K
* [KBAC](k-1.html#kbac), Kernal Based Association for Rare Variants
* [KIN](k-1.html#kin)
* [KING](k-1.html#king), Kinship-based INference for Gwas

## L
* [L-POP](l-1.html#l-pop)
* [LAMARC](l-1.html#lamarc), Likelihood Analysis with Metropolis Algorithm using Random Coalescence
* [LAMBDAA](l-1.html#lambdaa)
* [LAMP](l-1.html#lamp), Linkage and Association Modeling in Pedigrees
* [LAPSTRUCT](l-1.html#lapstruct), LAPlacian eigenfunctions learn population STRUCTure
* [LCP](l-1.html#lcp), Linkage Control Program
* [LDA](l-1.html#lda), Linkage Disequilibrium Analyzer
* [LDB/LDB+](l-1.html#ldbldb), Location DataBase
* [LDGROUP](l-1.html#ldgroup)
* [LDHAT](l-1.html#ldhat)
* [LDHEATMAP](l-1.html#ldheatmap)
* [LDMAP](l-1.html#ldmap)
* [LDMET](l-1.html#ldmet)
* [LDSELECT](l-1.html#ldselect)
* [LDSUPPORT](l-1.html#ldsupport)
* [LEA](l-1.html#lea), Likelihood-based Estimation of Admixture
* [LINKAGE - CEPH](l-1.html#linkage---ceph), LINKAGE CEPH (three-generation) pedigrees
* [LINKAGE - general pedigrees](l-1.html#linkage---general-pedigrees)
* [LINKAGE-IMPRINT](l-1.html#linkage-imprint), LINKAGE allowing for IMPRINITING
* [LINKBASE](l-1.html#linkbase)
* [LIPED](l-1.html#liped), Likelihoods in Pedigrees
* [LNKTOCRI](l-1.html#lnktocri)
* [LOCUSMAP](l-1.html#locusmap)
* [LOCUSZOOM](l-1.html#locuszoom)
* [LOGINSERM_ESTIHAPLO](l-1.html#loginserm_estihaplo)
* [LOH-LINKAGE](l-1.html#loh-linkage)
* [LOKI](l-1.html#loki)
* [LOT](l-1.html#lot), Linkage analysis of Ordinal Traits
* [LRP](l-1.html#lrp), Linkage Report Program
* [LRTAE](l-1.html#lrtae), Likelihood Ratio Test Allowing for Errors
* [LSP](l-1.html#lsp), Linkage Setup Program
* [LTSOFT](l-1.html#ltsoft)

## M
* [MACH](m-1.html#mach), MArkov Chain Haplotyper
* [MADELINE](m-1.html#madeline)
* [MADMAPPER](m-1.html#madmapper)
* [MAGIC](m-1.html#magic), Marker And Gene Interpolation and Correlation
* [MAIA](m-1.html#maia)
* [MAKEPED](m-1.html#makeped), MAKE PEDfiles
* [MALDSOFT](m-1.html#maldsoft)
* [MAMA](m-1.html#mama), Multivariate Association Mapping Algorithms
* [MANTEL-STRUCT](m-1.html#mantel-struct)
* [MAOS](m-1.html#maos), Meta-Analysis with Overlapping Subjects
* [MAP MANAGER QT](m-1.html#map-manager-qt)
* [MAP MANAGER QTX](m-1.html#map-manager-qtx)
* [MAP/MAP+/MAP+H/MAP2000](m-1.html#mapmapmaphmap2000), (MAP+H is the radiation Hybrid module of MAP+)
* [MAPCHART](m-1.html#mapchart), Windows software for the graphical presentation of linkage maps and QTLs
* [MAPCREATOR](m-1.html#mapcreator)
* [MAPDISTO](m-1.html#mapdisto)
* [MAPDRAW](m-1.html#mapdraw)
* [MAPINSPECT](m-1.html#mapinspect)
* [MAPL](m-1.html#mapl), MAPping and QTL analysis
* [MAPMAKER/EXP](m-1.html#mapmakerexp)
* [MAPMAKER/HOMOZ](m-1.html#mapmakerhomoz), homozygosity mapping
* [MAPMAKER/QTL](m-1.html#mapmakerqtl)
* [MAPMAKER/SIBS](m-1.html#mapmakersibs)
* [MAPPOP](m-1.html#mappop)
* [MAPQTL](m-1.html#mapqtl), Software for the calculation of QTL positions on genetic maps
* [MAREYMAP](m-1.html#mareymap)
* [MARGARITA](m-1.html#margarita)
* [MATLINK](m-1.html#matlink)
* [MCLEEPS](m-1.html#mcleeps), Monte Carlo Likelihood Estimation of Effective Population Size
* [MCQTL](m-1.html#mcqtl), Multi-Cross QTL mapping
* [MDBLOCKS](m-1.html#mdblocks), Minimum Description length method for haplotype BLOCKS
* [MDR](m-1.html#mdr), Multifactor Dimensionality Reduction
* [MDR-PDT](m-1.html#mdr-pdt), Multifactor Dimensionality Reduction and genotype Pedigree Disequilibrium Test
* [MECPM](m-1.html#mecpm), Maximum Entropy Conditional Probability Moldeling
* [MEGA2](m-1.html#mega2), a Manipulation Environment for Genetic Analyses
* [MEGASNPHUNTER](m-1.html#megasnphunter)
* [MENDEL](m-1.html#mendel)
* [MENDELSOFT](m-1.html#mendelsoft)
* [MERLIN](m-1.html#merlin), Multipoint Engine for Rapid Likelihood INference
* [METAL](m-1.html#metal), METa AnaLysis Helper
* [MFLINK](m-1.html#mflink), (nearly) model-free linkage analysis
* [MGA-MAPF2](m-1.html#mga-mapf2)
* [MIDAS](m-1.html#midas), Multiallelic Interallelic Disequilibrium Analysis Software
* [MILD](m-1.html#mild), MultIallelic Linkage Disequilibrium: a program for adjusted linkage disequilibrium (LD) calculations
* [MIM](m-1.html#mim), Multipoint Identical-by-descent Method
* [MINIMAC](m-1.html#minimac)
* [MINSAGE](m-1.html#minsage), MINimal SAmple size for GEnotypes
* [MITPENE](m-1.html#mitpene)
* [MIXSCORE](m-1.html#mixscore)
* [MKGST](m-1.html#mkgst)
* [MLBGH](m-1.html#mlbgh), Maximum-Likelihood-Binomial analysis of affected sib-pair and sibship data extended from the GENEHUNTER program
* [MLD](m-1.html#mld)
* [MLR-TAGGING](m-1.html#mlr-tagging), Multivariate Linear Regression TAGGING
* [MMDRAWER](m-1.html#mmdrawer)
* [MOLKIN](m-1.html#molkin)
* [MORGAN](m-1.html#morgan), MOnte caRlo Genetic ANalysis
* [MOSCPHASER](m-1.html#moscphaser), Mixture Of Cnv-Snp PHASER
* [MPDA](m-1.html#mpda), Microarray Pooled DNA Analyser
* [MRH](m-1.html#mrh), Minimum Recombinant Haplotype
* [MULTIDISEQ](m-1.html#multidiseq)
* [MULTIMAP](m-1.html#multimap)
* [MULTIMAPPER](m-1.html#multimapper)
* [MULTIMAPPER/OUTBRED](m-1.html#multimapperoutbred)
* [MULTIPOPTAGSELECT](m-1.html#multipoptagselect)
* [MULTIQTL](m-1.html#multiqtl)
* [MULTISIM](m-1.html#multisim)
* [MUTAGENESYS](m-1.html#mutagenesys)

## N
* [NOCOM](n-1.html#nocom)
* [NOPAR](n-1.html#nopar)
* [NUCULAR](n-1.html#nucular)

## O
* [OLORIN](o-1.html#olorin)
* [ONEMAP](o-1.html#onemap)
* [OSA](o-1.html#osa), Ordered Subset Analysis
* [OSIRIS](o-1.html#osiris)

## P
* [P_ACT](p-1.html#p_act), P-values: Adjustment for Correlated Tests
* [PANGAEA](p-1.html#pangaea), Pedigree ANalysis for Genetics (And Epidemiological Attributes)
* [PAP](p-1.html#pap), Pedigree Analysis Package
* [PARENTE](p-1.html#parente)
* [PASS PEDIGREE](p-1.html#pass-pedigree)
* [PATH](p-1.html#path)
* [PAWE](p-1.html#pawe), Power for Association With Error
* [PAWE-3D](p-1.html#pawe-3d), Power for Association With Error in 3D
* [PBAT](p-1.html#pbat), Power calculation of family-Based Association Tests
* [PDA](p-1.html#pda), Pooled DNA Analyzer
* [PDPSYS](p-1.html#pdpsys), Pedigree Data Processing SYStem
* [PDT](p-1.html#pdt), pedigree disequilibrium test
* [PED](p-1.html#ped), PEdigree Drawing software
* [PEDAGREE](p-1.html#pedagree)
* [PEDCHECK](p-1.html#pedcheck)
* [PEDFIDDLER](p-1.html#pedfiddler)
* [PEDHUNTER](p-1.html#pedhunter)
* [PEDIGRAPH](p-1.html#pedigraph)
* [PEDIGREE-VISUALIZER](p-1.html#pedigree-visualizer)
* [PEDIGREE/DRAW](p-1.html#pedigreedraw)
* [PEDIGREEQUERY](p-1.html#pedigreequery)
* [PEDJAVA](p-1.html#pedjava), PedJava
* [PEDMANAGER](p-1.html#pedmanager)
* [PEDPACK](p-1.html#pedpack)
* [PEDPEEL](p-1.html#pedpeel)
* [PEDPHASE](p-1.html#pedphase)
* [PEDPLOT](p-1.html#pedplot)
* [PEDRAW/WPEDRAW](p-1.html#pedrawwpedraw), Pedigree Drawing/ Window Pedigree Drawing (MS-Window and X-Window version of PEDRAW)
* [PEDSCRIPT](p-1.html#pedscript)
* [PEDSTATS](p-1.html#pedstats)
* [PEDSYS](p-1.html#pedsys)
* [PEDVIZAPI](p-1.html#pedvizapi), PEDigree VIsualiZation API
* [PEER](p-1.html#peer)
* [PELICAN](p-1.html#pelican), Pedigree Editor for LInkage Computer ANalysis
* [PHASE](p-1.html#phase)
* [PIAGE](p-1.html#piage), Power of Indirect Association Studies of Gene-Environment Interactions
* [PL-EM](p-1.html#pl-em), Partition-Ligation EM algorithm for haplotype inference with single nucleotide polymorphisms
* [PLABQTL](p-1.html#plabqtl), PLAnt Breeding QTL analysis
* [PLABSIM](p-1.html#plabsim), PLAnt Breeding SIMulation
* [PLAYPUS](p-1.html#playpus)
* [PLINK](p-1.html#plink)
* [PLINK/SEQ](p-1.html#plinkseq)
* [POINTER](p-1.html#pointer)
* [POLYMORPHISM](p-1.html#polymorphism)
* [POLYMUTT](p-1.html#polymutt), POLYmorphism and de novo MUTaTion call in families with sequencing data
* [POLYPHEN](p-1.html#polyphen), POLYmorphism PHENotyping
* [POOL_STR](p-1.html#pool_str), POOLing alleles of Short Tandem Repeat markers
* [POOLSCORE](p-1.html#poolscore)
* [POPDIST](p-1.html#popdist)
* [POPGEN](p-1.html#popgen)
* [POWER](p-1.html#power)
* [POWERMARKER](p-1.html#powermarker)
* [POWERTRIM](p-1.html#powertrim)
* [POWQ](p-1.html#powq)
* [POWTEST](p-1.html#powtest)
* [PREPLINK](p-1.html#preplink), MAKE PEDfiles
* [PREST](p-1.html#prest), Pedigree RElationship Statistical Test
* [PRESTO](p-1.html#presto)
* [PROBMAX](p-1.html#probmax)
* [PROC QTL](p-1.html#proc-qtl)
* [PROFILER](p-1.html#profiler)
* [PROGENY](p-1.html#progeny), Progeny Software, LLC
* [PRT](p-1.html#prt), Pedigree Reconstruction Tools
* [PSAT](p-1.html#psat), Population Stratification Association Test
* [PSEUDO](p-1.html#pseudo)
* [PSEUDOMARKER](p-1.html#pseudomarker)
* [PSEUDOMARKER.M](p-1.html#pseudomarker.m)
* [PYPOP](p-1.html#pypop), PYthon for POPulation genetics

## Q
* [QGENE](q-1.html#qgene)
* [QMSIM](q-1.html#qmsim), Qtl and Marker SIMulator
* [QTDT](q-1.html#qtdt), Quantitative (Trait) Transmission/Disequilibrium Test
* [QTL CAFE](q-1.html#qtl-cafe)
* [QTL Cartographer](q-1.html#qtl-cartographer)
* [QTL EXPRESS](q-1.html#qtl-express)
* [QTL-ALL](q-1.html#qtl-all), QTL Analysis and Linkage Library
* [QTLNETWORK](q-1.html#qtlnetwork)
* [QU-GENE](q-1.html#qu-gene), QUantitative GENEtics
* [QUANTO](q-1.html#quanto)
* [QUTIE](q-1.html#qutie), Case-Control RAre Variant Analysis Tool

## R
* [R/ADEGENET](r-1.html#radegenet)
* [R/ARP.GEE](r-1.html#rarp.gee)
* [R/BARS](r-1.html#rbars)
* [R/COMPOSITELD](r-1.html#rcompositeld)
* [R/COVIBD](r-1.html#rcovibd)
* [R/EHP](r-1.html#rehp)
* [R/ENTROPY_BLOCKER](r-1.html#rentropy_blocker)
* [R/FEST](r-1.html#rfest)
* [R/GAP](r-1.html#rgap), R/Genetic Analysis Package
* [R/GC R/GCF](r-1.html#rgc-rgcf)
* [R/GENABEL](r-1.html#rgenabel)
* [R/GWAPOWER](r-1.html#rgwapower)
* [R/HAPASSOC](r-1.html#rhapassoc)
* [R/HCLUST](r-1.html#rhclust)
* [R/IBDREG](r-1.html#ribdreg)
* [R/LAPSTRUCT](r-1.html#rlapstruct)
* [R/LDGROUP](r-1.html#rldgroup)
* [R/LDHEATMAP](r-1.html#rldheatmap)
* [R/LUCA](r-1.html#rluca), Likelihood Under Covariate Assumptions
* [R/META](r-1.html#rmeta)
* [R/METASIM](r-1.html#rmetasim), METApopulation SIMulation
* [R/ONEMAP](r-1.html#ronemap)
* [R/PIAGE](r-1.html#rpiage)
* [R/POOLSCORE](r-1.html#rpoolscore)
* [R/POPGEN](r-1.html#rpopgen)
* [R/QTL](r-1.html#rqtl)
* [R/QTLBIM](r-1.html#rqtlbim)
* [R/QTLDESIGN](r-1.html#rqtldesign)
* [R/SNP.PLOTTER](r-1.html#rsnp.plotter)
* [R/SPECTRAL-GEM](r-1.html#rspectral-gem), SPECTRAL graph approach for GEnetic Matching
* [R/STEPWISE](r-1.html#rstepwise)
* [R/TDTHAP](r-1.html#rtdthap)
* [R/WEIGHTED_FDR](r-1.html#rweighted_fdr)
* [RC-TDT](r-1.html#rc-tdt), Reconstruction-Combined Transmission Disequilibrium Test
* [REAPER](r-1.html#reaper)
* [RECORD](r-1.html#record), REcombination Counting and ORDering
* [RELATIVE](r-1.html#relative)
* [RELATIVEFINDER](r-1.html#relativefinder)
* [RELCHECK](r-1.html#relcheck)
* [RELPAIR](r-1.html#relpair)
* [RELTYPE](r-1.html#reltype)
* [RHMAP](r-1.html#rhmap), Radiation Hybrid Map
* [RHMAPPER](r-1.html#rhmapper)
* [RISCALW](r-1.html#riscalw), RISk CALculation in Windows
* [ROMPREV](r-1.html#romprev), Regression of Offspring on Mid-Parent (REVised)
* [ROSATTA SYLLEGO SYSTEM](r-1.html#rosatta-syllego-system)
* [RTDT](r-1.html#rtdt), Robust Transmission/Disequilibrium Test
* [RVTESTS](r-1.html#rvtests), Rare Variants TESTS

## S
* [SAGE](s-1.html#sage), Statistical Analysis for Genetic Epidemiology
* [SAMTOOLS](s-1.html#samtools), Sequence Alignment/Map TOOLS
* [SAS/GENETICS](s-1.html#sasgenetics)
* [SASGENE](s-1.html#sasgene)
* [SASQUANT](s-1.html#sasquant)
* [SCORE-SEQ](s-1.html#score-seq), SCORE-type tests for detecting disease associations with rare variants in SEQuencing Studies
* [SCOUT](s-1.html#scout)
* [SDMINP](s-1.html#sdminp), Step-Down MIN P-value
* [SEGPATH](s-1.html#segpath), SEGregation analysis and PATH analysis
* [SELSIM](s-1.html#selsim)
* [SEQUENCE LD/SEQUENCE LDHOT](s-1.html#sequence-ldsequence-ldhot)
* [SGS](s-1.html#sgs)
* [SIB-PAIR](s-1.html#sib-pair)
* [SIBERROR](s-1.html#siberror), SibError
* [SIBLINK](s-1.html#siblink)
* [SIBMED](s-1.html#sibmed), SIBling Mutation and Error Detection
* [SIBSIM](s-1.html#sibsim)
* [SIFT](s-1.html#sift), Sorting Intolerant From Tolerant
* [SILCLOD](s-1.html#silclod), SIgnificance Levels and Critical LODs
* [SIMCOAL](s-1.html#simcoal), SIMulate COAlescence
* [SIMHAP](s-1.html#simhap), SIMulation-based HAPlotypic analysis
* [SIMIBD](s-1.html#simibd)
* [SIMLA](s-1.html#simla), SIMulation of pedigree data for Linkage and Association studies
* [SIMLA 3](s-1.html#simla-3)
* [SIMLINK](s-1.html#simlink)
* [SIMM](s-1.html#simm)
* [SIMPED](s-1.html#simped)
* [SIMPLE](s-1.html#simple), Sequential Imputation for MultiPoint Linkage Estimation
* [SIMULAPLOT](s-1.html#simulaplot)
* [SIMULATE](s-1.html#simulate)
* [SIMUPOP](s-1.html#simupop)
* [SIMWALK](s-1.html#simwalk)
* [SKAT](s-1.html#skat), SNP-set (Sequence) Kernel Association Test
* [SLINK](s-1.html#slink)
* [SMOOTH](s-1.html#smooth)
* [SNAP](s-1.html#snap), Suite of Nucleotide Analysis Programs
* [SNAP 2](s-1.html#snap-2), SNp Annotation and Proxy search
* [SNAP 3](s-1.html#snap-3), Simulation of sNp haplotype data And Phenotypic traits
* [SNIPPEEP](s-1.html#snippeep)
* [SNP ASSISTANT](s-1.html#snp-assistant)
* [SNP CHART](s-1.html#snp-chart)
* [SNP HITLINK](s-1.html#snp-hitlink), SNP HIgh-Throughput LINKage analysis system
* [SNP-HWE](s-1.html#snp-hwe)
* [SNP.PLOTTER](s-1.html#snp.plotter)
* [SNPALYZE](s-1.html#snpalyze)
* [SNPEM](s-1.html#snpem), SNP Expectation Maximization
* [SNPFILE](s-1.html#snpfile)
* [SNPHAP](s-1.html#snphap)
* [SNPHARVESTER](s-1.html#snpharvester)
* [SNPLINK](s-1.html#snplink)
* [SNPMSTAT](s-1.html#snpmstat), SNP Missing data STATistics
* [SNPP](s-1.html#snpp), Single Nucleotide Polymorphism Processor
* [SNPSTATS](s-1.html#snpstats), SNP STATisticS
* [SNPTEST](s-1.html#snptest)
* [SOAP](s-1.html#soap), Short Oligonucleotide Analysis Package
* [SOLAR](s-1.html#solar), Sequential Oligogenic Linkage Analysis Routines
* [SPAM](s-1.html#spam), Statistics Program for Analysing Mixtures
* [SPECTRAL-GEM](s-1.html#spectral-gem)
* [SPERM](s-1.html#sperm)
* [SPERMSEG](s-1.html#spermseg)
* [SPIP](s-1.html#spip), Simulate Pedigree In Population
* [SPLAT](s-1.html#splat), Sib Pair Linkage Analysis Testing
* [SPLINK](s-1.html#splink), affected Sib Pairs LINKage analysis
* [SPREG](s-1.html#spreg), Secondary Phenotype REGression analysis
* [SQTDT/SPDT](s-1.html#sqtdtspdt)
* [SQTL](s-1.html#sqtl), Semiparametric QTL mapping in general pedigrees
* [sQTLSeekerR](s-1.html#sqtlseekerr)
* [SSAHASNP](s-1.html#ssahasnp), Sequence Search and Alignment by Hashing Algorithm for SNP detection
* [START](s-1.html#start)
* [STEPC](s-1.html#stepc), STatistical Explanation for Positional Cloning
* [STEPWISE](s-1.html#stepwise)
* [STRAT](s-1.html#strat)
* [STRUCTURE](s-1.html#structure)
* [SUMSTAT](s-1.html#sumstat)
* [SUP](s-1.html#sup), Slink Utility Program
* [SUPERLINK](s-1.html#superlink)
* [SVA](s-1.html#sva), Sequence Variant Analyzer
* [SVCC](s-1.html#svcc), Semiparametric Variance-Component for Censored data
* [SWEEP](s-1.html#sweep)
* [SYZYGY](s-1.html#syzygy)

## T
* [TAGGER](t-1.html#tagger)
* [TAGIMPUTE](t-1.html#tagimpute), TAGging-snp based IMPUTATE
* [TAGSNP](t-1.html#tagsnp)
* [TAGSTER](t-1.html#tagster)
* [TASSEL](t-1.html#tassel), Trait Analysis by aSSociation, Evolution, and Linkage
* [TDT-AE](t-1.html#tdt-ae), Transmission Disequilibrium Test Allowing for Errors
* [TDT-PC](t-1.html#tdt-pc), Transmission Disequilibrium Test Power Calculator
* [TDT/S-TDT](t-1.html#tdts-tdt), Transmission Disequilibrium Test and Sib Transmission Disequilibrium Test
* [TDTASP](t-1.html#tdtasp), Power and Sample-Size Calculations for the TDT and ASP Tests
* [TDTHAP](t-1.html#tdthap)
* [TDTPOWER](t-1.html#tdtpower)
* [TFPGA](t-1.html#tfpga), Tools For Population Genetic Analyses
* [THESIAS](t-1.html#thesias), Testing Haplotype EffectS In Association Studies
* [TKMAP](t-1.html#tkmap)
* [TLINKAGE](t-1.html#tlinkage), two-locus trait linkage programs
* [TOMCAT](t-1.html#tomcat)
* [TRANSMIT](t-1.html#transmit)
* [TRAP](t-1.html#trap), Tiled Regression Analysis Package
* [TREELD](t-1.html#treeld)
* [TREESCAN](t-1.html#treescan)
* [TREESELECT](t-1.html#treeselect)
* [TRIMHAP](t-1.html#trimhap), TRIMmed-HAPlotype (previously named HAL: Haplotype ALgorithm)
* [TUNA](t-1.html#tuna), Testing UNtyped Alleles
* [TWOLOC](t-1.html#twoloc)

## U
* [UMAKE](u-1.html#umake)
* [UNKNOWN](u-1.html#unknown)
* [UNPHASED](u-1.html#unphased)
* [UTIL](u-1.html#util), statistical genetics UTILity programs

## V
* [VAAST](v-1.html#vaast), Variant Annotation, Analysis and Search Tool
* [Variant Association Tools](v-1.html#variant-association-tools)
* [Variant Tools](v-1.html#variant-tools)
* [VARSCAN](v-1.html#varscan)
* [VG](v-1.html#vg), Visual Genotype
* [VH](v-1.html#vh), visual haplotype
* [VITESSE](v-1.html#vitesse), VITESSE (means "speed" in French)

## W
* [WEB-PREPLINK](w-1.html#web-preplink)
* [WEBQTL](w-1.html#webqtl)
* [WEIGHTED_FDR](w-1.html#weighted_fdr)
* [WGCNA](w-1.html#wgcna), Weighted Gene Co-expression Network Analysis
* [WHAIT](w-1.html#whait), Weighted Haplotype test And an Imputation-based Test.
* [WHAP](w-1.html#whap)
* [WHICHRUN](w-1.html#whichrun)

## Z
* [ZAPLO](z-1.html#zaplo)
