*   ACT

    *   full name: Analysis of Complex Traits

    *   version: 1.1 (Mar 17, 2000)

    *   description: It contains the following modules: ibd, calculates the proportion of gene shared identical by decent for a nuclear family; ibdn, (modified program of ERPA), which implements a method for assessing increased-allele sharing between all pairs of affected relatives within a pedigree; multic, multivariate analysis for complex traits; ml, estimation of variance components using maximum likelihood; ql, estimation of variance components using quasi likelihood; relcov, generates first degree relationship coefficients for extended families; sim2s, the simulation program that was used to test ACT; cage, Cohort Analysis for Genetic Epidemiology; gh: GeneHunter, heavily modified to assist multipoint calculation using multic; TDT: TDT programs written in SAS; gcc and f77 compilers are necessary. Executable programs are included for compatible operating systems, i.e., Solaris2.6.

    *   authors: Christopher I. Amos (email: camos@request.mdacc.tmc.edu), Mariza de Andrade (email: mandrade@request.mdacc.tmc.edu), JianFang Chen (email: cjf@request.mdacc.tmc.edu)

    *   web/ftp: http://www.epigenetic.org/Linkage/act.html, http://www.epigenetic.org/Linkage/act.tar.gz

    *   source code language: FORTRAN77, C++

    *   operating systems: UNIX(Solaris 2.4/..)

    *   executables: ibd, ml, ql, he, ibdn, multic

    *   reference: Amos, et al (1996), "Assessing genetic linkage and association with robust components of variance approaches", Annals of Human Genetics, 60:143-160.

    *   Amos (1994), "Robust variance-components approach for assessing genetic linkage in pedigrees", American Journal of Human Genetics, 54:535-543. [[abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1918121)]

    *   de Andreade, Amos, Thiel (1999), "Methods to estimate genetic parameters for quantitative traits", Genetic Epidemiology, 17(1):64-76.

*   ADEGENET (see also R/ADEGENET)

*   ADMIXMAP

    *   full name: Admixture mapping

    *   version: 3.7 (2007)

    *   description: ADMIXMAP is a general-purpose program for modelling admixture, using marker genotypes and trait data on a sample of individuals from an admixed population (such as African-Americans), where the markers have been chosen to have extreme differentials in allele frequencies between two or more of the ancestral populations between which admixture has occurred. The main difference between ADMIXMAP and classical programs for estimation of admixture such as ADMIX is that ADMIXMAP is based on a multilevel model for the distribution of individual admixture in the population and the stochastic variation of ancestry on hybrid chromosomes. This makes it possible to model the associations of ancestry between linked marker loci, and the association of a trait with individual admixture or with ancestry at a linked marker locus.

    *   authors: Clive Hoggart, Nigel Wetters, David Clayton, David O'Donnell (email: david.odonnell@ucd.ie), and Paul McKeigue (email: paul.mckeigue@ucd.ie)

    *   web/ftp: http://www.homepages.ed.ac.uk/pmckeigu/admixmap/index.html

    *   source code language: C++

    *   operating systems: Linux, MS-Windows

    *   executables: admixmap, admixmap.pl, and some R scripts to process the output

    *   reference: PM McKeigue (2005), "Prospects for admixture mapping of complex traits", American Journal of Human Genetics, 76(1):1-7

    *   CJ Hoggart, MD Shriver, RA Kittles, DG Clayton, PM McKeigue (2004), "Design and analysis of admixture mapping studies", American Journal of Human Genetics, 74(5):965-978

    *   CJ Hoggart, EJ Parra, MD Shriver, C Bonilla, RA Kittles, DC Clayton, PM McKeigue (2003), "Control of confounding of genetic associations in stratified populations", American Journal of Human Genetics, 72(6):1492-1504

    *   PM McKeigue, J Carpenter, EJ Parra, MD Shriver (2000), "Estimation of admixture and detection of linkage in admixed populations by a Bayesian approach: application to African-American populations", Annals of Human Genetics, 64:171-186.

*   AGEINF

    *   full name:

    *   version:

    *   description: inference of the age of a rare, selectively-neutral mutation

    *   authors: Jinko Graham

    *   web/ftp: http://www.math.sfu.ca/~jgraham/Papers/Programs/AgeCode/

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Graham, Thompson (2000), "A coalescent model of ancestry for a rare allele", Genetics, 156:375-384.

*   ALBERT

    *   full name: A Likelihood Based Estimation of Risk in Trios

    *   version: 1.0 (May 2006)

    *   description: ALBERT is a program that estimates genotype relative risks, genotyping error rates and population risk allele frequencies from marker genotype data in case-parent trios. ALBERT uses the distribution of trio marker genotypes to compute maximum likelihood estimates for the parameters.

    *   authors: Adele Mitchell, Elizabeth A Thompson (University of Washington)

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/Albert/albert.shtml

    *   source code language: C

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference:

*   ALLASS

    *   full name:

    *   version: February 2002

    *   description:

    *   authors: A Collins (email: arc@soton.ac.uk), NE Morton (University of Southampton, UK)

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ALLASS

    *   source code language: C

    *   operating systems: UNIX (SUN..)

    *   executables: allass

    *   reference: Collins, Morton (1998), "Mapping a disease locus by allelic association", Proceedings of the National Academy of Sciences , 95:1741-1745; online manual

*   ALLEGRO

    *   full name:

    *   version: 2.0 (October 2005)

    *   description: a faster version of GENEHUNTER and Allegro 1.2 (several degrees of increase of speed, can handle bigger families, up to 50 bits)

    *   authors: Daniel F Gudbjartsson, TA Thorvaldsson, G Gunnarsson, Augustine Kong, A Ingolfsdottir (DeCode Genetics, Inc. Iceland) send email to: allegro@decode.is

    *   web/ftp: http://www.decode.com/software/

    *   source code language: C++

    *   operating systems:

    *   executables:

    *   reference: D Gudbjartsson, K Jonasson, CA Kong (1999), "Fast multipoint linkage calculation with Allegro", American Journal of Human Genetics, 65(suppl):A60.

    *   Gudbjartsson, Jonasson, Frigge, Kong (2000) "Allegro, a new computer program for multipoint linkage analysis", Nature Genetics, 25:12-13.

    *   Gudbjartsson, Thorvaldsson, Gunnarsson, Kong, Ingolfsdottir (2004), "Decision diagram based multipoint linkage analysis: ALLEGRO2", American Journal of Human Genetics, 75(suppl):?.

    *   Gudbjartsson, Thorvaldsson, Kong, Gunnarsson, Ingolfsdottir (2005), "Allegro version 2", Nature Genetics, 37:1015-1016\. **PLEASE CITE THIS REFERENCE WHEN USING THE ALLEGRO2 PROGRAM**

*   ALLELIX

    *   full name: Allelix - Paternity Linkage Analysis Online

    *   version: December 2001

    *   description: One can enter the alleles of commonly used STR by clicking the mouse. The algorithm calculates the paternity index and the Essen-Moeller probability of kinship for the deficiency- and the trio case. Everybody can use the network-software online after registering. The usage on the internet is free. Academic users can ask me to unlock an option to display the details (formulas/frequencies etc.) and to have an export-funktion to MS Word. The program is in German and (non-professional) English. An expansion to other languages is easy, if somebody helps us with the translation. For those who are interested to have the software running on their own intranet (for database security reasons) an individual agreement can be found.

    *   authors: Thomas Krahn (biotix) (email: tk@biotix.de), Stefan Zinke (Software Erstellung- und Beratung, Germany), Dr. Michael Kraft (biotix)

    *   web/ftp: http://www.allelix.net (German version is: http://www.allelix.de)

    *   source code language: C++, Java Script

    *   operating systems: MS-Windows (2000 &IIS/ME & Personal Webserver)

    *   executables: allelix.dll

    *   reference:

*   ALOHOMORA

    *   full name:

    *   version: 0.33 (October 2009)

    *   description: designed to facilitate genome-wide linkage studies performed with high-density single nucleotide polymorphism (SNP) marker panels such as the Affymetrix GeneChip(R) Human Mapping 10K Array.

    *   authors: Franz Ruschendorf (email: fruesch@mdc-berlin.de), Peter Nurnberg

    *   web/ftp: http://gmc.mdc-berlin.de/alohomora/

    *   source code language: perl

    *   operating systems: MS-Window, Linux, UNIX (Solaris/..)

    *   executables:

    *   reference: Ruschendorf, Nurnberg (2005), "ALOHOMORA: a tool for linkage analysis using 10K SNP array data", Bioinformatics, 21(9):2123-2125.

*   ALP

    *   full name: Automated Linkage Preprocessor

    *   version: v1.40 (August 1995)

    *   description: ALP is a Microsoft Windows application designed to analyse microsatellite DNA fragments separated on an Automated Laser Fluorescence sequencer.

    *   authors: Alastair Brown (email: alastair.brown@hgu.mrc.ac.uk), Daryll Green, Steve Woess (MRC Human Genetics Unit, Edinburgh, UK).

    *   web/ftp: http://www.hgu.mrc.ac.uk/Softdata/ALP/

    *   ftp://ftp.hgu.mrc.ac.uk/pub/ALP

    *   source code language: C

    *   operating systems: MS-Windows (3.1/95/NT)

    *   executables: alpw.exe

    *   reference: Mansfield, Brown, Green, Carothers, Morris, Evans, Wright (1994), "Automation of genetic linkage analysis using fluorescent microsatellite markers", Genomics, 24(2):225-233.

*   ALTREE

    *   full name:

    *   version:

    *   description: ALTREE performs these two phylogeny-based analysis: (1) it tests the association between a candidate gene and a disease; (2) it pinpoints markers (SNPs) that are putative disease susceptibility loci

    *   authors: Claire Bardel (email: bardel@vjf.inserm.fr), Vincent Danjean, Jean-Pierre Hugot, Pierre Darlu, Emmanuelle Génin

    *   web/ftp: http://claire.bardel.free.fr/software.html

    *   source code language:

    *   operating systems:

    *   executables: altree, altree-convert, altree-add-s

    *   reference: Bardel, Danjean, Hugot, Darlu, Genin (2005), "On the use of haplotype phylogeny to detect disease susceptibility loci", BMC Genetics, 6:24.

*   AMELIA

    *   full name: Allele Matching Empirical Locus-specific Integrated Association test

    *   version:

    *   description: AMELIA is a program that employs allele matching to analyse the effects of rare variants within a specific locus. There is increasing evidence that rare variants play a role in some complex traits, but their analysis is not straightforward. Locus-based tests become necessary due to low power in rare variant single-point association analyses. In addition, variant quality scores are available for sequencing data, but are rarely taken into account. To enable this analysis, AMELIA has been developed as an allele-matching approach that is robust to the presence of both directions of effect for variants within the locus analysed.

    *   authors: Eleftheria Zeggini's group (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/amelia/

    *   source code language: R

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference:

*   ANALYZE

    *   full name:

    *   version: 2.1 (feb 14, 1996)

    *   description: a set of useful accessory programs to the LINKAGE package. It simplifies the performance of a large array of parametric and nonparametric tests for linkage and association on data entered in LINKAGE format pedigree and parameter files.

    *   authors: Joseph Terwilliger

    *   web/ftp: http://www.helsinki.fi/~tsjuntun/linkage/analyze/; http://linkage.cpmc.columbia.edu/index_files/Page434.htm;

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/linkage_cpmc_columbia/analyze/

    *   source code language: Pascal

    *   operating systems: UNIX (SunOS/Solaris/OSF1..)

    *   executables: analysis (script), analyzeprep, downfreq, extract, (homog), homogpos, homogpre, hrrlamb, hrrlambprep, hrrmult, hrrprep, interpretation, locusorder, pedprepare, polylocus, sib2link, sibpairna, sibprepna, tdtlikena

    *   reference:

*   ANCESTRY

    *   full name:

    *   version:

    *   description: This application aims at providing the user with an easy to use web-based user interface to compute individual ancestry estimates using the Maximum Likelihood Estimation method. This website also allows registered users to manage genotype and marker/allele frequency files. Registration is optional.

    *   authors: Van-Anh Tran (email: vananh.tran@case.edu), Thomas Dyer, Guo-Qiang Zhang, Jill Barnholtz-Sloan (email: jsb42@case.edu)

    *   web/ftp: http://ciil.case.edu:3000

    *   source code language: Ruby on Rails, FORTRAN

    *   operating systems: web-based

    *   executables:

    *   reference: Hanis, Chakraborty, Ferrell, Schull (1986), "Individual admixture estimates: disease associations and individual risk of diabetes and gallbladder disease among Mexican-Americans in Starr County, Texas", American Journal of Physical Anthropology, 70(4):433-441.

*   ANCESTRYMAP

    *   full name:

    *   version: 2.0

    *   description: ANCESTRYMAP screens through the genome in a recently mixed population such as African Americans, searching for segments with increased ancestry from one of the ancestral populations, which can indicate the position of disease genes

    *   authors: Arti Tandon (email: atandon@broad.mit.edu), Nick Patterson (email: nickp@broad.mit.edu), David Reich.

    *   web/ftp: http://genepath.med.harvard.edu/~reich/Software.htm

    *   source code language:

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference: Patterson, Hattangadi, Lane, Lohmueller, Hafler, Oksenberg, Hauser, Smith, O'Brien, Altshuler, Daly, Reich (2004), "Methods for high-density admixture mapping of disease genes", American Journal of Human Genetics, 74(5):97-1000

*   ANNOVAR

    *   full name: functional ANNOtation of genetic VARiants

    *   version: Feb 2010

    *   description: ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, as well as mouse, worm, fly, yeast and many others). Given a list of variants with chromosome, start position, end position, reference nucleotide and observed nucleotides, ANNOVAR can perform: 1\. gene-based annotation. 2\. region-based annotation. 3\. filter-based annotation. 4\. other functionalities.

    *   authors: Kai Wang (Univ Pennsylvania)

    *   web: http://www.openbioinformatics.org/annovar/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Wang, Li, Hakonarson (2010), "ANNOVAR: Functional annotation of genetic variants from next-generation sequencing data", Nucleic Acids Research, 38:e164.

*   ANTMAP

    *   full name:

    *   version: 1.2 (June 2006)

    *   description: Ordering markers when the number of loci is large is a special case of the traveling salesman problem. ANTMAP is a system based on the Ant Colony Optimization to solve this problem. ANYMAP performs segregation test, linkage grouping and locus ordering, and constructs a linkage map rapidly.

    *   authors: Hiroyoshi Iwata (email: iwata@lbm.ab.a.u-tokyo.ac.jp)

    *   web/ftp: http://lbm.ab.a.u-tokyo.ac.jp/~iwata/antmap/

    *   source code language: Java

    *   operating systems: MS-Windows, MacOS, Linux, UNIX(Solaris)

    *   executables:

    *   reference: Iwata, Ninomiya (2006) "AntMap: constructing genetic linkage maps using an ant colony optimization algorithm", Breeding Science, 56:371-377.


*   APE

    *   full name: Allelic Path Explorer

    *   version: 1.0

    *   description: Extends partially observed genotype data to the whole pedigree. Can be used for generating starting points for MCMC samplers and for checking that the genotype data are consistent with the pedigree structure.

    *   authors: Matti Pirinen, Dario Gasbarra (University of Helsinki, Finland)

    *   web/ftp: http://www.helsinki.fi/~mpirinen/download

    *   source code language: C

    *   operating systems: UNIX

    *   executables:

    *   reference: Pirinen, Gasbarra (2006), "Finding consistent gene transmission patterns on large and complex pedigrees", IEEE/ACM cransactions on Computational Biology and Bioinformatics, 3(3):252-262

*   APL-OSA

    *   full name: Association in the Presence of Linkage with Ordered Subset Analysis

    *   version:

    *   description: In the presence of genetic heterogeneity, APL-OSA can identify a genetically homogenous subset of families based on a trait-related covariate. APL-OSA then tests the relationship between the association statistics (i.e., the APL statistics) calculated based on the subset and the family-specific covariate. APL-OSA is based on the OSA method for linkage and the family-based association test, APL. Thus, APL-OSA has similar properties with OSA and APL. Bi-alleleic markers such as SNPs are accepted by APL-OSA. APL-OSA is a single-marker test and considers one covariate each time.

    *   authors: Elizabeth R. Hauser, email: apl-osa@chg.duhs.duke.edu

    *   web/ftp: http://wwwchg.duhs.duke.edu/research/aplosa.html

    *   source code language: C++

    *   operating systems: UNIX(Solaris), Linux(RedHat), MS-Windows(XP)

    *   executables:

    *   reference: Martin, Bass, Hauser, Kaplan (2003), "Accounting for linkage in family-based tests of association with missing parental genotypes", American Journal of Human Genetics, 73:1016-1026

    *   Hauser, Watanabe, Duren, Bass, Langefeld, Boehnke (2004), "Ordered subset analysis in genetic linkage mapping of complex traits", Genetic Epidemiology, 27:53-63.

    *   Chung, Hauser, martin (2006), "The APL test: extension to general nuclear families and haplotypes and examination of its robustness", Human Heredity, 61:189-199.

*   APM

    *   full name: Affected Pedigree-Member Method

    *   version: July 1993

    *   description:

    *   authors: Daniel E. Weeks (University of Oxford and University of Pittsburgh, email: dweeks@watson.hgen.pitt.edu), Mark Schroeder (email: mark@holmes.hgen.pitt.edu)

    *   web/ftp: http://watson.hgen.pitt.edu/register/soft_doc.html registration page at: http://watson.hgen.pitt.edu/register

    *   source code language: C and Pascal and FORTRAN

    *   operating systems: UNIX, VMS, MS-DOS

    *   executables: apm, sim, apmmult, simmult, chapm, hist

    *   reference: Weeks and Lange, American Journal of Human Genetics, 42(2):315-326 (1988). [[abstract](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1715269)]

*   ARIEL

    *   full name: Accumulation of Rare variants Integrated and Extended Locus-specific test

    *   version:

    *   description: ARIEL explores the effects of rare variants within complex traits through locus-based analysis. There is increasing evidence that rare variants play a role in some complex traits, but their analysis is not straightforward. Locus-based tests become necessary due to low power in rare variant single-point association analyses. In addition, variant quality scores are available for sequencing data, but are rarely taken into account. To enable this analysis, ARIEL has been developed as a locus-wide regression-based collapsing approach that incorporates variant quality scores.

    *   authors: Eleftheria Zeggini's group (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/ariel/

    *   source code language: R

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference:

*   ARLEQUIN

    *   full name: (French translation of "Arlecchino", a character of the Italian "Commedia dell'Arte". He has many aspects, but can switch among them very easily according to needs and necessities. Similarly, this software is about the study of genetic polymorphism.)

    *   version: 3.1 (September 2006)

    *   description: an exploratory population genetics software environment able to handle large samples of molecular data (RFLPs, DNA sequences, microsatellites), while retaining the capacity of analyzing conventional genetic data (standard multi-locus data or mere allele frequency data).

    *   authors: Laurent Excoffier (email: laurent.excoffier@zoo.unibe.ch), Guillaume Laval, Stefan Schneider (University of Berne, Switzerland)

    *   web/ftp: http://cmpg.unibe.ch/software/arlequin3/

    *   http://lgb.unige.ch/arlequin/

    *   source code language:

    *   operating systems: MS-Windows (95/98/NT/2000/XP)

    *   executables:

    *   reference: Excoffier, Laval, Schneider (2005), "Arlequin ver. 3.0: an integrated software package for population genetics data analysis", Evolutionary Bioinformatics Online, 1:47-50.

    *   Schneider, Kueffer, Roessli, Excofier, "Arlequin: A software for population genetic data analysis. Ver 1.1", (Genetics and Biometry Lab, Dept. of Anthropology, University of Geneva, 1997);

    *   User's manual (PDF, v1.1)

    *   User's manual (PDF, v3.1)

*   ARP.GEE

    *   full name:

    *   version: 0.1.0 (April 2005)

    *   description: Simultaneously estimate a trait-locus position and its genetic effects for affected relative pairs (ARP) by one of two methods. Either allow a different trait-locus effect for each ARP type, or constrain the trait-locus effects according to the marginal effect of a single susceptibility locus. We include a goodness of fit statistic for the constrained model.

    *   authors: Daniel Schaid, Jason Sinnwell (Mayo Clinic)

    *   web/ftp: http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm

    *   source code language: R/S-PLUS

    *   operating systems:

    *   executables:

    *   reference:

*   ASP/ASPSHARE

    *   full name:

    *   version: 1.2 (June 2002) (Warning: the earlier version contains a bug)

    *   description: ASP a power calculator for gene mapping using a sibpair design (concordant or discordant). ASPSHARE complements ASP in that it allows rapid calculation of the expected ibd sharing at the trait locus, based upon the model parameters, and the incidence corresponding to the respective parameters.

    *   authors: Michael Krawczak

    *   web/ftp: http://www.uni-kiel.de/medinfo/mitarbeiter/krawczak/download/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: online manual for ASP

*   ASPEX

    *   full name: Affected Sib Pairs EXclusion map

    *   version: 2.5 (September 2003)

    *   description: a set of programs for performing multipoint exclusion mapping of affected sibling pair data for discrete traits.

    *   authors: David Hinds (email: dahinds@users.sourceforge.net) Neil Risch (Stanford Univ),

    *   web/ftp: http://aspex.sourceforge.net/; ftp://lahmed.stanford.edu/pub/aspex

    *   source code language: C, Tcl, Perl (the graphic script requires xmgr, available from ftp://ftp.teleport.com/pub/users/pturner/acegr)

    *   operating systems: UNIX (Solaris/SunOS/IRIX/OSF-1), Linux

    *   executables: sib_ibd, sib_tdt, sib_phase, sib_map, sib_kin, sib_clean

    *   reference: online manual (PDF)

*   ASSOCIATIONVIEWER

    *   full name:

    *   version:

    *   description: AssociationViewer is a Java application used to display SNPs in a genetic context. Supplementary data (such as genes or LD plots) is downloaded from various public data sources on the fly and saved locally in a cache. Custom data can be added as supplementary tracks.

    *   authors:

    *   web/ftp: http://associationviewer.vital-it.ch/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Martin, Valsesia, Telenti, Xenarios, Stevenson (2009), "AssociationViewer: a scalable and integrated software tool for visualization of large-scale variation data in genomic context", Bioinformatics, 25(5):662-663.

*   AUTOSCAN

    *   full name:

    *   version: 1.0.1 (February 2000)

    *   description: a helper program to automate the tedious process of the creation of input files from genotype data of genome-wide scans

    *   authors: T Hiekkalinna, L Peltonen (UCLA)

    *   web/ftp: http://www.helsinki.fi/~tsjuntun/autoscan/

    *   source code language: C and UNIX-shell (Bourne)

    *   operating systems: UNIX (Solaris/DEC-UNIX)

    *   executables: autostart, autoscan10.sh

    *   reference: American Journal of Human Genetics, supp, 65 . ( [abstract](http://www.jurgott.org/linkage/absAUTOSCAN_AJHG99.htm)) (1999)

*   BAMA

    *   full name: Bayesian Analysis of Multilocus Association

    *   version:

    *   description: This is a program to select a trait-associated subset of markers among many candidates. The program is based on Bayesian modeling/estimation and it suits for both quantitative and qualitative traits. It can handle bi- and multiallelic markers as well as applied in situations where part of the marker genotypes may be missing. As an output of the program, one obtains posterior estimate of number and positions of trait-associated markers.

    *   authors: Mikko J Sillanpaa, Riika Kilpikari

    *   web/ftp: http://www.rni.helsinki.fi/~mjs/

    *   source code language: C

    *   operating systems: Linux

    *   executables:

    *   reference: R Kilpikari, MJ Sillanpaa (2003), "Bayesian analysis of multilocus association in quantitative and qualitative traits", Genetic Epidemiology, 25:122-135.

*   BARS

    *   full name: Bayesian Adaptive Regression Splines

    *   version:

    *   description: The BARS tests is a statistical method that bridges the gap between single-locus and haplotype-based tests of association. It is based on the non-parametric regression techniques embodied by Bayesian Adaptive Regression Splines.

    *   authors: Garrick Wallstrom, RE Kass

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/bars.htm

    *   source code language: R

    *   operating systems: Linux

    *   executables:

    *   reference: Zhang, Roeder, Wallstrom, Devlin (2003), "Integration of associative statistics over genomic regions using Bayesian Adaptive Regression Splines", Human Genomics, 1:20-29.

*   BAYESFST

    *   full name:

    *   version:

    *   description: Bayesian estimation of the coancestry coefficient FST

    *   authors: DJ Balding (email: d.balding@ic.ac.uk), KL Ayres, MA Beaumont, P O'Reilly

    *   web/ftp: http://www.reading.ac.uk/Statistics/genetics/software.html

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Balding (2003), "Likelihood-based inference for genetic correlation coefficients", Theoretical Population Biology, 63:221-230

    *   Beaumont, Balding (2004), "Identifying adaptive genetic divergence among populations from genome scans", Molecular Ecology, 13:969-980.

*   BDGEN

    *   full name:

    *   version: 1.7

    *   description: a powerful database software with improvement tools for paternity testing, database searching (like CODIS) and NRC II recommendations based formulae for investigating likelyhood ratios in putative contributors to crime evidences. Additional genetic population parameters estimations are added in this version. Only spanish version available.

    *   authors: Martinez Gustavo (email:ggmartinez@arnet.com.ar), Vazquez Sebastian, and Vicentin Ariel (email: bdgen@yahoo.com.ar)

    *   web/ftp: http://www.simedic.com.ar/bdgen.htm

    *   source code language:

    *   operating systems: MS-Windows

    *   executables:

    *   reference: V Vazquez, A Vicentin, L Vazquez, and G Martinez (2002) "Nuevo programa BDGen, como base de datos y herramienta de analisis estadistico de patrones geneticos". VII meeting of the Spanish-Portuguese Group for the International Society for Forensic Genertics. Barcelona, June 2002.

*   BEAGLE

    *   full name:

    *   version: 3.2.1 (May 2010)

    *   description: BEAGLE is a state of the art software package for analysis of large-scale genetic data sets with hundreds of thousands of markers genotyped on thousands of samples. BEAGLE can * phase genotype data (i.e. infer haplotypes) for unrelated individuals, parent-offspring pairs, and parent-offspring trios * infer sporadic missing genotype data * impute ungenotyped markers that have been genotyped in a reference panel * perform single marker and haplotypic association analysis * detect genetic regions that are homozygous-by-descent in an individual or identical-by-descent in pairs of individuals.

    *   authors: Brian L Browning, Sharon R Browning (email: browning@stat.auckland.ac.nz),

    *   web/ftp: http://www.stat.auckland.ac.nz/~browning/beagle/beagle.html

    *   source code language: Java

    *   operating systems: MS-Windows, Linux, UNIX(Solaris), MacOS

    *   executables:

    *   reference: Browning SR and Browning BR (2007), "Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering", American Journal of Human Genetics, 81(5):1084-1097

    *   Browning BR and Browning SR (2007), "Efficient multilocus association testing for whole genome association studies using localized haplotype clustering", Genetic Epidemiology, 31:365-375.

*   BEAGLECALL

    *   full name:

    *   version: 1.0.0 (September 2010)

    *   description: BEAGLECALL is a software package for simultaneous genotype calling and haplotype phasing for unrelated individuals. BEAGLECALL produces output posterior genotype probabilities and output phased haplotypes. BEAGLECALL generates extremely accurate genotype calls because it uses both allele signal intensity data and inter-marker correlation to call genotypes. BEAGLECALL is designed for use with high-density SNP arrays, and it uses the BEAGLE haplotype frequency model to model inter-marker correlation.

    *   authors:

    *   web/ftp: http://faculty.washington.edu/browning/beaglecall/beaglecall.html

    *   source code language: Java

    *   operating systems: MS-Windows, UNIX, Linux, MacOS

    *   executables:

    *   reference: Browning, Yu (2009), "Simultaneous genotype calling and haplotype phase inference improves genotype accuracy and reduces false positive associations for genome-wide association studies", American Journal of Human Genetics, 85:847-861.

*   BEAM

    *   full name: Bayesian Epistasis Association Mapping

    *   version:

    *   description: BEAM treats the disease-associated markers and their interactions via a bayesian partitioning model and computes, via Markov chain Monte Carlo, the posterior probability that each marker set is associated with the disease.

    *   authors: Yu Zhang, Jun S Liu (Harvard University)

    *   web/ftp: http://www.people.fas.harvard.edu/~junliu/BEAM/

    *   source code language: C++

    *   operating systems: UNIX, Linux, DOS

    *   executables:

    *   reference: Zhang, Liu (2007), "Bayesian inference of epistatic interactions in case-control studies", Nature, 39:1167-1173.

*   BETA

    *   full name:

    *   version: Jan 1998

    *   description: non-parametric linkage analysis using allele sharing in sib pairs

    *   authors: A Collins (email: arc@soton.ac.uk), NE Morton (University of Southampton, UK)

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/BETA

    *   source code language: C

    *   operating systems: UNIX(SUN..)

    *   executables: beta

    *   reference: Morton (1996), Proceedings of the National Academy of Sciences , 93:3471-3476; Collins, Morton (1996), Proceedings of the National Academy of Sciences , 93:9177-9181; Lio, Morton (1997), Proceedings of the National Academy of Sciences , 94:5344-5348; online manual

*   BIMBAM

    *   full name: Bayesian IMputation-Based Association Mapping

    *   version: 0.95

    *   description:

    *   authors: B Servin, M Stephens

    *   web/ftp: http://stephenslab.uchicago.edu/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Servin, Stephens (2007), "Imputation-based analysis of association studies: candidate genes and quantitative traits", PLoS Genetics, 3(7):e114. http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0030114

*   BIOIDE

    *   full name: BIOmedical Integrated Discovery Environment

    *   version:

    *   description: Developed by scientists for scientists, our Biomedical Integrated Discovery Environment provides easy access to a rapidly expanding collection of data analysis tools and interrelated biomedical information in a single easy-to-use software package with an extremely high benefit/cost ratio. Our component-based software/data integration platform can quickly transform fragmented enterprise legacy data and software into an integrated suite of tools and knowledge base to achieve maximal usability, interoperability, scalability, and extensibility.

    *   authors:

    *   web/ftp: http://www.htbiology.com/software.php

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   BIOLAD-DB

    *   full name:

    *   version: 3.0.1

    *   description: The BiolAD-DB system is a research bioinformatics system for inputting, validating, organizing, archiving, analyzing, and processing of complex clinical and genetic data. The database schema employs design principles for handling complex clinical information, such as response items in genetic questionnaires. Data access and validation is provided by the BiolAD-DB client application, which features a data validation engine tightly coupled to a graphical user interface. Data integrity is provided by the password protected BiolAD-DB SQL compliant server database. BiolAD-DB tools further provide functionalities for generating customized reports and views.

    *   authors: David Nielsen (email: nielsen@rockefeller.edu), Marty Leidner (email: marty@rockefeller.edu), Chad Haynes (email: haynesc@rockefeller.edu), Michael Krauthammer (email: michael.krauthammer@yale.edu), Mary Jeanne Kreek (email: kreek@rockefeller.edu)

    *   web/ftp: http://www.rockefeller.edu/biolad-db/

    *   source code language: Python

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Nielsen, Leidner, Haynes, Krauthammer, Kreek (2007), "The BiolAD-DB system - an informatics system for clinical and genetic data", Molecular Diagnosis & Therapy, 11(1):15-19.

*   BIRDSUITE

    *   full name:

    *   version:

    *   description: The Birdsuite is a fully open-source set of tools to detect and report SNP genotypes, common Copy-Number Polymorphisms (CNPs), and novel, rare, or de novo CNVs in samples processed with the Affymetrix platform. While most of the components of the suite can be run individually (for instance, to only do SNP genotyping), the Birdsuite is especially intended for integrated analysis of SNPs and CNVs.

    *   authors: Joshua Korn et al. (Broad Institute)

    *   web/ftp: http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/birdsuite/birdsuite

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Korn, Kuruvilla, McCarroll, Wysoker, Nemesh, Cawley, Hubbell, Veitch, Collins, Darvishi, Lee, Nizzari, Gabriel, Purcell, Daly, Altshuler (2008), "Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs", Nature Genetics, 40:1253-1260.

*   BLADE

    *   full name: Bayesian LinkAge DisEquilibrium mapping

    *   version: v2

    *   description:

    *   authors: Jun Liu (Harvard Univ) (email: jliu@stat.harvard.edu)

    *   web/ftp: http://www.people.fas.harvard.edu/~junliu/index1.html

    *   http://www.fas.harvard.edu/~junliu/TechRept/03folder/bladev2.tgz

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Liu, Sabatti, Teng, Keats, Risch (2001), "Bayesian analysis of haplotypes for linkage disequilibrium mapping", Genome Research, 11:1716-1724.

    *   Lu, Niu, Liu (2003), "Haplotype information and linkage disequilibrium mapping for single nucleotide polymorphisms", Genome Research, 13:2112-2117.

*   BLOCK

    *   full name: Blocking Gibbs sampler for pedigree analysis

    *   version: 2.0.0 (Dec 1996)

    *   description: the programs allow you to perform general pedigree analysis on a general pedigree with any number of loops. It also allows you to perform two-point linkage analysis on a general pedigree with an arbitrary number of alleles.

    *   authors: Claus Skaanning Jensen (email: claus@cs.auc.dk) (Aalborg University, Denmark)

    *   web/ftp: http://www.cs.auc.dk/~claus/block.html; http://hpcio.cit.nih.gov/lserver/BLOCK.html

    *   ftp://ftp.cs.auc.dk/pub/packages/block/current

    *   source code language:

    *   operating systems: MS-DOS, Linux, UNIX(Solaris, Irix 64, AIX 3.2.5, DEC alpha)

    *   executables: block, theta

    *   reference: Jensen, Kong and Kjaerulff, "Blocking-Gibbs Sampling in Very Large Probabilistic Expert Systems", International Journal of Human-Computer Studies, 647-666 (1995).

*   BMAPBUILDER

    *   full name:

    *   version: 2.1(beta)

    *   description:

    *   authors:

    *   web/ftp: http://bios.ugr.es/BMapBuilder/

    *   source code language: Java

    *   operating systems: MS-Windows, MacOS, UNIX, Linux

    *   executables:

    *   reference: Abad-Grau, Montes, Sebastiani (2006), "Building chromosome-wide LD maps", Bioinformatics, 22(16):1933-1934.

*   BOOLD

    *   full name:

    *   version: 1.1 (July 2002)

    *   description: a set of programs for calculations under different linkage disequilibrium (LD) distribution models

    *   authors: YS Aulchenko (email: aulchenko@epib.fgg.eur.nl) (Erasmus University Medical School. The Netherlands)

    *   web/ftp: http://www.geneticepi.com/Research/software/software.html

    *   http://www.geneticepi.com/Research/software/booLD-1.1.tar.gz

    *   source code language: C++, FORTRAN 77, Perl

    *   operating systems: UNIX, Linux

    *   executables: kLD, lsqLD, booit.pl

    *   reference:

*   BOOST

    *   full name: BOolean Operation based Screening and Testing

    *   version:

    *   description:

    *   authors: Xiang Wan, Can Yang (email: eeyang@ust.hk), Qiang Yang, Hong Xue, Xiaodan Fan, Nelson Tang, Weichuan Yu

    *   web/ftp: http://bioinformatics.ust.hk/BOOST.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Wan, Yang, Yang, Xue, Fan, Tang, Yu (2010), "BOOST: a Boolean representation-based method for detecting SNP-SNP interactions in genome-wide association studies"", American Journal of Human Genetics, 87:325-340.

*   BOOSTRAPPER

    *   full name:

    *   version: 1.0

    *   description: Robust haploblock border reliability estimation tool is implemented for LD based haploblock detection method. The most important new features are bootstrapping and overlapping block borders.

    *   authors:

    *   web/ftp: http://bioinfo.ebc.ee/download/

    *   source code language:

    *   operating systems: MS-Window, Linux

    *   executables:

    *   reference:

*   BOREL (see also PANGAEA)

    *   full name:

    *   version: July 1996

    *   description: Programs for inference of genealogical relationships from genetic data, including sibship inference.

    *   authors: Elizabeth A Thompson (University of Washington), Ian Painter (now at NCSU, email: ian@statgen.ncsu.edu)

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml

    *   ftp://ftp.u.washington.edu/pub/user-supported/pangaea/PANGAEA/BOREL

    *   source code language: C

    *   operating systems: UNIX(DEC-UNIX/..)

    *   executables:

    *   reference:

*   BOTTLENECK

    *   full name:

    *   version: 1.2.02

    *   description: a program for detecting recent effective population size reductions from allele data frequencies

    *   authors: Sylvian Piry

    *   web/ftp: http://www.montpellier.inra.fr/URLB/bottleneck/bottleneck.html

    *   source code language:

    *   operating systems: MS-Windows (95)

    *   executables:

    *   reference: Piry, Luikart, Cornuet (1999), "BOTTLENECK: a computer program for detecting recent reductions in the effective size using allele frequency data", Journal of Heredity, 90:502-503.

*   BPPH

    *   full name: Berkeley method for Perfect Phylogeney Haplotyping

    *   version: April 2003

    *   description: a program for inferring haplotypes from genotypes to determine if there are resulting haplotypes that fit a tree model (i.e. a perfect phylogeny, a coalescent). In population genetic terms, BPPH determines whether a set of SNP genotypes can be explained by haplotype pairs that could have evolved on a coalescent under the no-recombination, infinite sites model.

    *   authors: Ren-Hua Chuang, Dan Gusfield (UC Davis) (email: gusfield@cs.ucdavis.edu)

    *   web/ftp: http://wwwcsif.cs.ucdavis.edu/~gusfield/bpph.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: D. Gusfield (2002), "Haplotyping as perfect phylogeny: conceptual framework and efficient solutions", In Proceedings of RECOMB, Sixth Annual Conference on Research in Computational Molecular Biology, pp.?-?.

*   BQTL

    *   full name: Bayesian Quantitative Trait Locus mapping

    *   version:

    *   description: for the mapping of genetic traits from line crosses and recombinant inbred lines. It performs (1) maximum likelihood estimation of multi-gene models; (2) Bayesian estimation of multi-gene models via Laplace Approximations; and (3) interval mapping and composite interval mapping of genetic loci

    *   authors: Justin Borevitz, Charles C. Berry (UCSD)

    *   web/ftp: http://hacuna.ucsd.edu/bqtl/

    *   source code language: S, C, FORTRAN. The software is engineered to work in conjunction with R.

    *   operating systems: UNIX, MS-Windows, MacOS

    *   executables: cvl.blup.R(), cvl.epi.R(), cvl.jack.R(), cvl.qtl.effect.R(), read.map.R(), read.markers.R().

    *   reference: JO Borevitz, JN Maloof, J Lutes, T Dabi, JL Redfern, GT Trainer, JD Werner, T Asami, CC Berry, D Weigel, J Chory (2002), "Quantitative trait loci controlling light and hormone response in two accessions of Arabidopsis thaliana", Genetics, 160(2):683-696.

*   BREAKDANCER

    *   full name:

    *   version:

    *   description: BreakDancer provides genome-wide detection of structural variants from next generation paired-end sequencing reads. BreakDancerMax predicts five types of structural variants: insertions, deletions, inversions, inter- and intra-chromosomal translocations from next-generation short paired-end sequencing reads using read pairs that are mapped with unexpected separation distances or orientation.

    *   authors:

    *   web/ftp: http://gmt.genome.wustl.edu/breakdancer/current/

    *   source code language: Perl, C++

    *   operating systems:

    *   executables:

    *   reference: Chen, Wallis, McLellan, Larson, Kalicki, Pohl, McGrath, Wendl, Zhang, Locke, Shi, Fulton, Ley, Wilson, Ding, Mardis (2009), "BreakDancer: an algorithm for high-resolution mapping of genomic structural variation", Nature Methods, 6:677-681.

*   CAROL

    *   full name: Combined Annotation scoRing toOL

    *   version:

    *   description: CAROL is a combined functional annotation score of non-synonymous coding variants. A major challenge in interpreting whole-exome data is predicting which of the discovered variants are deleterious or neutral. To address this question in silico, we have developed a score called Combined Annotation scoRing toOL (CAROL),which combines information from two bioinformatics tools: PolyPhen-2 and SIFT, in order to improve the prediction of the effect of non-synonymous coding variants. The combination of annotation tools can help improve automated prediction of whole-genome/exome non-synonymous variant functional consequences.

    *   authors: Eleftheria Zeggini's group (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/carol/

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference:

*   CARTHAGENE

    *   full name:

    *   version: 1.0 (January 2006)

    *   description: It is a genetic/radiation hybrid mapping software. CARTHAGENE looks for multiple populations maximum likelihood consensus maps using a fast EM algorithm for maximum likelihood estimation and powerful ordering algorithms inspired from TSP (Traveling Salesman Problem) technology. It can handle large data sets made up of different populations (either F2 backcross, recombinant inbred lines, F2 intercross, phase known outbreds, haploid/diploid radiation hybrids). It can also exploit existing syntenic relationships between the organism mapped and a reference (sequenced) organism for accurate dense RH mapping.

    *   authors: P Chabrier, D de Givry, T Faraut, C Gaspin, T Schiex (INRA, Toulouse, France)

    *   web/ftp: http://www.inra.fr/mia/T/CarthaGene/

    *   source code language: C++,Tcl/Tk

    *   operating systems: MS-Windows, UNIX(Solaris), Linux. Sources are available for other operating systems (GPL/QPL).

    *   executables:

    *   reference: Schiex,Gaspin, Proceedings of the International Conference on Intelligent Systems for Molecular Biology (ISMB) (1997).

    *   Givry, Bouchez, Chabrier, Milan, Schiex (2005), "CARTHAGENE: multipopulation integrated genetic and radiation hybrid mapping", Bioinformatics, 21(8):1703-1704.

*   CASAVA

    *   full name: Consensus Assessment of Sequence And VAriation

    *   version: v1.6

    *   description: Illumina's CASAVA software captures summary information for resequencing and counting studies and places the data in a compact structure for visualization within GenomeStudio Software or publicly available analysis tools. CASAVA can create genomic builds, call SNPs, detects indels, and count reads from data generated from one or more runs of the Genome Analyzer across a broad range of sequencing applications. \\ v.1.6 highlights: * complete secondary analysis software package, including facilities for alignment, reference-guided assembly, SNP/indel calling and counting for RNA applications. * new methods for gapped, multiseed alignments that reduce artifactual mismatches. * new indel detection method, and improved SNP calling on longer Genome Analyzer reads.

    *   authors: (Illumina)

    *   web/ftp: http://www.illumina.com/software/genome_analyzer_software.ilmn

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:


*   CASPAR

    *   full name: Computerized Affected Sibling Pair Analyzer and Reporter

    *   version:

    *   description: CASPAR's main novel feature is conditional linkage analyses, in which the population can be subdivided according to criteria at some loci and analyzed for linkage at other loci. CASPAR uses simulation to overcome the problems inherent in such multiple testing.

    *   authors: Richa Agarwala (email: richa@helix.nih.gov) and Alejandro Schaffer (email: schaffer@helix.nih.gov); other participants include Jeremy Buhler, Kenneth Gabbay, Marek Kimmel, David Owerbach

    *   web/ftp: http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/caspar.html

    *   ftp://fastlink.nih.gov/pub/caspar

    *   source code language: C

    *   operating systems: UNIX

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting 1995 (Buhler, et. al. American Journal of Human Genetics, 57 Supp, A171 (1995));

    *   Buhler, et. al. (1997) Human Heredity, 47(4):211-222.

*   CATS

    *   full name: power Calculator for Association with Two Stage design

    *   version: 0.0.1

    *   description: CATS calculates the power and other useful quantities for two-stage genome-wide association studies

    *   authors: Andrew Skol (email: askol@umich.edu)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/CaTS/

    *   source code language: C++

    *   operating systems: UNIX, Linux, MS-Windows, MacOS

    *   executables: cats (graphical interface for MS-Windows, MacOS, and Linux), cats-test (Linux command line)

    *   reference: Skol, Scott, Abecasis, Boehnke (2006), "Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies", Nature Genetics, 38:209-213.

*   CC-QLS

    *   full name: Case-Control Quasi-Likelihood Score test

    *   version: 1.3 (March 2006)

    *   description:

    *   authors: Catherine Bourgain, Sabine Hoffjan, Raluca Nicolae, Dina Newman, Lori Steiner, Karen Walker, Rebecca Reynolds, Carole Ober, Mary Sara McPeek

    *   web/ftp: http://www.stat.uchicago.edu/~mcpeek/software/CCQLSpackage1.3/

    *   source code language:

    *   operating systems: UNIX

    *   executables:

    *   reference: Bourgain, Hoffjan, Nicolae, Newman, Steiner, Walker, Reynolds, Ober, McPeek (2003), "Novel case-control test in a founder population identifies p-selectin as an Atopy-susceptibility locus", American Journal of Human Genetics, 73(3):612-626

*   CCRAVAT (see also QUTIE)

    *   full name: Case-Control RAre Variant Analysis Tool

    *   version:

    *   description: Enabling the analysis of rare variants in large-scale case control and quantitative trait association studies. CCRaVAT (Case-Control Rare Variant Analysis Tool) and QuTie (Quantitative Trait) are software packages that enable efficient large-scale analysis of rare variants across specific regions or genome-wide. These programs implement a rare variant super-locus or collapsing method that investigates the accumulation of rare variant alleles in either a case-control or quantitative trait study design.

    *   authors: Eleftheria Zeggini's group (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/rarevariant/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   CCREL

    *   full name: Case-Control association analysis with RELlated individuals

    *   version: 0.3 (December 2005)

    *   description: CCREL is a program for case-control genetic analysis that takes relatedness between individuals into account. It will perform single-marker and haplotypic tests, however it will only work with SNP or other biallelic markers.

    *   authors: Sharon R. Browning (e-mail: browning@stat.auckland.ac.nz)

    *   web/ftp: http://www.stat.auckland.ac.nz/~browning/ccrel/ccrel.htm

    *   source code language: R with Perl and C

    *   operating systems: Unix/Linux

    *   executables:

    *   reference: Browning, Briley, Briley, Chandra, Charnecki, Ehm, Johansson, Jones, Karter, Yarnall, Wagner (2005), "Case-control single marker and haplotypic association analysis of pedigree data", Genetic Epidemiology, 28:110-122.

*   CEPH2CRI

    *   full name:

    *   version: May 1994

    *   description: convert output from CEPH DBMS to CRIMAP format

    *   authors: John Attwood (Sanger Centre, UK) ( email: ja1@sanger.ac.uk)

    *   web/ftp: http://www.gene.ucl.ac.uk/public-files/packages/linkage_utils/ceph2cri/

    *   source code language: Pascal

    *   operating systems: MS-DOS

    *   executables:

    *   reference:

*   CEPH2MAP

    *   full name:

    *   version:

    *   description: it is developed from crimap v2.4 for use with the map suite of programs.

    *   authors:

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ceph2map

    *   source code language: C

    *   operating systems:

    *   executables: ceph2map

    *   reference:

*   CFC

    *   full name: Contribution, Inbreeding (F) and Coancestry

    *   version: 1.0 (April 2006)

    *   description: This is a general-purpose program for monitoring genetic diversity. CFC allows for several pedigree analyses such as: (1) computing inbreeding coefficients and relationships; (2) computing average relationships very quickly within and between specified groups of individuals; (3) computing average relatedness; (4) ancestral decomposition of the average or the individual inbreeding coefficient; (5) ancestral decomposition of the average coancestry; (6) optimizing matings to minimize the average inbreeding coefficients in the next generation; (7) computing founder equivalent, founder genome equivalent and effective number of non-founders; (8) computing numerator relationship matrix, its Cholesky decomposition and its inverse; (9) providing useful information on the structure of pedigrees

    *   authors: Mehdi Sargolzaei (University of Guelph), Hiroaki Iwaisaki and Jean-Jacques Colleau

    *   web/ftp: http://agrews.agr.niigata-u.ac.jp/~iwsk/cfc.html

    *   source code language: Visual C++

    *   operating systems: MS-Windows(98/NT/XP)

    *   executables:

    *   reference: M Sargolzaei, H Iwaisaki, JJ Colleau (2006), "CFC: a tool for monitoring genetic diversity", Proceedings of the 8th World Congress on Genetics Applied to Livestock Production, CD-Rom comm. N 27-28.

*   CHAPLIN

    *   full name: Case-control HAPLotype INference package

    *   version: 1.2.2

    *   description: Chaplin is a software program for identifying specific haplotypes or haplotype features that are associated with disease using genotype data from a case-control study.

    *   authors: Richard Duncan, Michael Epstein, Glen Satten (Emory University)

    *   web/ftp: http://www.genetics.emory.edu/labs/epstein/software/chaplin/index.html

    *   source code language: FORTRAN90 (CVF 6.6) with IMSL routines

    *   operating systems: MS-Windows (2000/XP)

    *   executables: chaplin.exe

    *   reference: M Epstein, G Satten (2003), "Inference on haplotype effects in case-control studies using unphased genotype data", American Journal of Human Genetics, 73(6):1316-1329

    *   Satten, Epstein (2004), "Comparison of prospective and retrospective methods for haplotype inference in case-control studies", Genetic Epidemiology, 27(3):192-201

*   CHECKHET

    *   full name:

    *   version: March'02 version

    *   description: a program to detect genetically abnormal subjects in a case-control sample based on genotypes at multiple marker loci.

    *   authors: Dave Curtis (Royal London Hospital, UK) (email: dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   http://www.smd.qmul.ac.uk/statgen/software/dcurtis/chhet10.zip

    *   source code language:

    *   operating systems: MS-Windows, UNIX

    *   executables:

    *   reference: D Curtis, BV North, H Gurling, E Blaveri, PC Sham (2002), "A quick and simple method for detecting subjects with abnormal genetic background in case-control samples", Annals of Human Genetics, 66:235-244.

*   CHECKMATRIX (other name: PY_MATRIX_2D)

    *   full name:

    *   version: v248

    *   description: CheckMatrix serves as a visualization tool to validate constructed genetic maps. CheckMatrix generates graphical genotypes and two-dimensional heat plots of pairwise scores. Visualization of regions with positive and negative linkage as well as of allele fraction per marker simplifies genetic map validation without applying statistical approaches. CheckMatrix works in conjunction with MadMapper and freely available at http://www.atgc.org/XLinkage/MadMapper/

    *   authors: Alexander Kozik (UC Davis)

    *   web/ftp: http://www.atgc.org/XLinkage/

    *   source code language: Python

    *   operating systems:

    *   executables:

    *   reference:

*   CHIAMO

    *   full name:

    *   version: 0.2.1 (Oct 2007)

    *   description: CHIAMO is a program for calling genotypes from the Affymetrix 500K Mapping chip. The program allows for multiple cohorts which have potentially different intensity characteristics that can lead to elevated false-positive rates in genome-wide studies. The underlying model has a hierarchical structure that allows for correlation between the parameters of each cohort. The output files produced by CHIAMO feed directly into both the programs SNPTEST and IMPUTE. CHIAMO was used to call genotypes for the 7 genome-wide association studies carried out by the Wellcome Trust Case-Control Consortium (WTCCC).

    *   authors: Jonathan Marchini

    *   web/ftp: http://www.stats.ox.ac.uk/~marchini/software/gwas/chiamo.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Marchini, Spencer, Teo, Donnelly (2007), "A Bayesian hierarchical mixture model for genotype calling in a multi-cohort study",

*   CHIP2SPELL

    *   full name:

    *   version: 1.0.2 (September 2009)

    *   description: Chip2Spell aims to speed up the preparation of linkage data files passed on to the Alohomora software for data from the Affymetrix or Illumina assays. Chip2Spell gathers the information necessary to supplement the genotype data (such as the genetic map of the markers and the population-specific allele frequencies) from publicly available annoation files from Affymetrix and Illumina. The program formats the genotype data for input into the Alohomora program.

    *   authors: Frederic Fournier (email: frederic.fournier@statgen.org)

    *   web/ftp: http://www.statgen.org/main/index.php/Downloads/Downloads

    *   source code language: perl

    *   operating systems: Linux, MS-Windows

    *   executables:

    *   reference:

*   CHROMOSCAN

    *   full name:

    *   version: 1.0

    *   description: CHROMOSCAN is an implementation of a genome-based scan statistic that detects genomic regions, which are statistically significant for targeted measurements, such as genetic associations with disease, gene expression profiles, DNA copy number variations, as well as other genome-based measurements.

    *   authors: Yan V Sun (email:yansun@umich.edu), Douglas M Jacobsen and Sharon LR Kardia (email: kardiasoftware@umich.edu) (University of Michigan)

    *   web/ftp: http://www.epidkardia.sph.umich.edu/software/chromoscan/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Sun, Jacobsen, Kardia (2006), "ChromoScan: a scan statistic application for identifying chromosomal regions in genomic studies", Bioinformatics, 22(23):2945-2947.

    *   Sun, Levin, Boerwinkle, Robertson, Kardia (2006), "A scan statistic for identifying chromosomal patterns of SNP association", Genetic Epidemiology, 30(7):627-635.

*   CHROMOSEG

    *   full name:

    *   version: 1.0

    *   description: A simulation program that simulates and plots (in real time) ancestral recombination graphs. This is currently primarily a teaching/educational tool.

    *   authors: Eric C Anderson (email: eric.anderson@stanfordalumni.org)

    *   web/ftp: http://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   CHROMSCAN

    *   full name:

    *   version: beta (December 11, 2006)

    *   description: CHROMSCAN is a statistical based program for association mapping of disease genes. It utilises the Malecot model and the linkage disequilibrium (LD) map for the candidate region to analyse the genotypes derive from large sample of matched cases and controls.

    *   authors: A.R. Collins (University of Southampton)

    *   web/ftp: http://www.som.soton.ac.uk/research/geneticsdiv/epidemiology/chromscan/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   CLUMP

    *   full name:

    *   version:

    *   description: Monte Carlo method for assessing significance of a case-control association study with multi-allelic marker.

    *   authors: Dave Curtis (Royal London Hospital, UK) (email: dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Sham, Curtis (1995), Annals of Human Genetics, 59:97-105; online documentation

*   CLUSTAG

    *   full name:

    *   version: 2

    *   description: hierarchical clustering and graph methods for selecting tage SNPs

    *   authors: Sio Iong Ao, Michael K Ng (Dept of Mathematics, University of Hong Kong), Kevin Yip, David Cheung (Dept of Computer Science, University of Hong Kong), Pak Sham, Pui Yee Fong, Ian Melhado (Genome Research Center, University of Hong Kong),

    *   web/ftp: http://hkumath.hku.hk/web/link/CLUSTAG/CLUSTAG.html

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Ao, Yip, Ng, Cheung, Fong, Melhado, Sham (2005), "CLUSTAG: hierarchical clustering and graph methods for selecting tag SNPs", Bioinformatics, 21(8):1735-1736.

*   CMAP

    *   full name: genetic and comparative maps

    *   version: 0.06 (June 14, 2006)

    *   description: CMAP is a web-based tool that allows users to view comparisons of genetic and physical maps. The package also includes tools for curating map data.

    *   authors: Ken Y Clark (email: kclark@cshl.org), Lincoln Stein (email: lstein@cshl.org), Doreen Ware (email: ware@cshl.org)

    *   web/ftp: http://www.gmod.org/cmap/

    *   https://sourceforge.net/project/showfiles.php?group_id=27707

    *   source code language: Perl

    *   operating systems: UNIX (Solaris, FreeBSD), Linux

    *   executables: cmap_admin.pl

    *   reference: Ware et al. (2002), "Gramene, a tool for grass genomics", Plant Physiology, 130:1606-1613.

*   COMBIN

    *   full name:

    *   version: 1.1

    *   description: Designed for the construction of highly saturated linkage maps, based on BC1, DH, Radiation Hybrid or CP (CrossPollinators) data sets. F2 is not supported.

    *   authors: Jaap Buntjer (Lab of Plant Breeding, Wageningen University. email: jaap.buntjer@users.pv.wau.nl), Herman van Eck (email: herman.vaneck@users.pv.wau.nl)

    *   web/ftp: http://www.dpw.wau.nl/pv/pub/combin/

    *   source code language: Visual Basic 5

    *   operating systems: MS-Windows (95/98/NT)

    *   executables:

    *   reference:

*   COMDS

    *   full name:

    *   version:

    *   description: for combined segregation and linkage analysis, incorporating severity and diathesis.

    *   authors: NE Morton (University of Southampton, UK)

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/comds

    *   source code language: SUN FORTRAN (the command "fsplit" is needed)

    *   operating systems: UNIX (SunOS/..)

    *   executables: list1,com1, comds (shell script)

    *   reference: manual (November 1993)

*   COMPOSITELD

    *   full name:

    *   version: 0.2.1 (December 2006)

    *   description: A method to compute composite measures of linkage disequilibrium, their variances and covariances, and statistical tests, for all pairs of alleles from two loci when linkage phase is unkown. An extension of Weir and Cockerham (1989) to apply to multi-allelic loci.

    *   authors: Daniel Schaid (Mayo Clinic)

    *   web/ftp: http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm

    *   source code language: R/S-PLUS

    *   operating systems:

    *   executables:

    *   reference:

*   COPE

    *   full name: COllaborative Pedigree drawing Environment

    *   version:

    *   description: includes a Java program for drawing pedigrees and a standardized system for pedigree storage. Unlike other existing pedigree programs, this software is particularly intended for epidemiologists in the sense that it allows customized automatic drawing of large numbers of pedigrees and remote and distributed consultation of pedigrees.

    *   authors: L Brun-Samarcq, S Gallina, A Philippi, F Demenais, G Vaysseix, E Barillot (email: cope@infobiogen.fr)

    *   web/ftp: http://www.infobiogen.fr/services/CoPE (closed)

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Brun-Samarcq, et al, (1999) Bioinformatics, 15(4):345-346\. [ full text (PDF) ].

*   COVIBD

    *   full name:

    *   version:

    *   description: CovIBD refines linkage analysis of affected sibpairs by considering attributes or environmental exposures thought to affect disease liability. This refinement utilizes a mixture model in which a disease mutation segregates in only a fraction of the sibships, with the rest of the sibships unlinked. Covariate information is used to predict membership within the two groups corresponding to the linked and unlinked sibships. The pre-clustering model uses covariate information to first form two probabilistic clusters and then tests for excess IBD-sharing in the clusters. The Cov-IBD model determines probabilistic group membership by joint consideration of covariate and IBD values.

    *   authors:

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/newcovibd/covibd.htm

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Devlin, Jones, Bacanu, Roeder (2002), "Mixture models for linkage analysis of affected sibling pairs and covariates", Genetic Epidemiology, 22(1):52-65.

    *   Devlin, Jones, Bacanu, Roeder (2002), "Reply to Olson", Genetic Epidemiology, 23(4):449-455.

*   CRIMAP

    *   full name:

    *   version: 2.4

    *   description: constructing multilocus linkage map

    *   authors: Phil Green (Univ. of Washington, email: phg@u.washington.edu)

    *   web/ftp: http://compgen.rutgers.edu/Crimap/

    *   source code language: C

    *   operating systems: UNIX, MS-Windows(XP)

    *   executables: crimap

    *   reference: Lander, Green (1987), Proceedings of the National Academy of Sciences , 84:2363-2367.

    *   documentation (version 2.4, 1990).

    *   Tutorial (from biobase.dk)

*   CRIMAP-PVM

    *   full name: CRIMAP with Parallel Virtual Machine

    *   version:

    *   description: a parallel version of CRIMAP

    *   authors: Tara Matise (email: matise@biology.rutgers.edu), Mark Schroeder, Donald Chiarulli, Daniel E. Weeks

    *   web/ftp: http://compgen.rutgers.edu/multimap/crimappvm.html

    *   source code language: C, PVM

    *   operating systems:

    *   executables:

    *   reference: Matise, Schroeder, Chiarulli, Weeks, (1995) Human Heredity, 45:103-116.

*   CROSSFIND (other names: BREAKPOINT, BPT)

    *   full name:

    *   version: October 1995

    *   description: software for detecting and displaying well-characterized meiotic breakpoints in human family data. another program, CRIMAP, is used.

    *   authors: J. Attwood (Sanger Centre, email: ja1@sanger.ac.uk)

    *   web/ftp: http://www.gene.ucl.ac.uk/public-files/packages/jattwood/

    *   source code language: C

    *   operating systems: UNIX(SunOS)

    *   executables:

    *   reference: Attwood, Povey (1996), Annals of Human Genetics, 60:487-498.

*   CYRILLIC

    *   full name:

    *   version 3 (October 1999)

    *   description: (version 3) pedigree drawing with fully integrated risk analysis and support for industry standard databases (MS Access and Corel Paradox). designed for genetic counselors and others who work with patients. if you use genetic marker data you need Cyrillic 2\. (version 2) this version of Cyrillic draws pedigrees, works with genetic marker data, lets you do haplotyping and allows exports to a range of linkage analysis packages.

    *   authors: Cherwell Scientific Publishing and Cyril Chapman (email: cyrillic@cherwell.com)

    *   web/ftp: http://www.cyrillicsoftware.com

    *   http://www.cyrillicsoftware.com/download/downloads.htm

    *   source code language: Visual C++

    *   operating systems: MS-Windows (95/98/NT)

    *   executables:

    *   reference: Chapman (1990), American Journal of Medical Genetics, 36:155-160.

    *   availability: commercial v2.1 (US$599/379 UK pounds/570 euros), 3 (US$769/479 UK pounds/725 euro), upgrades from v2.1 to v3 (US$299), upgrades from v2 to v3 (US$399),

    *   vender: Cyrillic Software, UK; Exeter Software (USA reseller)

*   DCHIP LINKAGE

    *   full name:

    *   version:

    *   description:

    *   authors: Igor Leykin, Cheng Li (Harvard University)

    *   web/ftp: http://www.biostat.harvard.edu/complab/dchip/snp.htm

    *   http://www.biostat.harvard.edu/~ileykin/accessory.htm
    
    *   http://groups.yahoo.com/group/dchip/

    *   source code language: C++

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Leykin et al. (2005), "Comparative linkage analysis and visualization of high-density oligonucleotide SNP array data", BMC Genetics, 6:7.


*   DGENE

    *   full name:

    *   version:

    *   description: a simple dBASE III program for the management of pedigree and locus data. It permits easy extraction of genetic data for use with MENDEL and FISHER.

    *   authors: Kenneth Lange (UCLA), Daniel E. Weeks (Pittsburgh), M. Boehnke

    *   web/ftp: registration page at: http://www.biomath.medsch.ucla.edu/faculty/klange/software.html

    *   source code language: dBASE III

    *   operating systems: MS-DOS, ...

    *   executables:

    *   reference: Lange, Weeks, Boehnke, Genet Epid, 5, 471-471 (1988)

*   DHSMAP

    *   full name: Decay of Haplotype Sharing MAPping software

    *   version: 2.0 (October 18, 2001)

    *   description: DHSMAP is a program for fine-mapping of qualitative traits by linkage disequilibrium. Given a set of marker haplotypes or genotypes from affected individuals, haplotypes or genotypes from appropriately selected controls, and a genetic map of the markers at which both sets of individuals are typed, DHSMAP estimates the location of the trait-associated variant by maximum likelihood or maximum quasi-likelihood.

    *   authors: Andrew Strahs, Kenneth Wilder, Mary Sara McPeek (University of Chicago)

    *   web/ftp: http://galton.uchicago.edu/~mcpeek/software/dhsmap/

    *   source code language: C++

    *   operating systems: UNIX(Solaris/..), Linux

    *   executables: dhsmap.solaris, dhsmap.linux

    *   reference: McPeek and Strahs (1999), "Assessment of linkage disequilibrium by the decay of haplotype sharing, with application to fine-scale genetic mapping", American Journal of Human Genetics, 65(3):858-875

*   DINDEL

    *   full name:

    *   version:

    *   description: Dindel is a program for calling small indels from short-read sequence data ('next generation sequence data'). It is currently designed to handle only Illumina data. Dindel takes BAM files with mapped Illumina read data and enables researchers to detect small indels and produce a VCF file of all the variant calls.

    *   authors: Cornelis Albers (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/dindel/

    *   source code language: C++

    *   operating systems: Linux, MacOS

    *   executables:

    *   reference: Albers, Lunter, MacArthur, McVean, Ouwehand, Durbin (2011), "Dindel: Accurate indel calls from short-read data", Genome Research, 21:961-973.

*   DISENTANGLER

    *   full name:

    *   version: October 2011

    *   description: The disentangler is a visualization technique for linkage disequilibrium mapping and haplotype analysis of multiple multi-allelic genetic markers.

    *   authors: K Kumasaka et al.

    *   web/ftp: http://kumasakanatsuhiko.jp/projects/disentangler/

    *   source code language: Java/JRE1.5

    *   operating systems:

    *   executables:

    *   reference: Okada et al. (2011), "HLA-Cw*1202-B*5201-DRB1*1502 haplotype increases risk for ulcerative colitis but reduces risk for Crohn's disease", Gastroenterology, 141(3):864-871.

*   DMAP

    *   full name:

    *   version:

    *   description: fine-scale disequilibrium mapping of a rare allele

    *   authors: Jinko Graham

    *   web/ftp: http://www.math.sfu.ca/~jgraham/Papers/Programs/DisequilibriumMapping/

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Graham, Thompson (1998), "Disequilibrium likelihoods for fine-scale mapping of a rare allele", American Journal of Human Genetics, 63(5):1517-1530

*   DMLE

    *   full name: Disease Mapping using Linkage disEquilibrium

    *   version: 2.3 (October 2005)

    *   description: high-resolution mapping of the position of a disease mutation relative to a set of genetic markers using population linkage disequilibrium (LD).

    *   authors: Jeff Reeve, Ying Wang, Rannala (University Alberta, Canada)

    *   web/ftp: http://www.dmle.org/

    *   source code language:

    *   operating systems: Linux, MS-Windows

    *   executables:

    *   reference: Reeve, Rannala, Bioinformatics, 18(6):894-895 (2002).

*   DNABASER

    *   description: a tool for manual and automatic DNA sequence assembly, DNA sequence analysis, automatic sample processing, contig editing, metadata integration, file format conversion and mutation detection.

    *   web: http://www.dnabaser.com/

*   DOLINK (other name: WDLINK)

    *   full name:

    *   version:

    *   description: manage genetic data

    *   authors: Dave Curtis (Royal London Hospital, UK) (email: dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C++

    *   operating systems: MS-DOS, MS-Windows

    *   executables:

    *   reference: Cook, Gurling, Curtis (1993), "DOLINK: a computer program to facilitate management of genetic data and analyses", Annals of Human Genetics, 57:307-310.

*   DPPH

    *   full name: Direct method for Perfect Phylogeney Haplotyping

    *   version:

    *   description: similar to the BPPH program

    *   authors: Ren-Hua Chuang, Dan Gusfield (UC Davis) (email: gusfield@cs.ucdavis.edu)

    *   web/ftp: http://wwwcsif.cs.ucdavis.edu/~gusfield/dpph.html

    *   source code language:

    *   operating systems: Linux, MacOS

    *   executables:

    *   reference: Bafna, Gusfield, Lancia, Yooseph (2003), "Haplotyping as perfect phylogeny: a direct approach", Journal of Computational Biology, 10(3):323-340.

*   EAGLET

    *   full name: Efficient Analysis of Genetic Linkage: Testing and Estimation

    *   version: 1.1

    *   description: EAGLET is a software package that provides a number of improved statistics for detecting linkage and estimating trait location. EAGLET uses multiple subsamples of dense SNP data to detect linkage with increased power, and to construct sharp 95% confidence intervals for the true trait location. Subsamples with minimal amounts of LD are repeatedly drawn from the original dense SNP data, and the linkage information across subsamples is then combined. This minimizes the false positive evidence for linkage that LD can create in classical multipoint linkage analysis due to an apparent excess in allele sharing among affecteds. Another attractive feature of EAGLET is that by using multiple subsamples, as opposed to one, EAGLET's estimate of trait location is greatly improved. EAGLET can be used to generate subsamples for custom linkage packages as well, and computes standard LD statistics via the EM algorithm.

    *   authors: William C.L. Stewart (email: ws2267@columbia.edu)

    *   web/ftp: http://www.columbia.edu/~ws2267/SOFT/EAGLET/eaglet.html

    *   source code language: Perl, C

    *   operating systems: Linux, MacOS

    *   executables: eaglet.pl

    *   reference: Stewart, Peljto, Greenberg (2009), "Multiple subsampling of dense SNP data localizes disease genes with increased precision", Human Heredity, to appear.

*   EASYLINKAGE/EASYLINKAGE-PLUS

    *   full name:

    *   version: 5.02

    *   description: easyLINKAGE combines automated setup and performance of linkage analyses and simulation under an easy to handle graphical user interface for Microsoft Windows 2000/XP and standard UNIX systems. The program package supports currently single-point linkage analyses (FastLink, SPLink, SuperLink), multi-point linkage analyses (GeneHunter, GeneHunter Plus, GeneHunter-Imprinting/-TwoLocus, Allegro, SimWalk, Merlin), and the simulation package SLink, and provides genome-wide as well as chromosomal postscript plots of LOD scores, NPL scores, P values, and many other parameters. The software can analyze STRPs as well as SNP chip data from Affymetrix, Illumina, or self-defined SNP data. The program performs single- and multi-point simulation studies.

    *   authors: Tom H Lindner (email: t-lindner@gmx.de), Katrin Hoffmann (email: khoffma@gmx.de)

    *   web/ftp: http://compbio.charite.de/genetik/hoffmann/easyLINKAGE/

    *   http://sourceforge.net/projects/easylinkage/

    *   source code language: Perl v5.8 (program can be provided as PERL script or as a compiled exe for the use in Windows)

    *   operating systems: MS-Windows(2000/XP), Linux

    *   executables: linkage_v5.02.exe

    *   reference: Lindner, Hoffmann (2005), "easyLINKAGE: A PERL script for easy and automated two-/multi-point linkage analyses", Bioinformatics, 21(3):405-407.

    *   Hoffmann, Lindner (2005), "easyLINKAGE-Plus - automated linkage analyses using large-scale SNP data" Bioinformatics, 21(17):3565-3567.

*   EASYPOP

    *   full name:

    *   version: 2.0.1 (July 2006)

    *   description: EASYPOP is a computer program allowing to simulate population genetics datasets. It allows generating genetic data for haploid, diploid, and haplodiploid organisms under a variety of mating systems. It includes various migration and mutation models. Output can be generated for the FSTAT, GENEPOP, and ARLEQUIN genetic analysis packages.

    *   authors: Francois Balloux (email: fb255@gen.cam.ac.uk)

    *   web/ftp: http://www.unil.ch/dee/page36926_fr.html

    *   source code language:

    *   operating systems: MS-Windows (95/98/NT/2000/XP), MacOS(X)

    *   executables:

    *   reference: Balloux (2001), "EASYPOP (Version 1.7): a computer program for population genetics simulations", Journal of Heredity, 92:301-302.

*   ECLIPSE (see also PANGAEA)

    *   full name: Error Correcting Likelihoods In Pedigree Structure Estimation.

    *   version: 1.1 (October 2003)

    *   description: A set of three programs, preproc, eclipse2 and eclipse3 which analyze genetic marker data for genotypic errors and pedigree errors. Using a single preprocessing program (preproc), eclipse2 analyzes data on pairs of individuals, and eclise3 analyzes data jointly on trios.

    *   authors: Solveig Sieberts (email: solly@stat.washington.edu) based on some earlier prototype routines by Jochen Kumm.

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml

    *   http://www.stat.washington.edu/thompson/Genepi/Eclipse.shtml

    *   source code language: C++

    *   operating systems: Tested on Unix (Compaq Tru64 V5.0A)

    *   executables: prepro, eclipse2, eclipse3

    *   reference:

*   EDAC

    *   full name: power calculator for Extremely Discordant And Concordant sibpairs

    *   version:

    *   description:

    *   authors: C Charles Gu

    *   web/ftp: http://www.biostat.wustl.edu/~gc/cgi-bin/calcpwr.cgi

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Gu, Todorov, Rao (1996), "Combining extremely concordant sibpairs with extremely discordant sibpairs provides a cost effective way to linkage analysis of QTL", Genetic Epidemiology, 13:513-533.

    *   Gu, Rao (1997), "A linkage strategy for detection of human quantitative-trait loci: I. Generalized relative risk ratios and power of sib pairs with extreme trait values",. American Journal of Human Genetics, 61(1):200-210

    *   Gu, Rao (1997), "A linkage strategy for detection of human quantitative-trait loci: II. Optimization of study designs based on extreme sib pairs and generalized relative risk ratios", American Journal of Human Genetics, 61(1):211-222

*   EH (see also EHPLUS, FASTEHPLUS, EHP)

    *   full name: Estimating Haplotype-frequencies

    *   version:

    *   description:

    *   authors: Xiaoli Xie, Jurg Ott (Columbia University; Rockefeller University)

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: Pascal

    *   operating systems: MS-DOS, UNIX

    *   executables: eh

    *   reference: Am Soc Hum Genet annual meeting 1993 (Xie, Ott (1993), "Testing linkage disequilibrium between a disease gene and marker loci", Am J Hum Genet, 53 (suppl), 1107 ([abstract](http://www.jurgott.org/linkage/absXieOtt1993.pdf))

    *   online user's guide

    *   a product review from: http://www.scienceboard.net/resources/prodreviews.asp?cat=22&product=440

*   EHAP

    *   full name:

    *   version: 2.0

    *   description: detecting association between haplotypes and phenotypes

    *   authors: Howard Seltman, Shawn Wood (University of Pittburg)

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/ehap__v1.htm

    *   source code language:

    *   operating systems:

    *   executables: ehap.exe, haplinmod.exe

    *   reference:

*   EHP (see also EH)

    *   full name: Estimation of Haplotype frequencies from Pooled DNA samples

    *   version:

    *   description: This program provides variance estimates for haplotype frequency estimates, it allows several kinds of missing information in the genotype data, it also allows for combined genotype data of different pool sizes. This program can be used for testing haplotype-disease associations in case control studies by calculating the likelihood ratio test: 2 log(likelihood for cases) + 2 log(likelihood for controls) - 2 log(likelihood for case+controls).

    *   authors: Yaning Yang (University of Science and Technology, Hefei, Anhui Province, China) (email: ynyang@gmail.com)

    *   source code language: R/S-plus

    *   operating systems:

    *   executables:

    *   reference: Y Yang et al. (2003), "Efficiency of single-nucleotide polymorphism haplotype estimation from pooled DNA", Proceedings of the National Academy of Sciences , 100(12):7225-7230.

*   EHPLUS (see also EH, FASTEHPLUS)

    *   full name:

    *   version: 2.0

    *   description:

    *   authors: Jinghua Zhao (MRC Epidemiology Unit at Strangeways Research Lab, UK) (email: jinghua.zhao@mrc-epid.cam.ac.uk)

    *   web/ftp: http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/

    *   source code language: C

    *   operating systems: UNIX(Solaris/DEC alpha/..), MS-Windows

    *   executables: ehplus

    *   reference: Zhao, Curtis, Sham (2000), "Model-free and permutation tests for allelic associations", Human Heredity, 50(2):133-139.

*   EIGENSOFT/EIGENSTRAT

    *   full name:

    *   version: 4.2 (August 2011)

    *   description:

    *   authors: Alkes Price, David Reich (Harvard Medical School)

    *   web/ftp: http://www.hsph.harvard.edu/faculty/alkes-price/software/

    *   http://genepath.med.harvard.edu/~reich/Software.htm

    *   http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm

    *   source code language:

    *   operating systems: Linux

    *   executables:

    *   reference: Price, Patterson, Plenge, Weinblatt, Shadick, Reich (2006), "Principal components analysis corrects for stratification in genome-wide association studies", Nature Genetics, 38(8):904-909.

*   EM-DECODER

    *   full name:

    *   version:

    *   description: a haplotype inference program

    *   authors: Steve Qin, (email: qin@stat.harvard.edu), Jun S Liu (email: jliu@stat.harvard.edu)

    *   web/ftp: http://www.people.fas.harvard.edu/~junliu/em/em.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Niu, Qin, Xu, Liu (2002), "Bayesian haplotype inference for multiple linked single-nucleotide polymorphisms", American Journal of Human Genetics, 70(1):157-169

*   EMLD

    *   full name: EM estimation of haplotype frequencies and LD calculation

    *   version: September 2003

    *   description: A quick and handy program to calculate pair-wise linkage disequilibrium based on SNP genotype data from unrelated individuals. EM algorithm is used to estimate pair-wise haplotype frequencies. The output file is in the format of input file for GOLD program, thus it can be directly plug into GOLD to get LD plots.

    *   authors: Dakai Zhu (email: dkz@mdanderson.org)

    *   web/ftp: http://epi.mdanderson.org/~qhuang/Software/pub.htm

    *   source code language: Java

    *   operating systems: UNIX, MS-Windows

    *   executables: emld

    *   reference:

*   ENDOG

    *   full name:

    *   version: 4.0 (November 2006)

    *   description: The program calculates individual inbreeding (F) and average relatedness (AR) coefficients. Additionally, users can compute useful parameters in population genetics such as: the number of ancestors explaining genetic variability; the genetic importance of the herds; F statistics from genealogical information.

    *   authors: Juan Pablo Gutierrez Garcia

    *   web/ftp: http://www.ucm.es/info/prodanim/html/JP_Web.htm

    *   source code language: FORTRAN 77

    *   operating systems:

    *   executables:

    *   reference:

*   ENTROPY BLOCKER

    *   full name:

    *   version: 1.7

    *   description: aiming at identifying "haplotype blocks". The likelihood of the data is calculated minus the model complexity. The resulting blocks have very low diversity and the linkage disequilibrium with SNP's outside the blocks is low.

    *   authors:

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/blocks.htm

    *   source code language: R

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference:

*   EPISTACY

    *   full name:

    *   version: 2.0

    *   description:

    *   authors: Jim Holland (email: james_holland@ncsu.edu)

    *   web/ftp: http://www4.ncsu.edu/~jholland/Epistacy/epistacy.htm

    *   source code language: SAS

    *   operating systems:

    *   executables:

    *   reference: Holland (1998), "EPISTACY: A SAS program for detecting two-locus epistatic interactions using genetic marker information", Journal of Heredity, 89:374-375.

*   EPDT

    *   full name: Extended Pedigree Disequilibrium Test

    *   version: 1.0 (2004)

    *   description: EPDT is a major advancement in detecting linkage/disequilibrium signals # between genetic markers and disease loci, particularly if only one # or a few large pedigrees are available. The strategy differs from # conventional approaches that require at least a moderate number of # families to attain adequate statistical power. The proposed testing # procedure is advantageous in that it provides high statistical power # coupled with reduced sample collection. Furthermore, the proposed # method avoids problems such as potential population stratification # and genetic heterogeneity, and is robust with respect to misspecification # of phenotype.

    *   authors: Chun-Liang Chen, Cathy SJ Fann (Institute of Biomedical Sciences, Academia Sinica, Taiwan)

    *   web/ftp: http://www.ibms.sinica.edu.tw/~csjfann/first%20flow/programlist.htm

    *   source code language:

    *   operating systems: UNIX

    *   executables:

    *   reference: Yang, Chen, Fann "A pedigree-based disequilibrium test for a single extended pedigree". preprint.

*   EQTL EXPLORER

    *   full name:

    *   version: 5.0

    *   description: EQTL EXPORER is an eQTL visualization tool that allows users to mine and understand data from a repository of genetical genomics experiments

    *   authors: Michael Mueller, Jonathan Mangion (Imperial College, UK)

    *   web/ftp: http://web.bioinformatics.ic.ac.uk/eqtlexplorer/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Mueller, Goel, Thimma, Dickens, Aitman, Mangion (2006), "eQTL Explorer: integrated mining of combined genetic linkage and expression experiments", Bioinformatics, 22(4):509-511.

*   ERPA

    *   full name: Extended Relative Pair Analysis

    *   version:

    *   description: non-parametric analysis

    *   authors: Dave Curtis (Royal London Hospital, UK) ( dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Curtis, Sham (1994), Annals of Human Genetics, 58:151-162.

*   ETDT

    *   full name: extended transmission/disequilibrium test

    *   version:

    *   description: TDT test on markers with more than two alleles using a logistic regression analysis.

    *   authors: P.C. Sham (King's College London, UK), Dave Curtis (Royal London Hospital, UK) ( dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Sham, Curtis (1995), Annals of Human Genetics, 59:323-336.

*   ET-TDT

    *   full name: Evolutionary Tree - Transmission Disequilibrium Test

    *   version:

    *   description:

    *   authors: Kathryn Roeder (Carnegie Mellon, email: roeder@stat.cmu.edu)

    *   web/ftp: http://www.stat.cmu.edu/~roeder/=ettdt/

    *   ftp://wpicf.wpic.pitt.edu/WPICCompgen/ettdt/index.html

    *   source code language:

    *   operating systems: MS-DOS

    *   executables: inferhap.exe, ettdt.exe

    *   reference: Selman, Roeder, Devlin (2001), "Transmission/Disequilibrium Test Meets Measured haplotype analysis: family-based association analysis guided by evolution of haplotypes", American Journal of Human Genetics, 68(5):1250-1263

*   EVOKER

    *   full name:

    *   version: May 2010

    *   description: Evoker is a graphical tool for visualizing genotype intensity data in order to assess genotype calls as part of quality control procedures for genome-wide association studies. It provides a solution to the computational and storage problems related to being able to work with the huge volumes of data generated by such projects by implementing a compact, binary format that allows rapid access to data, even with hundreds of thousands of observations.

    *   authors: James Morris, Jeffrey Barrett (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/evoker/

    *   http://sourceforge.net/projects/evoker/

    *   source code language: Java, Perl

    *   operating systems:

    *   executables:

    *   reference:

*   EXOMEPICKS

    *   full name:

    *   version:

    *   description: ExomePicks is a program that suggests individuals to be sequenced in a large pedigree. ExomePicks assumes that a genotyping chip or another cost effective means will be used to determine IBD sharing in the pedigree and that, subsequently, one would like to sequence a minimal number of individuals and use their sequences together with IBD information to deduce the sequence of other individuals in the pedigree. We are currently using it in the context of whole exome and whole genome sequencing studies to pick individuals to be sequenced from large family collections.

    *   authors: Abecasis' group (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/ExomePicks

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   FAMHAP

    *   full name:

    *   version: 16

    *   description: FAMHAP is a tool for haplotype association analysis of unphased genotype data. It can be used both for population data (case-control) and nuclear family data. The program is optimized for haplotype frequency estimation with the EM-algorithm for many markers. FAMHAP provides a method which searches for potential genotyping errors and several tests for haplotype-based association analysis. Particular emphasis is on Monte-Carlo simulations, which are necessary in the context of haplotype association, where asymptotic theory often fails, and in the context of multiple testing problems.

    *   authors: Tim Becker (email: tim.becker@ukb.uni-bonn.de), Michael Knapp

    *   web/ftp: http://www.uni-bonn.de/~umt70e/becker.html

    *   source code language: C

    *   operating systems: UNIX, MS-Windows, MacOS

    *   executables:

    *   reference: Becker, Knapp (2004), "Maximum-likelihood estimation of haplotype frequencies in nuclear families", Genetic Epidemiology, 27;21-32.

    *   Becker, Knapp (2004), "A powerful strategy to account for multiple testing in the context of haplotype analysis", American Journal of Human Genetics, 75:561-570

    *   Becker, Knapp (2005), "Impact of missing genotype data on Monte-carlo simulation based haplotype analysis", Human Heredity, 59:185-189.

    *   Becker, Cichon, Jönson, Knapp (2005), "Multiple testing in the context of haplotype analysis revisited: application to case-control data", Annals of Human Genetics, 69:1-10

    *   Becker, Schumacher, Cichon, Baur, Knapp (2005), "Haplotype interaction analysis of unlinked regions", Genetic Epidemiology, 29:313-322.

    *   Becker, Valentonyte, Croucher, Strauch, Schreiber, Hampe, Knapp (2006), "Identification of probable genotyping errors by consideration of haplotypes", European Journal of Human Genetics, 14:450-8.

    *   Becker, Baur, Knapp (2006), "Detection of parent-of-origin effects using haplotype analysis", Human Heredity, 62:64-76

*   FAMOZ

    *   full name: FAther/MOther

    *   version: 2003

    *   description: the program uses likelihood calculation and simulation to perform parentage studies with codominant, dominant, cytoplasmic markers or combinations of the different types

    *   authors: Sophie Gerber (email: sophie@pierroton.inra.fr), Patrick Chabrier, Antoine Kremer (INRA - UMR BIOGECO)

    *   web/ftp: http://www.pierroton.inra.fr/genetics/labo/Software/Famoz/index.html

    *   source code language: C, Tcl/Tk

    *   operating systems: UNIX(Solaris/..), Linux, MS-Windows

    *   executables:

    *   reference: S Gerber, S Mariette, R Streiff, C Bodénès, A Kremer (2000), "Comparison of microsatellites and AFLP markers for parentage analysis", Molecular Ecology, 9:1037-1048.

    *   S Gerber, P Chabrier, A Kremer (2003), "FaMoz: a software for parentage analysis using dominant, codominant and uniparentally inherited markers", Molecular Ecology Notes, 3(3):479-481.

*   FASTEHPLUS (see also EH, EHPLUS)

    *   full name: faster EH-PLUS

    *   version:

    *   description:

    *   authors: Jinghua Zhao (MRC Epidemiology Unit at Strangeways Research Lab, UK) (email: jinghua.zhao@mrc-epid.cam.ac.uk), Pak Sham (King's College London, UK)

    *   web/ftp: http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/

    *   source code language: C

    *   operating systems: MS-Windows, UNIX (DEC,OSF,Solaris)

    *   executables:

    *   reference: Zhao, Sham (2002), "Faster allelic association analysis using unrelated subjects", Human Heredity, 53(1):36-41.

*   FASTER

    *   full name: FAmily SmarT EliminatoR

    *   version: 1.0 beta

    *   description: The program is to reduce family members so the families can be used in GENEHUNTER program. FASTER eliminates the most redundant individuals according to the set of weights (preferences) (Age, Information about genetic Markers, etc.) The program has several features such as automatical reducement of branches without any affected individuals, etc.

    *   authors: Cyril Hoschl (Charles Univ, Pague)

    *   web/ftp: http://www.hoschl.cz/faster/

    *   source code language:

    *   operating systems: MS-Windows(9x/Me/2000/XP)

    *   executables:

    *   reference:

*   FASTLINK (see also LINKAGE)

    *   full name: faster version of LINKAGE

    *   version: 4.1P (July 4, 1999)

    *   description:

    *   authors: Cottingham, Gupta, Idury, Schaffer ( schaffer@helix.nih.gov), Shriram

    *   collaborators for version 4: Ann Becker, Dan Geiger.

    *   web/ftp: http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/fastlink.html

    *   ftp://ftp.ncbi.nlm.nih.gov/pub/fastlink (since Aug 6, 1996) or ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/FASTLINK/fastlink

    *   source code language: C

    *   operating systems: UNIX, VMS, MS-DOS,.. and can also run in parallel on shared memory UNIX machines

    *   executables:

    *   reference: Cottingham, Idury, Schaffer (1993), "Fast sequential genetic linkage computation", American Journal of Human Genetics, 53(1):252-263

    *   Schaffer, Gupta, Shriram, Cottingham (1994), "Avoiding recomputation in linkage analysis", Human Heredity, 44:225-237

    *   online README files (version 3.0P, december 95).

    *   Dwarkadas, Schaffer, Cottingham, Cox, Keleher, Zwaenepoel, "Parallelization of general linkage analysis problems"

    *   Gupta, Schaffer, Cox,Zwaenepoel, "Integrating parallelization strategies for linkage analysis"

    *   Schaffer, "Faster linkage analysis computations for pedigrees with loops or unused alleles"

    *   Becker, Geiger, Schaffer, "Automatic selection of loop breakers for genetic linkage analysis", Human Heredity

    *   Becker, Bar-Yehuda, Geiger, "Random algorithms for the loop cutset problem"

    *   Schaffer, "Loops in FASTLINK"

    *   Schaffer, "The mystery of (the) unknown"

    *   Schaffer, "Pedigree traversal in FASTLINK"

*   FASTMAP (1)

    *   full name: Fluorescent Allele-calling Software Toolkit: Microsatellite Automation Package

    *   version: 1.04b (April, 1997)

    *   description: fluorescent allele-calling software toolkit: a computer software for fully automated microsatellite genotyping.

    *   authors: Mark W. Perlin (Carnegie Mellon, perlin@cmu.edu), See-Kiong Ng ( skng@cs.cmu.edu), Nandita Mukhopadhyay, Lillian Markind Bloch

    *   web/ftp: http://www.cs.cmu.edu/~genome/FAST-MAP.html

    *   source code language:

    *   operating systems: UNIX, MacOS. MatLab programming environment is required.

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting 1996 (Ng,Mukhopadhyay, Rayman,Ghosth,Perlin, Am J Hum Genet, suppl 59, A225 (1996); Am Soc Hum Genet annual meeting 1997 (Perlin, Bloch, Mukhopadhyay, Ng, Am J Hum Genet, suppl 61, A290 (1997)).

*   FASTMAP (2)

    *   full name:

    *   version:

    *   description:

    *   authors: Dave Curtis (Royal London Hospital, UK) ( dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Curtis, Gurling (1993), Human Heredity, 43:173-185.

*   FASTSLINK (see also SLINK)

    *   full name: faster SLINK

    *   version:

    *   description: a faster version of SLINK

    *   authors: Daniel E. Weeks (University of Oxford and University of Pittsburgh)

    *   web/ftp: http://watson.hgen.pitt.edu/register/soft_doc.html

    *   registration page at: http://watson.hgen.pitt.edu/register

    *   source code language: C

    *   operating systems: UNIX

    *   executables: slink

    *   reference: Am Soc Hum Genet annual meeting 1990 (Weeks, Ott, Lathrop, Am J Hum Genet, suppl 47, A204 (1990)).

    *   online manual pages (same as slink)

*   FBAT (see also PBAT)

    *   full name: (haplotype) Family Based Association Test

    *   version: Nov 2003

    *   description: Haplotype FBAT is a user-friendly, well-documented software that allows the user to test for association/linkage between disease phenotypes and haplotypes by utilizing family-based controls. The method extends the approach for testing described in Rabinowitz and Laird (2000) to handle multiple tightly linked markers. It is robust to population admixture, yet efficient in the sense that it utilizes data from families where phase cannot be completely resolved in all individuals by using weights, which are estimated from the sample. However, the method remains robust to population stratification and population admixture. The method can handle any type of phenotype, including multiple phenotypes and missing parents, marker data, and/or phase, and provides both bi-allelic and multi-allelic tests. PowerFBAT is a tool for power simulation of association analysis using FBAT with binary outcomes. XWXW is an extension to the Haseman-Elston method for non-parametric linkage test with quantitative traits. XDT is a software that performs classical TDT, "SDT" and "Rabinowitz TDT" for nuclear families (not supported anymore).

    *   authors: Xin Xu, Steve Horvath, Nan Laird (Harvard Univ)

    *   web/ftp: http://www.biostat.harvard.edu/~fbat/fbat.htm

    *   source code language:

    *   operating systems: UNIX(Solaris, Linux, MS-Windows, Mac(MacOS X/Darwin)

    *   executables: fbat, pbat, powerfbat, xwxw, xdt

    *   reference: Laird NM (2004) Family based tests for associating haplotypes with general phenotype data: application to asthma genetics. Genetic Epidemiology, 26(1):61-69;

    *   Laird, Horvath, Xu (2000), Implementing a unified approach to family based tests of association, Genetic Epi, supp 1, 19:36-42; Rabinowitz, Laird (2000), A unified approach to adjusting association tests for population admixture with arbitrary pedigree structure and arbitrary missing marker information, Human Heredity, 50:211-223.

    *   Horvath, Xu, Laird (2001), The family based association test method: strategies for studying general genotype-phenotype associations, European J Hum Genet. 9:301-306.

*   FEST (see R/FEST)

*   FESTA

    *   full name: Fragmented Exhaustive Seearch for TAgsnps

    *   version: March 2005

    *   description:

    *   authors: Shyam Gopalakrishnan (email: gopalakr@umich.edu), Steve Qin (email: qin@umich.edu) (University of Michigan)

    *   web/ftp: http://www.sph.umich.edu/csg/qin/FESTA/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   FINESSE (see also VITESSE)

    *   full name:

    *   version:

    *   description: To incorporate regressive models into the VITESSE likelihood engine

    *   authors: Jeff O'Connell (email: joconnel@medicine.umaryland.edu), Florence Demenais (email: florence.demenais@evry.inserm.fr)

    *   web/ftp: http://www.molecular-haplotype.org/finesse/finesse_intro.htm

    *   source code language:

    *   operating systems: Linux, UNIX(Solaris)

    *   executables:

    *   reference:

*   FINETTI

    *   full name: de FINETTI generator

    *   version: 3.0.5 (2008)

    *   description: Tests for deviation from Hardy-Weinberg equilibrium and tests for association in case controls studies; Plot genotype frequencies graphically using a de Finetti diagram.

    *   authors: Thomas F Wienker (email: wienker@uni-bonn.de) (University of Bonn, Germany), Web implementation: Tim M Strom (email: timstrom@gsf.de) (GSF Munich, Germany), Henning Henschke (email: hhenschk@imbie.meb.uni-bonn.de).

    *   web/ftp: http://ihg.gsf.de/cgi-bin/hw/hwa1.pl (testing part)

    *   http://finetti.meb.uni-bonn.de/ (graphic part, 2008 version)

    *   http://ihg.gsf.de/linkage/download/finetti.zip (graphic part, 2004 version)

    *   source code language: Pascal (web implementation: Perl, C, php)

    *   operating systems: web-based, MS-DOS, MS-Windows(32), Linux (for stand-alone version)

    *   executables:

    *   reference:

*   FIRSTORD

    *   full name:

    *   version:

    *   description: demonstration of method for preliminary ordering of loci based on two-point lod scores.

    *   authors: Dave Curtis (Royal London Hospital, UK) ( dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C

    *   operating systems: MS-DOS

    *   executables:

    *   reference:

*   FISHER

    *   full name:

    *   version:

    *   description: genetic analysis of classical biometric traits like blood pressure or height that are caused by a combination of polygenic inheritance and complex environmental forces.

    *   authors: Kenneth Lange (IUCLA), Daniel E. Weeks (Pittsburgh), M. Boehnke

    *   web/ftp: registration page at: http://www.biomath.medsch.ucla.edu/faculty/klange/software.html

    *   source code language: FORTRAN77

    *   operating systems:

    *   executables:

    *   reference: Lange, Weeks, Boehnke, Genet Epid, 5, 471-471 (1988)

*   FLOSS

    *   full name: FLexible Ordered SubSet analysis

    *   version: 1.4.1

    *   description: FLOSS performs ordered subset analysis using MERLIN's ouput ".lod" file created with the --perFamily option. Ordered subset analysis uses covariate information to identify a more homogenous subset of families for linkage analysis. The homogeneous subset of families does not need to be specified a priori, and the covariates can include environmental exposures, quantitative traits, or linkage scores at another locus in the genome. The evidence for linkage is evaluated with a permutation test.

    *   authors: Brian L Browning

    *   web/ftp: http://faculty.washington.edu/browning/floss/floss.htm

*   GAIA

    *   full name: Genetic Association Interaction Analysis

    *   version: 1.1 (2006)

    *   description: GAIA is an easy to use, web-based application for testing for locus-locus interaction using genetic association. It is based upon the case-control study design and is designed so that non-specialists may routinely apply tests for interaction. GAIA allows simple testing of both additive and additive plus dominance interaction models and includes permutation testing to appropriately correct for multiple testing. The application is useful for both candidate gene based studies and genome-wide association studies. For large scale studies GAIA includes a screening approach which prioritizes loci for further interaction analysis.

    *   authors: Stuart Macgregor (QIMR, Brisbane, Australia and Cardiff University, UK) (email: stuart.macgregor@qimr.edu.au), Imtiaz Khan

    *   web/ftp: http://www.bbu.cf.ac.uk/html/research/biostats.htm

    *   source code language: perl, R

    *   operating systems: web-based

    *   executables:

    *   reference: Macgregor and Khan (2006), "GAIA: An easy-to-use web-based application for interaction analysis of case-control data", BMC Medical Genetics, 7:34

*   GAP

    *   full name: genetic analysis package

    *   version: 1.0

    *   description: a comprehensive package for the management and analysis of pedigree data. it offers: 1\. Powerful database management tools, specifically designed for family data; 2\. Automatic pedigree drawing; 3. Segregation and linkage analysis, based on traditional maximum likelihood methods and newer, more powerful, Monte Carlo methods that can model both genetic and environmental factors.

    *   authors: "Epicenter Software": Jonathan Buckley (Dept of Preventive Medicine, USC, jbuckley@hsc.usc.edu), Jim Gauderman, Duncan Thomas, Cheryl Faucett and John Morrison ( jmorr@rcf.usc.edu).

    *   web/ftp: http://www.epicentersoftware.com/products.php

    *   source code language: c++

    *   operating systems: MS-Windows (3.1/95/NT), UNIX(Solaris)

    *   executables:

    *   reference:

    *   availability: price single user: US$795; LAN version: US$1295\. Kindred only: US$495\. (academic prices are US$595, US$1095, US$395, respectively)

*   GAS

    *   full name: Genetic Analysis System

    *   version: 2.0 (Dec 1995)

    *   description: statistical analysis of genetic linkage data, sib-pair analysis, association studies

    *   authors: Alan Young ( ayoung@vax.ox.ac.uk)

    *   web/ftp: http://users.ox.ac.uk/~ayoung/gas.html .

    *   source code language:

    *   operating systems: MS-DOS, UNIX(SunOS/Solaris/AIX/OSF1/Ultrix/SGI-IRIX),VMS

    *   executables:

    *   reference:

*   GAS2

    *   full name: statistical significance in Genomewide Association Studies in 2-stage

    *   version:

    *   description: Evaluating Statistical Significance in Two-Stage Genomewide Association Studies

    *   authors:

    *   web/ftp: http://www.bios.unc.edu/~lin/software/GAS2/

    *   source code language: FORTRAN77

    *   operating systems:

    *   executables:

    *   reference: Lin (2006), "Evaluating statistical significance in two-stage genomewide association studies", American Journal of Human Genetics, 78(3):505-509

*   GASP

    *   full name: Genometric Analysis Simulation Program

    *   version: 3.31 (April 1998)

    *   description: a software tool for testing and investigating methods in statistical genetics by generating samples of family data based on user specified models.

    *   authors: A.F. Wilson ( afw@nhgri.nih.gov), J.E.Bailey-Wilson, E.W. Pugh, A.J.M. Sorant (NCHGR/NIH)

    *   web/ftp: http://www.nhgri.nih.gov/DIR/IDRB/GASP/

    *   ftp://nhgri.nih.gov/pub/outgoing/gasp

    *   source code language: FORTRAN77

    *   operating systems: UNIX(DEC-UNIX 4.0B/Solaris 2.5/SGI-IRIX 6.2/..)

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting 1996 (Wilson, Bailey-Wilson, Pugh, Sorant, Am J. Hum Genet, suppl, 59, A193 (1996)); documentation draft (in postscript)

*   GASSOC

    *   full name: Genetic ASSOCiation analysis software for cases and parent

    *   version: 1.06 (Jun 2001)

    *   description: Statistical methods for disease and genetic marker associations using cases and their parents. These methods include an extension of the transmission/disequilibrium test (TDT) for multiple marker alleles, as well as additional general tests sensitive to associations that depend on dominant or recessive genetic mechanisms.

    *   authors: D.J. Schaid ( schaid@mayo.edu), C. M. Rowland (Mayo Clinic/Foundation)

    *   web/ftp: http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm

    *   source code language: C

    *   operating systems: UNIX(SunOS/Solaris/..)

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting 1996 (Rowland, Schaid, Am J. Hum Genet, suppl, 59, A234 (1996)).

    *   Schaid (1996), Genetic Epidemiology, 13:423-449.

*   GATK

    *   full name: Genome Analysis ToolKit

    *   version:

    *   description: The GATK is a structured software library that makes writing efficient analysis tools using next-generation sequencing data very easy, and second it's a suite of tools for working with human medical resequencing projects such as 1000 Genomes and The Cancer Genome Atlas. These tools include things like a depth of coverage analyzers, a quality score recalibrator, a SNP/indel caller and a local realigner.

    *   authors: Mark DePristo et al. (Broad Institute)

    *   web/ftp: http://www.broadinstitute.org/gsa/wiki/index.php/The_Genome_Analysis_Toolkit

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: DePristo, Banks, Poplin, Garimella, Maguire, Hartl, Philippakis, del Angel, Rivas, Hanna, McKenna, Fennell, Kernytsky, Sivachenko, Cibulskis, Gabriel, Altshuler, Daly (2011), "A framework for variation discovery and genotyping using next-generation DNA sequencing data", Nature Genetics, 43:491-498.

*   GC/GCF

    *   full name: Genomic Control

    *   version:

    *   description: GC implements the genomic control models. GCF implements the basic Genomic Control approach, but adjusts the p-values for uncertainty in the estimated effect of substructure. This approach is preferable if a large number of tests will be evaluated because it provides a more accurrate assessment of the significance level for small p-values.

    *   authors: B Devlin, K Roeder

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/genomic_control/genomic_control.htm

    *   source code language: R

    *   operating systems: Linux

    *   executables:

    *   reference: Devlin, Roeder (1999), "Genomic control for association studies", Biometrics, 55:997-1004.

    *   Bacanu, Devlin, Roeder (2002), "Association studies for quantitative traits in structured populations", Genetic Epidemiology, 22:78-93.

    *   Devlin, Bacanu, Roeder (2004), "Genomic control in the extreme", Nature Genetics, 36(11):1129-1130.

*   GCHAP (see also GENEPI.JAR)

    *   full name: Gene Counting method for HAPlotype analysis.

    *   version:

    *   description: This program finds maximum likelihood estimates of haplotype frequencies from a sample of genotyped individuals. By excluding haplotypes with zero MLE at an early stage, this implementation uses many orders of magnitude less space and time than naive implementations. A second program, ApproxGCHap, is provided to give alternate estimates for data sets with large numbers of loci or large amounts of missing genotypes.

    *   authors: Alun Thomas (email: alun@genepi.med.utah.edu)

    *   web/ftp: http://episun7.med.utah.edu/~alun/gchap/index.html

    *   http://balance.med.utah.edu/wiki/index.php/GCHap

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Thomas (2003), "GCHap: fast MLEs for haplotype frequencies by gene counting", Bioinformatics, 19:2002-2003.

    *   Thomas (2003), "Accelerated gene counting for haplotype frequency estimation", Annals of Human Genetics, 67:608-612.

*   GDA

    *   full name: Genetic Data Analysis

    *   version: 1.0d16c (Jan 2001)

    *   description: designed to accompany the second edition of Bruce Weir's book "Genetic Data Analysis" (1996\. Sinauer Associates)

    *   authors: Paul Lewis, Dmitri Zaykin

    *   web/ftp: http://hydrodictyon.eeb.uconn.edu/people/plewis/software.php

    *   source code language:

    *   operating systems: MS-Windows (3.1/95/NT), MacOS (10.2.8/10.3)

    *   executables:

    *   reference:

*   GEMS

    *   full name: Genetic Epidemiology ModelS

    *   version: 1.0

    *   description: a computer software for fitting Genetic Epidemiology Models. This version of GEMS provides the features for many statistical analyses of family data:

    *   authors: GE Bonney, V Apprey, J Kwagyan (Howard Univ)

    *   web/ftp: http://statgen.genomecenter.howard.edu/gems.htm

    *   source code language:

    *   operating systems: UNIX(DEC-alpha), Linux

    *   executables: gems

    *   reference: Am Soc Hum Genet annual meeting 1997 (Bonney, Slater, Slifker, Am. J. Hum. Genet. suppl 61, A212 (1997)); Bonney, Apprey, Kwagyan (2001), "A computer software for fitting Genetic Epidemiology Models", tech report 2001-02.

*   GENABEL (see R/GENABEL)

*   GENECLASS

    *   full name:

    *   version: v2.0g (Jan 2005)

    *   description: GENECLASS2 is a program employing multilocus genotypes to select or exclude populations as origins of individuals (Assignment and Migrants Detection).

    *   authors: Jean-Marie Cornuet

    *   web/ftp: http://www.montpellier.inra.fr/URLB/

    *   source code language: Delphi

    *   operating systems: MS-Windos, UNIX

    *   executables:

    *   reference: Piry, Alapetite, Cornuet, Paetkau, Baudouin, Estoup (2004), "GeneClass2: a software for genetic assignment and first-generation migrant detection", Journal of Heredity, 95:536-539.

*   GENECOUNTING

    *   full name:

    *   version: 2.2

    *   description: gene-counting for haplotype analysis with permutation tests for global association and specific haplotypes, accounting for missing data

    *   authors: Jinghua Zhao (MRC Epidemiology Unit at Strangeways Research Lab, UK) (email: jinghua.zhao@mrc-epid.cam.ac.uk)

    *   web/ftp: http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/

    *   source code language: C

    *   operating systems: MS-Windows, UNIX(Solaris), Linux

    *   executables: gc, gcp, gcx, pgc., RUNGC, SCANASSOC, COMPGR, SCANGROUP, LDPAIRS

    *   reference: Zhao, Lissarrague, Essioux, Sham (2002), "GENECOUNTING: haplotype analysis with missing genotypes", Bioinformatics, 18(12):1694-1695.

    *   Zhao (2004), "2LD, GENECOUNTING and HAP: Computer programs for linkage disequilibrium analysis", Bioinformatics, 20:1325-1326.

    *   Curtis, Knight, Sham (2006), "Program report: GENECOUNTING support programsProgram report: GENECOUNTING support programs", Annals of Human Genetics, 70:277-279.

*   GENEFINDER

    *   full name:

    *   version: 1.1.9 (June 2004)

    *   description: GeneFinder uses GEE method to estimate the location of the susceptibility gene based on the allele-sharing of affected sib pairs for multiple markers. GENEFINDER can further the analysis of GENEHUNTER by providing a more accurate gene location estimation and the corresponding confidence interval.

    *   authors: Wei-Min Chen (email: wechen@jhsph.edu), Kung-Yee Liang (email: kyliang@jhpsh.edu), Yen-Feng Chiu

    *   web/ftp: http://www.biostat.jhsph.edu/~wmchen/gf.html

    *   source code language: C++

    *   operating systems: MS-Windows, MS-DOS, UNIX(Solaris)

    *   executables: gf

    *   reference: Liang KY, Zeger SL (1986), "Longitudinal data analysis using generalized linear models", Biometrika, 73: 13-22.

    *   Liang KY, Chiu YF, Beaty TH (2001), "A robust identity by descent procedure using affected sib pairs: a multipoint approach for complex diseases", Human Heredity, 51:64-78.

    *   Liang KY, Chiu YF, Beaty TH, Wjst Matthias (2001), "A multipoint analysis using affected sib pairs: incorporation of linkage evidence from unlinked regions", Genetic Epidemiology, 21:105-122.

    *   Liang KY, Huang CY, Beaty TH (2000) A unified sampling approach for multipoint analysis of qualitative and quantitative traits in sib pairs. American Journal Human Genetics, 66(5):1631-1641

    *   Glidden DV, Liang KY, Chiu YF, Pulver AE (2003), "Multipoint affected sibpair linkage methods for localizing susceptibility genes of complex diseases", Genetic Epidemiology, 24:107-117

*   GENEHUNTER (see also ALLEGRO, GENEHUNTER-IMPRINTING, GENEHUNTER-MODSCORE, GENEHUNTER-PLUS, GENEHUNTER-TWOLOCUS)

    *   full name:

    *   version: 2.1 release 4 (April 2003)

    *   description: multipoint analysis of pedigree data including: non-parametric linkage analysis, LOD-score computation, information-content mapping, haplotype reconstruction

    *   authors: Leonid Kruglyak ( leonid@genome.wi.mit.edu), Mark Daly (mjdaly@genome.wi.mit.edu), Mary Pat Reeve-Daly ( mpreeve@genome.wi.mit.edu), Eric Lander ( lander@genome.wi.mit.edu) (Whitehead Institute)

    *   web/ftp: http://www.broad.mit.edu/ftp/distribution/software/genehunter/

    *   source code language: C

    *   operating systems: UNIX

    *   executables: gh

    *   reference: Kruglyak,Daly,Reeve-Daly,Lander (1996), "Parametric and nonparametric linkage analysis: a unified multipoint approach", American Journal of Human Genetics, 58:1347-1363

    *   online documentation (2.0beta)

    *   documentation (in PDF) (2.0beta).

    *   online instruction (1.1, from Falling Rain Genomics, Inc.)

*   GENEHUNTER-IMPRINTING (see also ALLEGRO, GENEHUNTER, GENEHUNTER-MODSCORE, GENEHUNTER-PLUS, GENEHUNTER-TWOLOCUS)

    *   full name:

    *   version: version: 1.3 (May 2000), 2.1 beta (Oct 2001).

    *   description: GENEHUNTER-IMPRINTING is a modification of the GENEHUNTER software package (version 1.3). It allows to perform parametric (LOD-score) analysis of traits caused by imprinted genes - that is, of traits showing a parent-of-origin effect.

    *   authors: K Strauch strauch@med.uni-marburg.de) (Philipps University Marburg, Germany)

    *   web/ftp: http://www.staff.uni-marburg.de/~strauchk/software.html

    *   source code language: C

    *   operating systems: UNIX(SUNOS,Solaris,OSF,HPUX,AIX,Ultrix), Linux, MS-Windows

    *   executables: ghi, xghi

    *   reference: Strauch, Fimmers, Kurz, Deichmann, Wienker, Baur (2000), American Journal of Human Genetics, 66(6):1945-1957

*   GENEHUNTER-MODSCORE

    *   full name:

    *   version: 2.0 (October 2006)

    *   description: GENEHUNTER-MODSCORE is a further extension of GENEHUNTER-IMPRINTING. The program is based on the original GENEHUNTER version 2.1 release 6\. GENEHUNTER-MODSCORE allows for a MOD-score analysis, in which parametric LOD scores are maximized over the parameters of the trait model, i.e., the penetrances and disease allele frequency. As of version 2.0, it is possible to use sex-specific recombination frequencies. The genetic positions of markers can be automatically read from a publicly available genetic map.

    *   authors: Konstantin Strauch (strauch@med.uni-marburg.de) (Philipps University Marburg, Germany) Johannes Dietter (dietter@med.uni-marburg.de) (Philipps University Marburg, Germany)

    *   web/ftp: http://www.staff.uni-marburg.de/~strauchk/software.html

    *   source code language: C

    *   operating systems: UNIX(SUNOS,Solaris,OSF,HPUX,AIX,Ultrix), Linux, MS-Windows

    *   executables: ghm

    *   reference: K Strauch (2003), "Parametric linkage analysis with automatic optimization of the disease model parameters". American Journal of Human Genetics 73(Suppl1):A2624.

    *   Dietter, Mattheisen, Fuerst, Rueschendorf, Wienker, Strauch (2007), "Linkage analysis using sex-specific recombination fractions with GENEHUNTER-MODSCORE", Bioinformatics, 23(1):64-70.

*   GENEHUNTER-PLUS (see also ALLEGRO, GENEHUNTER, GENEHUNTER-IMPRINTING, GENEHUNTER-MODSCORE, GENEHUNTER-TWOLOCUS)

    *   full name:

    *   version: 1.2 (Nov 1998)

    *   description: GENEHUNTER-PLUS is a modification of the GENEHUNTER software package which produces output files containing the null and conditional distributions of the test statistic (in nullprobs.dat and probs.dat, respectively). These files can then be used as input to the ASM program which incorporates the allele sharing modeling for lodscores and likelihood ratio tests as developed by Kong and Cox (1997).

    *   authors: Augustine Kong (kong@galton.uchicago.edu), Mike Frigge.

    *   web/ftp: http://galton.uchicago.edu/genehunterplus

    *   source code language: C

    *   operating systems:

    *   executables: ghp, xghp,asm

    *   reference: Kong, Cox (1997), "Allele-sharing models: LOD scores and accurate linkage tests", American Journal of Human Genetics, 61(5):1179-1188

*   GENEHUNTER++SAD

    *   full name:

    *   version:

    *   description: implementation of the Sad statistic, more robust to transmission ratio distortion in the context of allele sharing

    *   authors: Mathieu Lemire (email: mlemire@sdf.lonestar.org)

    *   web/ftp: http://mlemire.freeshell.org/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Lemire, Roslin, Laprise, Hudson, Morgan (2004), "Transmission-ratio distortion and allele sharing in affected sib pairs: a new linkage statistic with reduced bias, with application to chromosome 6q25.3", American Journal of Human Genetics, 75:571-586.

*   GENEHUNTER-TWOLOCUS (see also ALLEGRO, GENEHUNTER, GENEHUNTER-IMPRINTING, GENEHUNTER-MODSCORE, GENEHUNTER-PLUS)

    *   full name:

    *   version: 1.3 (May 2000), optimized and parallelized version (May 2004)

    *   description: GENEHUNTER-TWOLOCUS is a modification of the GENEHUNTER software package (version 1.3). The program performs complete parametric and nonparametric multi-marker linkage analysis with two autosomal, biallelic disease loci.

    *   authors: K Strauch strauch@med.uni-marburg.de) (Philipps University Marburg, Germany), Johannes Dietter ( dietter@med.uni-marburg.de) (Philipps University Marburg, Germany)

    *   web/ftp: http://www.staff.uni-marburg.de/~strauchk/software.html

    *   source code language: C

    *   operating systems: UNIX(SUNOS,Solaris,OSF,HPUX,AIX,Ultrix), Linux, MS-Windows

    *   executables: ght

    *   reference: Strauch, Fimmers, Kurz, Deichmann, Wienker, Baur (2000), "Parametric and nonparametric multipoint linkage analysis with imprinting and two-locus-trait models: application to mite sensitization", American Journal of Human Genetics, 66(6):1945-1957

    *   Dietter, Spiegel, an Mey, Pflug, Al-Kateb, Hoffmann, Wienker, Strauch (2004): "Efficient two-trait-locus linkage analysis through program optimization and parallelization: application to hypercholesterolemia", European Journal of Human Genetics, 12:542-550.

*   GENEPI.JAR (see also, JPSGCS)

    *   full name:

    *   version: September 2006

    *   description: a set of Java programs for genetic epidemiology analysis

    *   authors: Alun Thomas (University of Utah)

    *   web/ftp: http://bioinformatics.med.utah.edu/~alun/software/index.html

    *   http://balance.med.utah.edu/wiki/index.php/JPSGCS

    *   source code language: Java

    *   operating systems: web-based

    *   executables: CheckFormat, CheckError, DownCodeAlleles, GeneCountAlleles, SelectLoci, SelectKindreds, TrimPed, TwoPointLods, MaxTwoPointLods, McLink, McLinkLD, HapGraph, GaploFreqs, GCHap, ApproxGCHap, ViewGraph, ViewPed, ViewLinkPed,

    *   reference:

*   GENEPOOL

    *   full name:

    *   version: 0.84

    *   description: GenePool is a software package that provides analysis tools for the detection of shifts in relative allele frequency between pooled genomic DNA from cases and controls using SNP-based genotyping microarrays. GenePool supports genotyping platforms from Affymetrix and Illumina.

    *   authors: John Pearson (Translational Genomics Research Institute) (email: jpearson@tgen.org) David Craig (email: dcraig@tgen.org) Matt Huentelman (email: mhuentelman@tgen.org) Waibhav Tembe, Sotiris Mitranopolous, Nils Homer

    *   web/ftp: http://genepool.tgen.org/

    *   source code language: C/C++/Perl

    *   operating systems: Linux, MacOS (should compile on most UNIX systems with gcc)

    *   executables: gpcommand, gpextract, gpanalyze

    *   reference: Pearson, Huentelman, Halperin, Tembe, Melquist, Homer, Brun, Szelinger, Coon, Zismann, Webster, Beach, Sando, Aasly, Heun, Jessen, Kolsch, Tsolaki, Daniilidou, Reiman, Papassotiropoulos, Hutton, Stephan, Craig (2007), "Identification of the genetic basis for complex disorders by use of pooling-based genomewide single-nucleotide-polymorphism association studies", American Journal of Human Genetics, 80:126-139

*   GENEPOP

    *   full name:

    *   version: 3.4

    *   description: a population genetics software package

    *   authors: Michel Raymond (email: raymond@isem.univ-montp2.fr), Francois Rousset ( email: rousset@isem.univ-montp2.fr), Eleanor Morgan (email: nell@wbiomed.curtin.edu.au) (web version)

    *   web/ftp: http://wbiomed.curtin.edu.au/genepop/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Raymond, Rousset (1995), "GENEPOP (version 1.2): population genetics software for exact tests and ecumenicism", Journal of Heredity, 86:248-249

*   GENERECON

    *   full name:

    *   version:

    *   description: GeneRecon is a software package for linkage disequilibrium mapping using coalescent theory. It is based on a Bayesian Markov-chain Monte Carlo (MCMC) method for fine-scale linkage-disequilibrium gene mapping using high-density marker maps. GeneRecon explicitly models the genealogy of a sample of the case chromosomes in the vicinity of a disease locus. Given case and control data in the form of genotype or haplotype information, it estimates a number of parameters, most importantly, the disease position.

    *   authors: Thomas Mailund (email: mailund@birc.au.dk) (Univ of Oxford, UK and Univ of Aarhus, Denmark)

    *   web/ftp: http://www.daimi.au.dk/~mailund/GeneRecon/

    *   source code language: C++

    *   operating systems: UNIX, Linux, MacOS, MS-Windows

    *   executables:

    *   reference: Mailund, Schierup, Pedersen, Madsen, Hein, Schauser (2006), "GeneRecon -- A coalescent based tool for fine-scale association mapping", Bioinformatics, 22(18):2317-2318.

*   GENESPRING GT (previously VARIA)

    *   full name:

    *   version: 2.0

    *   description: GeneSpring GT is a desktop analysis workbench for analyzing high-volume, high-density genotyping data. The software provides a comprehensive set of linkage and association algorithms that allow researchers to discover relationships between genotypes and phenotypes. Researchers can visually explore fully annotated SNPs and genes at varying levels of detail. Designed for biologists and statisticians, GeneSpring GT is capable of importing, visualizing, and analyzing hundreds of thousands of variation measurements simultaneously, for rapid localization of disease or phenotype markers.

    *   authors: Agilent Technologies (previously Silicon Genetics)

    *   web/ftp: http://www.chem.agilent.com/scripts/pds.asp?lpage=34662

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

    *   price: for quote request, go to http://www.chem.agilent.com/scripts/cag_requestaquote.asp

*   GENETIC POWER CALCULATOR

    *   full name:

    *   version: 2001-2005

    *   description: automated power analysis for variance components (VC) quantitative trait locis (QTL) linkage and association tests in sibships, and other common tests

    *   authors: S. Purcell, P Sham

    *   web/ftp: http://pngu.mgh.harvard.edu/~purcell/gpc/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Sham, Cherny, Purcell, Hewitt (2000), "Power of linkage versus association analysis of quantitative traits, by use of variance-components models, for sibship data", American Journal of Human Genetics, 66(5):1616-1630

    *   Purcell, Cherny, Sham (2003), "Genetic Power Calculator: design of linkage and association genetic mapping studies of complex traits", Bioinformatics, 19(1):149-150.

*   GENETSIM

    *   full name:

    *   version:

    *   description: GENETSIM provides flexible simulations of family data within an easy-to-use, high-level programming language. GENETSIM has no limit on pedigree sizes or structures, or number of families, or number of marker, or number of chromosomes. Genetic transmission is modeled by first generating the locations of recombination events, and then performing gene dropping according to the given recombination pattern. Any pattern of missing data can be specified and genotyping errors can be simulated. GENETSIM can simulat multiple QTLs with pleiotropic effects, multivariate polygenic background and any number of environmental factors, age effects, epistasis and variable expression. Users can also select families based on ascertainment schemes by repeating simulations.

    *   authors: Mike Miller, Na Li (Univ of Minnesota)

    *   web/ftp: http://taxa.epi.umn.edu/GenetSim/

    *   source code language: Octave, R (future version)

    *   operating systems:

    *   executables:

    *   reference: Miller, Li (2005), "GenetSim: software for simulation of familial data in genetics and epidemiology" (abstract), Genetic Epidemiology, 29:269.

*   GENEVAR

    *   full name: GENe Expression VARiation

    *   version:

    *   description: Genevar is a database and Java tool designed to integrate multiple datasets, and provides analysis and visualization of associations between sequence variation and gene expression in eQTL studies. Genevar allows researchers to investigate eQTL (expression quantitative trait loci) associations within a gene locus of interest in real time. The database and application can be installed on a standard computer in database mode and, in addition, on a server to share discoveries among affiliations or the broader community over the internet via web services protocols.

    *   authors:

    *   web/ftp: http://www.sanger.ac.uk/resources/software/genevar/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Yang, Beazley, Montgomery, Dimas, Gutierrez-Arcelus, Stranger, Deloukas, Dermitzakis (2010), "Genevar: a database and Java application for the analysis and visualization of SNP-gene associations in eQTL studies", Bioinformatics, 24(18):2105-2107.

*   GENEWEAVER

    *   full name:

    *   version: 1.1 (2003)

    *   description: software for charting family medical/health history

    *   authors:

    *   web/ftp: http://www.geneweaveronline.com/

    *   source code language:

    *   operating systems: MS-Windows (95/98/2000/NT/XP)

    *   executables:

    *   reference:

    *   availability: commercialUS$39.95

*   GENIE

    *   full name:

    *   version: 2.7.2.1 (2010)

    *   description: GENIE (previously PEDGENIE and HAPMC) performs tests of association and transmission disequilibrium (TDT) between genetic markers and traits in studies of arbitrarily-sized families and/or independent individuals using Monte Carlo testing. For dichotomous traits, basic genotype-based or allele-based Chi-square statistics, OR, and a Chi-square trend statistic with user-defined weights, TDT, sib-TDT, combined-TDT are included. For quantitative outcomes, a difference in means test, ANOVA and QTDT are offered. Flexible haplotype testing and meta analysis across multiple centers are available. An automated haplotype building module, hapConstructor, is also offered that data mines multi-locus data for association signals. The Monte Carlo empirical significance assessment accounts for all relatedness between individuals for all tests.

    *   authors: Nicola Camp, K Allen-Brady, Jathine Wong, K Curtin, R Abo

    *   web/ftp: http://bioinformatics.med.utah.edu/Genie/index.html

    *   source code language: Java 1.6

    *   operating systems: web-based

    *   executables: PedGenie, hapMC, hapConstructor

    *   reference: Allen-Brady, Farnham, Weiler, Camp (2003), "A cautionary note on the appropriateness of using a linkage resource for an association study", BMC Genetics, 4(suppl 1):S89.

    *   Allen-Brady, Wong, Camp (2006), "PedGenie: an analysis approach for genetic association testing in extended pedigrees and genealogies of arbitrary size", BMC Bioinformatics, 7:209.

    *   Curtin, Wong, Allen-Brady, Camp (2007), "PedGenie: meta genetic association testing in mixed family and case-control designs", BMC Bioinformatics, 8:448.

    *   Abo, Knight, Wong, Cox, Camp (2008), "hapConstructor: automatic construction and testing of haplotypes in a Monte Carlo framework", Bioinformatics, 24(18):2105-2107.

*   GENOCHECK

    *   full name: GenoCheck

    *   version: 1.0 (sept 20, 1995)

    *   description: GenoCheck is a program designed to identify genotypes that are likely to be errors. The statistical method was designed to identify typing error rather than pedigree errors, but is general enough to pinpoint any unlikely genotype still consistent with Mendelian inheritance.

    *   authors: Meg Gelder Ehm (Glaxo Wellcome Inc. mge37216@glaxowellcome.com), Robert W. Cottingham, Marek Kimmel, Alejandro Schaffer, Ramana Idury, Shriram Krishnamurthi, Sandeep Gupta, G. Mark Lathrop

    *   web/ftp: http://softlib.rice.edu/geno.html

    *   source code language: C

    *   operating systems: UNIX

    *   executables: GenoCheck, toaff

    *   reference: Ehm, Cottingham, Kimmel (1995), J. Biological Systems, 3(1), 13-25.

    *   Ehm, Cottingham, Kimmel (1996), "Error detection for genetic data, using likelihood methods", American Journal of Human Genetics, 58(1):225-234

*   GENOGRAM-MAKER

    *   full name:

    *   version:

    *   description: a graphic way of organizing the mass of information gathered during a family assessment and finding patterns in the family system

    *   authors:

    *   web/ftp: http://www.genogram.org/

    *   source code language:

    *   operating systems: MS-Windows, MacOS

    *   executables:

    *   reference: Monica McGoldrick, Randy Gerson, , Sylvia Shellenberger (1999), Genograms: Assessment and Intervention , 2nd edition (W.W.Norton & Company).

    *   price: US$50 (student), US$125 (standard)

*   GENOME

    *   full name:

    *   version:

    *   description: GENOME is a program to simulate sequences drawn from a population under the Wright-Fisher neutral model. The purpose of this program is to simulate sequences on the whole genome scale within practical time.

    *   authors: Liming Liang (email: lianglim@umich.edu), Sebastian Zollner, Goncalo Abecasis (University of Michigan)

    *   web/ftp: http://www.sph.umich.edu/csg/liang/genome/

    *   source code language: C++

    *   operating systems: UNIX(Solaris), Linux, MS-Windows

    *   executables:

    *   reference: Liang, Zollner, Abecasis (2006), "GENOME: a rapid coalescent-based whole genome simulator", ASHG Annual Meeting.

*   GENOMESIMLA

    *   full name:

    *   version: beta

    *   description: genomeSIMLA is a forward-time population simulation method that can simulate realistic patterns of LD in both family-based and case-control datasets.

    *   authors: Marylyn Ritchie (email: genomesimla@chgr.mc.vanderbilt.edu)

    *   web/ftp: http://chgr.mc.vanderbilt.edu/genomeSIMLA/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Dudek, Motsinger, Velez, Williams, Ritchie (2006), "Data simulation software for whole-genome association and other studies in human genetics", Pacific Symposium on Biocomputing, 499-510.

*   GENOMIZER

    *   full name:

    *   version: 1.0

    *   description: Genomizer is a platform independent Java program for the analysis of genome wide association experiments.

    *   authors: Andre Franke (email: a.franke@mucosa.de), Jochen Hampe

    *   web/ftp: http://www.ikmb.uni-kiel.de/genomizer/

    *   source code language: Java

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Franke, Wollstein, Teuber, Wittig, Lu, Hoffmann, Nurnberg, Krawczak, Schreiber, Hampe (2006), "GENOMIZER: an integrated analysis system for genome-wide association data", Human Mutation, 27(6):583-588

*   GENOOM

    *   full name: GENetic Object Oriented Modeling in population genetics

    *   version: 3.66 (November 2003)

    *   description: GENOOM implements a virtual computer world where biological entities are digital objects. In this world, each individual of a population is represented by a set of chromosomes with genetic elements (markers, genes, Quantitative Traits Loci, etc.) and attributes (phenotype, age, fertility, living status, exposure, etc.). It comes with a built-in model allowing basic genetic studies by simulations, performed according to parameter values such a penetrance matrix (phenotype probabilities for each genotype), exposure, between-relative mating probabilities, and a probability of migration in a two-dimensional migration process. The simulated populations could be analysed with different packages for statistical or genetical analysis (e.g., SAS, S-PLUS, Linkage, Mapmaker/sibs, Genehunter). Pedigrees can also be simulated according to a pedigree file at the LINKAGE format.

    *   authors: Hadi Quesneville, Dominique Anxolabehere

    *   web/ftp: http://dynagen.ijm.jussieu.fr/research/topics/modeling-complex-genetic-systems/object-oriented-modeling-of-complex-genetic-systems

    *   ftp://ftp.ijm.jussieu.fr/Usr/quesneville/genoom/

    *   source code language: C++

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference: Quesneville, Anxolabehere (1997), "GENOOM: a simulation package for GENetic Object-Oriented Modeling", Abstract in the European Mathematical Genetics Meeting Proceedings, Annals of Human Genetics, 61:543.

*   GENOPROOF

    *   full name:

    *   version: 1.1 (July 2006)

    *   description: download software package for the analysis of multiplex PCR kits within the scope of paternity testing, kinship cases and population studies. GenoProof® offers: (1) probably the most extensive existing population database for all supported markers of more than 50 ethnic groups, (2) individually configurable quality assurance options, (3) complex concept of user rights in order to guarantee data security, (4) languages German and English

    *   authors: Dr. F. Gotz (Qualitype AG, Dresden, Germany) (email: info@genoproof.de)

    *   web/ftp: http://www.genoproof.com or http://qualitype.de/genoproof/

    *   source code language: Java

    *   operating systems: MS-Windows (95/98/2000/NT/XP)

    *   executables:

    *   reference:

*   GENTOOLS

    *   full name:

    *   version:

    *   description: software for analysis and manipulation of genetic linkage data of genetic linkage data, including conversions of pedigree files between CRI-MAP and LINKAGE format.

    *   authors: Raymond Weaver, Cynthia Helms, Santosh Mishra, Helen Donis-Keller (Washington University School of Medicine, St. Lious)

    *   web/ftp: http://www.genlink.wustl.edu/software/index.html

    *   ftp://mapper.wustl.edu/pub/gentools

    *   source code language: C

    *   operating systems: UNIX

    *   executables: faminfo, genlink, jim, pic, pichet, twotable,unmerge, inherit,linkgen,refbld,unmerge2

    *   reference: Weaver, Helms, Mishra, Donis-Keller (1992), "Software for analysis and manipulation of genetic linkage data", American Journal of Human Genetics, 50:1267-1274

*   GERMLINE

    *   full name:

    *   version: 1.2.1 (September 2008)

    *   description: GERMLINE is a program for discovering long shared segments of Identity by Descent (IBD) between pairs of individuals in a large population. It takes as input genotype or haplotype marker data for individuals (as well as an optional known pedigree) and generates a list of all pairwise segmental sharing.

    *   authors: Sasha Gusev (email: gusev@cs.columbia.edu)

    *   web/ftp: http://www1.cs.columbia.edu/~gusev/germline/

    *   source code language: C++

    *   operating systems: Linux

    *   executables:

    *   reference:

*   GEST

    *   full name: Gene-Environment interaction and STability

    *   version:

    *   description:

    *   authors: Yasuo Ukai (University of Tokyo, Japan)

    *   web/ftp: http://lbm.ab.a.u-tokyo.ac.jp/~ukai/gest98.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Ukai, Nesumi, Yakano (1996), "A package of computer programs for the statistical analysis of genotype x environment interaction and stability GEST" (in Japanese), Breeding Science, 46:73-79.

*   GEVALT

    *   full name: GEnotype Visualization and ALgorithmic Tool

    *   version: 2.0

    *   description: GEVALT is designed to simplify and expedite the process of genotype analysis and disease association tests by providing a common interface to several common tasks relating to such analyses. It is aimed for analysis of unrelated individuals as well as two-generation families.

    *   authors: Ofir Davidovich, Gad Kimmel, Eran Halperin, Oded Apel, Ron Shamir.

    *   web/ftp: http://acgt.cs.tau.ac.il/gevalt/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Davidovich, Kimmel, Shamir (2007), "GEVALT: An integrated software tool for genotype analysis", BMC Bioinformatics, 8:36.

*   GGSD

    *   full name: Generic Genetic Studies Database

    *   version:

    *   description: web-based, relational database driven data management software package for the management of large scale genetic studies.

    *   authors: Aaron Day-Williams (Wellcome Trust Sanger Institute)

    *   web/ftp: http://sourceforge.net/projects/ggsd/

    *   http://ggsd.sourceforge.net/

    *   source code language: Java, PHP, Perl

    *   operating systems: web-based

    *   executables:

    *   reference:

*   GGT (see also, MAPCOMP)

    *   full name: Graphical GenoTyping package

    *   version: December 2006

    *   description: GGT combines genetic map data and locus data to display Graphical Genotypes. Data can be viewed by individual or by linkage group, and sorted on markers. Statistics are calculated and simultaneous filtering/selection on multiple linkage groups is possible. Output of hi-res genotype drawings directly to printer or to emf/jpeg/bitmap image. A module for constructing GGT-datafiles from common mapmaker/joinmap style formatted data is available.

    *   authors: Ralph van Berloo (Wageningen University, The Netherlands, ralph.vanberloo@wur.nl)

    *   web/ftp: http://www.dpw.wau.nl/pv/PUB/ggt/

    *   source code language: Delphi (Pascal)

    *   operating systems: MS-Windows (95/98/2000/XP)

    *   executables: GGT32.EXE

    *   reference: R Van Berloo (1999),"GGT: Software for the display of graphical genotypes", Journal of Heredity, 90:328-329\. [ full text (PDF) : http://www.dpw.wau.nl/pv/papers/reprint/9903.pdf]

*   GHOST

    *   full name:

    *   version:

    *   description: GHOST is a software package for family-based genomewide association (GWA) analysis, with the ability to infer missing genotypes using the Elston-Stewart algorithm. When SNPs from an association panel are less complete (i.e., having more missing genotypes) than markers from a linkage panel, many of the missing genotypes can be determined. GHOST can handle large pedigrees -- when pedigrees are small, Merlin is also recommended for this analysis.

    *   authors:

    *   web/ftp: http://www.sph.umich.edu/csg/chen/ghost/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Burdick, Chen, Abecasis, Cheung (2006), "In silico method for inferring genotypes in pedigrees", Nature Genetics, 38:1002-1004.

    *   Chen, Abecasis (2007), "Family-based association tests for genomewide association scans", American Journal of Human Genetics, 81(5):913-926

*   GIST

    *   full name: Genotype Ibd Sharing Test

    *   version: 0.3

    *   description: testing whether the presence of a marker allale can partially account for the evidence of linkage in the region of the marker

    *   authors: Chun Li (Vanderbilt University). (email: chun.li@vanderbilt.edu)

    *   web/ftp: http://phg.mc.vanderbilt.edu/content/gist

    *   source code language:

    *   operating systems: UNIX, MS-Windows

    *   executables:

    *   reference: Li, Scott, Boehnke (2004), American Journal of Human Genetics, 74(3):418-431

*   GLFSINGLE/GLFTRIO/GLFMULTIPLES

    *   full name:

    *   version:

    *   description: glfTrio is a GLF-based variant caller for next-generation sequencing data. It takes one/three/multiple GLF format genotype likelihood files as input and generates a VCF-format set of variant calls as output.

    *   authors: Abecasis' group (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/GlfSingle

    *   http://genome.sph.umich.edu/wiki/GlfTrio

    *   http://genome.sph.umich.edu/wiki/GlfMultiples

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   GLIDERS

    *   full name: Genome-wide LInkage DisEquilibrium Repository and Search engine

    *   version:

    *   description: HapMap based long-range LD search engine

    *   authors: Robert Lawrence (Wellcome Trust Sanger Institute)

    *   web/ftp: http://mather.well.ox.ac.uk/GLIDERS/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   GLUE

    *   full name: Genetic Linkage User Environment

    *   version: 3.30 (Aug 2002)

    *   description: A web interface to several commonly used statistical genetics programs, including Linkage, Genehunter, Merlin, Unphased, and Transmit. It simplifies their use through graphical selection of program options, automation of multiple analyses, and viewing of graphical output. GLUE is available to HGMP account holders; registration is free to all academic users.

    *   authors: Frank Dudbridge (email: frank.dudbridge@mrc-bsu.cam.ac.uk)

    *   web/ftp: http://portal.litbio.org/Registered/Webapp/glue/

    *   source code language: Perl

    *   operating systems: any web browser

    *   executables:

    *   reference:

*   GMA

    *   full name: Genetic Mixture Analysis

    *   version:

    *   description:

    *   authors: Steven T Kalinowski (email: skalinowski@montana.edu)

    *   web/ftp: http://www.montana.edu/kalinowski/GMA/GMA_Home.htm

    *   source code language:

    *   operating systems: MS-Windows

    *   executables:

    *   reference:

*   GMCHECK

    *   full name:

    *   version:

    *   description: GMCheck finds the posterior probabilities for data errors for genotypes and phenotypes in pedigree data

    *   authors: Alun Thomas (email: alun@genepi.med.utah.edu)

    *   web/ftp: http://episun7.med.utah.edu/~alun/gmcheck/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Thomas (2005), "GMCheck: Bayesian error checking for pedigree genotypes and phenotypes", Bioinformatics, 21(14):3187-3189.

*   G-MENDEL

    *   full name:

    *   version: 3.0 (May 1994)

    *   description: a tool for construction of genetic linkage maps and analyzing Mendelian phenotypes.

    *   authors: J.L Holloway, S.J. Knapp (Department of Crop and Soil Science, Oregon State University)

    *   web/ftp: http://cropandsoil.oregonstate.edu/G-mendel

    *   source code language:

    *   operating systems: UNIX, MS-Windows

    *   executables:

    *   reference:

*   GOLD

    *   full name: Graphical Overview of Linkage Disequilibrium

    *   version: November 2001

    *   description: A software package that provides a graphical summary of linkage disequilibrium in human genetic data. The graphical summary is well suited to the analysis of dense genetic maps, where contingency tables are cumbersome to interpret. An interface to the Simwalk2 application allows for the analysis of family data.

    *   authors: G R Abecasis, William Cookson (Wellcome Trust Centre for Human Genetics, Oxford, UK)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/GOLD/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Abecasis, Cookson (2000), Bioinformatics, 16(2):182-183.

*   GREGOR

    *   full name:

    *   version:

    *   description: the program allows students and scientists to explore potential manifestations of genetic models and stochastic processes. It also provides a means to evaluate the effectiveness of statistical procedures such as linkage analysis or QTL detection.

    *   authors: Nick Tinker ( tinker@agr.gc.ca), Diane Mather (grain crops breeding and genetic research, McGill Univ, mather@agradm.lan.mcgill.ca)

    *   web/ftp: http://gnome.agrenv.mcgill.ca/tinker/gregor.htm

    *   ftp://gnome.agrenv.mcgill.ca/software/GREGOR/

    *   source code language:

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Tinker and Mather, Journal of Heredity, 84, 237 (1993)

*   GRIDQTL

    *   full name:

    *   version: 1.5.1 (August 2010)

    *   description: A publicly available Web-based application that can perform QTL mapping on a variety of population types. GridQTL will extend the functionality of QTLExpress by adding new and advanced approaches for modelling QTL analysis in simple and complex populations. These new methods will be available on a Grid system that will offer flexible workflow management, resource allocation, data persistence, detached execution of simulations and the scalability required for the increase in data volume, data sources and complexity required by the new models.

    *   authors:

    *   web/ftp: http://www.gridqtl.org.uk/

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference:

*   GRONLOD

    *   full name: GRONingen university LOR score calculation

    *   version:

    *   description: Conversion programs from LINKAGE files are available. The program uses peeling and can employ nested conditioning. There is an automatic peeling program that will unravel (multiple) loops. Alleles do not need to be recoded, so real allele sizes can be used. Genotype probabilities for a chosen person can be calculated for purposes of genetic risk calculation. Later versions include one for calculations using linked markers and mutations and mosaicism, made by Martin van der Meulen. A symbolic versions will generate the formula to compute the pedigree likelihood.

    *   authors: Gerard te Meerman (University of Groningen, The Netherlands) (g.j.te.meerman@med.rug.nl)

    *   web/ftp:

    *   source code language: PROLOG

    *   operating systems: MS-Windows (3.1/95/NT)

    *   executables: gronrec,gronreca,gronlod,groninbr, gronchec,gronped,gronpedc, grontree

    *   reference: Meerman, "A Logic Programming Approach to Pedigree Analysis", (Thesis Publishers Amsterdam, 1991). Available free of charge from the author. Contains documented sources and description.

*   GRR

    *   full name: Graphical Representation of Relationships

    *   version: Aug 2003

    *   description: a graphical tool designed for detection of errors in relationship specification in general pedigrees by use of genome scan marker data.

    *   authors: G R Abecasis (email: goncalo@well.ox.ac.uk), S S Cherny (email: cherny@qtl.well.ox.ac.uk), W O C Cookson, and L R Cardon (email: lon@well.ox.ac.uk)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/GRR/

    *   source code language: C++

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Abecasis, Cherny, Cookson, and Cardon (2001), "GRR: Graphical representation of relationship errors", Bioinformatics, 17(8):742-743.

*   GS-EM

    *   full name: GenoSpectrum Expectation-Maximuzation

    *   version:

    *   description:

    *   authors:

    *   web/ftp: http://www.people.fas.harvard.edu/~junliu/genotype/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Kang, Qin, Niu, Liu (2004), "Incorporating genotyping uncertainty in haplotype inference for single-nucleotide polymorphisms", American Journal of Human Genetics, 74(3):495-510

*   GSMA

    *   full name: Genome Search Meta Analysis

    *   version:

    *   description: The GSMA is a rank-based meta-analysis method for analyzing results from genome-wide linkage searches. A software package is now available. The gsma software calculates the summed rank for any number of studies and bins, then obtains p-values for the Summed Rank and the Ordered Rank statistics, by simulation. Weighted and unweighted analyses are performed. A test data set is included.

    *   authors: Cathryn Lewis ( email: cathryn.lewis@kcl.ac.uk), Douglas Levinson, Lesley Wise

    *   web/ftp: http://mmg.umds.ac.uk/GSMA

    *   source code language: C++

    *   operating systems: UNIX(Solaris), MS-DOS

    *   executables:

    *   reference: Wise et al. (1999), Annals of Human Genetics, 63:263-272.

    *   Levinson et al., (2003), American Journal of Human Genetics, 73(1):17-33

*   GTOOL

    *   full name:

    *   version: 0.4.1 (Feb 2008)

    *   description: GTOOL is a program for transforming sets of genotype data for use with the programs SNPTEST and IMPUTE.

    *   authors: Jonathan Marchini

    *   web/ftp: http://www.stats.ox.ac.uk/~marchini/software/gwas/gtool.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   GWAPOWER

    *   full name: Genome-Wide Association POWER

    *   version: 1.1

    *   description: GWApower is a R package for assessing the power of genome-wide association studies using commercially available genotyping chips. The package encapsulates extensive simulation results generated by our program HAPGEN.

    *   authors:

    *   web/ftp: http://www.stats.ox.ac.uk/~marchini/software.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Spencer, Su, Donnelly, Marchini (2009), "Designing genome-wide association studies: sample size, power, imputation, and the choice of genotyping chip", PLoS Genetics, 5(5):e1000477.

*   GWASELECT

    *   full name:

    *   version:

    *   description: GWASelect implements a novel variable selection method for GWAS data and is able to handle more than half million SNPs. Extensive simulation studies and real data analysis show that this method enjoys high power and low false discovery rate compared to existing variable selection methods. The variables selected by GWASelect can be readily placed into a logistic regression model for disease prediction. The current release is designed for binary outcome under the additive mode of inheritance.

    *   authors: Qianchuan He, Dan-Yu Lin (Univ North Carolina)

    *   web/ftp: http://www.bios.unc.edu/~lin/software/GWASelect/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: He, Lin (2010), "A variable selection method for genome-wide association studies", Bioinformatics, 27:1-8.

*   HAP (1)

    *   full name: haplotype resolution using imperfect phylogeny

    *   version: 3.0

    *   description:

    *   authors: Eleazar Eskin ( eeskin@cs.ucsd.edu)

    *   web/ftp: http://research.calit2.net/hap/

    *   http://www.cs.columbia.edu/compbio/hap/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   HAP (2)

    *   full name: HAPlotype analysis of polymorphic markers

    *   version:

    *   description:

    *   authors: (MRC Epidemiology Unit at Strangeways Research Lab, UK) (email: jinghua.zhao@mrc-epid.cam.ac.uk)

    *   web/ftp: http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/

    *   source code language: C

    *   operating systems: MS-Windows, UNIX(Solaris), Linux

    *   executables:

    *   reference: Zhao (2004), "2LD, GENECOUNTING and HAP: Computer programs for linkage disequilibrium analysis", Bioinformatics, 20:1325-1326.

*   HAPAR

    *   full name: HAplotype inference by PARsimony

    *   version:

    *   description: HAPAR is a program to infer haplotype from genotype data. It uses the parsimony principle, i.e. try to find the minimum number of haplotypes that can reconstruct the input genotypes.

    *   authors: Lusheng Wang (City University of Hong Kong), Ying Xu (Stanford Univ),

    *   web/ftp: http://theory.stanford.edu/~xuying/hapar/

    *   http://www.cs.cityu.edu.hk/~lwang/hapar/index.html

    *   source code language:

    *   operating systems: MS-Window, UNIX(Solaris/..)

    *   executables:

    *   reference: Wang, Xu (2003), "Haplotype inference by maximum parsimony", Bioinformatics, 19(14):1773-1780.

*   HAPASSOC (see R/HAPASSOC)

*   HAPBLOCK

    *   full name: a dynamical programming algorithm for haplotype block partitioning

    *   version: 2002

    *   description:

    *   authors: Kui Zhang, Fengzhu Sun (USC)

    *   web/ftp: http://www-hto.usc.edu/msms/HapBlock/

    *   source code language: C

    *   operating systems: MS-Windows, Linux, UNIX(Solaris)

    *   executables:

    *   reference: Kui Zhang, Minghua Deng, Ting Chen, Michael S Waterman, Fengzhu Sun (2002), Proceedings of the National Academy of Sciences , 99:7335-7339.

    *   Zhang, Qin, Chen, Liu, Waterman, Sun (2005), "HapBlock: haplotype block partitioning and tag SNP selection software using a set of dynamic programming algorithms", Bioinformatics, 21(1):131-134.

*   HAPBLOCK (2)

    *   full name:

    *   version: April 2004

    *   description:

    *   authors: Adrian Paul Mander (Boston College) (email: adrian.mander@mrc-hnr.cam.ac.uk)

    *   web/ftp: http://econpapers.repec.org/software/bocbocode/s438902.htm

    *   source code language: Stata

    *   operating systems:

    *   executables:

    *   reference:

*   HAPGEN

    *   full name:

    *   version: 1.3.0 (Jan 2008)

    *   description: HAPGEN is a program thats simulates case control datasets at SNP markers and can output data in the FILE FORMAT used by IMPUTE, SNPTEST and GTOOL. The approach can handle markers in LD and can simulate datasets over large regions such as whole chromosomes. Hapgen simulates haplotypes by conditioning on a set of population haplotypes and an estimate of the fine-scale recombination rate across the region. The disease model is specified through the choice of a single SNP as the disease causing variant together with the relative risks of the genotypes at the disease SNP. The program is designed to work with publically available files that contain the haplotypes estimated as part of the HapMap project and the estimated fine-scale recombination map derived from that data. Hapgen is computationally tractable. On a modern desktop HAPGEN can simulate several thousand case and control data on a whole chromosome at Hapmap Phase 2 marker density within minutes. This program has been used to assess the power of several different commercially available genotyping chips, in the design stage of the 7 genome-wide association studies carried out by the Wellcome Trust Case-Control Consortium (WTCCC) and for evaluating the power of different methods for detecting association in genome-wide studies.

    *   authors: Zhan Su, Jonathan Marchini, Peter Donnelly (Univ of Oxford, UK)

    *   web/ftp: http://www.stats.ox.ac.uk/~marchini/software/gwas/hapgen.html

    *   source code language: C++

    *   operating systems:

    *   executables:

    *   reference:

*   HAPINFERX

    *   full name:

    *   version: (Jul 1999)

    *   description:

    *   authors: Andrew Clark (email: ac347@cornell.edu)

    *   web/ftp: (temporary) source code , input example file , output example file

    *   source code language: FORTRAN

    *   operating systems:

    *   executables:

    *   reference: Clark (1990), "Inference of haplotype from PCR-amplified samples of diploid populations", Molecular Biology and Evolution, 7:111-122.

*   HAPLO (1)

    *   full name: Haplotyping with computation of conditional probabilities

    *   version 1.00 (feb 5, 1996)

    *   description: assigns each person the haplotype with the highest conditional probability using Elston-Stewart algorithm

    *   authors: Daniel E. Weeks (Pittsburgh) and Kenneth Lange (UCLA) (Collaborators: Jeffrey R O'Connell and Eric Sobel)

    *   web/ftp: http://watson.hgen.pitt.edu/register/soft_doc.html,

    *   registration page at: http://watson.hgen.pitt.edu/register

    *   source code language: Fortran 77 (ANSI standard)

    *   operating systems: UNIX (SunOS/Solaris/HP)

    *   executables:

    *   reference: Sobel, Lange, O'Connell and Weeks, in "Genetic Mapping and DNA Sequencing" (IMA Volumes in Mathematics and its Applications) eds. T.P. Speed and M.S. Waterman (Springer-Verlag, 1995).

    *   Weeks, Sobel, O'Connell, Lange (1995), American Journal of Human Genetics, 56(6):1506-1507

*   HAPLO (2)

    *   full name:

    *   version: 2.5 (1998)

    *   description: Estimates frequencies of multi-site haplotypes using the EM algorithm

    *   authors: ME Hawley, KK Kidd

    *   web/ftp: http://krunch.med.yale.edu/haplo/

    *   source code language: FORTRAN

    *   operating systems: DEC VAX

    *   executables:

    *   reference: Hawley, Kidd (1995), "HAPLO: A program using the EM algorithm to estimate the frequencies of multi-site haplotypes", Journal of Heredity, 86:409-411.

*   HAPLOBLOCK

    *   full name:

    *   version: 1.2 (April 28, 2004)

    *   description: HAPLOBLOCK is a software package which provides an integrated approach to haplotype block identification, haplotype resolution and linkage disequilibrium mapping, suitable for high-density phased or unphased SNP data.

    *   authors: Gideon Greenspan (web/ftp: http://www.gidgreen.com/) Dan Geiger (web/ftp: http://www.cs.technion.ac.il/people/dang/)

    *   web/ftp: http://bioinfo.cs.technion.ac.il/haploblock/

    *   source code language: C

    *   operating systems: UNIX(Solaris), Linux, MacOS X

    *   executables: haploblock_b, haploblock_n

    *   reference: Greenspan, Geiger (2004), "Model-based inference of haplotype block variation", Journal of Computational Biology, 11(2-3):493-504.

    *   Greenspan, Geiger (2004), "High density linkage disequilibrium mapping using models of haplotype block variation", Bioinformatics, 20(1):137-144.

    *   Instruction Manual: http://bioinfo.cs.technion.ac.il/haploblock/HaploBlock_Manual.pdf

    *   LD Mapping Supplement http://bioinfo.cs.technion.ac.il/haploblock/HaploBlock_Mapping.pdf

*   HAPLOBLOCKFINDER

    *   full name:

    *   version: 0.7

    *   description: HaploBlockFinder is a package for haplotype block identification, visualization and htSNP selection. It can also compare the haplotype block structure with local LD pattern. The program can be either run as a web service, or standalone executables on local machine.

    *   authors: Kun Zhang (kun.zhang@uth.tmc.edu)

    *   web/ftp: http://cgi.uc.edu/cgi-bin/kzhang/haploBlockFinder.cgi

    *   source code language: C and Perl

    *   operating systems: UNIX, MS-Windows

    *   executables: haploBlockFinder, hbfWrapper.pl, showLDmatrix.pl, drawBlocks.pl

    *   reference:

*   HAPLOBUILD

    *   full name:

    *   version:

    *   description: The analysis of large amounts of SNP data creates difficulties for the analysis of haplotypes and their association to traits of interest. Commonly fairly simple methods, such as two- or three-SNP sliding windows are used to create haplotypes across large regions, but these may be of limited value when adjacent SNPs are in strong LD and provide redundant information. We have created a novel program, "HaploBuild" for constructing and testing haplotypes for SNPs in close physical proximity to one another but which are not necessarily contiguous. Furthermore, the number of SNPs contained in the haplotype is not restricted, thereby permitting the evaluation of complex haplotype structures.

    *   authors: JM Laramie, JB Wilk, AL DeStefano, RH Myers (Boston University)

    *   web/ftp: http://snp.bumc.bu.edu/modules.php?name=HaploBuild

    *   source code language: Perl

    *   operating systems: Linux, MS-Windows, MacOS

    *   executables:

    *   reference: Laramie, Wilk, DeStefano, Myers (2007), "HaploBuild: an algorithm to construct non-contiguous associated haplotypes in family based genetic studies", Bioinformatics, 23(16):2190-2192.

*   HAPLOCLUSTERS

    *   full name:

    *   version:

    *   description: a program designed to detect excess haplotypes sharing in datasets consisting of case and control haplotypes. Excess haplotype sharing can be seen around disease loci in case samples since LD persists longer here than in the controls where LD is persisting only according to the relatedness of the individuals in the population, i.e. the age of the population.

    *   authors: Melanie Bahlo (email: bahlo@wehi.edu.au)

    *   web/ftp: http://bioinf.wehi.edu.au/folders/melanie/haploclusters.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Bahlo,Stankovich,Speed,Rubio,Burfoot,Foote (2006), "Detecting genome wide haplotype sharing using SNP or microsatellite haplotype data", Human Genetics, 119(1-2):38-50.

*   HAPLOPAINTER

    *   full name:

    *   version: 0.027b

    *   description: A pedigree drawing program, suitable in processing haplotype outputs from GENEHUNTER, ALLEGRO, MERLIN, and SIMWALK

    *   authors: Holger Thiele, Peter Nuernberg (Max-Delbruck-Centrum fur Molekulare Medizin, Berlin-Buch).

    *   web/ftp: http://haplopainter.sourceforge.net/html/ManualIndex.htm

    *   source code language: Perl

    *   operating systems:

    *   executables:

    *   reference: Thiele, Nurnberg (2005), "HaploPainter: a tool for drawing pedigrees with complex haplotypes", Bioinformatics, 21(8):1730-1732.

*   HAPLOPOOL

    *   full name:

    *   version:

    *   description: HaploPool is a program for estimating haplotype frequencies either from genotypes of individuals or from genotypes of pooled individuals. The genotypes must be for a block of bi-allelic SNPs (meaning that the SNPs should be in linkage disequilibrium with each other). The program assumes that it is given many genotypes of unrelated diploid individuals in Hardy-Weinberg equilibrium. If the genotypes are from pooled DNA, the program assumes that every pool contains the same number of individuals and the individuals were chosen at random when placed into the pools. For a reasonable running-time, the number of individuals in a pool needs to be between 2 and 4.

    *   authors:

    *   web/ftp: http://haplopool.icsi.berkeley.edu/haplopool/

    *   source code language: gcc/g++, perl, MATLAB

    *   operating systems:

    *   executables:

    *   reference: Kirkpatrick, Santos-Armendariz, Karp, Halperin (2007), "HaploPool: improving haplotype frequency estimation through DNA pools and phylogenetic modeling", Bioinformatics, 23(22):3048-3055.

*   HAPLORE

    *   full name: HAPLOtype REconstruction in pedigrees

    *   version:

    *   description: haplotype reconstruction in general pedigree without recombination

    *   authors: Hongyu Zhao (Yale, email: hongyu.zhao@yale.edu)

    *   web/ftp: http://bioinformatics.med.yale.edu/group/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Zhang, Sun, Zhao (2005), "HAPLORE: a program for haplotype reconstruction in general pedigrees without recombination", Bioinformatics, 21(1):90-103.

*   HAPLOREC

    *   full name:

    *   version: 2.0

    *   description: population-based haplotyping

    *   authors:

    *   web/ftp: http://www.cs.helsinki.fi/group/genetics/haplotyping.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Eronen, Geerts, Toivonen (2004), "A Markov Chain Approach to Reconstruction of Long Haplotypes", Pacific Symposium on Biocomputing, 9:104-115.

*   HAPLO.STAT (previously HAPLO.SCORE)

    *   full name:

    *   version: 1.2.0 (2005)

    *   description: A suite of routines for the analysis of indirectly measured haplotypes.

    *   authors: Dan Schaid, Jason Sinnwell, Charles Rowland, David Tines (Mayo Clinic)

    *   web/ftp: http://mayoresearch.mayo.edu/mayo/research/biostat/schaid.cfm

    *   source code language: S-plus 6.1.2, R 1.7.1

    *   operating systems: UNIX

    *   executables: haplo.em, haplo.glm, haplo.score

    *   reference: Schaid, Rowland, Tines, Jacobson, Poland (2002), American Journal of Human Genetics, 70(2)(2):425-434

*   HAPLOT

    *   full name:

    *   version: 2.5 (March 1998)

    *   description: A simple program for graphical presentation of haplotype block structures, tagSNP selection and SNP variation.

    *   authors:

    *   web/ftp: http://info.med.yale.edu/genetics/kkidd/programs.html

    *   source code language: FORTRAN

    *   operating systems: VMS

    *   executables:

    *   reference: Gu, Pakstis, Kidd (2005), "HAPLOT: a graphical comparison of haplotype blocks, tagSNP sets and SNP variation for multiple populations", Bioinformatics, 21:3938-3939.

*   HAPLOTTER

    *   full name:

    *   version:

    *   description: HAPLOTTER is a web application that has been developed to display the results of a scan for positive selection in the human genome using the HapMap data. It can be used as a resource to examine various population genetic measures in a genomic region. Measures that are currently displayed include iHS (a statistic developed to detect recent positive selection), Fay and Wu's H, Tajima's D and Fst.

    *   authors: S. Kudaravalli, X.Wen, J.K.Pritchard (email: pritch@uchicago.edu) (Univ of Chicago)

    *   web/ftp: http://hg-wen.uchicago.edu/selection/haplotter.htm

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference: Voight, Kudaravalli, Wen, Pritchard (2006), "A map of recent positive selection in the human genome", PLoS Biology, 4(3):e72.

*   HAPLOTYPE ESTIMATION

    *   full name:

    *   version:

    *   description: The haplotype estimation service is available in two variants: one for genetic data from unrelated probands (e.g., case/control studies) and one for core family data

    *   authors: K Rohde, R Furst (MDC Berlin)

    *   web/ftp: http://www.bioinf.mdc-berlin.de/projects/hap/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: K Rohde, R Furst (2001), "Haplotyping and estimation of haplotype frequencies for closely linked biallelic multilocus genetic phenotypes including nuclear family information", Human Mutation, 17(4):289-295.

*   HAPLOTYPER

    *   full name:

    *   version:

    *   description:

    *   authors: T Niu, Jun Liu (Harvard Univ)

    *   web/ftp: http://www.people.fas.harvard.edu/~junliu/Haplo/docMain.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Niu, Qin, Xu, Liu (2002), "Bayesian haplotype inference for multiple linked single-nucleotide polymorphisms", American Journal of Human Genetics, 70(1):157-169

*   HAPLOVIEW

    *   full name:

    *   version: 3.32 (May 2006)

    *   description: Haploview is designed to simplify and expedite the process of haplotype analysis by providing a common interface to several tasks relating to such analyses. Haploview currently allows users to examine block structures, generate haplotypes in these blocks, run association tests, and save the data in a number of formats. All functionalities are highly customizable.

    *   authors: Jeffrey Barrett (Whitehead Institute)

    *   web/ftp: http://www.broad.mit.edu/personal/jcbarret/haploview/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Barrett, Fry, Maller, Daly (2005), "Haploview: analysis and visualization of LD and haplotype maps", Bioinformatics, 21(2):263-265.

*   HAPLOVISUAL

    *   full name:

    *   version:

    *   description:

    *   authors: Esko Ukkonen, Pasi Rastas

    *   web/ftp: http://www.cs.helsinki.fi/u/prastas/haplovisual/

    *   source code language: java

    *   operating systems:

    *   executables:

    *   reference: Ukkonen (2002), "Finding founder sequences from a set of recombinants" Lecture Notes in Computer Science Vol 2452, pp.277-286 (Springer).

*   HAPLOWSER

    *   full name: HAPLotype brOWSER

    *   version:

    *   description:

    *   authors:

    *   web/ftp: http://embio.yonsei.ac.kr/haplowser/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Kim, Kim, Waterman, Park, Li (2009), "HAPLOWSER: a whole-genome haplotype browser for personal genome and metagenome", Bioinformatics, 25(18):2430-2431.

*   HAPMINER

    *   full name:

    *   version: 1.1 (2006)

    *   description: HapMiner is a computer program for association mapping based on directly mining the haplotypes from case-control data via a density-based clustering algorithm. HapMiner can be applied to whole-genome screens, as well as candidate-gene studies in small genomic regions.

    *   authors: Jing Li (email:jingli@eecs.case.edu)

    *   web/ftp: http://vorlon.case.edu/~jxl175/HapMiner.html

    *   source code language:

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Li and Jiang (2005), "Haplotype-based linkage disequilibrium mapping via direct data mining", Bioinformatics, 21(24):4384-4393.

*   HAPMIX

    *   full name:

    *   version: 1.2 (March 2011)

    *   description: HAPMIX is an application for accurately inferring chromosomal segments of distinct continental ancestry in admixed populations, using dense genetic data.

    *   authors: Alkes Price (Harvard Univ)

    *   web/ftp: http://www.hsph.harvard.edu/faculty/alkes-price/software/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Price, Tandon, Patterson, Barnes, Rafaels, Ruczinski, Beaty, Mathias, Reich, Myers (2009), "Sensitive detection of chromosomal segments of distinct ancestry in admixed populations", PLoS Genetics, 5(6):e1000519

*   HAPMIXMAP

    *   full name:

    *   version:

    *   description: HAPMIXMAP is a program for modelling extended haplotypes in genetic association studies, similar to the FASTPHASE program.

    *   authors: Paul McKeigue (email: paul.mckeigue@ucd.ie), David O'Donnell (University College Dublin, Ireland)

    *   web/ftp: http://www.homepages.ed.ac.uk/pmckeigu/hapmixmap/hapmixmap_manual.html

    *   http://sourceforge.net/projects/hapmixmap/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   HAPPY

    *   full name: reconstructing HAPlotYpes

    *   version:

    *   description: a software package for Multipoint QTL Mapping in Genetically Heterogeneous Animals

    *   authors: Richard Mott (Wellcome Trust Centre for Human Genetics, UK. rmott@well.ox.ac.uk)

    *   web/ftp: http://www.well.ox.ac.uk/~rmott/happy.html

    *   source code language: ANSI C (requires NAG C library)

    *   operating systems: UNIX(IRIX,SunOS), Linux

    *   executables:

    *   reference:

*   HAP-SAMPLE

    *   full name:

    *   version:

    *   description: HAP-SAMPLE is a web application for simulating SNP genotypes for case-control and affected-child trio studies by resampling from Phase I/II HapMap SNP data. The user provides a list of SNPs to be "genotyped," along with a disease model file that describes causal SNPs and their effect sizes. The simulation tool is appropriate for candidate regions or whole-genome scans.

    *   authors:

    *   web/ftp: http://www.hapsample.org/

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference: Wright, Huang, Guan, Gamiel, Jeffreis, Barry, Pardo-Manuel, Sullivan, Wilhelmsen, Zou (2007), "Simulating association studies: a data-based resampling method for candidate regions or whole genome scans", preprint.

*   HAPSCOPE

    *   full name:

    *   version:

    *   description: HapScope includes a comprehensive analysis pipeline and a sophisticated visualization tool for analyzing functionally annotated haplotypes.

    *   authors: Jinghui Zhang (National Cancer Institute, NIH)

    *   web/ftp: http://lpg.nci.nih.gov/lpg_small/protocols/HapScope/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Zhang, Rowe, Struewing, Buetow (2002), "HapScope: a software system for automated and visual analysis of functionally annotated haplotypes", Nucleic Acid Research, 30:5213-5221.

*   HAPSIMU

    *   full name:

    *   version:

    *   description: HAPSIMU is based on real haplotype data from the HapMap ENCODE project. This platform can simulate heterogeneous populations with various known and controllable structures under the continuous migration model or the discrete model. Moreover, both qualitative and quantitative traits can be simulated using additive genetic model with various genetic parameters designated by users.

    *   authors: Feng Zhang, Hong-Wen Deng

    *   web/ftp: http://l.web.umkc.edu/liujian/

    *   source code language: C++

    *   operating systems:

    *   executables:

    *   reference: Zhang, Liu, Chen, Deng (2008), "HAPSIMU: a genetic simulation platform for population-based association studies", BMC Bioinformatics, 9:331.

*   HAPSTAT

    *   full name:

    *   version:

    *   description: HAPSTAT is a user-friendly software interface for the statistical analysis of haplotype-disease association. HAPSTAT allows the user to estimate or test haplotype effects and haplotype-environment interactions by maximizing the (observed-data) likelihood that properly accounts for phase uncertainty and study design. The current version considers cross-sectional, case-control and cohort studies.

    *   authors: Danyu Lin, Donglin Zeng (email: linsoft@bios.unc.edu)

    *   web/ftp: http://www.bios.unc.edu/~lin/hapstat/

    *   source code language:

    *   operating systems: MS-Windows(XP/2000/NT/98)

    *   executables:

    *   reference: Lin, Zeng, Millikan (2005), "Maximum likelihood estimation of haplotype effects and haplotype-environment interactions in association studies", Genetic Epidemiology, 29:299-312.

    *   Lin, Zeng (2006), "Likelihood-based inference on haplotype effects in genetic association studies (with discussion)", Journal of American Statistical Association, 101:89-118.

    *   Zeng, Lin, Avery, North, Bray (2006), "Efficient semiparametric estimation of haplotype-disease associations in case-cohort and nested case-control studies", Biostatistics, 7(3):486-502.

*   HARDY (see also PANGAEA)

    *   full name:

    *   version: July 1996

    *   description: Markov chain Monte Carlo program for association in two-dimensional contingency tables, and for testing Hardy-Weinberg equilibrium.

    *   authors: Sun-Wei Guo (U. Minnesota, guo_s@epivax.epi.umn.edu

    *   E.A. Thompson (Univ Washington)

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/Hardy.shtml

    *   source code language: C

    *   operating systems: UNIX(DEC-UNIX/..)

    *   executables:

    *   reference: Guo and Thompson (1992), Biometrics, 48, 361-372.

*   HCLUST

    *   full name:

    *   version:

    *   description: a simple clustering method that can be used to rapidly identify a set of tag SNP's based upon genotype data

    *   authors:

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/hclust/hclust.htm

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference:

*   HEGESMA

    *   full name: HEterogeneity and GEnome Search Meta Analysis

    *   version:

    *   description: HEGESMA is a comprehensive software for performing genome scan meta-analysis, a quantitative method to identify genetic regions (bins) with consistently increased linkage score across multiple genome scans, and for testing the heterogeneity of the results of each bin across scans. The program provides as an output the average of ranks and three heterogeneity statistics, as well as corresponding significance levels.

    *   authors: Elias Zintzaras, (email: zintza@med.uth.gr) John PA Ioannidis

    *   web/ftp: http://biomath.med.uth.gr

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Zintzaras, Ioannidis (2005), "HEGESMA: genome search meta-analysis and heterogeneity testing", Bioinformatics, 21(18):3672-3673.

*   HELIXTREE

    *   full name: HelixTree Genetic Association Software

    *   version: 6

    *   description: Comprehensive toolset for population-based association studies. Perform allele, genotype, haplotype, multi-locus, runs of homozygosity, copy number variation, and regression-based association analysis on case-control, quantitative trait, and categorical type data.

    *   authors: Christophe Lambert, Dmitri Zaykin, Douglas Hawkins, Stanley Young, Alan Menius, Meg Ehm, Mike Mosteller

    *   web/ftp: http://www.goldenhelix.com/SNP_Variation/HelixTree

    *   Download request for free trial available at http://www.goldenhelix.com/SNP_Variation/svstrial.html

    *   source code language: C++

    *   operating systems: MS-Windows, Linux, MacOS X

    *   executables: helixtree.exe

    *   reference: Zaykin, Westfall, Young, Karnoub, Wagner, Ehm (2002), "Testing association of statistically inferred haplotypes with discrete and continuous traits in samples of unrelated individuals", Human Heredity, 53:79-91.

    *   Recursive Partitioning, A Bibliography http://www.goldenhelix.com/download/pdfs/RPbibliography.pdf

    *   price: Academic pricing available. Please contact Golden Helix, Inc. at info@goldenhelix.com

*   HIT

    *   full name: Haplotype Inference Technique

    *   version:

    *   description:

    *   authors: Pasi Rastas

    *   web/ftp: http://www.cs.helsinki.fi/u/prastas/hit/

    *   source code language: java

    *   operating systems:

    *   executables:

    *   reference: Rastas, Koivisto, Mannila, Ukkonen (2005), "A Hidden Markov Technique for Haplotype Reconstruction", 5th Workshop on Algorithms in Bioinformatics - WABI 2005

    *   Rastas, Koivisto, Mannila, Ukkonen (2008), "Phasing genotypes using a hidden Markov model", in eds. I Mandoiu and A Zelikovsky, Bioinformatics Algorithms: Techniques and Applications, pp. 373-391 (Wiley).

*   HOMOG/HOMOGM

    *   full name:

    *   version: HOMOGM: Dec 1998

    *   description:

    *   authors: Jurg Ott (Rockefeller University)

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: Pascal

    *   operating systems:

    *   executables: homog, homog1a, homog1b, homog2, homog3, homog4, homog3r, point4, mtest

    *   reference: Ott (1986), Genetic Epidemiology, Suppl,1,251-257.

    *   online documentation

*   HOMOZYGOSITYMAPPER

    *   full name:

    *   version:

    *   description: A web-based approach of homozygosity mapping that can handle tens of thousands markers. User can upload their own SNP genotype files to our database. Intuitive graphic interface is provided to view the homozygous stretches, with the ability of zooming into single chromosomes or user-defined chromosome regions. The underlying genotypes in all samples are displayed. The software is also integrated with our candidate gene search engine, GeneDistiller, so that users can interactively determine the most promising gene.

    *   authors: Dominik Seelow, Markus Schuelke (NeuroCure Clinical Research, Charite - Universitatsmedizin Berlin)

    *   web/ftp: http://www.homozygositymapper.org/

    *   source code language: Perl

    *   operating systems: web-based

    *   executables:

    *   reference: Seelow, Schuelke, Hildebrandt, Nürnberg (2009), "HomozygosityMapper--an interactive approach to homozygosity mapping", Nucleic Acid Research, 37(web server issue):W593-599.

*   HOTSPOTTER

    *   full name:

    *   version: 1.2.1

    *   description: Hotspotter implements the PAC (Products of Approximate Conditional) model for estimating recombination rate

    *   authors: Na Li

    *   web/ftp: http://www.biostat.umn.edu/~nali/SoftwareListing.html

    *   source code language: C++

    *   operating systems:

    *   executables: rholike, hotspot, fullopt

    *   reference: Li, Stephens (2003), Genetics, 165(4):2213-2233.

*   HPLUS

    *   full name:

    *   version: 2.5

    *   description: HPlus is an analysis tool for performing haplotype estimation on genetic markers such as SNPs and microsatellites. It is able to handle datasets that include case-control status as well as covariates and marker location variables (such as gene name, chromosome location, etc).

    *   authors: Sue Li, RJ Laws, LuePing Zhao (Fred Hutchinson Cancer Research Center)

    *   web/ftp: http://cougar.fhcrc.org/hplus/

    *   source code language:

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Zhao, Li, Khalid (2003), "A method for assessing disease associations with SNP haplotypes and environmental variables in case-control", American Journal of Human Genetics, 72(5):1231-1250

    *   Li, Khalid, Carlson, Zhao (2003), "Estimating haplotype frequencies and standard errors for multiple single nucleotide polymorphisms", Biostatistics, 4:513-522.

*   HS-TDT

    *   full name: Haplotype Sharing Transmission Disequilibrium Test

    *   version:

    *   description: testing association using tightly linked markers in nuclear pedigrees

    *   authors: Shuanglin Zhang (email: shuzhang@mtu.edu)

    *   web/ftp: http://www.math.mtu.edu/~shuzhang/software.html

    *   source code language:

    *   operating systems: UNIX, MS-Windows

    *   executables:

    *   reference: Zhang, Sha, Chen, Dong, Jiang (2003), "Transmission/disequilibrium test based on haplotype sharing for tightly linked markers", American Journal of Human Genetics, 73(3):566-579

    *   Zhang, Sha, Chen, Dong, Jiang (2004), "Impact of genotyping error on type I error rate of the haplotype-sharing transmission /disequilibrium test (HS-TDT): reply to Knapp and Becker" (letter), American Journal of Human Genetics, 74(3):591-593

*   HTR

    *   full name: Haplotype Trend Regression

    *   version: Aug 2002

    *   description: Haplotype association mapping using unrelated individuals; "fixed" and "sliding" window analysis; overall tests and tests for individual haplotype effects

    *   authors: Dmitri Zaykin

    *   web/ftp: http://statgen.ncsu.edu/zaykin/htr.html

    *   ftp://statgen.ncsu.edu/pub/zaykin/htr/

    *   source code language: C++, Bash shell

    *   operating systems: Window, UNIX(Solaris)

    *   executables: emgi.exe (fixed number of markers), bash emgi.sh (sliding window)

    *   reference: Zaykin DV, Westfall PH, Young SS, Karnoub MC, Wagner MJ, Ehm MG (2002), "Testing association of statistically inferred haplotypes with discrete and continuous traits in samples of unrelated individuals", Human Heredity, 53:79-91.

*   HTSNPER

    *   full name: Haplotype Tagging SNPer

    *   version: 1.0 (Dec 2003)

    *   description: htSNPer is a Java program for characterizing the haplotype block structure and selecting haplotype tagging SNP

    *   authors: Keyue Ding, Jing Zhang (email: zhjing98@mails.tsinghua.edu.cn), Kaixin Zhou, Yan Shen, Xuegong Zhang (Tsinghua University, China)

    *   web/ftp: http://www.chgb.org.cn/htSNPer/htSNPer.html

    *   source code language: C++, JVM

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference:

*   HWMET

    *   full name:

    *   version:

    *   description: Bayesian estimation of the population inbreeding coefficient f

    *   authors: KL Ayres (email: k.l.ayres@reading.ac.uk)

    *   web/ftp: http://www.reading.ac.uk/Statistics/genetics/software.html

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference:

*   HWESTRATA

    *   full name:

    *   version:

    *   description: Calculates an exact stratified test for HWE for diallelic markers, such as single nucleotide polymorphisms (SNPs), and an exact test for homogeneity of Hardy Weinberg disequilbrium. In addition, exact tests for HWE are calculated for each stratum.

    *   authors: Daniel Schaid (Mayo Clinic) (email: schaid@mayo.edu)

    *   web/ftp: http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm

    *   source code language: C

    *   operating systems: UNIX(Solaris), MS-Windows(XP)

    *   executables:

    *   reference:

*   IBDREG

    *   full name:

    *   version: 0.1.0 (December 2006)

    *   description: A package in S-PLUS and R to test genetic linkage with covariates by regression methods with response IBD sharing for relative pairs. Account for correlations of IBD statistics and covariates for relative pairs within the same pedigree.

    *   authors: Jason Sinnwell (email: sinnwell@mayo.edu), Daniel Schaid (Mayo Clinic)

    *   web/ftp: http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm

    *   source code language: R/S-PLUS

    *   operating systems:

    *   executables:

    *   reference:

*   IGG

    *   full name: Integrate Genotypes for genome-wide Genetic studies

    *   version: 3.0

    *   description: IGG is an open-source Java package with graphic interface to efficiently and consistently integrate genotypes across high throughput genotyping platforms (e.g., Affymetrix and Illumina), the HapMap genotype repository (http://www.hapmap.org/), and even genotypes from the collaborators' projects. It is equipped with a series of functions to control qualities of genotype integration and to flexibly export genotypes for genetic studies as well.

    *   authors: Miao-Xin Li (email: limx54@163.com)

    *   web/ftp: http://bioinfo.hku.hk:13080/iggweb/

    *   source code language: Java

    *   operating systems: web-based

    *   executables:

    *   reference: Li, Jiang, Ho, Song, Sham (2007), "IGG: A tool to integrate GeneChips for genetic studies", Bioinformatics, 23(22):3105-3107.

    *   Li, Jiang, Kao, Sham, Song (2009), "IGG3: a tool to rapidly integrate large genotype datasets for whole-genome imputation and individual-level meta-analysis", Bioinformatics, 25(11):1449-1450.

*   ILLUMINUS

    *   full name:

    *   version:

    *   description: Illuminus is a fast and accurate algorithm for assigning single nucleotide polymorphism (SNP) genotypes to microarray data from the Illumina BeadArray technology.

    *   authors: Yik Teo et al. (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/illuminus/

    *   source code language: C++

    *   operating systems:

    *   executables:

    *   reference: Teo, Inouye, Small, Gwilliam, Deloukas, Kwiatkowski, Clark (2007), "A genotype calling algorithm for the Illumina BeadArray platform", Bioinformatics, 23(20):2741-2746.

*   INTERSNP

    *   full name: genome-wide association and INTERaction analysis of SNP

    *   version: 1.0.10 (Jan 2012)

    *   description: INTERSNP is a software for genome-wide interaction analysis (GWIA) of case-control SNP data and quantitative traits. SNPs are selected for joint analysis using a priori information. Sources of information to define meaningful strategies can be statistical evidence (single marker association at a moderate level, computed from the own data) and genetic/biologic relevance (genomic location, function class or pathway information). Supports all PLINK file formats and permutation-based testing. Includes pathway association analysis, case-only-analysis, haplotype analysis and more.

    *   authors: Christine Herold, Tim Becker, Dmitriy Drichel, André Lacour, Tatsiana Vaitsiakhovich

    *   web/ftp: http://intersnp.meb.uni-bonn.de/

    *   source code language: C/C++

    *   operating systems: Linux, MS-Windows

    *   executables:

    *   reference: Herold, Steffens, Brockschmidt, Baur, Becker (2009), "INTERSNP: genome-wide interaction analysis guided by priori information", Bioinformatics, 25(24):3275-3281

*   ILR

    *   full name: Imprinting Likelihood Ratio

    *   version: 2.2 (November 2005)

    *   description: calculation of the restricted likelihood-ratio affected sib-pair test for linkage allowing for imprinting

    *   authors: Michael Knapp (Univ Bonn) and Konstantin Strauch (Univ Marburg)

    *   web/ftp: http://www.uni-bonn.de/~umt70e/soft.htm

    *   source code language: SAS macro

    *   operating systems:

    *   executables:

    *   reference: Knapp, Strauch (2004), "Affected-sib-pair test for linkage based on constraints for identical-by-descent distributions corresponding to disease models with imprinting", Genetic Epidemiology, 26:273-285 (Erratum: 28:288).

*   IMPUTE

    *   full name:

    *   version: 2 (2009)

    *   description: IMPUTE is a program for estimating ("imputing") unobserved genotypes in SNP association studies. The program is designed to work seamlessly with the output of the genotype calling program CHIAMO and the population genetic simulator HAPGEN, and it produces output that can be analyzed using the program SNPTEST.

    *   authors: Bryan Howie (email: bhowie@uchicago.edu), Jonathan Marchini (email: marchini@stats.ox.ac.uk)

    *   web/ftp: https://mathgen.stats.ox.ac.uk/impute/impute.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Marchini, Howie, Myers, McVean, Donnelly (2007), "A new multipoint method for genome-wide association studies via imputation of genotypes", Nature Genetics, 39:906-913.

    *   Howie, Donnelly, Marchini (2009), "A flexible and accurate genotype imputation method for the next generation of genome-wide association studies", PLoS Genetics, 5(6):e1000529

*   INSEGT (see also PANGAEA)

    *   full name: INference of SEGregation Types

    *   version: 1.1 (September 1999)

    *   description: the program constructs feasible haplotype configurations and the corresponding segregation types on pedigrees. the haplotype configuration minimizes recombinations on the pedigree.

    *   authors: Jochen Kumm (email: jochen@u.washington.edu)

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml

    *   source code language: C++ (the standard template library, STL, is used)

    *   operating systems:

    *   executables:

    *   reference:

*   INTEGRAYEDMAP

    *   full name:

    *   version:

    *   description: IntegratedMap is a web application and database schema for storing and interactively displaying genetic map data.

    *   authors: Hongyu Yang, Hongyu Wang, Alan Gingle (Center for Applied Genetic Technologies, University of Georgia)

    *   web/ftp: ftp://cggc.agtec.uga.edu/IntegratedMap/

    *   source code language:

    *   operating systems: MS-Windows (NT/2000/XP profession/), require either Oracle RDBMS or MySQL.

    *   executables:

    *   reference: Yang, Wang, Gingle (2005), "IntegratedMap: a web interface for integrating genetic map data", Bioinformatics, 21(9):2126-2127.

*   INTERSNP

    *   full name:

    *   version: 1.0.4 (March 2010)

    *   description: INTERSNP is a software for genome-wide interaction analysis (GWIA) of case-control SNP data. SNPs are selected for joint analysis using a priori information. Sources of information to define meaningful strategies can be statistical evidence (single marker association at a moderate level, computed from the own data) and genetic/biologic relevance (genomic location, function class or pathway information).

    *   authors: Christine Herold, Tim Becker

    *   web/ftp: http://intersnp.meb.uni-bonn.de/

    *   source code language: C/C++

    *   operating systems: UNIX, MS-Windows, MacOS

    *   executables:

    *   reference: Herold, Steffens, Brockschmidt, Baur, Becker (2009), "INTERSNP: genome-wide interaction analysis guided by a priori information", Bioinformatics, 25(24):3275-3281

*   JENTI

    *   full name:

    *   version:

    *   description: An efficient tool for mining complex inbred genealogies that identify clusters of individuals sharing the same expected amount of relatedness is described. Additionally it allows for the reconstruction of sub-pedigrees suitable for genetic mapping in a systematic way.

    *   authors:

    *   web/ftp: http://www.jenti.org

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Falchi, Fuchsberger (2008), "Jenti: an efficient tool for mining complex inbred genealogies", Bioinformatics, 24(5):724-726.

*   JLIN

    *   full name: a Java based LINkage disequilibrium plotter

    *   version: 1.5.2 (March 2006)

    *   description: JLIN (Java LINkage disequilibrium plotter) is designed for customisable, intuitive visualisation of LD analysis across all common computing platforms. Customisation allows the researcher to choose particular visualisation, statistical measures and measurement ranges. JLIN also allows the researcher to export images of the LD visualisation in several common document formats. As there appears to be no single best measure of LD under all possible circumstances, JLIN allows the researcher to visually compare and contrast the results of a range of statistical measures on the input data set(s). These measures include the commonly used D' and R2 statistics and empirical p-values. New additions include calculation of HWE, a completely revamped interface, and a numer of minor bug fixes. We have added a display measure to show marker distances visually, embedded fonts to improve image clarity and additional LD measures including d,OR,Pexcess and Q.

    *   authors: Kim W Carter (Univ of Western Australian), Pamela A. McCaskie, Lyle J Palmer

    *   web/ftp: http://www.genepi.org.au/jlin

    *   source code language: Java

    *   operating systems: Any with Java 1.5 support

    *   executables: Java-based installer available from website

    *   reference: Carter, McCaskie, Palmer (2006), "JLIN: A java based linkage disequilibrium plotter", BMC Bioinformatics, 7:60.

*   JOINMAP

    *   full name: Software for the calculation of genetic linkage maps

    *   version: 4 (October 2006)

    *   description: Construction of genetic linkage maps for several types of mapping populations: BC1, F2, RILs, (doubled) haploids, outbreeders full-sib family. Can combine ('join') data derived from several sources into an integrated map. Further: linkage group determination, automatic phase determination for outbreeders full-sib family, several diagnostics, and map charts. Everything available in an intuitive MS-Windows user interface.

    *   authors: Johan W Van Ooijen, Roeland E Voorrips (Plant Research International, Biometris, Wageningen, Netherlands) (email: mapping@biometris.nl)

    *   web/ftp: http://www.joinmap.nl/

    *   source code language: ANSI C, Delphi

    *   operating systems: MS-Windows (95/98/ME/NT4.0/2000)

    *   executables:

    *   reference: JW Van Ooijen, RE Voorrips (2001), "JoinMap (tm) 3.0: Software for the calculation of genetic linkage maps", Plant Research International, Wageningen, Netherlands; Stam (1993), The Plant Journal, 3, 739-744.

*   JPSGCS (see also GENEPI.JAR)

    *   full name: Java Programs for Statistical Genetics and Computational Statistics

    *   version:

    *   description: These programs have been developed since 2002 to address problems in statistical genetics. However, they include several programs and packages that may be more generally useful, for instance, programs to draw and manipulate graphs, simulation programs, and programs to estimate graphical models.

    *   authors: Alun Thomas

    *   web/ftp: http://balance.med.utah.edu/wiki/index.php/JPSGCS

    *   source code language: Java

    *   operating systems: Linux, MacOS, MS-Windows,

    *   executables: ApproxGCHap, CheckErrors, CheckFormat, CheckParameters, CheckPedigree, CheckTriplets, Complete, DownCodeAlleles, FitGM, FitGMLD, FitIntervalLD, GCHap, GeneCountAlleles, GeneDrop, GeneDrops, GetPolymorphisms, HGS, HapMapToLinkage, HetCutOff, HetLods, Heterozygosities, HomozygousSGS, LinkageToFastPhase, LinkageToPhase, LocusLogLikelihoods, MakePed, MakeProbands, MaxTwoPointLods, McCheck, McKinship, McLinkage, McMultiPoints, McTLods, ObligateErrors, SGS, SelectKindreds, SelectLoci, SelectPedigrees, SimHGS, SimHomozygousSGS, SimSGS, Triangulate, TrimPed, TwoPointLods, ViewDAG, ViewGraph, ViewLinkPed, ViewPed, ZeroLoopPed, cMorgansToTheta

    *   reference:

*   J/QTL

    *   full name:

    *   version:

    *   description: J/QTL is a Java GUI for the popular QTL data analysis software R/QTL. It provides a flexible and powerful working environment for users to perform a variety of tasks.

    *   authors: Hao Wu and Lei Wu (Jackson Lab) (email: jqtl@jax.com)

    *   web/ftp: http://www.jax.org/staff/churchill/labsite/software/Jqtl/index.html

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference:

*   KIN

    *   full name:

    *   version: 1.05 (Dec 1995)

    *   description: calculate kinship coefficient or coefficient of coancestry (the probability that alleles at a given locus are identical by descent)

    *   authors: Nick A. Tinker ( tinker@em.agr.ca), Diane Mather (grain crops breeding and genetic research, McGill Univ, mather@agradm.lan.mcgill.ca)

    *   web/ftp: http://gnome.agrenv.mcgill.ca/tinker/kin.htm

    *   ftp://gnome.agrenv.mcgill.ca/software/KIN/

    *   source code language:

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Tinker and Mather, Journal of Heredity, 84, 238 (1993)

*   KING

    *   full name: Kinship-based INference for Gwas

    *   version: 1.0.0 (2010)

    *   description: KING is a toolset that makes use of high-throughput SNP data typically seen in a genome-wide association study (GWAS) for applications such as family relationship inference and population structure identification

    *   authors: Wei-Min Chen (University of Virginia)

    *   web: http://people.virginia.edu/~wc9c/KING/

    *   source code language: C++

    *   operating systems: Linux (64bit), MS-Windows, MS-DOS, MacOS, Ubuntu (32bit)

    *   executables: king

    *   reference: Manichaikul, Mychaleckyj, Rich, Daly, Sale, Chen (2010), "Robust relationship inference in genome wide association studies", Bioinformatics. 2010 Nov 15;26(22):2867-73

*   LAMARC

    *   full name: Likelihood Analysis with Metropolis Algorithm using Random Coalescence

    *   version: 2.0.2 (August 2005)

    *   description: The LAMARC program estimates effective population sizes, exponential population growth rates, and past migration rates between two or n populations, and simultaneously estimates the per-nucleotide recombination rate. Currently Lamarc can use DNA or RNA sequence data, SNP data, and microsatellite data.

    *   authors: Mary Kuhner, Jon Yamato, Peter Beerli (email: lamarc@gs.washington.edu)

    *   web/ftp: http://evolution.genetics.washington.edu/lamarc/lamarc_prog.html

    *   source code language: C++

    *   operating systems: Linux, MS-Windows, MacOS

    *   executables: lamarc, migrate, coalesce, fluctuate, recombine

    *   reference:

*   LAMBDAA

    *   full name:

    *   version:

    *   description:

    *   authors: AP Morris (email: amorris@well.ox.ac.uk)

    *   web/ftp: http://www.reading.ac.uk/Statistics/genetics/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Morris, Whittaker, Curnow (1997), "A likelihood ratio test for detecting patterns of disease-marker association", Annals of Human Genetics, 61:335-350.

*   LAMP

    *   full name: Linkage and Association Modeling in Pedigrees

    *   version: 0.0.9

    *   description: LAMP carries out parametric association analysis in samples of unrelated individuals, trios or large families. In addition to a test-statistic, a genetic model is estimated for each SNP. The approach is based on retrospective likelihood and can be used to analyze selected samples.

    *   authors: Goncalo Abecasis (email: goncalo@umich.edu) and Mingyao Li

    *   web: http://www.sph.umich.edu/csg/abecasis/LAMP/download/

    *   source code language: C++

    *   operating systems: UNIX, LINUX, MS-Windows, MacOS

    *   executables: lamp

    *   reference: Li, Boehnke, Abecasis (2006), "Efficient study designs for test of genetic association using sibship data and unrelated cases and controls", American Journal of Human Genetics, 78(5):778-792

    *   Li, Boehnke, Abecasis (2005), "Joint Modeling of Linkage and Association: Identifying SNPs Responsible for a Linkage Signal", American Journal of Human Genetics, 76(6):934-949

*   LAPSTRUCT

    *   full name: LAPlacian eigenfunctions learn population STRUCTure

    *   version:

    *   description: LAPSTRUCT is a free program to describe population structure using biomarker data ( typically SNPs, CNVs etc.) available in a population sample. The main features different from PCA are: (1)geometrically motivated and graphic model based; (2)robustness of outliers.

    *   authors: Jun Zhang, P Niyogi, MS McPeek

    *   web/ftp: http://galton.uchicago.edu/~junzhang/LAPSTRUCT.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Zhang, Niyogi, McPeek (2009), "Laplacian eigenfunctions learn population structure", PLoS ONE, 4(12):e7928.

*   LCP

    *   full name: Linkage Control Program

    *   alias: LINKAGE

    *   version: This is a 16 bit program and will not run on 64 bit PCs, see [http://www.jurgott.org/linkage/LinkagePC.html](http://www.jurgott.org/linkage/LinkagePC.html)

    *   description: part of the LINKAGE auxiliary programs

    *   authors: Peter Cartwright (Univ Utah, now Cimarron Software [pc@cimsoft.com](mailto:pc@cimsoft.com))

    *   web: http://lab.rockefeller.edu/ott/programs or https://www.dropbox.com/sh/ulpuondvofkrvx1/AAAkIHrAqyQpHJlZaqtmTRb4a/Linkage%20Analysis (please contact Alberto Goldoni, email: alberto.goldoni1975@gmail.com if you have any questions on this version)

    *   source code language: C and Pascal

    *   operating systems: UNIX, VMS, MS-DOS, OS2

    *   executables:

    *   reference:

*   LDA

    *   full name: Linkage Disequilibrium Analyzer

    *   version:

    *   description: LDA is a Java program for analyzing the pairwise linkage disequilibrium.

    *   authors: Keyue Ding (email: keyued@chgb.org.cn), Kaixin Zhou, Fuchu He, Yan Shen

    *   web/ftp: http://www.chgb.org.cn/lda/lda.htm

    *   source code language: Java

    *   operating systems: MS-Windows, Linux, MacOS

    *   executables:

    *   reference: Ding, Zhou, He, Shen (2003), "LDA - A java-based linkage disequilibrium analyzer", Bioinformatics, 19:2147-2148.

*   LDB/LDB+

    *   full name: Location DataBase

    *   version:

    *   description: integrate genetic linkage map and physical map

    *   authors: A Collins ( arc@southampton.ac.uk), NE Morton, S. Lawrence, D. Shields, Jonathan Teague, Paul Linehan, Pietro Lio (University of Southampton, UK)

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ldb;

    *   http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ldb+

    *   source code language: FORTRAN

    *   operating systems: UNIX(SunOS/..)

    *   executables:

    *   reference: Morton (1991), Annals of Human Genetics, 55:235-241.

    *   Morton, Collins, Lawrence, Shield (1992), Annals of Human Genetics, 56:223-232.

    *   online manual and user guide

*   LDGROUP

    *   full name:

    *   version: Nov 2007

    *   description: inkage disequilibrium grouping of single nucleotide polymorphisms (SNPs) reflecting haplotype phylogeny for efficient selection of tag SNPs.

    *   authors: Fumihiko Takeuchi (email: fumihiko@takeuchi.name)

    *   web/ftp: http://www.fumihiko.takeuchi.name/publications.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Takeuchi, Yanai, Morii, Ishinaga, Taniguchi-Yanai, Nagano, Kato (2005), "Linkage disequilibrium grouping of single nucleotide polymorphisms (SNPs) reflecting haplotype phylogeny for efficient selection of tag SNPs", Genetics, 170:291-304.

*   LDHAT

    *   full name:

    *   version: 2.0

    *   description: a package for the analysis of recombination rates from population genetic data

    *   authors: Gil McVean (email: mcvean@stats.ox.ac.uk)

    *   web/ftp: http://www.stats.ox.ac.uk/~mcvean/LDhat/

    *   source code language: C

    *   operating systems: DOS

    *   executables:

    *   reference: McVean, Myers, Hunt, DeLoukas, Bentley, Donnelly (2004), "The fine-scale structure of recombination rate variation in the human genome", Science, 304:581-584.

*   LDHEATMAP (see R/LDHEATMAP)

*   LDMAP

    *   full name:

    *   version: 1.0 (November 2004)

    *   description: LDMAP is a program for constructing linkage disequilibrium (LD) maps.

    *   authors: Andy Collins (email: a.r.collins@soton.ac.uk)

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/LDMAP

    *   source code language: C

    *   operating systems: UNIX(Solaris)

    *   executables:

    *   reference: online manual: http://cedar.genetics.soton.ac.uk/public_html/helpld.html

*   LDMET

    *   full name:

    *   version:

    *   description:

    *   authors: KL Ayres (email: k.l.ayres@reading.ac.uk)

    *   web/ftp: http://www.reading.ac.uk/Statistics/genetics/software.html

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Ayres, Balding (2001), "Measuring gametic disequilibrium from multi-locus data", Genetics, 157:413-423.

*   LDSELECT

    *   full name:

    *   version: 1.0

    *   description: The program analyzes patterns of linkage disequilibrium (LD) between polymorphic sites in a locus, and bins the SNPs on the basis of a threshold level of LD as measured by r2.

    *   authors: Deborah A. Nickerson, Mark Rieder, Chris Carlson, Qian Yi (Univ of Washington)

    *   web/ftp: http://droog.gs.washington.edu/ldSelect.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Carlson, Eberle, Rieder, Yi, Kruglyak, Nickerson (2004), "Selecting a maximally informative set of single-nucleotide polymorphisms for association analysis using linkage disequilibrium", American Journal of Human Genetics, 74(1):106-120

*   LDSUPPORT

    *   full name:

    *   version:

    *   description:

    *   authors: Yutaka Kitamura (Institute of Rheumatology, Tokyo Women;s Medical University, email: kitamura@ior.twmu.ac.jp)

    *   web/ftp:

    *   source code language: C

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference: Kitamura, Moriguchi, Kaneko, Morisaki, Morisaki, Toyama, Kamatani (2002), "Determination of probability distribution of diplotype configuration (diplotype distribution) for each subject from genotypic data using the EM algorithm", Annals of Human Genetics, 66:183-193.

*   LEA

    *   full name: Likelihood-based Estimation of Admixture

    *   version:

    *   description:

    *   authors: Olivier Langella, Lounes Chikhi, Mark Beaumont

    *   web/ftp: http://www.cnrs-gif.fr/pge/bioinfo/lea/index.php?lang=en

    *   source code language: C++

    *   operating systems: MS-Windows

    *   executables: lea.exe

    *   reference: Langella, Chikhi, Beaumont (2001), "LEA (Likelihood-based estimation of admixture): a program to simultaneously estimate admixture and the time since admixture", Molecular Ecology Notes, 1(4):357-358.

*   LINKAGE - general pedigrees (see also FASTLINK)

    *   full name:

    *   version: 5.1, 5.2, 6.0 (testing)

    *   description:

    *   authors: 5.1, 5.2: Mark Lathrop, J. Lalouel, C. Julier, Jurg Ott;

    *   6.0: Alan Young, Daniel Weeks, Mark Lathrop

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: 5.1, 5.2: Pascal ; 6.0: C

    *   operating systems: MS-DOS, OS2, UNIX, VMS

    *   executables: 5.1, 5.2: unknown, mlink, ilink, lodscore, linkmap; 6.0: linkage

    *   reference: Lathrop, Lalouel, Julier, Ott (1984), Proceedings of the National Academy of Sciences , 81:3443-3446.

    *   Lathrop, Lalouel (1984), "Easy calculations of lod scores and genetic risks on small computers", American Journal of Human Genetics, 36(2):460-465

    *   Lathrop, Lalouel, White (1986), Genetic Epidemiology, 3:39-52.

    *   Am Soc Hum Genet annual meeting 1995 (Young, Weeks, Lathrop, Am J Hum Genet suppl, 57(4), A206 (1995)).

    *   online user's guide (may 93) .

    *   see also: the mystery of (the) unknown.

    *   see also: pedigree traversal

*   LINKAGE - CEPH (three-generation) pedigrees (see also FASTLINK)

    *   full name:

    *   version: 5.1, 5.2

    *   description:

    *   authors: Mark Lathrop, J. Lalouel, C. Julier, Jurg Ott

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: Pascal

    *   operating systems: MS-DOS, OS2, UNIX,VMS

    *   executables: cfactor,cilink,clodscore,cmap

    *   reference: Lathrop,Lalouel,Julier, Ott (1984), Proceedings of the National Academy of Sciences , 81:3443-3446.

    *   online user's guide (may 93)

*   LINKAGE-IMPRINT

    *   full name: LINKAGE allowing for IMPRINITING

    *   version: January 2005

    *   description: We have developed a parametric model-based approach to analyzing pedigree data for genomic imprinting. We have modified widely used LINKAGE program to incorporate imprinting. In addition, the LINKAGE-IMPRINT program allows for the use of sex-specific recombination in the analysis, which is of particular importance in a genome-wide analysis for imprinted genes.

    *   authors: Sanjay Shete (email: sshete@mdanderson.org), Xiaojun Zhou (email: xiaojunzhou@mdanderson.org) (Univ of Texas MD Anderson Cancer Cancer)

    *   web/ftp: https://epi.mdanderson.org/~xzhou/Software/Linkage_imprinting/

    *   source code language: C, Pascal

    *   operating systems: UNIX (Solaris)

    *   executables: mlinki, linkmapi

    *   reference: Shete and Zhou (2005), "Parametric approach to genomic imprinting analysis with applications to Angelman's Syndrome", Human Heredity, 59:26-33.

*   LINKBASE

    *   full name:

    *   version: beta 0.95

    *   description: an easy and practical database-program made for researchers who want to connect the genotype data produced by automatic sequencers [ ABI Prism 377 (Perkin Elmer) and ALF (Pharmacia) ] to linkage and sib-pair programs.

    *   authors: Aki Suomalainen (Dept of Human Molecular Genetics, Natl Pub Health Institute, Finland, aki.suomalainen@ktl.fi)

    *   web/ftp: http://www.ktl.fi/molbio/software/linkbase/index.html

    *   source code language: Delphi 1.0

    *   operating systems: MS-Windows (3.1/95/NT)

    *   executables:

    *   reference:

*   LIPED

    *   full name: Likelihoods in Pedigrees

    *   version:

    *   description:

    *   authors: Jurg Ott (Columbia Univ)

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: FORTRAN

    *   operating systems: Windows

    *   executables:

    *   reference: Ott (1974), "Estimation of the recombination fraction in human pedigrees: efficient computation of the likelihood for human linkage studies.", American Journal of Human Genetics, 26(5):588-597

    *   online documentation

*   LNKTOCRI

    *   full name:

    *   version:

    *   description: Conversion of LINKAGE format data files to CRI-MAP format

    *   authors:Jillian Blaschak and Tara Cox Matise (email: matise@biology.rutgers.edu)

    *   web/ftp: http://compgen.rutgers.edu/multimap/

    *   http://compgen.rutgers.edu/multimap/src/other/lnktocri.p

    *   source code language: Pascal

    *   operating systems: UNIX, MS-DOS

    *   executables:

    *   reference:

*   LOCUSMAP

    *   full name:

    *   version: 1.1

    *   description: A computer package designed for rapid linkage analysis and map construction of loci with a variety of inheritance modes.

    *   authors: John Garbe (University of Minnesota, email: jgarbe@umn.edu), Yang Da (University of Minnesota, email: yda@umn.edu)

    *   web/ftp: http://animalgene.umn.edu/locusmap/index.html

    *   source code language: Fortran 90/95

    *   operating systems: MS-Windows,Linux

    *   executables: locusmap

    *   reference: J Garbe, Y Da (2003), Locusmap user manual Version 1.0\. Department of Animal Science, University of Minnesota.

*   LOCUSZOOM

    *   full name:

    *   version:

    *   description: LocusZoom is designed to facilitate viewing of local association results together with useful information about a locus, such as the location and orientation of the genes it includes, linkage disequilibrium coefficients and local estimates of recombination rates. It was developed by popular demand, as a result of many questions we have had about "How did you make the figures in your talk?" or "How did you make the figures for your GWAS paper?"

    *   authors: Abecasis's group (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/LocusZoom

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   LOGINSERM_ESTIHAPLO

    *   full name:

    *   version:

    *   description:

    *   authors: Pierre-Antoine Gourraud (Unite INSERM, 558-Faculte de Medecine email: gourraud@cict.fr)

    *   web/ftp:

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Gourraud, Genin, Cambon-Thomsen (2004), "Handling missing values in population data: consequences for maximum likelihood estimation of haplotype frequencies", European Journal of Human Genetics, 12:805-812.

*   LOH-LINKAGE

    *   full name:

    *   version:

    *   description: a program using loss of heterozygosity data to enhance the power to detect linkage in cancer families.

    *   authors: Klaus Rohde (Max Delbruck Center for Molecular Medicine, Germany) (rohde@mdc-berlin.de), Mauro F. Santibanez Koref, M Dawn Teare

    *   web/ftp: http://www.bioinf.mdc-berlin.de/~rohde/

    *   source code language: C

    *   operating systems: UNIX

    *   executables:

    *   reference: Rohde et al, (1995), Human Heredity, 45:337-345.

    *   Teare et al, J Med Genet 1994; 31:448-452.

*   LOKI (see also PANGAEA)

    *   full name:

    *   version: 2.4.5 (March 2003)

    *   description: Program for analyses a quantitative trait observed on large pedigrees using Markov chain Monte Carlo multipoint linkage and segregation analysis. The trait may be determined by multiple loci.

    *   authors: Simon C. Heath (Centre National de Genotypage) (email: heath@cng.fr)

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/Loki.shtml

    *   http://loki.homeunix.net

    *   source code language: C

    *   operating systems: UNIX(Solaris 2.5, OSF1 3.2, and others), Linux (2.0.30)

    *   executables:

    *   reference: Heath (1997), "Markov chain segregation and linkage analysis for oligogenic models", American Journal of Human Genetics, 6(3):748-760

*   LOT

    *   full name: Linkage analysis of Ordinal Traits

    *   version:

    *   description:

    *   authors: Heping Zhang (email: heping@peace.med.yale.edu)

    *   web/ftp: http://peace.med.yale.edu/pub/LOT/

    *   source code language: C++

    *   operating systems: MS-Windows, Linux, UNIX(Solaris)

    *   executables:

    *   reference: Feng, Leckman, Zhang (2004), "Linkage analysis of ordinal traits for pedigree data", Proceedings of the National Academy of Sciences , 101:16739-16744.

*   L-POP

    *   full name:

    *   version: Jan 2001

    *   description: L-POP detects population stratification in samples of unrelated individuals for whom a number of unlinked genotypes have been measured. | Using a latent class analysis model, population strata are defined such that unlinked markers are in Hardy-Weinberg equilibrium and linkage equilibrium within classes. | L-POP calculates the posterior probabilities for each individual belonging to each of K classes, which can be used to correct for spurious effects of population stratification in tests of association.

    *   authors: Shaun Purcell, Pak Sham

    *   web/ftp: http://statgen.iop.kcl.ac.uk/lpop/

    *   source code language:

    *   operating systems: MS-DOS

    *   executables: lpop

    *   reference:

*   LRP

    *   full name: Linkage Report Program

    *   alias: LINKAGE/LRP

    *   version:

    *   description: part of the LINKAGE auxiliary program

    *   authors: Peter Cartwright (Univ Utah, now Cimarron Software [pc@cimsoft.com](mailto:pc@cimsoft.com))

    *   web: http://lab.rockefeller.edu/ott/programs or https://www.dropbox.com/sh/ulpuondvofkrvx1/AAAkIHrAqyQpHJlZaqtmTRb4a/Linkage%20Analysis (please contact Alberto Goldoni, email: alberto.goldoni1975@gmail.com if you have any questions on this version)

    *   source code language: C and Pascal

    *   operating systems: UNIX, VMS, MS-DOS, OS2

    *   executables:

    *   reference:

*   LRTAE

    *   full name: Likelihood Ratio Test Allowing for Errors

    *   version: 1.3.1 (June 2005)

    *   description: The purpose of this program is to compute a likelihood ratio test statistic that increases power to detect genetic association in the presence of phenotype, genotype, and/or haplotype misclassification errors. In addition, the program produces asymptotically unbiased estimates of frequency parameters.

    *   authors: Chad Haynes, Derek Gordon (email: Gordon@dls.rutgers.edu)

    *   web/ftp: ftp://linkage.rockefeller.edu/software/lrtae/

    *   source code language: C

    *   operating systems: MS-Windows, UNIX(Solaris), Linux

    *   executables:

    *   reference: Gordon, Yang, Haynes, Finch, Mendell, Brown, Haroutunian (2004), "Increasing power for tests of genetic association in the presence of phenotype and/or genotype error by use of double-sampling", Stat Appl Genet and Mol Biol 3:Article 26.

    *   online manual page

*   LSP

    *   full name: Linkage Setup Program

    *   alias: LINKAGE/LSP

    *   version: This is a 16 bit program and will not run on 64 bit PCs, see [http://www.jurgott.org/linkage/LinkagePC.html](http://www.jurgott.org/linkage/LinkagePC.html)

    *   description: part of the LINKAGE auxiliary programs

    *   authors: Peter Cartwright (Univ Utah, now Cimarron Software [pc@cimsoft.com](mailto:pc@cimsoft.com))

    *   web: http://lab.rockefeller.edu/ott/programs or https://www.dropbox.com/sh/ulpuondvofkrvx1/AAAkIHrAqyQpHJlZaqtmTRb4a/Linkage%20Analysis (please contact Alberto Goldoni, email: alberto.goldoni1975@gmail.com if you have any questions on this version)

    *   source code language: C and Pascal

    *   operating systems: UNIX, VMS, MS-DOS, OS2

    *   executables:

    *   reference:

*   LTSOFT

    *   full name:

    *   version: 1.0 beta

    *   description: LTSOFT is a software suite designed to more powerfully leverage clinical-covariates such as age, bmi, smoking status, and gender when conducting case-control association studies. Including these covariates in standard regression models is not only suboptimal, but can in many instances reduce power. LTSOFT employs a liability threshold model approach that takes advantage of known epidemiological results to better model the covariates' relationship to the phenotype of interest

    *   authors: Alkes Price (Harvard Univ)

    *   web/ftp: http://www.hsph.harvard.edu/faculty/alkes-price/software/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   MACH (see also MINIMAC)

    *   full name: MArkov Chain Haplotyper

    *   version: 1.0

    *   description: MACH is a Markov Chain based haplotyper. It can be resolve long haplotypes or infer missing genotypes in samples of unrelated individuals.

    *   authors: Yun Li (email: ylwtx@umich.edu), Goncalo Abecasis (email: goncalo@umich.edu)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/MACH/index.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   MADELINE

    *   full name:

    *   version: 0.935 (July 2004)

    *   description: Madeline is a program designed for preparing, visualizing, and exploring human pedigree data used in genetic linkage studies. In addition to converting pedigree and marker data into various formats required by linkage analysis software, Madeline also provides useful functionality for querying pedigree data sets and drawing pedigrees.

    *   Authors: Edward Trager (email: ehtrager@umich.edu)

    *   web/ftp: http://eyegene.ophthy.med.umich.edu/

    *   source code language: C

    *   operating systems: UNIX (Solaris/FreeBSD/OpenBSD/MacOS), MS-Windows, Cygwin, Linux,

    *   executables: madeline

    *   reference: http://eyegene.ophthy.med.umich.edu/madeline-0.935/

*   MADMAPPER

    *   full name:

    *   version: September 2005

    *   description: suite of Python scripts for quality control of genetic markers, group analysis and inference of linear order of markers on linkage groups. MadMapper_RECBIT analyses raw marker scores for recombinant inbred lines. MadMapper_RECBIT generates pairwise distance scores for all markers, clusters based on pairwise distances, identifies genetic bins, assigns new markers to known linkage groups, validates allele calls, and assigns quality classes to each marker based on several criteria and cutoff values. MadMapper_XDELTA utilizes new algorithm, Minimum Entropy Approach and Best-Fit Extension, to infer linear order of markers. MadMapper_XDELTA analyzes two-dimensional matrices of all pairwise scores and finds best map that has minimal total sum of differences between adjacent cells (map with lowest entropy). MadMapper is freely available at http://www.atgc.org/XLinkage/MadMapper/

    *   authors: Alexander Kozik (UC Davis Genome Center)

    *   web/ftp: http://www.atgc.org/XLinkage/MadMapper/

    *   source code language: Python

    *   operating systems:

    *   executables: MadMapper_RECBIT, MadMapper_XDELTA

    *   reference:

*   MAGIC

    *   full name: Marker And Gene Interpolation and Correlation

    *   version: 1.0

    *   description:

    *   authors: Claire Simpson (c.simpson@iop.kcl.ac.uk), Ammar Al-Chalabi (ammar@iop.kcl.ac.uk), John Powell (john.powell@iop.kcl.ac.uk)

    *   web/ftp: http://cogent.iop.kcl.ac.uk/MaGIC.cogx

    *   source code language: Microsoft Visual Foxpro

    *   operating systems: MS-Windows (95/98/NT/2000/XP)

    *   executables:

    *   reference: CL Simpson, VK Hansen, PC Sham, A Collins, JF Powell, A Al-Chalabi (2004), "MaGIC: A program to generate targeted marker sets for genome-wide association studies", BioTechniques, 37:996-999.

*   MAIA

    *   full name:

    *   version: 1.0 (Jan 2000)

    *   description: complex segregaton analysis in animal pedigrees

    *   authors: Yurii Aulchenko (email: yurii@bionet.nsc.ru)

    *   web/ftp: http://mga.bionet.nsc.ru/soft/maia-1.0/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   MAKEPED

    *   full name: MAKE PEDfiles

    *   alias: LINKAGE/MAKEPED

    *   version: This is a 16 bit program and will not run on 64 bit PCs, see [http://www.jurgott.org/linkage/LinkagePC.html](http://www.jurgott.org/linkage/LinkagePC.html)

    *   description: part of the LINKAGE auxiliary programs

    *   authors: Peter Cartwright (University of Utah, now Cimarron Software pc@cimsoft.com)

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: C and Pascal

    *   operating systems: UNIX, VMS, MS-DOS, OS2

    *   executables:

    *   reference:

*   MALDSOFT

    *   full name:

    *   version: 1.0

    *   description: MALDSoft is a program for admixture mapping of complex trait loci, using case-control data. The samples should come from a recently-admixed population; additional 'learning' samples from the parental populations are helpful.

    *   authors: Giovanni Montana, Jonathan Pritchard (email: jpritcha@genetics.uchicago.edu)

    *   web/ftp: http://pritch.bsd.uchicago.edu/software/maldsoft_download.html

    *   source code language:

    *   operating systems: UNIX, Linux, MS-Windows (DOS/Window...)

    *   executables:

    *   reference: G Montana, JK Pritchard (2004), "Statistical tests for admixture mapping with case-control and cases-only data", American Journal of Human Genetics, 75(5):771-789

*   MAMA

    *   full name: Multivariate Association Mapping Algorithms

    *   version:

    *   description:

    *   authors:

    *   web/ftp: http://polymorphism.ucsd.edu/cgi-bin/PRL/mama/mama.cgi

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference:

*   MANTEL-STRUCT

    *   full name:

    *   version:

    *   description: tests for population structure through the use of Mantel tests

    *   authors: Mark Miller (email: mark.miller@usu.edu)

    *   web/ftp: http://www.marksgeneticsoftware.net/

    *   source code language:

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Miller (1999), "MANTEL-STRUCT: a program for the detection of population structure via mantel tests", Journal of Heredity, 90:258-259.

*   MAOS

    *   full name: Meta-Analysis with Overlapping Subjects

    *   version:

    *   description: Data from genome-wide association studies are often analyzed jointly for the purposes of combining information from multiple studies of the same disease or comparing results across different disorders. In many instances, the same subjects appear in multiple studies. Failure to account for overlapping subjects can greatly inflate type I error when combining results from multiple studies of the same disease and can drastically reduce power when comparing results across different disorders. MAOS implements valid and efficient statistical methods for meta-analysis of genomewide association studies with overlapping subjects. The current release performs logistic regression analysis of individual level data under the additive mode of inheritance.

    *   authors: Danyu Lin, Patrick Sullivan (Univ North Carolina)

    *   web/ftp: http://www.bios.unc.edu/~lin/software/MAOS/

    *   source code language: C++

    *   operating systems:

    *   executables:

    *   reference: Lin, Sullivan (2009), "Meta-analysis of genome-wide association studies with overlapping subjects", American Journal of Human Genetics, 85(6):862-872.

*   MAP/MAP+/MAP+H/MAP2000

    *   full name: (MAP+H is the radiation Hybrid module of MAP+)

    *   version:

    *   description: multiple pairwise linkage analysis under interference

    *   authors: A Collins (email: arc@southampton.ac.uk),

    *   D. Shields,

    *   NE Morton (University of Southampton, UK)

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map

    *   http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map+

    *   http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map+h

    *   http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map2000

    *   source code language: FORTRAN

    *   operating systems: UNIX(SunOS/..)

    *   executables: map

    *   reference: Morton, Andrew (1989), Annals of Human Genetics, 53:263-269.

    *   Morton, Collins (1990), Annals of Human Genetics, 54:103-106.

    *   Shields, Collins, Buetow, Morton (1991), Proceedings of the National Academy of Sciences , 88:6501-6505.

*   MAPCHART

    *   full name: Windows software for the graphical presentation of linkage maps and QTLs

    *   version: 2.2

    *   description: Produces charts of genetic linkage and QTL data. The charts are composed of a sequence of vertical bars representing the linkage groups or chromosomes. On these bars the positions of loci are indicated, and next to the bars QTL intervals and QTL graphs can be shown. MapChart reads the linkage information (i.e. the locus and QTL names and their positions) from text files. Many options to adapt the charts to different purposes. Can produce graphic files (enhanced windows metafile format) which can be enhanced with other MS-Windows software.

    *   Authors: Roeland E Voorrips (Plant Research International, Wageningen, The Netherlands) (email: r.e.voorrips@plant.wag-ur.nl)

    *   web/ftp: http://www.biometris.wur.nl/uk/Software/MapChart/

    *   source code language: Delphi

    *   operating systems: MS-Windows (95/98/ME/NT4.0/2000)

    *   executables:

    *   reference: RE Voorrips (2001), "MapChart version 2.0: Windows software for the graphical presentation of linkage maps and QTLs", Plant Research International, Wageningen, Netherlands.

    *   Voorrips (2002), "MapChart: Software for the graphical presentation of linkage maps and QTLs", Journal of Heredity, 93(1):77-78.

*   MAPCREATOR

    *   full name:

    *   version: 4.0

    *   description: create gene maps using either radiation hybrid data or linkage data

    *   authors: Wes Barris, Rachel Hawken

    *   web/ftp: http://www.wesbarris.com/mapcreator/

    *   source code language: Perl

    *   operating systems:

    *   executables:

    *   reference:

    *   price: Postscript ($69 USD), Image ($69 USD), both ($99 USD)

*   MAPDISTO

    *   full name:

    *   version: v1.7.0 (October 2007)

    *   description: MAPDISTO is a program for mapping genetic markers in experimental segregating populations like backcross, doubled haploids, single-seed descent. Its specificity is to propose recombination fraction estimates in case of segregation distortion. It can (1) compute and draw genetic maps easily and quickly through a graphical interface; (2) facilitate the analysis of marker data showing segregation distortion due to differential viability of gametes or zygotes.

    *   authors: Mathias Lorieux (m.lorieux@cgiar.org)

    *   web/ftp: http://mapdisto.free.fr/

    *   source code language: MS-Excel

    *   operating systems:

    *   executables:

    *   reference:

*   MAPDRAW

    *   full name:

    *   version: 2.2

    *   description: MAPDRAW draws genetic linkage maps on PC same as what MAPMAKER does on Mac.

    *   authors: Renhu Liu (email: liurenhu@21cn.com)

    *   web/ftp: http://www.nslij-genetics.org/soft/mapdraw.v2.2.xls

    *   ftp://brassica.hzau.edu.cn

    *   ftp://211.69.140.177

    *   source code language: Microsoft Visual Basic, Excel macro

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Liu, Meng (2003), "MapDraw: a microsoft excel macro for drawing genetic linkage maps based on given genetic linkage data", Heraditas (Beijing), 25(3):317-321.

*   MAPINSPECT (see also GGT)

    *   full name:

    *   version: May 2002 (previously called MAPCOMP)

    *   description: MAPINSPECT can be used to compare linkage maps obtained from different sources/populations/etc.. It will draw the linkage maps and look for common marker names, these are then connected in the drawing with dashed lines. All neighboring maps are compared in this way. Images can be printed and saved. Orders of the MAPs (ie which map is compared with which other map) can be changed and maps can be flipped (right mouse button). remark: MapComp bears close relationships with the GGT software package

    *   Authors: Ralph van Berloo, Wageningen University, The Netherlands (email: Ralph.vanberloo@pv.dpw.wau.nl)

    *   web/ftp: http://www.dpw.wau.nl/pv/PUB/MapComp/

    *   source code language: Delphi Pascal

    *   operating systems: MS-Window (32-bit)

    *   executables: MapComp.exe

    *   reference:

*   MAPL

    *   full name: MAPping and QTL analysis

    *   version: March 2004

    *   description: The programs allow a user to get results on segregation ratio, linkage test, recombination value, grouping of markers, ordering of markers by metric multidimensional scaling, drawing map and graphical genotype. ALso QTL analysis by interval mapping and ANOVA are possible.

    *   Authors: Ukai Tasuo, Ohsawa Ryo, Saito Akira, Hayashi Takeshi, Yasuo Ukai (University of Tokyo, Japan) (email: tkawai@lbm.ab.a.u-tokyo.ac.jp)

    *   web/ftp: http://lbm.ab.a.u-tokyo.ac.jp/software.html

    *   http://lbm.ab.a.u-tokyo.ac.jp/~ukai/mapl98.html

    *   source code language: Visual Basic (V5, professional)

    *   operating systems: MS-Windows (95/98)

    *   executables:

    *   reference: Yasuo, Ryo, Akira, Takeshi (1995), "MAPL: a package of computer programs for construction of DNA polymorphismlinkage maps and analysis of QTL" (in Japanese), Breeding Science, 45(1):139-142.

*   MAREYMAP

    *   full name:

    *   version:

    *   description: MareyMap is a meiotic recombination rate estimation program.

    *   authors:

    *   web/ftp: http://pbil.univ-lyon1.fr/software/mareymap/

    *   source code language: R, tcl/tk

    *   operating systems:

    *   executables:

    *   reference: Rezvoy, Charif, Gueguen, Marais (2007), "MareyMap: an R-based tool with graphical interface for estimating recombination rate", Bioinformatics, 23:2188-2189.

*   MARGARITA

    *   full name:

    *   version:

    *   description: Margarita infers genealogies from population genotype data and uses these to map disease loci. These genealogies take the form of the Ancestral Recombination Graph (ARG). The ARG defines a genealogical tree for each locus, and as one moves along the chromosome the topologies of consecutive trees shift according to the impact of historical recombination events.

    *   authors: Richard Durbin (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/margarita/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference:

*   MATLINK

    *   full name:

    *   version:

    *   description: A MATLAB utility for estimating genetic linkage in exotic line-cross mating designs

    *   authors: J Clare Nelson (Kansas State University)

    *   web/ftp: http://coding.plantpath.ksu.edu/~jcn/MatLink.html

    *   source code language: MATLAB

    *   operating systems:

    *   executables:

    *   reference:

*   MDBLOCKS

    *   full name: Minimum Description length method for haplotype BLOCKS

    *   version:

    *   description: Using the mimimum description length model to delineate haplotype blocks

    *   authors: Eric C. Anderson and John Novembre (UC Berkeley)

    *   web/ftp: http://ib.berkeley.edu/labs/slatkin/eriq/software/mdb_web/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Anderson, Novembre (2003), "Finding haplotype block boundaries by using the minimum description length criterion", American Journal of Human Genetics, 73(2):336-354

*   MAPMAKER/EXP (see also MMDRAWER)

    *   full name:

    *   version: 3.0 (1992)

    *   description:

    *   authors: Eric Lander, Phil Green, J. Abrahamson, A. Barlow, Mark Daly, Steve Lincoln, L. Newburg (Whitehead Institute)

    *   web/ftp: http://www.broad.mit.edu/ftp/distribution/software/mapmaker3/

    *   http://www-genome.wi.mit.edu/genome_software

    *   source code language: C

    *   operating systems: UNIX, VMS, MS-DOS, MacOS

    *   executables: mapmaker

    *   reference: Lander et al (1987), Genomics,1,174-181.

    *   online tutorial (version 3.0, 1987-1993)

*   MAPMAKER/HOMOZ

    *   full name: homozygosity mapping

    *   version: May 1995

    *   description: calculates multipoint lod scores in pedigrees with inbreeding loops

    *   authors: Leonid Kruglyak (email: leonid@genome.wi.mit.edu),

    *   Mark Daly (email: mjdaly@genome.wi.mit.edu),

    *   Eric Lander (email: lander@genome.wi.mit.edu)

    *   web/ftp: ftp://ftp-genome.wi.mit.edu/distribution/software/homoz

    *   source code language: C

    *   operating systems: UNIX

    *   executables:

    *   reference: Kruglyak, Daly, Lander (1995), "Rapid multipoint linkage analysis of recessive traits in nuclear families, including homozygosity mapping", American Journal of Human Genetics, 56(2):519-527

*   MAPMAKER/QTL

    *   full name:

    *   version: new version in April 1995

    *   description:

    *   authors: Eric Lander, Phil Green, J. Abrahamson, A. Barlow, Mark Daly, Steve Lincoln, L. Newburg (Whitehead Institute)

    *   web/ftp: http://www-genome.wi.mit.edu/genome_software

    *   ftp://ftp-genome.wi.mit.edu/distribution/software/newqtl" and ftp://ftp-genome.wi.mit.edu/distribution/software/mapmaker3

    *   source code language: C

    *   operating systems: UNIX, VMS, MS-DOS, MacOS

    *   executables:

    *   reference: Lander et al (1987), Genomics,1,174-181.

    *   online tutorial and reference manual (v1.1, 1993)

*   MAPMAKER/SIBS (see also GENEHUNTER)

    *   full name:

    *   version : 2.1 (Dec 1996)

    *   description:

    *   authors: Leonid Kruglyak (email: leonid@genome.wi.mit.edu),

    *   Mark Daly (email: mjdaly@genome.wi.mit.edu),

    *   Eric Lander (email: lander@genome.wi.mit.edu) (Whitehead Institute)

    *   web/ftp: ftp://ftp-genome.wi.mit.edu/distribution/software/sibs

    *   source code language: C

    *   operating systems: UNIX

    *   executables: sibs

    *   reference: Kruglyak, Lander (1995), "Complete multipoint sib-pair analysis of qualitative and quantitative traits", American Journal of Human Genetics, 57(2):439-454

    *   the changes from version 1.1 to 2.0 .

    *   manual (in postscript)

*   MAP MANAGER QT

    *   full name:

    *   version : QTb29 (August 2000)

    *   description: A graphic, interactive program to map quantitative trait loci by regression methods; MAP MANAGER CLASSIC enhanced by quantitative trait mapping.

    *   authors: K.F. Manly

    *   web/ftp: http://www.mapmanager.org/mmQT.html

    *   source code language: Pascal

    *   operating systems: MacOS

    *   executables:

    *   reference: Manly and Olsen, Mammalian Genome, 10:327-334 (1999)

*   MAP MANAGER QTX

    *   full name:

    *   version : QTXb20 (April 2004)

    *   description: A graphic, interactive program to map quantitative trait loci using intercrosses, backcrosses or recombinant inbred strains in experimental plants or animals. A completely rewritten cross-platform version of Map Manager QT with enhanced analysis functions.

    *   Authors: KF Manly, RH Cudmore, Jr., and JM Meer

    *   web/ftp: http://www.mapmanager.org/mmQTX.html

    *   source code language: C++ and cross-platform code from XVT

    *   operating systems: MS-Windows, MacOS

    *   executables:

    *   reference: Manly, Cudmore, and Meer (2001), " Map Manager QTX, cross-platform software for genetic mapping", Mammalian Genome, 12(12):930-932.

*   MAPPOP

    *   full name:

    *   version: 1.0 (Jan 2000)

    *   description: selects high resolution mapping subsamples and performs bin mapping

    *   Authors: Dan Brown, Todd Vision (email: tjv@bio.unc.edu)

    *   web/ftp: http://www.bio.unc.edu/faculty/vision/lab/mappop/

    *   source code language: MatLab

    *   operating systems: UNIX, MS-Windows, MacOS, etc

    *   executables:

    *   reference: Vision, Brown, Shmoys, Durrett, Tanksley (2000), "Selective Mapping: A strategy for optimizing the construction of high-density linkage maps", Genetics, 155: 407-420

    *   Brown, Vision, Tanksley (2000), "Selective mapping: A discrete optimization approach to selecting a population subset for use in a high-density genetic mapping project", Proceedings of the 11th Annual {ACM-SIAM} Symposium on Discrete Algorithms, pp.419-428.

    *   Xu, Zou, Vision (2005), "Improving QTL mapping resolution in experimental crosses by the use of genotypically selected samples", Genetics, 170, 401-408.

*   MAPQTL

    *   full name: Software for the calculation of QTL positions on genetic maps

    *   version: 5 (October 2005)

    *   description: Mapping of quantitative trait loci (QTLs) for several types of mapping populations: BC1, F2, RILs, (doubled) haploids, full-sib family of outbreeders. Analyses: interval mapping, composite interval mapping, nonparametric mapping, automatic cofactor selection, permutation test for interval mapping. QTL charts. Everything available in an intuitive MS-Windows user interface.

    *   authors: Johan W Van Ooijen, Martin P Boer, Ritsert C Jansen, Chris Maliepaard (Plant Research International, Biometris, Wageningen, Netherlands) (email: mapping@biometris.nl)

    *   web/ftp: http://www.mapqtl.nl

    *   source code language: ANSI C, Delphi

    *   operating systems: MS-Windows (95/98/ME/NT4.0/2000)

    *   executables:

    *   reference: JW Van Ooijen, MP Boer, RC Jansen, C Maliepaard (2002), "MapQTL (tm) 4.0: Software for the calculation of QTL positions on genetic maps", Plant Research International, Wageningen, Netherlands; JW Van Ooijen, C Maliepaard (1996), "MapQTL Version 3.0: Software for the Calculation of QTL Positions on Genetic Maps", Plant Genome IV (1996) ( http://www.intl-pag.org/4/abstracts/p316.html); JW Van Ooijen, C Maliepaard (1996), "MapQTL Version 3.0: Advanced QTL Mapping Software", Plant Genome IV (1996) ( http://www.intl-pag.org/4/abstracts/c9.html).

*   MCLEEPS (see also PANGAEA)

    *   full name: Monte Carlo Likelihood Estimation of Effective Population Size

    *   version: 1.1 (September 2001)

    *   description:

    *   Authors: Eric Anderson (email: eriq@cqs.washington.edu , University of Washington) and Ellen Williamson (University of California at Berkeley)

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml

    *   http://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm

    *   source code language: C++

    *   operating systems: UNIX, MacOS

    *   executables:

    *   reference: Anderson, Williamson, Thompson (2000), "Monte Carlo evaluation of the likelihood for Ne from temporally spaced samples", Genetics, 156:2109-2118.

*   MCQTL

    *   full name: Multi-Cross QTL mapping

    *   version: v4.0

    *   description: The aim of the MCQTL software package is to perform QTL mapping in multi-cross designs. It allows the analysis of the usual populations derived from inbred lines and can link the families by assuming that the QTL locations are the same in all them. Moreover, a diallel modelling of the QTL genotypic effects is allowed in multiple related families. Obviously, the analysis of a single cross is also feasible.

    *   authors: Marie-Françoise Jourjon, Sylvain Jasson, Jacques Marcel, Baba Ngom, Brigitte Mangin (email: brigitte.mangin@toulouse.inra.fr)

    *   web/ftp: http://carlit.toulouse.inra.fr/MCQTL/

    *   source code language: Java, C++

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference: Jourjon, Jasson, Marcel, Ngom, Mangin (2005), "MCQTL: Multi-allelic QTL mapping in multi-cross design", Bioinformatics, 21:128-130.

*   MDR

    *   full name: Multifactor Dimensionality Reduction

    *   version: 2.0 beta 6

    *   description: MDR is a data mining strategy for detecting and characterizing nonlinear interactions among discrete attributes (e.g. SNPs, smoking, gender, etc.) that are predictive of a discrete outcome (e.g. case-control status). The MDR software combines attribute selection, attribute construction and classification with cross-validation to provide a powerful approach to modeling interactions.

    *   authors: Jason Moore (email:jason.h.moore@dartmouth.edu)

    *   web/ftp: http://www.multifactordimensionalityreduction.org/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Ritchie, Hahn, Roodi, Bailey, Dupont, Parl, Moore (2001), "Multifactorial dimensionality reduction reveals high-order interactions among estrogen metabolism genes in sporadic breat cancer", American Journal of Human Genetics, 69:138-147.

    *   Moore, Gilbert, Tsai, Chiang, Holden, Barney, White (2006), "A flexible computational framework for detecting, characterizing, and interpreting statistical patterns of epistasis in genetic studies of human disease susceptibility", Journal of Theoretical Biology, 241:252-261.

*   MDR-PDT

    *   full name: Multifactor Dimensionality Reduction and genotype Pedigree Disequilibrium Test

    *   version: beta

    *   description:

    *   authors: ER Martin, MD Ritchie, L Hahn, S Kang, JH Moore

    *   web/ftp: http://chgr.mc.vanderbilt.edu/ritchielab/method.php?method=mdrpdt

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Martin, Ritchie, Hahn, Kang, Moore (2006), "A novel method to identify gene-gene effects in nuclear families: the MDR-PDT", Genetic Epidemiology, 30:111-123.

*   MECPM

    *   full name: Maximum Entropy Conditional Probability Moldeling

    *   version:

    *   description:

    *   authors:

    *   web/ftp: http://www.cbil.ece.vt.edu/ResearchOngoingSNP.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Miller, Zhang, Yu, Liu, Chen, Langefeld, Herrington, Wang (2009), "An algorithm for learning maximum entropy probability models of disease risk that efficiently searches and sparingly encodes multilocus genomic interactions", Bioinformatics, to appear.

*   MEGA2

    *   full name: a Manipulation Environment for Genetic Analyses

    *   version: 3.0 R9 (July 2006)

    *   description: Mega2 uses as input a trio of files: 1) a LINKAGE-format locus file modified to contain locus name information; 2) a LINKAGE-format pedigree file; and 3) a map file. Mega2 then takes this trio of input files and, via a menu-driven interface, transforms them into various other file formats, thus greatly facilitating a variety of different analyses. In addition, for many of these options, it also sets up a C-shell script that then can automatically run these analyses (if you are using Mega2 in a Unix environment that supports C-shell scripts).

    *   authors: Nandita Mukhopadhyay, Lee Almasy, Mark Schroeder, William P. Mulvihill, Daniel E Weeks

    *   web/ftp: http://watson.hgen.pitt.edu/register/

    *   source code language: C

    *   operating systems: UNIX (Solaris, Silicon Graphics, OSF1, MacOS X), Linux, MS-Windows, MacOS(X)

    *   executables: mega2

    *   reference: Mukhopadhyay N, Almasy L, Schroeder M, Mulvihill WP, Weeks DE (1999), "Mega2, a data-handling program for facilitating genetic linkage and association analyses" (abstract), American Journal of Human Genetics, 65:A436

    *   Mukhopadhyay, Almasy, Schroeder, Mulvihill, Weeks (2005), "Mega2: data-handling for facilitating genetic linkage and association analyses", Bioinformatics, 21(10):2556-2557.

    *   online documentation: http://watson.hgen.pitt.edu/docs/mega2_html/mega2.html

    *   FAQ-O-MATIC: http://watson.hgen.pitt.edu/fom-serve/cache/2.html

*   MEGASNPHUNTER

    *   full name:

    *   version:

    *   description: MegaSNPHunter takes case-control genotype data as input and produces a ranked list of multi-SNP interactions. In particular, the whole genome is first partitioned into multiple short subgenomes and a boosting tree classifier is built for each subgenomes based on multi-SNP interactions and then used to measure the importance of SNPs. The method keeps relatively more important SNPs from all subgenomes and let them compete with each other in the same way at the next level. The competition terminates when the number of selected SNPs is less than the size of a subgenome.

    *   authors:

    *   web/ftp: http://bioinformatics.ust.hk/MegaSNPHunter.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Wan, Yang, Yang, Xue, Tang, Yu (2009), "MegaSNPHunter: a learning approach to detect disease predisposition SNPs and high level interactions from whole genome data", BMC Bioinformatics, 10:13.

*   MENDEL

    *   full name:

    *   version: 8.0.1 (expire April 2009)

    *   description: genetic analysis of human pedigree data under models involving a small number of loci. MENDEL is useful for segregation analysis, linkage calculations, genetic counseling, allele frequency estimation, and related kinds of problems.

    *   authors: Kenneth Lange (UCLA), Daniel Weeks (Pittsburgh), M. Boehnke

    *   web/ftp: http://www.genetics.ucla.edu/software/

    *   source code language: FORTRAN77

    *   operating systems:

    *   executables:

    *   reference: Lange, Weeks, Boehnke (1988), Genetic Epidemiology, 5:471-471.

    *   module USERM14: Stringham, Boehnke (1996), "Identifying marker typing incompatibilities in linkage analysis", American Journal of Human Genetics, 59(4):946-950

    *   Lange, Cantor, Horvath, Perola, Sabatti, Sinsheimer, Sobel (2001), Am. J. Hum. Genet. 69(suppl), A1886.

*   MENDELSOFT

    *   full name:

    *   version: 0.5b (July 2007)

    *   description: Identifying all Mendelian inconsistencies in complex pedigree data with thousand of individuals, including many loops and several errors. Can also infer missing genotypes.

    *   authors: M Sanchez, Simon de Givry (INRA, Centre de recherches de Toulouse, Station de biométrie et d'intelligence artificielle, France) (email: degivry@toulouse.inra.fr), T Schiex

    *   web/ftp: http://www.inra.fr/mia/T/MendelSoft/

    *   source code language: C++

    *   operating systems: UNIX, Linux/Cygwin, MS-Windows, MacOS

    *   executables:

    *   reference: Sanchez, de Givry, Schiex (2007), "Mendelian error detection in complex pedigrees using weighted constraint satisfaction techniques", Frontiers in Artificial Intelligence and Applications, 163:29-37.

*   MERLIN

    *   full name: Multipoint Engine for Rapid Likelihood INference

    *   version: 1.0.1

    *   description: Merlin carries out single-point and multipoint analyses of pedigree data, including IBD and kinship calculations, nonparametric and variance component linkage analyses, error detection and information content mapping. For multipoint analyses in dense maps, Merlin allows the user to impose constraints on the number of recombinants between consecutive markers. Merlin estimates haplotypes by finding the most likely path of gene flow or by sampling paths of gene flow at all markers jointly. It can also list all possible nonrecombinant haplotypes within short regions. Finally, Merlin provides swap-file support for handling very large numbers of markers as well as gene-dropping simulations for estimating empirical significance levels.

    *   Authors: Goncalo Abecasis (email: goncalo@umich.edu)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/Merlin

    *   http://www.sph.umich.edu/csg/abecasis/Merlin/download

    *   source code language: C++

    *   operating systems: UNIX. LINUX

    *   executables: merlin, pedstats, pedwipe

    *   reference: Abecasis GR, Cherny SS, Cookson WO and Cardon LR (2002), "Merlin-rapid analysis of dense genetic maps using sparse gene flow trees", Nature Genetics, 30:97-101.

*   METAL

    *   full name: METa AnaLysis Helper

    *   version:

    *   description: The METAL software is designed to facilitate meta-analysis of large datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner.

    *   authors: Goncalo Abecasis (email: goncalo@umich.edu)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/Metal/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   MFLINK

    *   full name: (nearly) model-free linkage analysis

    *   version:

    *   description:

    *   authors: Dave Curtis, (email: dcurtis@hgmp.mrc.ac.uk) P.C. Sham

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Curtis, Sham (1995), "Model-free linkage analysis using likelihoods", American Journal of Human Genetics, 57(3):703-716

*   MGA-MAPF2

    *   full name:

    *   version:

    *   description: map QTLs in F-2 intercross in model organisms

    *   authors:

    *   web/ftp: http://mga.bionet.nsc.ru/soft/mgamapf2/

    *   source code language: FORTRAN

    *   operating systems: MS-DOS, MS-Windows (95/98/NT)

    *   executables:

    *   reference:

*   MIDAS

    *   full name: Multiallelic Interallelic Disequilibrium Analysis Software

    *   version: 1.0

    *   description: software for analysis and visualisation of interallelic disequilibrium between multiallelic markers

    *   authors: Tom R Gaunt, Santiago Rodriguez, Carlos Zapata, Ian NM Day

    *   web/ftp: http://www.genes.org.uk/software/midas

    *   http://www.sgel.humgen.soton.ac.uk/midas

    *   source code language: Python

    *   operating systems: MS-Windows(2000/XP), Linux, MacOS

    *   executables:

    *   reference: Gaunt, Rodriguez, Zapata, Day (2006), "MIDAS: software for analysis and visualisation of interallelic disequilibrium between multiallelic markers", BMC Bioinformatics, 7:227.

*   MILD

    *   full name: MultIallelic Linkage Disequilibrium: a program for adjusted linkage disequilibrium (LD) calculations

    *   version: 2.1 (2002)

    *   description:

    *   authors: Yuril S Aulchenko (email: aulchenko@epib.fgg.eur.nl) (Erasmus University Medical School, The Netherlands)

    *   web/ftp: http://www.geneticepi.com/Research/software/software.html

    *   source code language:

    *   operating systems: UNIX, Linux

    *   executables:

    *   reference:

*   MIM

    *   full name: Multipoint Identical-by-descent Method

    *   version: 1.21 (March 1996)

    *   description: multipoint IBD method for partitioning genetic variance of quantitative traits to specific chromosome regions using data on nuclear families.

    *   authors: David Goldgar, Cathryn M. Lewis (Univ. Utah). contact person: Edward Kort at edward@episun2.med.utah.edu

    *   web/ftp: ftp://morgan.med.utah.edu/pub/Mim

    *   source code language: C

    *   operating systems: UNIX

    *   executables:

    *   reference: Goldgar (1990), "Multipoint analysis of human quantitative genetic variation", American Journal of Human Genetics, 47(6): 957-967

    *   Goldgar, Oniki (1992), "Comparison of a multipoint identity-by-descent method with parametric multipoint linkage analysis for mapping quantitative traits", American Journal of Human Genetics, 50(3):598-606

    *   online manual (from Wellcome Trust)

*   MINIMAC (see also MACH)

    *   full name:

    *   version:

    *   description: minimac is a low memory, computationally efficient implementation of the MaCH algorithm for genotype imputation. It is designed to work on phased genotypes and can handle very large reference panels with hundreds or thousands of haplotypes. The name has two parts. The first, "mini", refers to the modest amount of computational resources it requires. The second, "mac", is short hand for MaCH, our widely used algorithm for genotype imputation.

    *   authors: Goncalo Abecasis, Christian Fuchsberger (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/Minimac

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   MINSAGE

    *   full name: MINimal SAmple size for GEnotypes

    *   version:

    *   description: calculate the sample size of genotypes minimally required to ensure that all alleles with a specified frequency at one locus are detected with a given confidence

    *   authors: Andreas Ziegler (email: koenigir@imbs.uni-luebeck.de)

    *   web/ftp: http://www.imbs.uni-luebeck.de/pub/minsage/index.html

    *   source code language:

    *   operating systems: MS-Windows

    *   executables:

    *   reference: HG Gregorius (1980), "The probability of losing an allele when diploid genotypes are sampled", Biometrics, 36:643-652.

*   MITPENE

    *   full name:

    *   version:

    *   description: A program for analysis of mitochondrial diseases

    *   authors:

    *   web/ftp: http://mga.bionet.nsc.ru/soft/mitpene/mitpene.html (in Russian)

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   MIXSCORE

    *   full name:

    *   version: 1.9 (March 2011)

    *   description: MIXSCORE is a method for combining SNP association and admixture association statistics to increase power in GWAS in admixed populations.

    *   authors: Alkes Price (Harvard Univ)

    *   web/ftp: http://www.hsph.harvard.edu/faculty/alkes-price/software/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Pasaniuc et al. (2011), "Enhanced statistical tests for GWAS in admixed populations: assessment using African Americans from CARe and a breast cancer consortium", PLoS Genetics, 7:e1001371.

*   MKGST

    *   full name: Michael Krawczak's Genetics Software Tools, including ASP (power calculator for gene mapping using a sibpair design), ASPSHARE (rapid calculation of the expected ibd sharing at the trait locus, based upon the model), EASYPAT (calculation of likelihood ratios for single locus data, comparing specific types of simple hypotheses regarding the familial relationships involved), MUTPROF/MUTCOMP (comparison of mutation profiles), FINDSIRE (identify mothers or sires by means of the comparison of a large number of potential parents, typed at single locus DNA markers, with given infants or parent-infant duos), PATERN (calculation of paternity probabilities from the multilocus DNA profiles of trios, comprising mother, child and putative father)

    *   version:

    *   description:

    *   Authors: Michael Krawczak

    *   web/ftp: http://www.uni-kiel.de/medinfo/mitarbeiter/krawczak/download/index.html

    *   source code language:

    *   operating systems:

    *   executables: asp, aspshare,easypat, mutprof,mutcomp, findsire,patern,fingcomp,hapmax, mutpred

    *   reference:

*   MMDRAWER

    *   full name:

    *   version: 0.2.0.0 (April 2003)

    *   description: draw chromosome maps from the output of MAPMAKER/EXP. The output format is a Enhanced Metafile, which can be imported into most Windows-based presentation or document editing programs.

    *   authors: Michael Brown

    *   web/ftp: http://www.emboss.co.nz/products.php?pid=2

    *   source code language: Delphi

    *   operating systems: MS-Windows

    *   executables:

    *   reference:

*   MLBGH

    *   full name: Maximum-Likelihood-Binomial analysis of affected sib-pair and sibship data extended from the GENEHUNTER program

    *   version: 1.0

    *   description: an extension of the GENEHUNTER program to perform sib-pair and sib-ship linkage analysis using the Maximum Likelihood Binomial (MLB) method.

    *   Authors: Laurent Abel (INSERM Unit 436, Paris, France, email: abel@biomath.jussieu.fr), Bertram Muller-Myhsok (Bernhard Nocht Institute, Hamburg, Germany, email: bmm@bni.uni-hamburg.de), Anne Philippi (INSERM Unit 436, Paris, France, email: philippi@biomath.jussieu.fr)

    *   web/ftp: ftp://ftp.biomath.jussieu.fr/pub/mlbgh (not available)

    *   http://genamics.com/software/downloads/mlbgh-1.0.tar.Z

    *   source code language: C

    *   operating systems: UNIX

    *   executables: mlbgh

    *   reference: Abel, Muller-Myhsok (1998), "Robustness and power of the maximum-likelihood-binomial and maximum-likelihood-score methods, in multipoint linkage analysis of affected-sibship data", American Journal of Human Genetics, 63(2):638-647

*   MLD

    *   full name:

    *   version: Aug 1997

    *   description: Performing a shuffling version of the exact conditional tests for different combinations of allelic and genotypic disequilibrium on haploid and diploid data, or their combination.

    *   Authors: Dmitri Zaykin (email: zaykin@statgen.ncsu.edu)

    *   web/ftp: ftp://statgen.ncsu.edu/pub/zaykin/

    *   source code language: C++

    *   operating systems: UNIX, MS-Windows(95)

    *   executables: mld.x

    *   reference: Zaykin, Zhivotovsky, Weir (1995), Genetica, 96:169-178.

*   MLR-TAGGING

    *   full name: Multivariate Linear Regression TAGGING

    *   version: 1.1

    *   description: MLR-TAGGING can be used for tagging SNP selection and genotype prediction

    *   authors: Alexander Zelikovsky (Georgia State Univ) (email: alexz@cs.gsu.edu), Jingwu Jim He (email: jingwu@cs.gsu.edu)

    *   web/ftp: http://alla.cs.gsu.edu/~software/tagging/tagging.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: He, Zelikovsky (2006), "MLR-tagging: informative SNP selection for unphased genotypes based on multiple linear regression", Bioinformatics, 22(20):2558-2561.

*   MOSCPHASER

    *   full name: Mixture Of Cnv-Snp PHASER

    *   version:

    *   description: inferring haplotypes composed of both CNV alleles and SNP alleles

    *   authors: Mamoru Kato, Yusuke Nakamura, Tatsuhiko Tsunoda (email: tsunoda@src.riken.jp)

    *   web/ftp: http://emu.src.riken.jp/MOCSphase/MOSCphaser.zip

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Kato, Nakamura, Tsunoda (2008), "MOCSphaser: a haplotype inference tool from a mixture of copy number variation and single nucleotide polymorphism data", Bioinformatics, 24(14):1645-1646.

    *   http://emu.src.riken.jp/MOCSphase/SuppInfor.doc

*   MOLKIN

    *   full name:

    *   version: 2.0

    *   description: MOLKIN is a population genetics computer program that conducts several genetic analyses on multilocus information in a user-friendly environment. Primary functions carried out by MOLKIN are the computation of the between individuals (and populations) molecular coancestry coefficients, the Kinship distance at individual and population levels. Additionally, users can compute with MOLKIN a set of among populations, genetic distances and F statistics from multilocus information. The program will help researchers or those responsible for population management to assess genetic variability and population structure at reduced costs with respect to dataset preparation

    *   authors: JP Gutierrez, LJ Royo, I Alvarez, F Goyache (Universidad Complutense de Madrid, Spain)

    *   web/ftp: http://www.ucm.es/info/prodanim/html/JP_Web.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Gutierrez, Royo, Alvarez, Goyache (2005), "Molkin v2.0: a computer program for genetic analysis of populations using molecular coancestry information", Journal of Heredity, 96:718-721.

*   MORGAN (see also PANGAEA)

    *   full name: MOnte caRlo Genetic ANalysis

    *   version: version 2.8.3 (March 2008)

    *   description: Programs for segregation and linkage analysis, using a variety of Markov chain Monte Carlo (MCMC) methods. Includes MCMC methods for multilocus gene identity by descent (including homozygosity mapping) and Monte Carlo Lod scores. Also, other programs for EM analysis of quantitative traits. MORGAN 2.5 replaces MORGAN 2.4.1 (June 2002) which superceded MORGAN 2.4 (February 2002) and MORGAN 2.3.2 (September, 2001). MORGAN 2.4 supercedes MORGAN 2.3 (April 2001), MORGAN 2.2 (April 2000) and MORGAN 1.1.2 (July 1999)

    *   authors: For MORGAN 2.5: M A Jewett (University of Washington, email: myrna@stat.washington.edu) Other authors of MORGAN programs include: EA Thompson (University of Washington) AW George, CJ Geyer, S Guo, SC Heath, Michael Li, S Lin, NA Sheehan, and G Snow .

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/MORGAN/Morgan.shtml

    *   source code language: C

    *   operating systems: UNIX (Compaq-Alpha, Solaris and others), Linux

    *   reference: Thompson (1995) "Monte Carlo in Genetic Analysis", Technical report No. 294, Department of Statistics, University of Washington. [postscript]

    *   Thompson (2000) Statistical Inferences from Genetic Data on Pedigrees NSF-CBMS Regional Conference Series in Probability and Statistics. Volume 6\. IMS, Beachwood, OH.

*   MPDA

    *   full name: Microarray Pooled DNA Analyser

    *   version: 2.1 (March 2008)

    *   description: MPDA is an innovative tool for analyzing hybridization intensity data from microarray-based pooled DNA experiments. MPDA was developed under the software platform, MATLABR, and provided user-friendly interfaces adapted to Windows systems (Windows 98, Windows 2000 and Windows XP). or users without installing software MATLABR, we also developed stand-alone executables generated via the MATLABR compiler. MPDA provides four major functions: (1) Whole-genome DNA amplification/hybridization analysis, (2) Allele frequency estimation, (3) Association mapping, (4) Allelic imbalance detection. Graphic and numerical outputs from MPDA support global and detailed inspection for bulk of genomic data.

    *   authors: Hsin-Chou Yang (email:hsinchou@stat.sinica.edu.tw), Mei-Chu Huang, Ling-Hui Li, Chien-Hsing Lin, Alice L T Yu, Mitchell B Diccianni, Jer-Yuarn Wu, Yuan-Tsong Chen, Cathy S J Fann

    *   web/ftp: http://www.stat.sinica.edu.tw/hsinchou/genetics/pooledDNA/mpda.htm

    *   source code language: MATLABR

    *   operating systems: MS-Windows(Windows98/2000/XP)

    *   executables: mpda

    *   reference: Yang, Huang, Li, Lin, Yu, Diccianni, Wu, Chen, Fann (2008), "MPDA: Microarray Pooled DNA Analyzer", BMC Bioinformatics, 9:196.

*   MRH

    *   full name: Minimum Recombinant Haplotype

    *   version: 0.2 (May 2003)

    *   description:

    *   Authors: Dajun Qian (City of Hope National Medical Center, email: dqian@coh.org)

    *   web/ftp: http://www-rcf.usc.edu/~gqian/software.htm (not available)

    *   source code language:

    *   operating systems: MS-Windows, UNIX(Solaris)

    *   executables:

    *   reference: Qian, Beckmann (2002), "Minimum-recombinant haplotyping in pedigrees", American Journal of Human Genetics, 70(6):1434-1445

*   MULTIDISEQ

    *   full name:

    *   version:

    *   description: MULTIDISEQ is multipoint linkage analysis software which allows Marker-Marker LD

    *   authors: Tero Hiekkalinna (email: tero.hiekkalinna@helsinki.fi)

    *   web/ftp: http://www.helsinki.fi/~tsjuntun/multidiseq/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: T Hiekkalinna, L Peltonen1, JD Terwilliger (2003), "MULTIDISEQ: Computer software for multipoint linkage analysis of complex traits allowing for marker-marker LD (Linkage Disequilibrium)" (abstract), American Journal of Human Genetics, suppl, 73

*   MULTIMAP

    *   full name:

    *   version:

    *   description: program for automated construction of genetic maps

    *   authors: Tara Matise (email: matise@biology.rutgers.edu), Mark Perlin, Aravinda Chakravarti

    *   web/ftp: http://compgen.rutgers.edu/Multimap/

    *   http://compgen.rutgers.edu/multimap/multimapdist.html

    *   source code language: LISP

    *   operating systems: UNIX (Sun/Compaq-Alpha/HP..)

    *   executables:

    *   reference: Matise,Perlin, Chakravarti, Nature Genetics, 6, 384-390 (1994);

*   MULTIMAPPER

    *   full name:

    *   version:

    *   description: Bayesian QTL mapping software for analysing backcross, double haploid, and F2 data from designed crossing experiments of inbred lines

    *   authors: Mikko J Sillanpaa (mail: mjs@rolf.helsinki.fi)

    *   web/ftp: http://www.rni.helsinki.fi/~mjs/

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Sillanpaa, Arjas (1998), "Bayesian mapping of multiple quantitative trait loci from incomplete inbred line cross data", Genetics, 148:1373-1388.

    *   Martinez, Thorgaard, Robison, Sillanpaa (2005), "An application of Bayesian QTL mapping to early development in double haploid lines of rainbow trout including environmental effects", Genetical Research, 86:209-221.

*   MULTIMAPPER/OUTBRED

    *   full name:

    *   version: 1.1

    *   description: Bayesian QTL mapping software for analysing backcross and F2 data from designed crossing experiments of outbred lines

    *   authors: Mikko J Sillanpaa (mail: mjs@rolf.helsinki.fi)

    *   web/ftp: http://www.rni.helsinki.fi/~mjs/

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Sillanpaa, Arjas (1999), "Bayesian mapping of multiple quantitative trait loci from incomplete outbred offspring data", Genetics, 151:1605-1619.

*   MULTIPOPTAGSELECT

    *   full name:

    *   version: 1.1

    *   description: The program selects a near-minimal set of tagging single-nucleotide polymorphisms (tagSNPs) that account for all observed patterns of linkage disequilibrium (LD) in multiple populations.

    *   authors: Bryan N. Howie, Christopher S. Carlson, Mark J. Rieder, and Deborah A. Nickerson (University of Washington)

    *   web/ftp: http://droog.gs.washington.edu/multiPopTagSelect.html

    *   source code language: Perl

    *   operating systems:

    *   executables: multiPopTagSelect.pl

    *   reference: Howie, Carlson, Rieder, Nickerson (2006), "Efficient selection of tagging single-nucleotide polymorphisms in multiple populations", Human Genetics, 120(1):58-68.

*   MULTIQTL

    *   full name:

    *   version: 2.6

    *   description: MultiQTL software integrates a broad spectrum of data mining, statistical analysis, interactive visualization and modeling tools that allow QTL analysis based on advanced and sophisticated methods for maximum extraction of the mapping information from data.

    *   authors: Efim Ronin, David Mester, Dina Minkov, Ori Orion, Abraham Korol

    *   web/ftp: http://www.multiqtl.com/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

    *   availability: contact: sales@multiqtl.com. tel: 972-4824-0449\. FAX: 972-4824-6554. address: MultiQTL Ltd, Institute of Evolution, Haifa University, Haifa, Israel 31905

*   MULTISIM

    *   full name:

    *   version: 1.1

    *   description: analyze the numbers of individuals that founded new populations following a bottleneck or founding event

    *   authors: Mohamed Noor (email: noor@duke.edu)

    *   web/ftp: http://www.biology.duke.edu/noorlab/multsim.html

    *   source code language:

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Noor, Pascual, Smith (2000), "Genetic variation in the spread of Drosophila subobscura from a nonequilibrium population", Evolution, 54:696-703.

*   MUTAGENESYS

    *   full name:

    *   version:

    *   description: MutaGeneSys is a system that uses genome-wide genotype data to estimate disease susceptibility. Our system integrates three data sources: HapMap, whole-genome marker correlation data, and OMIM database. It accepts SNP data of individuals as query input and delivers disease susceptibility hypotheses even if the original set of typed SNPs is incomplete.

    *   authors: Julia Stoyanovich, Itsik Pe'er (Columbia University)

    *   web/ftp: http://magnet.c2b2.columbia.edu/mutagenesys/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Stoyanovich, Pe'er (2008), "MutaGeneSys: estimating individual disease susceptibility based on genome-wide SNP array data", Bioinformatics, 24(3):440-442.

*   NOCOM

    *   full name:

    *   version: Aug 1992

    *   description: estimate parameters for mixture of normal distributions

    *   Authors: Jurg Ott (Rockefeller University)

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: FORTRAN

    *   operating systems: MS-DOS, OS2

    *   executables:

    *   reference: Ott (1979), "Detection of rare major genes in lipid levels", Human Genetics, 51:79-91.

    *   online user's guide

*   NOPAR

    *   full name:

    *   version: Aug 1995

    *   description: non-parametric linkage and association tests primarily for a quantitative trait

    *   authors: A Collins (email: arc@southampton.ac.uk), NE Morton. (University of Southampton, UK)

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/nopar

    *   source code language: FORTRAN

    *   operating systems: UNIX(SunOS)

    *   executables:

    *   reference: Collins,Morton (1995), Human Heredity, 45:311-318.

*   NUCULAR

    *   full name:

    *   version:

    *   description: A program that splits extended pedigrees into nuclear families, with the option of recoding all sibs as half-sibs with distinct mothers of fathers.

    *   authors: Mathieu Lemire (email: mlemire@sdf.lonestar.org)

    *   web/ftp: http://mlemire.freeshell.org/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Lemire (2005), "A simple nonparametric multipoint procedure to test for linkage through mothers or fathers as well as imprinting effects in the presence of linkage", BMC Genetics, 6(Suppl 1):S159.

*   ONEMAP (see R/ONEMAP)

*   OLORIN

    *   full name:

    *   version: October 2011

    *   description: Olorin is an interactive filtering tool for next generation sequencing data coming from the study of large complex disease pedigrees. Olorin, is a tool which integrates gene flow output from Merlin and next generation sequencing data. Users can interactively filter and prioritize variants based on haplotype sharing across different sets of selected individuals and allele frequency in reference datasets.

    *   authors: James Morris, Jeffrey Barrett (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/olorin/

    *   http://sourceforge.net/p/olorin/home/

    *   source code language: Java

    *   operating systems: any platform with Java 1.6 or later

    *   executables:

    *   reference:

*   OSA

    *   full name: Ordered Subset Analysis

    *   version: 2.1

    *   description: OSA allows the researcher to evaluate evidence for linkage even when heterogeneity is present in a data set. This is not an unusual occurrence when studying diseases of complex origin. Families are ranked by covariate values in order to test evidence for linkage among homogeneous subsets of families. Because families are ranked, a priori covariate cutpoints are not necessary. Covariates may include linkage evidence at other genes, environmental exposures, or biological trait values such as cholesterol, age at onset, and so on.

    *   authors: William Duren, Mike Boehnke, Elizabeth Hauser

    *   web/ftp: http://wwwchg.duhs.duke.edu/software/osa.html

    *   source code language: C++

    *   operating systems: UNIX(Solaris), Linux

    *   executables: osa2.0

    *   reference: ER Hauser, MP Bass, ER Martin, RM Watanabe, WL Duren, M Boehnke (2001), Am J Hum Genet, suppl 69:529.

    *   ER Hauser, RM Watanabe, WL Duren, MP Bass, CD Langefeld, M Boehnke (2004), "Ordered subset analysis in genetic linkage mapping of complex traits", Genetic Epidemiology, 27(1):53-63.

*   OSIRIS

    *   full name:

    *   version: 1.1

    *   description: OSIRIS is a tool for the retrieval of articles from MEDLINE related to the sequence variants reported for a human gene. The variations considered are single nucleotide polymorphisms (SNPs), insertion/deletion polymorphisms (indel), microsatellite, and named variations (e.g. Alu sequences).

    *   authors: Julio Bonis , Laura Ines Furlong Ferran Sanz (University of Pompeu Fabra, Spain) (email: fsanz@imim.es)

    *   web/ftp: http://ibi.imim.es/osirisform.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Bonis, Furlong, Sanz (2006), "OSIRIS: a tool for retrieving literature about sequence variants", Bioinformatics, 22(20):2567-2569.

*   P_ACT

    *   full name: P-values: Adjustment for Correlated Tests

    *   version: 1.2

    *   description: P_ACT is an R program that adjusts sets of up to 1000 p-values from association tests between correlated traits and SNPs for multiple testing, accounting for the correlation between tests.

    *   authors:

    *   web/ftp: http://csg.sph.umich.edu/boehnke/p_act.php

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Conneely, Boehnke (2007), "So many correlated tests, so little time! Rapid adjustment of p-values for multiple correlated tests", American Journal of Human Genetics, 81(6):1158-1168

*   PANGAEA (see also BOREL, ECLIPSE, HARDY, INSEGT, LOKI, MCLEEPS, MORGAN, PEDFIDDLER, PEDPACK)

    *   full name: Pedigree ANalysis for Genetics (And Epidemiological Attributes)

    *   version:

    *   description: Collection of nine packages for genetic analysis; | the original four BOREL, HARDY, MORGAN and PEDPACK, plus INSEGT, LOKI, MCLEEPS, PEDFIDDLER, and ECLIPSE

    *   Authors: many authors: see individual packages. managed by EA Thompson (University of Washington, email: thompson@stat.washington.edu).

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml

    *   source code language: C or C++

    *   operating systems: UNIX (Compaq, Solaris, and others), Linux

    *   executables:

    *   reference:

*   PAP

    *   full name: Pedigree Analysis Package

    *   version: 6.0 (December 2006)

    *   description: it can perform (1) compute the likelihood of specified parameter values; (2) compute the probability of each genotype for pedigree members; (3) simulate phenotypes for output into files; (4) maximize the likelihood over specified parameters (with or without standard errors); (5) compute the standard errors of parameters for unknown estimates; (6) simulate phenotypes and estimate parameter values; (7) estimate expected lod score; (8) compute a grid of likelihood over one or two parameters. New additions to V5: assortative mating; TDT; additive multi-locus models

    *   authors: Sandra Hasstedt (University of Utah, email: sandy@genetics.utah.edu)

    *   Peter Cartwright contributed to versions 2 and 3; many others contributed to version 1 (University of Utah)

    *   web/ftp: http://hasstedt.genetics.utah.edu/

    *   source code language: FORTRAN 77

    *   operating systems: UNIX

    *   executables: preped, descstat, prepap,simul, papdr (with 8 options),gpe

    *   reference: manual (version 5, PDF) online manual (version 4) (from Wellcome Trust)

*   PARENTE

    *   full name:

    *   version: 1.2 (February 2003)

    *   description: parentage inference using molecular data from diploid codominant markers

    *   authors: Stephanie Blanc-Manel (email: stephanie.manel@ujf-grenoble.fr)

    *   web/ftp: http://www2.ujf-grenoble.fr/leca/membres/manel.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Cercueil, Bellemain, Manel (2002), "PARENTE: computer program for parentage analysis", Journal of Heredity, 93:458-459.

*   PASS PEDIGREE

    *   full name:

    *   version:

    *   description: PASS Pedigree is drawing the most complex family trees in a matter of minutes instead of hours of work. The basis of this is an algorithm for automatically builing a family tree. Of course, manual adjustments in the family tree can be made for your specific requirements. PASS Pedigree meets all international conventions concerning the drawing of pedigrees. A converter can convert historical Cyrillic pedigrees automatically to PASS Pedigree. Unlike before, all your family trees are stored in one single database. PASS Pedigree can intelligently connect to many genetic centers (e.g. three genetic centers in the Netherlands) with the existing patient information, via the lab system HELIX based on HL7 techniques.

    *   authors: PASS (Pro Active Software Solutions) Software

    *   web/ftp: http://www.pass-software.com/Producten/PASSPedigree.aspx (in Dutch)

    *   http://www.pass-software.com/Producten/PASSPedigree/tabid/113/language/en-US/Default.aspx (in English)

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

    *   availability: price

*   PATH

    *   full name:

    *   version: 1.0.7 (September 28, 2009)

    *   description: The Path web application is a first-step bioinformatics approach to investigate gene-gene interactions in genetic association studies. Path is designed to do: 1\. Interface your SNP data with biological information from several online bioinformatics databases. 2. Generate biologically plausible hypotheses for testing gene-gene interactions. 3\. Select a subset of SNPs and conduct SNP-SNP interaction tests. 4\. Store analysis results. 5\. Explore analysis results through interactive plots and summary tables.

    *   authors: email: genapha@mrl.ubc.ca

    *   web/ftp: http://genapha.icapture.ubc.ca/PathTutorial/

    *   source code language:

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Zamer, Tripp, Ellis, Daley (2009), "Path: a tool to facilitate pathway-based genetic association analysis", Bioinformatics, 25(18):2444-2446.

*   PAWE

    *   full name: Power for Association With Error

    *   version: Dec 2002

    *   description: Power and sample size calculations for genetic case-control association studies allowing for errors

    *   authors: Derek Gordon (email: gordon@linkage.rockefeller.edu), assisted by Michael Nothnagel

    *   web/ftp: http://linkage.rockefeller.edu/pawe/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Gordon, Finch, Nothnagel, Ott (2002), Human Heredity, 54:22-33.

*   PAWE-3D

    *   full name: Power for Association With Error in 3D

    *   version:

    *   description:

    *   authors: D Gordon, C Haynes, J Blumenfeld, SJ Finch

    *   web/ftp: http://linkage.rockefeller.edu/pawe3d/

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference: Gordon, Haynes, Blumenfeld, Finch (2005), "PAWE-3D: visualizing Power for Association With Error in case/control genetic studies of complex traits", Bioinformatics, 21:3935-3937.

*   PBAT (see also FBAT)

    *   full name: Power calculation of family-Based Association Tests

    *   version: 3.6.1

    *   description: PBAT is an interactive software package that provides tools for the design and the data analysis of family-based association studies.

    *   authors: Christoph Lange (email: clange@hsph.harvard.edu) (Harvard Univ).

    *   web/ftp: http://www.biostat.harvard.edu/~clange/default.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Lange, DeMeo, Silverman, Weiss, Laird (2003), "Using the noninformative families in family-based association tests: A powerful new testing strategy", American Journal of Human Genetics, 73(4):801-811

    *   Lange, DeMeo, Silverman, Weiss, Laird (2004), "PBAT: tools for family-based association studies", American Journal of Human Genetics, 74(2):367-369

    *   Steen, Lange (2005), "PBAT: A comprehensive software package for genome-wide association analysis of complex family-based studies", Human Genomics, 2(1):67-69.

    *   Hoffmann, Lange (2006), "P2BAT: a massive parallel implementation of PBAT for genome-wide association studies in R", Bioinformatics, 22:3103-3105.

*   PDA

    *   full name: Pooled DNA Analyzer

    *   version:

    *   description: a tool for analyses of pooled DNA data

    *   authors: Hsin-Chou Yang, Chia-Ching Pan, Chin-Yu Lin and Cathy SJ Fann (Institute of Biomedical Sciences, Academia Sinica, Taiwan)

    *   web/ftp: http://www.ibms.sinica.edu.tw/~csjfann/first%20flow/programlist.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Yang, Pan, Lin, Fann (2006), "PDA: pooled DNA analyzer", BMC Bioinformatics, 7:233.

*   PDPSYS

    *   full name: Pedigree Data Processing SYStem

    *   version: 1.0 (November 1998)

    *   description: PDPSys is a Windows-based system designed for pedigree data management, providing a graphical interface for pedigree construction and output.

    *   Authors: Fil Quiaoit (Fred Hutchinson Cancer Research Center, email: fquiaoit@fhcrc.org)

    *   web/ftp: http://cougar.fhcrc.org/software.php

    *   source code language: Visual Basic, MS-Access

    *   operating systems: MS-Windows (95)

    *   executables: pdpsys

    *   reference:

*   PDT

    *   full name: pedigree disequilibrium test

    *   version: 5.1

    *   description: The PDT analysis program allows the user to evaluate evidence of linkage disequilibrium (LD) in general pedigree data. All family data may be used without nullifying the validity of the association test, even when there is more than one affected in a family. The PDT program performs both allele-specific and genotype-specific LD analysis of individual markers. Version 5.1 adds the ability to perform genotype-specific analysis over marker sets.

    *   Authors: Eden Martin (Duke Univ)

    *   web/ftp: http://www.chg.duke.edu/software/pdt.html

    *   source code language:

    *   operating systems: UNIX(Solaris)

    *   executables:

    *   reference: Martin, Monks, Warren, Kaplan (2000), American Journal of Human Genetics, 67(1):146-154

*   PED

    *   full name: PEdigree Drawing software

    *   version: 5.1.0 (October 2006)

    *   description: Powerful pedigree drawing program with two drawing modes: Input mode for fast and automatic drawing; edit mode for text annotations, legends, special symbols. Both input and edit mode provide auto numbering, auto resizing of symbols and fonts, and zooming. PED 4.2a complies with the "Recommendations for standardized human pedigree nomenclature". Apart from fully sizeable printed output, pedigrees can be exported as metafiles to virtually any Windows word processor or drawing program.

    *   Authors: Hansjoerg Plendl (email: plendl@medgen.uni-kiel.de)

    *   web/ftp: http://www.medgen.de/index.html

    *   http://www.medgen.de/ped/index.htm

    *   source code language: Visual Smalltalk

    *   operating systems: MS-Windows (95/98/NT)

    *   executables: ped.exe

    *   reference: Bennet, Steinhaus, Uhrich, O'Sullivan, Resta, Lochner-Doyle, Markel, Vincent, Hamanishi(1995), "Recommendations for standardized human pedigree nomenclature. pedigree standardization task force of the national society of genetic counselors", American Journal of Human Genetics, 56(3):745-752

    *   A Golla (1997), Medizinische Genetik, 4:595-597.

    *   H Plendl (1998), Medizinische Genetik, 1:50-51.

    *   price: site license US$99.

*   PEDAGREE

    *   full name: 1.00 (February 2002)

    *   version:

    *   description: a program for detecting autosomal marker Mendelian incompatibilities in pedigree data

    *   authors: Christine Couillault (email: kristin.c@free.fr)

    *   web/ftp: http://pedagree.free.fr/

    *   source code language: C, C++

    *   operating systems: UNIX, Linux, MS-Windows, MS-DOS,

    *   executables: pedagree

    *   reference:

*   PEDCHECK

    *   full name:

    *   version: 1.0 (November 1998)

    *   description: identifying all Mendelian inconsistencies in pedigree data.

    *   Authors: J. R. O'Connell (email: jeff@sherlock.hgen.pitt.edu)

    *   web/ftp: http://watson.hgen.pitt.edu/register

    *   source code language: C, C++

    *   operating systems: UNIX(SunOS/Solaris/Compaq-Alpha/SGI-IRIX/..)

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting 1997 (O'Connell, Weeks, Am. J. Hum. Genet. suppl,61, A288 (1997)); O'Connell, Weeks (1998), "PedCheck: a program for identification of genotype incompatibilities in linkage analysis", American Journal of Human Genetics, 63(1):259-266

    *   FAQ-O-MATIC: http://watson.hgen.pitt.edu/fom-serve/cache/8.html

*   PEDFIDDLER (see also PEDPACK in PANGAEA)

    *   full name:

    *   version: 0.5 (August 2005)

    *   description: This suite of six programs can be used as a stand-alone extension of the pedigree drawing facilities found in the publicly available version of PEDPACK.

    *   Authors: J C. Loredo-Osti (email: josti@bagel.epi.mcgill.ca), Kenneth Morgan (email: ken@bagel.epi.mcgill.ca); Window version adapted by Michael Li (email:lina@u.washington.edu).

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/Pedfiddler.shtml

    *   source code language: C,C++

    *   operating systems: UNIX (OSF1, and Solaris 2.7, AlphaLinux) MS-Windows (XP Home/Win32/Win95), Linux

    *   executables:

    *   reference:

*   PEDHUNTER

    *   full name:

    *   version:

    *   description: a software package that facilitates creation and verification of pedigrees within large genealogies.

    *   Authors: Agarwala, Biesecker, Hopkins, Francomano, Schffer (NIH)

    *   web/ftp: http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/pedhunter.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Agarwala, Biesecker, Hopkins, Francomano, Schffer (1998), "Software for constructing and verifying pedigrees within large genealogies and an application to the Old Order Amish of Lancaster county", Genome Research, 8:211-221.

*   PEDIGRAPH

    *   full name:

    *   version: 2.3

    *   description: A pedigree visualization program specifically designed to draw large, complex pedigrees.

    *   authors: John Garbe (University of Minnesota, email: jgarbe@umn.edu), Yang Da (University of Minnesota, email: yda@umn.edu)

    *   web/ftp: http://animalgene.umn.edu/pedigraph/

    *   source code language: C,C++

    *   operating systems: MS-Windows, Linux

    *   executables: pedigraph

    *   reference: J Garbe, Y Da (2003), Pedigraph user manual Version 1.1\. Department of Animal Science, University of Minnesota.

*   PEDIGREE/DRAW

    *   full name:

    *   version: 6

    *   description: pedigree drawing

    *   authors: Bennett Dyke (email: bdyke@darwin.sfbr.org), Paul Mamelka (contact person, email: paul@darwin.sfbr.org), J. MacCleur, Southwest Foundation for Biomedical Research

    *   web/ftp: http://www.pedigree-draw.com/

    *   ftp://www.sfbr.org/pub/pedmgt/peddrw

    *   source code language:

    *   operating systems: MacOS

    *   executables:PedDraw

    *   reference: Am Soc Hum Genet annual meeting 1987 (Dyke, Mamelka, Am J. Hum Genet, suppl 41, A253 (1987))

    *   price: disk and user's guide: US$45; user's guide: US$20; upgrade for disk: US$15;

*   PEDIGREEQUERY

    *   full name:

    *   version:

    *   description: allows drawing pedigrees with a difficult structure, those containing consanguinity loops, and those individuals with multiple mates or several related families

    *   authors: A Kirichenko

    *   web/ftp: http://mga.bionet.nsc.ru/soft/index.html

    *   http://mga.bionet.nsc.ru/PedigreeQuery/PedigreeQuery.html (in Russian)

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: A Kirichenko (2003), "PedigreeQuery - software for pedigree drawing" (abstract), American Journal of Human Genetics, suppl

*   PEDIGREE-VISUALIZER

    *   full name:

    *   version:

    *   description:

    *   authors: Limsoon Wong (National University of Singapore, Singapore)

    *   web/ftp: http://research.i2r.a-star.edu.sg:8080/kleisli/demos/pedigree/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   PEDJAVA

    *   full name: PedJava

    *   version: 1.1

    *   description: Allows pedigree entry and retrieval from an internet browser into a distant MS ACCESS database. Includes IP access restriction, automatic numbering of families and individuals and database consistency checks.

    *   Authors: Matthias Wjst (email: wjst@gsf.de), Juergen Knauth (email: woodoo@bigfoot.de)

    *   web/ftp: http://cooke.gsf.de/wjst/download.cfm

    *   source code language: JDK

    *   operating systems: no restriction for client, server WIN 95/NT

    *   executables: pedJava, pedJavaServer

    *   reference: in preparation

*   PEDMANAGER

    *   full name:

    *   alias: MAPMAKER/PEDMANAGER

    *   version: 0.9 (1997)

    *   description:

    *   authors: Mary Pat Daly (previously Mary Pat Reeve) (MIT/Whitehead)

    *   web/ftp: http://www-genome.wi.mit.edu/ftp/distribution/software/pedmanager/

    *   source code language: C

    *   operating systems: UNIX

    *   executables: pedprep

    *   reference:

*   PEDPACK (see also PANGAEA)

    *   full name:

    *   version: Version 2.2, July 1996

    *   description: Programs for pedigree analysis, including segregation analysis, gene extinction, and pedigree graphics.

    *   authors: A.W. Thomas (Versions 1.0, 3.0, 3.1)

    *   E.A. Thompson (University of Washington, email: thompson@stat.washington.edu)

    *   C.J. Geyer (University of Minnesota, email: charlie@stat.umn.edu)

    *   web/ftp: http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml

    *   source code language: C

    *   operating systems: UNIX(Compaq-Alpha/..)

    *   executables:

    *   reference: Thomas, IMA J. Math Appl Med Biol, 5, 210-213 (1988)

    *   Thomas, "Pedpack: Users' Manual", Technical report No. 99, Dept Statistics, University of Washington.

    *   Thomas, "Pedpack: Managers' Manual", Technical report No. 100, Dept Statistics, University of Washington.

    *   Geyer, (1988), "Software for calculating gene survival and multigene descent probabilities, and for pedigree manipulation and drawing", Technical report No. 153, Dept Statistics, University of Washington.

*   PEDPEEL

    *   full name:

    *   version:

    *   description: The program prepares pedigree data for calculation of Elston-Stewarts' likelihood function. It finds an optimal way to peel a pedigree and returns text file containing 7 description arrays

    *   authors: Nadezhda Belonogova (email: adg101@mail.ru), Tatiana I. Axenovich

    *   web/ftp: http://mga.bionet.nsc.ru/soft/pedpeel/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Belonogova, Axenovich (2007), "Optimal peeling order for pedigrees with incomplete genotypic information", Computational Biology and Chemistry , 31(3):173-177.

*   PEDPHASE

    *   full name:

    *   version: 2.0

    *   description: inferring haplotypes from genotypes on pedigree data

    *   authors: Jing Li(email: jingli@eecs.cwru.edu)

    *   web/ftp: http://vorlon.case.edu/~jxl175/haplotyping.html

    *   source code language:

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Li, Jiang (2003), "Effecient rule-based haplotyping algorithms for pedigree data", Proceedings of the Seventh International Conference on Research in Computational Molecular Biology , pp.197-206.

    *   Li, Jiang (2004), " An exact solution for finding minimum recombinant haplotype configurations on pedigrees with missing data by integer linear programming", Proceedings of the Seventh International Conference on Research in Computational Molecular Biology , pp.101-110.

*   PEDPLOT

    *   full name:

    *   version:

    *   description: Pedigree Plotting Program for the Pedfile Format

    *   Authors: JE Stajich (email: jason@chg.mc.duke.edu), C Haynes (email: carol@chgd.mc.duke.edu), MA Pericak-Vance (Duke University Medical Center)

    *   web/ftp: http://www.chg.duke.edu/software/pedplot.html

    *   source code language: C++, Postscript

    *   operating systems: UNIX(Sparc-Solaris 2.5/DEC UNIX 4.0/x86-Solaris 2.6)

    *   executables: pedplot

    *   reference: Am Soc Hum Genet annual meeting 1998 (Stajich, Haynes, Pericak-Vance, Am J Hum Genet, suppl, 63,A242).

*   PEDRAW/WPEDRAW

    *   full name: Pedigree Drawing/ Window Pedigree Drawing (MS-Window and X-Window version of PEDRAW)

    *   version: 2.5

    *   description: a pedigree drawing program using LINKAGE data files

    *   authors: Dave Curtis (email: dcurtis@hgmp.mrc.ac.uk)

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/

    *   source code language: C++

    *   operating systems: MS-DOS, MS-Windows, X-window

    *   executables:

    *   reference: Curtis (1990), Annals of Human Genetics, 54:365-367.

*   PEDSCRIPT

    *   full name:

    *   version: 2004 (not actively maintained)

    *   description: PedScript is a tool that allows scripting of simple modifications to pedigree files.

    *   authors:

    *   web/ftp: http://genome.sph.umich.edu/wiki/PedScript

    *   source code language:

    *   operating systems:

    *   executables: anonymize, mendelcheck, nuke, trim, sample

    *   reference:

*   PEDSTATS

    *   full name:

    *   version: 0.6.8

    *   description: PEDSTATS does error checking and data summary of large or small data sets in QTDT, LINKAGE or MENDEL format. Checks for basic formatting errors, disconnected family groups, ancestor-descendant loops and can detect all Mendelian (including X-linked) inheritance errors in any pedigree without loops. Produces text and graphical (PDF) summaries of the family structure, trait and marker information of pedigree data and can break down summaries by sex, relative pair type or family. PEDSTATS also does Hardy-Weinberg testing using either a fast exact or asymptotic test and can summarize information in text or graphical PDF format. Additional features include a number of options for filtering data prior to summary and checks for inappropriate age or covariate values. Lastly, PEDSTATS can identify and trim uninformative individuals from a pedigree and rewrite the reorganized data to a new pedigree file.

    *   authors: Janis Wigginton (email: wiggie@umich.edu), Goncalo Abecasis (email: goncalo@umich.edu)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/Pedstats/

    *   source code language: C,C++

    *   operating systems: Linux, UNIX, MS-Windows

    *   executables: pedstats

    *   reference: Wigginton, Abecasis (2005), "PEDSTATS: descriptive statistics, graphics and quality assessment for gene mapping data", Bioinformatics, 21:3445-3447.

*   PEDSYS

    *   full name:

    *   version: 2.0

    *   description: a full-scale database system developed as a specialized tool for management of genetic, pedigree and demographic data.

    *   authors: Bennett Dyke (email: bdyke@darwin.sfbr.org), Paul Mamelka (email: paul@darwin.sfbr.org),

    *   web/ftp: http://www.sfbr.org/sfbr/public/software/pedsys/pedsys.html

    *   ftp://www.sfbr.org/pub/pedmgt/pedsys

    *   source code language:

    *   operating systems: UNIX(SunOS,Solaris), MS-DOS, MacOS

    *   executables:

    *   reference:

    *   price: disks or tapes and user's manual: US$55; user's manual: US$30; disk for registered users: US$20\. person to contact: Linda Freeman-Shade, Dept Genetics, Southwest Foundation for Biomedical Research, P.O. Box 760549, San Antonio, TX 78245-0549 (email: linda@darwin.sfbr.org)

*   PEDVIZAPI

    *   full name: PEDigree VIsualiZation API

    *   version: 0.15 (November 2007)

    *   description: PEDVIZAPI is a Java Api for the visual analysis of large and complex pedigrees. It provides all the necessary functionality for the interactive exploration of extended genealogies. Available packages are mostly focused on a static representation or cannot be added to an existing application; the goal of this open-source library is to provide an application program interface that enables the efficient construction of visual analytic applications for the analysis of family based data.

    *   authors: Christain Fuchsberger (email: christian.fuchsberger@eurac.edu), Mario Falchi, Lukas Forer, Peter Pramstaller

    *   web/ftp: http://www.pedvizapi.org/

    *   source code language: Java, R

    *   operating systems:

    *   executables:

    *   reference: Fuchsberger, Falchi, Forer, Pramstaller (2007), "PedVizApi: A Java API for the interactive, visual analysis of extended pedigrees", Bioinformatics, 24(2):279-281.

*   PEER

    *   full name:

    *   version:

    *   description: PEER is a collection of Bayesian approaches to infer hidden determinants and their effects from gene expression profiles using factor analysis methods. It can be used to detect eQTL.

    *   authors:

    *   web/ftp: http://www.sanger.ac.uk/resources/software/peer/

    *   source code language: C++

    *   operating systems:

    *   executables:

    *   reference: Stegle, Parts, Durbin, Winn (2010), "A Bayesian framework to account for complex non-genetic factors in gene expression levels greatly increases power in eQTL studies", PLoS Computational Biology, 6:e1000770.

*   PELICAN

    *   full name: Pedigree Editor for LInkage Computer ANalysis

    *   version:

    *   description: It is a utility for graphically editing the pedigree data files used by programs such as FASTLINK, VITESSE, GENEHUNTER and MERLIN.

    *   authors: Frank Dudbridge (MRC, email: frank.dudbridge@mrc-bsu.cam.ac.uk)

    *   web/ftp: http://www.mrc-bsu.cam.ac.uk/personal/frank/software/pelican/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: F Dudbridge, T Carver, GW Williams (2004), "Pelican: pedigree editor for linkage computer analysis", Bioinformatics, 20(14):2327-2328.

*   PHASE

    *   full name:

    *   version: 2.0 (Sept 2003)

    *   description: This program implements a new statistical method for reconstructing haplotypes from population genotype data. | new for v2: (1) the introduction of a new computational approach, resulting in much faster haplotype resolution. (2) the introduction of a new model that allows for recombination and decay of Linkage Disequilibrium (LD) with distance, which results in more accurate haplotype estimates. This model also allows the user to estimate recombination rates, and identify recombination hotspots from population genotype data. (3) the facility to perform a test for haplotype frequency differences between cases and controls. (4) more extensive output summarising the results.

    *   Authors: Matthew Stephens (email: stephens@stat.washington.edu)

    *   web/ftp: http://www.stat.washington.edu/stephens/software.html

    *   previous site: http://www.stats.ox.ac.uk/mathgen/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Stephens, Smith, Donnelly (2001), AJHG, 68:978-989.

*   PIAGE

    *   full name: Power of Indirect Association Studies of Gene-Environment Interactions

    *   version: 1.0 (March 2008)

    *   description: The program PIAGE performs estimation of power and sample sizes required to detect genetic and environmental main, as well as gene-environment interaction (GxE) effects in indirect matched case-control studies (1:1 matching). When the hypothesis of GxE is tested, power/sample size will be estimated for the detection of GxE, as well as for the detection of genetic and environmental marginal effects. Furthermore, power estimation is implemented for the joint test of genetic marginal and GxE effects (Kraft P et al., 2007). Power and sample size estimations are based on Gauderman's (2002) asymptotic approach for power and sample size estimations in direct studies of GxE. Hardy-Weinberg equilibrium and independence of genotypes and environmental exposures in the population are assumed. The estimates are based on genotypic codes (G=1 (G=0) for individuals who carry a (non-) risk genotype), which depend on the mode of inheritance (dominant, recessive, or multiplicative). A conditional logistic regression approach is used, which employs a likelihood-ratio test with respect to a biallelic candidate SNP, a binary environmental factor (E=1 (E=0) in (un)exposed individuals), and the interaction between these components.

    *   authors: Rebecca Hein (email:r.hein@dkfz-heidelberg.de), Lars Beckmann (email:l.beckmann@dkfz-heidelberg.de)

    *   web/ftp: http://www.dkfz.de/en/epidemiologie-krebserkrankungen/software/software.html

    *   source code language: R

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Hein, Beckmann, Chang-Claude (2008), "Sample size requirements for indirect association studies of gene-environment interactions (GxE)", Genetic Epidmiology, to appear.

    *   Gauderman (2002), "Sample size requirements for matched case-control studies of gene-environment interaction", Statistics in Medicine, 21:35-50.

    *   Kraft, Yen, Stram, Morrison, Gauderman (2007), "Exploiting gene-environment interaction to detect genetic associations", Human Heredity, 63:1141-1119.

*   PLABQTL

    *   full name: PLAnt Breeding QTL analysis

    *   version:

    *   description:

    *   authors: HF Utz (email: utzf@uni-hohenheim.de)

    *   web/ftp: https://www.uni-hohenheim.de/plantbreeding/software/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Utz, Melchinger (1996), "PLABQTL: A program for composite interval mapping of QTL", JAG

*   PLABSIM

    *   full name: PLAnt Breeding SIMulation

    *   version: 1.0.1 (June 2005)

    *   description:

    *   authors: Matthias Frisch (email: frisch@uni-hohenheim.de)

    *   web/ftp: http://www.uni-hohenheim.de/~frisch/software.html

    *   source code language: C++

    *   operating systems: UNIX(AIX), Linux, MS-Windows(NT)

    *   executables:

    *   reference: Frisch, Bohn, Melchinger (2000), "PLABSIM: software for simulation of marker-assisted backcrossing", Journal of Heredity, 91:86-87.

*   PLAYPUS

    *   full name:

    *   version:

    *   description: an integrated variant caller

    *   authors: Gerton Lunter, Martin Goodson (Wellcome Trust Centre of Human Genetics, Oxford Univ)

    *   web/ftp: http://www.well.ox.ac.uk/platypus

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   PL-EM

    *   full name: Partition-Ligation EM algorithm for haplotype inference with single nucleotide polymorphisms

    *   version:

    *   description:

    *   authors: Jun Liu, Steve Qin, Tianhua Niu (Harvard Univ)

    *   web/ftp: http://www.people.fas.harvard.edu/~junliu/plem/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Qin, Niu, Liu (2002), "Partition-ligation-expectation-maximization algorithm for haplotype inference with single-nucleotide polymorphisms", American Journal of Human Genetics, (letter), 71(5):1242-1247-?

*   PLINK (see also PLINK/SEQ)

    *   full name:

    *   version: 1.03

    *   description: a whole-genome association analysis toolset

    *   authors: Shaun Purcell

    *   web/ftp: http://pngu.mgh.harvard.edu/purcell/plink/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Purcell, Neale, Todd-Brown, Thomas, Ferreira, Bender, Maller, de Bakker, Daly, Sham (2007), "PLINK: a toolset for whole-genome association and population-based linkage analysis", American Journal of Human Genetics, 81(3):559-575

*   PLINK/SEQ (see also PLINK)

    *   full name:

    *   version:

    *   description: PLINK/SEQ is an open-source C/C++ library for working with human genetic variation data. The specific focus is to provide a platform for analytic tool development for variation data from large-scale resequencing projects, particularly whole-exome and whole-genome studies. However, the library could in principle be applied to other types of genetic studies, including whole-genome association studies of common SNPs.

    *   authors:

    *   web/ftp: http://atgu.mgh.harvard.edu/plinkseq/

    *   source code language: C/C++, R

    *   operating systems: MacOS, Linux

    *   executables:

    *   reference:

*   POINTER

    *   full name:

    *   version:

    *   description: for complex segregation analysis with the mixed model (major locus and polygenes).

    *   authors: NE Morton (University of Southampton, UK), D.C. Rao, J-M. Lalouel

    *   web/ftp: http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/pointer

    *   source code language: SUN FORTRAN (especially, the command "fsplit" is needed)

    *   operating systems: UNIX (SunOS/..)

    *   executables:

    *   reference: Morton, Rao, Lalouel, "Methods in Genetic Epidemiology" (Karger, 1983).

*   POLYMORPHISM

    *   full name:

    *   version: 1.4 (June 1999)

    *   description: calculating the heterozygosity, PIC, and LIC values for polymorphic markers

    *   Authors: Tianhua Niu, Sc.D. (Harvard Medical School, email: tniu@hohp.harvard.edu)

    *   web/ftp:

    *   source code language: C

    *   operating systems: UNIX

    *   executables: powerlic

    *   reference: Guo, Elston (1999), "Linkage information content of polymorphic genetic markers", Human Heredity, 49:112-118.

    *   Niu, Struk, Lindpaintner (2001), Human Heredity, 52(2):102-109.

*   POLYMUTT

    *   full name: POLYmorphism and de novo MUTaTion call in families with sequencing data

    *   version:

    *   description: The program polymutt implemented a likelihood-based framework for calling single nucleotide variants and detecting de novo point mutation events in families for next-generation sequencing data. The program takes as input genotype likelihood format (GLF) files which can be generated following the Creation of GLF files instruction and outputs the result in the [VCF] format. The variant calling and de novo mutation detection are modelled jointly within families and can handle both nuclear and extended pedigrees without consanguinity loops. The input is a set of GLF files for each of family members and the relationships are specified through the .ped file.

    *   authors: B Li, G Abecasis (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/Polymutt:_a_tool_for_calling_polymorphism_and_de_novo_mutations_in_families_for_sequencing_data

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   POLYPHEN

    *   full name: POLYmorphism PHENotyping

    *   version:

    *   description: PolyPhen-2 is a tool which predicts possible impact of an amino acid substitution on the structure and function of a human protein using straightforward physical and comparative considerations. It can be used in prioritize functional SNPs in a genetic study.

    *   authors:

    *   web/ftp: http://genetics.bwh.harvard.edu/pph2/ (POLYPHEN-2)

    *   http://genetics.bwh.harvard.edu/pph/ (POLYPHEN-1)

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Sunyaev, Eisenhaber, Rodchenkov, Eisenhaber, Tumanyan, Kuznetsov (1999), "PSIC: profile extraction from sequence alignments with position-specific counts of independent observations", Protein Engineering, 12:387-394.

    *   Ramensky, Bork, Sunyaev (2002), "Human non-synonymous SNPs: server and survey", Nucleic Acids Research, 30:3894-3900.

    *   Adzhubei, Schmidt, Peshkin, Ramensky, Gerasimova, Bork, Kondrashov, Sunyaev (2010), "A method and server for predicting damaging missense mutations", Nature Methods, 7:248-249.

*   POOLSCORE

    *   full name:

    *   version:

    *   description: a program for analysis of case-control genetic association studies using allele frequency measurements on DNA pools

    *   authors: Paul McKeiger (email: paul.mckeigue@ucd.ie) (University College Dublin, Ireland)

    *   web/ftp: http://www.homepages.ed.ac.uk/pmckeigu/pooling/poolscore.htm

    *   source code language: R

    *   operating systems:

    *   executables: poolscore.R

    *   reference:

*   POOL_STR

    *   full name: POOLing alleles of Short Tandem Repeat markers

    *   version: 1.1

    *   description:

    *   authors: Yurii Aulchenko, Aida Bertoli-Avella, Cornelia van Duijn (Erasmus University Medical School, The Netherlands)

    *   web/ftp: http://www.geneticepi.com/Research/software/software.html

    *   http://www.geneticepi.com/Research/software/pool_str-1.1.tar.gz

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Aulchenko, Beroli-Avella, van Duijn (2005), "A method for pooling alleles from different genotyping experiments", Annals of Human Genetics, 69(2):233-238.

*   POPDIST

    *   full name:

    *   version: 1.1.2

    *   description: POPDIST calculates a number of different genetic identities, phylogeny reconstructing measures, and distance reconstructing measures

    *   authors: Bernt Guldbrandtsen (email: bg@genetics.agrsci.dk)

    *   web/ftp: http://genetics.agrsci.dk/~bg/popgen/

    *   source code language:

    *   operating systems: UNIX (AIX/IRIX/DECalpha), Linux, MacOS, MS-DOS, MS-Windows(95)

    *   executables:

    *   reference: Guldbrandtsen, Tokiuk, Loeschcke (2000), "POPDIST, version 1.1.1: a program to calculate population genetic distance and identity measures", Journal of Heredity, 91:178-179.

*   POPGEN

    *   full name:

    *   version:

    *   description: POPGEN is an R package that specifically focuses on statistical and population genetics methods. The motivation behind the package is to produce an easy to use interface to many of the commonly used methods and models used in statistical and population genetics and an alternative interface for some of the methodology produced by our group.

    *   authors: Jonathan Marchini (email: marchini@stats.ox.ac.uk)

    *   web/ftp: http://www.stats.ox.ac.uk/mathgen/software.html

    *   http://www.stats.ox.ac.uk/~marchini/software.html

    *   http://cran.r-project.org/web/packages/popgen/index.html (removed from CRAN)

    *   source code language: R

    *   operating systems:

    *   executables: popdiv, ps, nps, treesim, simMD

    *   reference:

*   POWER

    *   full name:

    *   version: 3.0

    *   description: windows-based program for computation of sample size and power for binary outcome studies (case-control and cohort studies) based on a logistic-like regression model with one covariate or two covariates (e.g., gene-exposure interactions).

    *   authors:

    *   web/ftp: http://dceg.cancer.gov/POWER/

    *   source code language:

    *   operating systems: MS-Window

    *   executables:

    *   reference: Garcia-Closas, Lubin (1999), "Power and sample size calculations in case-control studies of gene-environmental interactions: Comments on different approaches", American Journal of Epidemiology, 149:689-693.

*   POWERMARKER

    *   full name:

    *   version: 3.25 (February 2006)

    *   description: PowerMarker is a comprehensive set of statistical methods for genetic marker data analysis, designed especially for SSR/SNP data analysis. PowerMarker builds a powerful user interface around both new and traditional statistical methods for population genetic analysis. See analysis to check out the versatility of PowerMarker. PowerMarker is also a 2D Viewer - which was used intensively for visualizing linkage disequilibria results.

    *   authors: Jack Liu

    *   web/ftp: http://statgen.ncsu.edu/powermarker/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Liu, Muse (2005), "PowerMarker: an integrated analysis environment for genetic marker analysis", Bioinformatics, 21(9):2128-2129.

*   POWERTRIM

    *   full name:

    *   version:

    *   description: automate the decision to remove objects from a pedigree with a minimum loss information

    *   authors: Tricia Thornton, Jonathan Haines (Venderbilt University Medical Center)

    *   web/ftp: http://phg.mc.vanderbilt.edu/content/powertrim

    *   source code language: Perl

    *   operating systems:

    *   executables:

    *   reference: Thornton, Haines (2003), "PowerTrim: an automated decision support algorithm for preprocessing family-based genetic data", American Journal of Human Genetics, 72(5):1280-1281

*   POWQ

    *   full name:

    *   version:

    *   description: A user-friendly, graphical package for power evaluation and enhancement planning through variance component linkage analysis in a multipoint framework.

    *   authors: Mario Falchi (email: mario.falchi@kcl.ac.uk) Cesare Cappio Borlino

    *   web/ftp: http://www.twin-research.ac.uk/WebPowQ/PowQ.htm

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Falchi and Borlino (2006), "PowQ: a user-friendly package for the design of variance component multipoint linkage analysis studies", Bioinformatics, 22(11):1404-1405.

*   POWTEST

    *   full name:

    *   version:

    *   description: Excel spreadsheet to calculate power of affected sibpairs and TDT analyses

    *   authors: Dave Curtis

    *   web/ftp: http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   PREPLINK

    *   full name: MAKE PEDfiles

    *   alias: LINKAGE/PREPLINK

    *   version:

    *   description: part of the LINKAGE auxiliary program

    *   authors: Peter Cartwright (University of Utah, now Cimarron Software pc@cimsoft.com)

    *   web: http://lab.rockefeller.edu/ott/programs or

    *   source code language: C and Pascal

    *   operating systems: UNIX, VMS, MS-DOS, OS2

    *   executables:

    *   reference:

*   PREST

    *   full name: Pedigree RElationship Statistical Test

    *   version: 3.0.2 (February 2005)

    *   description: PREST is a program that detects pedigree errors in general outbred pedigrees by use of genome-screen data. When a potential pedigree error is detected, our companion program, ALTERTEST, determines which relationships are compatible with the observed genotype data. Both programs are freely available on the web.

    *   authors: Lei Sun, Kenneth Wilder, Mary Sara McPeek (University of Chicago)

    *   web/ftp: http://fisher.utstat.toronto.edu/sun/Software/Prest or previously, http://galton.uchicago.edu/~mcpeek/software/prest

    *   source code language: C

    *   operating systems: Linux, UNIX(Solaris/..)

    *   executables: prest.solaris, prest.linux, altertest.solaris, altertest.linux

    *   reference: McPeek, Sun (2000), "Statistical tests for detection of misspecified relationships by use of genome-screen data", American Journal of Human Genetics, 66(3):1076-1094

*   PRESTO

    *   full name:

    *   version: 1.0.1 (October 2007)

    *   description: PRESTO performs permutation testing and computes empirical distributions of order statistics for one and two stage association studies with stratified or unstratified data.

    *   authors: Brian L Browning (email: b.browning@auckland.ac.nz)

    *   web/ftp: http://faculty.washington.edu/browning/presto/presto.html

    *   source code language: Java

    *   operating systems: MS-Windows, UNIX (Solaris/..), Linux, MacOS

    *   executables:

    *   reference: Browning (2008), "PRESTO: Rapid calculation of order statistic distributions and multiple-testing adjusted P-values via permutation for one and two-stage genetic association studies", BMC Bioinformatics, 9:309.

*   PROBMAX

    *   full name:

    *   version:

    *   description: assigning unknown parentage in pedigree analysis from known genotypic pools of parents and progeny

    *   authors: Roy Danzmann (email: rdanzman@uoguelph.ca)

    *   web/ftp: http://www.uoguelph.ca/~rdanzman/software/PROBMAX/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Danzmann (1997), "PROBMAX: a computer program for assigning unknown parentage in pedigree analysis from known genotypic pools of parents and progeny", Journal of Heredity, 88:333.

*   PROC QTL

    *   full name:

    *   version: 1.0 (January 2009)

    *   description: PROC QTL is a user defined SAS procedure for mapping quantitative trait loci (QTL). Since this procedure is not a built-in SAS procedure, users need to obtain a copy of the executable file of PROC QTL and install the software in their personal computers before PROC QTL can be executed. Of course, users need a regular SAS license prior to the installation of PROC QTL. Once PROC QTL is installed, it can be called just like any other SAS procedures. Users will not notice the differences between this customized procedure and other built-in SAS procedures.

    *   authors: Zhiqiu Hu (email: zhiqiu.hu@ucr.edu), Shizhong Xu (shizhong.xu@ucr.edu)

    *   web/ftp: http://www.statgen.ucr.edu/software.html

    *   source code language: C++

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Hu and Xu (2009), "PROC QTL - A SAS procedure for mapping Quantitative Trait Loci", Plant and Animal Genomes XVII Conference. January 10-14, 2009, San Diego, California (http://www.intl-pag.org/17/abstracts/C02_PAGXVII_854.html).

*   PROFILER

    *   full name:

    *   version:

    *   description: A flexible tool to generate the probability distribution of joint multilocus genotypes defined by sets of individuals within the pedigree and sets of markers within the framework map.

    *   authors: Jeff O'Connell

    *   web/ftp: http://www.molecular-haplotype.org/profiler/profiler_intro.htm

    *   source code language:

    *   operating systems: Linux, UNIX(Solaris)

    *   executables:

    *   reference:

*   PROGENY

    *   full name: Progeny Software, LLC

    *   version: 6.8 (December 2006)

    *   description: Fully customizable, multi-user relational database with an integrated pedigree drawing component to manage genetic and pedigree data in one database. Manage Pedigrees, Individuals, SNPs, STRs, Samples, Plates, Genotypes and exports to multiple analysis platforms.

    *   authors: David Deram (Progeny Software, LLC)

    *   web/ftp: http://www.progenygenetics.com/

    *   source code language: C++ and Active X Controls

    *   operating systems: MS-Windows (95/98/2000/ME/NT/XP)

    *   executables:

    *   reference:

*   PRT

    *   full name: Pedigree Reconstruction Tools

    *   version: 1.2

    *   description: Partition of single generation into sibling groups

    *   authors: Anthony Almudevar (Dept Biostatistics and Computational Biology, University of Rochester Medical Center, email: anthony_almudevar@urmc.rochester.edu)

    *   web/ftp: http://www.urmc.rochester.edu/smd/biostat/Projects/Help/PC/Software_Listings.htm

    *   source code language:

    *   operating systems: MS-Windows

    *   executables: msg_proj

    *   reference: A Almudevar, C Field (1999), "Estimation of single generation sibling relationships based on DNA markers", J. Agricultural, Biological and Environmental Statistics, 4(2):136-165.

*   PSAT

    *   full name: Population Stratification Association Test

    *   version:

    *   description:

    *   authors: G Kimmel, MI Jordan, E Halperin, R Shamir, RM Karp

    *   web/ftp: http://acgt.cs.tau.ac.il/psat/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Kimmel, Jordan, Halperin, Shamir, Karp (2007), "A randomization test for controlling population stratification in whole-genome association studies", American Journal of Human Genetics, 81(5):895-905

*   PSEUDO

    *   full name:

    *   version: 0.3.5

    *   description: Pseudo estimates genomewide empirical p-values for Kong and Cox tests of linkage using the replicate pool method, which for many data sets, improves upon the computational efficiency of conventional gene-dropping methods by several orders of magnitude. This allows Pseudo to handle data sets with large families and makes it particularly applicable to those situations where p-value estimation by standard methods is computationally prohibitive. Pseudo also estimates variance for reported p-values, produces graphical and text summaries of results, and is able to assess significance of multiple correlated outcomes. Pseudo is designed to work with the Merlin package and includes utilities for generating input files from standard Merlin output.

    *   authors: Janis Wigginton (email:wiggie@umich.edu) and Goncalo Abecasis

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/pseudo/download

    *   source code language: C,C++

    *   operating systems: Linux, UNIX, MacOS, MS-Windows

    *   executables: pseudo, scan

    *   reference: Wigginton, Abecasis (2006), "An evaluation of the replicate pool method: quick estimation of genome-wide linkage peak p-values", Genetic Epidemiology, 30(4):320-332.

*   PSEUDOMARKER

    *   full name:

    *   version: 1.0.6b (November 2011)

    *   description: PSEUDOMARKER is a linkage analysis software for joint linkage and/or linkage disequilibrium analysis. PSEUDOMARKER can analyze different data structures jointly such as cases-controls, trios, sib-pairs, sib-ships, and extended families.

    *   Authors: Tero Hiekkalinna (email: tero.hiekkalinna@helsinki.fi), Joseph D Terwilliger, Petri Norrgrann

    *   web/ftp: http://www.helsinki.fi/~tsjuntun/pseudomarker/

    *   source code language: C/C++

    *   operating systems: Linux

    *   executables: pseudomarker, ilinkpseudo, makepedpseudo, mlinkpseudo, linkmappseudo, unknownpseudo

    *   reference: Goring, Terwilliger (2000), "Linkage analysis in the presence of errors IV: joint pseudomarker analysis of linkage and/or linkage disequilibrium on a mixture of pedigrees and singletons when the mode of inheritance cannot be accurately specified", American Journal of Human Genetics, 66(4):1310-1327

    *   Hiekkalinna, Schäffer, Lambert, Norrgrann, Göring, Terwilliger (2011), "PSEUDOMARKER: a powerful program for joint linkage and/or linkage disequilibrium analysis on mixtures of singletons and related individuals", Human Heredity, 71:256-266

*   PSEUDOMARKER.M

    *   full name:

    *   version: 2.03

    *   description: PSEUDOMARKER is a set of programs written in MATLAB for the analysis of QTL data from inbred line crosses.

    *   authors: Saunak Sen, Gary Churhcill (Jackson Lab, email: pseudomarker@jax.org)

    *   web/ftp: http://www.jax.org/staff/churchill/labsite/software/pseudomarker/index.html

    *   source code language: MATLAB

    *   operating systems:

    *   executables:

    *   reference: Sen, Churchill (2001), "A statistical framework for quantitative trait mapping", Genetics, 159:371-387.

*   PYPOP

    *   full name: PYthon for POPulation genetics

    *   version: beta 0.7.0 (September 2008)

    *   description:

    *   authors: Alex Lancaster, Mark P Nelson, Richard M Single, Diogo Meyer, Glenys Thomson

    *   web/ftp: http://www.pypop.org/

    *   source code language: Python

    *   operating systems: MS-Windows(98/2000/XP), Linux

    *   executables:

    *   reference: Lancaster, Nelson, Single, Meyer, Thomson (2003), "PyPop: a software framework for population genomics: analyzing large-scale multi-locus genotype data", Proceedings of the Pacific Symposium on Biocomputing, 8:514-525.

    *   Lancaster, Single, Solberg, Nelson, Thomson (2007), "PyPop update - a software pipeline for large-scale multilocus population genomics", Tissue Antigens, 69(s1):192-197.

*   QGENE

    *   full name:

    *   version: 4.0.2

    *   description: A free, computationally effecient, easy-to-use Java program for comparative analyses of QTL mapping data and population simulation.

    *   authors: Roby Joehanes, Zhigang Guo, Prashanth Boddhireddy, J Clare Nelson (Kansas State University)

    *   web/ftp: http://www.qgene.org/

    *   http://coding.plantpath.ksu.edu/qgene/

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Nelson (1997), "QGENE: software for marker-based genomic analysis and breeding", Molecular Breeding, 3:239-245\. [[abstract](http://dx.doi.org/10.1023/A:1009604312050)]

*   QMSIM

    *   full name: Qtl and Marker SIMulator

    *   version: 1.0

    *   description: QMSim was designed to simulate a wide range of genetic architectures and population structures in livestock. Large scale genotyping data and complex pedigrees can be efficiently simulated. QMSim is a family based simulator, which can also take into account predefined evolutionary features, such as LD, mutation, bottlenecks and expansions. The simulation is basically carried out in two steps: In the first step, a historical population is simulated to establish mutation-drift equilibrium and, in the second step, recent population structures are generated, which can be complex. QMSim allows for a wide range of parameters to be incorporated in the simulation models in order to produce appropriate simulated data.

    *   authors: Mehdi Sargolzaei (email: msargol@uoguelph.ca), Flavio S Schenkel

    *   web/ftp: http://www.aps.uoguelph.ca/~msargol/qmsim/

    *   source code language: C++

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Sargolzaei, Schenkel (2009), "QMSim: a large-scale genome simulator for livestock", Bioinformatics, 25:680-681.

*   QTDT

    *   full name: Quantitative (Trait) Transmission/Disequilibrium Test

    *   version: 2.5.1 (January 2005)

    *   description: Performs linkage disequilibrium (TDT) and association analysis for quantitative traits. Includes support for the methods of Abecasis et al. (2000), Fulker et al. (1999), Monks et al. (1998), Allison (TDTQ5, 1997) and Rabinowitz (1997). Supports families of any size, with or without parental information. Includes simple variance components modelling. Interfaces with SimWalk2 for IBD estimation.

    *   Authors: Goncalo Abecasis (goncalo@well.ox.ac.uk)

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/QTDT/

    *   source code language: C++

    *   operating systems: UNIX(Solaris), Linux, MS-Windows

    *   executables: qtdt, prelude, finale, pedstats

    *   reference: Abecasis, Cardon and Cookson (2000), "A general test of association for quantitative traits in nuclear families", American Journal of Human Genetics, 66(1):279-292\. [ html]

*   QTL-ALL

    *   full name: QTL Analysis and Linkage Library

    *   version:

    *   description: QTL-ALL is a software package designed to make as many as possible of the new statistics (e.g. score statistics) widely available. The software consists of a MEGA2-like interface for data analysis preparation and a library of R routines that computes linkage statistics. QTL-ALL reads in input data, creates re-formatted output data files, calls external IBD-generation software such as MERLIN or SIMWALK2, then computes statistics using our R library, and finally produces tables and plots of statistics and p-values. This entire sequence is highly automated, requiring minimal user-intervention. The initial release of the software computes a number of newer QTL-mapping statistics, including several score statistic variants, and can handle nuclear family data, including specialty designs such as discordant and concordant (affected) pairs.

    *   authors: Nandita Mukhopadhyay, S Bhattacharjee, CL Kuo, Dan Weeks, E Feingold

    *   web/ftp: http://watson.hgen.pitt.edu/register/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting 2006 Mukhopadhyay, Bhattacharjee, Kuo, Weeks, Feingold, Am J Hum Genet, suppl. ?, ? (1995)). http://www.ashg.org/genetics/abstracts/abs06/f1567.htm

*   QTL CAFE

    *   full name:

    *   version: May 1998

    *   description: providing a user freiendly way to perform QTL analysis. The software currently allows 3 types of QTL analysis: (1) single marker ANOVA. (2) marker regression. (3) interval mapping by regression.

    *   authors:

    *   web/ftp: http://www.biosciences-labs.bham.ac.uk/Kearsey/

    *   source code language:

    *   operating systems: Java

    *   executables:

    *   reference:

*   QTL Cartographer

    *   full name:

    *   version: Window version 2.5 (September 2006)

    *   description: A program to map quantitative traits sing a map of molecular markers.

    *   Authors: Christopher J. Basten, Bruce S. Weir, Zhao-Bang Zeng, Shengchu Wang (Dept Statistics, North Carolina State University);

    *   web/ftp: http://statgen.ncsu.edu/qtlcart/cartographer.html

    *   ftp://statgen.ncsu.edu/pub/qtlcart/

    *   source code language:

    *   operating systems: UNIX, MS-Windows, MacOS

    *   executables: rmap, rqtl, rcross, qstats, lrmapqtl, srmapqtl, zmapqtl, prune, preplot, eqtl, qtlcart

    *   reference: Basten,Weir,Zeng (1994), "Zmap-a QTL cartographer", in Proceedings of the 5th World Congress on Genetics Applied to Livestock Production: Computing Strategies and Software, vol 22, eds. Smith, Gavora, Chesnais, Fairfull, Gibson, Kennedy, Burnside.

    *   online manual

*   QTL EXPRESS

    *   full name:

    *   version: August 2005

    *   description: A web-based user-friendly package to map Quantitative Trait Loci in outbred populations. Population structures catered for are line crosses, halfsib families, nuclear families and sibpairs. Permutation tests to determine empirical significance levels and bootstrapping to estimate empirical confidence intervals of QTL locations are optional. Fixed effects/covariates can be fitted and models may include single or multiple QTL. Results are presented in tabular and graphical format.

    *   authors: George Seaton (email: george.seaton@ed.ac.uk), Chris Haley, Sara Knott, Xijiang Yu, Peter Visscher (Institute of Cell, Animal and Population Biology, Univ. Edingbergh)

    *   web/ftp: http://qtl.cap.ed.ac.uk/

    *   source code language: Java

    *   operating systems: web-based

    *   executables:

    *   reference: G Seaton, CS Haley, SA Knott, M Kearsey, PM Visscher (2002), "QTL Express: mapping quantitative trait loci in simple and complex pedigrees", Bioinformatics, 18:339-340

*   QTLNETWORK

    *   full name:

    *   version: 2.0

    *   description: QTLNetwork is a software package for mapping and visualizing the genetic architecture underlying complex traits for experimental populations derived from a cross between two inbred lines.

    *   authors: Jian Yang, Chengcheng Hu, Xiuzi Ye, Jun Zhu (Zhejiang University)

    *   web/ftp: http://ibi.zju.edu.cn/software/qtlnetwork

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Yang, Hu, Hu, Yu, Xia, Ye, Zhu (2008), "QTLNetwork: mapping and visualizing genetic architecture of complex traits in experimental populations", Bioinformatics. 24(5):721-723.

*   QUANTO

    *   full name:

    *   version: 1.2.1 (January 2007)

    *   description: Quanto is a program that computes sample size or power for association studies of genes, environmental factors, gene-environment interaction, or gene-gene interaction. Available study designs for a disease (binary) outcome include the unmatched case-control, matched case-control, case-sibling, case-parent, and case-only designs. Study designs for a quantitative tra it include independent individuals and case parent designs. Quanto is a 32-bit Windows application requiring Windows 95, 98, NT, 2000, ME or XP to run. The graphical user interface allows th e user to easily change the model and view the results without having to edit an input file and rerun the program for every model. The results of a session are stored to a log file. This log can be printed or saved to a file for reviewing at a later date. An option is included to create a text file of the log that can be imported into other documents.

    *   authors: Jim Gauderman (email: jimg@usc.edu), John Morrison (email: jmorr@usc.edu)

    *   web/ftp: http://hydra.usc.edu/GxE

    *   source code language: C++

    *   operating systems: Window(98/NT/2000/..)

    *   executables: quanto.exe

    *   reference: Gauderman (2002), "Sample size requirements for matched case-control studies of gene-environment interaction", Statistics in Medicine, 21:35-50

    *   Gauderman (2002), "Sample size requirements for association studies of gene-gene interaction", American Journal of Epidemiology, 155:478-484

    *   WJ Gauderman (2003), "Candidate gene association studies for a quantitative trait, using parent-offspring trios", Genetic Epidemiology, 25:327-338

*   QU-GENE

    *   full name: QUantitative GENEtics

    *   version: v1 (1998)

    *   description: A package for quantitative analysis of genetic models

    *   Authors: D. Podlich (email: d.podlich@mailbox.uq.edu.au) and M. Cooper (email: m.cooper@mailbox.uq.edu.au)

    *   web/ftp: http://pig.ag.uq.edu.au/qu-gene/ (not available)

    *   source code language: Fortran

    *   operating systems: MS-Windows

    *   executables: qugene.exe

    *   reference: Podlich, Cooper (1998), "QU-GENE: a simulation platform for quantitative analysis of genetic models", Bioinformatics. 14:632-653. Online manual: http://pig.ag.uq.edu.au/qu-gene/manual.htm

*   QUTIE (see also CCRAVAT)

    *   full name: Case-Control RAre Variant Analysis Tool

    *   version:

    *   description: Enabling the analysis of rare variants in large-scale case control and quantitative trait association studies. CCRaVAT (Case-Control Rare Variant Analysis Tool) and QuTie (Quantitative Trait) are software packages that enable efficient large-scale analysis of rare variants across specific regions or genome-wide. These programs implement a rare variant super-locus or collapsing method that investigates the accumulation of rare variant alleles in either a case-control or quantitative trait study design.

    *   authors: Eleftheria Zeggini's group (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/rarevariant/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   RC-TDT

    *   full name: Reconstruction-Combined Transmission Disequilibrium Test

    *   version: 1.1d (Jul 2006)

    *   description: A family-based association method that allows testing for linkage in the presence of linkage disequilibrium between an autosomal marker and a disease even if there is only incomplete parental-marker information. Recently, Horvath et al. (2000) described a similar procedure (XRC-TDT) for X-linked markers. The distribution contains SAS macros that calculate the RC-TDT and XRC-TDT test statistics, as well as their respective exact P values.

    *   authors: Michael Knapp (Univ Bonn)

    *   web/ftp: http://www.uni-bonn.de/~umt70e/soft.htm

    *   source code language: SAS macro

    *   operating systems:

    *   executables:

    *   reference: Knapp (1999), "The transmission/disequilibrium test and parental-genotype reconstruction: the reconstruction-combined transmission/disequilibrium test", American Journal of Human Genetics, 64(3):861-870

    *   Knapp (1999b), "Using exact P values to compare the power between the reconstruction-combined transmission/disequilibrium test and the sib transmission/disequilibrium test" (letter), American Journal of Human Genetics, 65:1208-1210\. [ html]

    *   Horvath, Laird, Knapp (2000), "The transmission/disequilibrium test and parental-genotype reconstruction for X-chromosomal markers", American Journal of Human Genetics, 66(3):1161-1167\. [ html]

*   REAPER

    *   full name:

    *   version: 1.0

    *   description: tag SNP selelciton tool according to r2-bins method. It is specifically designed for full genome scale deterministic tagging.

    *   authors: Reedik Magi, Lauris Kaplinski, Maido Remm (Univ of Tartu, Estonia)

    *   web/ftp: http://bioinfo.ebc.ee/download/

    *   source code language:

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Magi, Kaplinski, Remm (2006), "The whole genome TagSNP selection and transferability among HapMap populations", Pacific Symposium on Biocomputing, 11:535-543.

*   RECORD

    *   full name: REcombination Counting and ORDering

    *   version: 2005 and 2006

    *   description: RECORD can be used for the ordering of loci on genetic linkage maps. The method is based on minimisation of the total number of recombination events. Since the criterion does not require intensive calculations, the algorithm rapidly produces an optimal ordering as well as a series of near-optimal ones. The latter provides insight into the local certainty of ordering along the map. RECORD can deal with the following types of mapping populations: BC1, F2, F3, RILs (in fact any generation obtained by repeated selfing of a hybrid between homozygous parents). Data files are .LOC JoinMap format. Mapping populations from non-inbreds should be split into BC1 or HAP data that represent the maternal and paternal gametes.

    *   authors: Piet Stam, Herman van Eck (Wageningen University, Laboratory for Plant Breeding, email: herman.vaneck@wur.nl)

    *   web/ftp: http://www.plantbreeding.wur.nl/UK/software_record.html

    *   source code language: C++

    *   operating systems: MS-Windows (Win98, XP) and MS-DOS

    *   executables: record_win.exe, ord.exe

    *   reference: Van Os, Stam, Visser, Van Eck (2005), "RECORD: A novel method for ordering loci on a genetic linkage map", Theoretical and Applied Genetics, 112(1):30-40

    *   Van Os, Andrzejewski, Bakker, Barrena, Bryan, Caromel, Ghareeb, Isidore, de Jong, van Koert, Lefebvre, Milbourne, Ritter, van der Voort, Rousselle-Bourgeois, van Vliet, Waugh, Visse, Bakke, Van Eck (2006), "Construction of a 10,000-marker ultradense genetic recombination map of potato: Providing a framework for accelerated gene isolation and a genomewide physical map", Genetics, 173(2):1075-1087

*   RELATIVE

    *   full name:

    *   version: 1.10 (dec 97)

    *   description: software for relationship estimation, in particular between putative sibs when parents are untyped

    *   authors: Harald Goring <hgoring@sfbrgenetics.org>

    *   web/ftp: ftp://linkage.rockefeller.edu/software/relative

    *   source code language: ANSI C

    *   operating systems: MS-DOS, MS-Windows, UNIX

    *   executables: relative

    *   reference: Am Soc Hum Genet annual meeting 1995 (Göring and Ott, Am J Hum Genet, suppl. 57, A192 (1995));

    *   Goring and Ott (1997), "Relationship estimation in affected sib pair analysis of late-onset diseases", European Journal of Human Geneics, 5:69-77.

*   RELATIVEFINDER

    *   full name:

    *   version:

    *   description: relativeFinder is a program for checking relationships between pairs of individuals. There are many excellent programs that carry out similar tasks. Some of the unique features in relativeFinder are the batch mode options, that allow large jobs to be divided into many smaller jobs (suitable for deployment on a compute cluster environment), and the flexibility of the underlying Merlin engine, which allows relative finder to handle large pedigrees and consider a variety of alternate relationships -- including potential relationships specified by the user on the fly.

    *   authors: Abecasis's group (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/RelativeFinder

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   RELCHECK

    *   full name:

    *   version: 0.67 (August 2000)

    *   description: Program for verifying the relationships between all pairs of individuals in a linkage study, by use of (autosomal) genome scan data, with allowance for the presence of genotyping errors.

    *   Authors: Karl W. Broman (Johns Hopkins Univ, kbroman@jhsph.edu)

    *   web/ftp: http://www.biostat.jhsph.edu/~kbroman/software/

    *   source code language: C

    *   operating systems: MS-DOS, MS-Windows, UNIX

    *   executables: relcheck

    *   reference: Boehnke, Cox (1997), "Accurate inference of relationships in sib-pair linkage studies", American Journal of Human Genetics, 61(2):423-429

    *   Broman, Weber (1998), "Estimation of pairwise relationships in the presence of genotyping errors" (letter), American Journal of Human Genetics, 63(5):1563-1564

*   RELPAIR

    *   full name:

    *   version: 2.01 (June 2004)

    *   description: A program that infers the relationships of pairs of individuals based on genetic marker data, either within families or across an entire sample.

    *   Authors: Michael Boehnke, William Duren (Biostat, U Michigan)

    *   web/ftp: http://csg.sph.umich.edu/boehnke/relpair.php

    *   source code language: FORTRAN77

    *   operating systems: MS-Windows (95), UNIX

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting 1997 (Duren,Cox,Hauser,Boehnke, Am J. Hum Genet. suppl 61, A273(1997)).

*   RELTYPE

    *   full name:

    *   version:

    *   description: a computer program for determining biological relatedness between individuals based on allele sharing at microsatellite loci

    *   authors: D.N. Stivers (dstivers@utsph.uth.tmc.edu), Y. Zhong, C.L. Hanis, R. Chakraborty (Univ of Texas at Houston)

    *   web/ftp: http://hgc.sph.uth.tmc.edu (not available yet)

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Chakraborty, Jin (1993), "Determination of relatedness between individuals using DNA fingerprinting", Human Biology, 65(6):875-895.

    *   Am Soc Hum Genet annual meeting 1996 (Stivers, Zhong, Hanis, Chakraborty, Am J. Hum Genet. suppl 59, A190 (1996)).

*   RHMAP

    *   full name: Radiation Hybrid Map

    *   version: 3.0 (Sept 1996)

    *   description:

    *   authors: Michael Boehnke (Univ Michigan, boehnke@sph.umich.edu),E. Hauser, K. Lange, K. Lunetta, J. Uro, J. VanderStoep

    *   web/ftp: http://csg.sph.umich.edu/boehnke/rhmap.php

    *   source code language: FORTRAN77

    *   operating systems:

    *   executables: rhmap, rh2pt, rhminbrk, rhmaxlik

    *   reference: Boehnke,Lange,Cox (1991), "Statistical methods for multipoint radiation hybrid mapping", American Journal of Human Genetics, 49(6):1174-1188

    *   Lange,Boehnke, Cox, Lunetta (1995), Genome Research, 5:136-150.

    *   Lunetta,Boehnke, Lange, Cox (1995), Genome Research, 5:151-163.

*   RHMAPPER

    *   full name:

    *   version: 1.22 (1998)

    *   description: an interactive program for radiation hybrid mapping

    *   authors: Lincoln Stein (Whitehead Institute, lstein@genome.wi.mit.edu)

    *   web/ftp: http://www-genome.wi.mit.edu/ftp/pub/software/rhmapper/

    *   ftp://ftp-genome.wi.mit.edu/pub/software/rhmapper

    *   source code language: Perl

    *   operating systems:

    *   executables:

    *   reference: Hudson, Stein, et al. (1995), Science, 270:1945-1954.

    *   Slonim, Stein, Kruglyak, Lander (1996), in preparation.

    *   online user's guide

*   RISCALW

    *   full name: RISk CALculation in Windows

    *   version:

    *   description: RISCALW is a novel convenient Windows program for risk calculation in families with Duchenne muscular dystrophy. It is based on an extended genetic model which includes germline mosaicism and different new mutation rates depending on sex and mutation type. Arbitrary family structures and additional diagnostic information like genotypes from intragenetic and flancing genetic markers of the dystrophin gene, creatin kinase values and female deletion test results can be taken into account.

    *   authors: Christine Fischer (email: christine.fischer@med.uni-heidelberg.de)

    *   web/ftp: http://www.riscalw.uni-hd.de

    *   source code language: Delphi 4

    *   operating systems: MS-Windows

    *   executables:

    *   reference: Fischer, Kruger, Gross (2006), "RISCALW: a windows program for risk calculation in families with Duchenne Muscular Dystrophy", Annals of Human Genetics, 70:249-254.

    *   Fischer, Gross, Kruger, Cremer, Vogel, Grimm (2006), "Modeling germline mosaicism and different new mutation rates simultaneously for appropriate risk calculations in families with Duchenne muscular dystrophy", Annals of Human Genetics, 70:237-248.

*   ROMPREV

    *   full name: Regression of Offspring on Mid-Parent (REVised)

    *   version: 0.2.1beta

    *   description: a software tool for testing for association between polymorphisms and quantitative traits, as well as estimating trait heritability and locus-specific heritability using family data.

    *   authors: Marie-Helene Roy-Gagnon and Alexander F. Wilson (afw@mail.nih.gov)

    *   web/ftp: http://research.nhgri.nih.gov/ROMPrev/

    *   ftp://ftp.nhgri.nih.gov/pub/software/ROMPrev/

    *   source code language: R

    *   operating systems: any with R installation

    *   executables:

    *   reference: Roy-Gagnon, Mathias, Fallin, Jee, Broman, Wilson (2008), "An extension of the Regression of Offspring on Mid-Parent to test for association and estimate locus-specific heritability: The revised ROMP method", Annals of Human Genetics, 72:115-125.

    *   Roy-Gagnon (2004), "Theoretical and applied investigations of a Regression of Offspring on Mid-Parent approach: Familial idiopathic scoliosis", Ph.D. dissertaion. Johns Hopkins University.

    *   Pugh, Papanicolaou, Justice, Roy-Gagnon, Sorant, Kingman, Wilson (2001), "Comparison of variance components, ANOVA and regression of offspring on midparent (ROMP) methods for SNP markers", Genetic Epidemiology, 21(Suppl 1):S794-S799.

*   ROSATTA SYLLEGO SYSTEM

    *   full name:

    *   version: evaluation version (October 2007)

    *   description: The Rosetta Syllego system is a genetic data management and analysis system designed to advance your whole genome association, linkage, and eQTL studies. Designed for biologists, statistical geneticists, and investigators responsible for generating genotyping data, the Syllego system provides you with an easy-to-use project workspace so that you can organize, analyze, and share your genotype and phenotype data along with your analysis results. With the Syllego system, generating high quality analysis data and meaningful results becomes simple. The Syllego system automates all tedious data management and data formatting tasks so that you can streamline your genetic analysis workflows using your analysis methods of choice. Managing all your genetic data and reference information is straightforward. The Syllego system converts public and private genotype data sets and reference annotations, such as dbSNP and HapMap, as well as individual (sample) information into a single, consistent repository for fast, convenient access.

    *   authors:

    *   web/ftp: http://www.rosettabio.com/products/syllego/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   RTDT

    *   full name: Robust Transmission/Disequilibrium Test

    *   version: 1.0

    *   description:

    *   authors:

    *   web/ftp: http://bios.ugr.es/~mabad/rTDT/index.html

    *   source code language: Matlab, C++

    *   operating systems:

    *   executables:

    *   reference: Sebastiani, Abad, Alpargu, Ramoni (2004), "Robust Transmission/Disequilibrium Test for incomplete family genotypes", Genetics, 168:2329-2337.

*   R/ADEGENET

    *   full name:

    *   version: 1.2-6 (September 2010)

    *   description: ADEGENET is a package dedicated to the handling of molecular marker data for multivariate analysis. This package is related to ADE4, a R package for multivariate analysis, graphics, phylogeny and spatial analysis.

    *   authors: Thibaut Jombart (email: jombart@biomserv.univ-lyon1.fr)

    *   web/ftp: http://adegenet.r-forge.r-project.org/

    *   http://cran.r-project.org/web/packages/adegenet/index.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Jombart (2008), "adegenet: a R package for the multivariate analysis of genetic markers", Bioinformatics, 24:1403-1405.

*   R/ARP.GEE (see ARP.GEE)

*   R/BARS (see BARS)

*   R/COMPOSITELD (see COMPOSITELD)

*   R/COVIBD (see COVIBD)

*   R/EHP (see EHP.R)

*   R/ENTROPY_BLOCKER (see ENTROPY BLOCKER)

*   R/FEST

    *   full name:

    *   version: 0.08 (August, 2010)

    *   description: an R package for simulations and likelihood calculations of pair-wise family relationships using DNA marker data.

    *   authors: Øivind Skare (email: oivind.skare@medisin.uio.no

    *   web/ftp: http://folk.uio.no/thoree/FEST/

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Skare, Sheehan, Egeland (2009), "Identification of distant family relationships", Bioinformatics, 25(18):2376-2382.

*   R/GAP

    *   full name: R/Genetic Analysis Package

    *   version: 1.17-39 (July 2010)

    *   description: This is an integrated package for genetic data analysis of both population and family data. Currently it contains functions for sample size calculations of both population-based and family-based designs, classic twin ACE/ADE/AE/CE models, probability of familial disease aggregation, kinship calculation, some statistics in linkage analysis, and association analysis involving one or more genetic markers including haplotype analysis with or without environmental covariates

    *   authors: Jinghua Zhao (MRC Epidemiology Unit at Strangeways Research Lab, UK) (email: jinghua.zhao@mrc-epid.cam.ac.uk) in collaboration with other colleagues, and with help from Kurt Hornik of the R core

    *   web/ftp: http://www.mrc-epid.cam.ac.uk/~jinghua.zhao/r-progs.htm

    *   source code language: R

    *   operating systems:

    *   executables: aldh2, apoeapoc, bt, chow.test, fbsize, fsnps, gc.em, gcontrol, genecounting, gif, hap, hap.em, hap.score, hla, htr, hwe, hwe.hardy, kbyl, kin.morgan, makeped, mia, mtdt, muvar, pbsize, pfc, pgc, s2k, tbyt, whscore

    *   reference: Zhao and Tan (2006), "Integrated analysis of genetic data with R", Human Genomics, 2(4):258-265.

    *   reference manual (PDF): http://cran.r-project.org/doc/packages/gap.pdf

*   R/GC, R/GCF (see GC/GCF)

*   R/GENABEL

    *   full name:

    *   version: 1.6-4 (August 2010)

    *   description: GenABEL is a software created to facilitate whole-genome association analysis. Specific features of GenABEL include: * Support of modern methods for analysis in outbred populations * Maximum Likelihood estimation of linear, logistic and Cox regression on genome-wide scale * A range of functions specifically designed to help GWA analysis in samples of individuals from genetically isolated populations, and/or samples of related individuals (family-based design), and other analysis in presence of stratification * Special functions oriented to meta-analysis * Extensive documentation (about 100 pages of reference manual and 150 pages of tutorial) * Many example data sets and exercises
    *   authors:

    *   web/ftp: http://mga.bionet.nsc.ru/~yurii/ABEL/GenABEL/

    *   http://cran.r-project.org/web/packages/GenABEL/index.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Aulchenko, Ripke, Isaacs, van Duijn (2007), "GenABEL: an R library for genome-wide association analysis", Bioinformatics, 23(10):1294-1296.

*   R/GWAPOWER (see GWAPOWER)

*   R/HAPASSOC

    *   full name:

    *   version: 1.2-4 (July 2010)

    *   description: a likelihood approach is used to inference of haplotype and nongenetic effects and their interactions in generalized linear models of disease penetrance, when haplotype phase is unknown for some subjects. Parameter estimates are obtained by use of an expectation-maximization (EM) algorithm and standard errors are calculated using Louis' formula.

    *   authors:

    *   web/ftp: http://stat-db.stat.sfu.ca:8080/statgen/research/hapassoc

    *   http://cran.r-project.org/web/packages/hapassoc/index.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference:

*   R/HCLUST (see HCLUST)

*   R/IBDREG (see IBDREG)

*   R/LAPSTRUCT (see LAPSTRUCT)

*   R/LDGROUP (see LDGROUP)

*   R/LDHEATMAP

    *   full name:

    *   version: 0.2-9 (August 2010)

    *   description: plots measures of pairwise linkage disequilibria for SNPs

    *   authors: Ji-Hyung Shin (email: shin@sfu.ca), Sigal Blay, Nicholas Lewin-Koh, Brad McNeney, Jinko Graham

    *   web/ftp: http://stat-db.stat.sfu.ca:8080/statgen/research/LDheatmap

    *   http://cran.r-project.org/web/packages/LDheatmap/index.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Shin, Blay, McNeney, Graham (2006), "LDheatmap: an R function for graphical display of pairwise linkage disequilibria between single nucleotide polymorphisms", Journal of Statistical Software, 16, code snippet 3.

*   R/LUCA

    *   full name: Likelihood Under Covariate Assumptions

    *   version:

    *   description:

    *   authors: Ji-Hyung Shin (email: shin@sfu.ca), Brad McNeney, Jinko Graham

    *   web/ftp: http://stat-db.stat.sfu.ca:8080/statgen/research/luca/

    *   http://cran.r-project.org/web/packages/luca/index.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Shin, McNeney, Graham (2007), "Case-control inference of interaction between genetic and nongenetic risk factors under assumptions on their distribution", Statistical Applications in Genetics and Molecular Biology, 6(1):13.

*   R/META

    *   full name:

    *   version: 1.6 (June 2010)

    *   description: Fixed and random effects meta-analysis. Functions for tests of bias, forest and funnel plot.

    *   authors: Guido Schwarzer (email: sc@imbi.uni-freiburg.de)

    *   web/ftp: http://cran.r-project.org/web/packages/meta/index.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference:

*   R/METASIM

    *   full name: METApopulation SIMulation

    *   version: 1.01

    *   description: an R package that uses an individual-based approach to simulate distributions of genotypes that result from arbitrary within and among population demographies (including extinction/recolonization). These distributions can be used to test new or existing population-genetics summary statistics or develop "null" distributions under various demographies.

    *   authors: Allan Strand (email: stranda@cofc.edu)

    *   web/ftp: http://linum.cofc.edu/software.html

    *   source code language: R, C++

    *   operating systems: UNIX, MS-Windows, MacOS

    *   executables:

    *   reference: A Strand (2002), "Metasim 1.0: an individual-based environment for simulating population genetics of complex population dynamics", Mol Ecol Notes, 2:373-376.

*   R/ONEMAP

    *   full name:

    *   version: 1.0-1 (April 2009)

    *   description: ONEMAP is an environment for constructing linkage maps in outcrossing plant species, using full-sib families derived from two outbreed (non-inbreeding) parent plants.

    *   authors: Gabriel Rodrigues Alves Margarido, Anete Pereira de Souza, Antonio Augusto Franco Garcia (email: aafgarci@esalq.usp.br)

    *   web/ftp: http://www.ciagri.usp.br/~aafgarci/OneMap/

    *   http://cran.r-project.org/web/packages/onemap/index.html

    *   source code language: R

    *   operating systems:

    *   executables: load.onemap, read.outcross, extract.group, remove.marker, arbitr.rf.2pts, est.rf.2pts, modigy.rf.2pts, group, map, make.map, est.rf.3pts

    *   reference: Garcia, Kido, Meza, Souza, Pinto, Pastina, Leite, Silva, Ulian, Figueira, Souza (2006), "Development of an integrated genetic map of a sugarcane (Saccharum spp.) commercial cross, based on a maximum-likelihood approach for estimation of linkage and linkage phases", Theoretical and Applied Genetics, 112:298-314.

*   R/PIAGE (see PIAGE)

*   R/POOLSCORE (see POOLSCORE)

*   R/POPGEN (see POPGEN)

*   R/QTL

    *   full name:

    *   version: 1.05-2 (November 2006)

    *   description: Program for mapping quantitative trait loci in experimental crosses.

    *   Authors: Karl W. Broman (Johns Hopkins Univ, kbroman@jhsph.edu) and Hao Wu (The Jackson Laboratory)

    *   web/ftp: http://www.rqtl.org

    *   source code language: C and R

    *   operating systems: UNIX, MS-Windows, MacOS

    *   executables:

    *   reference: Broman, Wu, Sen, Churchill (2003), Bioinformatics, 19:889-890\. [Medline]

*   R/QTLBIM

    *   full name:

    *   version: 1.7.7 (April 2008)

    *   description: QTLBIM, a library for QTL Bayesian Interval Mapping, provides a Bayesian model selection approach to map multiple interacting QTL. It works on experimentally inbred lines and performs a genome-wide search to locate multiple potential QTL. The package can handle continuous, binary and ordinal traits.

    *   authors:

    *   web/ftp: http://www.qtlbim.org/

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference:

*   R/QTLDESIGN

    *   full name:

    *   version: v0.91 (May 2007)

    *   description: R/qtlDesign is software to help plan quantitative trait locus (QTL) experiments.

    *   authors: Saunak Sen, Jaya Satagopan, Karl Broman, Gary Churchill

    *   web/ftp: http://www.epibiostat.ucsf.edu/biostat/sen/qtldesign-software/

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference:

*   R/SNP.PLOTTER

    *   full name:

    *   version: 0.3 (October 2009)

    *   description: snp.plotter is an R package that creates publishable-quality plots of p-values using single SNP and/or haplotype data. Main features of the package include options to display a linkage disequilibrium (LD) plot and the ability to plot multiple sets of results simultaneously. Plots can be created using global and/or individual haplotype p-values along with single SNP p-values. Images are created as either Portable Document Format (PDF) or Encapsulated (EPS) files.

    *   authors: Augustin Luna, Kristin Nicodemus (email: augustin@mail.nih.gov)

    *   web/ftp: http://cbdb.nimh.nih.gov/~kristin/snp.plotter.html

    *   http://cran.r-project.org/web/packages/snp.plotter/index.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Luna, Nicodemus (2007), "snp.plotter: an R-based SNP/haplotype association and linkage disequilibrium plotting package", Bioinformatics, 23(6):774-776.

*   R/SPECTRAL-GEM

    *   full name: SPECTRAL graph approach for GEnetic Matching

    *   version: 1 (July 2009)

    *   description:

    *   authors: Ann Lee, Diana Luca, Lambertus Klei, Bernie Devlin, Kathryn Roeder

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference: Lee, Luca, Klei, Devlin, Roeder (2010), "Discovering genetic ancestry using spectral graph theory", Genetic Epidemiology, 34(1):51-59.

*   R/STEPWISE

    *   full name:

    *   version: 0.2-4 (November 2008)

    *   description: A stepwise approach to identifying recombination breakpoints in a sequence alignment

    *   authors: Jinko Graham, Brad McNeney, Francoise Seillier-Moiseiwitsch, Sigal Blay

    *   web/ftp: http://stat-db.stat.sfu.ca:8080/statgen/research/stepwise/

    *   http://cran.r-project.org/web/packages/stepwise/index.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   R/TDTHAP

    *   full name:

    *   version: 1.1-2 (November 2007)

    *   description: a package for TDT with extended haplotypes in the "R" language. "R" is the public domain dialect of "S". It should be possible to port this library to the commercial "Splus" product. The main problem would be translation of the help files.

    *   Authors: David Clayton (Cambridge Institute, david.clayton@cimr.cam.ac.uk

    *   web/ftp: http://www-gene.cimr.cam.ac.uk/clayton/software/

    *   http://cran.r-project.org/web/packages/tdthap/index.html

    *   source code language: R/Splus

    *   operating systems:

    *   executables:

    *   reference: Clayton, Jones (1999), "Transmission/disequilibrium tests for extended marker haplotypes", American Journal of Human Genetics, 65(4):1161-1169

*   RVTESTS

    *   full name: Rare Variants TESTS

    *   version:

    *   description:

    *   authors: X Zhan, Youna Hu, G Abecasis (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/RvTests

    *   source code language: C++

    *   operating systems:

    *   executables:

    *   reference:

*   R/WEIGHTED_FDR (see WEIGHTED_FDR)

*   S

*   SAGE

    *   full name:Statistical Analysis for Genetic Epidemiology

    *   version: 5.3

    *   description: The S.A.G.E. software provides researchers with the tools necessary for various types of statistical genetic analysis of human family data.

    *   authors: R Elston, J Bailey-Wilson, G Bonney, L. Tran, B. Keats, A. Wilson

    *   web/ftp: http://darwin.cwru.edu/sage/

    *   source code language: C [version 4.0 will be in C++]

    *   operating systems: UNIX(DEC UNIX/Solaris), MS-Windows (95/NT), Linux

    *   executables: assoc, fcor, freq, genibd, lodlink, lodpal, markerinfo, mlod, pedinfo, reltest, segreg, sibpal, tdtex

    *   reference:

*   SAMTOOLS

    *   full name: Sequence Alignment/Map TOOLS

    *   version:

    *   description: SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format. The parts that relevant to genetic analysis include SNP and indel callings.

    *   authors: Heng Li (Broad Institute), Richard Durbin (Wellcome Trust Sanger Institute) et al.

    *   web/ftp: http://samtools.sourceforge.net/

    *   source code language: C, C++, Java, Perl, Python, Ruby, Common LISP, Haskell

    *   operating systems:

    *   executables:

    *   reference: Li, Handsaker, Wysoker, Fennell, Ruan, Homer, Marth, Abecasis, Durbin, and 1000 Genome Project Data Processing Subgroup (2009), "The sequence alignment/map (SAM) format and SAMtools", Bioinformatics, 25:2078-2079.

*   SASGENE

    *   full name:

    *   version: 1.2 (March 1999)

    *   description: a SAS computer program for gene segregation and linkage analysis in breeding population

    *   authors: Jinsheng Liu, Todd Wehner, Sandy Donaghy

    *   web/ftp: http://cuke.hort.ncsu.edu/cucurbit/wehner/software.html

    *   source code language: SAS

    *   operating systems:

    *   executables:

    *   reference: Liu, Wehner, Donaghy (1997), "SASGENE: A SAS computer program for genetic analysis of gene segregation and linkage", Journal of Heredity, 88:253-254.

*   SAS/GENETICS

    *   full name:

    *   version: add-on to the SAS System, Release 8.2

    *   description: Software for summarizing marker properties (allele & genotype frequencies, tests for Hardy-Weinberg equilibrium, measures of marker informativeness), examining marker-marker relationships (tests and measures of linkage disequilibrium, and haplotype frequency estimation), and exploring marker-trait associations using case-control or family-based tests

    *   Authors: W Czika, X Yu

    *   web/ftp:

    *   source code language: C

    *   operating systems: MS-Windows, UNIX(HPUX/AIXR/Solaris/..), MVS

    *   executables:

    *   reference: introduction

    *   price: please contact shannon.conners@jmp.com for details

*   SASQUANT

    *   full name:

    *   version: 1.1 (May 2005)

    *   description: a SAS software program to estimate genetic effects and heritabilities of quantitative traits in breeding populations consisting of six related generations

    *   authors: Gabriele Gusmini, Todd Wehner , Sandy B. Donaghy

    *   web/ftp: http://cuke.hort.ncsu.edu/cucurbit/wehner/software.html

    *   source code language: SAS

    *   operating systems:

    *   executables:

    *   reference: Gusmini, Wehner, Donaghy (2007), "SASQuant: A SAS software program to estimate genetic effects and heritabilities of quantitative traits in populations consisting of six related generations", Journal of Heredity, 98(4):345-350.

*   SCORE-SEQ

    *   full name: SCORE-type tests for detecting disease associations with rare variants in SEQuencing Studies

    *   version:

    *   description: SCORE-Seq is a command-line program for detecting disease associations with rare variants in sequencing studies. The mutation information is aggregated across multiple variant sites of a gene through a weighted linear combination and then related to disease phenotypes through appropriate regression models. The weights can be constant or dependent on allele frequencies and phenotypes. The association testing is based on score-type statistics. The allele-frequency threshold can be fixed or variable. Statistical significance can be assessed by using asymptotic normal approximation or resampling. The current release covers binary and continuous traits with arbitrary covariates under case-control and cross-sectional sampling.

    *   authors: Danyu Lin, Zheng-Zheng Tang (Univ North Carolina)

    *   web/ftp: http://www.bios.unc.edu/~lin/software/SCORE-Seq/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Lin, Tang (2011), "A general framework for detecting disease associations with rare variants in sequencing studies", American Journal of Human Genetics, 89:354-367.

*   SCOUT

    *   full name:

    *   version: 1.00 (Jan 2006)

    *   description: Scout is a software program for conducting combined association analysis of triads and unrelated subjects

    *   authors: Richard Duncan and Michael Epstein (Emory University, email: statgen@genetics.emory.edu)

    *   web/ftp: http://www.genetics.emory.edu/labs/epstein/software/scout/index.html

    *   source code language: Fortran (CVF 6.6) with IMSL routines

    *   operating systems: MS-Windows(2000/XP)

    *   executables: scout_cl.exe, scout_gui.exe

    *   reference: Epstein, Veal, Trembath, Barker, Li, Satten (2005), "Genetic association analysis using data from triads and unrelated subjects", American Journal of Human Genetics, 76(4):592-608\. [ html]

*   SDMINP

    *   full name: Step-Down MIN P-value

    *   version: 1.0 (December 2005)

    *   description: SDMinP is an easy-to-use program for fast calculation of empirical and adjusted p-values for correlated and uncorrelated hypotheses in multiple testing experiments. It is based on the Free Step-Down Resampling Method for controlling the Family Wise Error Rate, originally proposed by Westfall and Young (1993), and implements a variation of the efficient algorithm of Ge et al. (2003), in which the originally necessary re-sampling effort was reduced considerably and the method made computationally more feasible. The program is independent of the underlying test statistic and works with provided observed and permutation test statistics.

    *   authors: Markus Obreiter, Lars Beckmann (email: l.beckmann@dkfz.de)

    *   web/ftp: http://www.dkfz.de/SDMinP/

    *   source code language: Python 2.3.5

    *   operating systems: UNIX, Linux, MS-Windows

    *   executables:

    *   reference: Obreiter, Fischer, Chang-Claude, Beckmann (2005), "SDMinP: a program to control the family wise error rate using step-down minP adjusted p-values", Bioinformatics, 21(14):3183-3184.

*   SEGPATH

    *   full name: SEGregation analysis and PATH analysis

    *   version:

    *   description:

    *   authors: Michael A. Province (email: mike@wubios.wustl.edu)

    *   web/ftp: http://www.biostat.wustl.edu/~mike/segpath/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Province, Rao (1995), "General purpose model and a computer program for combined segregation and path analysis (SEGPATH): automatically creating computer programs from symbolic language model specifications", Genetic Epidemiology, 12(2):203-219.

    *   Province, Rice, Borecki, Gu, Kraja, Rao (2003), "Multivariate and multilocus variance components method, based on structural relationships to assess quantitative trait linkage via SEGPATH", Genetic Epidemiology, 24(2):128-138.

*   SELSIM

    *   full name:

    *   version: 2.1 (June 2005)

    *   description: When designing methods which provide the necessary power to detect regions of the genome which have experience historical selective pressures it is important to consider which patterns of genetic diversity are indicative of particular forms of natural selection. SELSIM is a program which can simulate population genetic data in which a single site has experienced natural selection.

    *   authors: Chris Spencer (Univ of Oxford, email: spencer@stats.ox.ac.uk)

    *   web/ftp: http://www.stats.ox.ac.uk/mathgen/software.html

    *   http://www.stats.ox.ac.uk/~spencer/SelSim/Controlfile.html

    *   source code language: C,C++

    *   operating systems: MS-Windows(XP), Linux

    *   executables:

    *   reference: Spencer, Coop (2004), "SelSim: a program to simulate population genetic data with natural selection and recombination", Bioinformatics, 20:3673-3675.

*   SEQUENCE LD/SEQUENCE LDHOT

    *   full name:

    *   version:

    *   description: The program sequenceLD analyzes sequence data. It obtains an approximation to the likelihood of a summary of the data (as such it can be thought of as a marginal likelihood approach). It does not use all the information in the data, but computationally it can be substantially more efficient than the full-likelihood methods (and hence able to analyze larger data sets). | The program sequenceLDhot is a method for detecting recombination hotspots from population genetic data. It takes as input phased (i.e. haplotype) data, together with an estimate of the background recombination rate within the region (this is allowed to vary across the region). Then sequenceLDhot considers a grid of putative hotspot positions, and for each putative hotspot calculates a likelihood ratio statistics for the presence of a hotspot.

    *   authors: Paul Fearnhead (Department of Mathematics and Statistics, Lancaster University, UK)

    *   web/ftp: http://www.maths.lancs.ac.uk/~fearnhea/Software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Fearnhead, Donnelly (2002), "Approximate likelihood methods for estimating local recombination rates", Journal of the Royal Statistical Society series B, 64:657-680.

    *   Fearnhead (2006), "SequenceLDhot: detecting recombination hotspots", Bioinformatics, 22:3061-3066.

*   SGS

    *   full name:

    *   version: 1.0d

    *   description:

    *   authors: Bernd Degen

    *   web/ftp: http://software.bfh-inst2.de/download3.html

    *   source code language: VisualBasic

    *   operating systems: MS-Windows (95/98/00/NT)

    *   executables:

    *   reference: Degen, Petit, Kremer (2001), "SGS - Spatial Genetic Software: a computer program for analysis of spatial genetic and phenotypic structures of individuals and populations", Journal of Heredity, 92(5):447-448.

*   SIBERROR

    *   full name: SibError

    *   version: 1.0 (November 20, 1997)

    *   description: SibError is an algorithm that identifies pedigree errors in sibship data. Examples include half siblings, unrelated individuals, identical twins, and parental exclusions. The test statistic is based on the summation of the number of alleles shared by a pair of relatives for a large number of markers and the number of alleles and allele frequencies for those markers.

    *   Authors: Drs. Meg Gelder Ehm (mge37216@glaxowellcome.com) and Michael Wagner ( mw29911@glaxowellcome.com) (Glaxo Wellcome, Inc.)

    *   web/ftp:

    *   source code language: C

    *   operating systems: UNIX

    *   executables: siberror

    *   reference: Ehm, Wagner (1998), "Test statistic to detect errors in sib-pair relationship", American Journal of Human Genetics, 62(1):181-188

*   SIBLINK

    *   full name:

    *   version: 3.0

    *   description: SIBLINK allows the user to perform multipoint linkage analysis based on estimated IBD sharing between affected sibpairs. IBD sharing is inferred from IBS status, given marker genotypes, frequencies, and locations. Resulting LOD scores are maximized across a grid of possible disease locations and IBD sharing vectors.

    *   authors: Elizabeth Hauser, Mike Boehnke, William Duren

    *   web/ftp: http://wwwchg.duhs.duke.edu/software/siblink.html

    *   source code language: C++

    *   operating systems: UNIX(Solaris), Linux

    *   executables: siblink3.0

    *   reference: ER Hauser, M Boehnke, SW Guo, N Risch (1996), Genetics Epidemiology. 13:117-137.

    *   ER Hauser, M Boehnke M (1998), Biometrics, 54:1238-1246.

*   SIBMED

    *   full name: SIBling Mutation and Error Detection

    *   version: 1.0

    *   description: SIBMED identifies likely genotyping errors and mutations for a sib pair in the context of multipoint mapping.

    *   Authors: Julie A Douglas, Michael Boehnke (Univ Michigan)

    *   web/ftp: http://csg.sph.umich.edu/boehnke/sibmed.php

    *   source code language: FORTRAN 77

    *   operating systems: MS-Windows, UNIX

    *   executables: sibmed.exe

    *   reference: Douglas, Boehnke, Lange (2000), "A multipoint method for detecting genotyping errors and mutations in sibling-pair linkage data", American Journal of Human Genetics, 66(4):1287-1297\. [ html]

*   SIB-PAIR

    *   full name:

    *   version: 1.00a (January 2007)

    *   description: Program SIB-PAIR performs a number of simple analyses of family data that tend to be "nonparametric" or "robust" in nature, includes IBD and IBS based APM, Haseman-Elston sib pair, TDT and association analyses.

    *   Authors: David Duffy (Queensland Institute of Med Res, Australia, davidd@qimr.edu.au)

    *   web/ftp: http://www2.qimr.edu.au/davidD

    *   source code language: Fortran77

    *   operating systems:

    *   executables:

    *   reference: Am. Soc. Hum. Genet. Annual Meeting 1997 (Duffy, Am.J. Hum. Genet. suppl. 61, A197 (1997)); online manual page

*   SIBSIM

    *   full name:

    *   version: 2.1.2 (November 2005)

    *   description: Simulates either genotype and/or quantitative phenotype data in family structures in a modern, easy to use and highly scalable way. It is based on XML, completely written in C++ and published under the GNU General Public License.

    *   authors: Daniel Franke (University Hospital Schleswig-Holstein)

    *   web/ftp: http://www.imbs.uni-luebeck.de/pub/sibsim/index.html

    *   source code language: C++, XML

    *   operating systems: UNIX

    *   executables:

    *   reference: D Franke, A Kleensang and A Ziegler (2006), "SIBSIM, quantitative phenotype simulation in extended pedigrees", GMS Med Inform Biom Epidemiol, 2(1):Doc04.

*   SIFT

    *   full name: Sorting Intolerant From Tolerant

    *   version:

    *   description: SIFT predicts whether an amino acid substitution affects protein function based on sequence homology and the physical properties of amino acids. SIFT can be applied to naturally occurring nonsynonymous polymorphisms and laboratory-induced missense mutations.

    *   authors: Pauling Ng (Genome Institute of Singapore), Steven Henikoff (Fred Hutchinson Cancer Research Institute)

    *   web/ftp: http://sift.bii.a-star.edu.sg/

    *   http://sift.jcvi.org/

    *   http://blocks.fhcrc.org/sift/SIFT.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Ng, Henikoff (2001), "Predicting deleterious amino acid substitutions", Genome Research, 11:863-874.

    *   Ng, Henikoff (2002), "Accounting for human polymorphisms predicted to affect protein function", Genome Research, 12:436-446.

    *   Ng, Henikoff (2003), "SIFT: Predicting amino acid changes that affect protein function", Nucleic Acids Research, 31:3812-3814.

    *   Kumar, Henikoff, Ng (2009), "Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm", Nature Protocols, 4:1073-1081.

*   SILCLOD

    *   full name: SIgnificance Levels and Critical LODs

    *   version:

    *   description: calculate nominal significance levels and critical LOD scores depending on the length of the investigated region, number of chromosomes, and the cross-over rate. The global significance level as well as the precision of the calculation have to be specified.

    *   authors: Andreas Ziegler (email: koenigir@imbs.uni-luebeck.de)

    *   web/ftp: http://www.imbs.uni-luebeck.de/pub/silcLOD/index.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: E Lander, L Kruglyak (1995), "Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results", Nature Genetics, 11:241-247.

*   SIMCOAL

    *   full name: SIMulate COAlescence

    *   version: 1.0

    *   description:

    *   authors: Laurent Excoffier (email: laurent.excoffier@zoo.unibe.ch)

    *   web/ftp: http://cmpg.unibe.ch/software/simcoal/

    *   source code language: C++

    *   operating systems: MS-Windows(9X/NT)

    *   executables: simcoal, sim_coal

    *   reference: Excoffier, Novembre, Schneider (2000), "SIMCOAL: a general coalescent program for the simulation of molecular data in interconnected populations with arbitrary demography", Journal of Heredity, 91:506-509.

*   SIMHAP

    *   full name: SIMulation-based HAPlotypic analysis

    *   version: 1.0.2 (January 2009)

    *   description: SimHap uses biallelic SNP genotype data to impute haplotype frequencies at the individual level. SimHap also tests for haplotype associations with outcomes of interest while incorporating the uncertainty around inferred haplotypes into the modelling procedure. SimHap allows epidemiological (ie, non-genetic) and both single SNP and haplotype association analyses of quantitative Normal, binary, longitudinal and right-censored outcomes under a range of genetic models. SimHap can accommodate large data sets, and can model genetic and environmental effects, including complex haplotype:environment interactions. SimHap features cross-platform functionality via Java, and a sophisticated graphical user interface (GUI). SimHap will also perform association analysis on more simple epidemiological models, with or without the inclusion of genetic covariates.

    *   authors: Pamela A. McCaskie (Univ. of Western Australia), Kim W Carter, Lyle J Palmer

    *   web/ftp: http://www.genepi.org.au/simhap

    *   source code language: Java (1.5)

    *   operating systems: web-based

    *   executables:

    *   reference: Carter, McCaskie, Palmer (2008), "SimHap GUI: An intuitive graphical user interface for genetic association analysis", BMC Bioinformatics, 9:557

*   SIMIBD

    *   full name:

    *   version: 1.21 (Jan 1997)

    *   description:

    *   authors: Sean Davis ( davis@moriarty.hgen.pitt.edu) and Daniel E. Weeks

    *   web/ftp: http://watson.hgen.pitt.edu/register/soft_doc.html, http://watson.hgen.pitt.edu/~davis/

    *   http://watson.hgen.pitt.edu/register

    *   source code language: C

    *   operating systems: UNIX(SunOS/Solaris/HP/DEC-UNIX)

    *   executables: simibd, xslink

    *   reference: Davis, Schroeder, Goldin, Weeks (1996), "Nonparametric simulation-based statistics for detecting linkage in general pedigrees", American Journal of Human Genetics, 58(4):867-880

    *   online README file

*   SIMLA

    *   full name: SIMulation of pedigree data for Linkage and Association studies

    *   version: 2.3 (Oct 2003)

    *   description: SIMLA is a SIMulation program that generates data sets of families for use in Linkage and Association studies. It allows the user flexibility in specifying marker and disease placement, locus heterogeneity, disequilibrium between markers and between markers and disease loci. Output is in the form of a LINKAGE pedigree file and is easily utilized, either directly or with minimal reformatting, as input for various genetic analysis packages

    *   authors: Elizabeth Hauser, Eden Martin, Meredyth Bass

    *   web/ftp: http://wwwchg.duhs.duke.edu/software/simla.html

    *   source code language: C++

    *   operating systems: UNIX(Solaris), Linux

    *   executables: simla2.3

    *   reference: MP Bass, ER Martin, ER Hauser(2002), Am J Hum Genet, suppl 71:569.

    *   MP Bass, ER Martin, ER Hauser (2004), "Pedigree generation for analysis of genetic linkage and association", Proceedings of the Pacific Symposium on Biocomputing, pp.93-103.

*   SIMLINK

    *   full name:

    *   version: 4.12 (April 1997)

    *   description: a computer program to estimate the probability ("power") of detecting linkage given family history information on a set of identified pedigrees.

    *   authors: Michael Boehnke ( boehnke@sph.umich.edu)

    *   web/ftp: http://csg.sph.umich.edu/boehnke/simlink.php

    *   source code language: FORTRAN

    *   operating systems: MS-DOS, UNIX(SunOS/..), VMS

    *   executables:

    *   reference: Boehnke (1986), "Estimating the power of a proposed linkage study: a practical computer simulation approach", American Journal of Human Genetics, 39(4):513-527

    *   Ploughman, Boehnke (1989), "Estimating the power of a proposed linkage study for a complex genetic trait", American Journal of Human Genetics, 44(4):543-551

    *   online documentation (version 4.1, april 1990)

*   SIMM

    *   full name:

    *   version:

    *   description: A gene dropping simulation software

    *   authors: Mathieu Lemire (email:mlemire@sdf.lonestar.org)

    *   web/ftp: http://mlemire.freeshell.org/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Lemire, Roslin, Laprise, Hudson, Morgan (2004), "Transmission-ratio distortion and allele sharing in affected sib pairs: a new linkage statistic with reduced bias, with application to chromosome 6q25.3", American Journal of Human Genetics, 75:571-586.

*   SIMPED

    *   full name:

    *   version:

    *   description: SimPed is a program that quickly generates haplotypes and/or genotype data for a large number of marker loci (>20,000) for pedigrees of virtually any size and complexity. Haplotypes and/or genotypes are generated using user specified genetic map distances and haplotypes and/or allele frequencies.

    *   authors: SM Leal, K Yan, B Müller-Myhsok

    *   web/ftp: http://www.hgsc.bcm.tmc.edu/genemapping

    *   source code language: C

    *   operating systems: UNIX, MS-Window, Linux

    *   executables:

    *   reference: Leal, Yan, Muller-Myhsok (2005), "SimPed: a simulation program to generate haplotype and genotype data for pedigree structures", Human Heredity, 60(2):119-122.

*   SIMPLE

    *   full name: Sequential Imputation for MultiPoint Linkage Estimation

    *   version:

    *   description: Calculates linkage statistics, such as lod scores and NPL statistics by Sequential Imputation.

    *   authors: S Lin, Z Skrivanek, M Irwin (Ohio State Univ)

    *   web/ftp: http://www.stat.ohio-state.edu/~statgen/SOFTWARE/SIMPLE/

    *   source code language: C

    *   operating systems: UNIX(Solaris)

    *   executables: simple, scan

    *   reference: M Irwin, N Cox, A Kong (1994), "Sequential imputation for multilocus linkage analysis", Proceedings of the National Academy of Sciences , 91:1684-1688

    *   Z Skrivanek, S Lin, ME Irwin (2003), "Linkage Analysis with Sequential Imputation", Genetic Epidemiology, 25:25-35.

    *   S Lin, Z Skrivanek, M Irwin (2003), "Haplotyping using SIMPLE - caution on ignoring interference", Genetic Epidemiology, 25:384-387.

*   SIMLA 3

    *   full name:

    *   version: 3.2, 3.3 with graphic user interface

    *   description: SIMLA is a simulation program that generates data sets of families for use in linkage and association studies. SIMLA_3.2 is a major upgrade to versions 2.3 and 3.0 that provides the ability to simulate two disease loci and two environmental covariates. Gene-gene and gene-environment interactions may also be simulated which jointly determine the disease risk of all pedigree members. | The SIMLA3.3 with Graphical User Interface (GUI) is designed for users to set up the SIMLA parameters conveniently. The GUI assists in creating a SIMLA control file based on the parameters specified on the GUI panels. It includes the SIMLA 3.2 program version.

    *   authors: email: simla@chg.duhs.duke.edu

    *   web/ftp: http://www.chg.duke.edu/research/simla30.html

    *   http://wwwchg.duhs.duke.edu/research/simla3_3gui.html

    *   source code language: Java, R

    *   operating systems: UNIX(Solaris), Linux, MS-Windows

    *   executables:

    *   reference: Schmidt, Hauser, Martin, Schmidt (2005), "Extension of the SIMLA package for generating pedigrees with complex inheritance patterns: Environmental covariates, gene-gene and gene-environment interaction", Statistical Applications in Genetics and Molecular Biology, 4(1):15.

*   SIMULAPLOT

    *   full name:

    *   version: 1.0

    *   description: SIMLAPLOT is a tool designed to help visualize the joint effect of genes and continuous environmental covariates on complex human disease simulation models. SIMLAPLOT takes as input the set of genetic model parameters used to generate the data, including risk ratios for genotypes (G), a normally distributed environmental covariate (E) and their interaction (GxE), or the risk ratio for a continuous covariate that is itself influenced by a quantitative trait locus (QTL) and thus follows a mixture normal distribution. It is also possible to use SIMLAPLOT to plot observed data distributions, if estimates of these parameters can be provided.

    *   authors:

    *   web/ftp: http://www.chg.duke.edu/research/simlaplot.html

    *   source code language: R

    *   operating systems:

    *   executables:

    *   reference:

*   SIMULATE

    *   full name:

    *   version:

    *   description: a program to simulate genotypes in family members for a map of linked markers unlinked to a given affection status locus. the output is ready for analysis with UNKNOWN, ISIM, LSIM, or MSIM of the SLINK package.

    *   authors: Joseph Terwilliger, http://linkage.cpmc.columbia.edu/

    *   web: http://www.helsinki.fi/~tsjuntun/linkage/analyze/

    *   source code language: Pascal

    *   operating systems: MS-DOS, VMS, UNIX

    *   executables:

    *   reference: online documentation

*   SIMUPOP

    *   full name:

    *   version: 0.6.0 (Jan 2005)

    *   description: simuPOP is a forward-based population genetics simulation program. It is capable of simulating very complex evolution processes on large (think of millions) populations. Major features include variable population size; many built-in and hybrid (write in python) mutation, migration, selection models. simuPOP can be extended in Python so there is no limit on what you can do with it.

    *   authors: Bo Peng (email:bpeng@rice.edu)

    *   web/ftp: http://bp6.stat.rice.edu:8080/simuPOP/

    *   http://simupop.sourceforge.net/

    *   source code language: C++ (provided as Python libraries)

    *   operating systems:

    *   executables:

    *   reference: Peng, Kimmel (2005), "simuPOP: a forward-time population genetics simulation environment", Bioinformatics, 21(18):3686-3687.

*   SIMWALK

    *   full name:

    *   alias: SIMWALK/SIMCROSS

    *   version: SIMCROSS: June 1995; SIMWALK2: 2.91 (December 2004)

    *   description: programs for generating optimal haplotype configurations on general pedigrees using a likelihood-based approach to correctly take intermarker recombination fractions into account. simcross ignores untyped parts of the pedigree, and it uses simulated annealing. simwalk combines simulated annealing with random walk method.

    *   authors: Eric Sobel (UCLA. email: simwalk@genetics.ucla.edu (in collaboration with Kenneth Lange, Jeffrey O'Connell, and Daniel E. Weeks)

    *   web/ftp: http://www.genetics.ucla.edu/software/simwalk

    *   registration page at: http://www.genetics.ucla.edu/software/registration or, http://watson.hgen.pitt.edu/register/

    *   source code language: FORTRAN77

    *   operating systems: UNIX(SunOS/Solaris)

    *   executables:

    *   reference: Weeks, Sobel, O'Connell, Lange (1995), "Computer programs for multilocus haplotyping of general pedigrees", American Journal of Human Genetics, 56(6):1506-1507

    *   Sobel, Lange, O'Connell, Weeks (1996), in Genetic Mapping and DNA Sequencing , Vol 81 in The IMA Volumes in Mathematics and its Applications, eds TP Speed, MS Waterman (Springer-Verlag), pages 49-63.

    *   Sobel, Lange (1996), "Descent graphs in pedigree analysis: applications to haplotyping, location scores, and marker sharing statistics", American Journal of Human Genetics, 58(6):1323-1337

    *   Sobel, Sengul, Weeks (2001), Human Heredity, 52:121-131.

    *   Sobel, Papp, Lange (2002), "Detection and integration of genotyping errors in statistical genetics", American Journal of Human Genetics, 70(2):496-508. [ html]

    *   SIMWALK manual

    *   FAQ-O-MATIC: http://watson.hgen.pitt.edu/fom-serve/cache/20.html

*   SKAT

    *   full name: SNP-set (Sequence) Kernel Association Test

    *   version:

    *   description: SKAT is a SNP-set (e.g., a gene or a region) level test for association between a set of rare (or common) variants and dichotomous or quantitative phenotypes. SKAT aggregates individual score test statistics of SNPs in a SNP set and efficiently computes SNP-set level p-values, e.g. a gene or a region level p-value, while adjusting for covariates, such as principal components to account for population stratification. SKAT also allows for power/sample size calculations for designing for sequence association studies.

    *   authors: Xihong Lin, Seunggeun Lee (Harvard Univ)

    *   web/ftp: http://www.hsph.harvard.edu/research/skat/index.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Wu, Lee, Cai, Li, Boehnke, Lin (2011), "Rare variant association testing for sequencing data using the sequence kernel association test (SKAT)", American Journal of Human Genetics, 89:82-93.

    *   Wu, Kraft, Epstein, Taylor, Chanock, Hunter, Lin (2010), "Powerful SNP set analysis for case-control genomeWide association studies", American Journal of Human Genetics , 86:929-942.

*   SLINK (see also FASTSLINK)

    *   full name:

    *   version:

    *   description:

    *   authors: Daniel E. Weeks, Jurg Ott

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: Pascal

    *   operating systems: MS-DOS

    *   executables: slink, isim, lsim, msim

    *   reference: Ott (1989), Proceedings of the National Academy of Sciences , 86:4175-4178.

    *   Weeks, Ott, Lathrop (1990), American Journal of Human Genetics, 47(suppl):A204.

    *   online documentation

    *   FAQ-O-MATIC: http://watson.hgen.pitt.edu/fom-serve/cache/20.html

*   SMOOTH

    *   full name:

    *   version: 2008

    *   description: The construction of accurate linkage maps is not so much depending on the quality of the mapping software, but mostly on the marker data quality. Missing values and scoring errors can severely influence the calculated marker order. The software tool SMOOTH recognises and removes the most unrealistic data points. This software was used to construct the 10,000 marker potato map. The removal of improbable data point is a good medicine for linkage maps, that is not easily overdosed. One error is more harmfull than ten missing values. The software was never intended as user-friendly software. In these days it would be more useful to re-do the programming of the pascal source code into a perl script. Anyone who takes the initiative to generate such a script is welcomed to contact the authors. SMOOTH works best in close cooperation with mapping algorithm RECORD

    *   authors: Hans van Os, Herman van Eck (Laboratory for Plant Breeding, Wageningen University, email: herman.vaneck@wur.nl)

    *   web/ftp: http://www.plantbreeding.wur.nl/UK/software_smooth.html

    *   source code language: Pascal

    *   operating systems: MS-DOS

    *   executables: smooth7.exe

    *   reference: Van Os, Stam, Visser, Van Eck (2005), "SMOOTH: a statistical method for successful removal of genotyping errors from high-density genetic linkage data", Theoretical and Applied Genetics, 112(1):187-194

    *   Van Os, Stam, Visser, Van Eck (2005), "RECORD: A novel method for ordering loci on a genetic linkage map", Theoretical and Applied Genetics, 112(1):30-40

    *   Van Os, Andrzejewski, Bakker, Barrena, Bryan, Caromel, Ghareeb, Isidore, de Jong, van Koert, Lefebvre, Milbourne, Ritter, van der Voort, Rousselle-Bourgeois, van Vliet, Waugh, Visse, Bakke, Van Eck (2006), "Construction of a 10,000-marker ultradense genetic recombination map of potato: Providing a framework for accelerated gene isolation and a genomewide physical map", Genetics, 173(2):1075-1087

*   SNAP

    *   full name: Suite of Nucleotide Analysis Programs

    *   version:

    *   description: The exploration of DNA sequence variation for making inferences on evolutionary processes in populations requires the coordinated implementation of a Suite of Nucleotide Analysis Programs (SNAP), each bound by specific assumptions and limitations. We have developed a workbench tool to make existing population genetic software more accessible and to facilitate the integration of new tools for analyzing patterns of DNA sequence variation, within a phylogenetic context. Collectively, SNAP tools can serve as a bridge between theoretical and applied population genetic analysis.

    *   authors: David Aylor, Eric Price, Ignazio Carbone (North Carolina State Univ)

    *   web/ftp: http://www.cals.ncsu.edu/plantpath/people/faculty/carbone/snap.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Aylor, Price, Carbone (2006), "SNAP: Combine and Map modules for multilocus population genetic analysis", Bioinformatics, 22(11):1399-1401.

*   SNAP (2)

    *   full name: SNp Annotation and Proxy search

    *   version: 1.0 (August 2008)

    *   description: SNAP finds proxy SNPs based on linkage disequilibrium, physical distance and/or membership in more than 15 selected commercial genotyping arrays. Pair-wise linkage disequilibrium is pre-calculated based on phased genotype data from the International HapMap Project . Information about the genotyping arrays is based on data published by the vendors. SNAP also optionally returns current gene and SNP annotations, and can be used to generate graphical plots of LD structure, recombination rate and gene annotation.

    *   authors: Robert Handsaker (email: handsake@broad.mit.edu), Andrew Johnson (email: johnsonad2@nhlbi.nih.gov), Sara Pulit, Marcia Nizzari, Chris O'Donnell, Paul de Bakker (email: debakker@broad.mit.edu)

    *   web/ftp: http://www.broad.mit.edu/mpg/snap/

    *   source code language: R, Oracle

    *   operating systems:

    *   executables:

    *   reference:

*   SNAP (3)

    *   full name: Simulation of sNp haplotype data And Phenotypic traits

    *   version:

    *   description: The SNaP program can be used to generate SNP haplotype sequence data of unrelated individuals and nuclear families with a fixed or random number of children.

    *   authors: Michael Nothnagel (email: nothnagel@medinfo.uni-kiel.de)

    *   web/ftp: http://capella.uni-kiel.de/snap/snap.htm

    *   source code language: C

    *   operating systems: Linux, UNIX(Solaris), MS-Windows

    *   executables:

    *   reference: Am Soc Hum Genet annual meeting (2002) Nothnagel (2002), "Simulation of LD block-structured SNP haplotype data and its use for the analysis of case-control data by supervised learning methods", Am J. Hum Genet, 71(4)(suppl), A2363.

*   SNIPPEEP

    *   full name:

    *   version:

    *   description: SnipPeep is an interactive graphic interface to visualise results from whole genome genotyping. It allows one to visualise single subjects and groups of subjects, and provides a direct connection with the UCSC Genome Browser.

    *   authors: Roberto Toro, (email: rto@pasteur.fr), Thomas Bourgeron (email: thomasb@pasteur.fr)

    *   web/ftp: http://snippeep.sourceforge.net/

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference:

*   SNPALYZE

    *   full name:

    *   version: 6.0 (December 2006)

    *   description:

    *   authors: dynacom.co.jp

    *   web/ftp: http://www.dynacom.co.jp/e/products/package/snpalyze/index.html

    *   source code language:

    *   operating systems: MS-Windows (98/Me/NT4.0/2000/XP)

    *   executables:

    *   reference: Shimo-Onoda, Tanaka, Furushima, Nakajima, Toh, Harata, Yone, Komiya, Adachi, Nakamura, Fujimiya, Inoue(2002), "Akaike's information criterion for a measure of linkage disequilibrium", J Hum Genet, 47(12): 649-655.

*   SNP ASSISTANT

    *   full name:

    *   version: 1.0.11 (April 2004)

    *   description: SNP data managing, import & export from linkage format, data validation, pairwise LD calculation and visualisation, case-control and TDT tests, visual comparison of two datasets, relationships testing. Suitable for large projects.

    *   authors: Biodata Ltd., Estonia (snpassistant@biodata.ee)

    *   web/ftp: http://www.biodata.ee/SNPassistant.htm (30 days trial version)

    *   source code language: C++

    *   operating systems: MS-Windows(95 and newer)

    *   executables:

    *   reference: online manual: http://www.biodata.ee/sa/SNP_Assistant_Manual_1.0.7.pdf

*   SNP CHART

    *   full name:

    *   version:

    *   description: The purpose of this software is to generate visual patterns that can be easily interpreted as a single specific Single Nucleotide Polymorphism (SNP) genotype, based on reiterative pattern recognition from validated SNP data

    *   authors: Scott Tebbutt (email: scott.tebbutt@hli.ubc.ca)

    *   web/ftp: http://www.flintbox.com/public/project/500/

    *   source code language: Java

    *   operating systems: web-based

    *   executables:

    *   reference: Tebbutt, Opushnyev, Tripp, Kassamali, Alexander, Andersen (2005), "SNP Chart: an integrated platform for visualization and interpretation of microarray genotyping data", Bioinformatics, 21(1):124-127.

*   SNPEM

    *   full name: SNP Expectation Maximization

    *   version:

    *   description: EM algorithm estimations of SNP haplotypes

    *   authors:

    *   web/ftp: http://polymorphism.ucsd.edu/snpem/

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference:

*   SNPFILE

    *   full name:

    *   version:

    *   description: SNPFile is a library and API for manipulating large SNP datasets with associated meta-data, such as marker names, marker locations, individuals' phenotypes, etc. in an I/O efficient binary file format. In its core, SNPFile assumes very little about the metadata associated with markers and individuals, but leaves this up to application program protocols.

    *   authors:

    *   web/ftp: http://www.daimi.au.dk/~mailund/SNPFile/

    *   source code language: C++

    *   operating systems: Linux, UNIX

    *   executables:

    *   reference: Nielsen, Mailund (2008), "SNPFile -- A software library and file format for large scale association mapping and population genetics studies", BMC Bioinformatics, 9:526.

*   SNPHAP

    *   full name:

    *   version: 1.3.1

    *   description: a program for estimating frequencies of haplotypes of large numbers of diallelic markers from unphased genotype data from unrelated subjects

    *   Authors: David Clayton (Cambridge Institute, david.clayton@cimr.cam.ac.uk

    *   web/ftp: http://www-gene.cimr.cam.ac.uk/clayton/software/

    *   source code language: C,C++

    *   operating systems:

    *   executables:

    *   reference:

*   SNPHARVESTER

    *   full name:

    *   version:

    *   description: a powerful tool for detecting epistatic interactions in genome-wide association studies

    *   authors:

    *   web/ftp: http://bioinformatics.ust.hk/SNPHarvester.html

    *   source code language: java

    *   operating systems:

    *   executables:

    *   reference: Yang, He, Wan, Yang, Xue, Yu (2009), "SNPHarvester: a filtering-based approach for detecting epistatic interactions in genome-wide association studies", Bioinformatics, 25:504-511.

*   SNP HITLINK

    *   full name: SNP HIgh-Throughput LINKage analysis system

    *   version: 5.1 (May 2010)

    *   description: SNP HiTLink is a computer program providing a useful pipeline to directly connect SNP data and linkage analysis program. SNP HiTLink currently supports the data from SNP chips provided by Affymetrix (Mapping 100k/500k array set, Genome-Wide Human SNP array 5.0/6.0) and Illumina (recently supported), carrying out typical linkage analysis programs of MLINK (FASTLINK/ LINKAGE package), Superlink, Merlin and Allegro.

    *   authors: info@dynacom.co.jp

    *   web/ftp: http://www.dynacom.co.jp/u-tokyo.ac.jp/snphitlink/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Fukuda, Nakahara, Date, Takahashi, Goto, Miyashita, Kuwano, Adachi, Nakamura, Tsuji (2009), "SNP HiTLink: a high-throughput linkage analysis system employing dense SNP data", BMC Bioinformatics, 10:121.

*   SNP-HWE

    *   full name:

    *   version:

    *   description: a fast exact Hardy-Weinberg Equilibrium test for SNPs

    *   authors: Janis Wigginton (email: "a href="mailto:wiggie@umich.edu"> wiggie@umich.edu) and Goncalo Abecasis

    *   web/ftp: http://www.sph.umich.edu/csg/abecasis/Exact/index.html

    *   source code language: Fortran, C,C++, R

    *   operating systems:

    *   executables:

    *   reference: Wigginton, Cutler, Abecasis (2005), "A note on exact tests of Hardy-Weinberg equilibrium", American Journal of Human Genetics, 76(5):887-893\. [ html]

*   SNPLINK

    *   full name:

    *   version:

    *   description: Multipoint linkage analysis of densely distributed SNP data incorporating automated linkage disequilibrium removal. SNPLINK requires these other programs installed on the system: MERLIN (used for nonparametric analysis), ALLEGRO (used for parametric analysis), R and PERL, all are freely available.

    *   authors: Emily Webb (The Institute for Cancer Research, email: emily.webb@icr.ac.uk)

    *   web/ftp: http://www.icr.ac.uk/cancgen/molgen/MolPopGen_Bioinformatics.htm

    *   source code language: Perl

    *   operating systems: UNIX

    *   executables:

    *   reference: Webb, Sellick, Houlston (2005), "SNPLINK: multipoint linkage analysis of densely distributed SNP data incorporating automated linkage disequilibrium removal", Bioinformatics, 21:3060-3061.

*   SNPMSTAT

    *   full name: SNP Missing data STATistics

    *   version: 4.0

    *   description: SNPMStat is a command-line program for the statistical analysis of SNP-disease association in case-control/cohort/cross-sectional studies with potentially missing genotype data. SNPMStat allows the user to estimate or test SNP effects and SNP-environment interactions by maximizing the (observed-data) likelihood that properly accounts for phase uncertainty, study design and gene-environment dependence. For SNPs without missing data, the program performs the standard association analysis. For typed SNPs with missing data or untyped SNPs, the program performs the maximum-likelihood analysis.

    *   authors: Danyu Lin's group (Univ North Carolina)

    *   web/ftp: http://www.bios.unc.edu/~lin/software/SNPMStat/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Lin, Hu, Huang (2008), "Simple and efficient analysis of SNP-disease association with missing genotype data", American Journal of Human Genetics, 82:444-452.

*   SNPP

    *   full name: Single Nucleotide Polymorphism Processor

    *   version: 1.0 (2003)

    *   description: a dynamic general database management system to manage high-throughput SNP genotyping data. It provides several functions, including data importing with comparison, Mendelian inheritance check within pedigrees, data compiling and exporting. Furthermore, SNPP may generate files for repeat genotyping and transform them into files that can be executed by a liquid handling system.

    *   authors: Lanjuan Zhao ( lanjuanzhao@creighton.edu), Miaoxin Li ( limx54@163.com), Yanfang Guo, Fuhua, Xu, Jinlong Li and Hongwen Deng.

    *   web/ftp: http://orclinux.creighton.edu/snpp/

    *   source code language: Java

    *   operating systems: Windows, Linux, Solaris, MacOS

    *   executables:

    *   reference: Zhao, Li, Guo, Xu, Li, Deng (2005), "SNPP: automating large-scale SNP genotype data management", Bioinformatics, 21:266-268

*   SNP.PLOTTER (see R/SNP.PLOTTER)

*   SNPSTATS

    *   full name: SNP STATisticS

    *   version:

    *   description: SNPStats is a simple, ready-to-use software which has been designed to analyze genetic-epidemiology studies of association using SNPs.

    *   authors: Victor Moreno (email: v.moreno@ico.scs.es) Xavier Sole (email: x.sole@ico.scs.es)

    *   web/ftp: http://bioinfo.iconcologia.net/snpstats/

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference: Sole, Guino, Valls, Iniesta, Moreno (2006), "SNPStats: a web tool for the analysis of association studies", Bioinformatics, 22(15):1928-1929.

*   SNPTEST

    *   full name:

    *   version: 1.1.5 (May 2008)

    *   description: SNPTEST is a program for the analysis of single SNP association in genome-wide studies. The tests implemented can cater for binary (case-control) and quantitative phenotypes, can condition upon an arbitrary set of covariates and properly account for the uncertainty in genotypes. The program is designed to work seamlessly with the output of both the genotype calling program CHIAMO, the genotype imputation program IMPUTE and the program GTOOL. This program was used in the analysis of the 7 genome-wide association studies carried out by the Wellcome Trust Case-Control Consortium (WTCCC).

    *   authors: Jonathan Marchini

    *   web/ftp: http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   SOAP

    *   full name: Short Oligonucleotide Analysis Package

    *   version:

    *   description: SOAP provides full solution to next generation sequencing data analysis. Currently, it consists of a new alignment tool (SOAPaligner/soap2), a re-sequencing consensus sequence builder (SOAPsnp), an indel finder (SOAPindel), a structural variation scanner (SOAPsv) and a de novo short reads assembler (SOAPdenovo).

    *   authors:

    *   web/ftp: http://soap.genomics.org.cn/

    *   http://soap.genomics.org.cn/soap3.html (SOAP3)

    *   http://soap.genomics.org.cn/soapsnp.html (SOAP SNP)

    *   http://soap.genomics.org.cn/soapindel.html (SOAP indel)

    *   http://soap.genomics.org.cn/SOAPsv.html (SOAP structural variants)

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Li, et al. (2008), "SOAP: short oligonucleotide alignment program", Bioinformatics, 24:713-714.

    *   Li, Li, Fang, et al. (2009), "SNP detection for massively parallel whole-genome resequencing", Genome Research, 19:1124-1132.

    *   Li, Yu, Li, Lam, Yiu, Kristiansen, Wang (2009), "SOAP2: an improved ultrafast tool for short read alignment", Bioinformatics, 25(15):1966-1967.

*   SOLAR

    *   full name: Sequential Oligogenic Linkage Analysis Routines

    *   version: 4.0.7 (December 2006).

    *   description: OLAR is a flexible and extensive software package for genetic variance components analysis, including linkage analysis, quantitative genetic analysis, and covariate screening. Operations are included for calculation of marker-specific or multipoint identity-by-descent (IBD) matrices in pedigrees of arbitrary size and complexity, and for linkage analysis of quantitative traits which may involve multiple loci (oligogenic analysis), dominance effects, and epistasis.

    *   Authors: John Blangero (john@darwin.sfbr.org), Kenneth Lange, Laura Almasy (almasy@darwin.sfbr.org), Tom Dyer (tdyer@darwin.sfbr.org), Charles Peterson (charlesp@darwin.sfbr.org).

    *   web/ftp: http://www.sfbr.org/Departments/genetics_detail.aspx?p=37

    *   source code language: FORTRAN, C, C++, TCL

    *   operating systems: UNIX(Solaris 7-10/Solaris x86 8-9/Digital Unix 4.0e/SG Irix), Linux, MacOS, MS-Windows

    *   executables: solar, ibdprep, ibdmat, relate, multipnt

    *   reference: Blangero, Almasy (1996), "SOLAR: sequential oligogenic linkage analysis routines", Population Genetics Laboratory Technical Report No. 6, Southwest Foundation for Biomedical Research, San Antonio, TX 78228.

    *   Almasy, Blangero (1998), "Multipoint quantitative-trait linkage analysis in general pedigrees", American Journal of Human Genetics, 62(5):1198-1211 (1998)

*   SPAM

    *   full name: Statistics Program for Analysing Mixtures

    *   version: 3.7b

    *   description: SPAM estimates the relative contributions of discrete populations to a mixture sample, solving what is commonly referred to in fisheries as the mixed stock analysis or genetic stock identification problem.

    *   authors:

    *   web/ftp: http://www.genetics.cf.adfg.state.ak.us/software/spampage.php

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Debevec, Gates, Masuda, Pella, Reynolds, Seeb (2000), "SPAM (version 3.2): statistics program for analyzing mixtures", Journal of Heredity, 91:509-511.

*   SPECTRAL-GEM (see R/SPECTRAL-GEM)

*   SPERM

    *   full name:

    *   version:

    *   description: analysis of sperm typing data.

    *   authors: Kenneth Lange (UCLA), Laura Lazzeroni

    *   web/ftp: registration page at: http://www.biomath.medsch.ucla.edu/faculty/klange/software.html

    *   source code language: FORTRAN77

    *   operating systems:

    *   executables:

    *   reference: Lazzeroni, Arnheim, Schmitt, Lange (1994), "Multipoint mapping calculations for sperm-typing data", American Journal of Human Genetics, 55(3):431-436

*   SPERMSEG

    *   full name:

    *   version:

    *   description: analysis of segregation in single-sperm data

    *   Authors: MS McPeek ( mcpeek@galton.uchicago.eduUniv Chicago)

    *   web/ftp: http://galton.uchicago.edu/~mcpeek/software/spermseg

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: MS McPeek (1999), "SPERMSEG: analysis of segregation distortion in single-sperm data", American Journal of Human Genetics, 65(4):1195-1197\. [ html]

*   SPIP

    *   full name: Simulate Pedigree In Population

    *   version:

    *   description: Simulate pedigrees and genetic data in age-structured populations

    *   authors: Eric C Anderson, K.K. Dunham (Southwest Fisherie Science Center)

    *   web/ftp: http://swfsc.noaa.gov/staff.aspx?id=740

    *   http://swfsc.noaa.gov/textblock.aspx?Division=FED&id=3434

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Anderson, Dunham (2005), "SPIP 1.0: a program for simulating pedigrees and genetic data in age-structured populations", Molecular Ecology Notes, 5(2):459-461.

*   SPLAT

    *   full name: Sib Pair Linkage Analysis Testing

    *   version: 1.0 (2006)

    *   description: SPLAT can calculate virtually any linkage test statistic under several sib pair study designs: affected, discordant, unaffected, and pairs defined by threshold values for quantitative traits, such as extreme discordant sib pairs. It uses the EM algorithm to compute maximum likelihood estimates of sharing (subject to any user-specified domain restrictions or null hypotheses) and then plots lod scores versus chromosomal position. It includes a novel grid scanning capability that enables simultaneous visualization of multiple test statistics. Phenotype definitions can be modified without recalculating inheritance vectors, thereby providing considerable analytical flexibility.

    *   authors: Katarzyna Adamska (email: adamska@elf.ii.uj.edu.pl), David Poznik (email: david.poznik@joslin.harvard.edu), Xin Xu (email: xin_xu@harvard.edu)

    *   web/ftp: http://www.joslinresearch.org/LabSites/Krolewski/splat/

    *   source code language: C++, Qt

    *   operating systems: UNIX(SunOS),Linux, MacOS, MS-Windows(2000/XP)

    *   executables:

    *   reference: Poznik, Adamska, Xu, Krolewski, Rogus (2006), "A novel framework for sib pair linkage analysis", American Journal of Human Genetics, 78(2):222-230\. [ html]

*   SPLINK

    *   full name: affected Sib Pairs LINKage analysis

    *   version: 1.09 (Nov 1999)

    *   description: linkage analysis using affected sib pairs

    *   authors: David Clayton (Cambridge Institute, david.clayton@cimr.cam.ac.uk

    *   web/ftp: http://www-gene.cimr.cam.ac.uk/clayton/software/

    *   source code language: c++

    *   operating systems: UNIX(SunOS/..), MS-DOS

    *   executables: splink, ped2spl

    *   reference: Holmans (1993), "Asymptotic properties of affected-sib-pair linkage analysis", American Journal of Human Genetics, 52(2):362-374

    *   Holmans and Clayton (1995), submitted.

    *   online manual

*   SPREG

    *   full name: Secondary Phenotype REGression analysis

    *   version: 2.0

    *   description: SPREG is a computer program for performing regression analysis of secondary phenotype data in case-control association studies. Secondary phenotypes are quantitative or qualitative traits other than the case-control status. Because the case-control sample is not a random sample of the general population, standard statistical analysis of secondary phenotype data can yield very misleading results. SPREG implements valid and efficient statistical methods.

    *   authors: Danyu Lin, Donglin Zeng (Univ North Carolina)

    *   web/ftp: http://www.bios.unc.edu/~lin/software/SPREG/

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Lin, Zeng (2009), "Proper analysis of secondary phenotype data in case-control association studies", Genetic Epidemiology, 33:256-265.

*   SQTDT/SPDT

    *   full name:

    *   version:

    *   description: semiparametric family-based association

    *   authors: Guoqing Diao (email: gdiao@gmu.edu), Danyu Lin

    *   web/ftp: http://www.bios.unc.edu/~lin/software/SQTDT/

    *   source code language:

    *   operating systems: Linux

    *   executables:

    *   reference: Diao, Lin (2006), "Improving the power of association tests for quantitative traits in family studies", Genetic Epidemiology, 30:301-313.

*   SQTL

    *   full name: Semiparametric QTL mapping in general pedigrees

    *   version:

    *   description:

    *   authors: Guoqing Diao (email: gdiao@gmu.edu), Danyu Lin

    *   web/ftp: http://www.bios.unc.edu/~lin/software/SQTL/

    *   source code language:

    *   operating systems: Linux

    *   executables:

    *   reference: Diao, Lin (2005), "A powerful and robust method for mapping quantitative trait loci in general pedigrees", American Journal of Human Genetics, 77(1):97-111\. [ html]

*   SSAHASNP

    *   full name: Sequence Search and Alignment by Hashing Algorithm for SNP detection

    *   version:

    *   description: ssahaSNP is a polymorphism detection tool. It detects homozygous SNPs and indels by aligning shotgun reads to the finished genome sequence. Highly repetitive elements are filtered out by ignoring those kmer words with high occurrence numbers. For those less repetitive or non-repetitive reads, we place them uniquely on the reference genome sequence and find the best alignment according to the pair-wise alignment score if there are multiple seeded regions. From the best alignment, SNP candidates are screened, taking into account the quality value of the bases with variation as well as the quality values in the neighbouring bases, using neighbourhood quality standard (NQS). For insertions/deletions, we check if the same indel is mapped by more than one read, ensuring the detected indel with high confidence.

    *   authors: Adam Spargo, Zemin Ning (Wellcome Trust Sanger Institute)

    *   web/ftp: http://www.sanger.ac.uk/resources/software/ssahasnp/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   START

    *   full name:

    *   version:

    *   description: Finds starting points for MCMC analysis performed on large, complex pedigrees and polymorphic markers.

    *   authors: Y Luo, S Lin (Ohio State Univ)

    *   web/ftp: http://www.stat.ohio-state.edu/~statgen/SOFTWARE/START/

    *   source code language:

    *   operating systems: UNIX(Solaris), Linux

    *   executables:

    *   reference: Y Luo, S Lin (2003), "Finding starting points for Markov chain Monte Carlo analysis of genetic data from large and complex pedigrees", Genetic Epidemiology, 25:14-24.

*   STEPC

    *   full name: STatistical Explanation for Positional Cloning

    *   version:

    *   description:

    *   authors: Lei Sun, Nancy COx, Mary Sara McPeek

    *   web/ftp: http://hg-wen.uchicago.edu/software.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Sun, Cox, McPeek (2002) "A statistical method for identification of polymorphisms that explain a linkage result", American Journal of Human Genetics, 70(2):399-411\. [ html]

*   STEPWISE (see R/STEPWISE)

*   STRAT (see also STRUCTURE)

    *   full name:

    *   version: 1.1

    *   description: a companion program to structure. This is a structured association method, for use in association mapping, enabling valid case-control studies even in the presence of population structure.

    *   authors: Jonathan Pritchard (email: pritch@genetics.uchicago.edu)

    *   web/ftp: http://pritch.bsd.uchicago.edu/software/STRAT.html

    *   source code language:

    *   operating systems: MS-DOS, MS-Windows,UNIX(Solaris), Linux

    *   executables:

    *   reference: JK Pritchard, M Stephens, NA Rosenberg, P Donnelly (2000), "Association mapping in structured populations", American Journal of Human Genetics, 67(1):170-181\. [ html]

*   STRUCTURE (see also STRAT)

    *   full name:

    *   version: 2.1 (August 2004)

    *   description: using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. It can be applied to most of the commonly-used genetic markers, including microsatellites, RFLPs and SNPs.

    *   authors: Jonathan Pritchard (email: pritch@genetics.uchicago.edu)

    *   web/ftp: http://pritch.bsd.uchicago.edu/software.html

    *   source code language:

    *   operating systems: MS-DOS, MS-Windows, UNIX(Solaris), Linux

    *   executables:

    *   reference: JK Pritchard, M Stephens, P Donnelly (2000), "Inference of population structure using multilocus genotype data", Genetics 155: 945-959.

*   SUMSTAT

    *   full name:

    *   version:

    *   description: assess the joint disease association of multiple unlinked SNPs via sums of SNP specific test statistics. Genome-wide significance levels are obtained by per mutation analysis.

    *   authors: Josephine Hoh, Jurg Ott (email: ottjurg@yahoo.com)

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: Free Pascal

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference: Hoh, Wille, Ott (2001), "Trimming, weighting, and grouping SNPs in human case-control association studies", Genome Research, 11:2115-2119

*   SUP

    *   full name: Slink Utility Program

    *   version: 060629

    *   description: an extension to SLINK/FastSLINK to allow more marker loci to be simulated in pedigrees conditional on trait values and in linkage equilibrium or disequilibrium with a trait locus.

    *   authors: Mathieu Lemire (email: mlemire@sdf.lonestar.org)

    *   web/ftp: http://mlemire.freeshell.org/software.html

    *   source code language: C,C++

    *   operating systems: UNIX, LINUX, CYGWIN

    *   executables:

    *   reference: Lemire (2006), "SUP: An extension to SLINK to allow a larger number of marker loci to be simulated in pedigrees conditional on trait values", BMC Genetics, 7:40.

*   SUPERLINK

    *   full name:

    *   version: 1.6 (March 2006)

    *   description: the program performs exact linkage analysis with the same input-output relationships as in standard genetic linkage programs such as LINKAGE, FASTLINK, VITESSE, but can run larger files than previous programs.

    *   authors: Ma'ayan Fishelson (email: fmaayan@cs.technion.ac.il)

    *   web/ftp: http://bioinfo.cs.technion.ac.il/superlink/

    *   source code language:

    *   operating systems: UNIX, MS-Windows, Linux, MacOS X

    *   executables:

    *   reference: M Fishelson, D Geiger (2002), Bioinformatics, 18(suppl 1):S189-S198.

*   SVA

    *   full name: Sequence Variant Analyzer

    *   version: v1.10 (March 2011)

    *   description: SVA is a software tool that assigns a predicted biological function to variants identified in next-generation sequencing studies and provides a browser to visualize the variants in their genomic contexts. SVA also provides for flexible interaction with software implementing variant association tests allowing users to consider both the bioinformatic annotation of identified variants and the strength of their associations with studied traits.

    *   authors: Dongliang Ge, David Goldstein (Duke)

    *   web/ftp: http://www.svaproject.org/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Ge,Ruzzo, Shianna,He,Pelak, Heinzen, Need, Cirulli, Maia, Dickson, Zhu,Singh, Allen, Goldstein (2011), "SVA: software for annotating and visualizing sequenced human genomes", Bioinformatics, 27:1998-2000.

*   SVCC

    *   full name: Semiparametric Variance-Component for Censored data

    *   version:

    *   description:

    *   authors: Guoqing Diao (email: gdiao@gmu.edu), Danyu Lin

    *   web/ftp: http://www.bios.unc.edu/~lin/software/SVCC/

    *   source code language:

    *   operating systems: Linux

    *   executables:

    *   reference: Diao, Lin (2006), "Semiparametric variance-component models for linkage and association analyses of censored trait data", Genetic Epidemiology, 30:570-581.

*   SWEEP

    *   full name:

    *   version: December 2005

    *   description: SWEEP allows large-scale analysis of haplotype structure in genomes for the primary purpose of detecting evidence of natural selection. Primarily, it uses the Long Range Haplotype test to look for alleles of high frequency with long-range linkage disequilibrium, which suggest the haplotype rapidly rose to high frequency before recombination could break down associations with nearby markers. SWEEP takes phased genotype data as input, detects all haplotype blocks in that data, and then determines the frequency and long-range LD for each allele in each block.

    *   authors: Patrick Varilly, Ben Fry, Oleg Shamovsky, Pardis Sabeti (email: sweep@broad.mit.edu)

    *   web/ftp: http://www.broad.mit.edu/mpg/sweep/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Sabeti, Reich, Higgins, Levine, Richter, Schaffner, Gabriel, Platko, Patterson, McDonald, Ackerman, Campbell, Altshuler, Cooper, Kwiatkowski, Ward, Lander (2002), "Detecting recent positive selection in the human genome from haplotype structure", Nature, 419:832-827.

*   SYZYGY

    *   full name:

    *   version:

    *   description: Syzygy is a targeted sequencing post processing analysis tool that allows: 1. SNP and indel detection; 2\. Allele frequency estimation; 3. Single-marker association test; 4\. Group-wise marker test association; 5\. Experimental QC summary (%dbSNP, Ts/Tv, Ns/S); 6. Power to detect variant.

    *   authors:

    *   web/ftp: http://www.broadinstitute.org/software/syzygy/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Rivas et al. (2011), "Deep resequencing of GWAS loci identifies independent rare variants associated with inflammatory bowel disease", Nature Genetics, in press.

*   TAGGER

    *   full name:

    *   version:

    *   description:

    *   authors: Paul de Bakker (email: debakker@molbio.mgh.harvard.edu, Borad Institute)

    *   web/ftp: http://www.broad.mit.edu/mpg/tagger/

    *   source code language:

    *   operating systems: web-based

    *   executables:

    *   reference: de Bakker, Yelensky, Pe'er, Gabriel, Daly, Altshuler (2005), "Efficiency and power in genetic association studies", Nature Genetics, 37:1217-1223.

*   TAGIMPUTE

    *   full name: TAGging-snp based IMPUTATE

    *   version: 1.0

    *   description: tagIMPUTE is a command-line program for the imputation of untyped SNPs. tagIMPUTE is based on a few flanking SNPs that can optimally predict the SNP under imputation.

    *   authors: Yijuan Hu, Danyu Lin (Univ North Carolina)

    *   web/ftp: http://www.bios.unc.edu/~lin/software/tagIMPUTE/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Hu, Lin (2010), "Analysis of untyped SNPs: maximum likelihood and imputation methods", Genetic Epidemiology, 34:803-815.

*   TAGSNP

    *   full name:

    *   version: 2 (beta)

    *   description:

    *   authors: Daniel O Stram (email: stram@usc.edu, Univ Southern California)

    *   web/ftp: http://www-rcf.usc.edu/~stram/tagSNPs.html

    *   source code language: FORTRAN90

    *   operating systems:

    *   executables:

    *   reference: Stram (2004), "Tag SNP selection for association studies", Genetic Epidemiology, 27(4):365-374.

*   TAGSTER

    *   full name:

    *   version:

    *   description: TAGster is a tool to select, evaluate and visualize LD tag SNPs for single or multiple populations. The input files can be a set of dumped genotype files from International HapMap Project (http://www.hapmap.org/) (Hapmap format) or Seattle SNPs (http://pga.gs.washington.edu/) (Prettybase format). The ouput is a set of LD tag SNPs for single or multiple populations.

    *   authors: Zongli Xu, Norman L. Kaplan, Jack A Taylor

    *   web/ftp: http://www.niehs.nih.gov/research/resources/software/tagster/

    *   source code language: Perl, R

    *   operating systems: Linux, MacOS, MS-Windows

    *   executables:

    *   reference: Xu, Kaplan, Taylor (2007), "TAGster: efficient selection of LD tag SNPs in single or multiple populations", Bioinformatics, 23:3254-3255.

*   TASSEL

    *   full name: Trait Analysis by aSSociation, Evolution, and Linkage

    *   version: 2.0 (January 2006)

    *   description: TASSEL is a software package which performs a variety of genetic analyses. The analyses include association mapping, diversity estimation and calculating linkage disequilibrium. The association analysis between genotypes and phenotypes can be performed by either a general linear model or a mixed linear model. The general linear model now allows users to analyze complex field designs, environmental interactions, and epistatic interactions. The mixed model is specially designed to handle polygenic effects at multiple levels of relatedness including pedigree information. These new analyses should permit association analysis in a wide range plant and animal species.

    *   authors: Peter J Bradbury, Edward S Buckler, Dallas E Kroon, Zhiwu Zhang (email: esb@cornell.edu)

    *   web/ftp: http://sourceforge.net/projects/tassel/

    *   source code language: Java

    *   operating systems: web-based

    *   executables:

    *   reference: Zhiwu Zhang, Peter J Bradbury, Dallas E Kroon, Terry M Casstevens, Edward S Buckler (2006), "TASSEL 2.0: a software package for association and diversity analyses in plants and animals", Plant & Animal Genomes XIV Conference. [[abstract](http://www.intl-pag.org/14/abstracts/PAG14_C012.html)]

*   TDT-AE

    *   full name: Transmission Disequilibrium Test Allowing for Errors

    *   version: 2.01 (July 2004)

    *   description: This program computes a likelihood-based transmission disequilibrium test. The data are genotypes on trios (father, mother, affected child) in which random genotyping errors leading to Mendelian inconsistencies may or may not have occurred. This program computes the TDT-AE statistic on all trios (whether Mendelianly consistent or not) and thereby maintains a correct type I error rate in the presence of random genotyping errors.

    *   Authors: Chad Haynes, Derek Gordon (email: Gordon@dls.rutgers.edu), Xin Liu (for version 1)

    *   web/ftp: ftp://linkage.rockefeller.edu/software/tdtae2

    *   source code language: C

    *   operating systems: MS-Windows (2000/DOS), UNIX(Sun Solaris), LINUX (RedHat)

    *   executables: tdt-ae (PC), tdtae (UNIX and LINUX)

    *   reference: Gordon, Heath, Liu, Ott (2001), "A transmission/disequilibrium test that allows for genotyping errors in the analysis of single-nucleotide polymorphism data", American Journal of Human Genetics, 69(2):371-380\. [ html]

    *   Gordon, Haynes, Johnnidis, Patel, Bowcock, Ott (2004), "A transmission disequilibrium test for general pedigrees that is robust to the presence of random genotyping errors and any number of untyped parents", European Journal of Human Genetics, 12:752-761.

    *   User's Guide

*   TDTASP

    *   full name: Power and Sample-Size Calculations for the TDT and ASP Tests

    *   version: 1.1 (April 2003)

    *   description: Power and sample-size calculations for the TDT and ASP tests under a wide variety of ascertainment schemes. Uses the flexible genetic model of McGinnis. Most calculations are exact rather than asymptotic.

    *   Authors: Barry W. Brown (bwb@mdanderson.org), Dan Serachitopol (dansera@mdanderson.org)

    *   web/ftp: http://biostatistics.mdanderson.org/SoftwareDownload/

    *   http://biostatistics.mdanderson.org/SoftwareDownload/SingleSoftware.aspx?Software_Id=20

    *   ftp://odin.mdacc.tmc.edu/pub/source/tdtasp_1.1.tar.gz (source)

    *   ftp://odin.mdacc.tmc.edu/pub/sun/tdtasp_1.0.tar.gz (Solaris Executable)

    *   ftp://odin.mdacc.tmc.edu/pub/win32/tdtasp_1.1_se.exe (Windows executable)

    *   source code language: Fortran95

    *   operating systems: UNIX, MS-Windows

    *   executables: tdtasp

    *   reference: Brown (2004), "Power calculations for the transmission/disequilibrium and affected sib pair tests using elementary probability methods", Genetic Research, 83(2):133-141.

*   TDTHAP (see R/TDTHAP)

*   TDT-PC

    *   full name: Transmission Disequilibrium Test Power Calculator

    *   version: 1.3 (2002)

    *   description: TDT Power Calculator (TDT-PC) is a computer program to compute the statistical power of the Transmission/Disequilibrium Test (TDT) analytically, based on the most accurate asymptotic algorithms up to date, and is applicable in very general situations, where different parental disease status, multiple children, mixed family type and recombination events are considered. Routine algorithms for Monte Carlo simulations with significant improvements are also implemented in this program.

    *   Authors: Wei-Min Chen (email: wc9c@virginia.edu) and Hong-Wen Deng (deng@creighton.edu) (John Hopkins)

    *   web/ftp: http://people.virginia.edu/~wc9c/TDTPC/Download.htm

    *   http://www.biostat.jhsph.edu/~wmchen/pc.html

    *   source code language: C++

    *   operating systems: MS-Windows, MS-DOS, UNIX(Solaris)

    *   executables:

    *   reference: Chen, Deng (2001), Genetic Epidemiology, 21:53-67.

*   TDTPOWER

    *   full name:

    *   version: 1.0b (Aug 2001)

    *   description: calculates the sample size required for obtaining a prescribed power against a specified alternative for TDT.

    *   authors: Michael Knapp (Univ Bonn)

    *   web/ftp: http://www.uni-bonn.de/~umt70e/soft.htm

    *   source code language: SAS macro

    *   operating systems:

    *   executables:

    *   reference: M Knapp (1999), "A note on power approximations for the transmission/disequilibrium test", American Journal of Human Genetics, 64(4):1177-1185

*   TDT/S-TDT

    *   full name: Transmission Disequilibrium Test and Sib Transmission Disequilibrium Test

    *   version: 1.1 (Feb 24, 1999)

    *   description: The program provides separate results for TDT, S-TDT, and the combined (overall) test, as appropriate.

    *   Authors:

    *   web/ftp: http://genomics.med.upenn.edu/spielman/TDT.htm

    *   source code language: Java

    *   operating systems: MS-Windows (95/NT)

    *   executables:

    *   reference: Spielman, McGinnis, Ewens (1993), "Transmission test for linkage disequilibrium: the insulin gene region and insulin-dependent diabetes mellitus (IDDM)", American Journal of Human Genetics, 52(3):506-516

    *   Spielman, Ewens (1998), "A sibship test for linkage in the presence of association: the sib transmission/disequilibrium test", American Journal of Human Genetics, 62(2):450-458

*   TFPGA

    *   full name: Tools For Population Genetic Analyses

    *   version:

    *   description: This program calculates descriptive statistics, genetic distances, and F-statistics. It also performs tests for Hardy-Weinberg equilibrium, exact tests for genetic differentiation, Mantel tests, and UPGMA cluster analyses.

    *   authors: Mark Miller (email: mark.miller@usu.edu)

    *   web/ftp: http://www.marksgeneticsoftware.net/

    *   source code language:

    *   operating systems: MS-Windows

    *   executables:

    *   reference:

*   THESIAS

    *   full name: Testing Haplotype EffectS In Association Studies

    *   version: 3.1 (March 2006)

    *   description: The objective of the THESIAS program is to perform haplotype-based association analysis in unrelated individuals. This program is based on a maximum likelihood model described in Tregouet et al. 2002 and is linked to the stochastic EM (SEM) algorithm. THESIAS allows the simultaneous estimation of haplotype frequencies and of their associated effects on the phenotype of interest. In its current version, both quantitative and qualitative phenotypes can be studied. Covariate-adjusted haplotype effects as well as haplotype x covariate interactions can be investigated.

    *   authors: David-Alexandre Tregouet (email: david.tregouet@chups.jussieu.fr)

    *   web/ftp: http://ecgene.net/genecanvas/downloads.php?cat_id=1

    *   source code language:

    *   operating systems: MS-Windows, Linux

    *   executables:

    *   reference:

*   TKMAP

    *   full name:

    *   version:

    *   description: a program for drawing genetic maps

    *   authors: Humberto Ortiz Zuazaga (email: humberto@hpcf.upr.edu), Rosemarie Plaetke ( rosi@momo.uthscsa.edu) (Univ Texas Health Science Ctr at San Antonio).

    *   web/ftp: http://www.hpcf.upr.edu/~humberto/software/TkMap/

    *   source code language: based on bioTk which is based on Tcl/Tk

    *   operating systems:

    *   executables:

    *   reference: manual page

*   TLINKAGE

    *   full name: two-locus trait linkage programs

    *   version:

    *   description: these are extensions of the LINKAGE programs that allow the disease phenotype to be under the control of disease two loci.

    *   authors: Lathrop, Schork, Boehnke, Terwilliger, Ott

    *   web/ftp:

    *   source code language: Pascal

    *   operating systems: MS-DOS

    *   executables:

    *   reference: Lathrop, Ott (1990), American Journal of Human Genetics, 47(supp):A188.

    *   Schork, Boehnke, Terwilliger, Ott (1993), "Two-trait-locus linkage analysis: a powerful strategy for mapping complex genetic traits", American Journal of Human Genetics, 53(5):1127-1136

    *   online documentation

*   TOMCAT

    *   full name:

    *   version: 1.0 (December 2005)

    *   description: The program implements the Mantel statistics as proposed by Beckmann et al. (2005) to test for association between genetic markers and phenotypes in case-control studies using haplotype information. The potential value of haplotypes has attracted widespread interest in the mapping of complex traits. Haplotype sharing methods take into account linkage disequilibrium information between multiple markers, and may have good power to detect predisposing genes. We present a new approach based on Mantel statistics for space time clustering, which we developed in order to improve the power of haplotype sharing analysis for gene mapping in complex disease. The new statistic correlates genetic similarity and phenotypic similarity across pairs of haplotypes for case-only and case-control studies. The genetic similarity is measured as the shared length between haplotypes around a putative disease locus. Alternative measures for the phenotypic similarity were implemented.

    *   authors: Michael Rabes, Matthias Schmitt, Lars Beckmann (email: l.beckmann@dkfz.de)

    *   web/ftp: http://www.dkfz.de/en/epidemiologie-krebserkrankungen/software/software.html

    *   source code language: Java 5.0

    *   operating systems:

    *   executables: tomcat1.0.jar

    *   reference: Beckmann, Thomas, Fischer, Chang-Claude (2005), "Haplotype sharing analysis using mantel statistics", Human Heredity, 59(2):67-78.

*   TRANSMIT

    *   full name:

    *   version: 2.5.4

    *   description: TRANSMIT tests for association between genetic marker and disease by examining the transmission of markers from parents to affected offspring. The main features which differ from other similar programs are: (1) It can deal with transmission of multi-locus haplotypes, even if phase is unknown, and (2) Parental genotypes may be unknown.

    *   Authors: David Clayton (Cambridge Institute, david.clayton@cimr.cam.ac.uk

    *   web/ftp: http://www-gene.cimr.cam.ac.uk/clayton/software/

    *   source code language: C++

    *   operating systems: UNIX

    *   executables: transmit

    *   reference: online documentation

*   TRAP

    *   full name: Tiled Regression Analysis Package

    *   version: 1.0

    *   description: a software tool for determining a regression model of quantitative or binary trait variation when the number of possible genetic predictors is very large, considering only a moderate number of predictors at one time, using unrelated or family data.

    *   authors: Alexander F. Wilson (afw@mail.nih.gov), Yoonhee Kim, Heejong Sung, Juanling Cai, Alexa J. M. Sorant (ajms@mail.nih.gov)

    *   web/ftp: http://research.nhgri.nih.gov/software/TRAP/

    *   ftp://ftp.nhgri.nih.gov/pub/software/TRAP/

    *   source code language: R

    *   operating systems: any with R installation

    *   executables:

    *   reference: Wilson, Kim, Sung, Cai, McMahon, Sorant (2009), "Tiled Regression: the use of regression methods in hotspot defined genomic segments to identify independent genetic variants responsible for variation in quantitative traits", Abstract 142\. In: "Abstracts from the Eighteenth Annual Meeting of the International Genetic Epidemiology Society", Genetic Epidemiology, 33:793.

    *   Sorant, Cai, Sung, Kim, Wilson (2010), "Tiled Regression Analysis Package (TRAP): software implementation of tiled regression methodology", Abstract 244. In: "Abstracts from the nineteenth annual meeting of the International Genetic Epidemiology Society", Genetic Epidemiology, 34:984-985.

*   TREELD

    *   full name:

    *   version: 1.0 (Jan 2005)

    *   description: The package TreeLD is a free software tool for mapping complex trait loci. TreeLD performs a multipoint LD-analysis by inferring the ancestry of a genomic region and analyzing this ancestry for signals of disease mutations. The generated likelihoods can be used to test for the presence of a disease locus and to fine-map its location, providing a point estimate and a credible region. Furthermore, the package provides a novel way of visualizing the association signal in a sample. TreeLD is designed for high-density SNP haplotypes and can be applied to case-control data, TDT trio data and quantitative trait data.

    *   authors: Sebastian Zollner, Xiaoquan Wen, Jonathan Pritchard

    *   web/ftp: http://pritch.bsd.uchicago.edu/treeld.html

    *   source code language:

    *   operating systems: UNIX(Solaris/..), Linux, MS-Windows(95/98/NT/2000/XP)

    *   executables:

    *   reference: Zollner, Pritchard (2004), "Coalescent-based association mapping and fine mapping of complex trait loci", Genetics, 169:1071-1092.

    *   Zollner, Wen, Pritchard (2005), "Association mapping and fine mapping with TreeLD", Bioinformatics, 21(14):3168-3170.

*   TREESCAN

    *   full name:

    *   version: 0.9

    *   description: TREESCAN is intended to provide p-values for the hypothesis of association between evolutionary clades and continuous traits, using haplotype trees.

    *   authors: David Posada (email: dposada@uvigo.es)

    *   web/ftp: http://darwin.uvigo.es/software/treescan.html

    *   source code language: C

    *   operating systems: UNIX, MS-Windows, MS-DOS, MacOS(X)

    *   executables:

    *   reference: Posada, Maxwell, Templeton (2005), "TreeScan: a bioinformatic application to search for genotype/phenotype associations using haplotype trees", Bioinformatics, 21(9):2130-2132.

*   TREESELECT

    *   full name:

    *   version:

    *   description: TreeSelect is a software package for inferring natural selection from unusual population differentiation between closely related populations.

    *   authors: Alkes Price (Harvard Univ)

    *   web/ftp: http://www.hsph.harvard.edu/faculty/alkes-price/software/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Bhatia et al. (2011), "Genome-wide comparison of African-ancestry populations from CARe and other cohorts reveals signals of natural selection", American Journal of Human Genetics, 89(3):368-381.

*   TRIMHAP

    *   full name: TRIMmed-HAPlotype (previously named HAL: Haplotype ALgorithm)

    *   version: 1.2

    *   description: linkage disequilibrium mapping based on ancestral founder haplotypes. Method uses haplotype data from general pedigrees.

    *   Authors: Rory Martin and Charles MacLean (Virginia Commonwealth University)

    *   web/ftp: http://www.vipbg.vcu.edu/trimhap/

    *   source code language: FORTRAN

    *   operating systems: UNIX

    *   executables: hal

    *   reference: MacLean, Martin, Sham, Straub, Kendler (2000), "The trimmed-haplotype test for linkage disequilibrium", American Journal of Human Genetics, 66(3):1062-1075\. [ html]

*   TUNA

    *   full name: Testing UNtyped Alleles

    *   version: 1.1.2 beta (July 2007)

    *   description: TUNA is a free software package for estimating allele frequencies for all the SNPs available in a population reference panel (e.g. HapMap data base) based on genotype data from a subset of markers (e.g. the Illumina HumanHap300 BeadChip SNP set) in a group of subjects (e.g. the cases in a case-control sample).

    *   authors:

    *   web/ftp: http://www.stat.uchicago.edu/~wen/tuna/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Nicolae (2006), "Quantifying the amount of missing information in genetic association studies", Genetic Epidemiology, 30(8):703-717.

    *   Nicolae (2006), "Testing untyped alleles (TUNA) - applications to genome-wide association studies", Genetic Epidemiology, 30(8):718-727.

    *   Wen, Nicolae (2007), "Case-control association studies of untyped markers with TUNA", Bioinformatics, 24(3):435-437.

*   2DMAP

    *   full name:

    *   version:

    *   description: constructing 2-d crossover-based map

    *   authors: Edward Kort ( edward@episun2.med.utah.edu)

    *   web/ftp: http://www.genlink.wustl.edu/software

    *   ftp://mapper.wustl.edu/pub/two-d

    *   source code language:

    *   operating systems: MS-DOS

    *   executables: pic-2d, 2d-map, 2d-color

    *   reference: Fain, Kort, Chance, Nguyen, Redd, Econ, Barker (1995), Nature Genetics, 9(3):261-265.

*   2LD

    *   full name:

    *   version: 2002

    *   description: 2LD is a simple program calculating linkage disequilibrium (LD) measures between two polymorphic markers.

    *   authors: Jinghua Zhao (MRC Epidemiology Unit at Strangeways Research Lab, UK) (email: jinghua.zhao@mrc-epid.cam.ac.uk)

    *   web/ftp: http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software.htm

    *   source code language: C

    *   operating systems:

    *   executables:

    *   reference: Zhao (2004), "2LD, GENECOUNTING and HAP: computer programs for linkage disequilibrium analysis", Bioinformatics, 20(8):1325-1326.

    *   Zapata, Carollo, Rodriguez (2001), "Sampling variance and distribution of the D' measure of overall gametic disequlibrium between multiallelic loci", Annals of Human Genetics, 65:395-406.

*   TWOLOC

    *   full name:

    *   version: v2 (May 1998)

    *   description: a software package for analyzing two-locus susceptibility gene models in affected sib-pair data

    *   Authors: Martin Farrall (Wellcome Trust Centre for Human Genetics, mfarrall@well.ox.ac.uk)

    *   web/ftp: http://www.well.ox.ac.uk/~mfarrall/twoloc.htm

    *   source code language: FORTRAN77, Pascal, awk

    *   operating systems: UNIX(Saloris/DEC UNIX/IRIX/..)

    *   executables: twoloc, calc_alpha, mkp, calc_ibd

    *   reference: Farrall (1997), Genetic Epidemiology, 14:103-115.

*   2SNP

    *   full name:

    *   version: 1.5 (Aug 4, 2006)

    *   description:

    *   authors: Alexander Zelikovsky (email: alexz@cs.gsu.edu), Dumitru Brinza (email: dima@cs.gsu.edu)

    *   web/ftp: http://alla.cs.gsu.edu/~software/2SNP/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Brinza and Zelikovsky (2006), "2SNP: scalable phasing based on 2-SNP haplotypes", Bioinformatics, 22(3):371-373.

*   UMAKE

    *   full name:

    *   version:

    *   description: UMAKE is a software pipeline to detect SNPs and call their genotypes from a list of BAM files. UMAKE pipeline has been successfully applied in detecting SNPs from many large-scale next-generation sequencing studies.

    *   authors: Hyun Min Kang, Goo Jun, Carlo Sidore, Yun Li, Paul Anderson, Mary Kate Trost, Wei Chen, Tom Blackwell, Goncalo Abecasis (Univ Michigan)

    *   web/ftp: http://genome.sph.umich.edu/wiki/UMAKE

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   UNKNOWN (see "LINKAGE - general pedigrees").

    *   reference: a relevant documentation is the mystery of (the) unknown

*   UNPHASED

    *   full name:

    *   version: 3.0.13 (May 2008)

    *   description: UNPHASED is a suite of programs for association analysis of multilocus haplotypes from unphased genotype data. These include TDTPHASE for case-parent trios, COCAPHASE for case/control data, QTPHASE for quantitative traits in unrelateds, PDTPHASE for general pedigrees, and QPDTPHASE for quantitative traits in general pedigrees. Features include global and individual haplotype tests, main effects and conditional tests, grouping of rare haplotypes, pairwise comparisons of haplotype risk, flexible permutation procedures and calculation of LD measures.

    *   authors: Frank Dudbridge (MRC Human Genome Mapping Project Resource Centre) (email: frank.dudbridge@mrc-bsu.cam.ac.uk

    *   web/ftp: http://www.mrc-bsu.cam.ac.uk/personal/frank/software/unphased/

    *   source code language: C++, Java

    *   operating systems: UNIX(SOlaris), Linux, MS-Windows

    *   executables: unphased, tdtphase, cocaphase, pdtphase, qpdtphase

    *   reference: F Dudbridge (2003), "Pedigree disequilibrium tests for multilocus haplotypes", Genetic Epidemiology, 25:115-121.

    *   Dudbridge (2008), "Likelihood-based association analysis for nuclear families and unrelated subjects with missing genotype data", Human Heredity, 66:87-98.

*   UTIL

    *   full name: statistical genetics UTILity programs

    *   version:

    *   description:

    *   authors: Jurg Ott (Rockefeller University, email:ott@rockefeller.edu)

    *   web: http://lab.rockefeller.edu/ott/programs

    *   source code language: Pascal

    *   operating systems: MS-Windows, Linux

    *   executables: multiple, binom, chiprob, conting, equiv, hist, hwe, ibd, mapfun, nocom, norinv, norprob, par, pvalues, relrisk, sexdif, tprob, trend

    *   reference: Ott (1999), Analysis of human genetic linkage 3rd edition (Johns Hospkins University Press, Baltimore).

*   VAAST

    *   full name: Variant Annotation, Analysis and Search Tool

    *   version:

    *   description: VAAST is a probabilistic search tool for identifying damaged genes and their disease-causing variants in personal genome sequences. VAAST builds upon existing amino acid substitution (AAS) and aggregative approaches to variant prioritization, combining elements of both into a single unified likelihood-framework that allows users to identify damaged genes and deleterious variants with greater accuracy, and in an easy-to-use fashion. VAAST can score both coding and non-coding variants, evaluating the cumulative impact of both types of variants simultaneously. VAAST can identify rare variants causing rare genetic diseases, and it can also use both rare and common variants to identify genes responsible for common diseases. VAAST thus has a much greater scope of use than any existing methodology.

    *   authors: Mark Yandell's group (Univ Utah)

    *   web: http://www.yandell-lab.org/software/vaast.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Yandell, Huff, Hu, Singleton, Moore, Xing, Jorde, Reese (2011), "A probabilistic disease-gene finder for personal genomes", Genome Research, 21(9):1529-1542.

*   VARSCAN

    *   full name:

    *   version:

    *   description: VarScan is a platform-independent, technology-independent software tool for identifying SNPs and indels in massively parallel sequencing of individual and pooled samples. Given data for a single sample, VarScan identifies and filters germline variants based on read counts, base quality, and allele frequency. Given data for a tumor-normal pair, VarScan also determines the somatic status of each variant (Germline, Somatic, or LOH) by comparing read counts between samples.

    *   authors:

    *   web/ftp: http://genome.wustl.edu/software/varscan

    *   source code language: Java

    *   operating systems:

    *   executables:

    *   reference: Koboldt, Chen, Wylie, Larson, McLellan, Mardis, Weinstock, Wilson, Ding (2009), "VarScan: variant detection in massively parallel sequencing of individual and pooled samples", Bioinformatics, 25:2283-2285.

*   VG

    *   full name: Visual Genotype

    *   version: 2.0

    *   description: The program presents complete raw datasets of individuals' genotype data using a display format with samples as rows and polymorphisms as columns. The color code is: (1) blue: homozygous genotype for the common allele; (2) red: heterozygous genotype; (3) yellow: homozygous genotype for the rare allele; and (4) grey: missing data

    *   authors:

    *   web/ftp: http://pga.gs.washington.edu/VG2.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Nickerson, Taylor, Weiss, Clark, Hutchinson, Stengard, Salomaa, Vartiainen, Boerwinkle, Sing (1998), "DNA sequence diversity in a 9.7-kb region of the human lipoprotein lipase gene", Nature Genetics, 19:233-240.

    *   Rieder, Taylor, Clark, Nickerson (1999), "Sequence variation in the human angiotensin converting enzyme", Nature Genetics, 22:59-60.

*   VH

    *   full name:visual haplotype

    *   version: 1.0

    *   description: displaying estimated haplotype data

    *   authors:

    *   web/ftp: http://pga.gs.washington.edu/VH1.html

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference: Nickerson, Taylor, Weiss, Clark, Hutchinson, Stengard, Salomaa, Vartiainen, Boerwinkle, Sing (1998), "DNA sequence diversity in a 9.7-kb region of the human lipoprotein lipase gene", Nature Genetics, 19:233-240.

    *   Rieder, Taylor, Clark, Nickerson (1999), "Sequence variation in the human angiotensin converting enzyme", Nature Genetics, 22:59-60.

*   VITESSE

    *   full name: VITESSE (means "speed" in French)

    *   version: 2.0beta (July 1998)

    *   description: calculates likelihood on pedigrees

    *   authors: Jeff O'Connell ( jeff@sherlock.hgen.pitt.edu),

    *   web/ftp: http://watson.hgen.pitt.edu/register/soft_doc.html,

    *   http://watson.hgen.pitt.edu/register

    *   source code language: ANSI C

    *   operating systems: UNIX, VMS, MS-DOS

    *   executables:

    *   reference: Am. Soc. Hum. Genet. Annual Meeting 1995 (O'Connell,Weeks, Am. J. Hum. Genet. suppl 57, A199 (1995)).

    *   O'Connell,Weeks (1995), Nature Genetics, 11:402-408.

    *   online README file

*   WEB-PREPLINK

    *   description: This was a simple program running on our old _linkage_ server at Rockefeller University. However, that server has been [updated](http://lab.rockefeller.edu/ott/) and no longer supports running scripts, so web-preplink is no longer available.

*   WEBQTL

    *   full name:

    *   version:

    *   description: An interactive web site useful for exploring the genetic modulation of thousands of phenotypes gathered over a 30-year period by hundreds of investigators using reference panels of recombinant inbred strains of mice. WebQTL includes dense error-checked genetic maps, as well as extensive gene expression data sets (Affymetrix) acquired across more than 35 strains of mice. WebQTL accepts user-entered traits for BXD, AXB/BXA, CXB, BXH, AKXD recombinant inbred strains.

    *   authors: Jintao Wang, Elissa J. Chesler, Lu Lu, Jing Gu, Yanhua Qu, Robert W. Williams, (email: rwilliams@nb.utmem.edu) Kenneth F. Manly (email:kmanly@utmem.edu) (Univ of Tennessee Health Science Center)

    *   web/ftp: http://www.webqtl.org/search.html

    *   source code language: C++ and Python

    *   operating systems: all via internet

    *   executables:

    *   reference: J Wang, RW Williams, EJ Chesler, KF Manly (2003), "WebQTL: Web-based complex trait analysis", Neuroinformatics 1(4):299-308.

    *   EJ Chesler, J Wang, L Lu, Y Qu, KF Manly, RW Williams (2003), "Genetic correlates of gene expression in recombinant inbred strains: a relational model system to explore neurobehavioral phenotypes", Neuroinformatics, 1(4):343-357.

*   WEIGHTED_FDR

    *   full name:

    *   version: 1.2 (March 2006)

    *   description:

    *   authors: Lambertus Klei (University of Pittsburgh Medical Center), Kathryn Roeder (Carnegie Mellon University)

    *   web/ftp: http://wpicr.wpic.pitt.edu/WPICCompGen/fdr/

    *   source code language: R

    *   operating systems: MS-Windows, Linux, UNIX

    *   executables:

    *   reference:

*   WHAIT

    *   full name: Weighted Haplotype test And an Imputation-based Test.

    *   version:

    *   description:

    *   authors: Yun Li (email: ylwtx@umich.edu)

    *   web/ftp: http://www.sph.umich.edu/csg/yli/whait/

    *   source code language:

    *   operating systems:

    *   executables:

    *   reference:

*   WHAP

    *   full name:

    *   version: 2.06 (Dec 2004)

    *   description:

    *   authors: Shaun Purcell (email: purcell@wi.mit.edu), Pak Sham

    *   web: http://pngu.mgh.harvard.edu/~purcell/whap/

    *   source code language: C,C++

    *   operating systems: UNIX, MS-Windows, MS-DOS, Linux

    *   executables:

    *   reference:

*   WHICHRUN

    *   full name:

    *   version: 4.1

    *   description: uses multilocus genotypic data to allocate individuals to their most likely source population

    *   authors: Michael Banks, (email: michael.banks@hmsc.orst.edu), Will Eichert (email: wfeichert@ucdavis.edu)

    *   web/ftp: http://www.bml.ucdavis.edu/whichrun.htm

    *   source code language: C++

    *   operating systems: MS-Windows (95/98/NT)

    *   executables:

    *   reference: Banks, Eichert (2000), "WHICHRUN (version 3.2): a computer program for population assignment of individuals based on multilocus genotype data", Journal of Heredity, 91:87-89.

*   ZAPLO

    *   full name:

    *   version: Oct 5, 2005

    *   description: zero recombinant haplotyping

    *   authors: Jeff O'Connell (email: joconnel@medicine.umaryland.edu, zaplo@molecular-haplotype.org)

    *   web/ftp: http://www.molecular-haplotype.org/zaplo/zaplo_index.html

    *   source code language:

    *   operating systems: Linux, UNIX(Solaris)

    *   executables:

    *   reference: O'Connell (2000), "Zero-recombinant haplotyping: applications to fine mapping using SNPs,", Genetic Epidemiology, 19:S64-S70.
