<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=windows-1252">
	<title>Genetic Analysis Software</title>
	<meta name="generator" content="LibreOffice 4.2.3.3 (Windows)">
	<meta name="author" content="Jurg Ott">
	<meta name="created" content="20121022;2340000">
	<meta name="changed" content="20140810;171128619000000">
	<meta name="AppVersion" content="12.0000">
	<meta name="DocSecurity" content="0">
	<meta name="HyperlinksChanged" content="false">
	<meta name="LinksUpToDate" content="false">
	<meta name="ScaleCrop" content="false">
	<meta name="ShareDoc" content="false">
	<style type="text/css">
	<!--
		@page { size: 8.27in 11.69in; margin: 0.5in }
		p { margin-bottom: 0.08in; direction: ltr; widows: 2; orphans: 2 }
		td p { margin-bottom: 0.08in; direction: ltr; widows: 2; orphans: 2 }
		a:link { color: #0000ff; so-language: en-US }
	-->
	</style>
</head>
<body lang="en-US" link="#0000ff" dir="ltr" style="background: transparent">
<p align="center" style="margin-top: 0.02in; margin-bottom: 0.17in; line-height: 100%">
<br><br>
</p>
<center>
	<table width="458" cellpadding="30" cellspacing="1" bgcolor="#239eee">
		<col width="394">
		<tr>
			<td width="394" bgcolor="#239eee" style="border: 3.00pt double #00000a; padding-top: 0.31in; padding-bottom: 0.31in; padding-left: 0.33in; padding-right: 0.31in">
				<p align="center"><font color="#df0273"><font face="Times New Roman, serif"><font size="6" style="font-size: 24pt"><b>An
				Alphabetic List </b></font></font></font><font face="Times New Roman, serif"><font size="3" style="font-size: 12pt"><br></font></font><font color="#a87b45"><font face="Times New Roman, serif"><font size="5" style="font-size: 18pt"><b>of
				</b></font></font></font><font face="Times New Roman, serif"><font size="3" style="font-size: 12pt"><br></font></font><font color="#ffffff"><font face="Times New Roman, serif"><font size="6" style="font-size: 24pt"><b>Genetic
				Analysis Software</b></font></font></font><font color="#ffffff"> </font>
				</p>
			</td>
		</tr>
	</table>
</center>
<p align="center" style="margin-top: 0.02in; margin-bottom: 0.02in; line-height: 100%">
<font color="#ef1602"><font face="Times New Roman, serif"><font size="2" style="font-size: 10pt"><b>Last
updated: 10 November 2013</b></font></font></font><font color="#ef1602">
</font>
</p>
<p style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 10pt">Computer
software on the following topics is included here: genetic linkage
analysis for human pedigree data, QTL analysis for animal/plant
breeding data, genetic marker ordering, genetic association analysis,
haplotype construction, pedigree drawing, and population genetics.
This list is offered here as a service to the gene mapping community.
The inclusion of a program should not be interpreted as an
endorsement from us.</font></font></p>
<p style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 10pt">In
the last few years, new technology has been producing new types of
genetic data, and the scope of genetic analyses changes dramatically.
It is no longer obvious whether a program should be included or
excluded from this list. Topics such as next-generation-sequencing
(NGS), gene expression, genomics annotation, etc. can all be relevant
to a genetic study, but we'll have to decide on a case-by-case basis
whether to include a program (<a href="mailto:ott@rockefeller.edu">email
us</a> if you are interested in having a program listed here).</font></font></p>
<p style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 10pt">This
page was created by Dr. <a href="http://scholar.google.com/citations?user=XsMM0oQAAAAJ&amp;hl=en">Wentian
Li</a>, when he was at Columbia University (1995-1996). It was later
moved to Rockefeller University (1996-2002) and now it takes its home
at North Shore LIJ Research Institute (2002-now). The present copy is
maintained by <a href="mailto:ott@rockefeller.edu">Jurg Ott</a> as a
single file. More than 240 programs were listed by December 2004,
close to 400 programs by December 2006, close to 480 programs by
November 2008, and over 600 programs by October 2012. A version of
the searchable database was developed by Zhiliang Hu of Iowa State
University, and a recent round of updating was assisted by Wei JIANG
of Harbin Medical School.</font></font></p>
<p style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 10pt">Some
earlier software can be downloaded from EBI:
<a href="ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/</a>
(Linkage and Mapping Software Repository).</font></font></p>
<p style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 10pt">More
and more packages are now written in R. To be consistent, any R
package in CRAN has been renamed from [package-name] to
R/[package-name]. If an R package is not submitted to CRAN, its
original name has been kept. Here is another partial list of
statistical genetics R packages summarized by CRAN:
<a href="http://cran.r-project.org/web/views/Genetics.html">http://cran.r-project.org/web/views/Genetics.html</a>.
More R packages yet can be found at:
<a href="http://www.mrc-epid.cam.ac.uk/~jinghua.zhao/r-genetics.htm">http://www.mrc-epid.cam.ac.uk/~jinghua.zhao/r-genetics.htm</a>,
<a href="http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm">http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm</a>,
<a href="http://wpicr.wpic.pitt.edu/WPICCompGen/software.htm">http://wpicr.wpic.pitt.edu/WPICCompGen/software.htm</a>,
<a href="http://www-gene.cimr.cam.ac.uk/clayton/software/">http://www-gene.cimr.cam.ac.uk/clayton/software/</a>,
and other places.</font></font></p>
<p style="margin-bottom: 0.14in"><br><br>
</p>
<ul>
	<li><p style="margin-bottom: 0.14in">ACT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Analysis of Complex
		Traits</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (Mar 17, 2000)</p>
		<li><p style="margin-bottom: 0.14in">description: It contains the
		following modules: ibd, calculates the proportion of gene shared
		identical by decent for a nuclear family; ibdn, (modified program
		of ERPA), which implements a method for assessing increased-allele
		sharing between all pairs of affected relatives within a pedigree;
		multic, multivariate analysis for complex traits; ml, estimation of
		variance components using maximum likelihood; ql, estimation of
		variance components using quasi likelihood; relcov, generates first
		degree relationship coefficients for extended families; sim2s, the
		simulation program that was used to test ACT; cage, Cohort Analysis
		for Genetic Epidemiology; gh: GeneHunter, heavily modified to
		assist multipoint calculation using multic; TDT: TDT programs
		written in SAS; gcc and f77 compilers are necessary. Executable
		programs are included for compatible operating systems, i.e.,
		Solaris2.6.</p>
		<li><p style="margin-bottom: 0.14in">authors: Christopher I. Amos
		(email: camos@request.mdacc.tmc.edu), Mariza de Andrade (email:
		mandrade@request.mdacc.tmc.edu), JianFang Chen (email:
		cjf@request.mdacc.tmc.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.epigenetic.org/Linkage/act.html,
		http://www.epigenetic.org/Linkage/act.tar.gz</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77, C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris 2.4/..)</p>
		<li><p style="margin-bottom: 0.14in">executables: ibd, ml, ql, he,
		ibdn, multic</p>
		<li><p style="margin-bottom: 0.14in">reference: Amos, et al (1996),
		&quot;Assessing genetic linkage and association with robust
		components of variance approaches&quot;, Annals of Human Genetics,
		60:143-160.</p>
		<li><p style="margin-bottom: 0.14in">Amos (1994), &quot;Robust
		variance-components approach for assessing genetic linkage in
		pedigrees&quot;, American Journal of Human Genetics, 54:535-543.
		[<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1918121">abstract</a>]</p>
		<li><p style="margin-bottom: 0.14in">de Andreade, Amos, Thiel
		(1999), &quot;Methods to estimate genetic parameters for
		quantitative traits&quot;, Genetic Epidemiology, 17(1):64-76.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ADEGENET (see also R/ADEGENET)</p>
	<li><p style="margin-bottom: 0.14in">ADMIXMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Admixture mapping</p>
		<li><p style="margin-bottom: 0.14in">version: 3.7 (2007)</p>
		<li><p style="margin-bottom: 0.14in">description: ADMIXMAP is a
		general-purpose program for modelling admixture, using marker
		genotypes and trait data on a sample of individuals from an admixed
		population (such as African-Americans), where the markers have been
		chosen to have extreme differentials in allele frequencies between
		two or more of the ancestral populations between which admixture
		has occurred. The main difference between ADMIXMAP and classical
		programs for estimation of admixture such as ADMIX is that ADMIXMAP
		is based on a multilevel model for the distribution of individual
		admixture in the population and the stochastic variation of
		ancestry on hybrid chromosomes. This makes it possible to model the
		associations of ancestry between linked marker loci, and the
		association of a trait with individual admixture or with ancestry
		at a linked marker locus.</p>
		<li><p style="margin-bottom: 0.14in">authors: Clive Hoggart, Nigel
		Wetters, David Clayton, David O'Donnell (email:
		david.odonnell@ucd.ie), and Paul McKeigue (email:
		paul.mckeigue@ucd.ie)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.homepages.ed.ac.uk/pmckeigu/admixmap/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: admixmap,
		admixmap.pl, and some R scripts to process the output</p>
		<li><p style="margin-bottom: 0.14in">reference: PM McKeigue (2005),
		&quot;Prospects for admixture mapping of complex traits&quot;,
		American Journal of Human Genetics, 76(1):1-7</p>
		<li><p style="margin-bottom: 0.14in">CJ Hoggart, MD Shriver, RA
		Kittles, DG Clayton, PM McKeigue (2004), &quot;Design and analysis
		of admixture mapping studies&quot;, American Journal of Human
		Genetics, 74(5):965-978</p>
		<li><p style="margin-bottom: 0.14in">CJ Hoggart, EJ Parra, MD
		Shriver, C Bonilla, RA Kittles, DC Clayton, PM McKeigue (2003),
		&quot;Control of confounding of genetic associations in stratified
		populations&quot;, American Journal of Human Genetics,
		72(6):1492-1504</p>
		<li><p style="margin-bottom: 0.14in">PM McKeigue, J Carpenter, EJ
		Parra, MD Shriver (2000), &quot;Estimation of admixture and
		detection of linkage in admixed populations by a Bayesian approach:
		application to African-American populations&quot;, Annals of Human
		Genetics, 64:171-186.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">AGEINF</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: inference of the
		age of a rare, selectively-neutral mutation</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinko Graham</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.math.sfu.ca/~jgraham/Papers/Programs/AgeCode/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Graham, Thompson
		(2000), &quot;A coalescent model of ancestry for a rare allele&quot;,
		Genetics, 156:375-384.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ALBERT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: A Likelihood Based
		Estimation of Risk in Trios</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (May 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: ALBERT is a
		program that estimates genotype relative risks, genotyping error
		rates and population risk allele frequencies from marker genotype
		data in case-parent trios. ALBERT uses the distribution of trio
		marker genotypes to compute maximum likelihood estimates for the
		parameters.</p>
		<li><p style="margin-bottom: 0.14in">authors: Adele Mitchell,
		Elizabeth A Thompson (University of Washington)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/Albert/albert.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ALLASS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: February 2002</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: A Collins (email:
		arc@soton.ac.uk), NE Morton (University of Southampton, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ALLASS</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(SUN..)</p>
		<li><p style="margin-bottom: 0.14in">executables: allass</p>
		<li><p style="margin-bottom: 0.14in">reference: Collins, Morton
		(1998), &quot;Mapping a disease locus by allelic association&quot;,
		Proceedings of the National Academy of Sciences , 95:1741-1745;
		online manual</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ALLEGRO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0 (October 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: a faster version
		of GENEHUNTER and Allegro 1.2 (several degrees of increase of
		speed, can handle bigger families, up to 50 bits)</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel F
		Gudbjartsson, TA Thorvaldsson, G Gunnarsson, Augustine Kong, A
		Ingolfsdottir (DeCode Genetics, Inc. Iceland) send email to:
		allegro@decode.is</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.decode.com/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: D Gudbjartsson, K
		Jonasson, CA Kong (1999), &quot;Fast multipoint linkage calculation
		with Allegro&quot;, American Journal of Human Genetics,
		65(suppl):A60.</p>
		<li><p style="margin-bottom: 0.14in">Gudbjartsson, Jonasson,
		Frigge, Kong (2000) &quot;Allegro, a new computer program for
		multipoint linkage analysis&quot;, Nature Genetics, 25:12-13.</p>
		<li><p style="margin-bottom: 0.14in">Gudbjartsson, Thorvaldsson,
		Gunnarsson, Kong, Ingolfsdottir (2004), &quot;Decision diagram
		based multipoint linkage analysis: ALLEGRO2&quot;, American Journal
		of Human Genetics, 75(suppl):?.</p>
		<li><p style="margin-bottom: 0.14in">Gudbjartsson, Thorvaldsson,
		Kong, Gunnarsson, Ingolfsdottir (2005), &quot;Allegro version 2&quot;,
		Nature Genetics, 37:1015-1016. [<font color="#ff0000"><b>PLEASE
		CITE THIS REFERENCE WHEN USING THE ALLEGRO2 PROGRAM</b></font>]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ALLELIX</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Allelix - Paternity
		Linkage Analysis Online</p>
		<li><p style="margin-bottom: 0.14in">version: December 2001</p>
		<li><p style="margin-bottom: 0.14in">description: One can enter the
		alleles of commonly used STR by clicking the mouse. The algorithm
		calculates the paternity index and the Essen-Moeller probability of
		kinship for the deficiency- and the trio case. Everybody can use
		the network-software online after registering. The usage on the
		internet is free. Academic users can ask me to unlock an option to
		display the details (formulas/frequencies etc.) and to have an
		export-funktion to MS Word. The program is in German and
		(non-professional) English. An expansion to other languages is
		easy, if somebody helps us with the translation. For those who are
		interested to have the software running on their own intranet (for
		database security reasons) an individual agreement can be found.</p>
		<li><p style="margin-bottom: 0.14in">authors: Thomas Krahn (biotix)
		(email: tk@biotix.de), Stefan Zinke (Software Erstellung- und
		Beratung, Germany), Dr. Michael Kraft (biotix)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.allelix.net (German version is: http://www.allelix.de)</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++,
		Java Script</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(2000 &amp;IIS/ME &amp; Personal Webserver)</p>
		<li><p style="margin-bottom: 0.14in">executables: allelix.dll</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ALOHOMORA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.33 (October 2009)</p>
		<li><p style="margin-bottom: 0.14in">description: designed to
		facilitate genome-wide linkage studies performed with high-density
		single nucleotide polymorphism (SNP) marker panels such as the
		Affymetrix GeneChip(R) Human Mapping 10K Array.</p>
		<li><p style="margin-bottom: 0.14in">authors: Franz Ruschendorf
		(email: fruesch@mdc-berlin.de), Peter Nurnberg</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://gmc.mdc-berlin.de/alohomora/</p>
		<li><p style="margin-bottom: 0.14in">source code language: perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Window,
		Linux, UNIX (Solaris/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ruschendorf,
		Nurnberg (2005), &quot;ALOHOMORA: a tool for linkage analysis using
		10K SNP array data&quot;, Bioinformatics, 21(9):2123-2125.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ALP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Automated Linkage
		Preprocessor</p>
		<li><p style="margin-bottom: 0.14in">version: v1.40 (August 1995)</p>
		<li><p style="margin-bottom: 0.14in">description: ALP is a
		Microsoft Windows application designed to analyse microsatellite
		DNA fragments separated on an Automated Laser Fluorescence
		sequencer.</p>
		<li><p style="margin-bottom: 0.14in">authors: Alastair Brown
		(email: alastair.brown@hgu.mrc.ac.uk), Daryll Green, Steve Woess
		(MRC Human Genetics Unit, Edinburgh, UK).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hgu.mrc.ac.uk/Softdata/ALP/</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.hgu.mrc.ac.uk/pub/ALP</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(3.1/95/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables: alpw.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Mansfield, Brown,
		Green, Carothers, Morris, Evans, Wright (1994), &quot;Automation of
		genetic linkage analysis using fluorescent microsatellite markers&quot;,
		Genomics, 24(2):225-233.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ALTREE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: ALTREE performs
		these two phylogeny-based analysis: (1) it tests the association
		between a candidate gene and a disease; (2) it pinpoints markers
		(SNPs) that are putative disease susceptibility loci</p>
		<li><p style="margin-bottom: 0.14in">authors: Claire Bardel (email:
		bardel@vjf.inserm.fr), Vincent Danjean, Jean-Pierre Hugot, Pierre
		Darlu, Emmanuelle G&eacute;nin</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://claire.bardel.free.fr/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: altree,
		altree-convert, altree-add-s</p>
		<li><p style="margin-bottom: 0.14in">reference: Bardel, Danjean,
		Hugot, Darlu, Genin (2005), &quot;On the use of haplotype phylogeny
		to detect disease susceptibility loci&quot;, BMC Genetics, 6:24.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">AMELIA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Allele Matching
		Empirical Locus-specific Integrated Association test</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: AMELIA is a
		program that employs allele matching to analyse the effects of rare
		variants within a specific locus. There is increasing evidence that
		rare variants play a role in some complex traits, but their
		analysis is not straightforward. Locus-based tests become necessary
		due to low power in rare variant single-point association analyses.
		In addition, variant quality scores are available for sequencing
		data, but are rarely taken into account. To enable this analysis,
		AMELIA has been developed as an allele-matching approach that is
		robust to the presence of both directions of effect for variants
		within the locus analysed.</p>
		<li><p style="margin-bottom: 0.14in">authors: Eleftheria Zeggini's
		group (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/amelia/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ANALYZE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1 (feb 14, 1996)</p>
		<li><p style="margin-bottom: 0.14in">description: a set of useful
		accessory programs to the LINKAGE package. It simplifies the
		performance of a large array of parametric and nonparametric tests
		for linkage and association on data entered in LINKAGE format
		pedigree and parameter files.</p>
		<li><p style="margin-bottom: 0.14in">authors: Joseph Terwilliger</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.helsinki.fi/~tsjuntun/linkage/analyze/;
		http://linkage.cpmc.columbia.edu/index_files/Page434.htm; 
		</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/linkage_cpmc_columbia/analyze/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(SunOS/Solaris/OSF1..)</p>
		<li><p style="margin-bottom: 0.14in">executables: analysis
		(script), analyzeprep, downfreq, extract, (homog), homogpos,
		homogpre, hrrlamb, hrrlambprep, hrrmult, hrrprep, interpretation,
		locusorder, pedprepare, polylocus, sib2link, sibpairna, sibprepna,
		tdtlikena</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ANCESTRY</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: This application
		aims at providing the user with an easy to use web-based user
		interface to compute individual ancestry estimates using the
		Maximum Likelihood Estimation method. This website also allows
		registered users to manage genotype and marker/allele frequency
		files. Registration is optional.</p>
		<li><p style="margin-bottom: 0.14in">authors: Van-Anh Tran (email:
		vananh.tran@case.edu), Thomas Dyer, Guo-Qiang Zhang, Jill
		Barnholtz-Sloan (email: jsb42@case.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://ciil.case.edu:3000</p>
		<li><p style="margin-bottom: 0.14in">source code language: Ruby on
		Rails, FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Hanis, Chakraborty,
		Ferrell, Schull (1986), &quot;Individual admixture estimates:
		disease associations and individual risk of diabetes and
		gallbladder disease among Mexican-Americans in Starr County,
		Texas&quot;, American Journal of Physical Anthropology,
		70(4):433-441.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ANCESTRYMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: ANCESTRYMAP
		screens through the genome in a recently mixed population such as
		African Americans, searching for segments with increased ancestry
		from one of the ancestral populations, which can indicate the
		position of disease genes</p>
		<li><p style="margin-bottom: 0.14in">authors: Arti Tandon (email:
		atandon@broad.mit.edu), Nick Patterson (email:
		nickp@broad.mit.edu), David Reich.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genepath.med.harvard.edu/~reich/Software.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Patterson,
		Hattangadi, Lane, Lohmueller, Hafler, Oksenberg, Hauser, Smith,
		O'Brien, Altshuler, Daly, Reich (2004), &quot;Methods for
		high-density admixture mapping of disease genes&quot;, American
		Journal of Human Genetics, 74(5):97-1000</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ANNOVAR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: functional
		ANNOtation of genetic VARiants</p>
		<li><p style="margin-bottom: 0.14in">version: Feb 2010</p>
		<li><p style="margin-bottom: 0.14in">description: ANNOVAR is an
		efficient software tool to utilize update-to-date information to
		functionally annotate genetic variants detected from diverse
		genomes (including human genome hg18, hg19, as well as mouse, worm,
		fly, yeast and many others). Given a list of variants with
		chromosome, start position, end position, reference nucleotide and
		observed nucleotides, ANNOVAR can perform: 1. gene-based
		annotation. 2. region-based annotation. 3. filter-based annotation.
		4. other functionalities.</p>
		<li><p style="margin-bottom: 0.14in">authors: Kai Wang (Univ
		Pennsylvania)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://www.openbioinformatics.org/annovar/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wang, Li,
		Hakonarson (2010), &quot;ANNOVAR: Functional annotation of genetic
		variants from next-generation sequencing data&quot;, Nucleic Acids
		Research, 38:e164.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ANTMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (June 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Ordering markers
		when the number of loci is large is a special case of the traveling
		salesman problem. ANTMAP is a system based on the Ant Colony
		Optimization to solve this problem. ANYMAP performs segregation
		test, linkage grouping and locus ordering, and constructs a linkage
		map rapidly.</p>
		<li><p style="margin-bottom: 0.14in">authors: Hiroyoshi Iwata
		(email: iwata@lbm.ab.a.u-tokyo.ac.jp)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://lbm.ab.a.u-tokyo.ac.jp/~iwata/antmap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		MacOS, Linux, UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Iwata, Ninomiya
		(2006) &quot;AntMap: constructing genetic linkage maps using an ant
		colony optimization algorithm&quot;, Breeding Science, 56:371-377.</p>
		<li><p style="margin-bottom: 0.14in">[PDF]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">APE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Allelic Path
		Explorer</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: Extends partially
		observed genotype data to the whole pedigree. Can be used for
		generating starting points for MCMC samplers and for checking that
		the genotype data are consistent with the pedigree structure.</p>
		<li><p style="margin-bottom: 0.14in">authors: Matti Pirinen, Dario
		Gasbarra (University of Helsinki, Finland)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.helsinki.fi/~mpirinen/download</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Pirinen, Gasbarra
		(2006), &quot;Finding consistent gene transmission patterns on
		large and complex pedigrees&quot;, IEEE/ACM cransactions on
		Computational Biology and Bioinformatics, 3(3):252-262</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">APL-OSA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Association in the
		Presence of Linkage with Ordered Subset Analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: In the presence
		of genetic heterogeneity, APL-OSA can identify a genetically
		homogenous subset of families based on a trait-related covariate.
		APL-OSA then tests the relationship between the association
		statistics (i.e., the APL statistics) calculated based on the
		subset and the family-specific covariate. APL-OSA is based on the
		OSA method for linkage and the family-based association test, APL.
		Thus, APL-OSA has similar properties with OSA and APL. Bi-alleleic
		markers such as SNPs are accepted by APL-OSA. APL-OSA is a
		single-marker test and considers one covariate each time.</p>
		<li><p style="margin-bottom: 0.14in">authors: Elizabeth R. Hauser,
		email: apl-osa@chg.duhs.duke.edu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wwwchg.duhs.duke.edu/research/aplosa.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux(RedHat), MS-Windows(XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Martin, Bass,
		Hauser, Kaplan (2003), &quot;Accounting for linkage in family-based
		tests of association with missing parental genotypes&quot;,
		American Journal of Human Genetics, 73:1016-1026</p>
		<li><p style="margin-bottom: 0.14in">Hauser, Watanabe, Duren, Bass,
		Langefeld, Boehnke (2004), &quot;Ordered subset analysis in genetic
		linkage mapping of complex traits&quot;, Genetic Epidemiology,
		27:53-63.</p>
		<li><p style="margin-bottom: 0.14in">Chung, Hauser, martin (2006),
		&quot;The APL test: extension to general nuclear families and
		haplotypes and examination of its robustness&quot;, Human Heredity,
		61:189-199.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">APM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Affected
		Pedigree-Member Method</p>
		<li><p style="margin-bottom: 0.14in">version: July 1993</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel E. Weeks
		(University of Oxford and University of Pittsburgh, email:
		dweeks@watson.hgen.pitt.edu), Mark Schroeder (email:
		mark@holmes.hgen.pitt.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register/soft_doc.html registration
		page at: http://watson.hgen.pitt.edu/register</p>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		Pascal and FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: apm, sim,
		apmmult, simmult, chapm, hist</p>
		<li><p style="margin-bottom: 0.14in">reference: Weeks and Lange,
		American Journal of Human Genetics, 42(2):315-326 (1988).
		[<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1715269">abstract</a>]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ARIEL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Accumulation of
		Rare variants Integrated and Extended Locus-specific test</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: ARIEL explores
		the effects of rare variants within complex traits through
		locus-based analysis. There is increasing evidence that rare
		variants play a role in some complex traits, but their analysis is
		not straightforward. Locus-based tests become necessary due to low
		power in rare variant single-point association analyses. In
		addition, variant quality scores are available for sequencing data,
		but are rarely taken into account. To enable this analysis, ARIEL
		has been developed as a locus-wide regression-based collapsing
		approach that incorporates variant quality scores.</p>
		<li><p style="margin-bottom: 0.14in">authors: Eleftheria Zeggini's
		group (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/ariel/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ARLEQUIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: (French translation
		of &quot;Arlecchino&quot;, a character of the Italian &quot;Commedia
		dell'Arte&quot;. He has many aspects, but can switch among them
		very easily according to needs and necessities. Similarly, this
		software is about the study of genetic polymorphism.)</p>
		<li><p style="margin-bottom: 0.14in">version: 3.1 (September 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: an exploratory
		population genetics software environment able to handle large
		samples of molecular data (RFLPs, DNA sequences, microsatellites),
		while retaining the capacity of analyzing conventional genetic data
		(standard multi-locus data or mere allele frequency data).</p>
		<li><p style="margin-bottom: 0.14in">authors: Laurent Excoffier
		(email: laurent.excoffier@zoo.unibe.ch), Guillaume Laval, Stefan
		Schneider (University of Berne, Switzerland)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cmpg.unibe.ch/software/arlequin3/</p>
		<li><p style="margin-bottom: 0.14in">http://lgb.unige.ch/arlequin/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/NT/2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Excoffier, Laval,
		Schneider (2005), &quot;Arlequin ver. 3.0: an integrated software
		package for population genetics data analysis&quot;, Evolutionary
		Bioinformatics Online, 1:47-50.</p>
		<li><p style="margin-bottom: 0.14in">Schneider, Kueffer, Roessli,
		Excofier, &quot;Arlequin: A software for population genetic data
		analysis. Ver 1.1&quot;, (Genetics and Biometry Lab, Dept. of
		Anthropology, University of Geneva, 1997);</p>
		<li><p style="margin-bottom: 0.14in">User's manual (PDF, v1.1)</p>
		<li><p style="margin-bottom: 0.14in">User's manual (PDF, v3.1)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ARP.GEE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.1.0 (April 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: Simultaneously
		estimate a trait-locus position and its genetic effects for
		affected relative pairs (ARP) by one of two methods. Either allow a
		different trait-locus effect for each ARP type, or constrain the
		trait-locus effects according to the marginal effect of a single
		susceptibility locus. We include a goodness of fit statistic for
		the constrained model.</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel Schaid, Jason
		Sinnwell (Mayo Clinic)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R/S-PLUS</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ASP/ASPSHARE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (June 2002)
		(Warning: the earlier version contains a bug)</p>
		<li><p style="margin-bottom: 0.14in">description: ASP a power
		calculator for gene mapping using a sibpair design (concordant or
		discordant). ASPSHARE complements ASP in that it allows rapid
		calculation of the expected ibd sharing at the trait locus, based
		upon the model parameters, and the incidence corresponding to the
		respective parameters.</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Krawczak</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uni-kiel.de/medinfo/mitarbeiter/krawczak/download/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: online manual for
		ASP</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ASPEX</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Affected Sib Pairs
		EXclusion map</p>
		<li><p style="margin-bottom: 0.14in">version: 2.5 (September 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: a set of programs
		for performing multipoint exclusion mapping of affected sibling
		pair data for discrete traits.</p>
		<li><p style="margin-bottom: 0.14in">authors: David Hinds (email:
		dahinds@users.sourceforge.net) Neil Risch (Stanford Univ),</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://aspex.sourceforge.net/</p>
		<li><p style="margin-bottom: 0.14in">previously:
		ftp://lahmed.stanford.edu/pub/aspex</p>
		<li><p style="margin-bottom: 0.14in">source code language: C, Tcl,
		Perl (the graphic script requires xmgr, available from
		ftp://ftp.teleport.com/pub/users/pturner/acegr)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Solaris/SunOS/IRIX/OSF-1), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: sib_ibd, sib_tdt,
		sib_phase, sib_map, sib_kin, sib_clean</p>
		<li><p style="margin-bottom: 0.14in">reference: online manual (PDF)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ASSOCIATIONVIEWER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: AssociationViewer
		is a Java application used to display SNPs in a genetic context.
		Supplementary data (such as genes or LD plots) is downloaded from
		various public data sources on the fly and saved locally in a
		cache. Custom data can be added as supplementary tracks.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://associationviewer.vital-it.ch/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Martin, Valsesia,
		Telenti, Xenarios, Stevenson (2009), &quot;AssociationViewer: a
		scalable and integrated software tool for visualization of
		large-scale variation data in genomic context&quot;,
		Bioinformatics, 25(5):662-663.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">AUTOSCAN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.1 (February 2000)</p>
		<li><p style="margin-bottom: 0.14in">description: a helper program
		to automate the tedious process of the creation of input files from
		genotype data of genome-wide scans</p>
		<li><p style="margin-bottom: 0.14in">authors: T Hiekkalinna, L
		Peltonen (UCLA)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.helsinki.fi/~tsjuntun/autoscan/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		UNIX-shell (Bourne)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Solaris/DEC-UNIX)</p>
		<li><p style="margin-bottom: 0.14in">executables: autostart,
		autoscan10.sh</p>
		<li><p style="margin-bottom: 0.14in">reference: American Journal of
		Human Genetics, supp, 65 . ( <a href="http://www.jurgott.org/linkage/absAUTOSCAN_AJHG99.htm">abstract</a>)
		(1999)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BAMA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Bayesian Analysis
		of Multilocus Association</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: This is a program
		to select a trait-associated subset of markers among many
		candidates. The program is based on Bayesian modeling/estimation
		and it suits for both quantitative and qualitative traits. It can
		handle bi- and multiallelic markers as well as applied in
		situations where part of the marker genotypes may be missing. As an
		output of the program, one obtains posterior estimate of number and
		positions of trait-associated markers.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mikko J Sillanpaa,
		Riika Kilpikari</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.rni.helsinki.fi/~mjs/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: R Kilpikari, MJ
		Sillanpaa (2003), &quot;Bayesian analysis of multilocus association
		in quantitative and qualitative traits&quot;, Genetic Epidemiology,
		25:122-135.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BARS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Bayesian Adaptive
		Regression Splines</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The BARS tests is
		a statistical method that bridges the gap between single-locus and
		haplotype-based tests of association. It is based on the
		non-parametric regression techniques embodied by Bayesian Adaptive
		Regression Splines.</p>
		<li><p style="margin-bottom: 0.14in">authors: Garrick Wallstrom, RE
		Kass</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/bars.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhang, Roeder,
		Wallstrom, Devlin (2003), &quot;Integration of associative
		statistics over genomic regions using Bayesian Adaptive Regression
		Splines&quot;, Human Genomics, 1:20-29.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BAYESFST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Bayesian
		estimation of the coancestry coefficient FST</p>
		<li><p style="margin-bottom: 0.14in">authors: DJ Balding (email:
		d.balding@ic.ac.uk), KL Ayres, MA Beaumont, P O'Reilly</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.reading.ac.uk/Statistics/genetics/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Balding (2003),
		&quot;Likelihood-based inference for genetic correlation
		coefficients&quot;, Theoretical Population Biology, 63:221-230</p>
		<li><p style="margin-bottom: 0.14in">Beaumont, Balding (2004),
		&quot;Identifying adaptive genetic divergence among populations
		from genome scans&quot;, Molecular Ecology, 13:969-980.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BDGEN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.7</p>
		<li><p style="margin-bottom: 0.14in">description: a powerful
		database software with improvement tools for paternity testing,
		database searching (like CODIS) and NRC II recommendations based
		formulae for investigating likelyhood ratios in putative
		contributors to crime evidences. Additional genetic population
		parameters estimations are added in this version. Only spanish
		version available.</p>
		<li><p style="margin-bottom: 0.14in">authors: Martinez Gustavo
		(email:ggmartinez@arnet.com.ar), Vazquez Sebastian, and Vicentin
		Ariel (email: bdgen@yahoo.com.ar)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.simedic.com.ar/bdgen.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: V Vazquez, A
		Vicentin, L Vazquez, and G Martinez (2002) &quot;Nuevo programa
		BDGen, como base de datos y herramienta de analisis estadistico de
		patrones geneticos&quot;. VII meeting of the Spanish-Portuguese
		Group for the International Society for Forensic Genertics.
		Barcelona, June 2002.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BEAGLE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.2.1 (May 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: BEAGLE is a state
		of the art software package for analysis of large-scale genetic
		data sets with hundreds of thousands of markers genotyped on
		thousands of samples. BEAGLE can</p>
		<li><p style="margin-bottom: 0.14in">phase genotype data (i.e.
		infer haplotypes) for unrelated individuals, parent-offspring
		pairs, and parent-offspring trios.</p>
		<li><p style="margin-bottom: 0.14in">infer sporadic missing
		genotype data.</p>
		<li><p style="margin-bottom: 0.14in">impute ungenotyped markers
		that have been genotyped in a reference panel.</p>
		<li><p style="margin-bottom: 0.14in">perform single marker and
		haplotypic association analysis.</p>
		<li><p style="margin-bottom: 0.14in">detect genetic regions that
		are homozygous-by-descent in an individual or identical-by-descent
		in pairs of individuals.</p>
		<li><p style="margin-bottom: 0.14in">authors: Brian L Browning,
		Sharon R Browning (email: browning@stat.auckland.ac.nz),</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.auckland.ac.nz/~browning/beagle/beagle.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux, UNIX(Solaris), MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Browning SR and
		Browning BR (2007), &quot;Rapid and accurate haplotype phasing and
		missing-data inference for whole-genome association studies by use
		of localized haplotype clustering&quot;, American Journal of Human
		Genetics, 81(5):1084-1097</p>
		<li><p style="margin-bottom: 0.14in">Browning BR and Browning SR
		(2007), &quot;Efficient multilocus association testing for whole
		genome association studies using localized haplotype clustering&quot;,
		Genetic Epidemiology, 31:365-375.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BEAGLECALL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.0 (September
		2010)</p>
		<li><p style="margin-bottom: 0.14in">description: BEAGLECALL is a
		software package for simultaneous genotype calling and haplotype
		phasing for unrelated individuals. BEAGLECALL produces output
		posterior genotype probabilities and output phased haplotypes.
		BEAGLECALL generates extremely accurate genotype calls because it
		uses both allele signal intensity data and inter-marker correlation
		to call genotypes. BEAGLECALL is designed for use with high-density
		SNP arrays, and it uses the BEAGLE haplotype frequency model to
		model inter-marker correlation.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://faculty.washington.edu/browning/beaglecall/beaglecall.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX, Linux, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Browning, Yu
		(2009), &quot;Simultaneous genotype calling and haplotype phase
		inference improves genotype accuracy and reduces false positive
		associations for genome-wide association studies&quot;, American
		Journal of Human Genetics, 85:847-861.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BEAM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Bayesian Epistasis
		Association Mapping</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: BEAM treats the
		disease-associated markers and their interactions via a bayesian
		partitioning model and computes, via Markov chain Monte Carlo, the
		posterior probability that each marker set is associated with the
		disease.</p>
		<li><p style="margin-bottom: 0.14in">authors: Yu Zhang, Jun S Liu
		(Harvard University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.people.fas.harvard.edu/~junliu/BEAM/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		Linux, DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhang, Liu (2007),
		&quot;Bayesian inference of epistatic interactions in case-control
		studies&quot;, Nature, 39:1167-1173.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BETA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Jan 1998</p>
		<li><p style="margin-bottom: 0.14in">description: non-parametric
		linkage analysis using allele sharing in sib pairs</p>
		<li><p style="margin-bottom: 0.14in">authors: A Collins (email:
		arc@soton.ac.uk), NE Morton (University of Southampton, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/BETA</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX(SUN..)</p>
		<li><p style="margin-bottom: 0.14in">executables: beta</p>
		<li><p style="margin-bottom: 0.14in">reference: Morton (1996),
		Proceedings of the National Academy of Sciences , 93:3471-3476;
		Collins, Morton (1996), Proceedings of the National Academy of
		Sciences , 93:9177-9181; Lio, Morton (1997), Proceedings of the
		National Academy of Sciences , 94:5344-5348; online manual</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BIMBAM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Bayesian
		IMputation-Based Association Mapping</p>
		<li><p style="margin-bottom: 0.14in">version: 0.95</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: B Servin, M Stephens</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://stephenslab.uchicago.edu/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Servin, Stephens
		(2007), &quot;Imputation-based analysis of association studies:
		candidate genes and quantitative traits&quot;, PLoS Genetics,
		3(7):e114.
		http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0030114</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BIOIDE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: BIOmedical
		Integrated Discovery Environment</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Developed by
		scientists for scientists, our Biomedical Integrated Discovery
		Environment provides easy access to a rapidly expanding collection
		of data analysis tools and interrelated biomedical information in a
		single easy-to-use software package with an extremely high
		benefit/cost ratio. Our component-based software/data integration
		platform can quickly transform fragmented enterprise legacy data
		and software into an integrated suite of tools and knowledge base
		to achieve maximal usability, interoperability, scalability, and
		extensibility.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.htbiology.com/software.php</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BIOLAD-DB</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0.1</p>
		<li><p style="margin-bottom: 0.14in">description: The BiolAD-DB
		system is a research bioinformatics system for inputting,
		validating, organizing, archiving, analyzing, and processing of
		complex clinical and genetic data. The database schema employs
		design principles for handling complex clinical information, such
		as response items in genetic questionnaires. Data access and
		validation is provided by the BiolAD-DB client application, which
		features a data validation engine tightly coupled to a graphical
		user interface. Data integrity is provided by the password
		protected BiolAD-DB SQL compliant server database. BiolAD-DB tools
		further provide functionalities for generating customized reports
		and views.</p>
		<li><p style="margin-bottom: 0.14in">authors: David Nielsen (email:
		nielsen@rockefeller.edu), Marty Leidner (email:
		marty@rockefeller.edu), Chad Haynes (email:
		haynesc@rockefeller.edu), Michael Krauthammer (email:
		michael.krauthammer@yale.edu), Mary Jeanne Kreek (email:
		kreek@rockefeller.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.rockefeller.edu/biolad-db/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Python</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Nielsen, Leidner,
		Haynes, Krauthammer, Kreek (2007), &quot;The BiolAD-DB system - an
		informatics system for clinical and genetic data&quot;, Molecular
		Diagnosis &amp; Therapy, 11(1):15-19.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BIRDSUITE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The Birdsuite is
		a fully open-source set of tools to detect and report SNP
		genotypes, common Copy-Number Polymorphisms (CNPs), and novel,
		rare, or de novo CNVs in samples processed with the Affymetrix
		platform. While most of the components of the suite can be run
		individually (for instance, to only do SNP genotyping), the
		Birdsuite is especially intended for integrated analysis of SNPs
		and CNVs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Joshua Korn et al.
		(Broad Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/birdsuite/birdsuite</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Korn, Kuruvilla,
		McCarroll, Wysoker, Nemesh, Cawley, Hubbell, Veitch, Collins,
		Darvishi, Lee, Nizzari, Gabriel, Purcell, Daly, Altshuler (2008),
		&quot;Integrated genotype calling and association analysis of SNPs,
		common copy number polymorphisms and rare CNVs&quot;, Nature
		Genetics, 40:1253-1260.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BLADE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Bayesian LinkAge
		DisEquilibrium mapping</p>
		<li><p style="margin-bottom: 0.14in">version: v2</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jun Liu (Harvard
		Univ) (email: jliu@stat.harvard.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.people.fas.harvard.edu/~junliu/index1.html</p>
		<li><p style="margin-bottom: 0.14in">http://www.fas.harvard.edu/~junliu/TechRept/03folder/bladev2.tgz</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Liu, Sabatti, Teng,
		Keats, Risch (2001), &quot;Bayesian analysis of haplotypes for
		linkage disequilibrium mapping&quot;, Genome Research,
		11:1716-1724.</p>
		<li><p style="margin-bottom: 0.14in">Lu, Niu, Liu (2003),
		&quot;Haplotype information and linkage disequilibrium mapping for
		single nucleotide polymorphisms&quot;, Genome Research,
		13:2112-2117.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BLOCK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Blocking Gibbs
		sampler for pedigree analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0.0 (Dec 1996)</p>
		<li><p style="margin-bottom: 0.14in">description: the programs
		allow you to perform general pedigree analysis on a general
		pedigree with any number of loops. It also allows you to perform
		two-point linkage analysis on a general pedigree with an arbitrary
		number of alleles.</p>
		<li><p style="margin-bottom: 0.14in">authors: Claus Skaanning
		Jensen (email: claus@cs.auc.dk) (Aalborg University, Denmark)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cs.auc.dk/~claus/block.html</p>
		<li><p style="margin-bottom: 0.14in">or a copy from
		http://hpcio.cit.nih.gov/lserver/BLOCK.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.cs.auc.dk/pub/packages/block/current</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		Linux, UNIX(Solaris, Irix 64, AIX 3.2.5, DEC alpha)</p>
		<li><p style="margin-bottom: 0.14in">executables: block, theta</p>
		<li><p style="margin-bottom: 0.14in">reference: Jensen, Kong and
		Kjaerulff, &quot;Blocking-Gibbs Sampling in Very Large
		Probabilistic Expert Systems&quot;, International Journal of
		Human-Computer Studies, 647-666 (1995).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BMAPBUILDER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1(beta)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bios.ugr.es/BMapBuilder/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		MacOS, UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Abad-Grau, Montes,
		Sebastiani (2006), &quot;Building chromosome-wide LD maps&quot;,
		Bioinformatics, 22(16):1933-1934.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BOOLD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (July 2002)</p>
		<li><p style="margin-bottom: 0.14in">description: a set of programs
		for calculations under different linkage disequilibrium (LD)
		distribution models</p>
		<li><p style="margin-bottom: 0.14in">authors: YS Aulchenko (email:
		aulchenko@epib.fgg.eur.nl) (Erasmus University Medical School. The
		Netherlands)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.geneticepi.com/Research/software/software.html</p>
		<li><p style="margin-bottom: 0.14in">http://www.geneticepi.com/Research/software/booLD-1.1.tar.gz</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++,
		FORTRAN 77, Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: kLD, lsqLD,
		booit.pl</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BOOST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: BOolean Operation
		based Screening and Testing</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Xiang Wan, Can Yang
		(email: eeyang@ust.hk), Qiang Yang, Hong Xue, Xiaodan Fan, Nelson
		Tang, Weichuan Yu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinformatics.ust.hk/BOOST.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wan, Yang, Yang,
		Xue, Fan, Tang, Yu (2010), &quot;BOOST: a Boolean
		representation-based method for detecting SNP-SNP interactions in
		genome-wide association studies&quot;&quot;, American Journal of
		Human Genetics, 87:325-340.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BOOSTRAPPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: Robust haploblock
		border reliability estimation tool is implemented for LD based
		haploblock detection method. The most important new features are
		bootstrapping and overlapping block borders.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinfo.ebc.ee/download/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Window,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BOREL (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: July 1996</p>
		<li><p style="margin-bottom: 0.14in">description: Programs for
		inference of genealogical relationships from genetic data,
		including sibship inference.</p>
		<li><p style="margin-bottom: 0.14in">authors: Elizabeth A Thompson
		(University of Washington), Ian Painter (now at NCSU, email:
		ian@statgen.ncsu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.u.washington.edu/pub/user-supported/pangaea/PANGAEA/BOREL</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(DEC-UNIX/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BOTTLENECK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2.02</p>
		<li><p style="margin-bottom: 0.14in">description: a program for
		detecting recent effective population size reductions from allele
		data frequencies</p>
		<li><p style="margin-bottom: 0.14in">authors: Sylvian Piry</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.montpellier.inra.fr/URLB/bottleneck/bottleneck.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Piry, Luikart,
		Cornuet (1999), &quot;BOTTLENECK: a computer program for detecting
		recent reductions in the effective size using allele frequency
		data&quot;, Journal of Heredity, 90:502-503.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BPPH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Berkeley method for
		Perfect Phylogeney Haplotyping</p>
		<li><p style="margin-bottom: 0.14in">version: April 2003</p>
		<li><p style="margin-bottom: 0.14in">description: a program for
		inferring haplotypes from genotypes to determine if there are
		resulting haplotypes that fit a tree model (i.e. a perfect
		phylogeny, a coalescent). In population genetic terms, BPPH
		determines whether a set of SNP genotypes can be explained by
		haplotype pairs that could have evolved on a coalescent under the
		no-recombination, infinite sites model.</p>
		<li><p style="margin-bottom: 0.14in">authors: Ren-Hua Chuang, Dan
		Gusfield (UC Davis) (email: gusfield@cs.ucdavis.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wwwcsif.cs.ucdavis.edu/~gusfield/bpph.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: D. Gusfield (2002),
		&quot;Haplotyping as perfect phylogeny: conceptual framework and
		efficient solutions&quot;, In Proceedings of RECOMB, Sixth Annual
		Conference on Research in Computational Molecular Biology, pp.?-?.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Bayesian
		Quantitative Trait Locus mapping</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: for the mapping
		of genetic traits from line crosses and recombinant inbred lines.
		It performs (1) maximum likelihood estimation of multi-gene models;
		(2) Bayesian estimation of multi-gene models via Laplace
		Approximations; and (3) interval mapping and composite interval
		mapping of genetic loci</p>
		<li><p style="margin-bottom: 0.14in">authors: Justin Borevitz,
		Charles C. Berry (UCSD)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hacuna.ucsd.edu/bqtl/</p>
		<li><p style="margin-bottom: 0.14in">source code language: S, C,
		FORTRAN. The software is engineered to work in conjunction with R.</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables: cvl.blup.R(),
		cvl.epi.R(), cvl.jack.R(), cvl.qtl.effect.R(), read.map.R(),
		read.markers.R().</p>
		<li><p style="margin-bottom: 0.14in">reference: JO Borevitz, JN
		Maloof, J Lutes, T Dabi, JL Redfern, GT Trainer, JD Werner, T
		Asami, CC Berry, D Weigel, J Chory (2002), &quot;Quantitative trait
		loci controlling light and hormone response in two accessions of
		Arabidopsis thaliana&quot;, Genetics, 160(2):683-696.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">BREAKDANCER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: BreakDancer
		provides genome-wide detection of structural variants from next
		generation paired-end sequencing reads. BreakDancerMax predicts
		five types of structural variants: insertions, deletions,
		inversions, inter- and intra-chromosomal translocations from
		next-generation short paired-end sequencing reads using read pairs
		that are mapped with unexpected separation distances or
		orientation.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://gmt.genome.wustl.edu/breakdancer/current/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl,
		C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Chen, Wallis,
		McLellan, Larson, Kalicki, Pohl, McGrath, Wendl, Zhang, Locke, Shi,
		Fulton, Ley, Wilson, Ding, Mardis (2009), &quot;BreakDancer: an
		algorithm for high-resolution mapping of genomic structural
		variation&quot;, Nature Methods, 6:677-681.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CAROL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Combined Annotation
		scoRing toOL</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: CAROL is a
		combined functional annotation score of non-synonymous coding
		variants. A major challenge in interpreting whole-exome data is
		predicting which of the discovered variants are deleterious or
		neutral. To address this question in silico, we have developed a
		score called Combined Annotation scoRing toOL (CAROL),which
		combines information from two bioinformatics tools: PolyPhen-2 and
		SIFT, in order to improve the prediction of the effect of
		non-synonymous coding variants. The combination of annotation tools
		can help improve automated prediction of whole-genome/exome
		non-synonymous variant functional consequences.</p>
		<li><p style="margin-bottom: 0.14in">authors: Eleftheria Zeggini's
		group (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/carol/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CARTHAGENE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (January 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: It is a
		genetic/radiation hybrid mapping software. CARTHAGENE looks for
		multiple populations maximum likelihood consensus maps using a fast
		EM algorithm for maximum likelihood estimation and powerful
		ordering algorithms inspired from TSP (Traveling Salesman Problem)
		technology. It can handle large data sets made up of different
		populations (either F2 backcross, recombinant inbred lines, F2
		intercross, phase known outbreds, haploid/diploid radiation
		hybrids). It can also exploit existing syntenic relationships
		between the organism mapped and a reference (sequenced) organism
		for accurate dense RH mapping.</p>
		<li><p style="margin-bottom: 0.14in">authors: P Chabrier, D de
		Givry, T Faraut, C Gaspin, T Schiex (INRA, Toulouse, France)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.inra.fr/mia/T/CarthaGene/</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		C++,Tcl/Tk</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX(Solaris), Linux. Sources are available for other operating
		systems (GPL/QPL).</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Schiex,Gaspin,
		Proceedings of the International Conference on Intelligent Systems
		for Molecular Biology (ISMB) (1997).</p>
		<li><p style="margin-bottom: 0.14in">Givry, Bouchez, Chabrier,
		Milan, Schiex (2005), &quot;CARTHAGENE: multipopulation integrated
		genetic and radiation hybrid mapping&quot;, Bioinformatics,
		21(8):1703-1704.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CASAVA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Consensus
		Assessment of Sequence And VAriation</p>
		<li><p style="margin-bottom: 0.14in">version: v1.6</p>
		<li><p style="margin-bottom: 0.14in">description: Illumina's CASAVA
		software captures summary information for resequencing and counting
		studies and places the data in a compact structure for
		visualization within GenomeStudio Software or publicly available
		analysis tools. CASAVA can create genomic builds, call SNPs,
		detects indels, and count reads from data generated from one or
		more runs of the Genome Analyzer across a broad range of sequencing
		applications.</p>
		<li><p style="margin-bottom: 0.14in">v.1.6 highlights: 1. complete
		secondary analysis software package, including facilities for
		alignment, reference-guided assembly, SNP/indel calling and
		counting for RNA applications. 2. new methods for gapped, multiseed
		alignments that reduce artifactual mismatches. 3. new indel
		detection method, and improved SNP calling on longer Genome
		Analyzer reads.</p>
		<li><p style="margin-bottom: 0.14in">authors: (Illumina)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.illumina.com/software/genome_analyzer_software.ilmn</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">price:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CASPAR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Computerized
		Affected Sibling Pair Analyzer and Reporter</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: CASPAR's main
		novel feature is conditional linkage analyses, in which the
		population can be subdivided according to criteria at some loci and
		analyzed for linkage at other loci. CASPAR uses simulation to
		overcome the problems inherent in such multiple testing.</p>
		<li><p style="margin-bottom: 0.14in">authors: Richa Agarwala
		(email: richa@helix.nih.gov) and Alejandro Schaffer (email:
		schaffer@helix.nih.gov); other participants include Jeremy Buhler,
		Kenneth Gabbay, Marek Kimmel, David Owerbach</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/caspar.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://fastlink.nih.gov/pub/caspar</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1995 (Buhler, et. al. American Journal of Human
		Genetics, 57 Supp, A171 (1995));</p>
		<li><p style="margin-bottom: 0.14in">Buhler, et. al. (1997) Human
		Heredity, 47(4):211-222.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CATS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: power Calculator
		for Association with Two Stage design</p>
		<li><p style="margin-bottom: 0.14in">version: 0.0.1</p>
		<li><p style="margin-bottom: 0.14in">description: CATS calculates
		the power and other useful quantities for two-stage genome-wide
		association studies</p>
		<li><p style="margin-bottom: 0.14in">authors: Andrew Skol (email:
		askol@umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/CaTS/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		Linux, MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables: cats (graphical
		interface for MS-Windows, MacOS, and Linux), cats-test (Linux
		command line)</p>
		<li><p style="margin-bottom: 0.14in">reference: Skol, Scott,
		Abecasis, Boehnke (2006), &quot;Joint analysis is more efficient
		than replication-based analysis for two-stage genome-wide
		association studies&quot;, Nature Genetics, 38:209-213.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CC-QLS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Case-Control
		Quasi-Likelihood Score test</p>
		<li><p style="margin-bottom: 0.14in">version: 1.3 (March 2006)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Catherine Bourgain,
		Sabine Hoffjan, Raluca Nicolae, Dina Newman, Lori Steiner, Karen
		Walker, Rebecca Reynolds, Carole Ober, Mary Sara McPeek</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.uchicago.edu/~mcpeek/software/CCQLSpackage1.3/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Bourgain, Hoffjan,
		Nicolae, Newman, Steiner, Walker, Reynolds, Ober, McPeek (2003),
		&quot;Novel case-control test in a founder population identifies
		p-selectin as an Atopy-susceptibility locus&quot;, American Journal
		of Human Genetics, 73(3):612-626</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CCRAVAT (see also QUTIE)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Case-Control RAre
		Variant Analysis Tool</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Enabling the
		analysis of rare variants in large-scale case control and
		quantitative trait association studies. CCRaVAT (Case-Control Rare
		Variant Analysis Tool) and QuTie (Quantitative Trait) are software
		packages that enable efficient large-scale analysis of rare
		variants across specific regions or genome-wide. These programs
		implement a rare variant super-locus or collapsing method that
		investigates the accumulation of rare variant alleles in either a
		case-control or quantitative trait study design.</p>
		<li><p style="margin-bottom: 0.14in">authors: Eleftheria Zeggini's
		group (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/rarevariant/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CCREL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Case-Control
		association analysis with RELlated individuals</p>
		<li><p style="margin-bottom: 0.14in">version: 0.3 (December 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: CCREL is a
		program for case-control genetic analysis that takes relatedness
		between individuals into account. It will perform single-marker and
		haplotypic tests, however it will only work with SNP or other
		biallelic markers.</p>
		<li><p style="margin-bottom: 0.14in">authors: Sharon R. Browning
		(e-mail: browning@stat.auckland.ac.nz)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.auckland.ac.nz/~browning/ccrel/ccrel.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R with
		Perl and C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Unix/Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Browning, Briley,
		Briley, Chandra, Charnecki, Ehm, Johansson, Jones, Karter, Yarnall,
		Wagner (2005), &quot;Case-control single marker and haplotypic
		association analysis of pedigree data&quot;, Genetic Epidemiology,
		28:110-122.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CEPH2CRI</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: May 1994</p>
		<li><p style="margin-bottom: 0.14in">description: convert output
		from CEPH DBMS to CRIMAP format</p>
		<li><p style="margin-bottom: 0.14in">authors: John Attwood (Sanger
		Centre, UK) ( email: ja1@sanger.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.gene.ucl.ac.uk/public-files/packages/linkage_utils/ceph2cri/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CEPH2MAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: it is developed
		from crimap v2.4 for use with the map suite of programs.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ceph2map</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: ceph2map</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CFC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Contribution,
		Inbreeding (F) and Coancestry</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (April 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: This is a
		general-purpose program for monitoring genetic diversity. CFC
		allows for several pedigree analyses such as: (1) computing
		inbreeding coefficients and relationships; (2) computing average
		relationships very quickly within and between specified groups of
		individuals; (3) computing average relatedness; (4) ancestral
		decomposition of the average or the individual inbreeding
		coefficient; (5) ancestral decomposition of the average coancestry;
		(6) optimizing matings to minimize the average inbreeding
		coefficients in the next generation; (7) computing founder
		equivalent, founder genome equivalent and effective number of
		non-founders; (8) computing numerator relationship matrix, its
		Cholesky decomposition and its inverse; (9) providing useful
		information on the structure of pedigrees</p>
		<li><p style="margin-bottom: 0.14in">authors: Mehdi Sargolzaei
		(University of Guelph), Hiroaki Iwaisaki and Jean-Jacques Colleau</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://agrews.agr.niigata-u.ac.jp/~iwsk/cfc.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Visual
		C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(98/NT/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: M Sargolzaei, H
		Iwaisaki, JJ Colleau (2006), &quot;CFC: a tool for monitoring
		genetic diversity&quot;, Proceedings of the 8th World Congress on
		Genetics Applied to Livestock Production, CD-Rom comm. N 27-28.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHAPLIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Case-control
		HAPLotype INference package</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2.2</p>
		<li><p style="margin-bottom: 0.14in">description: Chaplin is a
		software program for identifying specific haplotypes or haplotype
		features that are associated with disease using genotype data from
		a case-control study.</p>
		<li><p style="margin-bottom: 0.14in">authors: Richard Duncan,
		Michael Epstein, Glen Satten (Emory University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genetics.emory.edu/labs/epstein/software/chaplin/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN90 (CVF 6.6) with IMSL routines</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables: chaplin.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: M Epstein, G Satten
		(2003), &quot;Inference on haplotype effects in case-control
		studies using unphased genotype data&quot;, American Journal of
		Human Genetics, 73(6):1316-1329</p>
		<li><p style="margin-bottom: 0.14in">Satten, Epstein (2004),
		&quot;Comparison of prospective and retrospective methods for
		haplotype inference in case-control studies&quot;, Genetic
		Epidemiology, 27(3):192-201</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHECKHET</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: March'02 version</p>
		<li><p style="margin-bottom: 0.14in">description: a program to
		detect genetically abnormal subjects in a case-control sample based
		on genotypes at multiple marker loci.</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis (Royal
		London Hospital, UK) (email: dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">http://www.smd.qmul.ac.uk/statgen/software/dcurtis/chhet10.zip</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: D Curtis, BV North,
		H Gurling, E Blaveri, PC Sham (2002), &quot;A quick and simple
		method for detecting subjects with abnormal genetic background in
		case-control samples&quot;, Annals of Human Genetics, 66:235-244.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHECKMATRIX (other name:
	PY_MATRIX_2D)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: v248</p>
		<li><p style="margin-bottom: 0.14in">description: CheckMatrix
		serves as a visualization tool to validate constructed genetic
		maps. CheckMatrix generates graphical genotypes and two-dimensional
		heat plots of pairwise scores. Visualization of regions with
		positive and negative linkage as well as of allele fraction per
		marker simplifies genetic map validation without applying
		statistical approaches. CheckMatrix works in conjunction with
		MadMapper and freely available at
		http://www.atgc.org/XLinkage/MadMapper/</p>
		<li><p style="margin-bottom: 0.14in">authors: Alexander Kozik (UC
		Davis)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.atgc.org/XLinkage/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Python</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHIAMO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.2.1 (Oct 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: CHIAMO is a
		program for calling genotypes from the Affymetrix 500K Mapping
		chip. The program allows for multiple cohorts which have
		potentially different intensity characteristics that can lead to
		elevated false-positive rates in genome-wide studies. The
		underlying model has a hierarchical structure that allows for
		correlation between the parameters of each cohort. The output files
		produced by CHIAMO feed directly into both the programs SNPTEST and
		IMPUTE. CHIAMO was used to call genotypes for the 7 genome-wide
		association studies carried out by the Wellcome Trust Case-Control
		Consortium (WTCCC).</p>
		<li><p style="margin-bottom: 0.14in">authors: Jonathan Marchini</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/~marchini/software/gwas/chiamo.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Marchini, Spencer,
		Teo, Donnelly (2007), &quot;A Bayesian hierarchical mixture model
		for genotype calling in a multi-cohort study&quot;,</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHIP2SPELL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.2 (September
		2009)</p>
		<li><p style="margin-bottom: 0.14in">description: Chip2Spell aims
		to speed up the preparation of linkage data files passed on to the
		Alohomora software for data from the Affymetrix or Illumina assays.
		Chip2Spell gathers the information necessary to supplement the
		genotype data (such as the genetic map of the markers and the
		population-specific allele frequencies) from publicly available
		annoation files from Affymetrix and Illumina. The program formats
		the genotype data for input into the Alohomora program.</p>
		<li><p style="margin-bottom: 0.14in">authors: Frederic Fournier
		(email: frederic.fournier@statgen.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.statgen.org/main/index.php/Downloads/Downloads</p>
		<li><p style="margin-bottom: 0.14in">source code language: perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHROMOSCAN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: CHROMOSCAN is an
		implementation of a genome-based scan statistic that detects
		genomic regions, which are statistically significant for targeted
		measurements, such as genetic associations with disease, gene
		expression profiles, DNA copy number variations, as well as other
		genome-based measurements.</p>
		<li><p style="margin-bottom: 0.14in">authors: Yan V Sun
		(email:yansun@umich.edu), Douglas M Jacobsen and Sharon LR Kardia
		(email: kardiasoftware@umich.edu) (University of Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.epidkardia.sph.umich.edu/software/chromoscan/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sun, Jacobsen,
		Kardia (2006), &quot;ChromoScan: a scan statistic application for
		identifying chromosomal regions in genomic studies&quot;,
		Bioinformatics, 22(23):2945-2947.</p>
		<li><p style="margin-bottom: 0.14in">Sun, Levin, Boerwinkle,
		Robertson, Kardia (2006), &quot;A scan statistic for identifying
		chromosomal patterns of SNP association&quot;, Genetic
		Epidemiology, 30(7):627-635.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHROMOSEG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: A simulation
		program that simulates and plots (in real time) ancestral
		recombination graphs. This is currently primarily a
		teaching/educational tool.</p>
		<li><p style="margin-bottom: 0.14in">authors: Eric C Anderson
		(email: eric.anderson@stanfordalumni.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CHROMSCAN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: beta (December 11,
		2006)</p>
		<li><p style="margin-bottom: 0.14in">description: CHROMSCAN is a
		statistical based program for association mapping of disease genes.
		It utilises the Malecot model and the linkage disequilibrium (LD)
		map for the candidate region to analyse the genotypes derive from
		large sample of matched cases and controls.</p>
		<li><p style="margin-bottom: 0.14in">authors: A.R. Collins
		(University of Southampton)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.som.soton.ac.uk/research/geneticsdiv/epidemiology/chromscan/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CLUMP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Monte Carlo
		method for assessing significance of a case-control association
		study with multi-allelic marker.</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis (Royal
		London Hospital, UK) (email: dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sham, Curtis
		(1995), Annals of Human Genetics, 59:97-105; online documentation</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CLUSTAG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2</p>
		<li><p style="margin-bottom: 0.14in">description: hierarchical
		clustering and graph methods for selecting tage SNPs</p>
		<li><p style="margin-bottom: 0.14in">authors: Sio Iong Ao, Michael
		K Ng (Dept of Mathematics, University of Hong Kong), Kevin Yip,
		David Cheung (Dept of Computer Science, University of Hong Kong),
		Pak Sham, Pui Yee Fong, Ian Melhado (Genome Research Center,
		University of Hong Kong),</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hkumath.hku.hk/web/link/CLUSTAG/CLUSTAG.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ao, Yip, Ng,
		Cheung, Fong, Melhado, Sham (2005), &quot;CLUSTAG: hierarchical
		clustering and graph methods for selecting tag SNPs&quot;,
		Bioinformatics, 21(8):1735-1736.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: genetic and
		comparative maps</p>
		<li><p style="margin-bottom: 0.14in">version: 0.06 (June 14, 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: CMAP is a
		web-based tool that allows users to view comparisons of genetic and
		physical maps. The package also includes tools for curating map
		data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Ken Y Clark (email:
		kclark@cshl.org), Lincoln Stein (email: lstein@cshl.org), Doreen
		Ware (email: ware@cshl.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.gmod.org/cmap/</p>
		<li><p style="margin-bottom: 0.14in">http download:
		https://sourceforge.net/project/showfiles.php?group_id=27707</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Solaris, FreeBSD), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: cmap_admin.pl</p>
		<li><p style="margin-bottom: 0.14in">reference: Ware et al. (2002),
		&quot;Gramene, a tool for grass genomics&quot;, Plant Physiology,
		130:1606-1613.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">COMBIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: Designed for the
		construction of highly saturated linkage maps, based on BC1, DH,
		Radiation Hybrid or CP (CrossPollinators) data sets. F2 is not
		supported.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jaap Buntjer (Lab of
		Plant Breeding, Wageningen University. email:
		jaap.buntjer@users.pv.wau.nl), Herman van Eck (email:
		herman.vaneck@users.pv.wau.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dpw.wau.nl/pv/pub/combin/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Visual
		Basic 5</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">COMDS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: for combined
		segregation and linkage analysis, incorporating severity and
		diathesis.</p>
		<li><p style="margin-bottom: 0.14in">authors: NE Morton (University
		of Southampton, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/comds</p>
		<li><p style="margin-bottom: 0.14in">source code language: SUN
		FORTRAN (the command &quot;fsplit&quot; is needed)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(SunOS/..)</p>
		<li><p style="margin-bottom: 0.14in">executables: list1,com1, comds
		(shell script)</p>
		<li><p style="margin-bottom: 0.14in">reference: manual (November
		1993)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">COMPOSITELD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.2.1 (December 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: A method to
		compute composite measures of linkage disequilibrium, their
		variances and covariances, and statistical tests, for all pairs of
		alleles from two loci when linkage phase is unkown. An extension of
		Weir and Cockerham (1989) to apply to multi-allelic loci.</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel Schaid (Mayo
		Clinic)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R/S-PLUS</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">COPE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: COllaborative
		Pedigree drawing Environment</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: includes a Java
		program for drawing pedigrees and a standardized system for
		pedigree storage. Unlike other existing pedigree programs, this
		software is particularly intended for epidemiologists in the sense
		that it allows customized automatic drawing of large numbers of
		pedigrees and remote and distributed consultation of pedigrees.</p>
		<li><p style="margin-bottom: 0.14in">authors: L Brun-Samarcq, S
		Gallina, A Philippi, F Demenais, G Vaysseix, E Barillot (email:
		cope@infobiogen.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.infobiogen.fr/services/CoPE (closed)</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Brun-Samarcq, et
		al, (1999) Bioinformatics, 15(4):345-346. [ full text (PDF) ].</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">COVIBD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: CovIBD refines
		linkage analysis of affected sibpairs by considering attributes or
		environmental exposures thought to affect disease liability. This
		refinement utilizes a mixture model in which a disease mutation
		segregates in only a fraction of the sibships, with the rest of the
		sibships unlinked. Covariate information is used to predict
		membership within the two groups corresponding to the linked and
		unlinked sibships. The pre-clustering model uses covariate
		information to first form two probabilistic clusters and then tests
		for excess IBD-sharing in the clusters. The Cov-IBD model
		determines probabilistic group membership by joint consideration of
		covariate and IBD values.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/newcovibd/covibd.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Devlin, Jones,
		Bacanu, Roeder (2002), &quot;Mixture models for linkage analysis of
		affected sibling pairs and covariates&quot;, Genetic Epidemiology,
		22(1):52-65.</p>
		<li><p style="margin-bottom: 0.14in">Devlin, Jones, Bacanu, Roeder
		(2002), &quot;Reply to Olson&quot;, Genetic Epidemiology,
		23(4):449-455.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CRIMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.4</p>
		<li><p style="margin-bottom: 0.14in">description: constructing
		multilocus linkage map</p>
		<li><p style="margin-bottom: 0.14in">authors: Phil Green (Univ. of
		Washington, email: phg@u.washington.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://compgen.rutgers.edu/Crimap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows(XP)</p>
		<li><p style="margin-bottom: 0.14in">executables: crimap</p>
		<li><p style="margin-bottom: 0.14in">reference: Lander, Green
		(1987), Proceedings of the National Academy of Sciences ,
		84:2363-2367.</p>
		<li><p style="margin-bottom: 0.14in">documentation (version 2.4,
		1990).</p>
		<li><p style="margin-bottom: 0.14in">Tutorial (from biobase.dk)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CRIMAP-PVM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: CRIMAP with
		Parallel Virtual Machine</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a parallel
		version of CRIMAP</p>
		<li><p style="margin-bottom: 0.14in">authors: Tara Matise (email:
		matise@biology.rutgers.edu), Mark Schroeder, Donald Chiarulli,
		Daniel E. Weeks</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://compgen.rutgers.edu/multimap/crimappvm.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C, PVM</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Matise, Schroeder,
		Chiarulli, Weeks, (1995) Human Heredity, 45:103-116.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CROSSFIND (other names:
	BREAKPOINT, BPT)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: October 1995</p>
		<li><p style="margin-bottom: 0.14in">description: software for
		detecting and displaying well-characterized meiotic breakpoints in
		human family data. another program, CRIMAP, is used.</p>
		<li><p style="margin-bottom: 0.14in">authors: J. Attwood (Sanger
		Centre, email: ja1@sanger.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.gene.ucl.ac.uk/public-files/packages/jattwood/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX(SunOS)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Attwood, Povey
		(1996), Annals of Human Genetics, 60:487-498.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">CYRILLIC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version 3 (October 1999)</p>
		<li><p style="margin-bottom: 0.14in">description: (version 3)
		pedigree drawing with fully integrated risk analysis and support
		for industry standard databases (MS Access and Corel Paradox).
		designed for genetic counselors and others who work with patients.
		if you use genetic marker data you need Cyrillic 2. (version 2)
		this version of Cyrillic draws pedigrees, works with genetic marker
		data, lets you do haplotyping and allows exports to a range of
		linkage analysis packages.</p>
		<li><p style="margin-bottom: 0.14in">authors: Cherwell Scientific
		Publishing and Cyril Chapman (email: cyrillic@cherwell.com)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cyrillicsoftware.com</p>
		<li><p style="margin-bottom: 0.14in">http://www.cyrillicsoftware.com/download/downloads.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Visual
		C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Chapman (1990),
		American Journal of Medical Genetics, 36:155-160.</p>
		<li><p style="margin-bottom: 0.14in">availability: commercial v2.1
		(US$599/379 UK pounds/570 euros), 3 (US$769/479 UK pounds/725
		euro), upgrades from v2.1 to v3 (US$299), upgrades from v2 to v3
		(US$399),</p>
		<li><p style="margin-bottom: 0.14in">vender: Cyrillic Software, UK;
		Exeter Software (USA reseller)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DCHIP LINKAGE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Igor Leykin, Cheng Li
		(Harvard University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.harvard.edu/complab/dchip/snp.htm</p>
		<li><p style="margin-bottom: 0.14in">http://www.biostat.harvard.edu/~ileykin/accessory.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Leykin et al.
		(2005), &quot;Comparative linkage analysis and visualization of
		high-density oligonucleotide SNP array data&quot;, BMC Genetics,
		6:7.</p>
		<li><p style="margin-bottom: 0.14in">Yahoo discussion group:
		http://groups.yahoo.com/group/dchip/</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DGENE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a simple dBASE
		III program for the management of pedigree and locus data. It
		permits easy extraction of genetic data for use with MENDEL and
		FISHER.</p>
		<li><p style="margin-bottom: 0.14in">authors: Kenneth Lange (UCLA),
		Daniel E. Weeks (Pittsburgh), M. Boehnke</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: registration page at:
		http://www.biomath.medsch.ucla.edu/faculty/klange/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: dBASE
		III</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS, ...</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lange, Weeks,
		Boehnke, Genet Epid, 5, 471-471 (1988)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DHSMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Decay of Haplotype
		Sharing MAPping software</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0 (October 18,
		2001)</p>
		<li><p style="margin-bottom: 0.14in">description: DHSMAP is a
		program for fine-mapping of qualitative traits by linkage
		disequilibrium. Given a set of marker haplotypes or genotypes from
		affected individuals, haplotypes or genotypes from appropriately
		selected controls, and a genetic map of the markers at which both
		sets of individuals are typed, DHSMAP estimates the location of the
		trait-associated variant by maximum likelihood or maximum
		quasi-likelihood.</p>
		<li><p style="margin-bottom: 0.14in">authors: Andrew Strahs,
		Kenneth Wilder, Mary Sara McPeek (University of Chicago)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://galton.uchicago.edu/~mcpeek/software/dhsmap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris/..), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: dhsmap.solaris,
		dhsmap.linux</p>
		<li><p style="margin-bottom: 0.14in">reference: McPeek and Strahs
		(1999), &quot;Assessment of linkage disequilibrium by the decay of
		haplotype sharing, with application to fine-scale genetic mapping&quot;,
		American Journal of Human Genetics, 65(3):858-875</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DINDEL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Dindel is a
		program for calling small indels from short-read sequence data
		('next generation sequence data'). It is currently designed to
		handle only Illumina data. Dindel takes BAM files with mapped
		Illumina read data and enables researchers to detect small indels
		and produce a VCF file of all the variant calls.</p>
		<li><p style="margin-bottom: 0.14in">authors: Cornelis Albers
		(Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/dindel/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Albers, Lunter,
		MacArthur, McVean, Ouwehand, Durbin (2011), &quot;Dindel: Accurate
		indel calls from short-read data&quot;, Genome Research,
		21:961-973.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DISENTANGLER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: October 2011</p>
		<li><p style="margin-bottom: 0.14in">description: The disentangler
		is a visualization technique for linkage disequilibrium mapping and
		haplotype analysis of multiple multi-allelic genetic markers.</p>
		<li><p style="margin-bottom: 0.14in">authors: K Kumasaka et al.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://kumasakanatsuhiko.jp/projects/disentangler/</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		Java/JRE1.5</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Okada et al.
		(2011), &quot;HLA-Cw*1202-B*5201-DRB1*1502 haplotype increases risk
		for ulcerative colitis but reduces risk for Crohn's disease&quot;,
		Gastroenterology, 141(3):864-871.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: fine-scale
		disequilibrium mapping of a rare allele</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinko Graham</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.math.sfu.ca/~jgraham/Papers/Programs/DisequilibriumMapping/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Graham, Thompson
		(1998), &quot;Disequilibrium likelihoods for fine-scale mapping of
		a rare allele&quot;, American Journal of Human Genetics,
		63(5):1517-1530</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DMLE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Disease Mapping
		using Linkage disEquilibrium</p>
		<li><p style="margin-bottom: 0.14in">version: 2.3 (October 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: high-resolution
		mapping of the position of a disease mutation relative to a set of
		genetic markers using population linkage disequilibrium (LD).</p>
		<li><p style="margin-bottom: 0.14in">authors: Jeff Reeve, Ying
		Wang, Rannala (University Alberta, Canada)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: http://www.dmle.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Reeve, Rannala,
		Bioinformatics, 18(6):894-895 (2002).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DNABASER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">description: a tool for manual
		and automatic DNA sequence assembly, DNA sequence analysis,
		automatic sample processing, contig editing, metadata integration,
		file format conversion and mutation detection.</p>
		<li><p style="margin-bottom: 0.14in">web site:
		http://www.dnabaser.com/</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DOLINK (other name: WDLINK)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: manage genetic
		data</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis (Royal
		London Hospital, UK) (email: dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Cook, Gurling,
		Curtis (1993), &quot;DOLINK: a computer program to facilitate
		management of genetic data and analyses&quot;, Annals of Human
		Genetics, 57:307-310.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">DPPH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Direct method for
		Perfect Phylogeney Haplotyping</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: similar to the
		BPPH program</p>
		<li><p style="margin-bottom: 0.14in">authors: Ren-Hua Chuang, Dan
		Gusfield (UC Davis) (email: gusfield@cs.ucdavis.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wwwcsif.cs.ucdavis.edu/~gusfield/dpph.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Bafna, Gusfield,
		Lancia, Yooseph (2003), &quot;Haplotyping as perfect phylogeny: a
		direct approach&quot;, Journal of Computational Biology,
		10(3):323-340.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EAGLET</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Efficient Analysis
		of Genetic Linkage: Testing and Estimation</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: EAGLET is a
		software package that provides a number of improved statistics for
		detecting linkage and estimating trait location. EAGLET uses
		multiple subsamples of dense SNP data to detect linkage with
		increased power, and to construct sharp 95% confidence intervals
		for the true trait location. Subsamples with minimal amounts of LD
		are repeatedly drawn from the original dense SNP data, and the
		linkage information across subsamples is then combined. This
		minimizes the false positive evidence for linkage that LD can
		create in classical multipoint linkage analysis due to an apparent
		excess in allele sharing among affecteds. Another attractive
		feature of EAGLET is that by using multiple subsamples, as opposed
		to one, EAGLET's estimate of trait location is greatly improved.
		EAGLET can be used to generate subsamples for custom linkage
		packages as well, and computes standard LD statistics via the EM
		algorithm.</p>
		<li><p style="margin-bottom: 0.14in">authors: William C.L. Stewart
		(email: ws2267@columbia.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.columbia.edu/~ws2267/SOFT/EAGLET/eaglet.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl, C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables: eaglet.pl</p>
		<li><p style="margin-bottom: 0.14in">reference: Stewart, Peljto,
		Greenberg (2009), &quot;Multiple subsampling of dense SNP data
		localizes disease genes with increased precision&quot;, Human
		Heredity, to appear.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EASYLINKAGE/EASYLINKAGE-PLUS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 5.02</p>
		<li><p style="margin-bottom: 0.14in">description: easyLINKAGE
		combines automated setup and performance of linkage analyses and
		simulation under an easy to handle graphical user interface for
		Microsoft Windows 2000/XP and standard UNIX systems. The program
		package supports currently single-point linkage analyses (FastLink,
		SPLink, SuperLink), multi-point linkage analyses (GeneHunter,
		GeneHunter Plus, GeneHunter-Imprinting/-TwoLocus, Allegro, SimWalk,
		Merlin), and the simulation package SLink, and provides genome-wide
		as well as chromosomal postscript plots of LOD scores, NPL scores,
		P values, and many other parameters. The software can analyze STRPs
		as well as SNP chip data from Affymetrix, Illumina, or self-defined
		SNP data. The program performs single- and multi-point simulation
		studies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Tom H Lindner (email:
		t-lindner@gmx.de), Katrin Hoffmann (email: khoffma@gmx.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://compbio.charite.de/genetik/hoffmann/easyLINKAGE/</p>
		<li><p style="margin-bottom: 0.14in">http://sourceforge.net/projects/easylinkage/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl
		v5.8 (program can be provided as PERL script or as a compiled exe
		for the use in Windows)</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(2000/XP), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: linkage_v5.02.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Lindner, Hoffmann
		(2005), &quot;easyLINKAGE: A PERL script for easy and automated
		two-/multi-point linkage analyses&quot;, Bioinformatics,
		21(3):405-407.</p>
		<li><p style="margin-bottom: 0.14in">Hoffmann, Lindner (2005),
		&quot;easyLINKAGE-Plus - automated linkage analyses using
		large-scale SNP data&quot; Bioinformatics, 21(17):3565-3567.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EASYPOP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0.1 (July 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: EASYPOP is a
		computer program allowing to simulate population genetics datasets.
		It allows generating genetic data for haploid, diploid, and
		haplodiploid organisms under a variety of mating systems. It
		includes various migration and mutation models. Output can be
		generated for the FSTAT, GENEPOP, and ARLEQUIN genetic analysis
		packages.</p>
		<li><p style="margin-bottom: 0.14in">authors: Francois Balloux
		(email: fb255@gen.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.unil.ch/dee/page36926_fr.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/NT/2000/XP), MacOS(X)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Balloux (2001),
		&quot;EASYPOP (Version 1.7): a computer program for population
		genetics simulations&quot;, Journal of Heredity, 92:301-302.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ECLIPSE (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Error Correcting
		Likelihoods In Pedigree Structure Estimation.</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (October 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: A set of three
		programs, preproc, eclipse2 and eclipse3 which analyze genetic
		marker data for genotypic errors and pedigree errors. Using a
		single preprocessing program (preproc), eclipse2 analyzes data on
		pairs of individuals, and eclise3 analyzes data jointly on trios.</p>
		<li><p style="margin-bottom: 0.14in">authors: Solveig Sieberts
		(email: solly@stat.washington.edu)</p>
		<li><p style="margin-bottom: 0.14in">based on some earlier
		prototype routines by Jochen Kumm.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml</p>
		<li><p style="margin-bottom: 0.14in">http://www.stat.washington.edu/thompson/Genepi/Eclipse.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Tested on
		Unix (Compaq Tru64 V5.0A)</p>
		<li><p style="margin-bottom: 0.14in">executables: prepro, eclipse2,
		eclipse3</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EDAC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: power calculator
		for Extremely Discordant And Concordant sibpairs</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: C Charles Gu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.wustl.edu/~gc/cgi-bin/calcpwr.cgi</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gu, Todorov, Rao
		(1996), &quot;Combining extremely concordant sibpairs with
		extremely discordant sibpairs provides a cost effective way to
		linkage analysis of QTL&quot;, Genetic Epidemiology, 13:513-533.</p>
		<li><p style="margin-bottom: 0.14in">Gu, Rao (1997), &quot;A
		linkage strategy for detection of human quantitative-trait loci: I.
		Generalized relative risk ratios and power of sib pairs with
		extreme trait values&quot;,. American Journal of Human Genetics,
		61(1):200-210</p>
		<li><p style="margin-bottom: 0.14in">Gu, Rao (1997), &quot;A
		linkage strategy for detection of human quantitative-trait loci:
		II. Optimization of study designs based on extreme sib pairs and
		generalized relative risk ratios&quot;, American Journal of Human
		Genetics, 61(1):211-222</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EH (see also EHPLUS,
	FASTEHPLUS, EHP)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Estimating
		Haplotype-frequencies</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Xiaoli Xie, Jurg Ott
		(Columbia University; Rockefeller University)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: eh</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1993 (Xie, Ott (1993), &quot;Testing linkage
		disequilibrium between a disease gene and marker loci&quot;, Am J
		Hum Genet, 53 (suppl), 1107 (<a href="http://www.jurgott.org/linkage/absXieOtt1993.pdf">abstract</a>)</p>
		<li><p style="margin-bottom: 0.14in">online user's guide</p>
		<li><p style="margin-bottom: 0.14in">a product review from:
		http://www.scienceboard.net/resources/prodreviews.asp?cat=22&amp;product=440</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EHAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: detecting
		association between haplotypes and phenotypes</p>
		<li><p style="margin-bottom: 0.14in">authors: Howard Seltman, Shawn
		Wood (University of Pittburg)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/ehap__v1.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: ehap.exe,
		haplinmod.exe</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EHP (see also EH)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Estimation of
		Haplotype frequencies from Pooled DNA samples</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: This program
		provides variance estimates for haplotype frequency estimates, it
		allows several kinds of missing information in the genotype data,
		it also allows for combined genotype data of different pool sizes.
		This program can be used for testing haplotype-disease associations
		in case control studies by calculating the likelihood ratio test: 2
		log(likelihood for cases) + 2 log(likelihood for controls) - 2
		log(likelihood for case+controls).</p>
		<li><p style="margin-bottom: 0.14in">authors: Yaning Yang
		(University of Science and Technology, Hefei, Anhui Province,
		China) (email: ynyang@gmail.com)</p>
		<li><p style="margin-bottom: 0.14in">source code language: R/S-plus</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Y Yang et al.
		(2003), &quot;Efficiency of single-nucleotide polymorphism
		haplotype estimation from pooled DNA&quot;, Proceedings of the
		National Academy of Sciences , 100(12):7225-7230.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EHPLUS (see also EH,
	FASTEHPLUS)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinghua Zhao (MRC
		Epidemiology Unit at Strangeways Research Lab, UK) (email:
		jinghua.zhao@mrc-epid.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris/DEC alpha/..), MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: ehplus</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao, Curtis, Sham
		(2000), &quot;Model-free and permutation tests for allelic
		associations&quot;, Human Heredity, 50(2):133-139.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EIGENSOFT/EIGENSTRAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 4.2 (August 2011)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Alkes Price, David
		Reich (Harvard Medical School)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hsph.harvard.edu/faculty/alkes-price/software/</p>
		<li><p style="margin-bottom: 0.14in">http://genepath.med.harvard.edu/~reich/Software.htm</p>
		<li><p style="margin-bottom: 0.14in">http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Price, Patterson,
		Plenge, Weinblatt, Shadick, Reich (2006), &quot;Principal
		components analysis corrects for stratification in genome-wide
		association studies&quot;, Nature Genetics, 38(8):904-909.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EM-DECODER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a haplotype
		inference program</p>
		<li><p style="margin-bottom: 0.14in">authors: Steve Qin, (email:
		qin@stat.harvard.edu), Jun S Liu (email: jliu@stat.harvard.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.people.fas.harvard.edu/~junliu/em/em.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Niu, Qin, Xu, Liu
		(2002), &quot;Bayesian haplotype inference for multiple linked
		single-nucleotide polymorphisms&quot;, American Journal of Human
		Genetics, 70(1):157-169</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EMLD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: EM estimation of
		haplotype frequencies and LD calculation</p>
		<li><p style="margin-bottom: 0.14in">version: September 2003</p>
		<li><p style="margin-bottom: 0.14in">description: A quick and handy
		program to calculate pair-wise linkage disequilibrium based on SNP
		genotype data from unrelated individuals. EM algorithm is used to
		estimate pair-wise haplotype frequencies. The output file is in the
		format of input file for GOLD program, thus it can be directly plug
		into GOLD to get LD plots.</p>
		<li><p style="margin-bottom: 0.14in">authors: Dakai Zhu (email:
		dkz@mdanderson.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://epi.mdanderson.org/~qhuang/Software/pub.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: emld</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ENDOG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 4.0 (November 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		calculates individual inbreeding (F) and average relatedness (AR)
		coefficients. Additionally, users can compute useful parameters in
		population genetics such as: the number of ancestors explaining
		genetic variability; the genetic importance of the herds; F
		statistics from genealogical information.</p>
		<li><p style="margin-bottom: 0.14in">authors: Juan Pablo Gutierrez
		Garcia</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ucm.es/info/prodanim/html/JP_Web.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN
		77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ENTROPY BLOCKER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.7</p>
		<li><p style="margin-bottom: 0.14in">description: aiming at
		identifying &quot;haplotype blocks&quot;. The likelihood of the
		data is calculated minus the model complexity. The resulting blocks
		have very low diversity and the linkage disequilibrium with SNP's
		outside the blocks is low.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/blocks.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EPISTACY</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jim Holland (email:
		james_holland@ncsu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www4.ncsu.edu/~jholland/Epistacy/epistacy.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: SAS</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Holland (1998),
		&quot;EPISTACY: A SAS program for detecting two-locus epistatic
		interactions using genetic marker information&quot;, Journal of
		Heredity, 89:374-375.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EPDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Extended Pedigree
		Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (2004)</p>
		<li><p style="margin-bottom: 0.14in">description: EPDT is a major
		advancement in detecting linkage/disequilibrium signals # between
		genetic markers and disease loci, particularly if only one # or a
		few large pedigrees are available. The strategy differs from #
		conventional approaches that require at least a moderate number of
		# families to attain adequate statistical power. The proposed
		testing # procedure is advantageous in that it provides high
		statistical power # coupled with reduced sample collection.
		Furthermore, the proposed # method avoids problems such as
		potential population stratification # and genetic heterogeneity,
		and is robust with respect to misspecification # of phenotype.</p>
		<li><p style="margin-bottom: 0.14in">authors: Chun-Liang Chen,
		Cathy SJ Fann (Institute of Biomedical Sciences, Academia Sinica,
		Taiwan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ibms.sinica.edu.tw/~csjfann/first%20flow/programlist.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yang, Chen, Fann &quot;A
		pedigree-based disequilibrium test for a single extended pedigree&quot;.
		preprint.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EQTL EXPLORER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 5.0</p>
		<li><p style="margin-bottom: 0.14in">description: EQTL EXPORER is
		an eQTL visualization tool that allows users to mine and understand
		data from a repository of genetical genomics experiments</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Mueller,
		Jonathan Mangion (Imperial College, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://web.bioinformatics.ic.ac.uk/eqtlexplorer/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Mueller, Goel,
		Thimma, Dickens, Aitman, Mangion (2006), &quot;eQTL Explorer:
		integrated mining of combined genetic linkage and expression
		experiments&quot;, Bioinformatics, 22(4):509-511.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ERPA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Extended Relative
		Pair Analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: non-parametric
		analysis</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis (Royal
		London Hospital, UK) ( dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Curtis, Sham
		(1994), Annals of Human Genetics, 58:151-162.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ETDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: extended
		transmission/disequilibrium test</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: TDT test on
		markers with more than two alleles using a logistic regression
		analysis.</p>
		<li><p style="margin-bottom: 0.14in">authors: P.C. Sham (King's
		College London, UK), Dave Curtis (Royal London Hospital, UK) (
		dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sham, Curtis
		(1995), Annals of Human Genetics, 59:323-336.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ET-TDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Evolutionary Tree -
		Transmission Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Kathryn Roeder
		(Carnegie Mellon, email: roeder@stat.cmu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.cmu.edu/~roeder/=ettdt/</p>
		<li><p style="margin-bottom: 0.14in">ftp://wpicf.wpic.pitt.edu/WPICCompgen/ettdt/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: inferhap.exe,
		ettdt.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Selman, Roeder,
		Devlin (2001), &quot;Transmission/Disequilibrium Test Meets
		Measured haplotype analysis: family-based association analysis
		guided by evolution of haplotypes&quot;, American Journal of Human
		Genetics, 68(5):1250-1263</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EVOKER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: May 2010</p>
		<li><p style="margin-bottom: 0.14in">description: Evoker is a
		graphical tool for visualizing genotype intensity data in order to
		assess genotype calls as part of quality control procedures for
		genome-wide association studies. It provides a solution to the
		computational and storage problems related to being able to work
		with the huge volumes of data generated by such projects by
		implementing a compact, binary format that allows rapid access to
		data, even with hundreds of thousands of observations.</p>
		<li><p style="margin-bottom: 0.14in">authors: James Morris, Jeffrey
		Barrett (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/evoker/</p>
		<li><p style="margin-bottom: 0.14in">http://sourceforge.net/projects/evoker/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java,
		Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">EXOMEPICKS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: ExomePicks is a
		program that suggests individuals to be sequenced in a large
		pedigree. ExomePicks assumes that a genotyping chip or another cost
		effective means will be used to determine IBD sharing in the
		pedigree and that, subsequently, one would like to sequence a
		minimal number of individuals and use their sequences together with
		IBD information to deduce the sequence of other individuals in the
		pedigree. We are currently using it in the context of whole exome
		and whole genome sequencing studies to pick individuals to be
		sequenced from large family collections.</p>
		<li><p style="margin-bottom: 0.14in">authors: Abecasis' group (Univ
		Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/ExomePicks</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FAMHAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 16</p>
		<li><p style="margin-bottom: 0.14in">description: FAMHAP is a tool
		for haplotype association analysis of unphased genotype data. It
		can be used both for population data (case-control) and nuclear
		family data. The program is optimized for haplotype frequency
		estimation with the EM-algorithm for many markers. FAMHAP provides
		a method which searches for potential genotyping errors and several
		tests for haplotype-based association analysis. Particular emphasis
		is on Monte-Carlo simulations, which are necessary in the context
		of haplotype association, where asymptotic theory often fails, and
		in the context of multiple testing problems.</p>
		<li><p style="margin-bottom: 0.14in">authors: Tim Becker (email:
		tim.becker@ukb.uni-bonn.de), Michael Knapp</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uni-bonn.de/~umt70e/becker.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Becker, Knapp
		(2004), &quot;Maximum-likelihood estimation of haplotype
		frequencies in nuclear families&quot;, Genetic Epidemiology,
		27;21-32.</p>
		<li><p style="margin-bottom: 0.14in">Becker, Knapp (2004), &quot;A
		powerful strategy to account for multiple testing in the context of
		haplotype analysis&quot;, American Journal of Human Genetics,
		75:561-570</p>
		<li><p style="margin-bottom: 0.14in">Becker, Knapp (2005), &quot;Impact
		of missing genotype data on Monte-carlo simulation based haplotype
		analysis&quot;, Human Heredity, 59:185-189.</p>
		<li><p style="margin-bottom: 0.14in">Becker, Cichon, J&ouml;nson,
		Knapp (2005), &quot;Multiple testing in the context of haplotype
		analysis revisited: application to case-control data&quot;, Annals
		of Human Genetics, 69:1-10</p>
		<li><p style="margin-bottom: 0.14in">Becker, Schumacher, Cichon,
		Baur, Knapp (2005), &quot;Haplotype interaction analysis of
		unlinked regions&quot;, Genetic Epidemiology, 29:313-322.</p>
		<li><p style="margin-bottom: 0.14in">Becker, Valentonyte, Croucher,
		Strauch, Schreiber, Hampe, Knapp (2006), &quot;Identification of
		probable genotyping errors by consideration of haplotypes&quot;,
		European Journal of Human Genetics, 14:450-8.</p>
		<li><p style="margin-bottom: 0.14in">Becker, Baur, Knapp (2006),
		&quot;Detection of parent-of-origin effects using haplotype
		analysis&quot;, Human Heredity, 62:64-76</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FAMOZ</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: FAther/MOther</p>
		<li><p style="margin-bottom: 0.14in">version: 2003</p>
		<li><p style="margin-bottom: 0.14in">description: the program uses
		likelihood calculation and simulation to perform parentage studies
		with codominant, dominant, cytoplasmic markers or combinations of
		the different types</p>
		<li><p style="margin-bottom: 0.14in">authors: Sophie Gerber (email:
		sophie@pierroton.inra.fr), Patrick Chabrier, Antoine Kremer (INRA -
		UMR BIOGECO)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.pierroton.inra.fr/genetics/labo/Software/Famoz/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,
		Tcl/Tk</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris/..), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: S Gerber, S
		Mariette, R Streiff, C Bod&eacute;n&egrave;s, A Kremer (2000),
		&quot;Comparison of microsatellites and AFLP markers for parentage
		analysis&quot;, Molecular Ecology, 9:1037-1048.</p>
		<li><p style="margin-bottom: 0.14in">S Gerber, P Chabrier, A Kremer
		(2003), &quot;FaMoz: a software for parentage analysis using
		dominant, codominant and uniparentally inherited markers&quot;,
		Molecular Ecology Notes, 3(3):479-481.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FASTEHPLUS (see also EH,
	EHPLUS)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: faster EH-PLUS</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinghua Zhao (MRC
		Epidemiology Unit at Strangeways Research Lab, UK) (email:
		jinghua.zhao@mrc-epid.cam.ac.uk), Pak Sham (King's College London,
		UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX (DEC,OSF,Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao, Sham (2002),
		&quot;Faster allelic association analysis using unrelated
		subjects&quot;, Human Heredity, 53(1):36-41.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FASTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: FAmily SmarT
		EliminatoR</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 beta</p>
		<li><p style="margin-bottom: 0.14in">description: The program is to
		reduce family members so the families can be used in GENEHUNTER
		program. FASTER eliminates the most redundant individuals according
		to the set of weights (preferences) (Age, Information about genetic
		Markers, etc.) The program has several features such as automatical
		reducement of branches without any affected individuals, etc.</p>
		<li><p style="margin-bottom: 0.14in">authors: Cyril Hoschl (Charles
		Univ, Pague)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hoschl.cz/faster/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(9x/Me/2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FASTLINK (see also LINKAGE)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: faster version of
		LINKAGE</p>
		<li><p style="margin-bottom: 0.14in">version: 4.1P (July 4, 1999)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Cottingham, Gupta,
		Idury, Schaffer ( schaffer@helix.nih.gov), Shriram</p>
		<li><p style="margin-bottom: 0.14in">collaborators for version 4:
		Ann Becker, Dan Geiger.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/fastlink.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ncbi.nlm.nih.gov/pub/fastlink
		(since Aug 6, 1996) or
		ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/FASTLINK/fastlink</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS,.. and can also run in parallel on shared memory UNIX
		machines</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Cottingham, Idury,
		Schaffer (1993), &quot;Fast sequential genetic linkage
		computation&quot;, American Journal of Human Genetics,
		53(1):252-263</p>
		<li><p style="margin-bottom: 0.14in">Schaffer, Gupta, Shriram,
		Cottingham (1994), &quot;Avoiding recomputation in linkage
		analysis&quot;, Human Heredity, 44:225-237</p>
		<li><p style="margin-bottom: 0.14in">online README files (version
		3.0P, december 95).</p>
		<li><p style="margin-bottom: 0.14in">Dwarkadas, Schaffer,
		Cottingham, Cox, Keleher, Zwaenepoel, &quot;Parallelization of
		general linkage analysis problems&quot;</p>
		<li><p style="margin-bottom: 0.14in">Gupta, Schaffer,
		Cox,Zwaenepoel, &quot;Integrating parallelization strategies for
		linkage analysis&quot;</p>
		<li><p style="margin-bottom: 0.14in">Schaffer, &quot;Faster linkage
		analysis computations for pedigrees with loops or unused alleles&quot;</p>
		<li><p style="margin-bottom: 0.14in">Becker, Geiger, Schaffer,
		&quot;Automatic selection of loop breakers for genetic linkage
		analysis&quot;, Human Heredity</p>
		<li><p style="margin-bottom: 0.14in">Becker, Bar-Yehuda, Geiger,
		&quot;Random algorithms for the loop cutset problem&quot;</p>
		<li><p style="margin-bottom: 0.14in">Schaffer, &quot;Loops in
		FASTLINK&quot;</p>
		<li><p style="margin-bottom: 0.14in">Schaffer, &quot;The mystery of
		(the) unknown&quot;</p>
		<li><p style="margin-bottom: 0.14in">Schaffer, &quot;Pedigree
		traversal in FASTLINK&quot;</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FASTMAP (1)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Fluorescent
		Allele-calling Software Toolkit: Microsatellite Automation Package</p>
		<li><p style="margin-bottom: 0.14in">version: 1.04b (April, 1997)</p>
		<li><p style="margin-bottom: 0.14in">description: fluorescent
		allele-calling software toolkit: a computer software for fully
		automated microsatellite genotyping.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mark W. Perlin
		(Carnegie Mellon, perlin@cmu.edu), See-Kiong Ng ( skng@cs.cmu.edu),
		Nandita Mukhopadhyay, Lillian Markind Bloch</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cs.cmu.edu/~genome/FAST-MAP.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MacOS. MatLab programming environment is required.</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1996 (Ng,Mukhopadhyay, Rayman,Ghosth,Perlin, Am J
		Hum Genet, suppl 59, A225 (1996); Am Soc Hum Genet annual meeting
		1997 (Perlin, Bloch, Mukhopadhyay, Ng, Am J Hum Genet, suppl 61,
		A290 (1997)).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FASTMAP (2)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis (Royal
		London Hospital, UK) ( dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Curtis, Gurling
		(1993), Human Heredity, 43:173-185.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FASTSLINK (see also SLINK)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: faster SLINK</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a faster version
		of SLINK</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel E. Weeks
		(University of Oxford and University of Pittsburgh)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register/soft_doc.html</p>
		<li><p style="margin-bottom: 0.14in">registration page at:
		http://watson.hgen.pitt.edu/register</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: slink</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1990 (Weeks, Ott, Lathrop, Am J Hum Genet, suppl 47,
		A204 (1990)).</p>
		<li><p style="margin-bottom: 0.14in">online manual pages (same as
		slink)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FBAT (see also PBAT)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: (haplotype) Family
		Based Association Test</p>
		<li><p style="margin-bottom: 0.14in">version: Nov 2003</p>
		<li><p style="margin-bottom: 0.14in">description: Haplotype FBAT is
		a user-friendly, well-documented software that allows the user to
		test for association/linkage between disease phenotypes and
		haplotypes by utilizing family-based controls. The method extends
		the approach for testing described in Rabinowitz and Laird (2000)
		to handle multiple tightly linked markers. It is robust to
		population admixture, yet efficient in the sense that it utilizes
		data from families where phase cannot be completely resolved in all
		individuals by using weights, which are estimated from the sample.
		However, the method remains robust to population stratification and
		population admixture. The method can handle any type of phenotype,
		including multiple phenotypes and missing parents, marker data,
		and/or phase, and provides both bi-allelic and multi-allelic tests.
		PowerFBAT is a tool for power simulation of association analysis
		using FBAT with binary outcomes. XWXW is an extension to the
		Haseman-Elston method for non-parametric linkage test with
		quantitative traits. XDT is a software that performs classical TDT,
		&quot;SDT&quot; and &quot;Rabinowitz TDT&quot; for nuclear families
		(not supported anymore).</p>
		<li><p style="margin-bottom: 0.14in">authors: Xin Xu, Steve
		Horvath, Nan Laird (Harvard Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.harvard.edu/~fbat/fbat.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris, Linux, MS-Windows, Mac(MacOS X/Darwin)</p>
		<li><p style="margin-bottom: 0.14in">executables: fbat, pbat,
		powerfbat, xwxw, xdt</p>
		<li><p style="margin-bottom: 0.14in">reference: Laird NM (2004)
		Family based tests for associating haplotypes with general
		phenotype data: application to asthma genetics. Genetic
		Epidemiology, 26(1):61-69;</p>
		<li><p style="margin-bottom: 0.14in">Laird, Horvath, Xu (2000),
		Implementing a unified approach to family based tests of
		association, Genetic Epi, supp 1, 19:36-42; Rabinowitz, Laird
		(2000), A unified approach to adjusting association tests for
		population admixture with arbitrary pedigree structure and
		arbitrary missing marker information, Human Heredity, 50:211-223.</p>
		<li><p style="margin-bottom: 0.14in">Horvath, Xu, Laird (2001), The
		family based association test method: strategies for studying
		general genotype-phenotype associations, European J Hum Genet.
		9:301-306.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FEST (see R/FEST)</p>
	<li><p style="margin-bottom: 0.14in">FESTA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Fragmented
		Exhaustive Seearch for TAgsnps</p>
		<li><p style="margin-bottom: 0.14in">version: March 2005</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Shyam Gopalakrishnan
		(email: gopalakr@umich.edu), Steve Qin (email: qin@umich.edu)
		(University of Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/qin/FESTA/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FINESSE (see also VITESSE)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: To incorporate
		regressive models into the VITESSE likelihood engine</p>
		<li><p style="margin-bottom: 0.14in">authors: Jeff O'Connell
		(email: joconnel@medicine.umaryland.edu), Florence Demenais (email:
		florence.demenais@evry.inserm.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.molecular-haplotype.org/finesse/finesse_intro.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FINETTI</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: de FINETTI
		generator</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0.5 (2008)</p>
		<li><p style="margin-bottom: 0.14in">description: Tests for
		deviation from Hardy-Weinberg equilibrium and tests for association
		in case controls studies; Plot genotype frequencies graphically
		using a de Finetti diagram.</p>
		<li><p style="margin-bottom: 0.14in">authors: Thomas F Wienker
		(email: wienker@uni-bonn.de) (University of Bonn, Germany), Web
		implementation: Tim M Strom (email: timstrom@gsf.de) (GSF Munich,
		Germany), Henning Henschke (email: hhenschk@imbie.meb.uni-bonn.de).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://ihg.gsf.de/cgi-bin/hw/hwa1.pl (testing part)</p>
		<li><p style="margin-bottom: 0.14in">http://finetti.meb.uni-bonn.de/
		(graphic part, 2008 version)</p>
		<li><p style="margin-bottom: 0.14in">http://ihg.gsf.de/linkage/download/finetti.zip
		(graphic part, 2004 version)</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal
		(web implementation: Perl, C, php)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based,
		MS-DOS, MS-Windows(32), Linux (for stand-alone version)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FIRSTORD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: demonstration of
		method for preliminary ordering of loci based on two-point lod
		scores.</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis (Royal
		London Hospital, UK) ( dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FISHER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: genetic analysis
		of classical biometric traits like blood pressure or height that
		are caused by a combination of polygenic inheritance and complex
		environmental forces.</p>
		<li><p style="margin-bottom: 0.14in">authors: Kenneth Lange
		(IUCLA), Daniel E. Weeks (Pittsburgh), M. Boehnke</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: registration page at:
		http://www.biomath.medsch.ucla.edu/faculty/klange/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lange, Weeks,
		Boehnke, Genet Epid, 5, 471-471 (1988)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">FLOSS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: FLexible Ordered
		SubSet analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 1.4.1</p>
		<li><p style="margin-bottom: 0.14in">description: FLOSS performs
		ordered subset analysis using MERLIN's ouput &quot;.lod&quot; file
		created with the --perFamily option. Ordered subset analysis uses
		covariate information to identify a more homogenous subset of
		families for linkage analysis. The homogeneous subset of families
		does not need to be specified a priori, and the covariates can
		include environmental exposures, quantitative traits, or linkage
		scores at another locus in the genome. The evidence for linkage is
		evaluated with a permutation test.</p>
		<li><p style="margin-bottom: 0.14in">authors: Brian L Browning</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://faculty.washington.edu/browning/floss/floss.htm</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GAIA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genetic Association
		Interaction Analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (2006)</p>
		<li><p style="margin-bottom: 0.14in">description: GAIA is an easy
		to use, web-based application for testing for locus-locus
		interaction using genetic association. It is based upon the
		case-control study design and is designed so that non-specialists
		may routinely apply tests for interaction. GAIA allows simple
		testing of both additive and additive plus dominance interaction
		models and includes permutation testing to appropriately correct
		for multiple testing. The application is useful for both candidate
		gene based studies and genome-wide association studies. For large
		scale studies GAIA includes a screening approach which prioritizes
		loci for further interaction analysis.</p>
		<li><p style="margin-bottom: 0.14in">authors: Stuart Macgregor
		(QIMR, Brisbane, Australia and Cardiff University, UK) (email:
		stuart.macgregor@qimr.edu.au), Imtiaz Khan</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bbu.cf.ac.uk/html/research/biostats.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: perl, R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Macgregor and Khan
		(2006), &quot;GAIA: An easy-to-use web-based application for
		interaction analysis of case-control data&quot;, BMC Medical
		Genetics, 7:34</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: genetic analysis
		package</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: a comprehensive
		package for the management and analysis of pedigree data. it
		offers: 1. Powerful database management tools, specifically
		designed for family data; 2. Automatic pedigree drawing; 3.
		Segregation and linkage analysis, based on traditional maximum
		likelihood methods and newer, more powerful, Monte Carlo methods
		that can model both genetic and environmental factors.</p>
		<li><p style="margin-bottom: 0.14in">authors: &quot;Epicenter
		Software&quot;: Jonathan Buckley (Dept of Preventive Medicine, USC,
		jbuckley@hsc.usc.edu), Jim Gauderman, Duncan Thomas, Cheryl Faucett
		and John Morrison ( jmorr@rcf.usc.edu).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.epicentersoftware.com/products.php</p>
		<li><p style="margin-bottom: 0.14in">source code language: c++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(3.1/95/NT), UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">availability: price single
		user: US$795; LAN version: US$1295. Kindred only: US$495. (academic
		prices are US$595, US$1095, US$395, respectively)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GAS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genetic Analysis
		System</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0 (Dec 1995)</p>
		<li><p style="margin-bottom: 0.14in">description: statistical
		analysis of genetic linkage data, sib-pair analysis, association
		studies</p>
		<li><p style="margin-bottom: 0.14in">authors: Alan Young (
		ayoung@vax.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://users.ox.ac.uk/~ayoung/gas.html .</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		UNIX(SunOS/Solaris/AIX/OSF1/Ultrix/SGI-IRIX),VMS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GAS2</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: statistical
		significance in Genomewide Association Studies in 2-stage</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Evaluating
		Statistical Significance in Two-Stage Genomewide Association
		Studies</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/GAS2/</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lin (2006),
		&quot;Evaluating statistical significance in two-stage genomewide
		association studies&quot;, American Journal of Human Genetics,
		78(3):505-509</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GASP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genometric Analysis
		Simulation Program</p>
		<li><p style="margin-bottom: 0.14in">version: 3.31 (April 1998)</p>
		<li><p style="margin-bottom: 0.14in">description: a software tool
		for testing and investigating methods in statistical genetics by
		generating samples of family data based on user specified models.</p>
		<li><p style="margin-bottom: 0.14in">authors: A.F. Wilson (
		afw@nhgri.nih.gov), J.E.Bailey-Wilson, E.W. Pugh, A.J.M. Sorant
		(NCHGR/NIH)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.nhgri.nih.gov/DIR/IDRB/GASP/</p>
		<li><p style="margin-bottom: 0.14in">ftp://nhgri.nih.gov/pub/outgoing/gasp</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(DEC-UNIX 4.0B/Solaris 2.5/SGI-IRIX 6.2/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1996 (Wilson, Bailey-Wilson, Pugh, Sorant, Am J. Hum
		Genet, suppl, 59, A193 (1996)); documentation draft (in postscript)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GASSOC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genetic ASSOCiation
		analysis software for cases and parent</p>
		<li><p style="margin-bottom: 0.14in">version: 1.06 (Jun 2001)</p>
		<li><p style="margin-bottom: 0.14in">description: Statistical
		methods for disease and genetic marker associations using cases and
		their parents. These methods include an extension of the
		transmission/disequilibrium test (TDT) for multiple marker alleles,
		as well as additional general tests sensitive to associations that
		depend on dominant or recessive genetic mechanisms.</p>
		<li><p style="margin-bottom: 0.14in">authors: D.J. Schaid (
		schaid@mayo.edu), C. M. Rowland (Mayo Clinic/Foundation)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS/Solaris/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1996 (Rowland, Schaid, Am J. Hum Genet, suppl, 59,
		A234 (1996)).</p>
		<li><p style="margin-bottom: 0.14in">Schaid (1996), Genetic
		Epidemiology, 13:423-449.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GATK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genome Analysis
		ToolKit</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The GATK is a
		structured software library that makes writing efficient analysis
		tools using next-generation sequencing data very easy, and second
		it's a suite of tools for working with human medical resequencing
		projects such as 1000 Genomes and The Cancer Genome Atlas. These
		tools include things like a depth of coverage analyzers, a quality
		score recalibrator, a SNP/indel caller and a local realigner.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mark DePristo et al.
		(Broad Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broadinstitute.org/gsa/wiki/index.php/The_Genome_Analysis_Toolkit</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: DePristo, Banks,
		Poplin, Garimella, Maguire, Hartl, Philippakis, del Angel, Rivas,
		Hanna, McKenna, Fennell, Kernytsky, Sivachenko, Cibulskis, Gabriel,
		Altshuler, Daly (2011), &quot;A framework for variation discovery
		and genotyping using next-generation DNA sequencing data&quot;,
		Nature Genetics, 43:491-498.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GC/GCF</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genomic Control</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: GC implements the
		genomic control models. GCF implements the basic Genomic Control
		approach, but adjusts the p-values for uncertainty in the estimated
		effect of substructure. This approach is preferable if a large
		number of tests will be evaluated because it provides a more
		accurrate assessment of the significance level for small p-values.</p>
		<li><p style="margin-bottom: 0.14in">authors: B Devlin, K Roeder</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/genomic_control/genomic_control.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Devlin, Roeder
		(1999), &quot;Genomic control for association studies&quot;,
		Biometrics, 55:997-1004.</p>
		<li><p style="margin-bottom: 0.14in">Bacanu, Devlin, Roeder (2002),
		&quot;Association studies for quantitative traits in structured
		populations&quot;, Genetic Epidemiology, 22:78-93.</p>
		<li><p style="margin-bottom: 0.14in">Devlin, Bacanu, Roeder (2004),
		&quot;Genomic control in the extreme&quot;, Nature Genetics,
		36(11):1129-1130.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GCHAP (see also GENEPI.JAR)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Gene Counting
		method for HAPlotype analysis.</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: This program
		finds maximum likelihood estimates of haplotype frequencies from a
		sample of genotyped individuals. By excluding haplotypes with zero
		MLE at an early stage, this implementation uses many orders of
		magnitude less space and time than naive implementations. A second
		program, ApproxGCHap, is provided to give alternate estimates for
		data sets with large numbers of loci or large amounts of missing
		genotypes.</p>
		<li><p style="margin-bottom: 0.14in">authors: Alun Thomas (email:
		alun@genepi.med.utah.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://episun7.med.utah.edu/~alun/gchap/index.html</p>
		<li><p style="margin-bottom: 0.14in">http://balance.med.utah.edu/wiki/index.php/GCHap</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Thomas (2003),
		&quot;GCHap: fast MLEs for haplotype frequencies by gene counting&quot;,
		Bioinformatics, 19:2002-2003.</p>
		<li><p style="margin-bottom: 0.14in">Thomas (2003), &quot;Accelerated
		gene counting for haplotype frequency estimation&quot;, Annals of
		Human Genetics, 67:608-612.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GDA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genetic Data
		Analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0d16c (Jan 2001)</p>
		<li><p style="margin-bottom: 0.14in">description: designed to
		accompany the second edition of Bruce Weir's book &quot;Genetic
		Data Analysis&quot; (1996. Sinauer Associates)</p>
		<li><p style="margin-bottom: 0.14in">authors: Paul Lewis, Dmitri
		Zaykin</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hydrodictyon.eeb.uconn.edu/people/plewis/software.php</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(3.1/95/NT), MacOS (10.2.8/10.3)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GEMS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genetic
		Epidemiology ModelS</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: a computer
		software for fitting Genetic Epidemiology Models. This version of
		GEMS provides the features for many statistical analyses of family
		data:</p>
		<li><p style="margin-bottom: 0.14in">authors: GE Bonney, V Apprey,
		J Kwagyan (Howard Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://statgen.genomecenter.howard.edu/gems.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(DEC-alpha), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: gems</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1997 (Bonney, Slater, Slifker, Am. J. Hum. Genet.
		suppl 61, A212 (1997)); Bonney, Apprey, Kwagyan (2001), &quot;A
		computer software for fitting Genetic Epidemiology Models&quot;,
		tech report 2001-02.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENABEL (see R/GENABEL)</p>
	<li><p style="margin-bottom: 0.14in">GENECLASS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: v2.0g (Jan 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: GENECLASS2 is a
		program employing multilocus genotypes to select or exclude
		populations as origins of individuals (Assignment and Migrants
		Detection).</p>
		<li><p style="margin-bottom: 0.14in">authors: Jean-Marie Cornuet</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.montpellier.inra.fr/URLB/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Delphi</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windos,
		UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Piry, Alapetite,
		Cornuet, Paetkau, Baudouin, Estoup (2004), &quot;GeneClass2: a
		software for genetic assignment and first-generation migrant
		detection&quot;, Journal of Heredity, 95:536-539.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENECOUNTING</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.2</p>
		<li><p style="margin-bottom: 0.14in">description: gene-counting for
		haplotype analysis with permutation tests for global association
		and specific haplotypes, accounting for missing data</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinghua Zhao (MRC
		Epidemiology Unit at Strangeways Research Lab, UK) (email:
		jinghua.zhao@mrc-epid.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: gc, gcp, gcx,
		pgc.</p>
		<li><p style="margin-bottom: 0.14in">RUNGC, SCANASSOC, COMPGR,
		SCANGROUP, LDPAIRS</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao, Lissarrague,
		Essioux, Sham (2002), &quot;GENECOUNTING: haplotype analysis with
		missing genotypes&quot;, Bioinformatics, 18(12):1694-1695.</p>
		<li><p style="margin-bottom: 0.14in">Zhao (2004), &quot;2LD,
		GENECOUNTING and HAP: Computer programs for linkage disequilibrium
		analysis&quot;, Bioinformatics, 20:1325-1326.</p>
		<li><p style="margin-bottom: 0.14in">Curtis, Knight, Sham (2006),
		&quot;Program report: GENECOUNTING support programsProgram report:
		GENECOUNTING support programs&quot;, Annals of Human Genetics,
		70:277-279.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEFINDER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1.9 (June 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: GeneFinder uses
		GEE method to estimate the location of the susceptibility gene
		based on the allele-sharing of affected sib pairs for multiple
		markers. GENEFINDER can further the analysis of GENEHUNTER by
		providing a more accurate gene location estimation and the
		corresponding confidence interval.</p>
		<li><p style="margin-bottom: 0.14in">authors: Wei-Min Chen (email:
		wechen@jhsph.edu), Kung-Yee Liang (email: kyliang@jhpsh.edu),
		Yen-Feng Chiu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.jhsph.edu/~wmchen/gf.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		MS-DOS, UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables: gf</p>
		<li><p style="margin-bottom: 0.14in">reference: Liang KY, Zeger SL
		(1986), &quot;Longitudinal data analysis using generalized linear
		models&quot;, Biometrika, 73: 13-22.</p>
		<li><p style="margin-bottom: 0.14in">Liang KY, Chiu YF, Beaty TH
		(2001), &quot;A robust identity by descent procedure using affected
		sib pairs: a multipoint approach for complex diseases&quot;, Human
		Heredity, 51:64-78.</p>
		<li><p style="margin-bottom: 0.14in">Liang KY, Chiu YF, Beaty TH,
		Wjst Matthias (2001), &quot;A multipoint analysis using affected
		sib pairs: incorporation of linkage evidence from unlinked
		regions&quot;, Genetic Epidemiology, 21:105-122.</p>
		<li><p style="margin-bottom: 0.14in">Liang KY, Huang CY, Beaty TH
		(2000) A unified sampling approach for multipoint analysis of
		qualitative and quantitative traits in sib pairs. American Journal
		Human Genetics, 66(5):1631-1641</p>
		<li><p style="margin-bottom: 0.14in">Glidden DV, Liang KY, Chiu YF,
		Pulver AE (2003), &quot;Multipoint affected sibpair linkage methods
		for localizing susceptibility genes of complex diseases&quot;,
		Genetic Epidemiology, 24:107-117</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEHUNTER (see also ALLEGRO,
	GENEHUNTER-IMPRINTING, GENEHUNTER-MODSCORE, GENEHUNTER-PLUS,
	GENEHUNTER-TWOLOCUS)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1 release 4 (April
		2003)</p>
		<li><p style="margin-bottom: 0.14in">description: multipoint
		analysis of pedigree data including: non-parametric linkage
		analysis, LOD-score computation, information-content mapping,
		haplotype reconstruction</p>
		<li><p style="margin-bottom: 0.14in">authors: Leonid Kruglyak (
		leonid@genome.wi.mit.edu), Mark Daly (mjdaly@genome.wi.mit.edu),
		Mary Pat Reeve-Daly ( mpreeve@genome.wi.mit.edu), Eric Lander (
		lander@genome.wi.mit.edu) (Whitehead Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broad.mit.edu/ftp/distribution/software/genehunter/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: gh</p>
		<li><p style="margin-bottom: 0.14in">reference:
		Kruglyak,Daly,Reeve-Daly,Lander (1996), &quot;Parametric and
		nonparametric linkage analysis: a unified multipoint approach&quot;,
		American Journal of Human Genetics, 58:1347-1363</p>
		<li><p style="margin-bottom: 0.14in">online documentation (2.0beta)</p>
		<li><p style="margin-bottom: 0.14in">documentation (in PDF)
		(2.0beta).</p>
		<li><p style="margin-bottom: 0.14in">online instruction (1.1, from
		Falling Rain Genomics, Inc.)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEHUNTER-IMPRINTING (see also
	ALLEGRO, GENEHUNTER, GENEHUNTER-MODSCORE, GENEHUNTER-PLUS,
	GENEHUNTER-TWOLOCUS)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: version: 1.3 (May
		2000), 2.1 beta (Oct 2001).</p>
		<li><p style="margin-bottom: 0.14in">description:
		GENEHUNTER-IMPRINTING is a modification of the GENEHUNTER software
		package (version 1.3). It allows to perform parametric (LOD-score)
		analysis of traits caused by imprinted genes - that is, of traits
		showing a parent-of-origin effect.</p>
		<li><p style="margin-bottom: 0.14in">authors: K Strauch
		strauch@med.uni-marburg.de) (Philipps University Marburg, Germany)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.staff.uni-marburg.de/~strauchk/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SUNOS,Solaris,OSF,HPUX,AIX,Ultrix), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: ghi, xghi</p>
		<li><p style="margin-bottom: 0.14in">reference: Strauch, Fimmers,
		Kurz, Deichmann, Wienker, Baur (2000), American Journal of Human
		Genetics, 66(6):1945-1957</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEHUNTER-MODSCORE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0 (October 2006)</p>
		<li><p style="margin-bottom: 0.14in">description:
		GENEHUNTER-MODSCORE is a further extension of
		GENEHUNTER-IMPRINTING. The program is based on the original
		GENEHUNTER version 2.1 release 6. GENEHUNTER-MODSCORE allows for a
		MOD-score analysis, in which parametric LOD scores are maximized
		over the parameters of the trait model, i.e., the penetrances and
		disease allele frequency. As of version 2.0, it is possible to use
		sex-specific recombination frequencies. The genetic positions of
		markers can be automatically read from a publicly available genetic
		map.</p>
		<li><p style="margin-bottom: 0.14in">authors: Konstantin Strauch
		(strauch@med.uni-marburg.de) (Philipps University Marburg, Germany)
		Johannes Dietter (dietter@med.uni-marburg.de) (Philipps University
		Marburg, Germany)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.staff.uni-marburg.de/~strauchk/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SUNOS,Solaris,OSF,HPUX,AIX,Ultrix), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: ghm</p>
		<li><p style="margin-bottom: 0.14in">reference: K Strauch (2003),
		&quot;Parametric linkage analysis with automatic optimization of
		the disease model parameters&quot;. American Journal of Human
		Genetics 73(Suppl1):A2624.</p>
		<li><p style="margin-bottom: 0.14in">Dietter, Mattheisen, Fuerst,
		Rueschendorf, Wienker, Strauch (2007), &quot;Linkage analysis using
		sex-specific recombination fractions with GENEHUNTER-MODSCORE&quot;,
		Bioinformatics, 23(1):64-70.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEHUNTER-PLUS (see also
	ALLEGRO, GENEHUNTER, GENEHUNTER-IMPRINTING, GENEHUNTER-MODSCORE,
	GENEHUNTER-TWOLOCUS)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (Nov 1998)</p>
		<li><p style="margin-bottom: 0.14in">description: GENEHUNTER-PLUS
		is a modification of the GENEHUNTER software package which produces
		output files containing the null and conditional distributions of
		the test statistic (in nullprobs.dat and probs.dat, respectively).
		These files can then be used as input to the ASM program which
		incorporates the allele sharing modeling for lodscores and
		likelihood ratio tests as developed by Kong and Cox (1997).</p>
		<li><p style="margin-bottom: 0.14in">authors: Augustine Kong
		(kong@galton.uchicago.edu), Mike Frigge.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://galton.uchicago.edu/genehunterplus</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: ghp, xghp,asm</p>
		<li><p style="margin-bottom: 0.14in">reference: Kong, Cox (1997),
		&quot;Allele-sharing models: LOD scores and accurate linkage
		tests&quot;, American Journal of Human Genetics, 61(5):1179-1188</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEHUNTER++SAD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: implementation of
		the Sad statistic, more robust to transmission ratio distortion in
		the context of allele sharing</p>
		<li><p style="margin-bottom: 0.14in">authors: Mathieu Lemire
		(email: mlemire@sdf.lonestar.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mlemire.freeshell.org/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lemire, Roslin,
		Laprise, Hudson, Morgan (2004), &quot;Transmission-ratio distortion
		and allele sharing in affected sib pairs: a new linkage statistic
		with reduced bias, with application to chromosome 6q25.3&quot;,
		American Journal of Human Genetics, 75:571-586.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEHUNTER-TWOLOCUS (see also
	ALLEGRO, GENEHUNTER, GENEHUNTER-IMPRINTING, GENEHUNTER-MODSCORE,
	GENEHUNTER-PLUS)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.3 (May 2000),
		optimized and parallelized version (May 2004)</p>
		<li><p style="margin-bottom: 0.14in">description:
		GENEHUNTER-TWOLOCUS is a modification of the GENEHUNTER software
		package (version 1.3). The program performs complete parametric and
		nonparametric multi-marker linkage analysis with two autosomal,
		biallelic disease loci.</p>
		<li><p style="margin-bottom: 0.14in">authors: K Strauch
		strauch@med.uni-marburg.de) (Philipps University Marburg, Germany),
		Johannes Dietter ( dietter@med.uni-marburg.de) (Philipps University
		Marburg, Germany)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.staff.uni-marburg.de/~strauchk/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SUNOS,Solaris,OSF,HPUX,AIX,Ultrix), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: ght</p>
		<li><p style="margin-bottom: 0.14in">reference: Strauch, Fimmers,
		Kurz, Deichmann, Wienker, Baur (2000), &quot;Parametric and
		nonparametric multipoint linkage analysis with imprinting and
		two-locus-trait models: application to mite sensitization&quot;,
		American Journal of Human Genetics, 66(6):1945-1957</p>
		<li><p style="margin-bottom: 0.14in">Dietter, Spiegel, an Mey,
		Pflug, Al-Kateb, Hoffmann, Wienker, Strauch (2004): &quot;Efficient
		two-trait-locus linkage analysis through program optimization and
		parallelization: application to hypercholesterolemia&quot;,
		European Journal of Human Genetics, 12:542-550.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEPI.JAR (see also, JPSGCS)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: September 2006</p>
		<li><p style="margin-bottom: 0.14in">description: a set of Java
		programs for genetic epidemiology analysis</p>
		<li><p style="margin-bottom: 0.14in">authors: Alun Thomas
		(University of Utah)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinformatics.med.utah.edu/~alun/software/index.html</p>
		<li><p style="margin-bottom: 0.14in">http://balance.med.utah.edu/wiki/index.php/JPSGCS</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables: CheckFormat,
		CheckError, DownCodeAlleles, GeneCountAlleles, SelectLoci,
		SelectKindreds, TrimPed, TwoPointLods, MaxTwoPointLods, McLink,
		McLinkLD, HapGraph, GaploFreqs, GCHap, ApproxGCHap, ViewGraph,
		ViewPed, ViewLinkPed,</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEPOOL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.84</p>
		<li><p style="margin-bottom: 0.14in">description: GenePool is a
		software package that provides analysis tools for the detection of
		shifts in relative allele frequency between pooled genomic DNA from
		cases and controls using SNP-based genotyping microarrays. GenePool
		supports genotyping platforms from Affymetrix and Illumina.</p>
		<li><p style="margin-bottom: 0.14in">authors: John Pearson
		(Translational Genomics Research Institute) (email:
		jpearson@tgen.org) David Craig (email: dcraig@tgen.org) Matt
		Huentelman (email: mhuentelman@tgen.org) Waibhav Tembe, Sotiris
		Mitranopolous, Nils Homer</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genepool.tgen.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		C/C++/Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MacOS (should compile on most UNIX systems with gcc)</p>
		<li><p style="margin-bottom: 0.14in">executables: gpcommand,
		gpextract, gpanalyze</p>
		<li><p style="margin-bottom: 0.14in">reference: Pearson,
		Huentelman, Halperin, Tembe, Melquist, Homer, Brun, Szelinger,
		Coon, Zismann, Webster, Beach, Sando, Aasly, Heun, Jessen, Kolsch,
		Tsolaki, Daniilidou, Reiman, Papassotiropoulos, Hutton, Stephan,
		Craig (2007), &quot;Identification of the genetic basis for complex
		disorders by use of pooling-based genomewide
		single-nucleotide-polymorphism association studies&quot;, American
		Journal of Human Genetics, 80:126-139</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEPOP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.4</p>
		<li><p style="margin-bottom: 0.14in">description: a population
		genetics software package</p>
		<li><p style="margin-bottom: 0.14in">authors: Michel Raymond
		(email: raymond@isem.univ-montp2.fr), Francois Rousset ( email:
		rousset@isem.univ-montp2.fr), Eleanor Morgan (email:
		nell@wbiomed.curtin.edu.au) (web version)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wbiomed.curtin.edu.au/genepop/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Raymond, Rousset
		(1995), &quot;GENEPOP (version 1.2): population genetics software
		for exact tests and ecumenicism&quot;, Journal of Heredity,
		86:248-249</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENERECON</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: GeneRecon is a
		software package for linkage disequilibrium mapping using
		coalescent theory. It is based on a Bayesian Markov-chain Monte
		Carlo (MCMC) method for fine-scale linkage-disequilibrium gene
		mapping using high-density marker maps. GeneRecon explicitly models
		the genealogy of a sample of the case chromosomes in the vicinity
		of a disease locus. Given case and control data in the form of
		genotype or haplotype information, it estimates a number of
		parameters, most importantly, the disease position.</p>
		<li><p style="margin-bottom: 0.14in">authors: Thomas Mailund
		(email: mailund@birc.au.dk) (Univ of Oxford, UK and Univ of Aarhus,
		Denmark)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.daimi.au.dk/~mailund/GeneRecon/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		Linux, MacOS, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Mailund, Schierup,
		Pedersen, Madsen, Hein, Schauser (2006), &quot;GeneRecon -- A
		coalescent based tool for fine-scale association mapping&quot;,
		Bioinformatics, 22(18):2317-2318.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENESPRING GT (previously
	VARIA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: GeneSpring GT is
		a desktop analysis workbench for analyzing high-volume,
		high-density genotyping data. The software provides a comprehensive
		set of linkage and association algorithms that allow researchers to
		discover relationships between genotypes and phenotypes.
		Researchers can visually explore fully annotated SNPs and genes at
		varying levels of detail. Designed for biologists and
		statisticians, GeneSpring GT is capable of importing, visualizing,
		and analyzing hundreds of thousands of variation measurements
		simultaneously, for rapid localization of disease or phenotype
		markers.</p>
		<li><p style="margin-bottom: 0.14in">authors: Agilent Technologies
		(previously Silicon Genetics)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.chem.agilent.com/scripts/pds.asp?lpage=34662</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">price: for quote request, go
		to http://www.chem.agilent.com/scripts/cag_requestaquote.asp</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENETIC POWER CALCULATOR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2001-2005</p>
		<li><p style="margin-bottom: 0.14in">description: automated power
		analysis for variance components (VC) quantitative trait locis
		(QTL) linkage and association tests in sibships, and other common
		tests</p>
		<li><p style="margin-bottom: 0.14in">authors: S. Purcell, P Sham</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pngu.mgh.harvard.edu/~purcell/gpc/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sham, Cherny,
		Purcell, Hewitt (2000), &quot;Power of linkage versus association
		analysis of quantitative traits, by use of variance-components
		models, for sibship data&quot;, American Journal of Human Genetics,
		66(5):1616-1630</p>
		<li><p style="margin-bottom: 0.14in">Purcell, Cherny, Sham (2003),
		&quot;Genetic Power Calculator: design of linkage and association
		genetic mapping studies of complex traits&quot;, Bioinformatics,
		19(1):149-150.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENETSIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: GENETSIM provides
		flexible simulations of family data within an easy-to-use,
		high-level programming language. GENETSIM has no limit on pedigree
		sizes or structures, or number of families, or number of marker, or
		number of chromosomes. Genetic transmission is modeled by first
		generating the locations of recombination events, and then
		performing gene dropping according to the given recombination
		pattern. Any pattern of missing data can be specified and
		genotyping errors can be simulated. GENETSIM can simulat multiple
		QTLs with pleiotropic effects, multivariate polygenic background
		and any number of environmental factors, age effects, epistasis and
		variable expression. Users can also select families based on
		ascertainment schemes by repeating simulations.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mike Miller, Na Li
		(Univ of Minnesota)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://taxa.epi.umn.edu/GenetSim/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Octave,
		R (future version)</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Miller, Li (2005),
		&quot;GenetSim: software for simulation of familial data in
		genetics and epidemiology&quot; (abstract), Genetic Epidemiology,
		29:269.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEVAR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: GENe Expression
		VARiation</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Genevar is a
		database and Java tool designed to integrate multiple datasets, and
		provides analysis and visualization of associations between
		sequence variation and gene expression in eQTL studies. Genevar
		allows researchers to investigate eQTL (expression quantitative
		trait loci) associations within a gene locus of interest in real
		time. The database and application can be installed on a standard
		computer in database mode and, in addition, on a server to share
		discoveries among affiliations or the broader community over the
		internet via web services protocols.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/genevar/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yang, Beazley,
		Montgomery, Dimas, Gutierrez-Arcelus, Stranger, Deloukas,
		Dermitzakis (2010), &quot;Genevar: a database and Java application
		for the analysis and visualization of SNP-gene associations in eQTL
		studies&quot;, Bioinformatics, 24(18):2105-2107.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENEWEAVER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (2003)</p>
		<li><p style="margin-bottom: 0.14in">description: software for
		charting family medical/health history</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.geneweaveronline.com/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/2000/NT/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">availability:
		commercialUS$39.95</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENIE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.7.2.1 (2010)</p>
		<li><p style="margin-bottom: 0.14in">description: GENIE (previously
		PEDGENIE and HAPMC) performs tests of association and transmission
		disequilibrium (TDT) between genetic markers and traits in studies
		of arbitrarily-sized families and/or independent individuals using
		Monte Carlo testing. For dichotomous traits, basic genotype-based
		or allele-based Chi-square statistics, OR, and a Chi-square trend
		statistic with user-defined weights, TDT, sib-TDT, combined-TDT are
		included. For quantitative outcomes, a difference in means test,
		ANOVA and QTDT are offered. Flexible haplotype testing and meta
		analysis across multiple centers are available. An automated
		haplotype building module, hapConstructor, is also offered that
		data mines multi-locus data for association signals. The Monte
		Carlo empirical significance assessment accounts for all
		relatedness between individuals for all tests.</p>
		<li><p style="margin-bottom: 0.14in">authors: Nicola Camp, K
		Allen-Brady, Jathine Wong, K Curtin, R Abo</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinformatics.med.utah.edu/Genie/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java 1.6</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables: PedGenie, hapMC,
		hapConstructor</p>
		<li><p style="margin-bottom: 0.14in">reference: Allen-Brady,
		Farnham, Weiler, Camp (2003), &quot;A cautionary note on the
		appropriateness of using a linkage resource for an association
		study&quot;, BMC Genetics, 4(suppl 1):S89.</p>
		<li><p style="margin-bottom: 0.14in">Allen-Brady, Wong, Camp
		(2006), &quot;PedGenie: an analysis approach for genetic
		association testing in extended pedigrees and genealogies of
		arbitrary size&quot;, BMC Bioinformatics, 7:209.</p>
		<li><p style="margin-bottom: 0.14in">Curtin, Wong, Allen-Brady,
		Camp (2007), &quot;PedGenie: meta genetic association testing in
		mixed family and case-control designs&quot;, BMC Bioinformatics,
		8:448.</p>
		<li><p style="margin-bottom: 0.14in">Abo, Knight, Wong, Cox, Camp
		(2008), &quot;hapConstructor: automatic construction and testing of
		haplotypes in a Monte Carlo framework&quot;, Bioinformatics,
		24(18):2105-2107.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENOCHECK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: GenoCheck</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (sept 20, 1995)</p>
		<li><p style="margin-bottom: 0.14in">description: GenoCheck is a
		program designed to identify genotypes that are likely to be
		errors. The statistical method was designed to identify typing
		error rather than pedigree errors, but is general enough to
		pinpoint any unlikely genotype still consistent with Mendelian
		inheritance.</p>
		<li><p style="margin-bottom: 0.14in">authors: Meg Gelder Ehm (Glaxo
		Wellcome Inc. mge37216@glaxowellcome.com), Robert W. Cottingham,
		Marek Kimmel, Alejandro Schaffer, Ramana Idury, Shriram
		Krishnamurthi, Sandeep Gupta, G. Mark Lathrop</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://softlib.rice.edu/geno.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: GenoCheck, toaff</p>
		<li><p style="margin-bottom: 0.14in">reference: Ehm, Cottingham,
		Kimmel (1995), J. Biological Systems, 3(1), 13-25.</p>
		<li><p style="margin-bottom: 0.14in">Ehm, Cottingham, Kimmel
		(1996), &quot;Error detection for genetic data, using likelihood
		methods&quot;, American Journal of Human Genetics, 58(1):225-234</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENOGRAM-MAKER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a graphic way of
		organizing the mass of information gathered during a family
		assessment and finding patterns in the family system</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genogram.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Monica McGoldrick,
		Randy Gerson, , Sylvia Shellenberger (1999), Genograms: Assessment
		and Intervention , 2nd edition (W.W.Norton &amp; Company).</p>
		<li><p style="margin-bottom: 0.14in">price: US$50 (student), US$125
		(standard)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENOME</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: GENOME is a
		program to simulate sequences drawn from a population under the
		Wright-Fisher neutral model. The purpose of this program is to
		simulate sequences on the whole genome scale within practical time.</p>
		<li><p style="margin-bottom: 0.14in">authors: Liming Liang (email:
		lianglim@umich.edu), Sebastian Zollner, Goncalo Abecasis
		(University of Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/liang/genome/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Liang, Zollner,
		Abecasis (2006), &quot;GENOME: a rapid coalescent-based whole
		genome simulator&quot;, ASHG Annual Meeting.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENOMESIMLA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: beta</p>
		<li><p style="margin-bottom: 0.14in">description: genomeSIMLA is a
		forward-time population simulation method that can simulate
		realistic patterns of LD in both family-based and case-control
		datasets.</p>
		<li><p style="margin-bottom: 0.14in">authors: Marylyn Ritchie
		(email: genomesimla@chgr.mc.vanderbilt.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://chgr.mc.vanderbilt.edu/genomeSIMLA/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Dudek, Motsinger,
		Velez, Williams, Ritchie (2006), &quot;Data simulation software for
		whole-genome association and other studies in human genetics&quot;,
		Pacific Symposium on Biocomputing, 499-510.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENOMIZER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: Genomizer is a
		platform independent Java program for the analysis of genome wide
		association experiments.</p>
		<li><p style="margin-bottom: 0.14in">authors: Andre Franke (email:
		a.franke@mucosa.de), Jochen Hampe</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ikmb.uni-kiel.de/genomizer/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Franke, Wollstein,
		Teuber, Wittig, Lu, Hoffmann, Nurnberg, Krawczak, Schreiber, Hampe
		(2006), &quot;GENOMIZER: an integrated analysis system for
		genome-wide association data&quot;, Human Mutation, 27(6):583-588</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENOOM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: GENetic Object
		Oriented Modeling in population genetics</p>
		<li><p style="margin-bottom: 0.14in">version: 3.66 (November 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: GENOOM implements
		a virtual computer world where biological entities are digital
		objects. In this world, each individual of a population is
		represented by a set of chromosomes with genetic elements (markers,
		genes, Quantitative Traits Loci, etc.) and attributes (phenotype,
		age, fertility, living status, exposure, etc.). It comes with a
		built-in model allowing basic genetic studies by simulations,
		performed according to parameter values such a penetrance matrix
		(phenotype probabilities for each genotype), exposure,
		between-relative mating probabilities, and a probability of
		migration in a two-dimensional migration process. The simulated
		populations could be analysed with different packages for
		statistical or genetical analysis (e.g., SAS, S-PLUS, Linkage,
		Mapmaker/sibs, Genehunter). Pedigrees can also be simulated
		according to a pedigree file at the LINKAGE format.</p>
		<li><p style="margin-bottom: 0.14in">authors: Hadi Quesneville,
		Dominique Anxolabehere</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://dynagen.ijm.jussieu.fr/research/topics/modeling-complex-genetic-systems/object-oriented-modeling-of-complex-genetic-systems</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ijm.jussieu.fr/Usr/quesneville/genoom/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Quesneville,
		Anxolabehere (1997), &quot;GENOOM: a simulation package for GENetic
		Object-Oriented Modeling&quot;, Abstract in the European
		Mathematical Genetics Meeting Proceedings, Annals of Human
		Genetics, 61:543.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENOPROOF</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (July 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: download software
		package for the analysis of multiplex PCR kits within the scope of
		paternity testing, kinship cases and population studies. GenoProof&reg;
		offers: (1) probably the most extensive existing population
		database for all supported markers of more than 50 ethnic groups,
		(2) individually configurable quality assurance options, (3)
		complex concept of user rights in order to guarantee data security,
		(4) languages German and English</p>
		<li><p style="margin-bottom: 0.14in">authors: Dr. F. Gotz
		(Qualitype AG, Dresden, Germany) (email: info@genoproof.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genoproof.com or http://qualitype.de/genoproof/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/2000/NT/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GENTOOLS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: software for
		analysis and manipulation of genetic linkage data of genetic
		linkage data, including conversions of pedigree files between
		CRI-MAP and LINKAGE format.</p>
		<li><p style="margin-bottom: 0.14in">authors: Raymond Weaver,
		Cynthia Helms, Santosh Mishra, Helen Donis-Keller (Washington
		University School of Medicine, St. Lious)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genlink.wustl.edu/software/index.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://mapper.wustl.edu/pub/gentools</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: faminfo, genlink,
		jim, pic, pichet, twotable,unmerge, inherit,linkgen,refbld,unmerge2</p>
		<li><p style="margin-bottom: 0.14in">reference: Weaver, Helms,
		Mishra, Donis-Keller (1992), &quot;Software for analysis and
		manipulation of genetic linkage data&quot;, American Journal of
		Human Genetics, 50:1267-1274</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GERMLINE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2.1 (September
		2008)</p>
		<li><p style="margin-bottom: 0.14in">description: GERMLINE is a
		program for discovering long shared segments of Identity by Descent
		(IBD) between pairs of individuals in a large population. It takes
		as input genotype or haplotype marker data for individuals (as well
		as an optional known pedigree) and generates a list of all pairwise
		segmental sharing.</p>
		<li><p style="margin-bottom: 0.14in">authors: Sasha Gusev (email:
		gusev@cs.columbia.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www1.cs.columbia.edu/~gusev/germline/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GEST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Gene-Environment
		interaction and STability</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Yasuo Ukai
		(University of Tokyo, Japan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://lbm.ab.a.u-tokyo.ac.jp/~ukai/gest98.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ukai, Nesumi,
		Yakano (1996), &quot;A package of computer programs for the
		statistical analysis of genotype x environment interaction and
		stability GEST&quot; (in Japanese), Breeding Science, 46:73-79.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GEVALT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: GEnotype
		Visualization and ALgorithmic Tool</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: GEVALT is
		designed to simplify and expedite the process of genotype analysis
		and disease association tests by providing a common interface to
		several common tasks relating to such analyses. It is aimed for
		analysis of unrelated individuals as well as two-generation
		families.</p>
		<li><p style="margin-bottom: 0.14in">authors: Ofir Davidovich, Gad
		Kimmel, Eran Halperin, Oded Apel, Ron Shamir.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://acgt.cs.tau.ac.il/gevalt/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Davidovich, Kimmel,
		Shamir (2007), &quot;GEVALT: An integrated software tool for
		genotype analysis&quot;, BMC Bioinformatics, 8:36.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GGSD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Generic Genetic
		Studies Database</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: web-based,
		relational database driven data management software package for the
		management of large scale genetic studies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Aaron Day-Williams
		(Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://sourceforge.net/projects/ggsd/</p>
		<li><p style="margin-bottom: 0.14in">http://ggsd.sourceforge.net/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java,
		PHP, Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GGT (see also, MAPCOMP)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Graphical
		GenoTyping package</p>
		<li><p style="margin-bottom: 0.14in">version: December 2006</p>
		<li><p style="margin-bottom: 0.14in">description: GGT combines
		genetic map data and locus data to display Graphical Genotypes.
		Data can be viewed by individual or by linkage group, and sorted on
		markers. Statistics are calculated and simultaneous
		filtering/selection on multiple linkage groups is possible. Output
		of hi-res genotype drawings directly to printer or to
		emf/jpeg/bitmap image. A module for constructing GGT-datafiles from
		common mapmaker/joinmap style formatted data is available.</p>
		<li><p style="margin-bottom: 0.14in">authors: Ralph van Berloo
		(Wageningen University, The Netherlands, ralph.vanberloo@wur.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dpw.wau.nl/pv/PUB/ggt/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Delphi
		(Pascal)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables: GGT32.EXE</p>
		<li><p style="margin-bottom: 0.14in">reference: R Van Berloo
		(1999),&quot;GGT: Software for the display of graphical genotypes&quot;,
		Journal of Heredity, 90:328-329. [ full text (PDF) :
		http://www.dpw.wau.nl/pv/papers/reprint/9903.pdf]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GHOST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: GHOST is a
		software package for family-based genomewide association (GWA)
		analysis, with the ability to infer missing genotypes using the
		Elston-Stewart algorithm. When SNPs from an association panel are
		less complete (i.e., having more missing genotypes) than markers
		from a linkage panel, many of the missing genotypes can be
		determined. GHOST can handle large pedigrees -- when pedigrees are
		small, Merlin is also recommended for this analysis.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/chen/ghost/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Burdick, Chen,
		Abecasis, Cheung (2006), &quot;In silico method for inferring
		genotypes in pedigrees&quot;, Nature Genetics, 38:1002-1004.</p>
		<li><p style="margin-bottom: 0.14in">Chen, Abecasis (2007),
		&quot;Family-based association tests for genomewide association
		scans&quot;, American Journal of Human Genetics, 81(5):913-926</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GIST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genotype Ibd
		Sharing Test</p>
		<li><p style="margin-bottom: 0.14in">version: 0.3</p>
		<li><p style="margin-bottom: 0.14in">description: testing whether
		the presence of a marker allale can partially account for the
		evidence of linkage in the region of the marker</p>
		<li><p style="margin-bottom: 0.14in">authors: Chun Li (Vanderbilt
		University). (email: chun.li@vanderbilt.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://phg.mc.vanderbilt.edu/content/gist</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Li, Scott, Boehnke
		(2004), American Journal of Human Genetics, 74(3):418-431</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GLFSINGLE/GLFTRIO/GLFMULTIPLES</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: glfTrio is a
		GLF-based variant caller for next-generation sequencing data. It
		takes one/three/multiple GLF format genotype likelihood files as
		input and generates a VCF-format set of variant calls as output.</p>
		<li><p style="margin-bottom: 0.14in">authors: Abecasis' group (Univ
		Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/GlfSingle</p>
		<li><p style="margin-bottom: 0.14in">http://genome.sph.umich.edu/wiki/GlfTrio</p>
		<li><p style="margin-bottom: 0.14in">http://genome.sph.umich.edu/wiki/GlfMultiples</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GLIDERS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genome-wide LInkage
		DisEquilibrium Repository and Search engine</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HapMap based
		long-range LD search engine</p>
		<li><p style="margin-bottom: 0.14in">authors: Robert Lawrence
		(Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mather.well.ox.ac.uk/GLIDERS/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GLUE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genetic Linkage
		User Environment</p>
		<li><p style="margin-bottom: 0.14in">version: 3.30 (Aug 2002)</p>
		<li><p style="margin-bottom: 0.14in">description: A web interface
		to several commonly used statistical genetics programs, including
		Linkage, Genehunter, Merlin, Unphased, and Transmit. It simplifies
		their use through graphical selection of program options,
		automation of multiple analyses, and viewing of graphical output.
		GLUE is available to HGMP account holders; registration is free to
		all academic users.</p>
		<li><p style="margin-bottom: 0.14in">authors: Frank Dudbridge
		(email: frank.dudbridge@mrc-bsu.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://portal.litbio.org/Registered/Webapp/glue/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: any web
		browser</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GMA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genetic Mixture
		Analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Steven T Kalinowski
		(email: skalinowski@montana.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.montana.edu/kalinowski/GMA/GMA_Home.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GMCHECK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: GMCheck finds the
		posterior probabilities for data errors for genotypes and
		phenotypes in pedigree data</p>
		<li><p style="margin-bottom: 0.14in">authors: Alun Thomas (email:
		alun@genepi.med.utah.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://episun7.med.utah.edu/~alun/gmcheck/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Thomas (2005),
		&quot;GMCheck: Bayesian error checking for pedigree genotypes and
		phenotypes&quot;, Bioinformatics, 21(14):3187-3189.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">G-MENDEL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0 (May 1994)</p>
		<li><p style="margin-bottom: 0.14in">description: a tool for
		construction of genetic linkage maps and analyzing Mendelian
		phenotypes.</p>
		<li><p style="margin-bottom: 0.14in">authors: J.L Holloway, S.J.
		Knapp (Department of Crop and Soil Science, Oregon State
		University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cropandsoil.oregonstate.edu/G-mendel</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GOLD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Graphical Overview
		of Linkage Disequilibrium</p>
		<li><p style="margin-bottom: 0.14in">version: November 2001</p>
		<li><p style="margin-bottom: 0.14in">description: A software
		package that provides a graphical summary of linkage disequilibrium
		in human genetic data. The graphical summary is well suited to the
		analysis of dense genetic maps, where contingency tables are
		cumbersome to interpret. An interface to the Simwalk2 application
		allows for the analysis of family data.</p>
		<li><p style="margin-bottom: 0.14in">authors: G R Abecasis, William
		Cookson (Wellcome Trust Centre for Human Genetics, Oxford, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/GOLD/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Abecasis, Cookson
		(2000), Bioinformatics, 16(2):182-183.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GREGOR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: the program
		allows students and scientists to explore potential manifestations
		of genetic models and stochastic processes. It also provides a
		means to evaluate the effectiveness of statistical procedures such
		as linkage analysis or QTL detection.</p>
		<li><p style="margin-bottom: 0.14in">authors: Nick Tinker (
		tinker@agr.gc.ca), Diane Mather (grain crops breeding and genetic
		research, McGill Univ, mather@agradm.lan.mcgill.ca)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://gnome.agrenv.mcgill.ca/tinker/gregor.htm</p>
		<li><p style="margin-bottom: 0.14in">ftp://gnome.agrenv.mcgill.ca/software/GREGOR/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Tinker and Mather,
		Journal of Heredity, 84, 237 (1993)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GRIDQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.5.1 (August 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: A publicly
		available Web-based application that can perform QTL mapping on a
		variety of population types. GridQTL will extend the functionality
		of QTLExpress by adding new and advanced approaches for modelling
		QTL analysis in simple and complex populations. These new methods
		will be available on a Grid system that will offer flexible
		workflow management, resource allocation, data persistence,
		detached execution of simulations and the scalability required for
		the increase in data volume, data sources and complexity required
		by the new models.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.gridqtl.org.uk/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GRONLOD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: GRONingen
		university LOR score calculation</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Conversion
		programs from LINKAGE files are available. The program uses peeling
		and can employ nested conditioning. There is an automatic peeling
		program that will unravel (multiple) loops. Alleles do not need to
		be recoded, so real allele sizes can be used. Genotype
		probabilities for a chosen person can be calculated for purposes of
		genetic risk calculation. Later versions include one for
		calculations using linked markers and mutations and mosaicism, made
		by Martin van der Meulen. A symbolic versions will generate the
		formula to compute the pedigree likelihood.</p>
		<li><p style="margin-bottom: 0.14in">authors: Gerard te Meerman
		(University of Groningen, The Netherlands)
		(g.j.te.meerman@med.rug.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:</p>
		<li><p style="margin-bottom: 0.14in">source code language: PROLOG</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(3.1/95/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:
		gronrec,gronreca,gronlod,groninbr, gronchec,gronped,gronpedc,
		grontree</p>
		<li><p style="margin-bottom: 0.14in">reference: Meerman, &quot;A
		Logic Programming Approach to Pedigree Analysis&quot;, (Thesis
		Publishers Amsterdam, 1991). Available free of charge from the
		author. Contains documented sources and description.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GRR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Graphical
		Representation of Relationships</p>
		<li><p style="margin-bottom: 0.14in">version: Aug 2003</p>
		<li><p style="margin-bottom: 0.14in">description: a graphical tool
		designed for detection of errors in relationship specification in
		general pedigrees by use of genome scan marker data.</p>
		<li><p style="margin-bottom: 0.14in">authors: G R Abecasis (email:
		goncalo@well.ox.ac.uk), S S Cherny (email:
		cherny@qtl.well.ox.ac.uk), W O C Cookson, and L R Cardon (email:
		lon@well.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/GRR/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Abecasis, Cherny,
		Cookson, and Cardon (2001), &quot;GRR: Graphical representation of
		relationship errors&quot;, Bioinformatics, 17(8):742-743.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GS-EM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: GenoSpectrum
		Expectation-Maximuzation</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.people.fas.harvard.edu/~junliu/genotype/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kang, Qin, Niu, Liu
		(2004), &quot;Incorporating genotyping uncertainty in haplotype
		inference for single-nucleotide polymorphisms&quot;, American
		Journal of Human Genetics, 74(3):495-510</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GSMA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genome Search Meta
		Analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The GSMA is a
		rank-based meta-analysis method for analyzing results from
		genome-wide linkage searches. A software package is now available.
		The gsma software calculates the summed rank for any number of
		studies and bins, then obtains p-values for the Summed Rank and the
		Ordered Rank statistics, by simulation. Weighted and unweighted
		analyses are performed. A test data set is included.</p>
		<li><p style="margin-bottom: 0.14in">authors: Cathryn Lewis (
		email: cathryn.lewis@kcl.ac.uk), Douglas Levinson, Lesley Wise</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mmg.umds.ac.uk/GSMA</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wise et al. (1999),
		Annals of Human Genetics, 63:263-272.</p>
		<li><p style="margin-bottom: 0.14in">Levinson et al., (2003),
		American Journal of Human Genetics, 73(1):17-33</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GTOOL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.4.1 (Feb 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: GTOOL is a
		program for transforming sets of genotype data for use with the
		programs SNPTEST and IMPUTE.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jonathan Marchini</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/~marchini/software/gwas/gtool.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GWAPOWER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Genome-Wide
		Association POWER</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: GWApower is a R
		package for assessing the power of genome-wide association studies
		using commercially available genotyping chips. The package
		encapsulates extensive simulation results generated by our program
		HAPGEN.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/~marchini/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Spencer, Su,
		Donnelly, Marchini (2009), &quot;Designing genome-wide association
		studies: sample size, power, imputation, and the choice of
		genotyping chip&quot;, PLoS Genetics, 5(5):e1000477.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">GWASELECT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: GWASelect
		implements a novel variable selection method for GWAS data and is
		able to handle more than half million SNPs. Extensive simulation
		studies and real data analysis show that this method enjoys high
		power and low false discovery rate compared to existing variable
		selection methods. The variables selected by GWASelect can be
		readily placed into a logistic regression model for disease
		prediction. The current release is designed for binary outcome
		under the additive mode of inheritance.</p>
		<li><p style="margin-bottom: 0.14in">authors: Qianchuan He, Dan-Yu
		Lin (Univ North Carolina)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/GWASelect/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: He, Lin (2010), &quot;A
		variable selection method for genome-wide association studies&quot;,
		Bioinformatics, 27:1-8.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAP (1)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: haplotype
		resolution using imperfect phylogeny</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Eleazar Eskin (
		eeskin@cs.ucsd.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://research.calit2.net/hap/</p>
		<li><p style="margin-bottom: 0.14in">http://www.cs.columbia.edu/compbio/hap/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAP (2)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: HAPlotype analysis
		of polymorphic markers</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: (MRC Epidemiology
		Unit at Strangeways Research Lab, UK) (email:
		jinghua.zhao@mrc-epid.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao (2004), &quot;2LD,
		GENECOUNTING and HAP: Computer programs for linkage disequilibrium
		analysis&quot;, Bioinformatics, 20:1325-1326.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPAR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: HAplotype inference
		by PARsimony</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HAPAR is a
		program to infer haplotype from genotype data. It uses the
		parsimony principle, i.e. try to find the minimum number of
		haplotypes that can reconstruct the input genotypes.</p>
		<li><p style="margin-bottom: 0.14in">authors: Lusheng Wang (City
		University of Hong Kong), Ying Xu (Stanford Univ),</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://theory.stanford.edu/~xuying/hapar/</p>
		<li><p style="margin-bottom: 0.14in">http://www.cs.cityu.edu.hk/~lwang/hapar/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Window,
		UNIX(Solaris/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wang, Xu (2003),
		&quot;Haplotype inference by maximum parsimony&quot;,
		Bioinformatics, 19(14):1773-1780.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPASSOC (see R/HAPASSOC)</p>
	<li><p style="margin-bottom: 0.14in">HAPBLOCK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: a dynamical
		programming algorithm for haplotype block partitioning</p>
		<li><p style="margin-bottom: 0.14in">version: 2002</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Kui Zhang, Fengzhu
		Sun (USC)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-hto.usc.edu/msms/HapBlock/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux, UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kui Zhang, Minghua
		Deng, Ting Chen, Michael S Waterman, Fengzhu Sun (2002),
		Proceedings of the National Academy of Sciences , 99:7335-7339.</p>
		<li><p style="margin-bottom: 0.14in">Zhang, Qin, Chen, Liu,
		Waterman, Sun (2005), &quot;HapBlock: haplotype block partitioning
		and tag SNP selection software using a set of dynamic programming
		algorithms&quot;, Bioinformatics, 21(1):131-134.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPBLOCK (2)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: April 2004</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Adrian Paul Mander
		(Boston College) (email: adrian.mander@mrc-hnr.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://econpapers.repec.org/software/bocbocode/s438902.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Stata</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPGEN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.3.0 (Jan 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: HAPGEN is a
		program thats simulates case control datasets at SNP markers and
		can output data in the FILE FORMAT used by IMPUTE, SNPTEST and
		GTOOL. The approach can handle markers in LD and can simulate
		datasets over large regions such as whole chromosomes. Hapgen
		simulates haplotypes by conditioning on a set of population
		haplotypes and an estimate of the fine-scale recombination rate
		across the region. The disease model is specified through the
		choice of a single SNP as the disease causing variant together with
		the relative risks of the genotypes at the disease SNP. The program
		is designed to work with publically available files that contain
		the haplotypes estimated as part of the HapMap project and the
		estimated fine-scale recombination map derived from that data.
		Hapgen is computationally tractable. On a modern desktop HAPGEN can
		simulate several thousand case and control data on a whole
		chromosome at Hapmap Phase 2 marker density within minutes. This
		program has been used to assess the power of several different
		commercially available genotyping chips, in the design stage of the
		7 genome-wide association studies carried out by the Wellcome Trust
		Case-Control Consortium (WTCCC) and for evaluating the power of
		different methods for detecting association in genome-wide studies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Zhan Su, Jonathan
		Marchini, Peter Donnelly (Univ of Oxford, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/~marchini/software/gwas/hapgen.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPINFERX</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: (Jul 1999)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Andrew Clark (email:
		ac347@cornell.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: (temporary) source
		code , input example file , output example file</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Clark (1990),
		&quot;Inference of haplotype from PCR-amplified samples of diploid
		populations&quot;, Molecular Biology and Evolution, 7:111-122.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLO (1)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Haplotyping with
		computation of conditional probabilities</p>
		<li><p style="margin-bottom: 0.14in">version 1.00 (feb 5, 1996)</p>
		<li><p style="margin-bottom: 0.14in">description: assigns each
		person the haplotype with the highest conditional probability using
		Elston-Stewart algorithm</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel E. Weeks
		(Pittsburgh) and Kenneth Lange (UCLA) (Collaborators: Jeffrey R
		O'Connell and Eric Sobel)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register/soft_doc.html,</p>
		<li><p style="margin-bottom: 0.14in">registration page at:
		http://watson.hgen.pitt.edu/register</p>
		<li><p style="margin-bottom: 0.14in">source code language: Fortran
		77 (ANSI standard)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(SunOS/Solaris/HP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sobel, Lange,
		O'Connell and Weeks, in &quot;Genetic Mapping and DNA Sequencing&quot;
		(IMA Volumes in Mathematics and its Applications) eds. T.P. Speed
		and M.S. Waterman (Springer-Verlag, 1995).</p>
		<li><p style="margin-bottom: 0.14in">Weeks, Sobel, O'Connell, Lange
		(1995), American Journal of Human Genetics, 56(6):1506-1507</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLO (2)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.5 (1998)</p>
		<li><p style="margin-bottom: 0.14in">description: Estimates
		frequencies of multi-site haplotypes using the EM algorithm</p>
		<li><p style="margin-bottom: 0.14in">authors: ME Hawley, KK Kidd</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://krunch.med.yale.edu/haplo/</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: DEC VAX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Hawley, Kidd
		(1995), &quot;HAPLO: A program using the EM algorithm to estimate
		the frequencies of multi-site haplotypes&quot;, Journal of
		Heredity, 86:409-411.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOBLOCK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (April 28, 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: HAPLOBLOCK is a
		software package which provides an integrated approach to haplotype
		block identification, haplotype resolution and linkage
		disequilibrium mapping, suitable for high-density phased or
		unphased SNP data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Gideon Greenspan
		(web/ftp: http://www.gidgreen.com/) Dan Geiger (web/ftp:
		http://www.cs.technion.ac.il/people/dang/)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinfo.cs.technion.ac.il/haploblock/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux, MacOS X</p>
		<li><p style="margin-bottom: 0.14in">executables: haploblock_b,
		haploblock_n</p>
		<li><p style="margin-bottom: 0.14in">reference: Greenspan, Geiger
		(2004), &quot;Model-based inference of haplotype block variation&quot;,
		Journal of Computational Biology, 11(2-3):493-504.</p>
		<li><p style="margin-bottom: 0.14in">Greenspan, Geiger (2004),
		&quot;High density linkage disequilibrium mapping using models of
		haplotype block variation&quot;, Bioinformatics, 20(1):137-144.</p>
		<li><p style="margin-bottom: 0.14in">Instruction Manual:
		http://bioinfo.cs.technion.ac.il/haploblock/HaploBlock_Manual.pdf</p>
		<li><p style="margin-bottom: 0.14in">LD Mapping Supplement
		http://bioinfo.cs.technion.ac.il/haploblock/HaploBlock_Mapping.pdf</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOBLOCKFINDER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.7</p>
		<li><p style="margin-bottom: 0.14in">description: HaploBlockFinder
		is a package for haplotype block identification, visualization and
		htSNP selection. It can also compare the haplotype block structure
		with local LD pattern. The program can be either run as a web
		service, or standalone executables on local machine.</p>
		<li><p style="margin-bottom: 0.14in">authors: Kun Zhang
		(kun.zhang@uth.tmc.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cgi.uc.edu/cgi-bin/kzhang/haploBlockFinder.cgi</p>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: haploBlockFinder,
		hbfWrapper.pl, showLDmatrix.pl, drawBlocks.pl</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOBUILD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The analysis of
		large amounts of SNP data creates difficulties for the analysis of
		haplotypes and their association to traits of interest. Commonly
		fairly simple methods, such as two- or three-SNP sliding windows
		are used to create haplotypes across large regions, but these may
		be of limited value when adjacent SNPs are in strong LD and provide
		redundant information. We have created a novel program,
		&quot;HaploBuild&quot; for constructing and testing haplotypes for
		SNPs in close physical proximity to one another but which are not
		necessarily contiguous. Furthermore, the number of SNPs contained
		in the haplotype is not restricted, thereby permitting the
		evaluation of complex haplotype structures.</p>
		<li><p style="margin-bottom: 0.14in">authors: JM Laramie, JB Wilk,
		AL DeStefano, RH Myers (Boston University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://snp.bumc.bu.edu/modules.php?name=HaploBuild</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Laramie, Wilk,
		DeStefano, Myers (2007), &quot;HaploBuild: an algorithm to
		construct non-contiguous associated haplotypes in family based
		genetic studies&quot;, Bioinformatics, 23(16):2190-2192.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOCLUSTERS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a program
		designed to detect excess haplotypes sharing in datasets consisting
		of case and control haplotypes. Excess haplotype sharing can be
		seen around disease loci in case samples since LD persists longer
		here than in the controls where LD is persisting only according to
		the relatedness of the individuals in the population, i.e. the age
		of the population.</p>
		<li><p style="margin-bottom: 0.14in">authors: Melanie Bahlo (email:
		bahlo@wehi.edu.au)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinf.wehi.edu.au/folders/melanie/haploclusters.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:
		Bahlo,Stankovich,Speed,Rubio,Burfoot,Foote (2006), &quot;Detecting
		genome wide haplotype sharing using SNP or microsatellite haplotype
		data&quot;, Human Genetics, 119(1-2):38-50.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOPAINTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.027b</p>
		<li><p style="margin-bottom: 0.14in">description: A pedigree
		drawing program, suitable in processing haplotype outputs from
		GENEHUNTER, ALLEGRO, MERLIN, and SIMWALK</p>
		<li><p style="margin-bottom: 0.14in">authors: Holger Thiele, Peter
		Nuernberg (Max-Delbruck-Centrum fur Molekulare Medizin,
		Berlin-Buch).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://haplopainter.sourceforge.net/html/ManualIndex.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Thiele, Nurnberg
		(2005), &quot;HaploPainter: a tool for drawing pedigrees with
		complex haplotypes&quot;, Bioinformatics, 21(8):1730-1732.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOPOOL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HaploPool is a
		program for estimating haplotype frequencies either from genotypes
		of individuals or from genotypes of pooled individuals. The
		genotypes must be for a block of bi-allelic SNPs (meaning that the
		SNPs should be in linkage disequilibrium with each other). The
		program assumes that it is given many genotypes of unrelated
		diploid individuals in Hardy-Weinberg equilibrium. If the genotypes
		are from pooled DNA, the program assumes that every pool contains
		the same number of individuals and the individuals were chosen at
		random when placed into the pools. For a reasonable running-time,
		the number of individuals in a pool needs to be between 2 and 4.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://haplopool.icsi.berkeley.edu/haplopool/</p>
		<li><p style="margin-bottom: 0.14in">source code language: gcc/g++,
		perl, MATLAB</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kirkpatrick,
		Santos-Armendariz, Karp, Halperin (2007), &quot;HaploPool:
		improving haplotype frequency estimation through DNA pools and
		phylogenetic modeling&quot;, Bioinformatics, 23(22):3048-3055.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLORE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: HAPLOtype
		REconstruction in pedigrees</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: haplotype
		reconstruction in general pedigree without recombination</p>
		<li><p style="margin-bottom: 0.14in">authors: Hongyu Zhao (Yale,
		email: hongyu.zhao@yale.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinformatics.med.yale.edu/group/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhang, Sun, Zhao
		(2005), &quot;HAPLORE: a program for haplotype reconstruction in
		general pedigrees without recombination&quot;, Bioinformatics,
		21(1):90-103.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOREC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: population-based
		haplotyping</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cs.helsinki.fi/group/genetics/haplotyping.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Eronen, Geerts,
		Toivonen (2004), &quot;A Markov Chain Approach to Reconstruction of
		Long Haplotypes&quot;, Pacific Symposium on Biocomputing,
		9:104-115.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLO.STAT (previously
	HAPLO.SCORE)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2.0 (2005)</p>
		<li><p style="margin-bottom: 0.14in">description: A suite of
		routines for the analysis of indirectly measured haplotypes.</p>
		<li><p style="margin-bottom: 0.14in">authors: Dan Schaid, Jason
		Sinnwell, Charles Rowland, David Tines (Mayo Clinic)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mayoresearch.mayo.edu/mayo/research/biostat/schaid.cfm</p>
		<li><p style="margin-bottom: 0.14in">source code language: S-plus
		6.1.2, R 1.7.1</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: haplo.em,
		haplo.glm, haplo.score</p>
		<li><p style="margin-bottom: 0.14in">reference: Schaid, Rowland,
		Tines, Jacobson, Poland (2002), American Journal of Human Genetics,
		70(2)(2):425-434</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.5 (March 1998)</p>
		<li><p style="margin-bottom: 0.14in">description: A simple program
		for graphical presentation of haplotype block structures, tagSNP
		selection and SNP variation.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://info.med.yale.edu/genetics/kkidd/programs.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: VMS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gu, Pakstis, Kidd
		(2005), &quot;HAPLOT: a graphical comparison of haplotype blocks,
		tagSNP sets and SNP variation for multiple populations&quot;,
		Bioinformatics, 21:3938-3939.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOTTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HAPLOTTER is a
		web application that has been developed to display the results of a
		scan for positive selection in the human genome using the HapMap
		data. It can be used as a resource to examine various population
		genetic measures in a genomic region. Measures that are currently
		displayed include iHS (a statistic developed to detect recent
		positive selection), Fay and Wu's H, Tajima's D and Fst.</p>
		<li><p style="margin-bottom: 0.14in">authors: S. Kudaravalli,
		X.Wen, J.K.Pritchard (email: pritch@uchicago.edu) (Univ of Chicago)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hg-wen.uchicago.edu/selection/haplotter.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Voight,
		Kudaravalli, Wen, Pritchard (2006), &quot;A map of recent positive
		selection in the human genome&quot;, PLoS Biology, 4(3):e72.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOTYPE ESTIMATION</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The haplotype
		estimation service is available in two variants: one for genetic
		data from unrelated probands (e.g., case/control studies) and one
		for core family data</p>
		<li><p style="margin-bottom: 0.14in">authors: K Rohde, R Furst (MDC
		Berlin)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bioinf.mdc-berlin.de/projects/hap/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: K Rohde, R Furst
		(2001), &quot;Haplotyping and estimation of haplotype frequencies
		for closely linked biallelic multilocus genetic phenotypes
		including nuclear family information&quot;, Human Mutation,
		17(4):289-295.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOTYPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: T Niu, Jun Liu
		(Harvard Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.people.fas.harvard.edu/~junliu/Haplo/docMain.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Niu, Qin, Xu, Liu
		(2002), &quot;Bayesian haplotype inference for multiple linked
		single-nucleotide polymorphisms&quot;, American Journal of Human
		Genetics, 70(1):157-169</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOVIEW</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.32 (May 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Haploview is
		designed to simplify and expedite the process of haplotype analysis
		by providing a common interface to several tasks relating to such
		analyses. Haploview currently allows users to examine block
		structures, generate haplotypes in these blocks, run association
		tests, and save the data in a number of formats. All
		functionalities are highly customizable.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jeffrey Barrett
		(Whitehead Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broad.mit.edu/personal/jcbarret/haploview/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Barrett, Fry,
		Maller, Daly (2005), &quot;Haploview: analysis and visualization of
		LD and haplotype maps&quot;, Bioinformatics, 21(2):263-265.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOVISUAL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Esko Ukkonen, Pasi
		Rastas</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cs.helsinki.fi/u/prastas/haplovisual/</p>
		<li><p style="margin-bottom: 0.14in">source code language: java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ukkonen (2002),
		&quot;Finding founder sequences from a set of recombinants&quot;
		Lecture Notes in Computer Science Vol 2452, pp.277-286 (Springer).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPLOWSER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: HAPLotype brOWSER</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://embio.yonsei.ac.kr/haplowser/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kim, Kim, Waterman,
		Park, Li (2009), &quot;HAPLOWSER: a whole-genome haplotype browser
		for personal genome and metagenome&quot;, Bioinformatics,
		25(18):2430-2431.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPMINER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (2006)</p>
		<li><p style="margin-bottom: 0.14in">description: HapMiner is a
		computer program for association mapping based on directly mining
		the haplotypes from case-control data via a density-based
		clustering algorithm. HapMiner can be applied to whole-genome
		screens, as well as candidate-gene studies in small genomic
		regions.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jing Li
		(email:jingli@eecs.case.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://vorlon.case.edu/~jxl175/HapMiner.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Li and Jiang
		(2005), &quot;Haplotype-based linkage disequilibrium mapping via
		direct data mining&quot;, Bioinformatics, 21(24):4384-4393.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPMIX</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (March 2011)</p>
		<li><p style="margin-bottom: 0.14in">description: HAPMIX is an
		application for accurately inferring chromosomal segments of
		distinct continental ancestry in admixed populations, using dense
		genetic data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Alkes Price (Harvard
		Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hsph.harvard.edu/faculty/alkes-price/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Price, Tandon,
		Patterson, Barnes, Rafaels, Ruczinski, Beaty, Mathias, Reich, Myers
		(2009), &quot;Sensitive detection of chromosomal segments of
		distinct ancestry in admixed populations&quot;, PLoS Genetics,
		5(6):e1000519</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPMIXMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HAPMIXMAP is a
		program for modelling extended haplotypes in genetic association
		studies, similar to the FASTPHASE program.</p>
		<li><p style="margin-bottom: 0.14in">authors: Paul McKeigue (email:
		paul.mckeigue@ucd.ie), David O'Donnell (University College Dublin,
		Ireland)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.homepages.ed.ac.uk/pmckeigu/hapmixmap/hapmixmap_manual.html</p>
		<li><p style="margin-bottom: 0.14in">http://sourceforge.net/projects/hapmixmap/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPPY</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: reconstructing
		HAPlotYpes</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a software
		package for Multipoint QTL Mapping in Genetically Heterogeneous
		Animals</p>
		<li><p style="margin-bottom: 0.14in">authors: Richard Mott
		(Wellcome Trust Centre for Human Genetics, UK. rmott@well.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.well.ox.ac.uk/~rmott/happy.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: ANSI C
		(requires NAG C library)</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(IRIX,SunOS), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAP-SAMPLE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HAP-SAMPLE is a
		web application for simulating SNP genotypes for case-control and
		affected-child trio studies by resampling from Phase I/II HapMap
		SNP data. The user provides a list of SNPs to be &quot;genotyped,&quot;
		along with a disease model file that describes causal SNPs and
		their effect sizes. The simulation tool is appropriate for
		candidate regions or whole-genome scans.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hapsample.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wright, Huang,
		Guan, Gamiel, Jeffreis, Barry, Pardo-Manuel, Sullivan, Wilhelmsen,
		Zou (2007), &quot;Simulating association studies: a data-based
		resampling method for candidate regions or whole genome scans&quot;,
		preprint.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPSCOPE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HapScope includes
		a comprehensive analysis pipeline and a sophisticated visualization
		tool for analyzing functionally annotated haplotypes.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinghui Zhang
		(National Cancer Institute, NIH)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://lpg.nci.nih.gov/lpg_small/protocols/HapScope/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhang, Rowe,
		Struewing, Buetow (2002), &quot;HapScope: a software system for
		automated and visual analysis of functionally annotated
		haplotypes&quot;, Nucleic Acid Research, 30:5213-5221.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPSIMU</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HAPSIMU is based
		on real haplotype data from the HapMap ENCODE project. This
		platform can simulate heterogeneous populations with various known
		and controllable structures under the continuous migration model or
		the discrete model. Moreover, both qualitative and quantitative
		traits can be simulated using additive genetic model with various
		genetic parameters designated by users.</p>
		<li><p style="margin-bottom: 0.14in">authors: Feng Zhang, Hong-Wen
		Deng</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://l.web.umkc.edu/liujian/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhang, Liu, Chen,
		Deng (2008), &quot;HAPSIMU: a genetic simulation platform for
		population-based association studies&quot;, BMC Bioinformatics,
		9:331.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HAPSTAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HAPSTAT is a
		user-friendly software interface for the statistical analysis of
		haplotype-disease association. HAPSTAT allows the user to estimate
		or test haplotype effects and haplotype-environment interactions by
		maximizing the (observed-data) likelihood that properly accounts
		for phase uncertainty and study design. The current version
		considers cross-sectional, case-control and cohort studies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Danyu Lin, Donglin
		Zeng (email: linsoft@bios.unc.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/hapstat/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(XP/2000/NT/98)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lin, Zeng, Millikan
		(2005), &quot;Maximum likelihood estimation of haplotype effects
		and haplotype-environment interactions in association studies&quot;,
		Genetic Epidemiology, 29:299-312.</p>
		<li><p style="margin-bottom: 0.14in">Lin, Zeng (2006),
		&quot;Likelihood-based inference on haplotype effects in genetic
		association studies (with discussion)&quot;, Journal of American
		Statistical Association, 101:89-118.</p>
		<li><p style="margin-bottom: 0.14in">Zeng, Lin, Avery, North, Bray
		(2006), &quot;Efficient semiparametric estimation of
		haplotype-disease associations in case-cohort and nested
		case-control studies&quot;, Biostatistics, 7(3):486-502.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HARDY (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: July 1996</p>
		<li><p style="margin-bottom: 0.14in">description: Markov chain
		Monte Carlo program for association in two-dimensional contingency
		tables, and for testing Hardy-Weinberg equilibrium.</p>
		<li><p style="margin-bottom: 0.14in">authors: Sun-Wei Guo (U.
		Minnesota, guo_s@epivax.epi.umn.edu</p>
		<li><p style="margin-bottom: 0.14in">E.A. Thompson (Univ
		Washington)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/Hardy.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(DEC-UNIX/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Guo and Thompson
		(1992), Biometrics, 48, 361-372.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HCLUST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a simple
		clustering method that can be used to rapidly identify a set of tag
		SNP's based upon genotype data</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/hclust/hclust.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HEGESMA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: HEterogeneity and
		GEnome Search Meta Analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: HEGESMA is a
		comprehensive software for performing genome scan meta-analysis, a
		quantitative method to identify genetic regions (bins) with
		consistently increased linkage score across multiple genome scans,
		and for testing the heterogeneity of the results of each bin across
		scans. The program provides as an output the average of ranks and
		three heterogeneity statistics, as well as corresponding
		significance levels.</p>
		<li><p style="margin-bottom: 0.14in">authors: Elias Zintzaras,
		(email: zintza@med.uth.gr) John PA Ioannidis</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://biomath.med.uth.gr</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zintzaras,
		Ioannidis (2005), &quot;HEGESMA: genome search meta-analysis and
		heterogeneity testing&quot;, Bioinformatics, 21(18):3672-3673.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HELIXTREE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: HelixTree Genetic
		Association Software</p>
		<li><p style="margin-bottom: 0.14in">version: 6</p>
		<li><p style="margin-bottom: 0.14in">description: Comprehensive
		toolset for population-based association studies. Perform allele,
		genotype, haplotype, multi-locus, runs of homozygosity, copy number
		variation, and regression-based association analysis on
		case-control, quantitative trait, and categorical type data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Christophe Lambert,
		Dmitri Zaykin, Douglas Hawkins, Stanley Young, Alan Menius, Meg
		Ehm, Mike Mosteller</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.goldenhelix.com/SNP_Variation/HelixTree</p>
		<li><p style="margin-bottom: 0.14in">Download request for free
		trial available at
		http://www.goldenhelix.com/SNP_Variation/svstrial.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux, MacOS X</p>
		<li><p style="margin-bottom: 0.14in">executables: helixtree.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Zaykin, Westfall,
		Young, Karnoub, Wagner, Ehm (2002), &quot;Testing association of
		statistically inferred haplotypes with discrete and continuous
		traits in samples of unrelated individuals&quot;, Human Heredity,
		53:79-91.</p>
		<li><p style="margin-bottom: 0.14in">Recursive Partitioning, A
		Bibliography
		http://www.goldenhelix.com/download/pdfs/RPbibliography.pdf</p>
		<li><p style="margin-bottom: 0.14in">price: Academic pricing
		available. Please contact Golden Helix, Inc. at
		info@goldenhelix.com</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HIT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Haplotype Inference
		Technique</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Pasi Rastas</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cs.helsinki.fi/u/prastas/hit/</p>
		<li><p style="margin-bottom: 0.14in">source code language: java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Rastas, Koivisto,
		Mannila, Ukkonen (2005), &quot;A Hidden Markov Technique for
		Haplotype Reconstruction&quot;, 5th Workshop on Algorithms in
		Bioinformatics - WABI 2005</p>
		<li><p style="margin-bottom: 0.14in">Rastas, Koivisto, Mannila,
		Ukkonen (2008), &quot;Phasing genotypes using a hidden Markov
		model&quot;, in eds. I Mandoiu and A Zelikovsky, Bioinformatics
		Algorithms: Techniques and Applications, pp. 373-391 (Wiley).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HOMOG/HOMOGM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: HOMOGM: Dec 1998</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jurg Ott (Rockefeller
		University)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: homog, homog1a,
		homog1b, homog2, homog3, homog4, homog3r, point4, mtest</p>
		<li><p style="margin-bottom: 0.14in">reference: Ott (1986), Genetic
		Epidemiology, Suppl,1,251-257.</p>
		<li><p style="margin-bottom: 0.14in">online documentation</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HOMOZYGOSITYMAPPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: A web-based
		approach of homozygosity mapping that can handle tens of thousands
		markers. User can upload their own SNP genotype files to our
		database. Intuitive graphic interface is provided to view the
		homozygous stretches, with the ability of zooming into single
		chromosomes or user-defined chromosome regions. The underlying
		genotypes in all samples are displayed. The software is also
		integrated with our candidate gene search engine, GeneDistiller, so
		that users can interactively determine the most promising gene.</p>
		<li><p style="margin-bottom: 0.14in">authors: Dominik Seelow,
		Markus Schuelke (NeuroCure Clinical Research, Charite -
		Universitatsmedizin Berlin)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.homozygositymapper.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Seelow, Schuelke,
		Hildebrandt, N&uuml;rnberg (2009), &quot;HomozygosityMapper--an
		interactive approach to homozygosity mapping&quot;, Nucleic Acid
		Research, 37(web server issue):W593-599.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HOTSPOTTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2.1</p>
		<li><p style="margin-bottom: 0.14in">description: Hotspotter
		implements the PAC (Products of Approximate Conditional) model for
		estimating recombination rate</p>
		<li><p style="margin-bottom: 0.14in">authors: Na Li</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.umn.edu/~nali/SoftwareListing.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: rholike, hotspot,
		fullopt</p>
		<li><p style="margin-bottom: 0.14in">reference: Li, Stephens
		(2003), Genetics, 165(4):2213-2233.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HPLUS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.5</p>
		<li><p style="margin-bottom: 0.14in">description: HPlus is an
		analysis tool for performing haplotype estimation on genetic
		markers such as SNPs and microsatellites. It is able to handle
		datasets that include case-control status as well as covariates and
		marker location variables (such as gene name, chromosome location,
		etc).</p>
		<li><p style="margin-bottom: 0.14in">authors: Sue Li, RJ Laws,
		LuePing Zhao (Fred Hutchinson Cancer Research Center)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cougar.fhcrc.org/hplus/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao, Li, Khalid
		(2003), &quot;A method for assessing disease associations with SNP
		haplotypes and environmental variables in case-control&quot;,
		American Journal of Human Genetics, 72(5):1231-1250</p>
		<li><p style="margin-bottom: 0.14in">Li, Khalid, Carlson, Zhao
		(2003), &quot;Estimating haplotype frequencies and standard errors
		for multiple single nucleotide polymorphisms&quot;, Biostatistics,
		4:513-522.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HS-TDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Haplotype Sharing
		Transmission Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: testing
		association using tightly linked markers in nuclear pedigrees</p>
		<li><p style="margin-bottom: 0.14in">authors: Shuanglin Zhang
		(email: shuzhang@mtu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.math.mtu.edu/~shuzhang/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhang, Sha, Chen,
		Dong, Jiang (2003), &quot;Transmission/disequilibrium test based on
		haplotype sharing for tightly linked markers&quot;, American
		Journal of Human Genetics, 73(3):566-579</p>
		<li><p style="margin-bottom: 0.14in">Zhang, Sha, Chen, Dong, Jiang
		(2004), &quot;Impact of genotyping error on type I error rate of
		the haplotype-sharing transmission /disequilibrium test (HS-TDT):
		reply to Knapp and Becker&quot; (letter), American Journal of Human
		Genetics, 74(3):591-593</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HTR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Haplotype Trend
		Regression</p>
		<li><p style="margin-bottom: 0.14in">version: Aug 2002</p>
		<li><p style="margin-bottom: 0.14in">description: Haplotype
		association mapping using unrelated individuals; &quot;fixed&quot;
		and &quot;sliding&quot; window analysis; overall tests and tests
		for individual haplotype effects</p>
		<li><p style="margin-bottom: 0.14in">authors: Dmitri Zaykin</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://statgen.ncsu.edu/zaykin/htr.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://statgen.ncsu.edu/pub/zaykin/htr/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++,
		Bash shell</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Window,
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables: emgi.exe (fixed
		number of markers), bash emgi.sh (sliding window)</p>
		<li><p style="margin-bottom: 0.14in">reference: Zaykin DV, Westfall
		PH, Young SS, Karnoub MC, Wagner MJ, Ehm MG (2002), &quot;Testing
		association of statistically inferred haplotypes with discrete and
		continuous traits in samples of unrelated individuals&quot;, Human
		Heredity, 53:79-91.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HTSNPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Haplotype Tagging
		SNPer</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (Dec 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: htSNPer is a Java
		program for characterizing the haplotype block structure and
		selecting haplotype tagging SNP</p>
		<li><p style="margin-bottom: 0.14in">authors: Keyue Ding, Jing
		Zhang (email: zhjing98@mails.tsinghua.edu.cn), Kaixin Zhou, Yan
		Shen, Xuegong Zhang (Tsinghua University, China)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.chgb.org.cn/htSNPer/htSNPer.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++, JVM</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HWMET</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Bayesian
		estimation of the population inbreeding coefficient f</p>
		<li><p style="margin-bottom: 0.14in">authors: KL Ayres (email:
		k.l.ayres@reading.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.reading.ac.uk/Statistics/genetics/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">HWESTRATA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Calculates an
		exact stratified test for HWE for diallelic markers, such as single
		nucleotide polymorphisms (SNPs), and an exact test for homogeneity
		of Hardy Weinberg disequilbrium. In addition, exact tests for HWE
		are calculated for each stratum.</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel Schaid (Mayo
		Clinic) (email: schaid@mayo.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), MS-Windows(XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">IBDREG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.1.0 (December 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: A package in
		S-PLUS and R to test genetic linkage with covariates by regression
		methods with response IBD sharing for relative pairs. Account for
		correlations of IBD statistics and covariates for relative pairs
		within the same pedigree.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jason Sinnwell
		(email: sinnwell@mayo.edu), Daniel Schaid (Mayo Clinic)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R/S-PLUS</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">IGG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Integrate Genotypes
		for genome-wide Genetic studies</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0</p>
		<li><p style="margin-bottom: 0.14in">description: IGG is an
		open-source Java package with graphic interface to efficiently and
		consistently integrate genotypes across high throughput genotyping
		platforms (e.g., Affymetrix and Illumina), the HapMap genotype
		repository (http://www.hapmap.org/), and even genotypes from the
		collaborators' projects. It is equipped with a series of functions
		to control qualities of genotype integration and to flexibly export
		genotypes for genetic studies as well.</p>
		<li><p style="margin-bottom: 0.14in">authors: Miao-Xin Li (email:
		limx54@163.com)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinfo.hku.hk:13080/iggweb/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Li, Jiang, Ho,
		Song, Sham (2007), &quot;IGG: A tool to integrate GeneChips for
		genetic studies&quot;, Bioinformatics, 23(22):3105-3107.</p>
		<li><p style="margin-bottom: 0.14in">Li, Jiang, Kao, Sham, Song
		(2009), &quot;IGG3: a tool to rapidly integrate large genotype
		datasets for whole-genome imputation and individual-level
		meta-analysis&quot;, Bioinformatics, 25(11):1449-1450.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ILLUMINUS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Illuminus is a
		fast and accurate algorithm for assigning single nucleotide
		polymorphism (SNP) genotypes to microarray data from the Illumina
		BeadArray technology.</p>
		<li><p style="margin-bottom: 0.14in">authors: Yik Teo et al.
		(Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/illuminus/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Teo, Inouye, Small,
		Gwilliam, Deloukas, Kwiatkowski, Clark (2007), &quot;A genotype
		calling algorithm for the Illumina BeadArray platform&quot;,
		Bioinformatics, 23(20):2741-2746.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">INTERSNP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: genome-wide
		association and INTERaction analysis of SNP</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.10 (Jan 2012)</p>
		<li><p style="margin-bottom: 0.14in">description: INTERSNP is a
		software for genome-wide interaction analysis (GWIA) of
		case-control SNP data and quantitative traits. SNPs are selected
		for joint analysis using a priori information. Sources of
		information to define meaningful strategies can be statistical
		evidence (single marker association at a moderate level, computed
		from the own data) and genetic/biologic relevance (genomic
		location, function class or pathway information). Supports all
		PLINK file formats and permutation-based testing. Includes pathway
		association analysis, case-only-analysis, haplotype analysis and
		more.</p>
		<li><p style="margin-bottom: 0.14in">authors: Christine Herold, Tim
		Becker, Dmitriy Drichel, Andr&eacute; Lacour, Tatsiana
		Vaitsiakhovich</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://intersnp.meb.uni-bonn.de/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C/C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Herold, Steffens,
		Brockschmidt, Baur, Becker (2009), &quot;INTERSNP: genome-wide
		interaction analysis guided by priori information&quot;,
		Bioinformatics, 25(24):3275-3281</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ILR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Imprinting
		Likelihood Ratio</p>
		<li><p style="margin-bottom: 0.14in">version: 2.2 (November 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: calculation of
		the restricted likelihood-ratio affected sib-pair test for linkage
		allowing for imprinting</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Knapp (Univ
		Bonn) and Konstantin Strauch (Univ Marburg)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uni-bonn.de/~umt70e/soft.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: SAS
		macro</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Knapp, Strauch
		(2004), &quot;Affected-sib-pair test for linkage based on
		constraints for identical-by-descent distributions corresponding to
		disease models with imprinting&quot;, Genetic Epidemiology,
		26:273-285 (Erratum: 28:288).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">IMPUTE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2 (2009)</p>
		<li><p style="margin-bottom: 0.14in">description: IMPUTE is a
		program for estimating (&quot;imputing&quot;) unobserved genotypes
		in SNP association studies. The program is designed to work
		seamlessly with the output of the genotype calling program CHIAMO
		and the population genetic simulator HAPGEN, and it produces output
		that can be analyzed using the program SNPTEST.</p>
		<li><p style="margin-bottom: 0.14in">authors: Bryan Howie (email:
		bhowie@uchicago.edu), Jonathan Marchini (email:
		marchini@stats.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		https://mathgen.stats.ox.ac.uk/impute/impute.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Marchini, Howie,
		Myers, McVean, Donnelly (2007), &quot;A new multipoint method for
		genome-wide association studies via imputation of genotypes&quot;,
		Nature Genetics, 39:906-913.</p>
		<li><p style="margin-bottom: 0.14in">Howie, Donnelly, Marchini
		(2009), &quot;A flexible and accurate genotype imputation method
		for the next generation of genome-wide association studies&quot;,
		PLoS Genetics, 5(6):e1000529</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">INSEGT (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: INference of
		SEGregation Types</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (September 1999)</p>
		<li><p style="margin-bottom: 0.14in">description: the program
		constructs feasible haplotype configurations and the corresponding
		segregation types on pedigrees. the haplotype configuration
		minimizes recombinations on the pedigree.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jochen Kumm (email:
		jochen@u.washington.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++ (the
		standard template library, STL, is used)</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">INTEGRAYEDMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: IntegratedMap is
		a web application and database schema for storing and interactively
		displaying genetic map data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Hongyu Yang, Hongyu
		Wang, Alan Gingle (Center for Applied Genetic Technologies,
		University of Georgia)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://cggc.agtec.uga.edu/IntegratedMap/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(NT/2000/XP profession/), require either Oracle RDBMS or MySQL.</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yang, Wang, Gingle
		(2005), &quot;IntegratedMap: a web interface for integrating
		genetic map data&quot;, Bioinformatics, 21(9):2126-2127.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">INTERSNP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.4 (March 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: INTERSNP is a
		software for genome-wide interaction analysis (GWIA) of
		case-control SNP data. SNPs are selected for joint analysis using a
		priori information. Sources of information to define meaningful
		strategies can be statistical evidence (single marker association
		at a moderate level, computed from the own data) and
		genetic/biologic relevance (genomic location, function class or
		pathway information).</p>
		<li><p style="margin-bottom: 0.14in">authors: Christine Herold, Tim
		Becker</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://intersnp.meb.uni-bonn.de/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C/C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Herold, Steffens,
		Brockschmidt, Baur, Becker (2009), &quot;INTERSNP: genome-wide
		interaction analysis guided by a priori information&quot;,
		Bioinformatics, 25(24):3275-3281</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">JENTI</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: An efficient tool
		for mining complex inbred genealogies that identify clusters of
		individuals sharing the same expected amount of relatedness is
		described. Additionally it allows for the reconstruction of
		sub-pedigrees suitable for genetic mapping in a systematic way.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: http://www.jenti.org</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Falchi, Fuchsberger
		(2008), &quot;Jenti: an efficient tool for mining complex inbred
		genealogies&quot;, Bioinformatics, 24(5):724-726.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">JLIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: a Java based
		LINkage disequilibrium plotter</p>
		<li><p style="margin-bottom: 0.14in">version: 1.5.2 (March 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: JLIN (Java
		LINkage disequilibrium plotter) is designed for customisable,
		intuitive visualisation of LD analysis across all common computing
		platforms. Customisation allows the researcher to choose particular
		visualisation, statistical measures and measurement ranges. JLIN
		also allows the researcher to export images of the LD visualisation
		in several common document formats. As there appears to be no
		single best measure of LD under all possible circumstances, JLIN
		allows the researcher to visually compare and contrast the results
		of a range of statistical measures on the input data set(s). These
		measures include the commonly used D' and R2 statistics and
		empirical p-values. New additions include calculation of HWE, a
		completely revamped interface, and a numer of minor bug fixes. We
		have added a display measure to show marker distances visually,
		embedded fonts to improve image clarity and additional LD measures
		including d,OR,Pexcess and Q.</p>
		<li><p style="margin-bottom: 0.14in">authors: Kim W Carter (Univ of
		Western Australian), Pamela A. McCaskie, Lyle J Palmer</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genepi.org.au/jlin</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Any with
		Java 1.5 support</p>
		<li><p style="margin-bottom: 0.14in">executables: Java-based
		installer available from website</p>
		<li><p style="margin-bottom: 0.14in">reference: Carter, McCaskie,
		Palmer (2006), &quot;JLIN: A java based linkage disequilibrium
		plotter&quot;, BMC Bioinformatics, 7:60.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">JOINMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Software for the
		calculation of genetic linkage maps</p>
		<li><p style="margin-bottom: 0.14in">version: 4 (October 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Construction of
		genetic linkage maps for several types of mapping populations: BC1,
		F2, RILs, (doubled) haploids, outbreeders full-sib family. Can
		combine ('join') data derived from several sources into an
		integrated map. Further: linkage group determination, automatic
		phase determination for outbreeders full-sib family, several
		diagnostics, and map charts. Everything available in an intuitive
		MS-Windows user interface.</p>
		<li><p style="margin-bottom: 0.14in">authors: Johan W Van Ooijen,
		Roeland E Voorrips (Plant Research International, Biometris,
		Wageningen, Netherlands) (email: mapping@biometris.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.joinmap.nl/</p>
		<li><p style="margin-bottom: 0.14in">source code language: ANSI C,
		Delphi</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/ME/NT4.0/2000)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: JW Van Ooijen, RE
		Voorrips (2001), &quot;JoinMap (tm) 3.0: Software for the
		calculation of genetic linkage maps&quot;, Plant Research
		International, Wageningen, Netherlands; Stam (1993), The Plant
		Journal, 3, 739-744.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">JPSGCS (see also GENEPI.JAR)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Java Programs for
		Statistical Genetics and Computational Statistics</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: These programs
		have been developed since 2002 to address problems in statistical
		genetics. However, they include several programs and packages that
		may be more generally useful, for instance, programs to draw and
		manipulate graphs, simulation programs, and programs to estimate
		graphical models.</p>
		<li><p style="margin-bottom: 0.14in">authors: Alun Thomas</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://balance.med.utah.edu/wiki/index.php/JPSGCS</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MacOS, MS-Windows,</p>
		<li><p style="margin-bottom: 0.14in">executables: ApproxGCHap,
		CheckErrors, CheckFormat, CheckParameters, CheckPedigree,
		CheckTriplets, Complete, DownCodeAlleles, FitGM, FitGMLD,
		FitIntervalLD, GCHap, GeneCountAlleles, GeneDrop, GeneDrops,
		GetPolymorphisms, HGS, HapMapToLinkage, HetCutOff, HetLods,
		Heterozygosities, HomozygousSGS, LinkageToFastPhase,
		LinkageToPhase, LocusLogLikelihoods, MakePed, MakeProbands,
		MaxTwoPointLods, McCheck, McKinship, McLinkage, McMultiPoints,
		McTLods, ObligateErrors, SGS, SelectKindreds, SelectLoci,
		SelectPedigrees, SimHGS, SimHomozygousSGS, SimSGS, Triangulate,
		TrimPed, TwoPointLods, ViewDAG, ViewGraph, ViewLinkPed, ViewPed,
		ZeroLoopPed, cMorgansToTheta</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">J/QTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: J/QTL is a Java
		GUI for the popular QTL data analysis software R/QTL. It provides a
		flexible and powerful working environment for users to perform a
		variety of tasks.</p>
		<li><p style="margin-bottom: 0.14in">authors: Hao Wu and Lei Wu
		(Jackson Lab) (email: jqtl@jax.com)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.jax.org/staff/churchill/labsite/software/Jqtl/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">KIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.05 (Dec 1995)</p>
		<li><p style="margin-bottom: 0.14in">description: calculate kinship
		coefficient or coefficient of coancestry (the probability that
		alleles at a given locus are identical by descent)</p>
		<li><p style="margin-bottom: 0.14in">authors: Nick A. Tinker (
		tinker@em.agr.ca), Diane Mather (grain crops breeding and genetic
		research, McGill Univ, mather@agradm.lan.mcgill.ca)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://gnome.agrenv.mcgill.ca/tinker/kin.htm</p>
		<li><p style="margin-bottom: 0.14in">ftp://gnome.agrenv.mcgill.ca/software/KIN/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Tinker and Mather,
		Journal of Heredity, 84, 238 (1993)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">KING</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Kinship-based
		INference for Gwas</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.0 (2010)</p>
		<li><p style="margin-bottom: 0.14in">description: KING is a toolset
		that makes use of high-throughput SNP data typically seen in a
		genome-wide association study (GWAS) for applications such as
		family relationship inference and population structure
		identification</p>
		<li><p style="margin-bottom: 0.14in">authors: Wei-Min Chen
		(University of Virginia)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://people.virginia.edu/~wc9c/KING/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux
		(64bit), MS-Windows, MS-DOS, MacOS, Ubuntu (32bit)</p>
		<li><p style="margin-bottom: 0.14in">executables: king</p>
		<li><p style="margin-bottom: 0.14in">reference: Manichaikul,
		Mychaleckyj, Rich, Daly, Sale, Chen (2010), &quot;Robust
		relationship inference in genome wide association studies&quot;,
		Bioinformatics. 2010 Nov 15;26(22):2867-73</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LAMARC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Likelihood Analysis
		with Metropolis Algorithm using Random Coalescence</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0.2 (August 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: The LAMARC
		program estimates effective population sizes, exponential
		population growth rates, and past migration rates between two or n
		populations, and simultaneously estimates the per-nucleotide
		recombination rate. Currently Lamarc can use DNA or RNA sequence
		data, SNP data, and microsatellite data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mary Kuhner, Jon
		Yamato, Peter Beerli (email: lamarc@gs.washington.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://evolution.genetics.washington.edu/lamarc/lamarc_prog.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables: lamarc, migrate,
		coalesce, fluctuate, recombine</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LAMBDAA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: AP Morris (email:
		amorris@well.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.reading.ac.uk/Statistics/genetics/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Morris, Whittaker,
		Curnow (1997), &quot;A likelihood ratio test for detecting patterns
		of disease-marker association&quot;, Annals of Human Genetics,
		61:335-350.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LAMP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Linkage and
		Association Modeling in Pedigrees</p>
		<li><p style="margin-bottom: 0.14in">version: 0.0.9</p>
		<li><p style="margin-bottom: 0.14in">description: LAMP carries out
		parametric association analysis in samples of unrelated
		individuals, trios or large families. In addition to a
		test-statistic, a genetic model is estimated for each SNP. The
		approach is based on retrospective likelihood and can be used to
		analyze selected samples.</p>
		<li><p style="margin-bottom: 0.14in">authors: Goncalo Abecasis
		(email: goncalo@umich.edu) and Mingyao Li</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://www.sph.umich.edu/csg/abecasis/LAMP/download/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		LINUX, MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables: lamp</p>
		<li><p style="margin-bottom: 0.14in">reference: Li, Boehnke,
		Abecasis (2006), &quot;Efficient study designs for test of genetic
		association using sibship data and unrelated cases and controls&quot;,
		American Journal of Human Genetics, 78(5):778-792</p>
		<li><p style="margin-bottom: 0.14in">Li, Boehnke, Abecasis (2005),
		&quot;Joint Modeling of Linkage and Association: Identifying SNPs
		Responsible for a Linkage Signal&quot;, American Journal of Human
		Genetics, 76(6):934-949</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LAPSTRUCT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: LAPlacian
		eigenfunctions learn population STRUCTure</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: LAPSTRUCT is a
		free program to describe population structure using biomarker data
		( typically SNPs, CNVs etc.) available in a population sample. The
		main features different from PCA are: (1)geometrically motivated
		and graphic model based; (2)robustness of outliers.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jun Zhang, P Niyogi,
		MS McPeek</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://galton.uchicago.edu/~junzhang/LAPSTRUCT.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhang, Niyogi,
		McPeek (2009), &quot;Laplacian eigenfunctions learn population
		structure&quot;, PLoS ONE, 4(12):e7928.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LCP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Linkage Control
		Program</p>
		<li><p style="margin-bottom: 0.14in">alias: LINKAGE</p>
		<li><p style="margin-bottom: 0.14in">version: This is a 16 bit
		program and will not run on 64 bit PCs, see
		<a href="http://www.jurgott.org/linkage/LinkagePC.html">http://www.jurgott.org/linkage/LinkagePC.html</a></p>
		<li><p style="margin-bottom: 0.14in">description: part of the
		LINKAGE auxiliary programs</p>
		<li><p style="margin-bottom: 0.14in">authors: Peter Cartwright
		(Univ Utah, now Cimarron Software <a href="mailto:pc@cimsoft.com">pc@cimsoft.com</a>)</p>
	</ul>
	<li><p align="left" style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 11pt">web:
	<font color="#0000ff"><span lang="en-US"><u><font color="#0000ff">http://lab.rockefeller.edu/ott/programs</font></u></span></font>
	 or
	<font color="#0000ff"><font face="Times New Roman, serif"><u>https://www.dropbox.com/sh/ulpuondvofkrvx1/AAAkIHrAqyQpHJlZaqtmTRb4a/Linkage%20Analysis</u></font></font>
	(please contact Alberto Goldoni, email:
	<font color="#0000ff"><font face="Times New Roman, serif"><u>alberto.goldoni1975@gmail.com</u></font></font>
	if you have any questions on this version)</font></font></p>
	<ul>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS, OS2</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Linkage
		Disequilibrium Analyzer</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: LDA is a Java
		program for analyzing the pairwise linkage disequilibrium.</p>
		<li><p style="margin-bottom: 0.14in">authors: Keyue Ding (email:
		keyued@chgb.org.cn), Kaixin Zhou, Fuchu He, Yan Shen</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.chgb.org.cn/lda/lda.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ding, Zhou, He,
		Shen (2003), &quot;LDA - A java-based linkage disequilibrium
		analyzer&quot;, Bioinformatics, 19:2147-2148.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDB/LDB+</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Location DataBase</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: integrate genetic
		linkage map and physical map</p>
		<li><p style="margin-bottom: 0.14in">authors: A Collins (
		arc@southampton.ac.uk), NE Morton, S. Lawrence, D. Shields,
		Jonathan Teague, Paul Linehan, Pietro Lio (University of
		Southampton, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ldb;</p>
		<li><p style="margin-bottom: 0.14in">http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/ldb+</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Morton (1991),
		Annals of Human Genetics, 55:235-241.</p>
		<li><p style="margin-bottom: 0.14in">Morton, Collins, Lawrence,
		Shield (1992), Annals of Human Genetics, 56:223-232.</p>
		<li><p style="margin-bottom: 0.14in">online manual and user guide</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDGROUP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Nov 2007</p>
		<li><p style="margin-bottom: 0.14in">description: inkage
		disequilibrium grouping of single nucleotide polymorphisms (SNPs)
		reflecting haplotype phylogeny for efficient selection of tag SNPs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Fumihiko Takeuchi
		(email: fumihiko@takeuchi.name)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.fumihiko.takeuchi.name/publications.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Takeuchi, Yanai,
		Morii, Ishinaga, Taniguchi-Yanai, Nagano, Kato (2005), &quot;Linkage
		disequilibrium grouping of single nucleotide polymorphisms (SNPs)
		reflecting haplotype phylogeny for efficient selection of tag
		SNPs&quot;, Genetics, 170:291-304.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDHAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: a package for the
		analysis of recombination rates from population genetic data</p>
		<li><p style="margin-bottom: 0.14in">authors: Gil McVean (email:
		mcvean@stats.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/~mcvean/LDhat/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: McVean, Myers,
		Hunt, DeLoukas, Bentley, Donnelly (2004), &quot;The fine-scale
		structure of recombination rate variation in the human genome&quot;,
		Science, 304:581-584.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDHEATMAP (see R/LDHEATMAP)</p>
	<li><p style="margin-bottom: 0.14in">LDMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (November 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: LDMAP is a
		program for constructing linkage disequilibrium (LD) maps.</p>
		<li><p style="margin-bottom: 0.14in">authors: Andy Collins (email:
		a.r.collins@soton.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/LDMAP</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: online manual:
		http://cedar.genetics.soton.ac.uk/public_html/helpld.html</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDMET</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: KL Ayres (email:
		k.l.ayres@reading.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.reading.ac.uk/Statistics/genetics/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ayres, Balding
		(2001), &quot;Measuring gametic disequilibrium from multi-locus
		data&quot;, Genetics, 157:413-423.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDSELECT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		analyzes patterns of linkage disequilibrium (LD) between
		polymorphic sites in a locus, and bins the SNPs on the basis of a
		threshold level of LD as measured by r2.</p>
		<li><p style="margin-bottom: 0.14in">authors: Deborah A. Nickerson,
		Mark Rieder, Chris Carlson, Qian Yi (Univ of Washington)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://droog.gs.washington.edu/ldSelect.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Carlson, Eberle,
		Rieder, Yi, Kruglyak, Nickerson (2004), &quot;Selecting a maximally
		informative set of single-nucleotide polymorphisms for association
		analysis using linkage disequilibrium&quot;, American Journal of
		Human Genetics, 74(1):106-120</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LDSUPPORT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Yutaka Kitamura
		(Institute of Rheumatology, Tokyo Women;s Medical University,
		email: kitamura@ior.twmu.ac.jp)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kitamura,
		Moriguchi, Kaneko, Morisaki, Morisaki, Toyama, Kamatani (2002),
		&quot;Determination of probability distribution of diplotype
		configuration (diplotype distribution) for each subject from
		genotypic data using the EM algorithm&quot;, Annals of Human
		Genetics, 66:183-193.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LEA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Likelihood-based
		Estimation of Admixture</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Olivier Langella,
		Lounes Chikhi, Mark Beaumont</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cnrs-gif.fr/pge/bioinfo/lea/index.php?lang=en</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: lea.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Langella, Chikhi,
		Beaumont (2001), &quot;LEA (Likelihood-based estimation of
		admixture): a program to simultaneously estimate admixture and the
		time since admixture&quot;, Molecular Ecology Notes, 1(4):357-358.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LINKAGE - general pedigrees
	(see also FASTLINK)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 5.1, 5.2, 6.0
		(testing)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: 5.1, 5.2: Mark
		Lathrop, J. Lalouel, C. Julier, Jurg Ott;</p>
		<li><p style="margin-bottom: 0.14in">6.0: Alan Young, Daniel Weeks,
		Mark Lathrop</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: 5.1,
		5.2: Pascal ; 6.0: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		OS2, UNIX, VMS</p>
		<li><p style="margin-bottom: 0.14in">executables: 5.1, 5.2:
		unknown, mlink, ilink, lodscore, linkmap; 6.0: linkage</p>
		<li><p style="margin-bottom: 0.14in">reference: Lathrop, Lalouel,
		Julier, Ott (1984), Proceedings of the National Academy of Sciences
		, 81:3443-3446.</p>
		<li><p style="margin-bottom: 0.14in">Lathrop, Lalouel (1984), &quot;Easy
		calculations of lod scores and genetic risks on small computers&quot;,
		American Journal of Human Genetics, 36(2):460-465</p>
		<li><p style="margin-bottom: 0.14in">Lathrop, Lalouel, White
		(1986), Genetic Epidemiology, 3:39-52.</p>
		<li><p style="margin-bottom: 0.14in">Am Soc Hum Genet annual
		meeting 1995 (Young, Weeks, Lathrop, Am J Hum Genet suppl, 57(4),
		A206 (1995)).</p>
		<li><p style="margin-bottom: 0.14in">online user's guide (may 93) .</p>
		<li><p style="margin-bottom: 0.14in">see also: the mystery of (the)
		unknown.</p>
		<li><p style="margin-bottom: 0.14in">see also: pedigree traversal</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LINKAGE - CEPH
	(three-generation) pedigrees (see also FASTLINK)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 5.1, 5.2</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Mark Lathrop, J.
		Lalouel, C. Julier, Jurg Ott</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		OS2, UNIX,VMS</p>
		<li><p style="margin-bottom: 0.14in">executables:
		cfactor,cilink,clodscore,cmap</p>
		<li><p style="margin-bottom: 0.14in">reference:
		Lathrop,Lalouel,Julier, Ott (1984), Proceedings of the National
		Academy of Sciences , 81:3443-3446.</p>
		<li><p style="margin-bottom: 0.14in">online user's guide (may 93)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LINKAGE-IMPRINT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: LINKAGE allowing
		for IMPRINITING</p>
		<li><p style="margin-bottom: 0.14in">version: January 2005</p>
		<li><p style="margin-bottom: 0.14in">description: We have developed
		a parametric model-based approach to analyzing pedigree data for
		genomic imprinting. We have modified widely used LINKAGE program to
		incorporate imprinting. In addition, the LINKAGE-IMPRINT program
		allows for the use of sex-specific recombination in the analysis,
		which is of particular importance in a genome-wide analysis for
		imprinted genes.</p>
		<li><p style="margin-bottom: 0.14in">authors: Sanjay Shete (email:
		sshete@mdanderson.org), Xiaojun Zhou (email:
		xiaojunzhou@mdanderson.org) (Univ of Texas MD Anderson Cancer
		Cancer)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		https://epi.mdanderson.org/~xzhou/Software/Linkage_imprinting/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables: mlinki, linkmapi</p>
		<li><p style="margin-bottom: 0.14in">reference: Shete and Zhou
		(2005), &quot;Parametric approach to genomic imprinting analysis
		with applications to Angelman's Syndrome&quot;, Human Heredity,
		59:26-33.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LINKBASE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: beta 0.95</p>
		<li><p style="margin-bottom: 0.14in">description: an easy and
		practical database-program made for researchers who want to connect
		the genotype data produced by automatic sequencers [ ABI Prism 377
		(Perkin Elmer) and ALF (Pharmacia) ] to linkage and sib-pair
		programs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Aki Suomalainen (Dept
		of Human Molecular Genetics, Natl Pub Health Institute, Finland,
		aki.suomalainen@ktl.fi)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ktl.fi/molbio/software/linkbase/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Delphi
		1.0</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(3.1/95/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LIPED</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Likelihoods in
		Pedigrees</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jurg Ott (Columbia
		Univ)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ott (1974),
		&quot;Estimation of the recombination fraction in human pedigrees:
		efficient computation of the likelihood for human linkage
		studies.&quot;, American Journal of Human Genetics, 26(5):588-597</p>
		<li><p style="margin-bottom: 0.14in">online documentation</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LNKTOCRI</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Conversion of
		LINKAGE format data files to CRI-MAP format</p>
		<li><p style="margin-bottom: 0.14in">authors:Jillian Blaschak and
		Tara Cox Matise (email: matise@biology.rutgers.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://compgen.rutgers.edu/multimap/</p>
		<li><p style="margin-bottom: 0.14in">http://compgen.rutgers.edu/multimap/src/other/lnktocri.p</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LOCUSMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: A computer
		package designed for rapid linkage analysis and map construction of
		loci with a variety of inheritance modes.</p>
		<li><p style="margin-bottom: 0.14in">authors: John Garbe
		(University of Minnesota, email: jgarbe@umn.edu), Yang Da
		(University of Minnesota, email: yda@umn.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://animalgene.umn.edu/locusmap/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Fortran
		90/95</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows,Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: locusmap</p>
		<li><p style="margin-bottom: 0.14in">reference: J Garbe, Y Da
		(2003), Locusmap user manual Version 1.0. Department of Animal
		Science, University of Minnesota.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LOCUSZOOM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: LocusZoom is
		designed to facilitate viewing of local association results
		together with useful information about a locus, such as the
		location and orientation of the genes it includes, linkage
		disequilibrium coefficients and local estimates of recombination
		rates. It was developed by popular demand, as a result of many
		questions we have had about &quot;How did you make the figures in
		your talk?&quot; or &quot;How did you make the figures for your
		GWAS paper?&quot;</p>
		<li><p style="margin-bottom: 0.14in">authors: Abecasis's group
		(Univ Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/LocusZoom</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LOGINSERM_ESTIHAPLO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Pierre-Antoine
		Gourraud (Unite INSERM, 558-Faculte de Medecine email:
		gourraud@cict.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gourraud, Genin,
		Cambon-Thomsen (2004), &quot;Handling missing values in population
		data: consequences for maximum likelihood estimation of haplotype
		frequencies&quot;, European Journal of Human Genetics, 12:805-812.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LOH-LINKAGE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a program using
		loss of heterozygosity data to enhance the power to detect linkage
		in cancer families.</p>
		<li><p style="margin-bottom: 0.14in">authors: Klaus Rohde (Max
		Delbruck Center for Molecular Medicine, Germany)
		(rohde@mdc-berlin.de), Mauro F. Santibanez Koref, M Dawn Teare</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bioinf.mdc-berlin.de/~rohde/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Rohde et al,
		(1995), Human Heredity, 45:337-345.</p>
		<li><p style="margin-bottom: 0.14in">Teare et al, J Med Genet 1994;
		31:448-452.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LOKI (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.4.5 (March 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: Program for
		analyses a quantitative trait observed on large pedigrees using
		Markov chain Monte Carlo multipoint linkage and segregation
		analysis. The trait may be determined by multiple loci.</p>
		<li><p style="margin-bottom: 0.14in">authors: Simon C. Heath
		(Centre National de Genotypage) (email: heath@cng.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/Loki.shtml</p>
		<li><p style="margin-bottom: 0.14in">http://loki.homeunix.net</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris 2.5, OSF1 3.2, and others), Linux (2.0.30)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Heath (1997),
		&quot;Markov chain segregation and linkage analysis for oligogenic
		models&quot;, American Journal of Human Genetics, 6(3):748-760</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LOT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Linkage analysis of
		Ordinal Traits</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Heping Zhang (email:
		heping@peace.med.yale.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://peace.med.yale.edu/pub/LOT/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux, UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Feng, Leckman,
		Zhang (2004), &quot;Linkage analysis of ordinal traits for pedigree
		data&quot;, Proceedings of the National Academy of Sciences ,
		101:16739-16744.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">L-POP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Jan 2001</p>
		<li><p style="margin-bottom: 0.14in">description: L-POP detects
		population stratification in samples of unrelated individuals for
		whom a number of unlinked genotypes have been measured.</p>
		<li><p style="margin-bottom: 0.14in">Using a latent class analysis
		model, population strata are defined such that unlinked markers are
		in Hardy-Weinberg equilibrium and linkage equilibrium within
		classes.</p>
		<li><p style="margin-bottom: 0.14in">L-POP calculates the posterior
		probabilities for each individual belonging to each of K classes,
		which can be used to correct for spurious effects of population
		stratification in tests of association.</p>
		<li><p style="margin-bottom: 0.14in">authors: Shaun Purcell, Pak
		Sham</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://statgen.iop.kcl.ac.uk/lpop/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: lpop</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LRP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Linkage Report
		Program</p>
		<li><p style="margin-bottom: 0.14in">alias: LINKAGE/LRP</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: part of the
		LINKAGE auxiliary program</p>
		<li><p style="margin-bottom: 0.14in">authors: Peter Cartwright
		(Univ Utah, now Cimarron Software <a href="mailto:pc@cimsoft.com">pc@cimsoft.com</a>)</p>
	</ul>
	<li><p align="left" style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 11pt">web:
	<font color="#0000ff"><span lang="en-US"><u><font color="#0000ff">http://lab.rockefeller.edu/ott/programs</font></u></span></font>
	 or
	<font color="#0000ff"><font face="Times New Roman, serif"><u>https://www.dropbox.com/sh/ulpuondvofkrvx1/AAAkIHrAqyQpHJlZaqtmTRb4a/Linkage%20Analysis</u></font></font>
	(please contact Alberto Goldoni, email:
	<font color="#0000ff"><font face="Times New Roman, serif"><u>alberto.goldoni1975@gmail.com</u></font></font>
	if you have any questions on this version)</font></font></p>
	<ul>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS, OS2</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LRTAE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Likelihood Ratio
		Test Allowing for Errors</p>
		<li><p style="margin-bottom: 0.14in">version: 1.3.1 (June 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: The purpose of
		this program is to compute a likelihood ratio test statistic that
		increases power to detect genetic association in the presence of
		phenotype, genotype, and/or haplotype misclassification errors. In
		addition, the program produces asymptotically unbiased estimates of
		frequency parameters.</p>
		<li><p style="margin-bottom: 0.14in">authors: Chad Haynes, Derek
		Gordon (email: Gordon@dls.rutgers.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://linkage.rockefeller.edu/software/lrtae/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gordon, Yang,
		Haynes, Finch, Mendell, Brown, Haroutunian (2004), &quot;Increasing
		power for tests of genetic association in the presence of phenotype
		and/or genotype error by use of double-sampling&quot;, Stat Appl
		Genet and Mol Biol 3:Article 26.</p>
		<li><p style="margin-bottom: 0.14in">online manual page</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LSP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Linkage Setup
		Program</p>
		<li><p style="margin-bottom: 0.14in">alias: LINKAGE/LSP</p>
		<li><p style="margin-bottom: 0.14in">version: This is a 16 bit
		program and will not run on 64 bit PCs, see
		<a href="http://www.jurgott.org/linkage/LinkagePC.html">http://www.jurgott.org/linkage/LinkagePC.html</a></p>
		<li><p style="margin-bottom: 0.14in">description: part of the
		LINKAGE auxiliary programs</p>
		<li><p style="margin-bottom: 0.14in">authors: Peter Cartwright
		(Univ Utah, now Cimarron Software <a href="mailto:pc@cimsoft.com">pc@cimsoft.com</a>)</p>
	</ul>
	<li><p align="left" style="margin-bottom: 0.14in"><font face="Times New Roman, serif"><font size="2" style="font-size: 11pt">web:
	<font color="#0000ff"><span lang="en-US"><u><font color="#0000ff">http://lab.rockefeller.edu/ott/programs</font></u></span></font>
	 or
	<font color="#0000ff"><font face="Times New Roman, serif"><u>https://www.dropbox.com/sh/ulpuondvofkrvx1/AAAkIHrAqyQpHJlZaqtmTRb4a/Linkage%20Analysis</u></font></font>
	(please contact Alberto Goldoni, email:
	<font color="#0000ff"><font face="Times New Roman, serif"><u>alberto.goldoni1975@gmail.com</u></font></font>
	if you have any questions on this version)</font></font></p>
	<ul>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS, OS2</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">LTSOFT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 beta</p>
		<li><p style="margin-bottom: 0.14in">description: LTSOFT is a
		software suite designed to more powerfully leverage
		clinical-covariates such as age, bmi, smoking status, and gender
		when conducting case-control association studies. Including these
		covariates in standard regression models is not only suboptimal,
		but can in many instances reduce power. LTSOFT employs a liability
		threshold model approach that takes advantage of known
		epidemiological results to better model the covariates'
		relationship to the phenotype of interest</p>
		<li><p style="margin-bottom: 0.14in">authors: Alkes Price (Harvard
		Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hsph.harvard.edu/faculty/alkes-price/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MACH (see also MINIMAC)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: MArkov Chain
		Haplotyper</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: MACH is a Markov
		Chain based haplotyper. It can be resolve long haplotypes or infer
		missing genotypes in samples of unrelated individuals.</p>
		<li><p style="margin-bottom: 0.14in">authors: Yun Li (email:
		ylwtx@umich.edu), Goncalo Abecasis (email: goncalo@umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/MACH/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MADELINE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.935 (July 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: Madeline is a
		program designed for preparing, visualizing, and exploring human
		pedigree data used in genetic linkage studies. In addition to
		converting pedigree and marker data into various formats required
		by linkage analysis software, Madeline also provides useful
		functionality for querying pedigree data sets and drawing
		pedigrees.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Edward Trager (email:
		ehtrager@umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://eyegene.ophthy.med.umich.edu/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Solaris/FreeBSD/OpenBSD/MacOS), MS-Windows, Cygwin, Linux,</p>
		<li><p style="margin-bottom: 0.14in">executables: madeline</p>
		<li><p style="margin-bottom: 0.14in">reference:
		http://eyegene.ophthy.med.umich.edu/madeline-0.935/</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MADMAPPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: September 2005</p>
		<li><p style="margin-bottom: 0.14in">description: suite of Python
		scripts for quality control of genetic markers, group analysis and
		inference of linear order of markers on linkage groups.
		MadMapper_RECBIT analyses raw marker scores for recombinant inbred
		lines. MadMapper_RECBIT generates pairwise distance scores for all
		markers, clusters based on pairwise distances, identifies genetic
		bins, assigns new markers to known linkage groups, validates allele
		calls, and assigns quality classes to each marker based on several
		criteria and cutoff values. MadMapper_XDELTA utilizes new
		algorithm, Minimum Entropy Approach and Best-Fit Extension, to
		infer linear order of markers. MadMapper_XDELTA analyzes
		two-dimensional matrices of all pairwise scores and finds best map
		that has minimal total sum of differences between adjacent cells
		(map with lowest entropy). MadMapper is freely available at
		http://www.atgc.org/XLinkage/MadMapper/</p>
		<li><p style="margin-bottom: 0.14in">authors: Alexander Kozik (UC
		Davis Genome Center)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.atgc.org/XLinkage/MadMapper/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Python</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: MadMapper_RECBIT,
		MadMapper_XDELTA</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAGIC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Marker And Gene
		Interpolation and Correlation</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Claire Simpson
		(c.simpson@iop.kcl.ac.uk), Ammar Al-Chalabi (ammar@iop.kcl.ac.uk),
		John Powell (john.powell@iop.kcl.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cogent.iop.kcl.ac.uk/MaGIC.cogx</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		Microsoft Visual Foxpro</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/NT/2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: CL Simpson, VK
		Hansen, PC Sham, A Collins, JF Powell, A Al-Chalabi (2004), &quot;MaGIC:
		A program to generate targeted marker sets for genome-wide
		association studies&quot;, BioTechniques, 37:996-999.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAIA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (Jan 2000)</p>
		<li><p style="margin-bottom: 0.14in">description: complex
		segregaton analysis in animal pedigrees</p>
		<li><p style="margin-bottom: 0.14in">authors: Yurii Aulchenko
		(email: yurii@bionet.nsc.ru)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mga.bionet.nsc.ru/soft/maia-1.0/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAKEPED</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: MAKE PEDfiles</p>
		<li><p style="margin-bottom: 0.14in">alias: LINKAGE/MAKEPED</p>
		<li><p style="margin-bottom: 0.14in">version: This is a 16 bit
		program and will not run on 64 bit PCs, see
		<a href="http://www.jurgott.org/linkage/LinkagePC.html">http://www.jurgott.org/linkage/LinkagePC.html</a></p>
		<li><p style="margin-bottom: 0.14in">description: part of the
		LINKAGE auxiliary programs</p>
		<li><p style="margin-bottom: 0.14in">authors: Peter Cartwright
		(University of Utah, now Cimarron Software pc@cimsoft.com)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS, OS2</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MALDSOFT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: MALDSoft is a
		program for admixture mapping of complex trait loci, using
		case-control data. The samples should come from a recently-admixed
		population; additional 'learning' samples from the parental
		populations are helpful.</p>
		<li><p style="margin-bottom: 0.14in">authors: Giovanni Montana,
		Jonathan Pritchard (email: jpritcha@genetics.uchicago.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pritch.bsd.uchicago.edu/software/maldsoft_download.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		Linux, MS-Windows (DOS/Window...)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: G Montana, JK
		Pritchard (2004), &quot;Statistical tests for admixture mapping
		with case-control and cases-only data&quot;, American Journal of
		Human Genetics, 75(5):771-789</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAMA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multivariate
		Association Mapping Algorithms</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://polymorphism.ucsd.edu/cgi-bin/PRL/mama/mama.cgi</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MANTEL-STRUCT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: tests for
		population structure through the use of Mantel tests</p>
		<li><p style="margin-bottom: 0.14in">authors: Mark Miller (email:
		mark.miller@usu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.marksgeneticsoftware.net/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Miller (1999),
		&quot;MANTEL-STRUCT: a program for the detection of population
		structure via mantel tests&quot;, Journal of Heredity, 90:258-259.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAOS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Meta-Analysis with
		Overlapping Subjects</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Data from
		genome-wide association studies are often analyzed jointly for the
		purposes of combining information from multiple studies of the same
		disease or comparing results across different disorders. In many
		instances, the same subjects appear in multiple studies. Failure to
		account for overlapping subjects can greatly inflate type I error
		when combining results from multiple studies of the same disease
		and can drastically reduce power when comparing results across
		different disorders. MAOS implements valid and efficient
		statistical methods for meta-analysis of genomewide association
		studies with overlapping subjects. The current release performs
		logistic regression analysis of individual level data under the
		additive mode of inheritance.</p>
		<li><p style="margin-bottom: 0.14in">authors: Danyu Lin, Patrick
		Sullivan (Univ North Carolina)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/MAOS/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lin, Sullivan
		(2009), &quot;Meta-analysis of genome-wide association studies with
		overlapping subjects&quot;, American Journal of Human Genetics,
		85(6):862-872.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAP/MAP+/MAP+H/MAP2000</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: (MAP+H is the
		radiation Hybrid module of MAP+)</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: multiple pairwise
		linkage analysis under interference</p>
		<li><p style="margin-bottom: 0.14in">authors: A Collins (email:
		arc@southampton.ac.uk),</p>
		<li><p style="margin-bottom: 0.14in">D. Shields,</p>
		<li><p style="margin-bottom: 0.14in">NE Morton (University of
		Southampton, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map</p>
		<li><p style="margin-bottom: 0.14in">http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map+</p>
		<li><p style="margin-bottom: 0.14in">http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map+h</p>
		<li><p style="margin-bottom: 0.14in">http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/map2000</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS/..)</p>
		<li><p style="margin-bottom: 0.14in">executables: map</p>
		<li><p style="margin-bottom: 0.14in">reference: Morton, Andrew
		(1989), Annals of Human Genetics, 53:263-269.</p>
		<li><p style="margin-bottom: 0.14in">Morton, Collins (1990), Annals
		of Human Genetics, 54:103-106.</p>
		<li><p style="margin-bottom: 0.14in">Shields, Collins, Buetow,
		Morton (1991), Proceedings of the National Academy of Sciences ,
		88:6501-6505.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPCHART</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Windows software
		for the graphical presentation of linkage maps and QTLs</p>
		<li><p style="margin-bottom: 0.14in">version: 2.2</p>
		<li><p style="margin-bottom: 0.14in">description: Produces charts
		of genetic linkage and QTL data. The charts are composed of a
		sequence of vertical bars representing the linkage groups or
		chromosomes. On these bars the positions of loci are indicated, and
		next to the bars QTL intervals and QTL graphs can be shown.
		MapChart reads the linkage information (i.e. the locus and QTL
		names and their positions) from text files. Many options to adapt
		the charts to different purposes. Can produce graphic files
		(enhanced windows metafile format) which can be enhanced with other
		MS-Windows software.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Roeland E Voorrips
		(Plant Research International, Wageningen, The Netherlands) (email:
		r.e.voorrips@plant.wag-ur.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biometris.wur.nl/uk/Software/MapChart/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Delphi</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/ME/NT4.0/2000)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: RE Voorrips (2001),
		&quot;MapChart version 2.0: Windows software for the graphical
		presentation of linkage maps and QTLs&quot;, Plant Research
		International, Wageningen, Netherlands.</p>
		<li><p style="margin-bottom: 0.14in">Voorrips (2002), &quot;MapChart:
		Software for the graphical presentation of linkage maps and QTLs&quot;,
		Journal of Heredity, 93(1):77-78.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPCREATOR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 4.0</p>
		<li><p style="margin-bottom: 0.14in">description: create gene maps
		using either radiation hybrid data or linkage data</p>
		<li><p style="margin-bottom: 0.14in">authors: Wes Barris, Rachel
		Hawken</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.wesbarris.com/mapcreator/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">price: Postscript ($69 USD),
		Image ($69 USD), both ($99 USD)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPDISTO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: v1.7.0 (October 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: MAPDISTO is a
		program for mapping genetic markers in experimental segregating
		populations like backcross, doubled haploids, single-seed descent.
		Its specificity is to propose recombination fraction estimates in
		case of segregation distortion. It can (1) compute and draw genetic
		maps easily and quickly through a graphical interface; (2)
		facilitate the analysis of marker data showing segregation
		distortion due to differential viability of gametes or zygotes.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mathias Lorieux
		(m.lorieux@cgiar.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mapdisto.free.fr/</p>
		<li><p style="margin-bottom: 0.14in">source code language: MS-Excel</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPDRAW</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.2</p>
		<li><p style="margin-bottom: 0.14in">description: MAPDRAW draws
		genetic linkage maps on PC same as what MAPMAKER does on Mac.</p>
		<li><p style="margin-bottom: 0.14in">authors: Renhu Liu (email:
		liurenhu@21cn.com)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.nslij-genetics.org/soft/mapdraw.v2.2.xls</p>
		<li><p style="margin-bottom: 0.14in">ftp://brassica.hzau.edu.cn</p>
		<li><p style="margin-bottom: 0.14in">ftp://211.69.140.177</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		Microsoft Visual Basic, Excel macro</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Liu, Meng (2003),
		&quot;MapDraw: a microsoft excel macro for drawing genetic linkage
		maps based on given genetic linkage data&quot;, Heraditas
		(Beijing), 25(3):317-321.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPINSPECT (see also GGT)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: May 2002 (previously
		called MAPCOMP)</p>
		<li><p style="margin-bottom: 0.14in">description: MAPINSPECT can be
		used to compare linkage maps obtained from different
		sources/populations/etc.. It will draw the linkage maps and look
		for common marker names, these are then connected in the drawing
		with dashed lines. All neighboring maps are compared in this way.
		Images can be printed and saved. Orders of the MAPs (ie which map
		is compared with which other map) can be changed and maps can be
		flipped (right mouse button). remark: MapComp bears close
		relationships with the GGT software package</p>
		<li><p style="margin-bottom: 0.14in">Authors: Ralph van Berloo,
		Wageningen University, The Netherlands (email:
		Ralph.vanberloo@pv.dpw.wau.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dpw.wau.nl/pv/PUB/MapComp/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Delphi
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Window
		(32-bit)</p>
		<li><p style="margin-bottom: 0.14in">executables: MapComp.exe</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: MAPping and QTL
		analysis</p>
		<li><p style="margin-bottom: 0.14in">version: March 2004</p>
		<li><p style="margin-bottom: 0.14in">description: The programs
		allow a user to get results on segregation ratio, linkage test,
		recombination value, grouping of markers, ordering of markers by
		metric multidimensional scaling, drawing map and graphical
		genotype. ALso QTL analysis by interval mapping and ANOVA are
		possible.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Ukai Tasuo, Ohsawa
		Ryo, Saito Akira, Hayashi Takeshi, Yasuo Ukai (University of Tokyo,
		Japan) (email: tkawai@lbm.ab.a.u-tokyo.ac.jp)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://lbm.ab.a.u-tokyo.ac.jp/software.html</p>
		<li><p style="margin-bottom: 0.14in">http://lbm.ab.a.u-tokyo.ac.jp/~ukai/mapl98.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Visual
		Basic (V5, professional)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yasuo, Ryo, Akira,
		Takeshi (1995), &quot;MAPL: a package of computer programs for
		construction of DNA polymorphismlinkage maps and analysis of QTL&quot;
		(in Japanese), Breeding Science, 45(1):139-142.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAREYMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: MareyMap is a
		meiotic recombination rate estimation program.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pbil.univ-lyon1.fr/software/mareymap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R,
		tcl/tk</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Rezvoy, Charif,
		Gueguen, Marais (2007), &quot;MareyMap: an R-based tool with
		graphical interface for estimating recombination rate&quot;,
		Bioinformatics, 23:2188-2189.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MARGARITA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Margarita infers
		genealogies from population genotype data and uses these to map
		disease loci. These genealogies take the form of the Ancestral
		Recombination Graph (ARG). The ARG defines a genealogical tree for
		each locus, and as one moves along the chromosome the topologies of
		consecutive trees shift according to the impact of historical
		recombination events.</p>
		<li><p style="margin-bottom: 0.14in">authors: Richard Durbin
		(Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/margarita/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MATLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: A MATLAB utility
		for estimating genetic linkage in exotic line-cross mating designs</p>
		<li><p style="margin-bottom: 0.14in">authors: J Clare Nelson
		(Kansas State University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://coding.plantpath.ksu.edu/~jcn/MatLink.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: MATLAB</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MDBLOCKS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Minimum Description
		length method for haplotype BLOCKS</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Using the mimimum
		description length model to delineate haplotype blocks</p>
		<li><p style="margin-bottom: 0.14in">authors: Eric C. Anderson and
		John Novembre (UC Berkeley)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://ib.berkeley.edu/labs/slatkin/eriq/software/mdb_web/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Anderson, Novembre
		(2003), &quot;Finding haplotype block boundaries by using the
		minimum description length criterion&quot;, American Journal of
		Human Genetics, 73(2):336-354</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPMAKER/EXP (see also
	MMDRAWER)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0 (1992)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Eric Lander, Phil
		Green, J. Abrahamson, A. Barlow, Mark Daly, Steve Lincoln, L.
		Newburg (Whitehead Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broad.mit.edu/ftp/distribution/software/mapmaker3/</p>
		<li><p style="margin-bottom: 0.14in">http://www-genome.wi.mit.edu/genome_software</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables: mapmaker</p>
		<li><p style="margin-bottom: 0.14in">reference: Lander et al
		(1987), Genomics,1,174-181.</p>
		<li><p style="margin-bottom: 0.14in">online tutorial (version 3.0,
		1987-1993)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPMAKER/HOMOZ</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: homozygosity
		mapping</p>
		<li><p style="margin-bottom: 0.14in">version: May 1995</p>
		<li><p style="margin-bottom: 0.14in">description: calculates
		multipoint lod scores in pedigrees with inbreeding loops</p>
		<li><p style="margin-bottom: 0.14in">authors: Leonid Kruglyak
		(email: leonid@genome.wi.mit.edu),</p>
		<li><p style="margin-bottom: 0.14in">Mark Daly (email:
		mjdaly@genome.wi.mit.edu),</p>
		<li><p style="margin-bottom: 0.14in">Eric Lander (email:
		lander@genome.wi.mit.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://ftp-genome.wi.mit.edu/distribution/software/homoz</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kruglyak, Daly,
		Lander (1995), &quot;Rapid multipoint linkage analysis of recessive
		traits in nuclear families, including homozygosity mapping&quot;,
		American Journal of Human Genetics, 56(2):519-527</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPMAKER/QTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: new version in April
		1995</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Eric Lander, Phil
		Green, J. Abrahamson, A. Barlow, Mark Daly, Steve Lincoln, L.
		Newburg (Whitehead Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-genome.wi.mit.edu/genome_software</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp-genome.wi.mit.edu/distribution/software/newqtl&quot;
		and ftp://ftp-genome.wi.mit.edu/distribution/software/mapmaker3</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lander et al
		(1987), Genomics,1,174-181.</p>
		<li><p style="margin-bottom: 0.14in">online tutorial and reference
		manual (v1.1, 1993)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPMAKER/SIBS (see also
	GENEHUNTER)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version : 2.1 (Dec 1996)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Leonid Kruglyak
		(email: leonid@genome.wi.mit.edu),</p>
		<li><p style="margin-bottom: 0.14in">Mark Daly (email:
		mjdaly@genome.wi.mit.edu),</p>
		<li><p style="margin-bottom: 0.14in">Eric Lander (email:
		lander@genome.wi.mit.edu) (Whitehead Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://ftp-genome.wi.mit.edu/distribution/software/sibs</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: sibs</p>
		<li><p style="margin-bottom: 0.14in">reference: Kruglyak, Lander
		(1995), &quot;Complete multipoint sib-pair analysis of qualitative
		and quantitative traits&quot;, American Journal of Human Genetics,
		57(2):439-454</p>
		<li><p style="margin-bottom: 0.14in">the changes from version 1.1
		to 2.0 .</p>
		<li><p style="margin-bottom: 0.14in">manual (in postscript)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAP MANAGER QT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version : QTb29 (August 2000)</p>
		<li><p style="margin-bottom: 0.14in">description: A graphic,
		interactive program to map quantitative trait loci by regression
		methods; MAP MANAGER CLASSIC enhanced by quantitative trait
		mapping.</p>
		<li><p style="margin-bottom: 0.14in">authors: K.F. Manly</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mapmanager.org/mmQT.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Manly and Olsen,
		Mammalian Genome, 10:327-334 (1999)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAP MANAGER QTX</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version : QTXb20 (April 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: A graphic,
		interactive program to map quantitative trait loci using
		intercrosses, backcrosses or recombinant inbred strains in
		experimental plants or animals. A completely rewritten
		cross-platform version of Map Manager QT with enhanced analysis
		functions.</p>
		<li><p style="margin-bottom: 0.14in">Authors: KF Manly, RH Cudmore,
		Jr., and JM Meer</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mapmanager.org/mmQTX.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++ and
		cross-platform code from XVT</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Manly, Cudmore, and
		Meer (2001), &quot; Map Manager QTX, cross-platform software for
		genetic mapping&quot;, Mammalian Genome, 12(12):930-932.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPPOP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (Jan 2000)</p>
		<li><p style="margin-bottom: 0.14in">description: selects high
		resolution mapping subsamples and performs bin mapping</p>
		<li><p style="margin-bottom: 0.14in">Authors: Dan Brown, Todd
		Vision (email: tjv@bio.unc.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bio.unc.edu/faculty/vision/lab/mappop/</p>
		<li><p style="margin-bottom: 0.14in">source code language: MatLab</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MacOS, etc</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Vision, Brown,
		Shmoys, Durrett, Tanksley (2000), &quot;Selective Mapping: A
		strategy for optimizing the construction of high-density linkage
		maps&quot;, Genetics, 155: 407-420</p>
		<li><p style="margin-bottom: 0.14in">Brown, Vision, Tanksley
		(2000), &quot;Selective mapping: A discrete optimization approach
		to selecting a population subset for use in a high-density genetic
		mapping project&quot;, Proceedings of the 11th Annual {ACM-SIAM}
		Symposium on Discrete Algorithms, pp.419-428.</p>
		<li><p style="margin-bottom: 0.14in">Xu, Zou, Vision (2005),
		&quot;Improving QTL mapping resolution in experimental crosses by
		the use of genotypically selected samples&quot;, Genetics, 170,
		401-408.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MAPQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Software for the
		calculation of QTL positions on genetic maps</p>
		<li><p style="margin-bottom: 0.14in">version: 5 (October 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: Mapping of
		quantitative trait loci (QTLs) for several types of mapping
		populations: BC1, F2, RILs, (doubled) haploids, full-sib family of
		outbreeders. Analyses: interval mapping, composite interval
		mapping, nonparametric mapping, automatic cofactor selection,
		permutation test for interval mapping. QTL charts. Everything
		available in an intuitive MS-Windows user interface.</p>
		<li><p style="margin-bottom: 0.14in">authors: Johan W Van Ooijen,
		Martin P Boer, Ritsert C Jansen, Chris Maliepaard (Plant Research
		International, Biometris, Wageningen, Netherlands) (email:
		mapping@biometris.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: http://www.mapqtl.nl</p>
		<li><p style="margin-bottom: 0.14in">source code language: ANSI C,
		Delphi</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/ME/NT4.0/2000)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: JW Van Ooijen, MP
		Boer, RC Jansen, C Maliepaard (2002), &quot;MapQTL (tm) 4.0:
		Software for the calculation of QTL positions on genetic maps&quot;,
		Plant Research International, Wageningen, Netherlands; JW Van
		Ooijen, C Maliepaard (1996), &quot;MapQTL Version 3.0: Software for
		the Calculation of QTL Positions on Genetic Maps&quot;, Plant
		Genome IV (1996) ( http://www.intl-pag.org/4/abstracts/p316.html);
		JW Van Ooijen, C Maliepaard (1996), &quot;MapQTL Version 3.0:
		Advanced QTL Mapping Software&quot;, Plant Genome IV (1996) (
		http://www.intl-pag.org/4/abstracts/c9.html).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MCLEEPS (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Monte Carlo
		Likelihood Estimation of Effective Population Size</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (September 2001)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">Authors: Eric Anderson (email:
		eriq@cqs.washington.edu , University of Washington) and Ellen
		Williamson (University of California at Berkeley)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml</p>
		<li><p style="margin-bottom: 0.14in">http://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Anderson,
		Williamson, Thompson (2000), &quot;Monte Carlo evaluation of the
		likelihood for Ne from temporally spaced samples&quot;, Genetics,
		156:2109-2118.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MCQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multi-Cross QTL
		mapping</p>
		<li><p style="margin-bottom: 0.14in">version: v4.0</p>
		<li><p style="margin-bottom: 0.14in">description: The aim of the
		MCQTL software package is to perform QTL mapping in multi-cross
		designs. It allows the analysis of the usual populations derived
		from inbred lines and can link the families by assuming that the
		QTL locations are the same in all them. Moreover, a diallel
		modelling of the QTL genotypic effects is allowed in multiple
		related families. Obviously, the analysis of a single cross is also
		feasible.</p>
		<li><p style="margin-bottom: 0.14in">authors: Marie-Fran&ccedil;oise
		Jourjon, Sylvain Jasson, Jacques Marcel, Baba Ngom, Brigitte Mangin
		(email: brigitte.mangin@toulouse.inra.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://carlit.toulouse.inra.fr/MCQTL/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java,
		C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Jourjon, Jasson,
		Marcel, Ngom, Mangin (2005), &quot;MCQTL: Multi-allelic QTL mapping
		in multi-cross design&quot;, Bioinformatics, 21:128-130.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MDR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multifactor
		Dimensionality Reduction</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0 beta 6</p>
		<li><p style="margin-bottom: 0.14in">description: MDR is a data
		mining strategy for detecting and characterizing nonlinear
		interactions among discrete attributes (e.g. SNPs, smoking, gender,
		etc.) that are predictive of a discrete outcome (e.g. case-control
		status). The MDR software combines attribute selection, attribute
		construction and classification with cross-validation to provide a
		powerful approach to modeling interactions.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jason Moore
		(email:jason.h.moore@dartmouth.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.multifactordimensionalityreduction.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ritchie, Hahn,
		Roodi, Bailey, Dupont, Parl, Moore (2001), &quot;Multifactorial
		dimensionality reduction reveals high-order interactions among
		estrogen metabolism genes in sporadic breat cancer&quot;, American
		Journal of Human Genetics, 69:138-147.</p>
		<li><p style="margin-bottom: 0.14in">Moore, Gilbert, Tsai, Chiang,
		Holden, Barney, White (2006), &quot;A flexible computational
		framework for detecting, characterizing, and interpreting
		statistical patterns of epistasis in genetic studies of human
		disease susceptibility&quot;, Journal of Theoretical Biology,
		241:252-261.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MDR-PDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multifactor
		Dimensionality Reduction and genotype Pedigree Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version: beta</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: ER Martin, MD
		Ritchie, L Hahn, S Kang, JH Moore</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://chgr.mc.vanderbilt.edu/ritchielab/method.php?method=mdrpdt</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Martin, Ritchie,
		Hahn, Kang, Moore (2006), &quot;A novel method to identify
		gene-gene effects in nuclear families: the MDR-PDT&quot;, Genetic
		Epidemiology, 30:111-123.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MECPM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Maximum Entropy
		Conditional Probability Moldeling</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cbil.ece.vt.edu/ResearchOngoingSNP.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Miller, Zhang, Yu,
		Liu, Chen, Langefeld, Herrington, Wang (2009), &quot;An algorithm
		for learning maximum entropy probability models of disease risk
		that efficiently searches and sparingly encodes multilocus genomic
		interactions&quot;, Bioinformatics, to appear.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MEGA2</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: a Manipulation
		Environment for Genetic Analyses</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0 R9 (July 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Mega2 uses as
		input a trio of files: 1) a LINKAGE-format locus file modified to
		contain locus name information; 2) a LINKAGE-format pedigree file;
		and 3) a map file. Mega2 then takes this trio of input files and,
		via a menu-driven interface, transforms them into various other
		file formats, thus greatly facilitating a variety of different
		analyses. In addition, for many of these options, it also sets up a
		C-shell script that then can automatically run these analyses (if
		you are using Mega2 in a Unix environment that supports C-shell
		scripts).</p>
		<li><p style="margin-bottom: 0.14in">authors: Nandita Mukhopadhyay,
		Lee Almasy, Mark Schroeder, William P. Mulvihill, Daniel E Weeks</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Solaris, Silicon Graphics, OSF1, MacOS X), Linux, MS-Windows,
		MacOS(X)</p>
		<li><p style="margin-bottom: 0.14in">executables: mega2</p>
		<li><p style="margin-bottom: 0.14in">reference: Mukhopadhyay N,
		Almasy L, Schroeder M, Mulvihill WP, Weeks DE (1999), &quot;Mega2,
		a data-handling program for facilitating genetic linkage and
		association analyses&quot; (abstract), American Journal of Human
		Genetics, 65:A436</p>
		<li><p style="margin-bottom: 0.14in">Mukhopadhyay, Almasy,
		Schroeder, Mulvihill, Weeks (2005), &quot;Mega2: data-handling for
		facilitating genetic linkage and association analyses&quot;,
		Bioinformatics, 21(10):2556-2557.</p>
		<li><p style="margin-bottom: 0.14in">online documentation:
		http://watson.hgen.pitt.edu/docs/mega2_html/mega2.html</p>
		<li><p style="margin-bottom: 0.14in">FAQ-O-MATIC:
		http://watson.hgen.pitt.edu/fom-serve/cache/2.html</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MEGASNPHUNTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: MegaSNPHunter
		takes case-control genotype data as input and produces a ranked
		list of multi-SNP interactions. In particular, the whole genome is
		first partitioned into multiple short subgenomes and a boosting
		tree classifier is built for each subgenomes based on multi-SNP
		interactions and then used to measure the importance of SNPs. The
		method keeps relatively more important SNPs from all subgenomes and
		let them compete with each other in the same way at the next level.
		The competition terminates when the number of selected SNPs is less
		than the size of a subgenome.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinformatics.ust.hk/MegaSNPHunter.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wan, Yang, Yang,
		Xue, Tang, Yu (2009), &quot;MegaSNPHunter: a learning approach to
		detect disease predisposition SNPs and high level interactions from
		whole genome data&quot;, BMC Bioinformatics, 10:13.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MENDEL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 8.0.1 (expire April
		2009)</p>
		<li><p style="margin-bottom: 0.14in">description: genetic analysis
		of human pedigree data under models involving a small number of
		loci. MENDEL is useful for segregation analysis, linkage
		calculations, genetic counseling, allele frequency estimation, and
		related kinds of problems.</p>
		<li><p style="margin-bottom: 0.14in">authors: Kenneth Lange (UCLA),
		Daniel Weeks (Pittsburgh), M. Boehnke</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genetics.ucla.edu/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lange, Weeks,
		Boehnke (1988), Genetic Epidemiology, 5:471-471.</p>
		<li><p style="margin-bottom: 0.14in">module USERM14: Stringham,
		Boehnke (1996), &quot;Identifying marker typing incompatibilities
		in linkage analysis&quot;, American Journal of Human Genetics,
		59(4):946-950</p>
		<li><p style="margin-bottom: 0.14in">Lange, Cantor, Horvath,
		Perola, Sabatti, Sinsheimer, Sobel (2001), Am. J. Hum. Genet.
		69(suppl), A1886.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MENDELSOFT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.5b (July 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: Identifying all
		Mendelian inconsistencies in complex pedigree data with thousand of
		individuals, including many loops and several errors. Can also
		infer missing genotypes.</p>
		<li><p style="margin-bottom: 0.14in">authors: M Sanchez, Simon de
		Givry (INRA, Centre de recherches de Toulouse, Station de biom&eacute;trie
		et d'intelligence artificielle, France) (email:
		degivry@toulouse.inra.fr), T Schiex</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.inra.fr/mia/T/MendelSoft/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		Linux/Cygwin, MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sanchez, de Givry,
		Schiex (2007), &quot;Mendelian error detection in complex pedigrees
		using weighted constraint satisfaction techniques&quot;, Frontiers
		in Artificial Intelligence and Applications, 163:29-37.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MERLIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multipoint Engine
		for Rapid Likelihood INference</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.1</p>
		<li><p style="margin-bottom: 0.14in">description: Merlin carries
		out single-point and multipoint analyses of pedigree data,
		including IBD and kinship calculations, nonparametric and variance
		component linkage analyses, error detection and information content
		mapping. For multipoint analyses in dense maps, Merlin allows the
		user to impose constraints on the number of recombinants between
		consecutive markers. Merlin estimates haplotypes by finding the
		most likely path of gene flow or by sampling paths of gene flow at
		all markers jointly. It can also list all possible nonrecombinant
		haplotypes within short regions. Finally, Merlin provides swap-file
		support for handling very large numbers of markers as well as
		gene-dropping simulations for estimating empirical significance
		levels.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Goncalo Abecasis
		(email: goncalo@umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/Merlin</p>
		<li><p style="margin-bottom: 0.14in">http://www.sph.umich.edu/csg/abecasis/Merlin/download</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX. LINUX</p>
		<li><p style="margin-bottom: 0.14in">executables: merlin, pedstats,
		pedwipe</p>
		<li><p style="margin-bottom: 0.14in">reference: Abecasis GR, Cherny
		SS, Cookson WO and Cardon LR (2002), &quot;Merlin-rapid analysis of
		dense genetic maps using sparse gene flow trees&quot;, Nature
		Genetics, 30:97-101.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">METAL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: METa AnaLysis
		Helper</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The METAL
		software is designed to facilitate meta-analysis of large datasets
		(such as several whole genome scans) in a convenient, rapid and
		memory efficient manner.</p>
		<li><p style="margin-bottom: 0.14in">authors: Goncalo Abecasis
		(email: goncalo@umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/Metal/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MFLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: (nearly) model-free
		linkage analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis, (email:
		dcurtis@hgmp.mrc.ac.uk) P.C. Sham</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Curtis, Sham
		(1995), &quot;Model-free linkage analysis using likelihoods&quot;,
		American Journal of Human Genetics, 57(3):703-716</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MGA-MAPF2</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: map QTLs in F-2
		intercross in model organisms</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mga.bionet.nsc.ru/soft/mgamapf2/</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		MS-Windows (95/98/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MIDAS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multiallelic
		Interallelic Disequilibrium Analysis Software</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: software for
		analysis and visualisation of interallelic disequilibrium between
		multiallelic markers</p>
		<li><p style="margin-bottom: 0.14in">authors: Tom R Gaunt, Santiago
		Rodriguez, Carlos Zapata, Ian NM Day</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genes.org.uk/software/midas</p>
		<li><p style="margin-bottom: 0.14in">http://www.sgel.humgen.soton.ac.uk/midas</p>
		<li><p style="margin-bottom: 0.14in">source code language: Python</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(2000/XP), Linux, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gaunt, Rodriguez,
		Zapata, Day (2006), &quot;MIDAS: software for analysis and
		visualisation of interallelic disequilibrium between multiallelic
		markers&quot;, BMC Bioinformatics, 7:227.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MILD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: MultIallelic
		Linkage Disequilibrium: a program for adjusted linkage
		disequilibrium (LD) calculations</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1 (2002)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Yuril S Aulchenko
		(email: aulchenko@epib.fgg.eur.nl) (Erasmus University Medical
		School, The Netherlands)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.geneticepi.com/Research/software/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multipoint
		Identical-by-descent Method</p>
		<li><p style="margin-bottom: 0.14in">version: 1.21 (March 1996)</p>
		<li><p style="margin-bottom: 0.14in">description: multipoint IBD
		method for partitioning genetic variance of quantitative traits to
		specific chromosome regions using data on nuclear families.</p>
		<li><p style="margin-bottom: 0.14in">authors: David Goldgar,
		Cathryn M. Lewis (Univ. Utah). contact person: Edward Kort at
		edward@episun2.med.utah.edu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://morgan.med.utah.edu/pub/Mim</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Goldgar (1990),
		&quot;Multipoint analysis of human quantitative genetic variation&quot;,
		American Journal of Human Genetics, 47(6): 957-967</p>
		<li><p style="margin-bottom: 0.14in">Goldgar, Oniki (1992),
		&quot;Comparison of a multipoint identity-by-descent method with
		parametric multipoint linkage analysis for mapping quantitative
		traits&quot;, American Journal of Human Genetics, 50(3):598-606</p>
		<li><p style="margin-bottom: 0.14in">online manual (from Wellcome
		Trust)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MINIMAC (see also MACH)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: minimac is a low
		memory, computationally efficient implementation of the MaCH
		algorithm for genotype imputation. It is designed to work on phased
		genotypes and can handle very large reference panels with hundreds
		or thousands of haplotypes. The name has two parts. The first,
		&quot;mini&quot;, refers to the modest amount of computational
		resources it requires. The second, &quot;mac&quot;, is short hand
		for MaCH, our widely used algorithm for genotype imputation.</p>
		<li><p style="margin-bottom: 0.14in">authors: Goncalo Abecasis,
		Christian Fuchsberger (Univ Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/Minimac</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MINSAGE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: MINimal SAmple size
		for GEnotypes</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: calculate the
		sample size of genotypes minimally required to ensure that all
		alleles with a specified frequency at one locus are detected with a
		given confidence</p>
		<li><p style="margin-bottom: 0.14in">authors: Andreas Ziegler
		(email: koenigir@imbs.uni-luebeck.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.imbs.uni-luebeck.de/pub/minsage/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: HG Gregorius
		(1980), &quot;The probability of losing an allele when diploid
		genotypes are sampled&quot;, Biometrics, 36:643-652.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MITPENE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: A program for
		analysis of mitochondrial diseases</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mga.bionet.nsc.ru/soft/mitpene/mitpene.html (in Russian)</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MIXSCORE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.9 (March 2011)</p>
		<li><p style="margin-bottom: 0.14in">description: MIXSCORE is a
		method for combining SNP association and admixture association
		statistics to increase power in GWAS in admixed populations.</p>
		<li><p style="margin-bottom: 0.14in">authors: Alkes Price (Harvard
		Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hsph.harvard.edu/faculty/alkes-price/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Pasaniuc et al.
		(2011), &quot;Enhanced statistical tests for GWAS in admixed
		populations: assessment using African Americans from CARe and a
		breast cancer consortium&quot;, PLoS Genetics, 7:e1001371.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MKGST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Michael Krawczak's
		Genetics Software Tools, including ASP (power calculator for gene
		mapping using a sibpair design), ASPSHARE (rapid calculation of the
		expected ibd sharing at the trait locus, based upon the model),
		EASYPAT (calculation of likelihood ratios for single locus data,
		comparing specific types of simple hypotheses regarding the
		familial relationships involved), MUTPROF/MUTCOMP (comparison of
		mutation profiles), FINDSIRE (identify mothers or sires by means of
		the comparison of a large number of potential parents, typed at
		single locus DNA markers, with given infants or parent-infant
		duos), PATERN (calculation of paternity probabilities from the
		multilocus DNA profiles of trios, comprising mother, child and
		putative father)</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">Authors: Michael Krawczak</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uni-kiel.de/medinfo/mitarbeiter/krawczak/download/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: asp,
		aspshare,easypat, mutprof,mutcomp, findsire,patern,fingcomp,hapmax,
		mutpred</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MMDRAWER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.2.0.0 (April 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: draw chromosome
		maps from the output of MAPMAKER/EXP. The output format is a
		Enhanced Metafile, which can be imported into most Windows-based
		presentation or document editing programs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Brown</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.emboss.co.nz/products.php?pid=2</p>
		<li><p style="margin-bottom: 0.14in">source code language: Delphi</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MLBGH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:
		Maximum-Likelihood-Binomial analysis of affected sib-pair and
		sibship data extended from the GENEHUNTER program</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: an extension of
		the GENEHUNTER program to perform sib-pair and sib-ship linkage
		analysis using the Maximum Likelihood Binomial (MLB) method.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Laurent Abel (INSERM
		Unit 436, Paris, France, email: abel@biomath.jussieu.fr), Bertram
		Muller-Myhsok (Bernhard Nocht Institute, Hamburg, Germany, email:
		bmm@bni.uni-hamburg.de), Anne Philippi (INSERM Unit 436, Paris,
		France, email: philippi@biomath.jussieu.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://ftp.biomath.jussieu.fr/pub/mlbgh (not available)</p>
		<li><p style="margin-bottom: 0.14in">http://genamics.com/software/downloads/mlbgh-1.0.tar.Z</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: mlbgh</p>
		<li><p style="margin-bottom: 0.14in">reference: Abel, Muller-Myhsok
		(1998), &quot;Robustness and power of the
		maximum-likelihood-binomial and maximum-likelihood-score methods,
		in multipoint linkage analysis of affected-sibship data&quot;,
		American Journal of Human Genetics, 63(2):638-647</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MLD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Aug 1997</p>
		<li><p style="margin-bottom: 0.14in">description: Performing a
		shuffling version of the exact conditional tests for different
		combinations of allelic and genotypic disequilibrium on haploid and
		diploid data, or their combination.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Dmitri Zaykin (email:
		zaykin@statgen.ncsu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://statgen.ncsu.edu/pub/zaykin/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows(95)</p>
		<li><p style="margin-bottom: 0.14in">executables: mld.x</p>
		<li><p style="margin-bottom: 0.14in">reference: Zaykin,
		Zhivotovsky, Weir (1995), Genetica, 96:169-178.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MLR-TAGGING</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Multivariate Linear
		Regression TAGGING</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: MLR-TAGGING can
		be used for tagging SNP selection and genotype prediction</p>
		<li><p style="margin-bottom: 0.14in">authors: Alexander Zelikovsky
		(Georgia State Univ) (email: alexz@cs.gsu.edu), Jingwu Jim He
		(email: jingwu@cs.gsu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://alla.cs.gsu.edu/~software/tagging/tagging.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: He, Zelikovsky
		(2006), &quot;MLR-tagging: informative SNP selection for unphased
		genotypes based on multiple linear regression&quot;,
		Bioinformatics, 22(20):2558-2561.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MOSCPHASER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Mixture Of Cnv-Snp
		PHASER</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: inferring
		haplotypes composed of both CNV alleles and SNP alleles</p>
		<li><p style="margin-bottom: 0.14in">authors: Mamoru Kato, Yusuke
		Nakamura, Tatsuhiko Tsunoda (email: tsunoda@src.riken.jp)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://emu.src.riken.jp/MOCSphase/MOSCphaser.zip</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kato, Nakamura,
		Tsunoda (2008), &quot;MOCSphaser: a haplotype inference tool from a
		mixture of copy number variation and single nucleotide polymorphism
		data&quot;, Bioinformatics, 24(14):1645-1646.</p>
		<li><p style="margin-bottom: 0.14in">http://emu.src.riken.jp/MOCSphase/SuppInfor.doc</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MOLKIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: MOLKIN is a
		population genetics computer program that conducts several genetic
		analyses on multilocus information in a user-friendly environment.
		Primary functions carried out by MOLKIN are the computation of the
		between individuals (and populations) molecular coancestry
		coefficients, the Kinship distance at individual and population
		levels. Additionally, users can compute with MOLKIN a set of among
		populations, genetic distances and F statistics from multilocus
		information. The program will help researchers or those responsible
		for population management to assess genetic variability and
		population structure at reduced costs with respect to dataset
		preparation</p>
		<li><p style="margin-bottom: 0.14in">authors: JP Gutierrez, LJ
		Royo, I Alvarez, F Goyache (Universidad Complutense de Madrid,
		Spain)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ucm.es/info/prodanim/html/JP_Web.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gutierrez, Royo,
		Alvarez, Goyache (2005), &quot;Molkin v2.0: a computer program for
		genetic analysis of populations using molecular coancestry
		information&quot;, Journal of Heredity, 96:718-721.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MORGAN (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: MOnte caRlo Genetic
		ANalysis</p>
		<li><p style="margin-bottom: 0.14in">version: version 2.8.3 (March
		2008)</p>
		<li><p style="margin-bottom: 0.14in">description: Programs for
		segregation and linkage analysis, using a variety of Markov chain
		Monte Carlo (MCMC) methods. Includes MCMC methods for multilocus
		gene identity by descent (including homozygosity mapping) and Monte
		Carlo Lod scores. Also, other programs for EM analysis of
		quantitative traits. MORGAN 2.5 replaces MORGAN 2.4.1 (June 2002)
		which superceded MORGAN 2.4 (February 2002) and MORGAN 2.3.2
		(September, 2001). MORGAN 2.4 supercedes MORGAN 2.3 (April 2001),
		MORGAN 2.2 (April 2000) and MORGAN 1.1.2 (July 1999)</p>
		<li><p style="margin-bottom: 0.14in">authors: For MORGAN 2.5: M A
		Jewett (University of Washington, email: myrna@stat.washington.edu)
		Other authors of MORGAN programs include: EA Thompson (University
		of Washington) AW George, CJ Geyer, S Guo, SC Heath, Michael Li, S
		Lin, NA Sheehan, and G Snow .</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/MORGAN/Morgan.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Compaq-Alpha, Solaris and others), Linux</p>
		<li><p style="margin-bottom: 0.14in">reference: Thompson (1995)
		&quot;Monte Carlo in Genetic Analysis&quot;, Technical report No.
		294, Department of Statistics, University of Washington.
		[postscript]</p>
		<li><p style="margin-bottom: 0.14in">Thompson (2000) Statistical
		Inferences from Genetic Data on Pedigrees NSF-CBMS Regional
		Conference Series in Probability and Statistics. Volume 6. IMS,
		Beachwood, OH.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MPDA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Microarray Pooled
		DNA Analyser</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1 (March 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: MPDA is an
		innovative tool for analyzing hybridization intensity data from
		microarray-based pooled DNA experiments. MPDA was developed under
		the software platform, MATLABR, and provided user-friendly
		interfaces adapted to Windows systems (Windows 98, Windows 2000 and
		Windows XP). or users without installing software MATLABR, we also
		developed stand-alone executables generated via the MATLABR
		compiler. MPDA provides four major functions: (1) Whole-genome DNA
		amplification/hybridization analysis, (2) Allele frequency
		estimation, (3) Association mapping, (4) Allelic imbalance
		detection. Graphic and numerical outputs from MPDA support global
		and detailed inspection for bulk of genomic data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Hsin-Chou Yang
		(email:hsinchou@stat.sinica.edu.tw), Mei-Chu Huang, Ling-Hui Li,
		Chien-Hsing Lin, Alice L T Yu, Mitchell B Diccianni, Jer-Yuarn Wu,
		Yuan-Tsong Chen, Cathy S J Fann</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.sinica.edu.tw/hsinchou/genetics/pooledDNA/mpda.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: MATLABR</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(Windows98/2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables: mpda</p>
		<li><p style="margin-bottom: 0.14in">reference: Yang, Huang, Li,
		Lin, Yu, Diccianni, Wu, Chen, Fann (2008), &quot;MPDA: Microarray
		Pooled DNA Analyzer&quot;, BMC Bioinformatics, 9:196.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MRH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Minimum Recombinant
		Haplotype</p>
		<li><p style="margin-bottom: 0.14in">version: 0.2 (May 2003)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">Authors: Dajun Qian (City of
		Hope National Medical Center, email: dqian@coh.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-rcf.usc.edu/~gqian/software.htm (not available)</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Qian, Beckmann
		(2002), &quot;Minimum-recombinant haplotyping in pedigrees&quot;,
		American Journal of Human Genetics, 70(6):1434-1445</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MULTIDISEQ</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: MULTIDISEQ is
		multipoint linkage analysis software which allows Marker-Marker LD</p>
		<li><p style="margin-bottom: 0.14in">authors: Tero Hiekkalinna
		(email: tero.hiekkalinna@helsinki.fi)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.helsinki.fi/~tsjuntun/multidiseq/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: T Hiekkalinna, L
		Peltonen1, JD Terwilliger (2003), &quot;MULTIDISEQ: Computer
		software for multipoint linkage analysis of complex traits allowing
		for marker-marker LD (Linkage Disequilibrium)&quot; (abstract),
		American Journal of Human Genetics, suppl, 73</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MULTIMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: program for
		automated construction of genetic maps</p>
		<li><p style="margin-bottom: 0.14in">authors: Tara Matise (email:
		matise@biology.rutgers.edu), Mark Perlin, Aravinda Chakravarti</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://compgen.rutgers.edu/Multimap/</p>
		<li><p style="margin-bottom: 0.14in">http://compgen.rutgers.edu/multimap/multimapdist.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: LISP</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Sun/Compaq-Alpha/HP..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Matise,Perlin,
		Chakravarti, Nature Genetics, 6, 384-390 (1994);</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MULTIMAPPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Bayesian QTL
		mapping software for analysing backcross, double haploid, and F2
		data from designed crossing experiments of inbred lines</p>
		<li><p style="margin-bottom: 0.14in">authors: Mikko J Sillanpaa
		(mail: mjs@rolf.helsinki.fi)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.rni.helsinki.fi/~mjs/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sillanpaa, Arjas
		(1998), &quot;Bayesian mapping of multiple quantitative trait loci
		from incomplete inbred line cross data&quot;, Genetics,
		148:1373-1388.</p>
		<li><p style="margin-bottom: 0.14in">Martinez, Thorgaard, Robison,
		Sillanpaa (2005), &quot;An application of Bayesian QTL mapping to
		early development in double haploid lines of rainbow trout
		including environmental effects&quot;, Genetical Research,
		86:209-221.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MULTIMAPPER/OUTBRED</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: Bayesian QTL
		mapping software for analysing backcross and F2 data from designed
		crossing experiments of outbred lines</p>
		<li><p style="margin-bottom: 0.14in">authors: Mikko J Sillanpaa
		(mail: mjs@rolf.helsinki.fi)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.rni.helsinki.fi/~mjs/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sillanpaa, Arjas
		(1999), &quot;Bayesian mapping of multiple quantitative trait loci
		from incomplete outbred offspring data&quot;, Genetics,
		151:1605-1619.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MULTIPOPTAGSELECT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		selects a near-minimal set of tagging single-nucleotide
		polymorphisms (tagSNPs) that account for all observed patterns of
		linkage disequilibrium (LD) in multiple populations.</p>
		<li><p style="margin-bottom: 0.14in">authors: Bryan N. Howie,
		Christopher S. Carlson, Mark J. Rieder, and Deborah A. Nickerson
		(University of Washington)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://droog.gs.washington.edu/multiPopTagSelect.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:
		multiPopTagSelect.pl</p>
		<li><p style="margin-bottom: 0.14in">reference: Howie, Carlson,
		Rieder, Nickerson (2006), &quot;Efficient selection of tagging
		single-nucleotide polymorphisms in multiple populations&quot;,
		Human Genetics, 120(1):58-68.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MULTIQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.6</p>
		<li><p style="margin-bottom: 0.14in">description: MultiQTL software
		integrates a broad spectrum of data mining, statistical analysis,
		interactive visualization and modeling tools that allow QTL
		analysis based on advanced and sophisticated methods for maximum
		extraction of the mapping information from data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Efim Ronin, David
		Mester, Dina Minkov, Ori Orion, Abraham Korol</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.multiqtl.com/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">availability: contact:
		sales@multiqtl.com. tel: 972-4824-0449. FAX: 972-4824-6554.
		address: MultiQTL Ltd, Institute of Evolution, Haifa University,
		Haifa, Israel 31905</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MULTISIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: analyze the
		numbers of individuals that founded new populations following a
		bottleneck or founding event</p>
		<li><p style="margin-bottom: 0.14in">authors: Mohamed Noor (email:
		noor@duke.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biology.duke.edu/noorlab/multsim.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Noor, Pascual,
		Smith (2000), &quot;Genetic variation in the spread of Drosophila
		subobscura from a nonequilibrium population&quot;, Evolution,
		54:696-703.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">MUTAGENESYS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: MutaGeneSys is a
		system that uses genome-wide genotype data to estimate disease
		susceptibility. Our system integrates three data sources: HapMap,
		whole-genome marker correlation data, and OMIM database. It accepts
		SNP data of individuals as query input and delivers disease
		susceptibility hypotheses even if the original set of typed SNPs is
		incomplete.</p>
		<li><p style="margin-bottom: 0.14in">authors: Julia Stoyanovich,
		Itsik Pe'er (Columbia University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://magnet.c2b2.columbia.edu/mutagenesys/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Stoyanovich, Pe'er
		(2008), &quot;MutaGeneSys: estimating individual disease
		susceptibility based on genome-wide SNP array data&quot;,
		Bioinformatics, 24(3):440-442.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">NOCOM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Aug 1992</p>
		<li><p style="margin-bottom: 0.14in">description: estimate
		parameters for mixture of normal distributions</p>
		<li><p style="margin-bottom: 0.14in">Authors: Jurg Ott (Rockefeller
		University)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS, OS2</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ott (1979),
		&quot;Detection of rare major genes in lipid levels&quot;, Human
		Genetics, 51:79-91.</p>
		<li><p style="margin-bottom: 0.14in">online user's guide</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">NOPAR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Aug 1995</p>
		<li><p style="margin-bottom: 0.14in">description: non-parametric
		linkage and association tests primarily for a quantitative trait</p>
		<li><p style="margin-bottom: 0.14in">authors: A Collins (email:
		arc@southampton.ac.uk), NE Morton. (University of Southampton, UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/nopar</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX(SunOS)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Collins,Morton
		(1995), Human Heredity, 45:311-318.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">NUCULAR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: A program that
		splits extended pedigrees into nuclear families, with the option of
		recoding all sibs as half-sibs with distinct mothers of fathers.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mathieu Lemire
		(email: mlemire@sdf.lonestar.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mlemire.freeshell.org/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lemire (2005), &quot;A
		simple nonparametric multipoint procedure to test for linkage
		through mothers or fathers as well as imprinting effects in the
		presence of linkage&quot;, BMC Genetics, 6(Suppl 1):S159.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ONEMAP (see R/ONEMAP)</p>
	<li><p style="margin-bottom: 0.14in">OLORIN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: October 2011</p>
		<li><p style="margin-bottom: 0.14in">description: Olorin is an
		interactive filtering tool for next generation sequencing data
		coming from the study of large complex disease pedigrees. Olorin,
		is a tool which integrates gene flow output from Merlin and next
		generation sequencing data. Users can interactively filter and
		prioritize variants based on haplotype sharing across different
		sets of selected individuals and allele frequency in reference
		datasets.</p>
		<li><p style="margin-bottom: 0.14in">authors: James Morris, Jeffrey
		Barrett (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/olorin/</p>
		<li><p style="margin-bottom: 0.14in">http://sourceforge.net/p/olorin/home/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: any
		platform with Java 1.6 or later</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">OSA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Ordered Subset
		Analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1</p>
		<li><p style="margin-bottom: 0.14in">description: OSA allows the
		researcher to evaluate evidence for linkage even when heterogeneity
		is present in a data set. This is not an unusual occurrence when
		studying diseases of complex origin. Families are ranked by
		covariate values in order to test evidence for linkage among
		homogeneous subsets of families. Because families are ranked, a
		priori covariate cutpoints are not necessary. Covariates may
		include linkage evidence at other genes, environmental exposures,
		or biological trait values such as cholesterol, age at onset, and
		so on.</p>
		<li><p style="margin-bottom: 0.14in">authors: William Duren, Mike
		Boehnke, Elizabeth Hauser</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wwwchg.duhs.duke.edu/software/osa.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: osa2.0</p>
		<li><p style="margin-bottom: 0.14in">reference: ER Hauser, MP Bass,
		ER Martin, RM Watanabe, WL Duren, M Boehnke (2001), Am J Hum Genet,
		suppl 69:529.</p>
		<li><p style="margin-bottom: 0.14in">ER Hauser, RM Watanabe, WL
		Duren, MP Bass, CD Langefeld, M Boehnke (2004), &quot;Ordered
		subset analysis in genetic linkage mapping of complex traits&quot;,
		Genetic Epidemiology, 27(1):53-63.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">OSIRIS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: OSIRIS is a tool
		for the retrieval of articles from MEDLINE related to the sequence
		variants reported for a human gene. The variations considered are
		single nucleotide polymorphisms (SNPs), insertion/deletion
		polymorphisms (indel), microsatellite, and named variations (e.g.
		Alu sequences).</p>
		<li><p style="margin-bottom: 0.14in">authors: Julio Bonis , Laura
		Ines Furlong Ferran Sanz (University of Pompeu Fabra, Spain)
		(email: fsanz@imim.es)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://ibi.imim.es/osirisform.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Bonis, Furlong,
		Sanz (2006), &quot;OSIRIS: a tool for retrieving literature about
		sequence variants&quot;, Bioinformatics, 22(20):2567-2569.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">P_ACT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: P-values:
		Adjustment for Correlated Tests</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2</p>
		<li><p style="margin-bottom: 0.14in">description: P_ACT is an R
		program that adjusts sets of up to 1000 p-values from association
		tests between correlated traits and SNPs for multiple testing,
		accounting for the correlation between tests.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://csg.sph.umich.edu/boehnke/p_act.php</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Conneely, Boehnke
		(2007), &quot;So many correlated tests, so little time! Rapid
		adjustment of p-values for multiple correlated tests&quot;,
		American Journal of Human Genetics, 81(6):1158-1168</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PANGAEA (see also BOREL,
	ECLIPSE, HARDY, INSEGT, LOKI, MCLEEPS, MORGAN, PEDFIDDLER, PEDPACK)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pedigree ANalysis
		for Genetics (And Epidemiological Attributes)</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Collection of
		nine packages for genetic analysis;</p>
		<li><p style="margin-bottom: 0.14in">the original four BOREL,
		HARDY, MORGAN and PEDPACK, plus INSEGT, LOKI, MCLEEPS, PEDFIDDLER,
		and ECLIPSE</p>
		<li><p style="margin-bottom: 0.14in">Authors: many authors: see
		individual packages. managed by EA Thompson (University of
		Washington, email: thompson@stat.washington.edu).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C or C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(Compaq, Solaris, and others), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pedigree Analysis
		Package</p>
		<li><p style="margin-bottom: 0.14in">version: 6.0 (December 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: it can perform
		(1) compute the likelihood of specified parameter values; (2)
		compute the probability of each genotype for pedigree members; (3)
		simulate phenotypes for output into files; (4) maximize the
		likelihood over specified parameters (with or without standard
		errors); (5) compute the standard errors of parameters for unknown
		estimates; (6) simulate phenotypes and estimate parameter values;
		(7) estimate expected lod score; (8) compute a grid of likelihood
		over one or two parameters. New additions to V5: assortative
		mating; TDT; additive multi-locus models</p>
		<li><p style="margin-bottom: 0.14in">authors: Sandra Hasstedt
		(University of Utah, email: sandy@genetics.utah.edu)</p>
		<li><p style="margin-bottom: 0.14in">Peter Cartwright contributed
		to versions 2 and 3; many others contributed to version 1
		(University of Utah)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hasstedt.genetics.utah.edu/</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN
		77</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: preped, descstat,
		prepap,simul, papdr (with 8 options),gpe</p>
		<li><p style="margin-bottom: 0.14in">reference: manual (version 5,
		PDF) online manual (version 4) (from Wellcome Trust)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PARENTE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (February 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: parentage
		inference using molecular data from diploid codominant markers</p>
		<li><p style="margin-bottom: 0.14in">authors: Stephanie Blanc-Manel
		(email: stephanie.manel@ujf-grenoble.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www2.ujf-grenoble.fr/leca/membres/manel.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Cercueil,
		Bellemain, Manel (2002), &quot;PARENTE: computer program for
		parentage analysis&quot;, Journal of Heredity, 93:458-459.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PASS PEDIGREE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: PASS Pedigree is
		drawing the most complex family trees in a matter of minutes
		instead of hours of work. The basis of this is an algorithm for
		automatically builing a family tree. Of course, manual adjustments
		in the family tree can be made for your specific requirements. PASS
		Pedigree meets all international conventions concerning the drawing
		of pedigrees. A converter can convert historical Cyrillic pedigrees
		automatically to PASS Pedigree. Unlike before, all your family
		trees are stored in one single database. PASS Pedigree can
		intelligently connect to many genetic centers (e.g. three genetic
		centers in the Netherlands) with the existing patient information,
		via the lab system HELIX based on HL7 techniques.</p>
		<li><p style="margin-bottom: 0.14in">authors: PASS (Pro Active
		Software Solutions) Software</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.pass-software.com/Producten/PASSPedigree.aspx (in Dutch)</p>
		<li><p style="margin-bottom: 0.14in">http://www.pass-software.com/Producten/PASSPedigree/tabid/113/language/en-US/Default.aspx
		(in English)</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">availability: price</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PATH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.7 (September 28,
		2009)</p>
		<li><p style="margin-bottom: 0.14in">description: The Path web
		application is a first-step bioinformatics approach to investigate
		gene-gene interactions in genetic association studies. Path is
		designed to do: 1. Interface your SNP data with biological
		information from several online bioinformatics databases. 2.
		Generate biologically plausible hypotheses for testing gene-gene
		interactions. 3. Select a subset of SNPs and conduct SNP-SNP
		interaction tests. 4. Store analysis results. 5. Explore analysis
		results through interactive plots and summary tables.</p>
		<li><p style="margin-bottom: 0.14in">authors: email:
		genapha@mrl.ubc.ca</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genapha.icapture.ubc.ca/PathTutorial/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zamer, Tripp,
		Ellis, Daley (2009), &quot;Path: a tool to facilitate pathway-based
		genetic association analysis&quot;, Bioinformatics,
		25(18):2444-2446.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PAWE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Power for
		Association With Error</p>
		<li><p style="margin-bottom: 0.14in">version: Dec 2002</p>
		<li><p style="margin-bottom: 0.14in">description: Power and sample
		size calculations for genetic case-control association studies
		allowing for errors</p>
		<li><p style="margin-bottom: 0.14in">authors: Derek Gordon (email:
		gordon@linkage.rockefeller.edu), assisted by Michael Nothnagel</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://linkage.rockefeller.edu/pawe/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gordon, Finch,
		Nothnagel, Ott (2002), Human Heredity, 54:22-33.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PAWE-3D</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Power for
		Association With Error in 3D</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: D Gordon, C Haynes, J
		Blumenfeld, SJ Finch</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://linkage.rockefeller.edu/pawe3d/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gordon, Haynes,
		Blumenfeld, Finch (2005), &quot;PAWE-3D: visualizing Power for
		Association With Error in case/control genetic studies of complex
		traits&quot;, Bioinformatics, 21:3935-3937.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PBAT (see also FBAT)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Power calculation
		of family-Based Association Tests</p>
		<li><p style="margin-bottom: 0.14in">version: 3.6.1</p>
		<li><p style="margin-bottom: 0.14in">description: PBAT is an
		interactive software package that provides tools for the design and
		the data analysis of family-based association studies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Christoph Lange
		(email: clange@hsph.harvard.edu) (Harvard Univ).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.harvard.edu/~clange/default.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lange, DeMeo,
		Silverman, Weiss, Laird (2003), &quot;Using the noninformative
		families in family-based association tests: A powerful new testing
		strategy&quot;, American Journal of Human Genetics, 73(4):801-811</p>
		<li><p style="margin-bottom: 0.14in">Lange, DeMeo, Silverman,
		Weiss, Laird (2004), &quot;PBAT: tools for family-based association
		studies&quot;, American Journal of Human Genetics, 74(2):367-369</p>
		<li><p style="margin-bottom: 0.14in">Steen, Lange (2005), &quot;PBAT:
		A comprehensive software package for genome-wide association
		analysis of complex family-based studies&quot;, Human Genomics,
		2(1):67-69.</p>
		<li><p style="margin-bottom: 0.14in">Hoffmann, Lange (2006),
		&quot;P2BAT: a massive parallel implementation of PBAT for
		genome-wide association studies in R&quot;, Bioinformatics,
		22:3103-3105.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PDA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pooled DNA Analyzer</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a tool for
		analyses of pooled DNA data</p>
		<li><p style="margin-bottom: 0.14in">authors: Hsin-Chou Yang,
		Chia-Ching Pan, Chin-Yu Lin and Cathy SJ Fann (Institute of
		Biomedical Sciences, Academia Sinica, Taiwan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ibms.sinica.edu.tw/~csjfann/first%20flow/programlist.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yang, Pan, Lin,
		Fann (2006), &quot;PDA: pooled DNA analyzer&quot;, BMC
		Bioinformatics, 7:233.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PDPSYS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pedigree Data
		Processing SYStem</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (November 1998)</p>
		<li><p style="margin-bottom: 0.14in">description: PDPSys is a
		Windows-based system designed for pedigree data management,
		providing a graphical interface for pedigree construction and
		output.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Fil Quiaoit (Fred
		Hutchinson Cancer Research Center, email: fquiaoit@fhcrc.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cougar.fhcrc.org/software.php</p>
		<li><p style="margin-bottom: 0.14in">source code language: Visual
		Basic, MS-Access</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95)</p>
		<li><p style="margin-bottom: 0.14in">executables: pdpsys</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: pedigree
		disequilibrium test</p>
		<li><p style="margin-bottom: 0.14in">version: 5.1</p>
		<li><p style="margin-bottom: 0.14in">description: The PDT analysis
		program allows the user to evaluate evidence of linkage
		disequilibrium (LD) in general pedigree data. All family data may
		be used without nullifying the validity of the association test,
		even when there is more than one affected in a family. The PDT
		program performs both allele-specific and genotype-specific LD
		analysis of individual markers. Version 5.1 adds the ability to
		perform genotype-specific analysis over marker sets.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Eden Martin (Duke
		Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.chg.duke.edu/software/pdt.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Martin, Monks,
		Warren, Kaplan (2000), American Journal of Human Genetics,
		67(1):146-154</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PED</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: PEdigree Drawing
		software</p>
		<li><p style="margin-bottom: 0.14in">version: 5.1.0 (October 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Powerful pedigree
		drawing program with two drawing modes: Input mode for fast and
		automatic drawing; edit mode for text annotations, legends, special
		symbols. Both input and edit mode provide auto numbering, auto
		resizing of symbols and fonts, and zooming. PED 4.2a complies with
		the &quot;Recommendations for standardized human pedigree
		nomenclature&quot;. Apart from fully sizeable printed output,
		pedigrees can be exported as metafiles to virtually any Windows
		word processor or drawing program.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Hansjoerg Plendl
		(email: plendl@medgen.uni-kiel.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.medgen.de/index.html</p>
		<li><p style="margin-bottom: 0.14in">http://www.medgen.de/ped/index.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Visual
		Smalltalk</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables: ped.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Bennet, Steinhaus,
		Uhrich, O'Sullivan, Resta, Lochner-Doyle, Markel, Vincent,
		Hamanishi(1995), &quot;Recommendations for standardized human
		pedigree nomenclature. pedigree standardization task force of the
		national society of genetic counselors&quot;, American Journal of
		Human Genetics, 56(3):745-752</p>
		<li><p style="margin-bottom: 0.14in">A Golla (1997), Medizinische
		Genetik, 4:595-597.</p>
		<li><p style="margin-bottom: 0.14in">H Plendl (1998), Medizinische
		Genetik, 1:50-51.</p>
		<li><p style="margin-bottom: 0.14in">price: site license US$99.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDAGREE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: 1.00 (February
		2002)</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a program for
		detecting autosomal marker Mendelian incompatibilities in pedigree
		data</p>
		<li><p style="margin-bottom: 0.14in">authors: Christine Couillault
		(email: kristin.c@free.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pedagree.free.fr/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C, C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		Linux, MS-Windows, MS-DOS,</p>
		<li><p style="margin-bottom: 0.14in">executables: pedagree</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDCHECK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (November 1998)</p>
		<li><p style="margin-bottom: 0.14in">description: identifying all
		Mendelian inconsistencies in pedigree data.</p>
		<li><p style="margin-bottom: 0.14in">Authors: J. R. O'Connell
		(email: jeff@sherlock.hgen.pitt.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register</p>
		<li><p style="margin-bottom: 0.14in">source code language: C, C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS/Solaris/Compaq-Alpha/SGI-IRIX/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1997 (O'Connell, Weeks, Am. J. Hum. Genet. suppl,61,
		A288 (1997)); O'Connell, Weeks (1998), &quot;PedCheck: a program
		for identification of genotype incompatibilities in linkage
		analysis&quot;, American Journal of Human Genetics, 63(1):259-266</p>
		<li><p style="margin-bottom: 0.14in">FAQ-O-MATIC:
		http://watson.hgen.pitt.edu/fom-serve/cache/8.html</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDFIDDLER (see also PEDPACK in
	PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.5 (August 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: This suite of six
		programs can be used as a stand-alone extension of the pedigree
		drawing facilities found in the publicly available version of
		PEDPACK.</p>
		<li><p style="margin-bottom: 0.14in">Authors: J C. Loredo-Osti
		(email: josti@bagel.epi.mcgill.ca), Kenneth Morgan (email:
		ken@bagel.epi.mcgill.ca); Window version adapted by Michael Li
		(email:lina@u.washington.edu).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/Pedfiddler.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX (OSF1,
		and Solaris 2.7, AlphaLinux) MS-Windows (XP Home/Win32/Win95),
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDHUNTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a software
		package that facilitates creation and verification of pedigrees
		within large genealogies.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Agarwala, Biesecker,
		Hopkins, Francomano, Schffer (NIH)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/pedhunter.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Agarwala,
		Biesecker, Hopkins, Francomano, Schffer (1998), &quot;Software for
		constructing and verifying pedigrees within large genealogies and
		an application to the Old Order Amish of Lancaster county&quot;,
		Genome Research, 8:211-221.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDIGRAPH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.3</p>
		<li><p style="margin-bottom: 0.14in">description: A pedigree
		visualization program specifically designed to draw large, complex
		pedigrees.</p>
		<li><p style="margin-bottom: 0.14in">authors: John Garbe
		(University of Minnesota, email: jgarbe@umn.edu), Yang Da
		(University of Minnesota, email: yda@umn.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://animalgene.umn.edu/pedigraph/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: pedigraph</p>
		<li><p style="margin-bottom: 0.14in">reference: J Garbe, Y Da
		(2003), Pedigraph user manual Version 1.1. Department of Animal
		Science, University of Minnesota.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDIGREE/DRAW</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 6</p>
		<li><p style="margin-bottom: 0.14in">description: pedigree drawing</p>
		<li><p style="margin-bottom: 0.14in">authors: Bennett Dyke (email:
		bdyke@darwin.sfbr.org), Paul Mamelka (contact person, email:
		paul@darwin.sfbr.org), J. MacCleur, Southwest Foundation for
		Biomedical Research</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.pedigree-draw.com/</p>
		<li><p style="margin-bottom: 0.14in">ftp://www.sfbr.org/pub/pedmgt/peddrw</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:PedDraw</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1987 (Dyke, Mamelka, Am J. Hum Genet, suppl 41, A253
		(1987))</p>
		<li><p style="margin-bottom: 0.14in">price: disk and user's guide:
		US$45; user's guide: US$20; upgrade for disk: US$15;</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDIGREEQUERY</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: allows drawing
		pedigrees with a difficult structure, those containing
		consanguinity loops, and those individuals with multiple mates or
		several related families</p>
		<li><p style="margin-bottom: 0.14in">authors: A Kirichenko</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mga.bionet.nsc.ru/soft/index.html</p>
		<li><p style="margin-bottom: 0.14in">http://mga.bionet.nsc.ru/PedigreeQuery/PedigreeQuery.html
		(in Russian)</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: A Kirichenko
		(2003), &quot;PedigreeQuery - software for pedigree drawing&quot;
		(abstract), American Journal of Human Genetics, suppl</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDIGREE-VISUALIZER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Limsoon Wong
		(National University of Singapore, Singapore)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://research.i2r.a-star.edu.sg:8080/kleisli/demos/pedigree/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDJAVA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: PedJava</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: Allows pedigree
		entry and retrieval from an internet browser into a distant MS
		ACCESS database. Includes IP access restriction, automatic
		numbering of families and individuals and database consistency
		checks.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Matthias Wjst (email:
		wjst@gsf.de), Juergen Knauth (email: woodoo@bigfoot.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cooke.gsf.de/wjst/download.cfm</p>
		<li><p style="margin-bottom: 0.14in">source code language: JDK</p>
		<li><p style="margin-bottom: 0.14in">operating systems: no
		restriction for client, server WIN 95/NT</p>
		<li><p style="margin-bottom: 0.14in">executables: pedJava,
		pedJavaServer</p>
		<li><p style="margin-bottom: 0.14in">reference: in preparation</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDMANAGER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">alias: MAPMAKER/PEDMANAGER</p>
		<li><p style="margin-bottom: 0.14in">version: 0.9 (1997)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Mary Pat Daly
		(previously Mary Pat Reeve) (MIT/Whitehead)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-genome.wi.mit.edu/ftp/distribution/software/pedmanager/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: pedprep</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDPACK (see also PANGAEA)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Version 2.2, July
		1996</p>
		<li><p style="margin-bottom: 0.14in">description: Programs for
		pedigree analysis, including segregation analysis, gene extinction,
		and pedigree graphics.</p>
		<li><p style="margin-bottom: 0.14in">authors: A.W. Thomas (Versions
		1.0, 3.0, 3.1)</p>
		<li><p style="margin-bottom: 0.14in">E.A. Thompson (University of
		Washington, email: thompson@stat.washington.edu)</p>
		<li><p style="margin-bottom: 0.14in">C.J. Geyer (University of
		Minnesota, email: charlie@stat.umn.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/thompson/Genepi/pangaea.shtml</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Compaq-Alpha/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Thomas, IMA J. Math
		Appl Med Biol, 5, 210-213 (1988)</p>
		<li><p style="margin-bottom: 0.14in">Thomas, &quot;Pedpack: Users'
		Manual&quot;, Technical report No. 99, Dept Statistics, University
		of Washington.</p>
		<li><p style="margin-bottom: 0.14in">Thomas, &quot;Pedpack:
		Managers' Manual&quot;, Technical report No. 100, Dept Statistics,
		University of Washington.</p>
		<li><p style="margin-bottom: 0.14in">Geyer, (1988), &quot;Software
		for calculating gene survival and multigene descent probabilities,
		and for pedigree manipulation and drawing&quot;, Technical report
		No. 153, Dept Statistics, University of Washington.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDPEEL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		prepares pedigree data for calculation of Elston-Stewarts'
		likelihood function. It finds an optimal way to peel a pedigree and
		returns text file containing 7 description arrays</p>
		<li><p style="margin-bottom: 0.14in">authors: Nadezhda Belonogova
		(email: adg101@mail.ru), Tatiana I. Axenovich</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mga.bionet.nsc.ru/soft/pedpeel/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Belonogova,
		Axenovich (2007), &quot;Optimal peeling order for pedigrees with
		incomplete genotypic information&quot;, Computational Biology and
		Chemistry , 31(3):173-177.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDPHASE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: inferring
		haplotypes from genotypes on pedigree data</p>
		<li><p style="margin-bottom: 0.14in">authors: Jing Li(email:
		jingli@eecs.cwru.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://vorlon.case.edu/~jxl175/haplotyping.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Li, Jiang (2003),
		&quot;Effecient rule-based haplotyping algorithms for pedigree
		data&quot;, Proceedings of the Seventh International Conference on
		Research in Computational Molecular Biology , pp.197-206.</p>
		<li><p style="margin-bottom: 0.14in">Li, Jiang (2004), &quot; An
		exact solution for finding minimum recombinant haplotype
		configurations on pedigrees with missing data by integer linear
		programming&quot;, Proceedings of the Seventh International
		Conference on Research in Computational Molecular Biology ,
		pp.101-110.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDPLOT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Pedigree Plotting
		Program for the Pedfile Format</p>
		<li><p style="margin-bottom: 0.14in">Authors: JE Stajich (email:
		jason@chg.mc.duke.edu), C Haynes (email: carol@chgd.mc.duke.edu),
		MA Pericak-Vance (Duke University Medical Center)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.chg.duke.edu/software/pedplot.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++,
		Postscript</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Sparc-Solaris 2.5/DEC UNIX 4.0/x86-Solaris 2.6)</p>
		<li><p style="margin-bottom: 0.14in">executables: pedplot</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1998 (Stajich, Haynes, Pericak-Vance, Am J Hum
		Genet, suppl, 63,A242).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDRAW/WPEDRAW</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pedigree Drawing/
		Window Pedigree Drawing (MS-Window and X-Window version of PEDRAW)</p>
		<li><p style="margin-bottom: 0.14in">version: 2.5</p>
		<li><p style="margin-bottom: 0.14in">description: a pedigree
		drawing program using LINKAGE data files</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis (email:
		dcurtis@hgmp.mrc.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.ebi.ac.uk/pub/software/linkage_and_mapping/statgen/dcurtis/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		MS-Windows, X-window</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Curtis (1990),
		Annals of Human Genetics, 54:365-367.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDSCRIPT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2004 (not actively
		maintained)</p>
		<li><p style="margin-bottom: 0.14in">description: PedScript is a
		tool that allows scripting of simple modifications to pedigree
		files.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/PedScript</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: anonymize,
		mendelcheck, nuke, trim, sample</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDSTATS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.6.8</p>
		<li><p style="margin-bottom: 0.14in">description: PEDSTATS does
		error checking and data summary of large or small data sets in
		QTDT, LINKAGE or MENDEL format. Checks for basic formatting errors,
		disconnected family groups, ancestor-descendant loops and can
		detect all Mendelian (including X-linked) inheritance errors in any
		pedigree without loops. Produces text and graphical (PDF) summaries
		of the family structure, trait and marker information of pedigree
		data and can break down summaries by sex, relative pair type or
		family. PEDSTATS also does Hardy-Weinberg testing using either a
		fast exact or asymptotic test and can summarize information in text
		or graphical PDF format. Additional features include a number of
		options for filtering data prior to summary and checks for
		inappropriate age or covariate values. Lastly, PEDSTATS can
		identify and trim uninformative individuals from a pedigree and
		rewrite the reorganized data to a new pedigree file.</p>
		<li><p style="margin-bottom: 0.14in">authors: Janis Wigginton
		(email: wiggie@umich.edu), Goncalo Abecasis (email:
		goncalo@umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/Pedstats/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		UNIX, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: pedstats</p>
		<li><p style="margin-bottom: 0.14in">reference: Wigginton, Abecasis
		(2005), &quot;PEDSTATS: descriptive statistics, graphics and
		quality assessment for gene mapping data&quot;, Bioinformatics,
		21:3445-3447.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDSYS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: a full-scale
		database system developed as a specialized tool for management of
		genetic, pedigree and demographic data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Bennett Dyke (email:
		bdyke@darwin.sfbr.org), Paul Mamelka (email: paul@darwin.sfbr.org),</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sfbr.org/sfbr/public/software/pedsys/pedsys.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://www.sfbr.org/pub/pedmgt/pedsys</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS,Solaris), MS-DOS, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
		<li><p style="margin-bottom: 0.14in">price: disks or tapes and
		user's manual: US$55; user's manual: US$30; disk for registered
		users: US$20. person to contact: Linda Freeman-Shade, Dept
		Genetics, Southwest Foundation for Biomedical Research, P.O. Box
		760549, San Antonio, TX 78245-0549 (email: linda@darwin.sfbr.org)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEDVIZAPI</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: PEDigree
		VIsualiZation API</p>
		<li><p style="margin-bottom: 0.14in">version: 0.15 (November 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: PEDVIZAPI is a
		Java Api for the visual analysis of large and complex pedigrees. It
		provides all the necessary functionality for the interactive
		exploration of extended genealogies. Available packages are mostly
		focused on a static representation or cannot be added to an
		existing application; the goal of this open-source library is to
		provide an application program interface that enables the efficient
		construction of visual analytic applications for the analysis of
		family based data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Christain Fuchsberger
		(email: christian.fuchsberger@eurac.edu), Mario Falchi, Lukas
		Forer, Peter Pramstaller</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.pedvizapi.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java, R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Fuchsberger,
		Falchi, Forer, Pramstaller (2007), &quot;PedVizApi: A Java API for
		the interactive, visual analysis of extended pedigrees&quot;,
		Bioinformatics, 24(2):279-281.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PEER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: PEER is a
		collection of Bayesian approaches to infer hidden determinants and
		their effects from gene expression profiles using factor analysis
		methods. It can be used to detect eQTL.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/peer/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Stegle, Parts,
		Durbin, Winn (2010), &quot;A Bayesian framework to account for
		complex non-genetic factors in gene expression levels greatly
		increases power in eQTL studies&quot;, PLoS Computational Biology,
		6:e1000770.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PELICAN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pedigree Editor for
		LInkage Computer ANalysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: It is a utility
		for graphically editing the pedigree data files used by programs
		such as FASTLINK, VITESSE, GENEHUNTER and MERLIN.</p>
		<li><p style="margin-bottom: 0.14in">authors: Frank Dudbridge (MRC,
		email: frank.dudbridge@mrc-bsu.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-bsu.cam.ac.uk/personal/frank/software/pelican/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: F Dudbridge, T
		Carver, GW Williams (2004), &quot;Pelican: pedigree editor for
		linkage computer analysis&quot;, Bioinformatics, 20(14):2327-2328.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PHASE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0 (Sept 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: This program
		implements a new statistical method for reconstructing haplotypes
		from population genotype data</p>
		<li><p style="margin-bottom: 0.14in">new for v2: (1) the
		introduction of a new computational approach, resulting in much
		faster haplotype resolution. (2) the introduction of a new model
		that allows for recombination and decay of Linkage Disequilibrium
		(LD) with distance, which results in more accurate haplotype
		estimates. This model also allows the user to estimate
		recombination rates, and identify recombination hotspots from
		population genotype data. (3) the facility to perform a test for
		haplotype frequency differences between cases and controls. (4)
		more extensive output summarising the results.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Matthew Stephens
		(email: stephens@stat.washington.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.washington.edu/stephens/software.html</p>
		<li><p style="margin-bottom: 0.14in">previous site:
		http://www.stats.ox.ac.uk/mathgen/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Stephens, Smith,
		Donnelly (2001), AJHG, 68:978-989.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PIAGE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Power of Indirect
		Association Studies of Gene-Environment Interactions</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (March 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: The program PIAGE
		performs estimation of power and sample sizes required to detect
		genetic and environmental main, as well as gene-environment
		interaction (GxE) effects in indirect matched case-control studies
		(1:1 matching). When the hypothesis of GxE is tested, power/sample
		size will be estimated for the detection of GxE, as well as for the
		detection of genetic and environmental marginal effects.
		Furthermore, power estimation is implemented for the joint test of
		genetic marginal and GxE effects (Kraft P et al., 2007). Power and
		sample size estimations are based on Gauderman's (2002) asymptotic
		approach for power and sample size estimations in direct studies of
		GxE. Hardy-Weinberg equilibrium and independence of genotypes and
		environmental exposures in the population are assumed. The
		estimates are based on genotypic codes (G=1 (G=0) for individuals
		who carry a (non-) risk genotype), which depend on the mode of
		inheritance (dominant, recessive, or multiplicative). A conditional
		logistic regression approach is used, which employs a
		likelihood-ratio test with respect to a biallelic candidate SNP, a
		binary environmental factor (E=1 (E=0) in (un)exposed individuals),
		and the interaction between these components.</p>
		<li><p style="margin-bottom: 0.14in">authors: Rebecca Hein
		(email:r.hein@dkfz-heidelberg.de), Lars Beckmann
		(email:l.beckmann@dkfz-heidelberg.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dkfz.de/en/epidemiologie-krebserkrankungen/software/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Hein, Beckmann,
		Chang-Claude (2008), &quot;Sample size requirements for indirect
		association studies of gene-environment interactions (GxE)&quot;,
		Genetic Epidmiology, to appear.</p>
		<li><p style="margin-bottom: 0.14in">Gauderman (2002), &quot;Sample
		size requirements for matched case-control studies of
		gene-environment interaction&quot;, Statistics in Medicine,
		21:35-50.</p>
		<li><p style="margin-bottom: 0.14in">Kraft, Yen, Stram, Morrison,
		Gauderman (2007), &quot;Exploiting gene-environment interaction to
		detect genetic associations&quot;, Human Heredity, 63:1141-1119.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PLABQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: PLAnt Breeding QTL
		analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: HF Utz (email:
		utzf@uni-hohenheim.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		https://www.uni-hohenheim.de/plantbreeding/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Utz, Melchinger
		(1996), &quot;PLABQTL: A program for composite interval mapping of
		QTL&quot;, JAG</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PLABSIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: PLAnt Breeding
		SIMulation</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.1 (June 2005)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Matthias Frisch
		(email: frisch@uni-hohenheim.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uni-hohenheim.de/~frisch/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX(AIX),
		Linux, MS-Windows(NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Frisch, Bohn,
		Melchinger (2000), &quot;PLABSIM: software for simulation of
		marker-assisted backcrossing&quot;, Journal of Heredity, 91:86-87.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PLAYPUS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: an integrated
		variant caller</p>
		<li><p style="margin-bottom: 0.14in">authors: Gerton Lunter, Martin
		Goodson (Wellcome Trust Centre of Human Genetics, Oxford Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.well.ox.ac.uk/platypus</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PL-EM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Partition-Ligation
		EM algorithm for haplotype inference with single nucleotide
		polymorphisms</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jun Liu, Steve Qin,
		Tianhua Niu (Harvard Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.people.fas.harvard.edu/~junliu/plem/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Qin, Niu, Liu
		(2002), &quot;Partition-ligation-expectation-maximization algorithm
		for haplotype inference with single-nucleotide polymorphisms&quot;,
		American Journal of Human Genetics, (letter), 71(5):1242-1247-?</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PLINK (see also PLINK/SEQ)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.03</p>
		<li><p style="margin-bottom: 0.14in">description: a whole-genome
		association analysis toolset</p>
		<li><p style="margin-bottom: 0.14in">authors: Shaun Purcell</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pngu.mgh.harvard.edu/purcell/plink/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Purcell, Neale,
		Todd-Brown, Thomas, Ferreira, Bender, Maller, de Bakker, Daly, Sham
		(2007), &quot;PLINK: a toolset for whole-genome association and
		population-based linkage analysis&quot;, American Journal of Human
		Genetics, 81(3):559-575</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PLINK/SEQ (see also PLINK)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: PLINK/SEQ is an
		open-source C/C++ library for working with human genetic variation
		data. The specific focus is to provide a platform for analytic tool
		development for variation data from large-scale resequencing
		projects, particularly whole-exome and whole-genome studies.
		However, the library could in principle be applied to other types
		of genetic studies, including whole-genome association studies of
		common SNPs.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://atgu.mgh.harvard.edu/plinkseq/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C/C++, R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MacOS,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POINTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: for complex
		segregation analysis with the mixed model (major locus and
		polygenes).</p>
		<li><p style="margin-bottom: 0.14in">authors: NE Morton (University
		of Southampton, UK), D.C. Rao, J-M. Lalouel</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cedar.genetics.soton.ac.uk/pub/PROGRAMS/pointer</p>
		<li><p style="margin-bottom: 0.14in">source code language: SUN
		FORTRAN (especially, the command &quot;fsplit&quot; is needed)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(SunOS/..)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Morton, Rao,
		Lalouel, &quot;Methods in Genetic Epidemiology&quot; (Karger,
		1983).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POLYMORPHISM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.4 (June 1999)</p>
		<li><p style="margin-bottom: 0.14in">description: calculating the
		heterozygosity, PIC, and LIC values for polymorphic markers</p>
		<li><p style="margin-bottom: 0.14in">Authors: Tianhua Niu, Sc.D.
		(Harvard Medical School, email: tniu@hohp.harvard.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: powerlic</p>
		<li><p style="margin-bottom: 0.14in">reference: Guo, Elston (1999),
		&quot;Linkage information content of polymorphic genetic markers&quot;,
		Human Heredity, 49:112-118.</p>
		<li><p style="margin-bottom: 0.14in">Niu, Struk, Lindpaintner
		(2001), Human Heredity, 52(2):102-109.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POLYMUTT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: POLYmorphism and de
		novo MUTaTion call in families with sequencing data</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		polymutt implemented a likelihood-based framework for calling
		single nucleotide variants and detecting de novo point mutation
		events in families for next-generation sequencing data. The program
		takes as input genotype likelihood format (GLF) files which can be
		generated following the Creation of GLF files instruction and
		outputs the result in the [VCF] format. The variant calling and de
		novo mutation detection are modelled jointly within families and
		can handle both nuclear and extended pedigrees without
		consanguinity loops. The input is a set of GLF files for each of
		family members and the relationships are specified through the .ped
		file.</p>
		<li><p style="margin-bottom: 0.14in">authors: B Li, G Abecasis
		(Univ Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/Polymutt:_a_tool_for_calling_polymorphism_and_de_novo_mutations_in_families_for_sequencing_data</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POLYPHEN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: POLYmorphism
		PHENotyping</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: PolyPhen-2 is a
		tool which predicts possible impact of an amino acid substitution
		on the structure and function of a human protein using
		straightforward physical and comparative considerations. It can be
		used in prioritize functional SNPs in a genetic study.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genetics.bwh.harvard.edu/pph2/ (POLYPHEN-2)</p>
		<li><p style="margin-bottom: 0.14in">http://genetics.bwh.harvard.edu/pph/
		(POLYPHEN-1)</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sunyaev,
		Eisenhaber, Rodchenkov, Eisenhaber, Tumanyan, Kuznetsov (1999),
		&quot;PSIC: profile extraction from sequence alignments with
		position-specific counts of independent observations&quot;, Protein
		Engineering, 12:387-394.</p>
		<li><p style="margin-bottom: 0.14in">Ramensky, Bork, Sunyaev
		(2002), &quot;Human non-synonymous SNPs: server and survey&quot;,
		Nucleic Acids Research, 30:3894-3900.</p>
		<li><p style="margin-bottom: 0.14in">Adzhubei, Schmidt, Peshkin,
		Ramensky, Gerasimova, Bork, Kondrashov, Sunyaev (2010), &quot;A
		method and server for predicting damaging missense mutations&quot;,
		Nature Methods, 7:248-249.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POOLSCORE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a program for
		analysis of case-control genetic association studies using allele
		frequency measurements on DNA pools</p>
		<li><p style="margin-bottom: 0.14in">authors: Paul McKeiger (email:
		paul.mckeigue@ucd.ie) (University College Dublin, Ireland)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.homepages.ed.ac.uk/pmckeigu/pooling/poolscore.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: poolscore.R</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POOL_STR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: POOLing alleles of
		Short Tandem Repeat markers</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Yurii Aulchenko, Aida
		Bertoli-Avella, Cornelia van Duijn (Erasmus University Medical
		School, The Netherlands)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.geneticepi.com/Research/software/software.html</p>
		<li><p style="margin-bottom: 0.14in">http://www.geneticepi.com/Research/software/pool_str-1.1.tar.gz</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Aulchenko,
		Beroli-Avella, van Duijn (2005), &quot;A method for pooling alleles
		from different genotyping experiments&quot;, Annals of Human
		Genetics, 69(2):233-238.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POPDIST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1.2</p>
		<li><p style="margin-bottom: 0.14in">description: POPDIST
		calculates a number of different genetic identities, phylogeny
		reconstructing measures, and distance reconstructing measures</p>
		<li><p style="margin-bottom: 0.14in">authors: Bernt Guldbrandtsen
		(email: bg@genetics.agrsci.dk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genetics.agrsci.dk/~bg/popgen/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX
		(AIX/IRIX/DECalpha), Linux, MacOS, MS-DOS, MS-Windows(95)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Guldbrandtsen,
		Tokiuk, Loeschcke (2000), &quot;POPDIST, version 1.1.1: a program
		to calculate population genetic distance and identity measures&quot;,
		Journal of Heredity, 91:178-179.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POPGEN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: POPGEN is an R
		package that specifically focuses on statistical and population
		genetics methods. The motivation behind the package is to produce
		an easy to use interface to many of the commonly used methods and
		models used in statistical and population genetics and an
		alternative interface for some of the methodology produced by our
		group.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jonathan Marchini
		(email: marchini@stats.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/mathgen/software.html</p>
		<li><p style="margin-bottom: 0.14in">http://www.stats.ox.ac.uk/~marchini/software.html</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/popgen/index.html
		(removed from CRAN)</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: popdiv, ps, nps,
		treesim, simMD</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POWER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0</p>
		<li><p style="margin-bottom: 0.14in">description: windows-based
		program for computation of sample size and power for binary outcome
		studies (case-control and cohort studies) based on a logistic-like
		regression model with one covariate or two covariates (e.g.,
		gene-exposure interactions).</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://dceg.cancer.gov/POWER/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Window</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Garcia-Closas,
		Lubin (1999), &quot;Power and sample size calculations in
		case-control studies of gene-environmental interactions: Comments
		on different approaches&quot;, American Journal of Epidemiology,
		149:689-693.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POWERMARKER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.25 (February 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: PowerMarker is a
		comprehensive set of statistical methods for genetic marker data
		analysis, designed especially for SSR/SNP data analysis.
		PowerMarker builds a powerful user interface around both new and
		traditional statistical methods for population genetic analysis.
		See analysis to check out the versatility of PowerMarker.
		PowerMarker is also a 2D Viewer - which was used intensively for
		visualizing linkage disequilibria results.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jack Liu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://statgen.ncsu.edu/powermarker/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Liu, Muse (2005),
		&quot;PowerMarker: an integrated analysis environment for genetic
		marker analysis&quot;, Bioinformatics, 21(9):2128-2129.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POWERTRIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: automate the
		decision to remove objects from a pedigree with a minimum loss
		information</p>
		<li><p style="margin-bottom: 0.14in">authors: Tricia Thornton,
		Jonathan Haines (Venderbilt University Medical Center)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://phg.mc.vanderbilt.edu/content/powertrim</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Thornton, Haines
		(2003), &quot;PowerTrim: an automated decision support algorithm
		for preprocessing family-based genetic data&quot;, American Journal
		of Human Genetics, 72(5):1280-1281</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POWQ</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: A user-friendly,
		graphical package for power evaluation and enhancement planning
		through variance component linkage analysis in a multipoint
		framework.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mario Falchi (email:
		mario.falchi@kcl.ac.uk) Cesare Cappio Borlino</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.twin-research.ac.uk/WebPowQ/PowQ.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Falchi and Borlino
		(2006), &quot;PowQ: a user-friendly package for the design of
		variance component multipoint linkage analysis studies&quot;,
		Bioinformatics, 22(11):1404-1405.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">POWTEST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Excel spreadsheet
		to calculate power of affected sibpairs and TDT analyses</p>
		<li><p style="margin-bottom: 0.14in">authors: Dave Curtis</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mds.qmw.ac.uk/statgen/dcurtis/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PREPLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: MAKE PEDfiles</p>
		<li><p style="margin-bottom: 0.14in">alias: LINKAGE/PREPLINK</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: part of the
		LINKAGE auxiliary program</p>
		<li><p style="margin-bottom: 0.14in">authors: Peter Cartwright
		(University of Utah, now Cimarron Software pc@cimsoft.com)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs or</p>
		<li><p style="margin-bottom: 0.14in">source code language: C and
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS, OS2</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PREST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pedigree
		RElationship Statistical Test</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0.2 (February 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: PREST is a
		program that detects pedigree errors in general outbred pedigrees
		by use of genome-screen data. When a potential pedigree error is
		detected, our companion program, ALTERTEST, determines which
		relationships are compatible with the observed genotype data. Both
		programs are freely available on the web.</p>
		<li><p style="margin-bottom: 0.14in">authors: Lei Sun, Kenneth
		Wilder, Mary Sara McPeek (University of Chicago)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://fisher.utstat.toronto.edu/sun/Software/Prest or previously,
		http://galton.uchicago.edu/~mcpeek/software/prest</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		UNIX(Solaris/..)</p>
		<li><p style="margin-bottom: 0.14in">executables: prest.solaris,
		prest.linux, altertest.solaris, altertest.linux</p>
		<li><p style="margin-bottom: 0.14in">reference: McPeek, Sun (2000),
		&quot;Statistical tests for detection of misspecified relationships
		by use of genome-screen data&quot;, American Journal of Human
		Genetics, 66(3):1076-1094</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PRESTO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.1 (October 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: PRESTO performs
		permutation testing and computes empirical distributions of order
		statistics for one and two stage association studies with
		stratified or unstratified data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Brian L Browning
		(email: b.browning@auckland.ac.nz)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://faculty.washington.edu/browning/presto/presto.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX (Solaris/..), Linux, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Browning (2008),
		&quot;PRESTO: Rapid calculation of order statistic distributions
		and multiple-testing adjusted P-values via permutation for one and
		two-stage genetic association studies&quot;, BMC Bioinformatics,
		9:309.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PROBMAX</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: assigning unknown
		parentage in pedigree analysis from known genotypic pools of
		parents and progeny</p>
		<li><p style="margin-bottom: 0.14in">authors: Roy Danzmann (email:
		rdanzman@uoguelph.ca)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uoguelph.ca/~rdanzman/software/PROBMAX/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Danzmann (1997),
		&quot;PROBMAX: a computer program for assigning unknown parentage
		in pedigree analysis from known genotypic pools of parents and
		progeny&quot;, Journal of Heredity, 88:333.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PROC QTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (January 2009)</p>
		<li><p style="margin-bottom: 0.14in">description: PROC QTL is a
		user defined SAS procedure for mapping quantitative trait loci
		(QTL). Since this procedure is not a built-in SAS procedure, users
		need to obtain a copy of the executable file of PROC QTL and
		install the software in their personal computers before PROC QTL
		can be executed. Of course, users need a regular SAS license prior
		to the installation of PROC QTL. Once PROC QTL is installed, it can
		be called just like any other SAS procedures. Users will not notice
		the differences between this customized procedure and other
		built-in SAS procedures.</p>
		<li><p style="margin-bottom: 0.14in">authors: Zhiqiu Hu (email:
		zhiqiu.hu@ucr.edu), Shizhong Xu (shizhong.xu@ucr.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.statgen.ucr.edu/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Hu and Xu (2009),
		&quot;PROC QTL - A SAS procedure for mapping Quantitative Trait
		Loci&quot;, Plant and Animal Genomes XVII Conference. January
		10-14, 2009, San Diego, California
		(http://www.intl-pag.org/17/abstracts/C02_PAGXVII_854.html).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PROFILER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: A flexible tool
		to generate the probability distribution of joint multilocus
		genotypes defined by sets of individuals within the pedigree and
		sets of markers within the framework map.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jeff O'Connell</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.molecular-haplotype.org/profiler/profiler_intro.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PROGENY</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Progeny Software,
		LLC</p>
		<li><p style="margin-bottom: 0.14in">version: 6.8 (December 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Fully
		customizable, multi-user relational database with an integrated
		pedigree drawing component to manage genetic and pedigree data in
		one database. Manage Pedigrees, Individuals, SNPs, STRs, Samples,
		Plates, Genotypes and exports to multiple analysis platforms.</p>
		<li><p style="margin-bottom: 0.14in">authors: David Deram (Progeny
		Software, LLC)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.progenygenetics.com/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++ and
		Active X Controls</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/2000/ME/NT/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PRT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Pedigree
		Reconstruction Tools</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2</p>
		<li><p style="margin-bottom: 0.14in">description: Partition of
		single generation into sibling groups</p>
		<li><p style="margin-bottom: 0.14in">authors: Anthony Almudevar
		(Dept Biostatistics and Computational Biology, University of
		Rochester Medical Center, email:
		anthony_almudevar@urmc.rochester.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.urmc.rochester.edu/smd/biostat/Projects/Help/PC/Software_Listings.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: msg_proj</p>
		<li><p style="margin-bottom: 0.14in">reference: A Almudevar, C
		Field (1999), &quot;Estimation of single generation sibling
		relationships based on DNA markers&quot;, J. Agricultural,
		Biological and Environmental Statistics, 4(2):136-165.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PSAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Population
		Stratification Association Test</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: G Kimmel, MI Jordan,
		E Halperin, R Shamir, RM Karp</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://acgt.cs.tau.ac.il/psat/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Kimmel, Jordan,
		Halperin, Shamir, Karp (2007), &quot;A randomization test for
		controlling population stratification in whole-genome association
		studies&quot;, American Journal of Human Genetics, 81(5):895-905</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PSEUDO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.3.5</p>
		<li><p style="margin-bottom: 0.14in">description: Pseudo estimates
		genomewide empirical p-values for Kong and Cox tests of linkage
		using the replicate pool method, which for many data sets, improves
		upon the computational efficiency of conventional gene-dropping
		methods by several orders of magnitude. This allows Pseudo to
		handle data sets with large families and makes it particularly
		applicable to those situations where p-value estimation by standard
		methods is computationally prohibitive. Pseudo also estimates
		variance for reported p-values, produces graphical and text
		summaries of results, and is able to assess significance of
		multiple correlated outcomes. Pseudo is designed to work with the
		Merlin package and includes utilities for generating input files
		from standard Merlin output.</p>
		<li><p style="margin-bottom: 0.14in">authors: Janis Wigginton
		(email:wiggie@umich.edu) and Goncalo Abecasis</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/pseudo/download</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		UNIX, MacOS, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: pseudo, scan</p>
		<li><p style="margin-bottom: 0.14in">reference: Wigginton, Abecasis
		(2006), &quot;An evaluation of the replicate pool method: quick
		estimation of genome-wide linkage peak p-values&quot;, Genetic
		Epidemiology, 30(4):320-332.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PSEUDOMARKER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.6b (November
		2011)</p>
		<li><p style="margin-bottom: 0.14in">description: PSEUDOMARKER is a
		linkage analysis software for joint linkage and/or linkage
		disequilibrium analysis. PSEUDOMARKER can analyze different data
		structures jointly such as cases-controls, trios, sib-pairs,
		sib-ships, and extended families.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Tero Hiekkalinna
		(email: tero.hiekkalinna@helsinki.fi), Joseph D Terwilliger, Petri
		Norrgrann</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.helsinki.fi/~tsjuntun/pseudomarker/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C/C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: pseudomarker,
		ilinkpseudo, makepedpseudo, mlinkpseudo, linkmappseudo,
		unknownpseudo</p>
		<li><p style="margin-bottom: 0.14in">reference: Goring, Terwilliger
		(2000), &quot;Linkage analysis in the presence of errors IV: joint
		pseudomarker analysis of linkage and/or linkage disequilibrium on a
		mixture of pedigrees and singletons when the mode of inheritance
		cannot be accurately specified&quot;, American Journal of Human
		Genetics, 66(4):1310-1327</p>
		<li><p style="margin-bottom: 0.14in">Hiekkalinna, Sch&auml;ffer,
		Lambert, Norrgrann, G&ouml;ring, Terwilliger (2011), &quot;PSEUDOMARKER:
		a powerful program for joint linkage and/or linkage disequilibrium
		analysis on mixtures of singletons and related individuals&quot;,
		Human Heredity, 71:256-266</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PSEUDOMARKER.M</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.03</p>
		<li><p style="margin-bottom: 0.14in">description: PSEUDOMARKER is a
		set of programs written in MATLAB for the analysis of QTL data from
		inbred line crosses.</p>
		<li><p style="margin-bottom: 0.14in">authors: Saunak Sen, Gary
		Churhcill (Jackson Lab, email: pseudomarker@jax.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.jax.org/staff/churchill/labsite/software/pseudomarker/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: MATLAB</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sen, Churchill
		(2001), &quot;A statistical framework for quantitative trait
		mapping&quot;, Genetics, 159:371-387.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">PYPOP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: PYthon for
		POPulation genetics</p>
		<li><p style="margin-bottom: 0.14in">version: beta 0.7.0 (September
		2008)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Alex Lancaster, Mark
		P Nelson, Richard M Single, Diogo Meyer, Glenys Thomson</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: http://www.pypop.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Python</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(98/2000/XP), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lancaster, Nelson,
		Single, Meyer, Thomson (2003), &quot;PyPop: a software framework
		for population genomics: analyzing large-scale multi-locus genotype
		data&quot;, Proceedings of the Pacific Symposium on Biocomputing,
		8:514-525.</p>
		<li><p style="margin-bottom: 0.14in">Lancaster, Single, Solberg,
		Nelson, Thomson (2007), &quot;PyPop update - a software pipeline
		for large-scale multilocus population genomics&quot;, Tissue
		Antigens, 69(s1):192-197.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QGENE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 4.0.2</p>
		<li><p style="margin-bottom: 0.14in">description: A free,
		computationally effecient, easy-to-use Java program for comparative
		analyses of QTL mapping data and population simulation.</p>
		<li><p style="margin-bottom: 0.14in">authors: Roby Joehanes,
		Zhigang Guo, Prashanth Boddhireddy, J Clare Nelson (Kansas State
		University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: http://www.qgene.org/</p>
		<li><p style="margin-bottom: 0.14in">http://coding.plantpath.ksu.edu/qgene/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Nelson (1997),
		&quot;QGENE: software for marker-based genomic analysis and
		breeding&quot;, Molecular Breeding, 3:239-245. [<a href="http://dx.doi.org/10.1023/A:1009604312050">abstract</a>]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QMSIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Qtl and Marker
		SIMulator</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: QMSim was
		designed to simulate a wide range of genetic architectures and
		population structures in livestock. Large scale genotyping data and
		complex pedigrees can be efficiently simulated. QMSim is a family
		based simulator, which can also take into account predefined
		evolutionary features, such as LD, mutation, bottlenecks and
		expansions. The simulation is basically carried out in two steps:
		In the first step, a historical population is simulated to
		establish mutation-drift equilibrium and, in the second step,
		recent population structures are generated, which can be complex.
		QMSim allows for a wide range of parameters to be incorporated in
		the simulation models in order to produce appropriate simulated
		data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mehdi Sargolzaei
		(email: msargol@uoguelph.ca), Flavio S Schenkel</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.aps.uoguelph.ca/~msargol/qmsim/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sargolzaei,
		Schenkel (2009), &quot;QMSim: a large-scale genome simulator for
		livestock&quot;, Bioinformatics, 25:680-681.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QTDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Quantitative
		(Trait) Transmission/Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version: 2.5.1 (January 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: Performs linkage
		disequilibrium (TDT) and association analysis for quantitative
		traits. Includes support for the methods of Abecasis et al. (2000),
		Fulker et al. (1999), Monks et al. (1998), Allison (TDTQ5, 1997)
		and Rabinowitz (1997). Supports families of any size, with or
		without parental information. Includes simple variance components
		modelling. Interfaces with SimWalk2 for IBD estimation.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Goncalo Abecasis
		(goncalo@well.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/QTDT/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: qtdt, prelude,
		finale, pedstats</p>
		<li><p style="margin-bottom: 0.14in">reference: Abecasis, Cardon
		and Cookson (2000), &quot;A general test of association for
		quantitative traits in nuclear families&quot;, American Journal of
		Human Genetics, 66(1):279-292. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QTL-ALL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: QTL Analysis and
		Linkage Library</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: QTL-ALL is a
		software package designed to make as many as possible of the new
		statistics (e.g. score statistics) widely available. The software
		consists of a MEGA2-like interface for data analysis preparation
		and a library of R routines that computes linkage statistics.
		QTL-ALL reads in input data, creates re-formatted output data
		files, calls external IBD-generation software such as MERLIN or
		SIMWALK2, then computes statistics using our R library, and finally
		produces tables and plots of statistics and p-values. This entire
		sequence is highly automated, requiring minimal user-intervention.
		The initial release of the software computes a number of newer
		QTL-mapping statistics, including several score statistic variants,
		and can handle nuclear family data, including specialty designs
		such as discordant and concordant (affected) pairs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Nandita Mukhopadhyay,
		S Bhattacharjee, CL Kuo, Dan Weeks, E Feingold</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 2006 Mukhopadhyay, Bhattacharjee, Kuo, Weeks,
		Feingold, Am J Hum Genet, suppl. ?, ? (1995)).
		http://www.ashg.org/genetics/abstracts/abs06/f1567.htm</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QTL CAFE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: May 1998</p>
		<li><p style="margin-bottom: 0.14in">description: providing a user
		freiendly way to perform QTL analysis. The software currently
		allows 3 types of QTL analysis: (1) single marker ANOVA. (2) marker
		regression. (3) interval mapping by regression.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biosciences-labs.bham.ac.uk/Kearsey/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Java</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QTL Cartographer</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Window version 2.5
		(September 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: A program to map
		quantitative traits sing a map of molecular markers.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Christopher J.
		Basten, Bruce S. Weir, Zhao-Bang Zeng, Shengchu Wang (Dept
		Statistics, North Carolina State University);</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://statgen.ncsu.edu/qtlcart/cartographer.html</p>
		<li><p style="margin-bottom: 0.14in">ftp://statgen.ncsu.edu/pub/qtlcart/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables: rmap, rqtl,
		rcross, qstats, lrmapqtl, srmapqtl, zmapqtl, prune, preplot, eqtl,
		qtlcart</p>
		<li><p style="margin-bottom: 0.14in">reference: Basten,Weir,Zeng
		(1994), &quot;Zmap-a QTL cartographer&quot;, in Proceedings of the
		5th World Congress on Genetics Applied to Livestock Production:
		Computing Strategies and Software, vol 22, eds. Smith, Gavora,
		Chesnais, Fairfull, Gibson, Kennedy, Burnside.</p>
		<li><p style="margin-bottom: 0.14in">online manual</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QTL EXPRESS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: August 2005</p>
		<li><p style="margin-bottom: 0.14in">description: A web-based
		user-friendly package to map Quantitative Trait Loci in outbred
		populations. Population structures catered for are line crosses,
		halfsib families, nuclear families and sibpairs. Permutation tests
		to determine empirical significance levels and bootstrapping to
		estimate empirical confidence intervals of QTL locations are
		optional. Fixed effects/covariates can be fitted and models may
		include single or multiple QTL. Results are presented in tabular
		and graphical format.</p>
		<li><p style="margin-bottom: 0.14in">authors: George Seaton (email:
		george.seaton@ed.ac.uk), Chris Haley, Sara Knott, Xijiang Yu, Peter
		Visscher (Institute of Cell, Animal and Population Biology, Univ.
		Edingbergh)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://qtl.cap.ed.ac.uk/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: G Seaton, CS Haley,
		SA Knott, M Kearsey, PM Visscher (2002), &quot;QTL Express: mapping
		quantitative trait loci in simple and complex pedigrees&quot;,
		Bioinformatics, 18:339-340</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QTLNETWORK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: QTLNetwork is a
		software package for mapping and visualizing the genetic
		architecture underlying complex traits for experimental populations
		derived from a cross between two inbred lines.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jian Yang, Chengcheng
		Hu, Xiuzi Ye, Jun Zhu (Zhejiang University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://ibi.zju.edu.cn/software/qtlnetwork</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yang, Hu, Hu, Yu,
		Xia, Ye, Zhu (2008), &quot;QTLNetwork: mapping and visualizing
		genetic architecture of complex traits in experimental
		populations&quot;, Bioinformatics. 24(5):721-723.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QUANTO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2.1 (January 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: Quanto is a
		program that computes sample size or power for association studies
		of genes, environmental factors, gene-environment interaction, or
		gene-gene interaction. Available study designs for a disease
		(binary) outcome include the unmatched case-control, matched
		case-control, case-sibling, case-parent, and case-only designs.
		Study designs for a quantitative tra it include independent
		individuals and case parent designs. Quanto is a 32-bit Windows
		application requiring Windows 95, 98, NT, 2000, ME or XP to run.
		The graphical user interface allows th e user to easily change the
		model and view the results without having to edit an input file and
		rerun the program for every model. The results of a session are
		stored to a log file. This log can be printed or saved to a file
		for reviewing at a later date. An option is included to create a
		text file of the log that can be imported into other documents.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jim Gauderman (email:
		jimg@usc.edu), John Morrison (email: jmorr@usc.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hydra.usc.edu/GxE</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		Window(98/NT/2000/..)</p>
		<li><p style="margin-bottom: 0.14in">executables: quanto.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Gauderman (2002),
		&quot;Sample size requirements for matched case-control studies of
		gene-environment interaction&quot;, Statistics in Medicine,
		21:35-50</p>
		<li><p style="margin-bottom: 0.14in">Gauderman (2002), &quot;Sample
		size requirements for association studies of gene-gene
		interaction&quot;, American Journal of Epidemiology, 155:478-484</p>
		<li><p style="margin-bottom: 0.14in">WJ Gauderman (2003),
		&quot;Candidate gene association studies for a quantitative trait,
		using parent-offspring trios&quot;, Genetic Epidemiology,
		25:327-338</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QU-GENE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: QUantitative
		GENEtics</p>
		<li><p style="margin-bottom: 0.14in">version: v1 (1998)</p>
		<li><p style="margin-bottom: 0.14in">description: A package for
		quantitative analysis of genetic models</p>
		<li><p style="margin-bottom: 0.14in">Authors: D. Podlich (email:
		d.podlich@mailbox.uq.edu.au) and M. Cooper (email:
		m.cooper@mailbox.uq.edu.au)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pig.ag.uq.edu.au/qu-gene/ (not available)</p>
		<li><p style="margin-bottom: 0.14in">source code language: Fortran</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: qugene.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Podlich, Cooper
		(1998), &quot;QU-GENE: a simulation platform for quantitative
		analysis of genetic models&quot;, Bioinformatics. 14:632-653.
		Online manual: http://pig.ag.uq.edu.au/qu-gene/manual.htm</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">QUTIE (see also CCRAVAT)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Case-Control RAre
		Variant Analysis Tool</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Enabling the
		analysis of rare variants in large-scale case control and
		quantitative trait association studies. CCRaVAT (Case-Control Rare
		Variant Analysis Tool) and QuTie (Quantitative Trait) are software
		packages that enable efficient large-scale analysis of rare
		variants across specific regions or genome-wide. These programs
		implement a rare variant super-locus or collapsing method that
		investigates the accumulation of rare variant alleles in either a
		case-control or quantitative trait study design.</p>
		<li><p style="margin-bottom: 0.14in">authors: Eleftheria Zeggini's
		group (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/rarevariant/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RC-TDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:
		Reconstruction-Combined Transmission Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1d (Jul 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: A family-based
		association method that allows testing for linkage in the presence
		of linkage disequilibrium between an autosomal marker and a disease
		even if there is only incomplete parental-marker information.
		Recently, Horvath et al. (2000) described a similar procedure
		(XRC-TDT) for X-linked markers. The distribution contains SAS
		macros that calculate the RC-TDT and XRC-TDT test statistics, as
		well as their respective exact P values.</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Knapp (Univ
		Bonn)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uni-bonn.de/~umt70e/soft.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: SAS
		macro</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Knapp (1999), &quot;The
		transmission/disequilibrium test and parental-genotype
		reconstruction: the reconstruction-combined
		transmission/disequilibrium test&quot;, American Journal of Human
		Genetics, 64(3):861-870</p>
		<li><p style="margin-bottom: 0.14in">Knapp (1999b), &quot;Using
		exact P values to compare the power between the
		reconstruction-combined transmission/disequilibrium test and the
		sib transmission/disequilibrium test&quot; (letter), American
		Journal of Human Genetics, 65:1208-1210. [ html]</p>
		<li><p style="margin-bottom: 0.14in">Horvath, Laird, Knapp (2000),
		&quot;The transmission/disequilibrium test and parental-genotype
		reconstruction for X-chromosomal markers&quot;, American Journal of
		Human Genetics, 66(3):1161-1167. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">REAPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: tag SNP
		selelciton tool according to r2-bins method. It is specifically
		designed for full genome scale deterministic tagging.</p>
		<li><p style="margin-bottom: 0.14in">authors: Reedik Magi, Lauris
		Kaplinski, Maido Remm (Univ of Tartu, Estonia)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinfo.ebc.ee/download/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Magi, Kaplinski,
		Remm (2006), &quot;The whole genome TagSNP selection and
		transferability among HapMap populations&quot;, Pacific Symposium
		on Biocomputing, 11:535-543.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RECORD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: REcombination
		Counting and ORDering</p>
		<li><p style="margin-bottom: 0.14in">version: 2005 and 2006</p>
		<li><p style="margin-bottom: 0.14in">description: RECORD can be
		used for the ordering of loci on genetic linkage maps. The method
		is based on minimisation of the total number of recombination
		events. Since the criterion does not require intensive
		calculations, the algorithm rapidly produces an optimal ordering as
		well as a series of near-optimal ones. The latter provides insight
		into the local certainty of ordering along the map. RECORD can deal
		with the following types of mapping populations: BC1, F2, F3, RILs
		(in fact any generation obtained by repeated selfing of a hybrid
		between homozygous parents). Data files are .LOC JoinMap format.
		Mapping populations from non-inbreds should be split into BC1 or
		HAP data that represent the maternal and paternal gametes.</p>
		<li><p style="margin-bottom: 0.14in">authors: Piet Stam, Herman van
		Eck (Wageningen University, Laboratory for Plant Breeding, email:
		herman.vaneck@wur.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.plantbreeding.wur.nl/UK/software_record.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(Win98, XP) and MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: record_win.exe,
		ord.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Van Os, Stam,
		Visser, Van Eck (2005), &quot;RECORD: A novel method for ordering
		loci on a genetic linkage map&quot;, Theoretical and Applied
		Genetics, 112(1):30-40</p>
		<li><p style="margin-bottom: 0.14in">Van Os, Andrzejewski, Bakker,
		Barrena, Bryan, Caromel, Ghareeb, Isidore, de Jong, van Koert,
		Lefebvre, Milbourne, Ritter, van der Voort, Rousselle-Bourgeois,
		van Vliet, Waugh, Visse, Bakke, Van Eck (2006), &quot;Construction
		of a 10,000-marker ultradense genetic recombination map of potato:
		Providing a framework for accelerated gene isolation and a
		genomewide physical map&quot;, Genetics, 173(2):1075-1087</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RELATIVE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.10 (dec 97)</p>
		<li><p style="margin-bottom: 0.14in">description: software for
		relationship estimation, in particular between putative sibs when
		parents are untyped</p>
		<li><p style="margin-bottom: 0.14in">authors: Harald Goring
		&lt;hgoring@sfbrgenetics.org&gt;</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://linkage.rockefeller.edu/software/relative</p>
		<li><p style="margin-bottom: 0.14in">source code language: ANSI C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		MS-Windows, UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: relative</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1995 (G&ouml;ring and Ott, Am J Hum Genet, suppl.
		57, A192 (1995));</p>
		<li><p style="margin-bottom: 0.14in">Goring and Ott (1997),
		&quot;Relationship estimation in affected sib pair analysis of
		late-onset diseases&quot;, European Journal of Human Geneics,
		5:69-77.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RELATIVEFINDER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: relativeFinder is
		a program for checking relationships between pairs of individuals.
		There are many excellent programs that carry out similar tasks.
		Some of the unique features in relativeFinder are the batch mode
		options, that allow large jobs to be divided into many smaller jobs
		(suitable for deployment on a compute cluster environment), and the
		flexibility of the underlying Merlin engine, which allows relative
		finder to handle large pedigrees and consider a variety of
		alternate relationships -- including potential relationships
		specified by the user on the fly.</p>
		<li><p style="margin-bottom: 0.14in">authors: Abecasis's group
		(Univ Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/RelativeFinder</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RELCHECK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.67 (August 2000)</p>
		<li><p style="margin-bottom: 0.14in">description: Program for
		verifying the relationships between all pairs of individuals in a
		linkage study, by use of (autosomal) genome scan data, with
		allowance for the presence of genotyping errors.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Karl W. Broman (Johns
		Hopkins Univ, kbroman@jhsph.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.jhsph.edu/~kbroman/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		MS-Windows, UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: relcheck</p>
		<li><p style="margin-bottom: 0.14in">reference: Boehnke, Cox
		(1997), &quot;Accurate inference of relationships in sib-pair
		linkage studies&quot;, American Journal of Human Genetics,
		61(2):423-429</p>
		<li><p style="margin-bottom: 0.14in">Broman, Weber (1998),
		&quot;Estimation of pairwise relationships in the presence of
		genotyping errors&quot; (letter), American Journal of Human
		Genetics, 63(5):1563-1564</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RELPAIR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.01 (June 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: A program that
		infers the relationships of pairs of individuals based on genetic
		marker data, either within families or across an entire sample.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Michael Boehnke,
		William Duren (Biostat, U Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://csg.sph.umich.edu/boehnke/relpair.php</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95), UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting 1997 (Duren,Cox,Hauser,Boehnke, Am J. Hum Genet.
		suppl 61, A273(1997)).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RELTYPE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a computer
		program for determining biological relatedness between individuals
		based on allele sharing at microsatellite loci</p>
		<li><p style="margin-bottom: 0.14in">authors: D.N. Stivers
		(dstivers@utsph.uth.tmc.edu), Y. Zhong, C.L. Hanis, R. Chakraborty
		(Univ of Texas at Houston)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hgc.sph.uth.tmc.edu (not available yet)</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Chakraborty, Jin
		(1993), &quot;Determination of relatedness between individuals
		using DNA fingerprinting&quot;, Human Biology, 65(6):875-895.</p>
		<li><p style="margin-bottom: 0.14in">Am Soc Hum Genet annual
		meeting 1996 (Stivers, Zhong, Hanis, Chakraborty, Am J. Hum Genet.
		suppl 59, A190 (1996)).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RHMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Radiation Hybrid
		Map</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0 (Sept 1996)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Boehnke (Univ
		Michigan, boehnke@sph.umich.edu),E. Hauser, K. Lange, K. Lunetta,
		J. Uro, J. VanderStoep</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://csg.sph.umich.edu/boehnke/rhmap.php</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: rhmap, rh2pt,
		rhminbrk, rhmaxlik</p>
		<li><p style="margin-bottom: 0.14in">reference: Boehnke,Lange,Cox
		(1991), &quot;Statistical methods for multipoint radiation hybrid
		mapping&quot;, American Journal of Human Genetics, 49(6):1174-1188</p>
		<li><p style="margin-bottom: 0.14in">Lange,Boehnke, Cox, Lunetta
		(1995), Genome Research, 5:136-150.</p>
		<li><p style="margin-bottom: 0.14in">Lunetta,Boehnke, Lange, Cox
		(1995), Genome Research, 5:151-163.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RHMAPPER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.22 (1998)</p>
		<li><p style="margin-bottom: 0.14in">description: an interactive
		program for radiation hybrid mapping</p>
		<li><p style="margin-bottom: 0.14in">authors: Lincoln Stein
		(Whitehead Institute, lstein@genome.wi.mit.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-genome.wi.mit.edu/ftp/pub/software/rhmapper/</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp-genome.wi.mit.edu/pub/software/rhmapper</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Hudson, Stein, et
		al. (1995), Science, 270:1945-1954.</p>
		<li><p style="margin-bottom: 0.14in">Slonim, Stein, Kruglyak,
		Lander (1996), in preparation.</p>
		<li><p style="margin-bottom: 0.14in">online user's guide</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RISCALW</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: RISk CALculation in
		Windows</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: RISCALW is a
		novel convenient Windows program for risk calculation in families
		with Duchenne muscular dystrophy. It is based on an extended
		genetic model which includes germline mosaicism and different new
		mutation rates depending on sex and mutation type. Arbitrary family
		structures and additional diagnostic information like genotypes
		from intragenetic and flancing genetic markers of the dystrophin
		gene, creatin kinase values and female deletion test results can be
		taken into account.</p>
		<li><p style="margin-bottom: 0.14in">authors: Christine Fischer
		(email: christine.fischer@med.uni-heidelberg.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.riscalw.uni-hd.de</p>
		<li><p style="margin-bottom: 0.14in">source code language: Delphi 4</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Fischer, Kruger,
		Gross (2006), &quot;RISCALW: a windows program for risk calculation
		in families with Duchenne Muscular Dystrophy&quot;, Annals of Human
		Genetics, 70:249-254.</p>
		<li><p style="margin-bottom: 0.14in">Fischer, Gross, Kruger,
		Cremer, Vogel, Grimm (2006), &quot;Modeling germline mosaicism and
		different new mutation rates simultaneously for appropriate risk
		calculations in families with Duchenne muscular dystrophy&quot;,
		Annals of Human Genetics, 70:237-248.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ROMPREV</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Regression of
		Offspring on Mid-Parent (REVised)</p>
		<li><p style="margin-bottom: 0.14in">version: 0.2.1beta</p>
		<li><p style="margin-bottom: 0.14in">description: a software tool
		for testing for association between polymorphisms and quantitative
		traits, as well as estimating trait heritability and locus-specific
		heritability using family data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Marie-Helene
		Roy-Gagnon and Alexander F. Wilson (afw@mail.nih.gov)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://research.nhgri.nih.gov/ROMPrev/</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.nhgri.nih.gov/pub/software/ROMPrev/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: any with R
		installation</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Roy-Gagnon,
		Mathias, Fallin, Jee, Broman, Wilson (2008), &quot;An extension of
		the Regression of Offspring on Mid-Parent to test for association
		and estimate locus-specific heritability: The revised ROMP method&quot;,
		Annals of Human Genetics, 72:115-125.</p>
		<li><p style="margin-bottom: 0.14in">Roy-Gagnon (2004),
		&quot;Theoretical and applied investigations of a Regression of
		Offspring on Mid-Parent approach: Familial idiopathic scoliosis&quot;,
		Ph.D. dissertaion. Johns Hopkins University.</p>
		<li><p style="margin-bottom: 0.14in">Pugh, Papanicolaou, Justice,
		Roy-Gagnon, Sorant, Kingman, Wilson (2001), &quot;Comparison of
		variance components, ANOVA and regression of offspring on midparent
		(ROMP) methods for SNP markers&quot;, Genetic Epidemiology,
		21(Suppl 1):S794-S799.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ROSATTA SYLLEGO SYSTEM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: evaluation version
		(October 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: The Rosetta
		Syllego system is a genetic data management and analysis system
		designed to advance your whole genome association, linkage, and
		eQTL studies. Designed for biologists, statistical geneticists, and
		investigators responsible for generating genotyping data, the
		Syllego system provides you with an easy-to-use project workspace
		so that you can organize, analyze, and share your genotype and
		phenotype data along with your analysis results. With the Syllego
		system, generating high quality analysis data and meaningful
		results becomes simple. The Syllego system automates all tedious
		data management and data formatting tasks so that you can
		streamline your genetic analysis workflows using your analysis
		methods of choice. Managing all your genetic data and reference
		information is straightforward. The Syllego system converts public
		and private genotype data sets and reference annotations, such as
		dbSNP and HapMap, as well as individual (sample) information into a
		single, consistent repository for fast, convenient access.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.rosettabio.com/products/syllego/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RTDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Robust
		Transmission/Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bios.ugr.es/~mabad/rTDT/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Matlab,
		C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sebastiani, Abad,
		Alpargu, Ramoni (2004), &quot;Robust Transmission/Disequilibrium
		Test for incomplete family genotypes&quot;, Genetics,
		168:2329-2337.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/ADEGENET</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2-6 (September
		2010)</p>
		<li><p style="margin-bottom: 0.14in">description: ADEGENET is a
		package dedicated to the handling of molecular marker data for
		multivariate analysis. This package is related to ADE4, a R package
		for multivariate analysis, graphics, phylogeny and spatial
		analysis.</p>
		<li><p style="margin-bottom: 0.14in">authors: Thibaut Jombart
		(email: jombart@biomserv.univ-lyon1.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://adegenet.r-forge.r-project.org/</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/adegenet/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Jombart (2008),
		&quot;adegenet: a R package for the multivariate analysis of
		genetic markers&quot;, Bioinformatics, 24:1403-1405.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/ARP.GEE (see ARP.GEE)</p>
	<li><p style="margin-bottom: 0.14in">R/BARS (see BARS)</p>
	<li><p style="margin-bottom: 0.14in">R/COMPOSITELD (see COMPOSITELD)</p>
	<li><p style="margin-bottom: 0.14in">R/COVIBD (see COVIBD)</p>
	<li><p style="margin-bottom: 0.14in">R/EHP (see EHP.R)</p>
	<li><p style="margin-bottom: 0.14in">R/ENTROPY_BLOCKER (see ENTROPY
	BLOCKER)</p>
	<li><p style="margin-bottom: 0.14in">R/FEST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.08 (August, 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: an R package for
		simulations and likelihood calculations of pair-wise family
		relationships using DNA marker data.</p>
		<li><p style="margin-bottom: 0.14in">authors: &Oslash;ivind Skare
		(email: oivind.skare@medisin.uio.no</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://folk.uio.no/thoree/FEST/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Skare, Sheehan,
		Egeland (2009), &quot;Identification of distant family
		relationships&quot;, Bioinformatics, 25(18):2376-2382.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/GAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: R/Genetic Analysis
		Package</p>
		<li><p style="margin-bottom: 0.14in">version: 1.17-39 (July 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: This is an
		integrated package for genetic data analysis of both population and
		family data. Currently it contains functions for sample size
		calculations of both population-based and family-based designs,
		classic twin ACE/ADE/AE/CE models, probability of familial disease
		aggregation, kinship calculation, some statistics in linkage
		analysis, and association analysis involving one or more genetic
		markers including haplotype analysis with or without environmental
		covariates</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinghua Zhao (MRC
		Epidemiology Unit at Strangeways Research Lab, UK) (email:
		jinghua.zhao@mrc-epid.cam.ac.uk) in collaboration with other
		colleagues, and with help from Kurt Hornik of the R core</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-epid.cam.ac.uk/~jinghua.zhao/r-progs.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: aldh2, apoeapoc,
		bt, chow.test, fbsize, fsnps, gc.em, gcontrol, genecounting, gif,
		hap, hap.em, hap.score, hla, htr, hwe, hwe.hardy, kbyl, kin.morgan,
		makeped, mia, mtdt, muvar, pbsize, pfc, pgc, s2k, tbyt, whscore</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao and Tan
		(2006), &quot;Integrated analysis of genetic data with R&quot;,
		Human Genomics, 2(4):258-265.</p>
		<li><p style="margin-bottom: 0.14in">reference manual (PDF):
		http://cran.r-project.org/doc/packages/gap.pdf</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/GC, R/GCF (see GC/GCF)</p>
	<li><p style="margin-bottom: 0.14in">R/GENABEL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.6-4 (August 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: GenABEL is a
		software created to facilitate whole-genome association analysis.
		Specific features of GenABEL include:</p>
		<li><p style="margin-bottom: 0.14in">Support of modern methods for
		analysis in outbred populations</p>
		<li><p style="margin-bottom: 0.14in">Maximum Likelihood estimation
		of linear, logistic and Cox regression on genome-wide scale</p>
		<li><p style="margin-bottom: 0.14in">A range of functions
		specifically designed to help GWA analysis in samples of
		individuals from genetically isolated populations, and/or samples
		of related individuals (family-based design), and other analysis in
		presence of stratification</p>
		<li><p style="margin-bottom: 0.14in">Special functions oriented to
		meta-analysis</p>
		<li><p style="margin-bottom: 0.14in">Extensive documentation (about
		100 pages of reference manual and 150 pages of tutorial)</p>
		<li><p style="margin-bottom: 0.14in">Many example data sets and
		exercises</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mga.bionet.nsc.ru/~yurii/ABEL/GenABEL/</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/GenABEL/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Aulchenko, Ripke,
		Isaacs, van Duijn (2007), &quot;GenABEL: an R library for
		genome-wide association analysis&quot;, Bioinformatics,
		23(10):1294-1296.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/GWAPOWER (see GWAPOWER)</p>
	<li><p style="margin-bottom: 0.14in">R/HAPASSOC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2-4 (July 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: a likelihood
		approach is used to inference of haplotype and nongenetic effects
		and their interactions in generalized linear models of disease
		penetrance, when haplotype phase is unknown for some subjects.
		Parameter estimates are obtained by use of an
		expectation-maximization (EM) algorithm and standard errors are
		calculated using Louis' formula.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://stat-db.stat.sfu.ca:8080/statgen/research/hapassoc</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/hapassoc/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/HCLUST (see HCLUST)</p>
	<li><p style="margin-bottom: 0.14in">R/IBDREG (see IBDREG)</p>
	<li><p style="margin-bottom: 0.14in">R/LAPSTRUCT (see LAPSTRUCT)</p>
	<li><p style="margin-bottom: 0.14in">R/LDGROUP (see LDGROUP)</p>
	<li><p style="margin-bottom: 0.14in">R/LDHEATMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.2-9 (August 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: plots measures of
		pairwise linkage disequilibria for SNPs</p>
		<li><p style="margin-bottom: 0.14in">authors: Ji-Hyung Shin (email:
		shin@sfu.ca), Sigal Blay, Nicholas Lewin-Koh, Brad McNeney, Jinko
		Graham</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://stat-db.stat.sfu.ca:8080/statgen/research/LDheatmap</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/LDheatmap/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Shin, Blay,
		McNeney, Graham (2006), &quot;LDheatmap: an R function for
		graphical display of pairwise linkage disequilibria between single
		nucleotide polymorphisms&quot;, Journal of Statistical Software,
		16, code snippet 3.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/LUCA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Likelihood Under
		Covariate Assumptions</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Ji-Hyung Shin (email:
		shin@sfu.ca), Brad McNeney, Jinko Graham</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://stat-db.stat.sfu.ca:8080/statgen/research/luca/</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/luca/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Shin, McNeney,
		Graham (2007), &quot;Case-control inference of interaction between
		genetic and nongenetic risk factors under assumptions on their
		distribution&quot;, Statistical Applications in Genetics and
		Molecular Biology, 6(1):13.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/META</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.6 (June 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: Fixed and random
		effects meta-analysis. Functions for tests of bias, forest and
		funnel plot.</p>
		<li><p style="margin-bottom: 0.14in">authors: Guido Schwarzer
		(email: sc@imbi.uni-freiburg.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cran.r-project.org/web/packages/meta/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/METASIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: METApopulation
		SIMulation</p>
		<li><p style="margin-bottom: 0.14in">version: 1.01</p>
		<li><p style="margin-bottom: 0.14in">description: an R package that
		uses an individual-based approach to simulate distributions of
		genotypes that result from arbitrary within and among population
		demographies (including extinction/recolonization). These
		distributions can be used to test new or existing
		population-genetics summary statistics or develop &quot;null&quot;
		distributions under various demographies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Allan Strand (email:
		stranda@cofc.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://linum.cofc.edu/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R, C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: A Strand (2002),
		&quot;Metasim 1.0: an individual-based environment for simulating
		population genetics of complex population dynamics&quot;, Mol Ecol
		Notes, 2:373-376.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/ONEMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0-1 (April 2009)</p>
		<li><p style="margin-bottom: 0.14in">description: ONEMAP is an
		environment for constructing linkage maps in outcrossing plant
		species, using full-sib families derived from two outbreed
		(non-inbreeding) parent plants.</p>
		<li><p style="margin-bottom: 0.14in">authors: Gabriel Rodrigues
		Alves Margarido, Anete Pereira de Souza, Antonio Augusto Franco
		Garcia (email: aafgarci@esalq.usp.br)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.ciagri.usp.br/~aafgarci/OneMap/</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/onemap/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: load.onemap,
		read.outcross, extract.group, remove.marker, arbitr.rf.2pts,
		est.rf.2pts, modigy.rf.2pts, group, map, make.map, est.rf.3pts</p>
		<li><p style="margin-bottom: 0.14in">reference: Garcia, Kido, Meza,
		Souza, Pinto, Pastina, Leite, Silva, Ulian, Figueira, Souza (2006),
		&quot;Development of an integrated genetic map of a sugarcane
		(Saccharum spp.) commercial cross, based on a maximum-likelihood
		approach for estimation of linkage and linkage phases&quot;,
		Theoretical and Applied Genetics, 112:298-314.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/PIAGE (see PIAGE)</p>
	<li><p style="margin-bottom: 0.14in">R/POOLSCORE (see POOLSCORE)</p>
	<li><p style="margin-bottom: 0.14in">R/POPGEN (see POPGEN)</p>
	<li><p style="margin-bottom: 0.14in">R/QTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.05-2 (November
		2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Program for
		mapping quantitative trait loci in experimental crosses.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Karl W. Broman (Johns
		Hopkins Univ, kbroman@jhsph.edu) and Hao Wu (The Jackson
		Laboratory)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: http://www.rqtl.org</p>
		<li><p style="margin-bottom: 0.14in">source code language: C and R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Broman, Wu, Sen,
		Churchill (2003), Bioinformatics, 19:889-890. [Medline]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/QTLBIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.7.7 (April 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: QTLBIM, a library
		for QTL Bayesian Interval Mapping, provides a Bayesian model
		selection approach to map multiple interacting QTL. It works on
		experimentally inbred lines and performs a genome-wide search to
		locate multiple potential QTL. The package can handle continuous,
		binary and ordinal traits.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.qtlbim.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/QTLDESIGN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: v0.91 (May 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: R/qtlDesign is
		software to help plan quantitative trait locus (QTL) experiments.</p>
		<li><p style="margin-bottom: 0.14in">authors: Saunak Sen, Jaya
		Satagopan, Karl Broman, Gary Churchill</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.epibiostat.ucsf.edu/biostat/sen/qtldesign-software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/SNP.PLOTTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.3 (October 2009)</p>
		<li><p style="margin-bottom: 0.14in">description: snp.plotter is an
		R package that creates publishable-quality plots of p-values using
		single SNP and/or haplotype data. Main features of the package
		include options to display a linkage disequilibrium (LD) plot and
		the ability to plot multiple sets of results simultaneously. Plots
		can be created using global and/or individual haplotype p-values
		along with single SNP p-values. Images are created as either
		Portable Document Format (PDF) or Encapsulated (EPS) files.</p>
		<li><p style="margin-bottom: 0.14in">authors: Augustin Luna,
		Kristin Nicodemus (email: augustin@mail.nih.gov)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cbdb.nimh.nih.gov/~kristin/snp.plotter.html</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/snp.plotter/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Luna, Nicodemus
		(2007), &quot;snp.plotter: an R-based SNP/haplotype association and
		linkage disequilibrium plotting package&quot;, Bioinformatics,
		23(6):774-776.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/SPECTRAL-GEM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SPECTRAL graph
		approach for GEnetic Matching</p>
		<li><p style="margin-bottom: 0.14in">version: 1 (July 2009)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Ann Lee, Diana Luca,
		Lambertus Klei, Bernie Devlin, Kathryn Roeder</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lee, Luca, Klei,
		Devlin, Roeder (2010), &quot;Discovering genetic ancestry using
		spectral graph theory&quot;, Genetic Epidemiology, 34(1):51-59.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/STEPWISE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.2-4 (November 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: A stepwise
		approach to identifying recombination breakpoints in a sequence
		alignment</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinko Graham, Brad
		McNeney, Francoise Seillier-Moiseiwitsch, Sigal Blay</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://stat-db.stat.sfu.ca:8080/statgen/research/stepwise/</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/stepwise/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/TDTHAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1-2 (November 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: a package for TDT
		with extended haplotypes in the &quot;R&quot; language. &quot;R&quot;
		is the public domain dialect of &quot;S&quot;. It should be
		possible to port this library to the commercial &quot;Splus&quot;
		product. The main problem would be translation of the help files.</p>
		<li><p style="margin-bottom: 0.14in">Authors: David Clayton
		(Cambridge Institute, david.clayton@cimr.cam.ac.uk</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-gene.cimr.cam.ac.uk/clayton/software/</p>
		<li><p style="margin-bottom: 0.14in">http://cran.r-project.org/web/packages/tdthap/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R/Splus</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Clayton, Jones
		(1999), &quot;Transmission/disequilibrium tests for extended marker
		haplotypes&quot;, American Journal of Human Genetics,
		65(4):1161-1169</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">RVTESTS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Rare Variants TESTS</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: X Zhan, Youna Hu, G
		Abecasis (Univ Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/RvTests</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">R/WEIGHTED_FDR (see
	WEIGHTED_FDR)</p>
	<li><p style="margin-bottom: 0.14in">S</p>
	<li><p style="margin-bottom: 0.14in">SAGE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:Statistical Analysis
		for Genetic Epidemiology</p>
		<li><p style="margin-bottom: 0.14in">version: 5.3</p>
		<li><p style="margin-bottom: 0.14in">description: The S.A.G.E.
		software provides researchers with the tools necessary for various
		types of statistical genetic analysis of human family data.</p>
		<li><p style="margin-bottom: 0.14in">authors: R Elston, J
		Bailey-Wilson, G Bonney, L. Tran, B. Keats, A. Wilson</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://darwin.cwru.edu/sage/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C
		[version 4.0 will be in C++]</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX(DEC
		UNIX/Solaris), MS-Windows (95/NT), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: assoc, fcor,
		freq, genibd, lodlink, lodpal, markerinfo, mlod, pedinfo, reltest,
		segreg, sibpal, tdtex</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SAMTOOLS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Sequence
		Alignment/Map TOOLS</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SAM Tools provide
		various utilities for manipulating alignments in the SAM format,
		including sorting, merging, indexing and generating alignments in a
		per-position format. The parts that relevant to genetic analysis
		include SNP and indel callings.</p>
		<li><p style="margin-bottom: 0.14in">authors: Heng Li (Broad
		Institute), Richard Durbin (Wellcome Trust Sanger Institute) et al.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://samtools.sourceforge.net/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C, C++,
		Java, Perl, Python, Ruby, Common LISP, Haskell</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Li, Handsaker,
		Wysoker, Fennell, Ruan, Homer, Marth, Abecasis, Durbin, and 1000
		Genome Project Data Processing Subgroup (2009), &quot;The sequence
		alignment/map (SAM) format and SAMtools&quot;, Bioinformatics,
		25:2078-2079.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SASGENE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (March 1999)</p>
		<li><p style="margin-bottom: 0.14in">description: a SAS computer
		program for gene segregation and linkage analysis in breeding
		population</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinsheng Liu, Todd
		Wehner, Sandy Donaghy</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cuke.hort.ncsu.edu/cucurbit/wehner/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: SAS</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Liu, Wehner,
		Donaghy (1997), &quot;SASGENE: A SAS computer program for genetic
		analysis of gene segregation and linkage&quot;, Journal of
		Heredity, 88:253-254.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SAS/GENETICS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: add-on to the SAS
		System, Release 8.2</p>
		<li><p style="margin-bottom: 0.14in">description: Software for
		summarizing marker properties (allele &amp; genotype frequencies,
		tests for Hardy-Weinberg equilibrium, measures of marker
		informativeness), examining marker-marker relationships (tests and
		measures of linkage disequilibrium, and haplotype frequency
		estimation), and exploring marker-trait associations using
		case-control or family-based tests</p>
		<li><p style="margin-bottom: 0.14in">Authors: W Czika, X Yu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX(HPUX/AIXR/Solaris/..), MVS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: introduction</p>
		<li><p style="margin-bottom: 0.14in">price: please contact
		shannon.conners@jmp.com for details</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SASQUANT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (May 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: a SAS software
		program to estimate genetic effects and heritabilities of
		quantitative traits in breeding populations consisting of six
		related generations</p>
		<li><p style="margin-bottom: 0.14in">authors: Gabriele Gusmini,
		Todd Wehner , Sandy B. Donaghy</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cuke.hort.ncsu.edu/cucurbit/wehner/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: SAS</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Gusmini, Wehner,
		Donaghy (2007), &quot;SASQuant: A SAS software program to estimate
		genetic effects and heritabilities of quantitative traits in
		populations consisting of six related generations&quot;, Journal of
		Heredity, 98(4):345-350.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SCORE-SEQ</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SCORE-type tests
		for detecting disease associations with rare variants in SEQuencing
		Studies</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SCORE-Seq is a
		command-line program for detecting disease associations with rare
		variants in sequencing studies. The mutation information is
		aggregated across multiple variant sites of a gene through a
		weighted linear combination and then related to disease phenotypes
		through appropriate regression models. The weights can be constant
		or dependent on allele frequencies and phenotypes. The association
		testing is based on score-type statistics. The allele-frequency
		threshold can be fixed or variable. Statistical significance can be
		assessed by using asymptotic normal approximation or resampling.
		The current release covers binary and continuous traits with
		arbitrary covariates under case-control and cross-sectional
		sampling.</p>
		<li><p style="margin-bottom: 0.14in">authors: Danyu Lin,
		Zheng-Zheng Tang (Univ North Carolina)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/SCORE-Seq/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lin, Tang (2011),
		&quot;A general framework for detecting disease associations with
		rare variants in sequencing studies&quot;, American Journal of
		Human Genetics, 89:354-367.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SCOUT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.00 (Jan 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: Scout is a
		software program for conducting combined association analysis of
		triads and unrelated subjects</p>
		<li><p style="margin-bottom: 0.14in">authors: Richard Duncan and
		Michael Epstein (Emory University, email:
		statgen@genetics.emory.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genetics.emory.edu/labs/epstein/software/scout/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Fortran
		(CVF 6.6) with IMSL routines</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables: scout_cl.exe,
		scout_gui.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Epstein, Veal,
		Trembath, Barker, Li, Satten (2005), &quot;Genetic association
		analysis using data from triads and unrelated subjects&quot;,
		American Journal of Human Genetics, 76(4):592-608. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SDMINP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Step-Down MIN
		P-value</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (December 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: SDMinP is an
		easy-to-use program for fast calculation of empirical and adjusted
		p-values for correlated and uncorrelated hypotheses in multiple
		testing experiments. It is based on the Free Step-Down Resampling
		Method for controlling the Family Wise Error Rate, originally
		proposed by Westfall and Young (1993), and implements a variation
		of the efficient algorithm of Ge et al. (2003), in which the
		originally necessary re-sampling effort was reduced considerably
		and the method made computationally more feasible. The program is
		independent of the underlying test statistic and works with
		provided observed and permutation test statistics.</p>
		<li><p style="margin-bottom: 0.14in">authors: Markus Obreiter, Lars
		Beckmann (email: l.beckmann@dkfz.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dkfz.de/SDMinP/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Python
		2.3.5</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Obreiter, Fischer,
		Chang-Claude, Beckmann (2005), &quot;SDMinP: a program to control
		the family wise error rate using step-down minP adjusted p-values&quot;,
		Bioinformatics, 21(14):3183-3184.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SEGPATH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SEGregation
		analysis and PATH analysis</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael A. Province
		(email: mike@wubios.wustl.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biostat.wustl.edu/~mike/segpath/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Province, Rao
		(1995), &quot;General purpose model and a computer program for
		combined segregation and path analysis (SEGPATH): automatically
		creating computer programs from symbolic language model
		specifications&quot;, Genetic Epidemiology, 12(2):203-219.</p>
		<li><p style="margin-bottom: 0.14in">Province, Rice, Borecki, Gu,
		Kraja, Rao (2003), &quot;Multivariate and multilocus variance
		components method, based on structural relationships to assess
		quantitative trait linkage via SEGPATH&quot;, Genetic Epidemiology,
		24(2):128-138.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SELSIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1 (June 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: When designing
		methods which provide the necessary power to detect regions of the
		genome which have experience historical selective pressures it is
		important to consider which patterns of genetic diversity are
		indicative of particular forms of natural selection. SELSIM is a
		program which can simulate population genetic data in which a
		single site has experienced natural selection.</p>
		<li><p style="margin-bottom: 0.14in">authors: Chris Spencer (Univ
		of Oxford, email: spencer@stats.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/mathgen/software.html</p>
		<li><p style="margin-bottom: 0.14in">http://www.stats.ox.ac.uk/~spencer/SelSim/Controlfile.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(XP), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Spencer, Coop
		(2004), &quot;SelSim: a program to simulate population genetic data
		with natural selection and recombination&quot;, Bioinformatics,
		20:3673-3675.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SEQUENCE LD/SEQUENCE LDHOT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		sequenceLD analyzes sequence data. It obtains an approximation to
		the likelihood of a summary of the data (as such it can be thought
		of as a marginal likelihood approach). It does not use all the
		information in the data, but computationally it can be
		substantially more efficient than the full-likelihood methods (and
		hence able to analyze larger data sets).</p>
		<li><p style="margin-bottom: 0.14in">The program sequenceLDhot is a
		method for detecting recombination hotspots from population genetic
		data. It takes as input phased (i.e. haplotype) data, together with
		an estimate of the background recombination rate within the region
		(this is allowed to vary across the region). Then sequenceLDhot
		considers a grid of putative hotspot positions, and for each
		putative hotspot calculates a likelihood ratio statistics for the
		presence of a hotspot.</p>
		<li><p style="margin-bottom: 0.14in">authors: Paul Fearnhead
		(Department of Mathematics and Statistics, Lancaster University,
		UK)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.maths.lancs.ac.uk/~fearnhea/Software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Fearnhead, Donnelly
		(2002), &quot;Approximate likelihood methods for estimating local
		recombination rates&quot;, Journal of the Royal Statistical Society
		series B, 64:657-680.</p>
		<li><p style="margin-bottom: 0.14in">Fearnhead (2006),
		&quot;SequenceLDhot: detecting recombination hotspots&quot;,
		Bioinformatics, 22:3061-3066.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SGS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0d</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Bernd Degen</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://software.bfh-inst2.de/download3.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		VisualBasic</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/00/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Degen, Petit,
		Kremer (2001), &quot;SGS - Spatial Genetic Software: a computer
		program for analysis of spatial genetic and phenotypic structures
		of individuals and populations&quot;, Journal of Heredity,
		92(5):447-448.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIBERROR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SibError</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (November 20,
		1997)</p>
		<li><p style="margin-bottom: 0.14in">description: SibError is an
		algorithm that identifies pedigree errors in sibship data. Examples
		include half siblings, unrelated individuals, identical twins, and
		parental exclusions. The test statistic is based on the summation
		of the number of alleles shared by a pair of relatives for a large
		number of markers and the number of alleles and allele frequencies
		for those markers.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Drs. Meg Gelder Ehm
		(mge37216@glaxowellcome.com) and Michael Wagner (
		mw29911@glaxowellcome.com) (Glaxo Wellcome, Inc.)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: siberror</p>
		<li><p style="margin-bottom: 0.14in">reference: Ehm, Wagner (1998),
		&quot;Test statistic to detect errors in sib-pair relationship&quot;,
		American Journal of Human Genetics, 62(1):181-188</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIBLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0</p>
		<li><p style="margin-bottom: 0.14in">description: SIBLINK allows
		the user to perform multipoint linkage analysis based on estimated
		IBD sharing between affected sibpairs. IBD sharing is inferred from
		IBS status, given marker genotypes, frequencies, and locations.
		Resulting LOD scores are maximized across a grid of possible
		disease locations and IBD sharing vectors.</p>
		<li><p style="margin-bottom: 0.14in">authors: Elizabeth Hauser,
		Mike Boehnke, William Duren</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wwwchg.duhs.duke.edu/software/siblink.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: siblink3.0</p>
		<li><p style="margin-bottom: 0.14in">reference: ER Hauser, M
		Boehnke, SW Guo, N Risch (1996), Genetics Epidemiology. 13:117-137.</p>
		<li><p style="margin-bottom: 0.14in">ER Hauser, M Boehnke M (1998),
		Biometrics, 54:1238-1246.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIBMED</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SIBling Mutation
		and Error Detection</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: SIBMED identifies
		likely genotyping errors and mutations for a sib pair in the
		context of multipoint mapping.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Julie A Douglas,
		Michael Boehnke (Univ Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://csg.sph.umich.edu/boehnke/sibmed.php</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN
		77</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: sibmed.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Douglas, Boehnke,
		Lange (2000), &quot;A multipoint method for detecting genotyping
		errors and mutations in sibling-pair linkage data&quot;, American
		Journal of Human Genetics, 66(4):1287-1297. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIB-PAIR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.00a (January 2007)</p>
		<li><p style="margin-bottom: 0.14in">description: Program SIB-PAIR
		performs a number of simple analyses of family data that tend to be
		&quot;nonparametric&quot; or &quot;robust&quot; in nature, includes
		IBD and IBS based APM, Haseman-Elston sib pair, TDT and association
		analyses.</p>
		<li><p style="margin-bottom: 0.14in">Authors: David Duffy
		(Queensland Institute of Med Res, Australia, davidd@qimr.edu.au)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www2.qimr.edu.au/davidD</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		Fortran77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am. Soc. Hum.
		Genet. Annual Meeting 1997 (Duffy, Am.J. Hum. Genet. suppl. 61,
		A197 (1997)); online manual page</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIBSIM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1.2 (November 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: Simulates either
		genotype and/or quantitative phenotype data in family structures in
		a modern, easy to use and highly scalable way. It is based on XML,
		completely written in C++ and published under the GNU General
		Public License.</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel Franke
		(University Hospital Schleswig-Holstein)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.imbs.uni-luebeck.de/pub/sibsim/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++, XML</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: D Franke, A
		Kleensang and A Ziegler (2006), &quot;SIBSIM, quantitative
		phenotype simulation in extended pedigrees&quot;, GMS Med Inform
		Biom Epidemiol, 2(1):Doc04.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIFT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Sorting Intolerant
		From Tolerant</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SIFT predicts
		whether an amino acid substitution affects protein function based
		on sequence homology and the physical properties of amino acids.
		SIFT can be applied to naturally occurring nonsynonymous
		polymorphisms and laboratory-induced missense mutations.</p>
		<li><p style="margin-bottom: 0.14in">authors: Pauling Ng (Genome
		Institute of Singapore), Steven Henikoff (Fred Hutchinson Cancer
		Research Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://sift.bii.a-star.edu.sg/</p>
		<li><p style="margin-bottom: 0.14in">http://sift.jcvi.org/</p>
		<li><p style="margin-bottom: 0.14in">http://blocks.fhcrc.org/sift/SIFT.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ng, Henikoff
		(2001), &quot;Predicting deleterious amino acid substitutions&quot;,
		Genome Research, 11:863-874.</p>
		<li><p style="margin-bottom: 0.14in">Ng, Henikoff (2002),
		&quot;Accounting for human polymorphisms predicted to affect
		protein function&quot;, Genome Research, 12:436-446.</p>
		<li><p style="margin-bottom: 0.14in">Ng, Henikoff (2003), &quot;SIFT:
		Predicting amino acid changes that affect protein function&quot;,
		Nucleic Acids Research, 31:3812-3814.</p>
		<li><p style="margin-bottom: 0.14in">Kumar, Henikoff, Ng (2009),
		&quot;Predicting the effects of coding non-synonymous variants on
		protein function using the SIFT algorithm&quot;, Nature Protocols,
		4:1073-1081.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SILCLOD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SIgnificance Levels
		and Critical LODs</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: calculate nominal
		significance levels and critical LOD scores depending on the length
		of the investigated region, number of chromosomes, and the
		cross-over rate. The global significance level as well as the
		precision of the calculation have to be specified.</p>
		<li><p style="margin-bottom: 0.14in">authors: Andreas Ziegler
		(email: koenigir@imbs.uni-luebeck.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.imbs.uni-luebeck.de/pub/silcLOD/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: E Lander, L
		Kruglyak (1995), &quot;Genetic dissection of complex traits:
		guidelines for interpreting and reporting linkage results&quot;,
		Nature Genetics, 11:241-247.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMCOAL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SIMulate
		COAlescence</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Laurent Excoffier
		(email: laurent.excoffier@zoo.unibe.ch)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://cmpg.unibe.ch/software/simcoal/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(9X/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables: simcoal, sim_coal</p>
		<li><p style="margin-bottom: 0.14in">reference: Excoffier,
		Novembre, Schneider (2000), &quot;SIMCOAL: a general coalescent
		program for the simulation of molecular data in interconnected
		populations with arbitrary demography&quot;, Journal of Heredity,
		91:506-509.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMHAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SIMulation-based
		HAPlotypic analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.2 (January 2009)</p>
		<li><p style="margin-bottom: 0.14in">description: SimHap uses
		biallelic SNP genotype data to impute haplotype frequencies at the
		individual level. SimHap also tests for haplotype associations with
		outcomes of interest while incorporating the uncertainty around
		inferred haplotypes into the modelling procedure. SimHap allows
		epidemiological (ie, non-genetic) and both single SNP and haplotype
		association analyses of quantitative Normal, binary, longitudinal
		and right-censored outcomes under a range of genetic models. SimHap
		can accommodate large data sets, and can model genetic and
		environmental effects, including complex haplotype:environment
		interactions. SimHap features cross-platform functionality via
		Java, and a sophisticated graphical user interface (GUI). SimHap
		will also perform association analysis on more simple
		epidemiological models, with or without the inclusion of genetic
		covariates.</p>
		<li><p style="margin-bottom: 0.14in">authors: Pamela A. McCaskie
		(Univ. of Western Australia), Kim W Carter, Lyle J Palmer</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genepi.org.au/simhap</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java
		(1.5)</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Carter, McCaskie,
		Palmer (2008), &quot;SimHap GUI: An intuitive graphical user
		interface for genetic association analysis&quot;, BMC
		Bioinformatics, 9:557</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMIBD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.21 (Jan 1997)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Sean Davis (
		davis@moriarty.hgen.pitt.edu) and Daniel E. Weeks</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register/soft_doc.html,
		http://watson.hgen.pitt.edu/~davis/</p>
		<li><p style="margin-bottom: 0.14in">http://watson.hgen.pitt.edu/register</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS/Solaris/HP/DEC-UNIX)</p>
		<li><p style="margin-bottom: 0.14in">executables: simibd, xslink</p>
		<li><p style="margin-bottom: 0.14in">reference: Davis, Schroeder,
		Goldin, Weeks (1996), &quot;Nonparametric simulation-based
		statistics for detecting linkage in general pedigrees&quot;,
		American Journal of Human Genetics, 58(4):867-880</p>
		<li><p style="margin-bottom: 0.14in">online README file</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMLA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SIMulation of
		pedigree data for Linkage and Association studies</p>
		<li><p style="margin-bottom: 0.14in">version: 2.3 (Oct 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: SIMLA is a
		SIMulation program that generates data sets of families for use in
		Linkage and Association studies. It allows the user flexibility in
		specifying marker and disease placement, locus heterogeneity,
		disequilibrium between markers and between markers and disease
		loci. Output is in the form of a LINKAGE pedigree file and is
		easily utilized, either directly or with minimal reformatting, as
		input for various genetic analysis packages</p>
		<li><p style="margin-bottom: 0.14in">authors: Elizabeth Hauser,
		Eden Martin, Meredyth Bass</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wwwchg.duhs.duke.edu/software/simla.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: simla2.3</p>
		<li><p style="margin-bottom: 0.14in">reference: MP Bass, ER Martin,
		ER Hauser(2002), Am J Hum Genet, suppl 71:569.</p>
		<li><p style="margin-bottom: 0.14in">MP Bass, ER Martin, ER Hauser
		(2004), &quot;Pedigree generation for analysis of genetic linkage
		and association&quot;, Proceedings of the Pacific Symposium on
		Biocomputing, pp.93-103.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 4.12 (April 1997)</p>
		<li><p style="margin-bottom: 0.14in">description: a computer
		program to estimate the probability (&quot;power&quot;) of
		detecting linkage given family history information on a set of
		identified pedigrees.</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Boehnke (
		boehnke@sph.umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://csg.sph.umich.edu/boehnke/simlink.php</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		UNIX(SunOS/..), VMS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Boehnke (1986),
		&quot;Estimating the power of a proposed linkage study: a practical
		computer simulation approach&quot;, American Journal of Human
		Genetics, 39(4):513-527</p>
		<li><p style="margin-bottom: 0.14in">Ploughman, Boehnke (1989),
		&quot;Estimating the power of a proposed linkage study for a
		complex genetic trait&quot;, American Journal of Human Genetics,
		44(4):543-551</p>
		<li><p style="margin-bottom: 0.14in">online documentation (version
		4.1, april 1990)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: A gene dropping
		simulation software</p>
		<li><p style="margin-bottom: 0.14in">authors: Mathieu Lemire
		(email:mlemire@sdf.lonestar.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mlemire.freeshell.org/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lemire, Roslin,
		Laprise, Hudson, Morgan (2004), &quot;Transmission-ratio distortion
		and allele sharing in affected sib pairs: a new linkage statistic
		with reduced bias, with application to chromosome 6q25.3&quot;,
		American Journal of Human Genetics, 75:571-586.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMPED</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SimPed is a
		program that quickly generates haplotypes and/or genotype data for
		a large number of marker loci (&gt;20,000) for pedigrees of
		virtually any size and complexity. Haplotypes and/or genotypes are
		generated using user specified genetic map distances and haplotypes
		and/or allele frequencies.</p>
		<li><p style="margin-bottom: 0.14in">authors: SM Leal, K Yan, B
		M&uuml;ller-Myhsok</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hgsc.bcm.tmc.edu/genemapping</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Window, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Leal, Yan,
		Muller-Myhsok (2005), &quot;SimPed: a simulation program to
		generate haplotype and genotype data for pedigree structures&quot;,
		Human Heredity, 60(2):119-122.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMPLE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Sequential
		Imputation for MultiPoint Linkage Estimation</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Calculates
		linkage statistics, such as lod scores and NPL statistics by
		Sequential Imputation.</p>
		<li><p style="margin-bottom: 0.14in">authors: S Lin, Z Skrivanek, M
		Irwin (Ohio State Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.ohio-state.edu/~statgen/SOFTWARE/SIMPLE/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables: simple, scan</p>
		<li><p style="margin-bottom: 0.14in">reference: M Irwin, N Cox, A
		Kong (1994), &quot;Sequential imputation for multilocus linkage
		analysis&quot;, Proceedings of the National Academy of Sciences ,
		91:1684-1688</p>
		<li><p style="margin-bottom: 0.14in">Z Skrivanek, S Lin, ME Irwin
		(2003), &quot;Linkage Analysis with Sequential Imputation&quot;,
		Genetic Epidemiology, 25:25-35.</p>
		<li><p style="margin-bottom: 0.14in">S Lin, Z Skrivanek, M Irwin
		(2003), &quot;Haplotyping using SIMPLE - caution on ignoring
		interference&quot;, Genetic Epidemiology, 25:384-387.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMULA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.2, 3.3 with graphic
		user interface</p>
		<li><p style="margin-bottom: 0.14in">description: SIMLA is a
		simulation program that generates data sets of families for use in
		linkage and association studies. SIMLA_3.2 is a major upgrade to
		versions 2.3 and 3.0 that provides the ability to simulate two
		disease loci and two environmental covariates. Gene-gene and
		gene-environment interactions may also be simulated which jointly
		determine the disease risk of all pedigree members.</p>
		<li><p style="margin-bottom: 0.14in">The SIMLA3.3 with Graphical
		User Interface (GUI) is designed for users to set up the SIMLA
		parameters conveniently. The GUI assists in creating a SIMLA
		control file based on the parameters specified on the GUI panels.
		It includes the SIMLA 3.2 program version.</p>
		<li><p style="margin-bottom: 0.14in">authors: email:
		simla@chg.duhs.duke.edu</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.chg.duke.edu/research/simla30.html</p>
		<li><p style="margin-bottom: 0.14in">http://wwwchg.duhs.duke.edu/research/simla3_3gui.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java, R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Schmidt, Hauser,
		Martin, Schmidt (2005), &quot;Extension of the SIMLA package for
		generating pedigrees with complex inheritance patterns:
		Environmental covariates, gene-gene and gene-environment
		interaction&quot;, Statistical Applications in Genetics and
		Molecular Biology, 4(1):15.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMULAPLOT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: SIMLAPLOT is a
		tool designed to help visualize the joint effect of genes and
		continuous environmental covariates on complex human disease
		simulation models. SIMLAPLOT takes as input the set of genetic
		model parameters used to generate the data, including risk ratios
		for genotypes (G), a normally distributed environmental covariate
		(E) and their interaction (GxE), or the risk ratio for a continuous
		covariate that is itself influenced by a quantitative trait locus
		(QTL) and thus follows a mixture normal distribution. It is also
		possible to use SIMLAPLOT to plot observed data distributions, if
		estimates of these parameters can be provided.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.chg.duke.edu/research/simlaplot.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMULATE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a program to
		simulate genotypes in family members for a map of linked markers
		unlinked to a given affection status locus. the output is ready for
		analysis with UNKNOWN, ISIM, LSIM, or MSIM of the SLINK package.</p>
		<li><p style="margin-bottom: 0.14in">authors: Joseph Terwilliger,
		http://linkage.cpmc.columbia.edu/</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://www.helsinki.fi/~tsjuntun/linkage/analyze/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		VMS, UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: online
		documentation</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMUPOP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.6.0 (Jan 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: simuPOP is a
		forward-based population genetics simulation program. It is capable
		of simulating very complex evolution processes on large (think of
		millions) populations. Major features include variable population
		size; many built-in and hybrid (write in python) mutation,
		migration, selection models. simuPOP can be extended in Python so
		there is no limit on what you can do with it.</p>
		<li><p style="margin-bottom: 0.14in">authors: Bo Peng
		(email:bpeng@rice.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bp6.stat.rice.edu:8080/simuPOP/</p>
		<li><p style="margin-bottom: 0.14in">http://simupop.sourceforge.net/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++
		(provided as Python libraries)</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Peng, Kimmel
		(2005), &quot;simuPOP: a forward-time population genetics
		simulation environment&quot;, Bioinformatics, 21(18):3686-3687.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SIMWALK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">alias: SIMWALK/SIMCROSS</p>
		<li><p style="margin-bottom: 0.14in">version: SIMCROSS: June 1995;
		SIMWALK2: 2.91 (December 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: programs for
		generating optimal haplotype configurations on general pedigrees
		using a likelihood-based approach to correctly take intermarker
		recombination fractions into account. simcross ignores untyped
		parts of the pedigree, and it uses simulated annealing. simwalk
		combines simulated annealing with random walk method.</p>
		<li><p style="margin-bottom: 0.14in">authors: Eric Sobel (UCLA.
		email: simwalk@genetics.ucla.edu (in collaboration with Kenneth
		Lange, Jeffrey O'Connell, and Daniel E. Weeks)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genetics.ucla.edu/software/simwalk</p>
		<li><p style="margin-bottom: 0.14in">registration page at:
		http://www.genetics.ucla.edu/software/registration or,
		http://watson.hgen.pitt.edu/register/</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS/Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Weeks, Sobel,
		O'Connell, Lange (1995), &quot;Computer programs for multilocus
		haplotyping of general pedigrees&quot;, American Journal of Human
		Genetics, 56(6):1506-1507</p>
		<li><p style="margin-bottom: 0.14in">Sobel, Lange, O'Connell, Weeks
		(1996), in Genetic Mapping and DNA Sequencing , Vol 81 in The IMA
		Volumes in Mathematics and its Applications, eds TP Speed, MS
		Waterman (Springer-Verlag), pages 49-63.</p>
		<li><p style="margin-bottom: 0.14in">Sobel, Lange (1996), &quot;Descent
		graphs in pedigree analysis: applications to haplotyping, location
		scores, and marker sharing statistics&quot;, American Journal of
		Human Genetics, 58(6):1323-1337</p>
		<li><p style="margin-bottom: 0.14in">Sobel, Sengul, Weeks (2001),
		Human Heredity, 52:121-131.</p>
		<li><p style="margin-bottom: 0.14in">Sobel, Papp, Lange (2002),
		&quot;Detection and integration of genotyping errors in statistical
		genetics&quot;, American Journal of Human Genetics, 70(2):496-508.
		[ html]</p>
		<li><p style="margin-bottom: 0.14in">SIMWALK manual</p>
		<li><p style="margin-bottom: 0.14in">FAQ-O-MATIC:
		http://watson.hgen.pitt.edu/fom-serve/cache/20.html</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SKAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SNP-set (Sequence)
		Kernel Association Test</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SKAT is a SNP-set
		(e.g., a gene or a region) level test for association between a set
		of rare (or common) variants and dichotomous or quantitative
		phenotypes. SKAT aggregates individual score test statistics of
		SNPs in a SNP set and efficiently computes SNP-set level p-values,
		e.g. a gene or a region level p-value, while adjusting for
		covariates, such as principal components to account for population
		stratification. SKAT also allows for power/sample size calculations
		for designing for sequence association studies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Xihong Lin, Seunggeun
		Lee (Harvard Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hsph.harvard.edu/research/skat/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wu, Lee, Cai, Li,
		Boehnke, Lin (2011), &quot;Rare variant association testing for
		sequencing data using the sequence kernel association test (SKAT)&quot;,
		American Journal of Human Genetics, 89:82-93.</p>
		<li><p style="margin-bottom: 0.14in">Wu, Kraft, Epstein, Taylor,
		Chanock, Hunter, Lin (2010), &quot;Powerful SNP set analysis for
		case-control genomeWide association studies&quot;, American Journal
		of Human Genetics , 86:929-942.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SLINK (see also FASTSLINK)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel E. Weeks, Jurg
		Ott</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: slink, isim,
		lsim, msim</p>
		<li><p style="margin-bottom: 0.14in">reference: Ott (1989),
		Proceedings of the National Academy of Sciences , 86:4175-4178.</p>
		<li><p style="margin-bottom: 0.14in">Weeks, Ott, Lathrop (1990),
		American Journal of Human Genetics, 47(suppl):A204.</p>
		<li><p style="margin-bottom: 0.14in">online documentation</p>
		<li><p style="margin-bottom: 0.14in">FAQ-O-MATIC:
		http://watson.hgen.pitt.edu/fom-serve/cache/20.html</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SMOOTH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2008</p>
		<li><p style="margin-bottom: 0.14in">description: The construction
		of accurate linkage maps is not so much depending on the quality of
		the mapping software, but mostly on the marker data quality.
		Missing values and scoring errors can severely influence the
		calculated marker order. The software tool SMOOTH recognises and
		removes the most unrealistic data points. This software was used to
		construct the 10,000 marker potato map. The removal of improbable
		data point is a good medicine for linkage maps, that is not easily
		overdosed. One error is more harmfull than ten missing values. The
		software was never intended as user-friendly software. In these
		days it would be more useful to re-do the programming of the pascal
		source code into a perl script. Anyone who takes the initiative to
		generate such a script is welcomed to contact the authors. SMOOTH
		works best in close cooperation with mapping algorithm RECORD</p>
		<li><p style="margin-bottom: 0.14in">authors: Hans van Os, Herman
		van Eck (Laboratory for Plant Breeding, Wageningen University,
		email: herman.vaneck@wur.nl)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.plantbreeding.wur.nl/UK/software_smooth.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: smooth7.exe</p>
		<li><p style="margin-bottom: 0.14in">reference: Van Os, Stam,
		Visser, Van Eck (2005), &quot;SMOOTH: a statistical method for
		successful removal of genotyping errors from high-density genetic
		linkage data&quot;, Theoretical and Applied Genetics,
		112(1):187-194</p>
		<li><p style="margin-bottom: 0.14in">Van Os, Stam, Visser, Van Eck
		(2005), &quot;RECORD: A novel method for ordering loci on a genetic
		linkage map&quot;, Theoretical and Applied Genetics, 112(1):30-40</p>
		<li><p style="margin-bottom: 0.14in">Van Os, Andrzejewski, Bakker,
		Barrena, Bryan, Caromel, Ghareeb, Isidore, de Jong, van Koert,
		Lefebvre, Milbourne, Ritter, van der Voort, Rousselle-Bourgeois,
		van Vliet, Waugh, Visse, Bakke, Van Eck (2006), &quot;Construction
		of a 10,000-marker ultradense genetic recombination map of potato:
		Providing a framework for accelerated gene isolation and a
		genomewide physical map&quot;, Genetics, 173(2):1075-1087</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Suite of Nucleotide
		Analysis Programs</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The exploration
		of DNA sequence variation for making inferences on evolutionary
		processes in populations requires the coordinated implementation of
		a Suite of Nucleotide Analysis Programs (SNAP), each bound by
		specific assumptions and limitations. We have developed a workbench
		tool to make existing population genetic software more accessible
		and to facilitate the integration of new tools for analyzing
		patterns of DNA sequence variation, within a phylogenetic context.
		Collectively, SNAP tools can serve as a bridge between theoretical
		and applied population genetic analysis.</p>
		<li><p style="margin-bottom: 0.14in">authors: David Aylor, Eric
		Price, Ignazio Carbone (North Carolina State Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.cals.ncsu.edu/plantpath/people/faculty/carbone/snap.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Aylor, Price,
		Carbone (2006), &quot;SNAP: Combine and Map modules for multilocus
		population genetic analysis&quot;, Bioinformatics,
		22(11):1399-1401.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNAP (2)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SNp Annotation and
		Proxy search</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (August 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: SNAP finds proxy
		SNPs based on linkage disequilibrium, physical distance and/or
		membership in more than 15 selected commercial genotyping arrays.
		Pair-wise linkage disequilibrium is pre-calculated based on phased
		genotype data from the International HapMap Project . Information
		about the genotyping arrays is based on data published by the
		vendors. SNAP also optionally returns current gene and SNP
		annotations, and can be used to generate graphical plots of LD
		structure, recombination rate and gene annotation.</p>
		<li><p style="margin-bottom: 0.14in">authors: Robert Handsaker
		(email: handsake@broad.mit.edu), Andrew Johnson (email:
		johnsonad2@nhlbi.nih.gov), Sara Pulit, Marcia Nizzari, Chris
		O'Donnell, Paul de Bakker (email: debakker@broad.mit.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broad.mit.edu/mpg/snap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R,
		Oracle</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNAP (3)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Simulation of sNp
		haplotype data And Phenotypic traits</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The SNaP program
		can be used to generate SNP haplotype sequence data of unrelated
		individuals and nuclear families with a fixed or random number of
		children.</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Nothnagel
		(email: nothnagel@medinfo.uni-kiel.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://capella.uni-kiel.de/snap/snap.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		UNIX(Solaris), MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am Soc Hum Genet
		annual meeting (2002) Nothnagel (2002), &quot;Simulation of LD
		block-structured SNP haplotype data and its use for the analysis of
		case-control data by supervised learning methods&quot;, Am J. Hum
		Genet, 71(4)(suppl), A2363.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNIPPEEP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SnipPeep is an
		interactive graphic interface to visualise results from whole
		genome genotyping. It allows one to visualise single subjects and
		groups of subjects, and provides a direct connection with the UCSC
		Genome Browser.</p>
		<li><p style="margin-bottom: 0.14in">authors: Roberto Toro, (email:
		rto@pasteur.fr), Thomas Bourgeron (email: thomasb@pasteur.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://snippeep.sourceforge.net/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPALYZE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 6.0 (December 2006)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: dynacom.co.jp</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dynacom.co.jp/e/products/package/snpalyze/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(98/Me/NT4.0/2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Shimo-Onoda,
		Tanaka, Furushima, Nakajima, Toh, Harata, Yone, Komiya, Adachi,
		Nakamura, Fujimiya, Inoue(2002), &quot;Akaike's information
		criterion for a measure of linkage disequilibrium&quot;, J Hum
		Genet, 47(12): 649-655.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNP ASSISTANT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0.11 (April 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: SNP data
		managing, import &amp; export from linkage format, data validation,
		pairwise LD calculation and visualisation, case-control and TDT
		tests, visual comparison of two datasets, relationships testing.
		Suitable for large projects.</p>
		<li><p style="margin-bottom: 0.14in">authors: Biodata Ltd., Estonia
		(snpassistant@biodata.ee)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.biodata.ee/SNPassistant.htm (30 days trial version)</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		MS-Windows(95 and newer)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: online manual:
		http://www.biodata.ee/sa/SNP_Assistant_Manual_1.0.7.pdf</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNP CHART</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: The purpose of
		this software is to generate visual patterns that can be easily
		interpreted as a single specific Single Nucleotide Polymorphism
		(SNP) genotype, based on reiterative pattern recognition from
		validated SNP data</p>
		<li><p style="margin-bottom: 0.14in">authors: Scott Tebbutt (email:
		scott.tebbutt@hli.ubc.ca)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.flintbox.com/public/project/500/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Tebbutt, Opushnyev,
		Tripp, Kassamali, Alexander, Andersen (2005), &quot;SNP Chart: an
		integrated platform for visualization and interpretation of
		microarray genotyping data&quot;, Bioinformatics, 21(1):124-127.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPEM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SNP Expectation
		Maximization</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: EM algorithm
		estimations of SNP haplotypes</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://polymorphism.ucsd.edu/snpem/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPFILE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SNPFile is a
		library and API for manipulating large SNP datasets with associated
		meta-data, such as marker names, marker locations, individuals'
		phenotypes, etc. in an I/O efficient binary file format. In its
		core, SNPFile assumes very little about the metadata associated
		with markers and individuals, but leaves this up to application
		program protocols.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.daimi.au.dk/~mailund/SNPFile/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux, UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Nielsen, Mailund
		(2008), &quot;SNPFile -- A software library and file format for
		large scale association mapping and population genetics studies&quot;,
		BMC Bioinformatics, 9:526.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPHAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.3.1</p>
		<li><p style="margin-bottom: 0.14in">description: a program for
		estimating frequencies of haplotypes of large numbers of diallelic
		markers from unphased genotype data from unrelated subjects</p>
		<li><p style="margin-bottom: 0.14in">Authors: David Clayton
		(Cambridge Institute, david.clayton@cimr.cam.ac.uk</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-gene.cimr.cam.ac.uk/clayton/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPHARVESTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a powerful tool
		for detecting epistatic interactions in genome-wide association
		studies</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinformatics.ust.hk/SNPHarvester.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yang, He, Wan,
		Yang, Xue, Yu (2009), &quot;SNPHarvester: a filtering-based
		approach for detecting epistatic interactions in genome-wide
		association studies&quot;, Bioinformatics, 25:504-511.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNP HITLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SNP HIgh-Throughput
		LINKage analysis system</p>
		<li><p style="margin-bottom: 0.14in">version: 5.1 (May 2010)</p>
		<li><p style="margin-bottom: 0.14in">description: SNP HiTLink is a
		computer program providing a useful pipeline to directly connect
		SNP data and linkage analysis program. SNP HiTLink currently
		supports the data from SNP chips provided by Affymetrix (Mapping
		100k/500k array set, Genome-Wide Human SNP array 5.0/6.0) and
		Illumina (recently supported), carrying out typical linkage
		analysis programs of MLINK (FASTLINK/ LINKAGE package), Superlink,
		Merlin and Allegro.</p>
		<li><p style="margin-bottom: 0.14in">authors: info@dynacom.co.jp</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dynacom.co.jp/u-tokyo.ac.jp/snphitlink/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Fukuda, Nakahara,
		Date, Takahashi, Goto, Miyashita, Kuwano, Adachi, Nakamura, Tsuji
		(2009), &quot;SNP HiTLink: a high-throughput linkage analysis
		system employing dense SNP data&quot;, BMC Bioinformatics, 10:121.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNP-HWE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a fast exact
		Hardy-Weinberg Equilibrium test for SNPs</p>
		<li><p style="margin-bottom: 0.14in">authors: Janis Wigginton
		(email: &quot;a href=&quot;mailto:wiggie@umich.edu&quot;&gt;
		wiggie@umich.edu) and Goncalo Abecasis</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/abecasis/Exact/index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Fortran,
		C,C++, R</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wigginton, Cutler,
		Abecasis (2005), &quot;A note on exact tests of Hardy-Weinberg
		equilibrium&quot;, American Journal of Human Genetics,
		76(5):887-893. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Multipoint
		linkage analysis of densely distributed SNP data incorporating
		automated linkage disequilibrium removal. SNPLINK requires these
		other programs installed on the system: MERLIN (used for
		nonparametric analysis), ALLEGRO (used for parametric analysis), R
		and PERL, all are freely available.</p>
		<li><p style="margin-bottom: 0.14in">authors: Emily Webb (The
		Institute for Cancer Research, email: emily.webb@icr.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.icr.ac.uk/cancgen/molgen/MolPopGen_Bioinformatics.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Webb, Sellick,
		Houlston (2005), &quot;SNPLINK: multipoint linkage analysis of
		densely distributed SNP data incorporating automated linkage
		disequilibrium removal&quot;, Bioinformatics, 21:3060-3061.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPMSTAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SNP Missing data
		STATistics</p>
		<li><p style="margin-bottom: 0.14in">version: 4.0</p>
		<li><p style="margin-bottom: 0.14in">description: SNPMStat is a
		command-line program for the statistical analysis of SNP-disease
		association in case-control/cohort/cross-sectional studies with
		potentially missing genotype data. SNPMStat allows the user to
		estimate or test SNP effects and SNP-environment interactions by
		maximizing the (observed-data) likelihood that properly accounts
		for phase uncertainty, study design and gene-environment
		dependence. For SNPs without missing data, the program performs the
		standard association analysis. For typed SNPs with missing data or
		untyped SNPs, the program performs the maximum-likelihood analysis.</p>
		<li><p style="margin-bottom: 0.14in">authors: Danyu Lin's group
		(Univ North Carolina)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/SNPMStat/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lin, Hu, Huang
		(2008), &quot;Simple and efficient analysis of SNP-disease
		association with missing genotype data&quot;, American Journal of
		Human Genetics, 82:444-452.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Single Nucleotide
		Polymorphism Processor</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (2003)</p>
		<li><p style="margin-bottom: 0.14in">description: a dynamic general
		database management system to manage high-throughput SNP genotyping
		data. It provides several functions, including data importing with
		comparison, Mendelian inheritance check within pedigrees, data
		compiling and exporting. Furthermore, SNPP may generate files for
		repeat genotyping and transform them into files that can be
		executed by a liquid handling system.</p>
		<li><p style="margin-bottom: 0.14in">authors: Lanjuan Zhao (
		lanjuanzhao@creighton.edu), Miaoxin Li ( limx54@163.com), Yanfang
		Guo, Fuhua, Xu, Jinlong Li and Hongwen Deng.</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://orclinux.creighton.edu/snpp/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Windows,
		Linux, Solaris, MacOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao, Li, Guo, Xu,
		Li, Deng (2005), &quot;SNPP: automating large-scale SNP genotype
		data management&quot;, Bioinformatics, 21:266-268</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNP.PLOTTER (see R/SNP.PLOTTER)</p>
	<li><p style="margin-bottom: 0.14in">SNPSTATS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: SNP STATisticS</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SNPStats is a
		simple, ready-to-use software which has been designed to analyze
		genetic-epidemiology studies of association using SNPs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Victor Moreno (email:
		v.moreno@ico.scs.es) Xavier Sole (email: x.sole@ico.scs.es)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinfo.iconcologia.net/snpstats/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sole, Guino, Valls,
		Iniesta, Moreno (2006), &quot;SNPStats: a web tool for the analysis
		of association studies&quot;, Bioinformatics, 22(15):1928-1929.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SNPTEST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1.5 (May 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: SNPTEST is a
		program for the analysis of single SNP association in genome-wide
		studies. The tests implemented can cater for binary (case-control)
		and quantitative phenotypes, can condition upon an arbitrary set of
		covariates and properly account for the uncertainty in genotypes.
		The program is designed to work seamlessly with the output of both
		the genotype calling program CHIAMO, the genotype imputation
		program IMPUTE and the program GTOOL. This program was used in the
		analysis of the 7 genome-wide association studies carried out by
		the Wellcome Trust Case-Control Consortium (WTCCC).</p>
		<li><p style="margin-bottom: 0.14in">authors: Jonathan Marchini</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SOAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Short
		Oligonucleotide Analysis Package</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: SOAP provides
		full solution to next generation sequencing data analysis.
		Currently, it consists of a new alignment tool (SOAPaligner/soap2),
		a re-sequencing consensus sequence builder (SOAPsnp), an indel
		finder (SOAPindel), a structural variation scanner (SOAPsv) and a
		de novo short reads assembler (SOAPdenovo).</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://soap.genomics.org.cn/</p>
		<li><p style="margin-bottom: 0.14in">http://soap.genomics.org.cn/soap3.html
		(SOAP3)</p>
		<li><p style="margin-bottom: 0.14in">http://soap.genomics.org.cn/soapsnp.html
		(SOAP SNP)</p>
		<li><p style="margin-bottom: 0.14in">http://soap.genomics.org.cn/soapindel.html
		(SOAP indel)</p>
		<li><p style="margin-bottom: 0.14in">http://soap.genomics.org.cn/SOAPsv.html
		(SOAP structural variants)</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Li, et al. (2008),
		&quot;SOAP: short oligonucleotide alignment program&quot;,
		Bioinformatics, 24:713-714.</p>
		<li><p style="margin-bottom: 0.14in">Li, Li, Fang, et al. (2009),
		&quot;SNP detection for massively parallel whole-genome
		resequencing&quot;, Genome Research, 19:1124-1132.</p>
		<li><p style="margin-bottom: 0.14in">Li, Yu, Li, Lam, Yiu,
		Kristiansen, Wang (2009), &quot;SOAP2: an improved ultrafast tool
		for short read alignment&quot;, Bioinformatics, 25(15):1966-1967.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SOLAR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Sequential
		Oligogenic Linkage Analysis Routines</p>
		<li><p style="margin-bottom: 0.14in">version: 4.0.7 (December
		2006).</p>
		<li><p style="margin-bottom: 0.14in">description: OLAR is a
		flexible and extensive software package for genetic variance
		components analysis, including linkage analysis, quantitative
		genetic analysis, and covariate screening. Operations are included
		for calculation of marker-specific or multipoint
		identity-by-descent (IBD) matrices in pedigrees of arbitrary size
		and complexity, and for linkage analysis of quantitative traits
		which may involve multiple loci (oligogenic analysis), dominance
		effects, and epistasis.</p>
		<li><p style="margin-bottom: 0.14in">Authors: John Blangero
		(john@darwin.sfbr.org), Kenneth Lange, Laura Almasy
		(almasy@darwin.sfbr.org), Tom Dyer (tdyer@darwin.sfbr.org), Charles
		Peterson (charlesp@darwin.sfbr.org).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sfbr.org/Departments/genetics_detail.aspx?p=37</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN,
		C, C++, TCL</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris 7-10/Solaris x86 8-9/Digital Unix 4.0e/SG Irix),
		Linux, MacOS, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: solar, ibdprep,
		ibdmat, relate, multipnt</p>
		<li><p style="margin-bottom: 0.14in">reference: Blangero, Almasy
		(1996), &quot;SOLAR: sequential oligogenic linkage analysis
		routines&quot;, Population Genetics Laboratory Technical Report No.
		6, Southwest Foundation for Biomedical Research, San Antonio, TX
		78228.</p>
		<li><p style="margin-bottom: 0.14in">Almasy, Blangero (1998),
		&quot;Multipoint quantitative-trait linkage analysis in general
		pedigrees&quot;, American Journal of Human Genetics,
		62(5):1198-1211 (1998)</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SPAM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Statistics Program
		for Analysing Mixtures</p>
		<li><p style="margin-bottom: 0.14in">version: 3.7b</p>
		<li><p style="margin-bottom: 0.14in">description: SPAM estimates
		the relative contributions of discrete populations to a mixture
		sample, solving what is commonly referred to in fisheries as the
		mixed stock analysis or genetic stock identification problem.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genetics.cf.adfg.state.ak.us/software/spampage.php</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Debevec, Gates,
		Masuda, Pella, Reynolds, Seeb (2000), &quot;SPAM (version 3.2):
		statistics program for analyzing mixtures&quot;, Journal of
		Heredity, 91:509-511.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SPECTRAL-GEM (see
	R/SPECTRAL-GEM)</p>
	<li><p style="margin-bottom: 0.14in">SPERM</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: analysis of sperm
		typing data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Kenneth Lange (UCLA),
		Laura Lazzeroni</p>
		<li><p style="margin-bottom: 0.14in">web/ftp: registration page at:
		http://www.biomath.medsch.ucla.edu/faculty/klange/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lazzeroni, Arnheim,
		Schmitt, Lange (1994), &quot;Multipoint mapping calculations for
		sperm-typing data&quot;, American Journal of Human Genetics,
		55(3):431-436</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SPERMSEG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: analysis of
		segregation in single-sperm data</p>
		<li><p style="margin-bottom: 0.14in">Authors: MS McPeek (
		mcpeek@galton.uchicago.eduUniv Chicago)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://galton.uchicago.edu/~mcpeek/software/spermseg</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: MS McPeek (1999),
		&quot;SPERMSEG: analysis of segregation distortion in single-sperm
		data&quot;, American Journal of Human Genetics, 65(4):1195-1197. [
		html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SPIP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Simulate Pedigree
		In Population</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Simulate
		pedigrees and genetic data in age-structured populations</p>
		<li><p style="margin-bottom: 0.14in">authors: Eric C Anderson, K.K.
		Dunham (Southwest Fisherie Science Center)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://swfsc.noaa.gov/staff.aspx?id=740</p>
		<li><p style="margin-bottom: 0.14in">http://swfsc.noaa.gov/textblock.aspx?Division=FED&amp;id=3434</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Anderson, Dunham
		(2005), &quot;SPIP 1.0: a program for simulating pedigrees and
		genetic data in age-structured populations&quot;, Molecular Ecology
		Notes, 5(2):459-461.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SPLAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Sib Pair Linkage
		Analysis Testing</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (2006)</p>
		<li><p style="margin-bottom: 0.14in">description: SPLAT can
		calculate virtually any linkage test statistic under several sib
		pair study designs: affected, discordant, unaffected, and pairs
		defined by threshold values for quantitative traits, such as
		extreme discordant sib pairs. It uses the EM algorithm to compute
		maximum likelihood estimates of sharing (subject to any
		user-specified domain restrictions or null hypotheses) and then
		plots lod scores versus chromosomal position. It includes a novel
		grid scanning capability that enables simultaneous visualization of
		multiple test statistics. Phenotype definitions can be modified
		without recalculating inheritance vectors, thereby providing
		considerable analytical flexibility.</p>
		<li><p style="margin-bottom: 0.14in">authors: Katarzyna Adamska
		(email: adamska@elf.ii.uj.edu.pl), David Poznik (email:
		david.poznik@joslin.harvard.edu), Xin Xu (email:
		xin_xu@harvard.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.joslinresearch.org/LabSites/Krolewski/splat/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++, Qt</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS),Linux, MacOS, MS-Windows(2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Poznik, Adamska,
		Xu, Krolewski, Rogus (2006), &quot;A novel framework for sib pair
		linkage analysis&quot;, American Journal of Human Genetics,
		78(2):222-230. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SPLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: affected Sib Pairs
		LINKage analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 1.09 (Nov 1999)</p>
		<li><p style="margin-bottom: 0.14in">description: linkage analysis
		using affected sib pairs</p>
		<li><p style="margin-bottom: 0.14in">authors: David Clayton
		(Cambridge Institute, david.clayton@cimr.cam.ac.uk</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-gene.cimr.cam.ac.uk/clayton/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: c++</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SunOS/..), MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: splink, ped2spl</p>
		<li><p style="margin-bottom: 0.14in">reference: Holmans (1993),
		&quot;Asymptotic properties of affected-sib-pair linkage analysis&quot;,
		American Journal of Human Genetics, 52(2):362-374</p>
		<li><p style="margin-bottom: 0.14in">Holmans and Clayton (1995),
		submitted.</p>
		<li><p style="margin-bottom: 0.14in">online manual</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SPREG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Secondary Phenotype
		REGression analysis</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: SPREG is a
		computer program for performing regression analysis of secondary
		phenotype data in case-control association studies. Secondary
		phenotypes are quantitative or qualitative traits other than the
		case-control status. Because the case-control sample is not a
		random sample of the general population, standard statistical
		analysis of secondary phenotype data can yield very misleading
		results. SPREG implements valid and efficient statistical methods.</p>
		<li><p style="margin-bottom: 0.14in">authors: Danyu Lin, Donglin
		Zeng (Univ North Carolina)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/SPREG/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lin, Zeng (2009),
		&quot;Proper analysis of secondary phenotype data in case-control
		association studies&quot;, Genetic Epidemiology, 33:256-265.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SQTDT/SPDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: semiparametric
		family-based association</p>
		<li><p style="margin-bottom: 0.14in">authors: Guoqing Diao (email:
		gdiao@gmu.edu), Danyu Lin</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/SQTDT/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Diao, Lin (2006),
		&quot;Improving the power of association tests for quantitative
		traits in family studies&quot;, Genetic Epidemiology, 30:301-313.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Semiparametric QTL
		mapping in general pedigrees</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Guoqing Diao (email:
		gdiao@gmu.edu), Danyu Lin</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/SQTL/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Diao, Lin (2005),
		&quot;A powerful and robust method for mapping quantitative trait
		loci in general pedigrees&quot;, American Journal of Human
		Genetics, 77(1):97-111. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SSAHASNP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Sequence Search and
		Alignment by Hashing Algorithm for SNP detection</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: ssahaSNP is a
		polymorphism detection tool. It detects homozygous SNPs and indels
		by aligning shotgun reads to the finished genome sequence. Highly
		repetitive elements are filtered out by ignoring those kmer words
		with high occurrence numbers. For those less repetitive or
		non-repetitive reads, we place them uniquely on the reference
		genome sequence and find the best alignment according to the
		pair-wise alignment score if there are multiple seeded regions.
		From the best alignment, SNP candidates are screened, taking into
		account the quality value of the bases with variation as well as
		the quality values in the neighbouring bases, using neighbourhood
		quality standard (NQS). For insertions/deletions, we check if the
		same indel is mapped by more than one read, ensuring the detected
		indel with high confidence.</p>
		<li><p style="margin-bottom: 0.14in">authors: Adam Spargo, Zemin
		Ning (Wellcome Trust Sanger Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sanger.ac.uk/resources/software/ssahasnp/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">START</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Finds starting
		points for MCMC analysis performed on large, complex pedigrees and
		polymorphic markers.</p>
		<li><p style="margin-bottom: 0.14in">authors: Y Luo, S Lin (Ohio
		State Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.ohio-state.edu/~statgen/SOFTWARE/START/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Y Luo, S Lin
		(2003), &quot;Finding starting points for Markov chain Monte Carlo
		analysis of genetic data from large and complex pedigrees&quot;,
		Genetic Epidemiology, 25:14-24.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">STEPC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: STatistical
		Explanation for Positional Cloning</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Lei Sun, Nancy COx,
		Mary Sara McPeek</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://hg-wen.uchicago.edu/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sun, Cox, McPeek
		(2002) &quot;A statistical method for identification of
		polymorphisms that explain a linkage result&quot;, American Journal
		of Human Genetics, 70(2):399-411. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">STEPWISE (see R/STEPWISE)</p>
	<li><p style="margin-bottom: 0.14in">STRAT (see also STRUCTURE)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1</p>
		<li><p style="margin-bottom: 0.14in">description: a companion
		program to structure. This is a structured association method, for
		use in association mapping, enabling valid case-control studies
		even in the presence of population structure.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jonathan Pritchard
		(email: pritch@genetics.uchicago.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pritch.bsd.uchicago.edu/software/STRAT.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		MS-Windows,UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: JK Pritchard, M
		Stephens, NA Rosenberg, P Donnelly (2000), &quot;Association
		mapping in structured populations&quot;, American Journal of Human
		Genetics, 67(1):170-181. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">STRUCTURE (see also STRAT)</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.1 (August 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: using multi-locus
		genotype data to investigate population structure. Its uses include
		inferring the presence of distinct populations, assigning
		individuals to populations, studying hybrid zones, identifying
		migrants and admixed individuals, and estimating population allele
		frequencies in situations where many individuals are migrants or
		admixed. It can be applied to most of the commonly-used genetic
		markers, including microsatellites, RFLPs and SNPs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jonathan Pritchard
		(email: pritch@genetics.uchicago.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pritch.bsd.uchicago.edu/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS,
		MS-Windows, UNIX(Solaris), Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: JK Pritchard, M
		Stephens, P Donnelly (2000), &quot;Inference of population
		structure using multilocus genotype data&quot;, Genetics 155:
		945-959.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SUMSTAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: assess the joint
		disease association of multiple unlinked SNPs via sums of SNP
		specific test statistics. Genome-wide significance levels are
		obtained by per mutation analysis.</p>
		<li><p style="margin-bottom: 0.14in">authors: Josephine Hoh, Jurg
		Ott (email: ottjurg@yahoo.com)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: Free
		Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Hoh, Wille, Ott
		(2001), &quot;Trimming, weighting, and grouping SNPs in human
		case-control association studies&quot;, Genome Research,
		11:2115-2119</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SUP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Slink Utility
		Program</p>
		<li><p style="margin-bottom: 0.14in">version: 060629</p>
		<li><p style="margin-bottom: 0.14in">description: an extension to
		SLINK/FastSLINK to allow more marker loci to be simulated in
		pedigrees conditional on trait values and in linkage equilibrium or
		disequilibrium with a trait locus.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mathieu Lemire
		(email: mlemire@sdf.lonestar.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://mlemire.freeshell.org/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		LINUX, CYGWIN</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lemire (2006),
		&quot;SUP: An extension to SLINK to allow a larger number of marker
		loci to be simulated in pedigrees conditional on trait values&quot;,
		BMC Genetics, 7:40.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SUPERLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.6 (March 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: the program
		performs exact linkage analysis with the same input-output
		relationships as in standard genetic linkage programs such as
		LINKAGE, FASTLINK, VITESSE, but can run larger files than previous
		programs.</p>
		<li><p style="margin-bottom: 0.14in">authors: Ma'ayan Fishelson
		(email: fmaayan@cs.technion.ac.il)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://bioinfo.cs.technion.ac.il/superlink/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, Linux, MacOS X</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: M Fishelson, D
		Geiger (2002), Bioinformatics, 18(suppl 1):S189-S198.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SVA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Sequence Variant
		Analyzer</p>
		<li><p style="margin-bottom: 0.14in">version: v1.10 (March 2011)</p>
		<li><p style="margin-bottom: 0.14in">description: SVA is a software
		tool that assigns a predicted biological function to variants
		identified in next-generation sequencing studies and provides a
		browser to visualize the variants in their genomic contexts. SVA
		also provides for flexible interaction with software implementing
		variant association tests allowing users to consider both the
		bioinformatic annotation of identified variants and the strength of
		their associations with studied traits.</p>
		<li><p style="margin-bottom: 0.14in">authors: Dongliang Ge, David
		Goldstein (Duke)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.svaproject.org/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Ge,Ruzzo,
		Shianna,He,Pelak, Heinzen, Need, Cirulli, Maia, Dickson, Zhu,Singh,
		Allen, Goldstein (2011), &quot;SVA: software for annotating and
		visualizing sequenced human genomes&quot;, Bioinformatics,
		27:1998-2000.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SVCC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Semiparametric
		Variance-Component for Censored data</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Guoqing Diao (email:
		gdiao@gmu.edu), Danyu Lin</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/SVCC/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Diao, Lin (2006),
		&quot;Semiparametric variance-component models for linkage and
		association analyses of censored trait data&quot;, Genetic
		Epidemiology, 30:570-581.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SWEEP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: December 2005</p>
		<li><p style="margin-bottom: 0.14in">description: SWEEP allows
		large-scale analysis of haplotype structure in genomes for the
		primary purpose of detecting evidence of natural selection.
		Primarily, it uses the Long Range Haplotype test to look for
		alleles of high frequency with long-range linkage disequilibrium,
		which suggest the haplotype rapidly rose to high frequency before
		recombination could break down associations with nearby markers.
		SWEEP takes phased genotype data as input, detects all haplotype
		blocks in that data, and then determines the frequency and
		long-range LD for each allele in each block.</p>
		<li><p style="margin-bottom: 0.14in">authors: Patrick Varilly, Ben
		Fry, Oleg Shamovsky, Pardis Sabeti (email: sweep@broad.mit.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broad.mit.edu/mpg/sweep/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Sabeti, Reich,
		Higgins, Levine, Richter, Schaffner, Gabriel, Platko, Patterson,
		McDonald, Ackerman, Campbell, Altshuler, Cooper, Kwiatkowski, Ward,
		Lander (2002), &quot;Detecting recent positive selection in the
		human genome from haplotype structure&quot;, Nature, 419:832-827.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">SYZYGY</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: Syzygy is a
		targeted sequencing post processing analysis tool that allows: 1.
		SNP and indel detection; 2. Allele frequency estimation; 3.
		Single-marker association test; 4. Group-wise marker test
		association; 5. Experimental QC summary (%dbSNP, Ts/Tv, Ns/S); 6.
		Power to detect variant.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broadinstitute.org/software/syzygy/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Rivas et al.
		(2011), &quot;Deep resequencing of GWAS loci identifies independent
		rare variants associated with inflammatory bowel disease&quot;,
		Nature Genetics, in press.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TAGGER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Paul de Bakker
		(email: debakker@molbio.mgh.harvard.edu, Borad Institute)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.broad.mit.edu/mpg/tagger/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: de Bakker,
		Yelensky, Pe'er, Gabriel, Daly, Altshuler (2005), &quot;Efficiency
		and power in genetic association studies&quot;, Nature Genetics,
		37:1217-1223.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TAGIMPUTE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: TAGging-snp based
		IMPUTATE</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: tagIMPUTE is a
		command-line program for the imputation of untyped SNPs. tagIMPUTE
		is based on a few flanking SNPs that can optimally predict the SNP
		under imputation.</p>
		<li><p style="margin-bottom: 0.14in">authors: Yijuan Hu, Danyu Lin
		(Univ North Carolina)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bios.unc.edu/~lin/software/tagIMPUTE/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Hu, Lin (2010),
		&quot;Analysis of untyped SNPs: maximum likelihood and imputation
		methods&quot;, Genetic Epidemiology, 34:803-815.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TAGSNP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2 (beta)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Daniel O Stram
		(email: stram@usc.edu, Univ Southern California)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-rcf.usc.edu/~stram/tagSNPs.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN90</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Stram (2004), &quot;Tag
		SNP selection for association studies&quot;, Genetic Epidemiology,
		27(4):365-374.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TAGSTER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: TAGster is a tool
		to select, evaluate and visualize LD tag SNPs for single or
		multiple populations. The input files can be a set of dumped
		genotype files from International HapMap Project
		(http://www.hapmap.org/) (Hapmap format) or Seattle SNPs
		(http://pga.gs.washington.edu/) (Prettybase format). The ouput is a
		set of LD tag SNPs for single or multiple populations.</p>
		<li><p style="margin-bottom: 0.14in">authors: Zongli Xu, Norman L.
		Kaplan, Jack A Taylor</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.niehs.nih.gov/research/resources/software/tagster/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Perl, R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		MacOS, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Xu, Kaplan, Taylor
		(2007), &quot;TAGster: efficient selection of LD tag SNPs in single
		or multiple populations&quot;, Bioinformatics, 23:3254-3255.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TASSEL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Trait Analysis by
		aSSociation, Evolution, and Linkage</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0 (January 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: TASSEL is a
		software package which performs a variety of genetic analyses. The
		analyses include association mapping, diversity estimation and
		calculating linkage disequilibrium. The association analysis
		between genotypes and phenotypes can be performed by either a
		general linear model or a mixed linear model. The general linear
		model now allows users to analyze complex field designs,
		environmental interactions, and epistatic interactions. The mixed
		model is specially designed to handle polygenic effects at multiple
		levels of relatedness including pedigree information. These new
		analyses should permit association analysis in a wide range plant
		and animal species.</p>
		<li><p style="margin-bottom: 0.14in">authors: Peter J Bradbury,
		Edward S Buckler, Dallas E Kroon, Zhiwu Zhang (email:
		esb@cornell.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://sourceforge.net/projects/tassel/</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: web-based</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhiwu Zhang, Peter
		J Bradbury, Dallas E Kroon, Terry M Casstevens, Edward S Buckler
		(2006), &quot;TASSEL 2.0: a software package for association and
		diversity analyses in plants and animals&quot;, Plant &amp; Animal
		Genomes XIV Conference. [<a href="http://www.intl-pag.org/14/abstracts/PAG14_C012.html">abstract</a>]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TDT-AE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Transmission
		Disequilibrium Test Allowing for Errors</p>
		<li><p style="margin-bottom: 0.14in">version: 2.01 (July 2004)</p>
		<li><p style="margin-bottom: 0.14in">description: This program
		computes a likelihood-based transmission disequilibrium test. The
		data are genotypes on trios (father, mother, affected child) in
		which random genotyping errors leading to Mendelian inconsistencies
		may or may not have occurred. This program computes the TDT-AE
		statistic on all trios (whether Mendelianly consistent or not) and
		thereby maintains a correct type I error rate in the presence of
		random genotyping errors.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Chad Haynes, Derek
		Gordon (email: Gordon@dls.rutgers.edu), Xin Liu (for version 1)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		ftp://linkage.rockefeller.edu/software/tdtae2</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(2000/DOS), UNIX(Sun Solaris), LINUX (RedHat)</p>
		<li><p style="margin-bottom: 0.14in">executables: tdt-ae (PC),
		tdtae (UNIX and LINUX)</p>
		<li><p style="margin-bottom: 0.14in">reference: Gordon, Heath, Liu,
		Ott (2001), &quot;A transmission/disequilibrium test that allows
		for genotyping errors in the analysis of single-nucleotide
		polymorphism data&quot;, American Journal of Human Genetics,
		69(2):371-380. [ html]</p>
		<li><p style="margin-bottom: 0.14in">Gordon, Haynes, Johnnidis,
		Patel, Bowcock, Ott (2004), &quot;A transmission disequilibrium
		test for general pedigrees that is robust to the presence of random
		genotyping errors and any number of untyped parents&quot;, European
		Journal of Human Genetics, 12:752-761.</p>
		<li><p style="margin-bottom: 0.14in">User's Guide</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TDTASP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Power and
		Sample-Size Calculations for the TDT and ASP Tests</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (April 2003)</p>
		<li><p style="margin-bottom: 0.14in">description: Power and
		sample-size calculations for the TDT and ASP tests under a wide
		variety of ascertainment schemes. Uses the flexible genetic model
		of McGinnis. Most calculations are exact rather than asymptotic.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Barry W. Brown
		(bwb@mdanderson.org), Dan Serachitopol (dansera@mdanderson.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://biostatistics.mdanderson.org/SoftwareDownload/</p>
		<li><p style="margin-bottom: 0.14in">http://biostatistics.mdanderson.org/SoftwareDownload/SingleSoftware.aspx?Software_Id=20</p>
		<li><p style="margin-bottom: 0.14in">ftp://odin.mdacc.tmc.edu/pub/source/tdtasp_1.1.tar.gz
		(source)</p>
		<li><p style="margin-bottom: 0.14in">ftp://odin.mdacc.tmc.edu/pub/sun/tdtasp_1.0.tar.gz
		(Solaris Executable)</p>
		<li><p style="margin-bottom: 0.14in">ftp://odin.mdacc.tmc.edu/pub/win32/tdtasp_1.1_se.exe
		(Windows executable)</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		Fortran95</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: tdtasp</p>
		<li><p style="margin-bottom: 0.14in">reference: Brown (2004),
		&quot;Power calculations for the transmission/disequilibrium and
		affected sib pair tests using elementary probability methods&quot;,
		Genetic Research, 83(2):133-141.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TDTHAP (see R/TDTHAP)</p>
	<li><p style="margin-bottom: 0.14in">TDT-PC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Transmission
		Disequilibrium Test Power Calculator</p>
		<li><p style="margin-bottom: 0.14in">version: 1.3 (2002)</p>
		<li><p style="margin-bottom: 0.14in">description: TDT Power
		Calculator (TDT-PC) is a computer program to compute the
		statistical power of the Transmission/Disequilibrium Test (TDT)
		analytically, based on the most accurate asymptotic algorithms up
		to date, and is applicable in very general situations, where
		different parental disease status, multiple children, mixed family
		type and recombination events are considered. Routine algorithms
		for Monte Carlo simulations with significant improvements are also
		implemented in this program.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Wei-Min Chen (email:
		wc9c@virginia.edu) and Hong-Wen Deng (deng@creighton.edu) (John
		Hopkins)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://people.virginia.edu/~wc9c/TDTPC/Download.htm</p>
		<li><p style="margin-bottom: 0.14in">http://www.biostat.jhsph.edu/~wmchen/pc.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		MS-DOS, UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Chen, Deng (2001),
		Genetic Epidemiology, 21:53-67.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TDTPOWER</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0b (Aug 2001)</p>
		<li><p style="margin-bottom: 0.14in">description: calculates the
		sample size required for obtaining a prescribed power against a
		specified alternative for TDT.</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Knapp (Univ
		Bonn)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.uni-bonn.de/~umt70e/soft.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: SAS
		macro</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: M Knapp (1999), &quot;A
		note on power approximations for the transmission/disequilibrium
		test&quot;, American Journal of Human Genetics, 64(4):1177-1185</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TDT/S-TDT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Transmission
		Disequilibrium Test and Sib Transmission Disequilibrium Test</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1 (Feb 24, 1999)</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		provides separate results for TDT, S-TDT, and the combined
		(overall) test, as appropriate.</p>
		<li><p style="margin-bottom: 0.14in">Authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genomics.med.upenn.edu/spielman/TDT.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Spielman, McGinnis,
		Ewens (1993), &quot;Transmission test for linkage disequilibrium:
		the insulin gene region and insulin-dependent diabetes mellitus
		(IDDM)&quot;, American Journal of Human Genetics, 52(3):506-516</p>
		<li><p style="margin-bottom: 0.14in">Spielman, Ewens (1998), &quot;A
		sibship test for linkage in the presence of association: the sib
		transmission/disequilibrium test&quot;, American Journal of Human
		Genetics, 62(2):450-458</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TFPGA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Tools For
		Population Genetic Analyses</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: This program
		calculates descriptive statistics, genetic distances, and
		F-statistics. It also performs tests for Hardy-Weinberg
		equilibrium, exact tests for genetic differentiation, Mantel tests,
		and UPGMA cluster analyses.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mark Miller (email:
		mark.miller@usu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.marksgeneticsoftware.net/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">THESIAS</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Testing Haplotype
		EffectS In Association Studies</p>
		<li><p style="margin-bottom: 0.14in">version: 3.1 (March 2006)</p>
		<li><p style="margin-bottom: 0.14in">description: The objective of
		the THESIAS program is to perform haplotype-based association
		analysis in unrelated individuals. This program is based on a
		maximum likelihood model described in Tregouet et al. 2002 and is
		linked to the stochastic EM (SEM) algorithm. THESIAS allows the
		simultaneous estimation of haplotype frequencies and of their
		associated effects on the phenotype of interest. In its current
		version, both quantitative and qualitative phenotypes can be
		studied. Covariate-adjusted haplotype effects as well as haplotype
		x covariate interactions can be investigated.</p>
		<li><p style="margin-bottom: 0.14in">authors: David-Alexandre
		Tregouet (email: david.tregouet@chups.jussieu.fr)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://ecgene.net/genecanvas/downloads.php?cat_id=1</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TKMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: a program for
		drawing genetic maps</p>
		<li><p style="margin-bottom: 0.14in">authors: Humberto Ortiz
		Zuazaga (email: humberto@hpcf.upr.edu), Rosemarie Plaetke (
		rosi@momo.uthscsa.edu) (Univ Texas Health Science Ctr at San
		Antonio).</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hpcf.upr.edu/~humberto/software/TkMap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: based on
		bioTk which is based on Tcl/Tk</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: manual page</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TLINKAGE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: two-locus trait
		linkage programs</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: these are
		extensions of the LINKAGE programs that allow the disease phenotype
		to be under the control of disease two loci.</p>
		<li><p style="margin-bottom: 0.14in">authors: Lathrop, Schork,
		Boehnke, Terwilliger, Ott</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Lathrop, Ott
		(1990), American Journal of Human Genetics, 47(supp):A188.</p>
		<li><p style="margin-bottom: 0.14in">Schork, Boehnke, Terwilliger,
		Ott (1993), &quot;Two-trait-locus linkage analysis: a powerful
		strategy for mapping complex genetic traits&quot;, American Journal
		of Human Genetics, 53(5):1127-1136</p>
		<li><p style="margin-bottom: 0.14in">online documentation</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TOMCAT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (December 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		implements the Mantel statistics as proposed by Beckmann et al.
		(2005) to test for association between genetic markers and
		phenotypes in case-control studies using haplotype information. The
		potential value of haplotypes has attracted widespread interest in
		the mapping of complex traits. Haplotype sharing methods take into
		account linkage disequilibrium information between multiple
		markers, and may have good power to detect predisposing genes. We
		present a new approach based on Mantel statistics for space time
		clustering, which we developed in order to improve the power of
		haplotype sharing analysis for gene mapping in complex disease. The
		new statistic correlates genetic similarity and phenotypic
		similarity across pairs of haplotypes for case-only and
		case-control studies. The genetic similarity is measured as the
		shared length between haplotypes around a putative disease locus.
		Alternative measures for the phenotypic similarity were
		implemented.</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Rabes,
		Matthias Schmitt, Lars Beckmann (email: l.beckmann@dkfz.de)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.dkfz.de/en/epidemiologie-krebserkrankungen/software/software.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java 5.0</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables: tomcat1.0.jar</p>
		<li><p style="margin-bottom: 0.14in">reference: Beckmann, Thomas,
		Fischer, Chang-Claude (2005), &quot;Haplotype sharing analysis
		using mantel statistics&quot;, Human Heredity, 59(2):67-78.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TRANSMIT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.5.4</p>
		<li><p style="margin-bottom: 0.14in">description: TRANSMIT tests
		for association between genetic marker and disease by examining the
		transmission of markers from parents to affected offspring. The
		main features which differ from other similar programs are: (1) It
		can deal with transmission of multi-locus haplotypes, even if phase
		is unknown, and (2) Parental genotypes may be unknown.</p>
		<li><p style="margin-bottom: 0.14in">Authors: David Clayton
		(Cambridge Institute, david.clayton@cimr.cam.ac.uk</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www-gene.cimr.cam.ac.uk/clayton/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: transmit</p>
		<li><p style="margin-bottom: 0.14in">reference: online
		documentation</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TRAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Tiled Regression
		Analysis Package</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: a software tool
		for determining a regression model of quantitative or binary trait
		variation when the number of possible genetic predictors is very
		large, considering only a moderate number of predictors at one
		time, using unrelated or family data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Alexander F. Wilson
		(afw@mail.nih.gov), Yoonhee Kim, Heejong Sung, Juanling Cai, Alexa
		J. M. Sorant (ajms@mail.nih.gov)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://research.nhgri.nih.gov/software/TRAP/</p>
		<li><p style="margin-bottom: 0.14in">ftp://ftp.nhgri.nih.gov/pub/software/TRAP/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: any with R
		installation</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Wilson, Kim, Sung,
		Cai, McMahon, Sorant (2009), &quot;Tiled Regression: the use of
		regression methods in hotspot defined genomic segments to identify
		independent genetic variants responsible for variation in
		quantitative traits&quot;, Abstract 142. In: &quot;Abstracts from
		the Eighteenth Annual Meeting of the International Genetic
		Epidemiology Society&quot;, Genetic Epidemiology, 33:793.</p>
		<li><p style="margin-bottom: 0.14in">Sorant, Cai, Sung, Kim, Wilson
		(2010), &quot;Tiled Regression Analysis Package (TRAP): software
		implementation of tiled regression methodology&quot;, Abstract 244.
		In: &quot;Abstracts from the nineteenth annual meeting of the
		International Genetic Epidemiology Society&quot;, Genetic
		Epidemiology, 34:984-985.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TREELD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0 (Jan 2005)</p>
		<li><p style="margin-bottom: 0.14in">description: The package
		TreeLD is a free software tool for mapping complex trait loci.
		TreeLD performs a multipoint LD-analysis by inferring the ancestry
		of a genomic region and analyzing this ancestry for signals of
		disease mutations. The generated likelihoods can be used to test
		for the presence of a disease locus and to fine-map its location,
		providing a point estimate and a credible region. Furthermore, the
		package provides a novel way of visualizing the association signal
		in a sample. TreeLD is designed for high-density SNP haplotypes and
		can be applied to case-control data, TDT trio data and quantitative
		trait data.</p>
		<li><p style="margin-bottom: 0.14in">authors: Sebastian Zollner,
		Xiaoquan Wen, Jonathan Pritchard</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pritch.bsd.uchicago.edu/treeld.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Solaris/..), Linux, MS-Windows(95/98/NT/2000/XP)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zollner, Pritchard
		(2004), &quot;Coalescent-based association mapping and fine mapping
		of complex trait loci&quot;, Genetics, 169:1071-1092.</p>
		<li><p style="margin-bottom: 0.14in">Zollner, Wen, Pritchard
		(2005), &quot;Association mapping and fine mapping with TreeLD&quot;,
		Bioinformatics, 21(14):3168-3170.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TREESCAN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 0.9</p>
		<li><p style="margin-bottom: 0.14in">description: TREESCAN is
		intended to provide p-values for the hypothesis of association
		between evolutionary clades and continuous traits, using haplotype
		trees.</p>
		<li><p style="margin-bottom: 0.14in">authors: David Posada (email:
		dposada@uvigo.es)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://darwin.uvigo.es/software/treescan.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MS-DOS, MacOS(X)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Posada, Maxwell,
		Templeton (2005), &quot;TreeScan: a bioinformatic application to
		search for genotype/phenotype associations using haplotype trees&quot;,
		Bioinformatics, 21(9):2130-2132.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TREESELECT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: TreeSelect is a
		software package for inferring natural selection from unusual
		population differentiation between closely related populations.</p>
		<li><p style="margin-bottom: 0.14in">authors: Alkes Price (Harvard
		Univ)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.hsph.harvard.edu/faculty/alkes-price/software/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Bhatia et al.
		(2011), &quot;Genome-wide comparison of African-ancestry
		populations from CARe and other cohorts reveals signals of natural
		selection&quot;, American Journal of Human Genetics, 89(3):368-381.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TRIMHAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: TRIMmed-HAPlotype
		(previously named HAL: Haplotype ALgorithm)</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2</p>
		<li><p style="margin-bottom: 0.14in">description: linkage
		disequilibrium mapping based on ancestral founder haplotypes.
		Method uses haplotype data from general pedigrees.</p>
		<li><p style="margin-bottom: 0.14in">Authors: Rory Martin and
		Charles MacLean (Virginia Commonwealth University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.vipbg.vcu.edu/trimhap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: FORTRAN</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables: hal</p>
		<li><p style="margin-bottom: 0.14in">reference: MacLean, Martin,
		Sham, Straub, Kendler (2000), &quot;The trimmed-haplotype test for
		linkage disequilibrium&quot;, American Journal of Human Genetics,
		66(3):1062-1075. [ html]</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TUNA</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Testing UNtyped
		Alleles</p>
		<li><p style="margin-bottom: 0.14in">version: 1.1.2 beta (July
		2007)</p>
		<li><p style="margin-bottom: 0.14in">description: TUNA is a free
		software package for estimating allele frequencies for all the SNPs
		available in a population reference panel (e.g. HapMap data base)
		based on genotype data from a subset of markers (e.g. the Illumina
		HumanHap300 BeadChip SNP set) in a group of subjects (e.g. the
		cases in a case-control sample).</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.stat.uchicago.edu/~wen/tuna/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Nicolae (2006),
		&quot;Quantifying the amount of missing information in genetic
		association studies&quot;, Genetic Epidemiology, 30(8):703-717.</p>
		<li><p style="margin-bottom: 0.14in">Nicolae (2006), &quot;Testing
		untyped alleles (TUNA) - applications to genome-wide association
		studies&quot;, Genetic Epidemiology, 30(8):718-727.</p>
		<li><p style="margin-bottom: 0.14in">Wen, Nicolae (2007),
		&quot;Case-control association studies of untyped markers with
		TUNA&quot;, Bioinformatics, 24(3):435-437.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">2DMAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: constructing 2-d
		crossover-based map</p>
		<li><p style="margin-bottom: 0.14in">authors: Edward Kort (
		edward@episun2.med.utah.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.genlink.wustl.edu/software</p>
		<li><p style="margin-bottom: 0.14in">ftp://mapper.wustl.edu/pub/two-d</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables: pic-2d, 2d-map,
		2d-color</p>
		<li><p style="margin-bottom: 0.14in">reference: Fain, Kort, Chance,
		Nguyen, Redd, Econ, Barker (1995), Nature Genetics, 9(3):261-265.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">2LD</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2002</p>
		<li><p style="margin-bottom: 0.14in">description: 2LD is a simple
		program calculating linkage disequilibrium (LD) measures between
		two polymorphic markers.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jinghua Zhao (MRC
		Epidemiology Unit at Strangeways Research Lab, UK) (email:
		jinghua.zhao@mrc-epid.cam.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-epid.cam.ac.uk/Personal/jinghua.zhao/software.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: C</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Zhao (2004), &quot;2LD,
		GENECOUNTING and HAP: computer programs for linkage disequilibrium
		analysis&quot;, Bioinformatics, 20(8):1325-1326.</p>
		<li><p style="margin-bottom: 0.14in">Zapata, Carollo, Rodriguez
		(2001), &quot;Sampling variance and distribution of the D' measure
		of overall gametic disequlibrium between multiallelic loci&quot;,
		Annals of Human Genetics, 65:395-406.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">TWOLOC</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: v2 (May 1998)</p>
		<li><p style="margin-bottom: 0.14in">description: a software
		package for analyzing two-locus susceptibility gene models in
		affected sib-pair data</p>
		<li><p style="margin-bottom: 0.14in">Authors: Martin Farrall
		(Wellcome Trust Centre for Human Genetics, mfarrall@well.ox.ac.uk)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.well.ox.ac.uk/~mfarrall/twoloc.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language:
		FORTRAN77, Pascal, awk</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(Saloris/DEC UNIX/IRIX/..)</p>
		<li><p style="margin-bottom: 0.14in">executables: twoloc,
		calc_alpha, mkp, calc_ibd</p>
		<li><p style="margin-bottom: 0.14in">reference: Farrall (1997),
		Genetic Epidemiology, 14:103-115.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">2SNP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.5 (Aug 4, 2006)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Alexander Zelikovsky
		(email: alexz@cs.gsu.edu), Dumitru Brinza (email: dima@cs.gsu.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://alla.cs.gsu.edu/~software/2SNP/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Brinza and
		Zelikovsky (2006), &quot;2SNP: scalable phasing based on 2-SNP
		haplotypes&quot;, Bioinformatics, 22(3):371-373.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">UMAKE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: UMAKE is a
		software pipeline to detect SNPs and call their genotypes from a
		list of BAM files. UMAKE pipeline has been successfully applied in
		detecting SNPs from many large-scale next-generation sequencing
		studies.</p>
		<li><p style="margin-bottom: 0.14in">authors: Hyun Min Kang, Goo
		Jun, Carlo Sidore, Yun Li, Paul Anderson, Mary Kate Trost, Wei
		Chen, Tom Blackwell, Goncalo Abecasis (Univ Michigan)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.sph.umich.edu/wiki/UMAKE</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">UNKNOWN (see &quot;LINKAGE -
	general pedigrees&quot;).</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">reference: a relevant
		documentation is the mystery of (the) unknown</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">UNPHASED</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 3.0.13 (May 2008)</p>
		<li><p style="margin-bottom: 0.14in">description: UNPHASED is a
		suite of programs for association analysis of multilocus haplotypes
		from unphased genotype data. These include TDTPHASE for case-parent
		trios, COCAPHASE for case/control data, QTPHASE for quantitative
		traits in unrelateds, PDTPHASE for general pedigrees, and QPDTPHASE
		for quantitative traits in general pedigrees. Features include
		global and individual haplotype tests, main effects and conditional
		tests, grouping of rare haplotypes, pairwise comparisons of
		haplotype risk, flexible permutation procedures and calculation of
		LD measures.</p>
		<li><p style="margin-bottom: 0.14in">authors: Frank Dudbridge (MRC
		Human Genome Mapping Project Resource Centre) (email:
		frank.dudbridge@mrc-bsu.cam.ac.uk</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.mrc-bsu.cam.ac.uk/personal/frank/software/unphased/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++,
		Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:
		UNIX(SOlaris), Linux, MS-Windows</p>
		<li><p style="margin-bottom: 0.14in">executables: unphased,
		tdtphase, cocaphase, pdtphase, qpdtphase</p>
		<li><p style="margin-bottom: 0.14in">reference: F Dudbridge (2003),
		&quot;Pedigree disequilibrium tests for multilocus haplotypes&quot;,
		Genetic Epidemiology, 25:115-121.</p>
		<li><p style="margin-bottom: 0.14in">Dudbridge (2008),
		&quot;Likelihood-based association analysis for nuclear families
		and unrelated subjects with missing genotype data&quot;, Human
		Heredity, 66:87-98.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">UTIL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: statistical
		genetics UTILity programs</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Jurg Ott (Rockefeller
		University, email:ott@rockefeller.edu)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://lab.rockefeller.edu/ott/programs</p>
		<li><p style="margin-bottom: 0.14in">source code language: Pascal</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux</p>
		<li><p style="margin-bottom: 0.14in">executables: multiple, binom,
		chiprob, conting, equiv, hist, hwe, ibd, mapfun, nocom, norinv,
		norprob, par, pvalues, relrisk, sexdif, tprob, trend</p>
		<li><p style="margin-bottom: 0.14in">reference: Ott (1999),
		Analysis of human genetic linkage 3rd edition (Johns Hospkins
		University Press, Baltimore).</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">VAAST</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Variant Annotation,
		Analysis and Search Tool</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: VAAST is a
		probabilistic search tool for identifying damaged genes and their
		disease-causing variants in personal genome sequences. VAAST builds
		upon existing amino acid substitution (AAS) and aggregative
		approaches to variant prioritization, combining elements of both
		into a single unified likelihood-framework that allows users to
		identify damaged genes and deleterious variants with greater
		accuracy, and in an easy-to-use fashion. VAAST can score both
		coding and non-coding variants, evaluating the cumulative impact of
		both types of variants simultaneously. VAAST can identify rare
		variants causing rare genetic diseases, and it can also use both
		rare and common variants to identify genes responsible for common
		diseases. VAAST thus has a much greater scope of use than any
		existing methodology.</p>
		<li><p style="margin-bottom: 0.14in">authors: Mark Yandell's group
		(Univ Utah)</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://www.yandell-lab.org/software/vaast.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Yandell, Huff, Hu,
		Singleton, Moore, Xing, Jorde, Reese (2011), &quot;A probabilistic
		disease-gene finder for personal genomes&quot;, Genome Research,
		21(9):1529-1542.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">VARSCAN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: VarScan is a
		platform-independent, technology-independent software tool for
		identifying SNPs and indels in massively parallel sequencing of
		individual and pooled samples. Given data for a single sample,
		VarScan identifies and filters germline variants based on read
		counts, base quality, and allele frequency. Given data for a
		tumor-normal pair, VarScan also determines the somatic status of
		each variant (Germline, Somatic, or LOH) by comparing read counts
		between samples.</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://genome.wustl.edu/software/varscan</p>
		<li><p style="margin-bottom: 0.14in">source code language: Java</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Koboldt, Chen,
		Wylie, Larson, McLellan, Mardis, Weinstock, Wilson, Ding (2009),
		&quot;VarScan: variant detection in massively parallel sequencing
		of individual and pooled samples&quot;, Bioinformatics,
		25:2283-2285.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">VG</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Visual Genotype</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0</p>
		<li><p style="margin-bottom: 0.14in">description: The program
		presents complete raw datasets of individuals' genotype data using
		a display format with samples as rows and polymorphisms as columns.
		The color code is: (1) blue: homozygous genotype for the common
		allele; (2) red: heterozygous genotype; (3) yellow: homozygous
		genotype for the rare allele; and (4) grey: missing data</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pga.gs.washington.edu/VG2.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Nickerson, Taylor,
		Weiss, Clark, Hutchinson, Stengard, Salomaa, Vartiainen,
		Boerwinkle, Sing (1998), &quot;DNA sequence diversity in a 9.7-kb
		region of the human lipoprotein lipase gene&quot;, Nature Genetics,
		19:233-240.</p>
		<li><p style="margin-bottom: 0.14in">Rieder, Taylor, Clark,
		Nickerson (1999), &quot;Sequence variation in the human angiotensin
		converting enzyme&quot;, Nature Genetics, 22:59-60.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">VH</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:visual haplotype</p>
		<li><p style="margin-bottom: 0.14in">version: 1.0</p>
		<li><p style="margin-bottom: 0.14in">description: displaying
		estimated haplotype data</p>
		<li><p style="margin-bottom: 0.14in">authors:</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://pga.gs.washington.edu/VH1.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Nickerson, Taylor,
		Weiss, Clark, Hutchinson, Stengard, Salomaa, Vartiainen,
		Boerwinkle, Sing (1998), &quot;DNA sequence diversity in a 9.7-kb
		region of the human lipoprotein lipase gene&quot;, Nature Genetics,
		19:233-240.</p>
		<li><p style="margin-bottom: 0.14in">Rieder, Taylor, Clark,
		Nickerson (1999), &quot;Sequence variation in the human angiotensin
		converting enzyme&quot;, Nature Genetics, 22:59-60.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">VITESSE</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: VITESSE (means
		&quot;speed&quot; in French)</p>
		<li><p style="margin-bottom: 0.14in">version: 2.0beta (July 1998)</p>
		<li><p style="margin-bottom: 0.14in">description: calculates
		likelihood on pedigrees</p>
		<li><p style="margin-bottom: 0.14in">authors: Jeff O'Connell (
		jeff@sherlock.hgen.pitt.edu),</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://watson.hgen.pitt.edu/register/soft_doc.html,</p>
		<li><p style="margin-bottom: 0.14in">http://watson.hgen.pitt.edu/register</p>
		<li><p style="margin-bottom: 0.14in">source code language: ANSI C</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX, VMS,
		MS-DOS</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Am. Soc. Hum.
		Genet. Annual Meeting 1995 (O'Connell,Weeks, Am. J. Hum. Genet.
		suppl 57, A199 (1995)).</p>
		<li><p style="margin-bottom: 0.14in">O'Connell,Weeks (1995), Nature
		Genetics, 11:402-408.</p>
		<li><p style="margin-bottom: 0.14in">online README file</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">WEB-PREPLINK</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">This was a simple program
		running on our old <i>linkage</i> server at Rockefeller University.
		However, that server has been <a href="http://lab.rockefeller.edu/ott/">updated</a>
		and no longer supports running scripts, so web-preplink is no
		longer available.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">WEBQTL</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description: An interactive
		web site useful for exploring the genetic modulation of thousands
		of phenotypes gathered over a 30-year period by hundreds of
		investigators using reference panels of recombinant inbred strains
		of mice. WebQTL includes dense error-checked genetic maps, as well
		as extensive gene expression data sets (Affymetrix) acquired across
		more than 35 strains of mice. WebQTL accepts user-entered traits
		for BXD, AXB/BXA, CXB, BXH, AKXD recombinant inbred strains.</p>
		<li><p style="margin-bottom: 0.14in">authors: Jintao Wang, Elissa
		J. Chesler, Lu Lu, Jing Gu, Yanhua Qu, Robert W. Williams, (email:
		rwilliams@nb.utmem.edu) Kenneth F. Manly (email:kmanly@utmem.edu)
		(Univ of Tennessee Health Science Center)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.webqtl.org/search.html</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++ and
		Python</p>
		<li><p style="margin-bottom: 0.14in">operating systems: all via
		internet</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: J Wang, RW
		Williams, EJ Chesler, KF Manly (2003), &quot;WebQTL: Web-based
		complex trait analysis&quot;, Neuroinformatics 1(4):299-308.</p>
		<li><p style="margin-bottom: 0.14in">EJ Chesler, J Wang, L Lu, Y
		Qu, KF Manly, RW Williams (2003), &quot;Genetic correlates of gene
		expression in recombinant inbred strains: a relational model system
		to explore neurobehavioral phenotypes&quot;, Neuroinformatics,
		1(4):343-357.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">WEIGHTED_FDR</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 1.2 (March 2006)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Lambertus Klei
		(University of Pittsburgh Medical Center), Kathryn Roeder (Carnegie
		Mellon University)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://wpicr.wpic.pitt.edu/WPICCompGen/fdr/</p>
		<li><p style="margin-bottom: 0.14in">source code language: R</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows,
		Linux, UNIX</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">WHAIT</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name: Weighted Haplotype
		test And an Imputation-based Test.</p>
		<li><p style="margin-bottom: 0.14in">version:</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Yun Li (email:
		ylwtx@umich.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.sph.umich.edu/csg/yli/whait/</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems:</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">WHAP</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 2.06 (Dec 2004)</p>
		<li><p style="margin-bottom: 0.14in">description:</p>
		<li><p style="margin-bottom: 0.14in">authors: Shaun Purcell (email:
		purcell@wi.mit.edu), Pak Sham</p>
		<li><p style="margin-bottom: 0.14in">web:
		http://pngu.mgh.harvard.edu/~purcell/whap/</p>
		<li><p style="margin-bottom: 0.14in">source code language: C,C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: UNIX,
		MS-Windows, MS-DOS, Linux</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference:</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">WHICHRUN</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: 4.1</p>
		<li><p style="margin-bottom: 0.14in">description: uses multilocus
		genotypic data to allocate individuals to their most likely source
		population</p>
		<li><p style="margin-bottom: 0.14in">authors: Michael Banks,
		(email: michael.banks@hmsc.orst.edu), Will Eichert (email:
		wfeichert@ucdavis.edu)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.bml.ucdavis.edu/whichrun.htm</p>
		<li><p style="margin-bottom: 0.14in">source code language: C++</p>
		<li><p style="margin-bottom: 0.14in">operating systems: MS-Windows
		(95/98/NT)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: Banks, Eichert
		(2000), &quot;WHICHRUN (version 3.2): a computer program for
		population assignment of individuals based on multilocus genotype
		data&quot;, Journal of Heredity, 91:87-89.</p>
	</ul>
	<li><p style="margin-bottom: 0.14in">ZAPLO</p>
	<ul>
		<li><p style="margin-bottom: 0.14in">full name:</p>
		<li><p style="margin-bottom: 0.14in">version: Oct 5, 2005</p>
		<li><p style="margin-bottom: 0.14in">description: zero recombinant
		haplotyping</p>
		<li><p style="margin-bottom: 0.14in">authors: Jeff O'Connell
		(email: joconnel@medicine.umaryland.edu,
		zaplo@molecular-haplotype.org)</p>
		<li><p style="margin-bottom: 0.14in">web/ftp:
		http://www.molecular-haplotype.org/zaplo/zaplo_index.html</p>
		<li><p style="margin-bottom: 0.14in">source code language:</p>
		<li><p style="margin-bottom: 0.14in">operating systems: Linux,
		UNIX(Solaris)</p>
		<li><p style="margin-bottom: 0.14in">executables:</p>
		<li><p style="margin-bottom: 0.14in">reference: O'Connell (2000),
		&quot;Zero-recombinant haplotyping: applications to fine mapping
		using SNPs,&quot;, Genetic Epidemiology, 19:S64-S70.</p>
	</ul>
</ul>
</body>
</html>